{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'865eda9aa6b27bd84ff8832558fce532'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import requests\n",
    "from dotenv import load_dotenv\n",
    "import os\n",
    "from bs4 import BeautifulSoup\n",
    "import lxml\n",
    "\n",
    "# Load variables from .env file\n",
    "load_dotenv()\n",
    "\n",
    "# Access the API key\n",
    "SPRINGER_API_KEY = os.getenv(\"SPRINGER_API_KEY\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n",
       "<?xml-stylesheet type=\"text/xsl\" href=\"/resources/spdi-openaccess-jats.xsl\"?><!DOCTYPE response>\n",
       "<response><apiMessage>This XML was provided by Springer Nature</apiMessage><query>subject:Chemistry</query><apiKey>865eda9aa6b27bd84ff8832558fce532</apiKey><nextPage>/openaccess/jats?q=subject:Chemistry&amp;s=11&amp;p=10&amp;api_key=865eda9aa6b27bd84ff8832558fce532</nextPage><result><total>62325</total><start>1</start><pageLength>10</pageLength><recordsDisplayed>10</recordsDisplayed></result><records><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02810-7</article-id><article-id pub-id-type=\"manuscript\">2810</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02810-7</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Evaluation of acyclovir adsorption on granular activated carbon from aqueous solutions: batch and fixed-bed parametric studies</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Gupta</surname><given-names>Anju</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-9330-1539</contrib-id><name><surname>Vyas</surname><given-names>Raj K.</given-names></name><address><email>rkvyas.chem@mnit.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023028107_cor2\">b</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444471.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1764 2536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering</institution><institution content-type=\"org-name\">Malaviya National Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">302017</addr-line><addr-line content-type=\"city\">Jaipur</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023028107_cor2\"><label>b</label><email>rkvyas.chem@mnit.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>13</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"40\">8</issue><fpage>4599</fpage><lpage>4612</lpage><history><date date-type=\"registration\"><day>3</day><month>4</month><year>2023</year></date><date date-type=\"received\"><day>13</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>1</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>13</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><sec><p id=\"Par1\">The present study is aimed to assess the adsorptive potential of carbonaceous material for the acyclovir (ACVR) removal from the aquatic environment using batch and fixed-bed processes. In batch mode, the impact of various process conditions (contact time, pH, adsorbent dose, initial ACVR concentration, and temperature) on ACVR adsorption was investigated. Experimental results revealed that Langmuir isotherm and the pseudo-second-order kinetic model adequately represent the ACVR adsorption mechanism, indicating homogeneous adsorption. The process was found exothermic and spontaneous. Thermodynamic studies concluded that adsorption is a result of both physisorption and chemisorption. To understand the dynamic regime for the design of large-scale column studies, experimental data obtained from breakthrough curve were fitted to various analytical kinetic models. Yan model followed by Thomas model demonstrated a greater correlation of breakthrough data, confirming that the results are significant and are in line with Langmuir isotherm and pseudo-second-order kinetic. G-AC exhibits sufficient adsorption capacity for ACVR. Hence, it is concluded that it can be used in a fixed-bed column in continuous mode for the treatment of ACVR-contaminated wastewater.</p></sec><sec id=\"ASec2\"><title>Graphical abstract</title><p id=\"Par2\"><fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"PNG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Figa_HTML.png\"/></fig></p></sec></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Granular material</kwd><kwd>Kinetics</kwd><kwd>Isotherms</kwd><kwd>Fixed-bed</kwd><kwd>Breakthrough curve</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2810.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02810-7\">https://doi.org/10.1007/s11696-023-02810-7</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par48\">Over the past few decades, the new strains of life-threatening pandemic viruses, including Nipah virus, severe acute respiratory syndrome (SARS) virus, Middle East respiratory syndrome (MERS), and the most recent one, coronavirus disease (COVID-19), have hit the world. Therefore, the viral infections are of the global concern. Researchers are exploring new promising antibodies, antiviral treatments, and ways to improve the efficacy of existing medications on a laboratory scale (Al-Otaibi et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>; Heidary et al. <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>; Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). Antiviral drugs have received a lot of attention recently (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR30\">2013</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). Acyclovir (ACVR) (Fig. S1), a synthetically prepared purine-based nucleoside analog, shows antiviral activities for the treatment of herpes simplex virus (HSV), hepatitis B, varicella-zoster virus, Epstein-Barr virus, chickenpox, etc. (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). ACVR is usually favored over other drugs because of its low cytotoxicity and high selectivity towards infected cells (Shetti et al. <xref ref-type=\"bibr\" rid=\"CR54\">2020</xref>). More than two-thirds of the population is infected by α-HSV (Tognarelli et al. <xref ref-type=\"bibr\" rid=\"CR57\">2019</xref>). ACVR reaches drinking water via several routes from domestic/clinic waste, bulk and formulation drug industries, etc. (Gupta et al. <xref ref-type=\"bibr\" rid=\"CR25\">2021</xref>). The simultaneous presence of a pertinent virus in living beings and the modest amount of ACVR can render the virus drug-resistant (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Its presence has been found in industrial effluents, surface water, drinking water, and wastewater influent/effluent stream(s) (Funke et al. <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>; Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR47\">2010</xref>; Mascolo et al. <xref ref-type=\"bibr\" rid=\"CR41\">2010</xref>). Even though ACVR is present at a low concentration ranging from ng/L to μg/L (Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>), its persistent nature and harmful effects on the aquatic environment necessitate the treatment of contaminated water (Russo et al. <xref ref-type=\"bibr\" rid=\"CR51\">2017</xref>). A high therapeutic dose is mainly recommended as it only lasts a brief time in the human body (Saifi et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>). As a result, water contamination and possible environmental issues with the widespread usage of ACVR should be seriously considered. Remediation of industrial wastewater before its discharge is more economical than treating low concentrated large volume of water among preventive routes. Consequently, to prevent the adverse effects of ACVR on the ecosystem, ACVR polluted wastewater must be treated before being discharged.</p><p id=\"Par49\">Adsorption (Hosseini Nasr et al. <xref ref-type=\"bibr\" rid=\"CR29\">2018</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>, <xref ref-type=\"bibr\" rid=\"CR32\">2014b</xref>), UV/H<sub>2</sub>O<sub>2</sub> (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>), catalytic ozonation (Bing et al. <xref ref-type=\"bibr\" rid=\"CR5\">2017</xref>), photocatalysis (Wu et al. <xref ref-type=\"bibr\" rid=\"CR58\">2018</xref>; Li et al. <xref ref-type=\"bibr\" rid=\"CR35\">2016</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR10\">2014</xref>), photolysis (Zhou et al. <xref ref-type=\"bibr\" rid=\"CR62\">2015</xref>), ozonation (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>), and biological treatment techniques (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR61\">2017</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>) have been attempted so far till date for the removal of ACVR from wastewater. AOP using oxone has been proven efficient (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>, <xref ref-type=\"bibr\" rid=\"CR38\">2020</xref>); however, the high cost of oxone used in the process makes it un-economical. Oxidation and biodegradation processes lead to more toxic intermediates like COFA and carboxy-ACV (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>). Bio-refractory compounds like ACVR on degradation get converted into more toxic intermediate product(s). For such compounds, adsorption appears to be the most efficient approach, as it does not degrade the compound. Most adsorption studies are carried out in batches, which do not provide precise scale-up information required for practical large-scale treatment. So far column studies on ACVR removal from wastewater have been scarcely reported.</p><p id=\"Par50\">Considering its high surface area and porous structure, granular activated carbon (G-AC) is an efficient adsorbent for ACVR. Hence, it is a viable treatment solution for eliminating ACVR from water. Additionally, G-AC is cost-effective, environmentally benign and easily available and can be used in a variety of treatment systems, including fixed-bed columns. Therefore, the present research was aimed to evaluate the performance of G-AC column for ACVR removal from the synthetically prepared aqueous solution of ACVR. The batch studies were also performed to know the effects of operating parameters on the removal efficiency of ACVR. The breakthrough curves were used to characterize the dynamic response of the fixed-bed column. Effects of initial ACVR concentration, G-AC-bed height, and flow rate on ACVR adsorption were evaluated as part of the column studies along with the analytical model studies. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p></sec><sec id=\"Sec2\"><title>Experimental</title><p id=\"Par51\">Chemicals/reagents, solution preparation, and analytical techniques used are described in supplementary file.</p><sec id=\"Sec3\"><title>Adsorbent characterization</title><p id=\"Par52\">The Brunauer–Emmett–Teller (BET) method was used to determine the specific surface area, pore volume, and pore size distribution of G-AC (Nova Touch LX2 gas sorption analyzer, Quantachrome Instruments). The scanning electron microscope (SEM) and energy-dispersive X-ray spectroscopy (EDX) were used to perform morphological characterization of the raw and spent G-AC sample (Nova Nanosem 450, FEITM).</p></sec><sec id=\"Sec4\"><title>Batch experiments</title><p id=\"Par53\">For determining the maximum removal of ACVR (dosage form, make: Ocuvir, FDC Limited, India) batch adsorption studies were conducted at varying initial concentrations of ACVR, adsorbent dose, pH of the solution, and temperature. A pre-determined amount of G-AC (make: Loba Chemie, India, LR grade) was mixed with 100 mL of synthetically prepared aqueous ACVR solution of known concentration for each group of experiments. Each batch was agitated in an orbital shaker at a fixed temperature and RPM. The solution was filtered after completion of each set, and the residual ACVR concentration was measured in a UV–Vis spectrophotometer.</p><p id=\"Par54\">The adsorption capacity of the G-AC at the time (t min) and the percent removal of ACVR were estimated using the following equations, respectively:<disp-formula id=\"Equa\"><alternatives><mml:math display=\"block\" id=\"Equa_Math\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mi>t</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mi>V</mml:mi></mml:mrow><mml:mi>W</mml:mi></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equa_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$q_{t} = \\frac{{\\left( {C_{0} - C_{f} } \\right) \\times V}}{W}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equa.gif\"/></alternatives></disp-formula><disp-formula id=\"Equb\"><alternatives><mml:math display=\"block\" id=\"Equb_Math\"><mml:mrow><mml:mo>%</mml:mo><mml:mrow><mml:mtext>Removal</mml:mtext><mml:mspace width=\"0.333333em\"/></mml:mrow><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mn>100</mml:mn></mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equb_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\% {\\text{Removal }} = \\frac{{\\left( {C_{O} - C_{f} } \\right) \\times 100}}{{C_{O} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equb.gif\"/></alternatives></disp-formula></p><p id=\"Par55\">Following equations were used to calculate standard deviation percent (SD%) and root-mean-square error (RMSE):<disp-formula id=\"Equc\"><alternatives><mml:math display=\"block\" id=\"Equc_Math\"><mml:mrow><mml:mtext>SD</mml:mtext><mml:mo>%</mml:mo><mml:mo>=</mml:mo><mml:mn>100</mml:mn><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub></mml:mrow></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equc_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{SD}}\\% = 100\\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)/q_{\\exp } } \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - 1}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equc.gif\"/></alternatives></disp-formula><disp-formula id=\"Equd\"><alternatives><mml:math display=\"block\" id=\"Equd_Math\"><mml:mrow><mml:mtext>RMSE</mml:mtext><mml:mo>=</mml:mo><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mtext>parameters</mml:mtext></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equd_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{RMSE}} = \\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)} \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - {\\text{parameters}}}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equd.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec5\"><title>Kinetic studies</title><p id=\"Par56\">Several kinetic models can test the experimental results and analyze the kinetic mechanism that governs the adsorption rate based on ACVR diffusion in adsorbent particles. The present work used pseudo-first and second-order, Elovich, and intra-particle diffusion kinetic models to evaluate experimental results. A batch size of 100 mL of 100 mg/L concentrated solution was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C.</p><p id=\"Par57\">Pseudo-first-order kinetic model deals with the adsorption potential of the adsorbent against an adsorbate. According to the pseudo-second-order kinetic model, physisorption and chemisorption contribute to adsorption (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>). Elovich model explains chemisorption (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). This model shows the declining adsorption rate with time as the vacant adsorbent sites get occupied by adsorbate molecules and as a result surface coverage increases (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The intraparticle diffusion model explains the adsorption mechanism. It is generally the last step to know about adsorption and type of diffusion within the adsorbent. Model equations used in kinetic studies are summarized in Table S1.</p><p id=\"Par58\">The low C value represents the boundary layer’s minimal influence (diffusional resistance) on adsorption (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The initial rate of adsorption h (mg/g/min) is determined using the following relation:<disp-formula id=\"Eque\"><alternatives><mml:math display=\"block\" id=\"Eque_Math\"><mml:mrow><mml:mi>h</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>k</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msubsup><mml:mi>q</mml:mi><mml:mrow><mml:mi>e</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:math><tex-math id=\"Eque_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$h = k_{2} q_{e}^{2}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Eque.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec6\"><title>Equilibrium studies</title><p id=\"Par59\">Adsorption equilibrium studies were carried out to study the adsorption phenomenon and to assess the ACVR interaction behavior with the adsorbent surface. For which solution with varying concentration was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C. Temkin, Freundlich, and Langmuir isotherm models were used to examine the equilibrium parameters. Freundlich isotherm reveals the heterogeneous nature of the sorption process (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). On the other hand, Langmuir isotherm typically shows the homogeneous nature of the adsorption process. It is assumed that adsorbents are mainly composed of fixed vacant sites, each adsorbing single adsorbate particle leading to a monolayer adsorption process (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Table S2 provides a summary of the model equations used in isotherm studies.</p><p id=\"Par60\">The nature of adsorption can be determined based on the value of ‘<italic>n</italic>’ (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Favorability of the process was checked using the separating factor (<italic>R</italic><sub><italic>L</italic></sub>):<disp-formula id=\"Equf\"><alternatives><mml:math display=\"block\" id=\"Equf_Math\"><mml:mrow><mml:msub><mml:mi>R</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mn>1</mml:mn><mml:mrow><mml:mn>1</mml:mn><mml:mo>+</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equf_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$R_{L} = \\frac{1}{{1 + K_{L} C_{0} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equf.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec7\"><title>Thermodynamic studies</title><p id=\"Par61\">To study the temperature dependence of adsorptive removal of ACVR and the spontaneity of the process, thermodynamic studies were conducted. Thermodynamic parameters, namely, Gibbs free energy and heat of adsorption, were also estimated using equations given in Table S3.</p></sec><sec id=\"Sec8\"><title>Column studies and data analysis</title><p id=\"Par62\">A glass column with an inner diameter of 1.25 cm and a height of 45 cm was used to conduct fixed-bed column studies (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). The required flow rate was maintained using a peristaltic pump. Nylon cloth was used at the bottom to stop any adsorbent loss, and glass beads were used on both sides of the column to provide mechanical support to the bed and for uniform distribution of the feed solution preventing channelling. All the experiments were performed at an ambient temperature of 30 ± 2 °C. Effects of various process parameters such as bed height, volumetric flow rate, and inlet ACVR concentration were studied. After predetermined intervals, samples were taken and examined to determine the ACVR concentration. The adsorption capacity and breakthrough time were estimated to assess column performance.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic diagram of a fixed-bed adsorption column used for ACVR removal</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig1_HTML.png\"/></fig></p><p id=\"Par63\">Fixed-bed analytical models, namely, Thomas, Adams–Bohart, Yoon–Nelson, Clark, and Yan models were used to characterize experimental breakthrough curves. Table S4 summarizes the model equations used in the study.</p></sec></sec><sec id=\"Sec9\"><title>Results and discussion</title><sec id=\"Sec10\"><title>Adsorbent characterization</title><p id=\"Par64\">BET surface area and pore volume of G-AC were found to be 470 m<sup>2</sup>/g and 0.23 cm<sup>3</sup>/g, respectively. The average pore diameter of G-AC (1.44 nm) suggests the prominent presence of supra-micropores in G-AC (0.7–2 nm) (Thommes <xref ref-type=\"bibr\" rid=\"CR56\">2010</xref>). Fig. S2 shows the nitrogen adsorption–desorption isotherm at 77 K and the G-AC pore size distribution curve.</p><p id=\"Par65\">The isotherm is a type IV isotherm, indicating the presence of mesoporous particles. Micro-mesoporous carbons usually feature H4 loops, which are closely attributed to the thin and slit-like pores (Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). The prominence of micropores causes the volume adsorbed to increase rapidly at lower relative pressure. The pore size distribution estimated by the BJH plot validated the presence of mesopores along with the micropores. Fig. S3 further summarizes the characteristics of G-AC, confirming that it is an efficacious adsorbent.</p><p id=\"Par66\">Higher magnification SEM micrograph (Fig. S3) of the G-AC sample confirmed its porous structure. Additionally, the EDX findings revealed the elemental analysis of G-AC. Fig. S3 clearly demonstrates the adsorption of ACVR on pores of G-AC.</p></sec><sec id=\"Sec11\"><title>Point of zero charge and pH effect</title><p id=\"Par67\">The liquid-phase pH significantly influences the adsorbent and their ability to adsorb, affecting the binding sites. The effect of pH on ACVR removal using G-AC was examined for 100 mg/L concentrated ACVR solution (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a). For a contact time of 75 min, q<sub>t</sub> increased from 22.75 to 24.62 mg/g on increasing pH from 3 to 7; further increasing pH to 11, q<sub>t</sub> decreased to 20.25 mg/g. From Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a, it can be seen that maximum removal was obtained at pH 7. The increased electrostatic affinity between the surface charge of G-AC and ACVR is most likely responsible for the improved adsorption capacity of the system at pH 7. As a result, possibly the G-AC and ACVR surface charges in the aqueous medium are in opposite directions. The point of zero charge has been further used to explain pH influence and to comprehend the interaction between G-AC and ACVR.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p><bold>a</bold> pH effect on ACVR removal at a contact time of 75 min, <bold>b</bold> Point of zero charge of G-AC and ACVR</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig2_HTML.png\"/></fig></p><p id=\"Par68\">The pH at which the surface of an adsorbent is believed to behave neutrally is referred to as the point of zero charge (pH<sub>ZPC</sub>). Hence, it is significant to determine the pH<sub>ZPC</sub> before conducting the adsorptive removal experiments involving ionic species adsorption from aqueous environments onto solid surfaces. The pH<sub>ZPC</sub> for G-AC and ACVR is illustrated in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b. The results show that the ACVR surface is more basic (pH<sub>ZPC</sub>: 8.2) than G-AC's surface (pH<sub>ZPC</sub>: 6.9). The ACVR and G-AC surfaces are charged positively at pH &lt; pH<sub>ZPC</sub> and negatively at pH &gt; pH<sub>ZPC</sub>. The surface of G-AC shows a negative charge at pH &gt; pH<sub>ZPC</sub>. As the initial pH of the solution is increased, the surplus hydroxide ions in the bulk liquid are generally counterbalanced by the G-AC released hydrogen ions. Furthermore, deprotonation of G-AC lowers the solution’s final pH and results in a net negative charge on the G-AC surface.</p><p id=\"Par69\">As discussed earlier, the G-AC surface is positively charged below pH<sub>ZPC</sub>, as consequence hydrogen ions compete with ACVR cations to reach the surface. The hydrogen ions will gradually surround major active sites on the G-AC surface. At pH &lt; 7, both G-AC and ACVR are positively charged which repelled from each other. As such, ACVR molecules were not effectively adsorbed on G-AC. Similar patterns for pH<sub>ZPC</sub> values for G-AC and ACVR are reported in the literature (Borah et al. <xref ref-type=\"bibr\" rid=\"CR7\">2008</xref>).</p></sec><sec id=\"Sec12\"><title>Effect of adsorbent dose</title><p id=\"Par70\">Experiments were carried out at varying doses of adsorbents ranging from 1 to 4 g/L at 25 °C for 2 h to investigate the effects of the G-AC dose on the ACVR removal efficiency (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>).<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Impact of varying adsorbent dose on ACVR removal</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig3_HTML.png\"/></fig></p><p id=\"Par71\">When adsorbent dose was increased from 1 to 4 g/L, there was a polynomial increase in the removal of ACVR from 39.7 to 95.2%. It can be seen from Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref> that there was a continuous rise in the percent removal of ACVR, but 4 g/L was taken as optimum as already 95% removal was being achieved with this dose. A reduction of 95.2% in ACVR concentration indicates that almost all of the vacant-active sites on the adsorbent surface are occupied at this point, and increasing the adsorbent dosage beyond this point will not be cost-effective.</p></sec><sec id=\"Sec13\"><title>Impact of contact time</title><p id=\"Par72\">At natural pH (~ 6.8), the impact of contact time on ACVR adsorption by G-AC was examined for 105 min. Figure <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a depicts the performance of adsorbents for the removal of ACVR with contact time.<fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p><bold>a</bold> Impact of Contact time (initial ACVR concentration: 100 mg/L) and <bold>b</bold> Impact of ACVR initial concentration on percent removal at equilibrium</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig4_HTML.png\"/></fig></p><p id=\"Par73\">The polynomial increase in percent removal was observed on increasing the contact time of adsorbent and ACVR. More than 50% of the removal was achieved within the initial 30 min using G-AC. Initially, the removal rate was rapid; however slow reduction was observed later. This could be due to more unoccupied sites on adsorbent are available at the initial stage with a high driving force which decreases with time and active sites also diminishes. Finally, for an initial ACVR concentration of 100 mg/L system reached the state of saturation at 75 min.</p></sec><sec id=\"Sec14\"><title>Adsorption kinetics</title><p id=\"Par74\">The adsorption kinetics of ACVR onto G-AC using pseudo-first- and -second-order, Elovich, and intra-particle diffusion kinetic models is depicted in Fig. S4. Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> summarizes the values of regression coefficients obtained and other kinetic parameters of ACVR adsorption.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Parameters for various kinetic models for ACVR removal</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Kinetic model</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Values</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Pseudo-first-order model</p></td><td align=\"left\"><p><italic>q</italic><sub>e,exp</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>24.62</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>k</italic><sub>1</sub> (min<sup>−1</sup>)</p></td><td align=\"char\" char=\".\"><p>0.04</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>8.98</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9899</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>8.1%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>7.82</p></td></tr><tr><td align=\"left\"><p>Pseudo-second-order model</p></td><td align=\"left\"><p><italic>k</italic><sub>2</sub> (g/mg/min)</p></td><td align=\"char\" char=\".\"><p>0.01</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>25.58</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9991</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>h</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>7.20</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>0.02%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>0.43</p></td></tr><tr><td align=\"left\"><p>Elovich kinetic model</p></td><td align=\"left\"><p><italic>α</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>296.61</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>β</italic> (g/mg)</p></td><td align=\"char\" char=\".\"><p>0.36</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9771</p></td></tr><tr><td align=\"left\"><p>Intraparticle diffusion model</p></td><td align=\"left\"><p><italic>K</italic><sub>d</sub> (mg/g/min<sup>0.5</sup>)</p></td><td align=\"char\" char=\".\"><p>1.12</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>C</italic> (mg/g)</p></td><td align=\"char\" char=\".\"><p>15.48</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9579</p></td></tr></tbody></table></table-wrap></p><p id=\"Par75\">The significance of studying the adsorption kinetics of ACVR onto G-AC using different kinetic models lies in understanding the rate and mechanism of the adsorption process. The comparison of the results obtained from different models can help determine the most suitable kinetic model for describing the adsorption process and provide insights into the mechanism of adsorption. Kinetic data obtained in the present study are significant and indicate that the adsorption process satisfies pseudo-second-order kinetic model.</p><p id=\"Par76\">The smaller difference in the equilibrium amount adsorbed between experimental and calculated values confirms the pseudo-second-order behavior of the process (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Moreover, the slightly lower correlation coefficient (<italic>R</italic><sup>2</sup>) also shows the inappropriateness of the pseudo-first-order kinetic model. The adsorption kinetics of numerous bio-refractory pollutant-adsorbent systems have exhibited comparable kinetic results (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p><p id=\"Par77\">If the value of C is not zero for any adsorption process, the intraparticle diffusion is not the sole controlling mechanism (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). Thus, it can be concluded that during ACVR sorption on G-AC, film and intraparticle diffusion occur simultaneously (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>).</p></sec><sec id=\"Sec15\"><title>Influence of ACVR initial concentration</title><p id=\"Par78\">It is vital to investigate the effect of the initial concentration of the adsorbate on the sorption rate for effective adsorption. The variation of the adsorptive removal efficiency of G-AC at different initial concentrations of ACVR (100–400 mg/L) is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b.</p><p id=\"Par79\">It is evident from Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b that as the drug concentration rise, so does the quantity of adsorbed ACVR. This could be because the difference between available adsorbent sites and the initial number of ACVR molecules is high at higher concentrations. Therefore, an initial concentration of 100 mg/L ACVR was used in the present study; moreover, further change in percent removal was significantly less. Additionally, it is common that the lower the initial adsorbate concentration, the higher removal efficiency while the other experimental conditions were kept constant.</p></sec><sec id=\"Sec16\"><title>Adsorption isotherms</title><p id=\"Par80\">Fig. S5 depicts the adsorption isotherms (Freundlich, Langmuir, and Temkin) and Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> shows the isotherm parameters for the ACVR adsorption onto G-AC.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Isotherm parameters for ACVR sorption</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Particulars</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Value</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Freundlich isotherm</p></td><td align=\"left\"><p><italic>K</italic><sub>F</sub> (mg/g)/(L/mg)<sup>n</sup></p></td><td align=\"left\"><p>24.51</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>n</italic></p></td><td align=\"left\"><p>2.37</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9293</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>12.69</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>8.72</p></td></tr><tr><td align=\"left\"><p>Langmuir isotherm</p></td><td align=\"left\"><p><italic>Q</italic><sub>m</sub> (mg/g)</p></td><td align=\"left\"><p>105.26</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>K</italic><sub>L</sub> (L/mg)</p></td><td align=\"left\"><p>0.21</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9981</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>3.46%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>2.35</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sub>L</sub></p></td><td align=\"left\"><p>0 &lt; <italic>R</italic><sub>L</sub> &lt; 1</p></td></tr><tr><td align=\"left\"><p>Temkin isotherm</p></td><td align=\"left\"><p><italic>A</italic><sub>T</sub> (L/g)</p></td><td align=\"left\"><p>2.10</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>b</italic><sub>T</sub></p></td><td align=\"left\"><p>109.93</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>B</italic><sub>T</sub> (mg/g)</p></td><td align=\"left\"><p>22.54</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9831</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>16.865</p></td></tr></tbody></table></table-wrap></p><p id=\"Par81\">The highest <italic>R</italic><sup>2</sup> value confirms the Langmuir adsorption isotherm model’s good fit to the experimental data. In the present work, the value of <italic>R</italic><sub><italic>L</italic></sub> was found 0.045 indicating the ACVR adsorption to be a favorable process. Furthermore, the low <italic>R</italic><sub><italic>L</italic></sub> value suggests that ACVR interaction with adsorbents is strong (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). The value of the heterogeneity factor also confirms the favorability of the process, and the process follows the Langmuir isotherm (0 &lt; 1/<italic>n</italic> &lt; 1) (Fytianos et al. <xref ref-type=\"bibr\" rid=\"CR22\">2000</xref>). It also signifies that monolayer chemisorption occurs with uniform adsorptive sites and energy (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>). The increase in adsorption capacity indicates weak interaction between adsorbed ACVR molecules (Hasnain Isa et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). Similar results are reported for numerous pollutant-adsorbent systems (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p></sec><sec id=\"Sec17\"><title>Thermodynamic studies</title><p id=\"Par82\">Adsorption studies at varying temperatures were conducted to examine the influence of temperature on ACVR uptake (Fig. S6). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> highlights the thermodynamic parameters for ACVR adsorption onto G-AC. Both 1/<italic>T</italic> and ln (<italic>q</italic><sub><italic>e</italic></sub>/<italic>C</italic><sub><italic>e</italic></sub>) values decrease with rise in temperature, indicating that the adsorption process is exothermic. Following equations were used to calculate change in Gibbs free energy,<disp-formula id=\"Equg\"><alternatives><mml:math display=\"block\" id=\"Equg_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mo>-</mml:mo><mml:mi>R</mml:mi><mml:mi>T</mml:mi><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:math><tex-math id=\"Equg_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = - RT\\ln K_{0}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equg.gif\"/></alternatives></disp-formula><disp-formula id=\"Equh\"><alternatives><mml:math display=\"block\" id=\"Equh_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>-</mml:mo><mml:mi>T</mml:mi><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mo>∘</mml:mo></mml:msup></mml:mrow></mml:math><tex-math id=\"Equh_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = \\Delta H^\\circ - T\\Delta S^\\circ$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equh.gif\"/></alternatives></disp-formula><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Thermodynamic parameters for ACVR adsorption onto G-AC</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Temperature (°C)</p></th><th align=\"left\"><p>∆<italic>H</italic><sup>0</sup> (kJ/mol)</p></th><th align=\"left\"><p>∆<italic>S</italic><sup>0</sup> (kJ/mol/K)</p></th><th align=\"left\"><p>∆<italic>G</italic><sup>0</sup> (kJ/mol)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>25</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 56.88</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 0.168</p></td><td align=\"char\" char=\".\"><p>− 6.93</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"char\" char=\".\"><p>− 6.04</p></td></tr><tr><td align=\"left\"><p>35</p></td><td align=\"char\" char=\".\"><p>− 4.59</p></td></tr><tr><td align=\"left\"><p>40</p></td><td align=\"char\" char=\".\"><p>− 4.19</p></td></tr><tr><td align=\"left\"><p>45</p></td><td align=\"char\" char=\".\"><p>− 3.38</p></td></tr><tr><td align=\"left\"><p>50</p></td><td align=\"char\" char=\".\"><p>− 2.64</p></td></tr><tr><td align=\"left\"><p>55</p></td><td align=\"char\" char=\".\"><p>− 1.78</p></td></tr></tbody></table></table-wrap></p><p id=\"Par83\">Combining above equations,<disp-formula id=\"Equi\"><alternatives><mml:math display=\"block\" id=\"Equi_Math\"><mml:mrow><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>=</mml:mo><mml:mo>ln</mml:mo><mml:mfrac><mml:msub><mml:mi>q</mml:mi><mml:mi>e</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mi>e</mml:mi></mml:msub></mml:mfrac><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mi>R</mml:mi></mml:mfrac><mml:mo>-</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equi_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\ln K_{0} = \\ln \\frac{{q_{e} }}{{C_{e} }} = \\frac{{\\Delta S^{0} }}{R} - \\frac{{\\Delta H^{0} }}{RT}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equi.gif\"/></alternatives></disp-formula></p><p id=\"Par84\">The heat of adsorption was negative, indicating exothermic nature for ACVR sorption onto G-AC. ACVR is soluble in water and may lose some of its hydration molecules, releasing heat. The spontaneous physisorption process is further established by the ∆<italic>G</italic>° value (0–20 kJ/mol) (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>) (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Further, as the temperature rises from 25 to 55 °C, ∆<italic>G</italic>° increases, indicating that the feasibility of the process declines with temperature rise (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>; Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The favorability of the process was also confirmed with the value of <italic>R</italic><sub><italic>L</italic></sub> reported in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>. Decrease in <italic>K</italic><sub>0</sub> value with temperature also confirms the exothermic nature of ACVR adsorption (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Moreover, the value of ∆<italic>S</italic>° was negative, showing that the ACVR adsorption was enthalpy driven (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Several pollutant-adsorbent systems have also produced similar results (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Since the Langmuir isotherm supports chemisorption while ∆<italic>G</italic>° (0–20 kJ/mol) supports physisorption. This was further clarified using ∆H° which indicates physisorption is being accompanied by chemisorption. This concludes that the adsorption of ACVR onto G-AC is in the transition phase from physisorption to chemisorption.</p></sec><sec id=\"Sec18\"><title>Effect of volumetric flow rate</title><p id=\"Par85\">The breakthrough response for ACVR adsorption was examined at various volumetric flow rates (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). As the volumetric flow rate rises from 3.1 to 3.8 mL/min, the breakthrough times at C<sub>t</sub>/C<sub>0</sub> = 0.1 shifted to 75 min from 120 min. It can be explained as at a high flow rate, the contact time between the G-AC and ACVR reduces, limiting the bed's adsorption capacity and processing times (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>). Furthermore, at a higher flow rate, the breakthrough curve shifts near the origin, become steeper, and reaches saturation more quickly.<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Effect of volumetric flow rate</p></caption><graphic id=\"MO6\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig5_HTML.png\"/></fig></p><p id=\"Par86\">Figure <xref ref-type=\"fig\" rid=\"Fig5\">5</xref> demonstrates that C<sub>t</sub>/C<sub>0</sub> increased from 0.12 to 0.32 at 3.1 and 4.5 mL/min flow rates, respectively, after 2 h. ACVR has more time to permeate through bed when the flow rate decreases, resulting in an increased column uptake. Likewise, higher volumetric flow rate diminishes the thickness of fluid film surrounding the sorbent particles, reducing mass transfer resistance and a quick increase in mass transfer (Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>). Similar trends have been reported in previous studies for the adsorptive removal of various pharmaceutical drugs (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec19\"><title>Effect of bed height</title><p id=\"Par87\">Figure <xref ref-type=\"fig\" rid=\"Fig6\">6</xref> presents breakthrough curves data at varying bed depths for a 200 mg/L concentrated solution and a volumetric flow rate of 3.1 mL/min.<fig id=\"Fig6\"><label>Fig. 6</label><caption xml:lang=\"en\"><p>Effect of bed height</p></caption><graphic id=\"MO7\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig6_HTML.png\"/></fig></p><p id=\"Par88\">The breakthrough time and exhaustion time decreased on reducing bed depth, moving the breakthrough curve towards the origin, as shown in Fig. <xref ref-type=\"fig\" rid=\"Fig6\">6</xref>. Also, C<sub>t</sub>/C<sub>0</sub> for a short-fixed bed rises fast than for a long bed. Breakthrough concentrations (C<sub>t</sub>/C<sub>0</sub> = 0.1) were attained at contact times of 2 h and 7 h for 3.5 and 12 cm of bed heights, respectively. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub>0</sub> varied from 0.59 and 0.11 at bed heights of 3.5 and 12 cm, respectively, after 7 h. With a greater bed height, more adsorbent load is used resulting in more adsorption sites and a longer contact time to attain 100% saturation (Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Furthermore, as the bed height rises, the adsorbate has considerable time for diffusion through the pores of the adsorbent, increasing the overall adsorbed amount and ACVR removal efficiency (Reynel-Avila et al. <xref ref-type=\"bibr\" rid=\"CR50\">2015</xref>). Similar results for the adsorption of several pharmacological drugs have been reported in the literature (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec20\"><title>Impact of inlet ACVR concentration</title><p id=\"Par89\">Figure <xref ref-type=\"fig\" rid=\"Fig7\">7</xref> shows that the inlet ACVR concentration considerably impacts the breakthrough curve.<fig id=\"Fig7\"><label>Fig. 7</label><caption xml:lang=\"en\"><p>Effect of inlet ACVR concentration</p></caption><graphic id=\"MO8\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig7_HTML.png\"/></fig></p><p id=\"Par90\">Time to reach breakthrough and exhaustion increased as the inlet ACVR concentration decreased. The breakthrough curve moved away from the origin at lower inlet concentration. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub><italic>0</italic></sub> varied from 0.03 to 0.10 at inlet concentrations of 200 and 300 mg/L, respectively, at a contact time of 90 min. This phenomenon may be explained by an increase in the sorption rate and the mass transfer driving force across the liquid film, which causes the fixed bed to saturate more quickly. These study results on the impact of inflow concentration on breakthrough curves concur with other bio-refractory compounds reported literature studies (Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>).</p></sec><sec id=\"Sec21\"><title>Thomas model</title><p id=\"Par91\">The Thomas model investigates how maximum column uptake and breakthrough curves are evaluated. This model emphasizes that the process follows Langmuir kinetics, and there is no axial dispersion (Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). It is also derived under the presumption that the rate of the driving force is subject to reversible 2nd order kinetics. It is advisable for the adsorption system when there is essentially no resistance to internal diffusion (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> provides more information on the obtained <italic>k</italic><sub>TH</sub>, <italic>q</italic><sub>TH</sub>, and <italic>R</italic><sup>2</sup> values at various process conditions (Fig. S7).<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>Parameters of kinetic analytical models for ACVR adsorption for different sets of process conditions</p></caption><table frame=\"hsides\" rules=\"groups\"><tbody><tr><td align=\"left\" rowspan=\"3\"><p>Process conditions</p></td><td align=\"left\"><p><italic>Z</italic> (cm)</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>8</p></td><td align=\"left\"><p>12</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td></tr><tr><td align=\"left\"><p><italic>Q</italic> (mL/min)</p></td><td align=\"left\"><p>3.8</p></td><td align=\"left\"><p>4.5</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td></tr><tr><td align=\"left\"><p><italic>C</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>300</p></td><td align=\"left\"><p>400</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Thomas model</p></td><td align=\"left\"><p>k<sub>TH</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>3.15</p></td><td align=\"left\"><p>4.45</p></td><td align=\"left\"><p>2.20</p></td><td align=\"left\"><p>1.35</p></td><td align=\"left\"><p>2.90</p></td><td align=\"left\"><p>1.97</p></td><td align=\"left\"><p>1.57</p></td></tr><tr><td align=\"left\"><p>q<sub>TH</sub> (mg/g)</p></td><td align=\"left\"><p>211.81</p></td><td align=\"left\"><p>199.54</p></td><td align=\"left\"><p>248.98</p></td><td align=\"left\"><p>242.69</p></td><td align=\"left\"><p>224.91</p></td><td align=\"left\"><p>302.20</p></td><td align=\"left\"><p>312.93</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"4\"><p>Yoon–Nelson model</p></td><td align=\"left\"><p><italic>k</italic><sub>YN</sub> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0063</p></td><td align=\"left\"><p>0.0089</p></td><td align=\"left\"><p>0.0044</p></td><td align=\"left\"><p>0.0027</p></td><td align=\"left\"><p>0.0058</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p>0.0063</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic> (min)</p></td><td align=\"left\"><p>278.70</p></td><td align=\"left\"><p>221.71</p></td><td align=\"left\"><p>803.16</p></td><td align=\"left\"><p>1174.33</p></td><td align=\"left\"><p>362.76</p></td><td align=\"left\"><p>324.95</p></td><td align=\"left\"><p>252.36</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic><sub>exp</sub> (min)</p></td><td align=\"left\"><p>270</p></td><td align=\"left\"><p>210</p></td><td align=\"left\"><p>660</p></td><td align=\"left\"><p>1200</p></td><td align=\"left\"><p>360</p></td><td align=\"left\"><p>330</p></td><td align=\"left\"><p>210</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Adams–Bohart model</p></td><td align=\"left\"><p><italic>k</italic><sub>BA</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>1.70</p></td><td align=\"left\"><p>2.25</p></td><td align=\"left\"><p>1.60</p></td><td align=\"left\"><p>0.95</p></td><td align=\"left\"><p>1.75</p></td><td align=\"left\"><p>1.13</p></td><td align=\"left\"><p>0.82</p></td></tr><tr><td align=\"left\"><p><italic>N</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>7733.02</p></td><td align=\"left\"><p>7393.39</p></td><td align=\"left\"><p>5684.98</p></td><td align=\"left\"><p>5538.09</p></td><td align=\"left\"><p>7182.78</p></td><td align=\"left\"><p>10388.09</p></td><td align=\"left\"><p>12462.19</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.81</p></td><td align=\"left\"><p>0.80</p></td><td align=\"left\"><p>0.85</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.82</p></td><td align=\"left\"><p>0.77</p></td><td align=\"left\"><p>0.77</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Clark model</p></td><td align=\"left\"><p><italic>r</italic> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0071</p></td><td align=\"left\"><p>0.01</p></td><td align=\"left\"><p>0.0054</p></td><td align=\"left\"><p>0.0032</p></td><td align=\"left\"><p>0.0067</p></td><td align=\"left\"><p>0.0068</p></td><td align=\"left\"><p>0.0072</p></td></tr><tr><td align=\"left\"><p><italic>A</italic></p></td><td align=\"left\"><p>11.89</p></td><td align=\"left\"><p>16.26</p></td><td align=\"left\"><p>115.75</p></td><td align=\"left\"><p>68.27</p></td><td align=\"left\"><p>18.30</p></td><td align=\"left\"><p>14.71</p></td><td align=\"left\"><p>9.94</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.88</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.87</p></td><td align=\"left\"><p>0.89</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Yan model</p></td><td align=\"left\"><p><italic>k</italic><sub>Y</sub> (L/mg/min)</p></td><td align=\"left\"><p>0.030</p></td><td align=\"left\"><p>0.040</p></td><td align=\"left\"><p>0.035</p></td><td align=\"left\"><p>0.032</p></td><td align=\"left\"><p>0.027</p></td><td align=\"left\"><p>0.015</p></td><td align=\"left\"><p>0.010</p></td></tr><tr><td align=\"left\"><p><italic>q</italic><sub>0</sub> (mg/g)</p></td><td align=\"left\"><p>0.756</p></td><td align=\"left\"><p>0.892</p></td><td align=\"left\"><p>0.307</p></td><td align=\"left\"><p>0.205</p></td><td align=\"left\"><p>0.617</p></td><td align=\"left\"><p>0.929</p></td><td align=\"left\"><p>1.239</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.98</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.99</p></td></tr></tbody></table></table-wrap></p><p id=\"Par92\">It indicates agreement between the Thomas model and experimental data for breakthrough curves (0.89 &gt; <italic>R</italic><sup>2</sup> &gt; 0.99), signifying the good fit of the model for ACVR adsorption on G-AC. Since the experimental data fitted the Thomas model well, film and intraparticle diffusion are not expected to be the limiting step. But, the limiting adsorption stage is the solid surface mass transfer (de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Flow resistance increases on increasing bed height, resulting in decreased mass transfer effect, causing a decrement in <italic>k</italic><sub>TH</sub> and increase in <italic>q</italic><sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>). On the other hand, increasing flow rate will result in insufficient contact time between ACVR and G-AC for fixed number of vacant sites causing decrease in the value of <italic>q</italic><sub>TH</sub> whereas increase in k<sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>). An increased driving force achieved a higher column performance at higher ACVR concentration. The results are consistent with the literature (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Pholosi et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>).</p><p id=\"Par93\">Table <xref ref-type=\"table\" rid=\"Tab5\">5</xref> compares the value of q<sub>TH</sub> of G-AC obtained in the present study with the literature. The maximum adsorption capacity of G-AC (q<sub>TH</sub>) in the present study has been found to be higher than other adsorbate-adsorbent systems.<table-wrap id=\"Tab5\"><label>Table 5</label><caption xml:lang=\"en\"><p>Comparison of q<sub>TH</sub> with values reported in literature</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Adsorbent</p></th><th align=\"left\"><p>Adsorbate</p></th><th align=\"left\"><p>Models investigated*</p></th><th align=\"left\"><p><italic>q</italic><sub>TH</sub> (mg/g)</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>G-AC</p></td><td align=\"left\"><p>ACVR</p></td><td align=\"left\"><p>TH, YN, AB, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>224.91</p></td><td align=\"left\"><p>Present study</p></td></tr><tr><td align=\"left\"><p>Thermally activated weed Salvinia Cucullata</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>159.20</p></td><td align=\"left\"><p>Baral et al. (<xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.09</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Norfloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>1.99</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Levofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.56</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon</p></td><td align=\"left\"><p>Amoxicillin</p></td><td align=\"left\"><p>TH, AB, Yan</p></td><td align=\"char\" char=\".\"><p>11.48</p></td><td align=\"left\"><p>de Franco et al. (<xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated sludge derived activated carbon</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>14.00</p></td><td align=\"left\"><p>Gupta and Garg (<xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Carboxyl-modified jute fiber</p></td><td align=\"left\"><p>Cu (II)</p></td><td align=\"left\"><p>TH, YN</p></td><td align=\"char\" char=\".\"><p>42.21</p></td><td align=\"left\"><p>Du et al. (<xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date palm petiole</p></td><td align=\"left\"><p>Indigo carmine</p></td><td align=\"left\"><p>TH, YN, AB, Yan</p></td><td align=\"char\" char=\".\"><p>48.88</p></td><td align=\"left\"><p>Khadhri et al. (<xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Bone char</p></td><td align=\"left\"><p>Crystal violet</p></td><td align=\"left\"><p>TH, Yan</p></td><td align=\"char\" char=\".\"><p>22.44</p></td><td align=\"left\"><p>Cruz et al. (<xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Magnetic pine cone composite</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>33.08</p></td><td align=\"left\"><p>Pholosi et al. (<xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Acid modified alumina</p></td><td align=\"left\"><p>Fluoride</p></td><td align=\"left\"><p>TH, YN, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>19.12</p></td><td align=\"left\"><p>Kumari et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>*<italic>TH</italic> Thomas model, <italic>YN</italic> Yoon–Nelson model, <italic>AB</italic> Adams–Bohart model</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec22\"><title>Yoon–Nelson model</title><p id=\"Par94\">This model presupposes that the decline in the adsorption rate of each adsorbate molecule is proportionally related to the adsorbate adsorption probability and probability of its breakthrough capacity onto the adsorbent (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Yoon and Nelson <xref ref-type=\"bibr\" rid=\"CR60\">1984</xref>). This model is significant in determining the nature of the adsorption process with its exhaustion time for a given concentration (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>). The estimated values of various Yoon–Nelson model parameters are summarized in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. It shows the strong correlation between experimental data and the Yoon–Nelson model (0.89 &lt; <italic>R</italic><sup>2</sup> &lt; 0.99) (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> and Fig. S8), indicating the adequacy of the model. According to the Yoon–Nelson model's fitting parameters, an increase in the initial concentration of ACVR leads to the enhancement of <italic>K</italic><sub>YN</sub> value and reduction in τ value. It might be because the G-AC in the column quickly saturates.</p><p id=\"Par95\">On the contrary, as the bed height was raised, the <italic>K</italic><sub>YN</sub> declined and the value of <italic>τ</italic> improved. This is because the amount of adsorbent increases as the height of the bed rises. Other studies have also reported consistent patterns (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Demarchi et al. <xref ref-type=\"bibr\" rid=\"CR16\">2015</xref>).</p></sec><sec id=\"Sec23\"><title>Adams–Bohart model</title><p id=\"Par96\">The Adams–Bohart model is based on the premise that equilibrium is not instantaneous and primarily concerned with the surface reaction theory. Therefore, rate of adsorption is related to adsorbate concentration in the solution and the adsorbent’s residual capacity (Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The initial segment of the breakthrough curve has been described by Adams–Bohart model in several previous studies. However, in the current study, the model has been fitted from the beginning of breakthrough curve to final point (Fig. S9). The estimated model parameters and R<sup>2</sup> values for various process conditions are provided in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> shows that the values of k<sub>BA</sub> were found to increase with increasing flow rate but deceased with increasing initial ACVR concentration and bed height. At most of the process conditions, the correlation coefficient is low (&lt; 0.86); hence, the correlation between experimental data and the Adams–Bohart model is not strong. Other investigations have also reported similar results. (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec24\"><title>Clark model</title><p id=\"Par97\">The Clark model anticipates the breakthrough curve’s behavior by studying mass transfer and adsorption equilibrium (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). It assumes that the flow in the column is plug flow type, and the Freundlich isotherm is followed by mass transfer occurring in the column (Patel <xref ref-type=\"bibr\" rid=\"CR44\">2019</xref>). Fig. S10 shows the linear curve fitting of breakthrough curves using the Clark model. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> presents the value of Clark model parameters, ‘<italic>A</italic>’ and ‘<italic>r</italic>’, at different process conditions. On increasing the bed height of the column, the value of <italic>r</italic> decreased. Further, <italic>r</italic> increases with flow rate and inlet ACVR concentration. At almost all process conditions, the value of correlation coefficient is in the range (0.87–0.92), indicating relatively poor fit of the Clark model. Similar results have been obtained earlier (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>).</p></sec><sec id=\"Sec25\"><title>Yan model</title><p id=\"Par98\">It has been reported earlier that, at absurdly short and very long operation times, the Yan model is proposed to reduce the inaccuracy that occurs from using the Thomas model (Maheshwari and Gupta <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR59\">2001</xref>). Figure <xref ref-type=\"fig\" rid=\"Fig8\">8</xref> illustrates the linear curve fitting of experimental data using the Yan model for various parameters.<fig id=\"Fig8\"><label>Fig. 8</label><caption xml:lang=\"en\"><p>Yan model for varying <bold>a</bold> flow rate, <bold>b</bold> bed height, and <bold>c</bold> initial ACVR concentration</p></caption><graphic id=\"MO9\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig8_HTML.png\"/></fig></p><p id=\"Par99\">The findings indicate a good agreement between the experimental data and the model, with R<sup>2</sup> values in the ranges of 0.94–0.99 (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>). The value of k<sub>Y</sub> increases on increasing flow rate, while decreases with concentration. According to Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>, the estimated adsorption capacity values from the Thomas and Yan model are of the same order. However, it is observed that the Yan model is better at evaluating adsorption capacity of the bed than the Thomas model which estimates inflated value (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>). Other works of literature have also reported similar overall results (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Comparison of breakthrough curve fitting</title><p id=\"Par100\">Thomas and Yan models suggest that external and intraparticle diffusion are not controlling, but the solid surface mass transfer (surface diffusion) is the limiting step in the adsorption process. (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). Adams–Bohart model demonstrated the lowest average <italic>R</italic><sup>2</sup> (0.82) signifying the weak correlation between experimental data and the model. Although, the average value of correlation coefficient for Clark model is 0.90. The poor fit of the Clark model for breakthrough data is in accordance with the batch study results, which fitted well to Langmuir isotherm. The inability of Adams–Bohart and Clark models to accurately predict the removal of ACVR by G-AC can be attributed to two factors: the existence of numerous mechanisms (possibly, adsorption and ion exchange) that contribute to ACVR adsorption onto G-AC and the significant involvement of mass transfer resistance in the removal of ACVR using G-AC (Dorado et al. <xref ref-type=\"bibr\" rid=\"CR17\">2014</xref>).</p><p id=\"Par101\">The Yan model exhibited the highest average <italic>R</italic><sup>2</sup> (0.98), indicating the best fit of the model and the experimental data. As noted in the previous section, batch ACVR adsorption onto G-AC was found to follow the pseudo-second-order kinetic model and Langmuir isotherm model. Both of these conclusions have been supported by Yan models applied to fixed-bed column studies.</p><p id=\"Par102\">Continuous column studies of ACVR adsorption on any of the adsorbent has been scarcely reported. Therefore, q<sub>TH</sub> value signifying adsorption capacity of the adsorbent obtained using Thomas model has been compared for the present study with results reported in literature for other adsorbate-adsorbent systems.</p></sec></sec><sec id=\"Sec27\" sec-type=\"conclusions\"><title>Conclusions</title><p id=\"Par103\">Antiviral pollutants, such as ACVR have been found in groundwater, surface water, and industrial wastewater. Their inadequate removal through conventional treatment is a significant challenge in water reuse. G-AC has proven to be an efficient adsorbent for the ACVR removal from aqueous solutions. The characterization showed the morphology and surface area of G-AC along with the ACVR adsorption (SEM, BET, and EDX).</p><p id=\"Par104\">Batch experiments showed that Langmuir isotherm adequately represents the equilibrium data indicating monolayer chemisorption is the predominant mechanism for ACVR adsorption onto G-AC. Pseudo-second-order kinetic model predicted the adsorption process. Negative values of heat of adsorption and Gibbs free energy indicated exothermic nature of the process and confirmed the process feasibility. The process of adsorption was found to be physisorption accompanied by chemisorption. The feasibility of adsorption decreases with increase in temperature. The adsorption process slightly decreases the randomness of molecules at the solid/liquid interface and is enthalpy driven.</p><p id=\"Par105\">The fixed-bed column experimental data were fitted to different analytical models. The Yan model fitted the data best, followed by the Thomas model confirming the pseudo-second-order kinetics and Langmuir isotherm. Adsorption capacity of the fixed bed of G-AC estimated using Thomas model indicated that the adsorption capacity is excellent for ACVR when compared to other adsorbate-adsorbent system studied in literature. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p><p id=\"Par106\">The current research's findings will be useful for small volumes of wastewater generated by ACVR manufacturing units. ACVR removal from the industrial effluents released by bulk manufacturing and formulation industries can be achieved in a continuous flow unit using a packed bed of a commercial adsorbent. This will prevent the entry of strong inhibitory substances, like ACVR, into the biological wastewater treatment units, which may inhibit the activity of the micro-organism that treat the soluble organics.</p></sec></body><back><ack><title>Acknowledgements</title><p>No specific grant was given to this research by funding organizations in the public, commercial, or not-for-profit sectors. However, one of the authors (AG) expresses her gratitude to Malaviya National Institute of Technology (MNIT) Jaipur and Ministry of Education, Govt. of India for providing the financial support in the form of Institute Fellowship for carrying out Ph. D. research work. The authors also wish to thank the Material Research Centre (MRC), MNIT Jaipur, for carrying out SEM, EDX and BET analysis of the samples.</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par107\">The authors report no declarations of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Al-Otaibi</surname><given-names>JS</given-names></name><name><surname>Mary</surname><given-names>YS</given-names></name><name><surname>Thomas</surname><given-names>R</given-names></name><name><surname>Kaya</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Detailed electronic structure, physico-chemical properties, excited state properties, virtual bioactivity screening and SERS analysis of three guanine based antiviral drugs valacyclovir HCl hydrate, acyclovir and ganciclovir</article-title><source>Polycycl Aromat Compd</source><year>2020</year><volume>42</volume><fpage>1260</fpage><lpage>1270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVKmu7jE</pub-id><pub-id pub-id-type=\"doi\">10.1080/10406638.2020.1773876</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azuma</surname><given-names>T</given-names></name><name><surname>Arima</surname><given-names>N</given-names></name><name><surname>Tsukada</surname><given-names>A</given-names></name><name><surname>Hirami</surname><given-names>S</given-names></name><name><surname>Matsuoka</surname><given-names>R</given-names></name><name><surname>Moriwake</surname><given-names>R</given-names></name><name><surname>Ishiuchi</surname><given-names>H</given-names></name><name><surname>Inoyama</surname><given-names>T</given-names></name><name><surname>Teranishi</surname><given-names>Y</given-names></name><name><surname>Yamaoka</surname><given-names>M</given-names></name><name><surname>Mino</surname><given-names>Y</given-names></name><name><surname>Hayashi</surname><given-names>T</given-names></name><name><surname>Fujita</surname><given-names>Y</given-names></name><name><surname>Masada</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Science of the Total Environment Detection of pharmaceuticals and phytochemicals together with their metabolites in hospital effluents in Japan, and their contribution to sewage treatment plant influents</article-title><source>Sci Total Environ</source><year>2016</year><volume>548–549</volume><fpage>189</fpage><lpage>197</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KqsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2015.12.157</pub-id><pub-id pub-id-type=\"pmid\">26802347</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Babas</surname><given-names>H</given-names></name><name><surname>Kaichouh</surname><given-names>G</given-names></name><name><surname>Khachani</surname><given-names>M</given-names></name><name><surname>Karbane</surname><given-names>ME</given-names></name><name><surname>Chakir</surname><given-names>A</given-names></name><name><surname>Guenbour</surname><given-names>A</given-names></name><name><surname>Bellaouchou</surname><given-names>A</given-names></name><name><surname>Warad</surname><given-names>I</given-names></name><name><surname>Zarrouk</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Equilibrium and kinetic studies for removal of antiviral sofosbuvir from aqueous solution by adsorption on expanded perlite: experimental, modelling and optimization</article-title><source>Surf Interfaces</source><year>2021</year><volume>23</volume><fpage>100962</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsFaqtr%2FF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.surfin.2021.100962</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Baral</surname><given-names>SS</given-names></name><name><surname>Das</surname><given-names>N</given-names></name><name><surname>Ramulu</surname><given-names>TS</given-names></name><name><surname>Sahoo</surname><given-names>SK</given-names></name><name><surname>Das</surname><given-names>SN</given-names></name><name><surname>Chaudhury</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) by thermally activated weed Salvinia cucullata in a fixed-bed column</article-title><source>J Hazard Mater</source><year>2009</year><volume>161</volume><fpage>1427</fpage><lpage>1435</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhsVSqsbnE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2008.04.127</pub-id><pub-id pub-id-type=\"pmid\">18571842</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bing</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Enhanced mineralization of pharmaceuticals by surface oxidation over mesoporous γ-Ti-Al<sub>2</sub>O<sub>3</sub> suspension with ozone</article-title><source>Appl Catal B Environ</source><year>2017</year><volume>202</volume><fpage>118</fpage><lpage>126</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFWnur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2016.09.019</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Biswas</surname><given-names>S</given-names></name><name><surname>Mishra</surname><given-names>U</given-names></name></person-group><article-title xml:lang=\"en\">Continuous fixed-bed column study and adsorption modeling: Removal of lead ion from aqueous solution by charcoal originated from chemical carbonization of rubber wood sawdust</article-title><source>J Chem</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1155/2015/907379</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Borah</surname><given-names>D</given-names></name><name><surname>Satokawa</surname><given-names>S</given-names></name><name><surname>Kato</surname><given-names>S</given-names></name><name><surname>Kojima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Surface-modified carbon black for As(V) removal</article-title><source>J Colloid Interface Sci</source><year>2008</year><volume>319</volume><fpage>53</fpage><lpage>62</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXnvVWntg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2007.11.019</pub-id><pub-id pub-id-type=\"pmid\">18082179</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bradley</surname><given-names>PM</given-names></name><name><surname>Barber</surname><given-names>LB</given-names></name><name><surname>Duris</surname><given-names>JW</given-names></name><name><surname>Foreman</surname><given-names>WT</given-names></name><name><surname>Furlong</surname><given-names>ET</given-names></name><name><surname>Hubbard</surname><given-names>LE</given-names></name><name><surname>Hutchinson</surname><given-names>KJ</given-names></name><name><surname>Keefe</surname><given-names>SH</given-names></name><name><surname>Kolpin</surname><given-names>DW</given-names></name></person-group><article-title xml:lang=\"en\">Riverbank filtration potential of pharmaceuticals in a wastewater-impacted stream</article-title><source>Environ Pollut</source><year>2014</year><volume>193</volume><fpage>173</fpage><lpage>180</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1ensbnK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.envpol.2014.06.028</pub-id><pub-id pub-id-type=\"pmid\">25038376</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Yue</surname><given-names>Q</given-names></name><name><surname>Gao</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Xu</surname><given-names>X</given-names></name><name><surname>Fu</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of hexavalent chromium from aqueous solution by modified corn stalk: a fixed-bed column study</article-title><source>Bioresour Technol</source><year>2012</year><volume>113</volume><fpage>114</fpage><lpage>120</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFWntrg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2011.11.110</pub-id><pub-id pub-id-type=\"pmid\">22189077</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Luo</surname><given-names>H</given-names></name><name><surname>Shi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Anatase TiO<sub>2</sub> nanoparticles-carbon nanotubes composite: optimization synthesis and the relationship of photocatalytic degradation activity of acyclovir in water</article-title><source>Appl Catal A Gen</source><year>2014</year><volume>485</volume><fpage>188</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhsVSmsLbP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcata.2014.08.004</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cruz</surname><given-names>MAP</given-names></name><name><surname>Guimarães</surname><given-names>LCM</given-names></name><name><surname>da Costa Júnior</surname><given-names>EF</given-names></name><name><surname>Rocha</surname><given-names>SDF</given-names></name><name><surname>da Mesquita</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of crystal violet from aqueous solution in continuous flow system using bone char</article-title><source>Chem Eng Commun</source><year>2020</year><volume>207</volume><fpage>372</fpage><lpage>381</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXms1amsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/00986445.2019.1596899</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Batch and fixed bed adsorption of levofloxacin on granular activated carbon from date (<italic>Phoenix dactylifera</italic> L.) stones by KOH chemical activation</article-title><source>Environ Toxicol Pharmacol</source><year>2017</year><volume>50</volume><fpage>159</fpage><lpage>166</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitlOls7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.etap.2017.02.005</pub-id><pub-id pub-id-type=\"pmid\">28189062</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of ciprofloxacin and norfloxacin from aqueous solution onto granular activated carbon in fixed bed column</article-title><source>Ecotoxicol Environ Saf</source><year>2017</year><volume>138</volume><fpage>139</fpage><lpage>145</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXislShsQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2016.12.032</pub-id><pub-id pub-id-type=\"pmid\">28040619</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Clercq</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Antivirals and antiviral strategies</article-title><source>Nat Rev Microbiol</source><year>2004</year><volume>2</volume><fpage>704</fpage><lpage>720</lpage><pub-id pub-id-type=\"doi\">10.1038/nrmicro975</pub-id><pub-id pub-id-type=\"pmid\">15372081</pub-id><pub-id pub-id-type=\"pmcid\">7097272</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Franco</surname><given-names>MAE</given-names></name><name><surname>de Carvalho</surname><given-names>CB</given-names></name><name><surname>Bonetto</surname><given-names>MM</given-names></name><name><surname>de Soares</surname><given-names>RS</given-names></name><name><surname>Féris</surname><given-names>LA</given-names></name></person-group><article-title xml:lang=\"en\">Removal of amoxicillin from water by adsorption onto activated carbon in batch process and fixed bed column: kinetics, isotherms, experimental design and breakthrough curves modelling</article-title><source>J Clean Prod</source><year>2017</year><volume>161</volume><fpage>947</fpage><lpage>956</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVant7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2017.05.197</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Demarchi</surname><given-names>CA</given-names></name><name><surname>Debrassi</surname><given-names>A</given-names></name><name><surname>Magro</surname><given-names>JD</given-names></name><name><surname>Nedelko</surname><given-names>N</given-names></name><name><surname>Ślawska-Waniewska</surname><given-names>A</given-names></name><name><surname>Dłuzewski</surname><given-names>P</given-names></name><name><surname>Greneche</surname><given-names>JM</given-names></name><name><surname>Rodrigues</surname><given-names>CA</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of Cr(VI) on crosslinked chitosan-Fe(III) complex in fixed-bed systems</article-title><source>J Water Process Eng</source><year>2015</year><volume>7</volume><fpage>141</fpage><lpage>152</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2015.05.003</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dorado</surname><given-names>AD</given-names></name><name><surname>Gamisans</surname><given-names>X</given-names></name><name><surname>Valderrama</surname><given-names>C</given-names></name><name><surname>Solé</surname><given-names>M</given-names></name><name><surname>Lao</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Cr(III) removal from aqueous solutions: a straightforward model approaching of the adsorption in a fixed-bed column</article-title><source>J Environ Sci Heal Part A Toxic/hazardous Subst Environ Eng</source><year>2014</year><volume>49</volume><fpage>179</fpage><lpage>186</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslWisbfJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/10934529.2013.838855</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>Z</given-names></name><name><surname>Zheng</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Experimental and modelling studies on fixed bed adsorption for Cu(II) removal from aqueous solution by carboxyl modified jute fiber</article-title><source>Powder Technol</source><year>2018</year><volume>338</volume><fpage>952</fpage><lpage>959</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVOltLjP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.powtec.2018.06.015</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fan</surname><given-names>L</given-names></name><name><surname>Luo</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>F</given-names></name><name><surname>Qiu</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Fabrication of novel magnetic chitosan grafted with graphene oxide to enhance adsorption properties for methyl blue</article-title><source>J Hazard Mater</source><year>2012</year><volume>215–216</volume><fpage>272</fpage><lpage>279</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XkvVWju74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2012.02.068</pub-id><pub-id pub-id-type=\"pmid\">22429622</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fountouli</surname><given-names>TV</given-names></name><name><surname>Chrysikopoulos</surname><given-names>CV</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and thermodynamics of pharmaceuticals, acyclovir and fluconazole, onto quartz sand under static and dynamic conditions</article-title><source>Environ Eng Sci</source><year>2018</year><volume>35</volume><fpage>909</fpage><lpage>917</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslSgu7jF</pub-id><pub-id pub-id-type=\"doi\">10.1089/ees.2017.0499</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Funke</surname><given-names>J</given-names></name><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Identification of transformation products of antiviral drugs formed during biological wastewater treatment and their occurrence in the urban water cycle</article-title><source>Water Res</source><year>2016</year><volume>98</volume><fpage>75</fpage><lpage>83</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XlvV2qt7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2016.03.045</pub-id><pub-id pub-id-type=\"pmid\">27082694</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fytianos</surname><given-names>K</given-names></name><name><surname>Voudrias</surname><given-names>E</given-names></name><name><surname>Kokkalis</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sorption-desorption behaviour of 2,4-dichlorophenol by marine sediments</article-title><source>Chemosphere</source><year>2000</year><volume>40</volume><fpage>3</fpage><lpage>6</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXivFSrsw%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0045-6535(99)00214-3</pub-id><pub-id pub-id-type=\"pmid\">10665437</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Gan</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>H</given-names></name><name><surname>Cai</surname><given-names>N</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Guo</surname><given-names>P</given-names></name><name><surname>Chen</surname><given-names>B</given-names></name><name><surname>Pan</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Effective degradation of COVID-19 related drugs by biochar-supported red mud catalyst activated persulfate process: Mechanism and pathway</article-title><source>J Clean Prod</source><year>2022</year><volume>340</volume><fpage>130753</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2022.130753</pub-id><pub-id pub-id-type=\"pmid\">36032562</pub-id><pub-id pub-id-type=\"pmcid\">9396784</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Garg</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and oxidation of ciprofloxacin in a fixed bed column using activated sludge derived activated carbon</article-title><source>J Environ Manag</source><year>2019</year><volume>250</volume><fpage>109474</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhslKnu7bJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jenvman.2019.109474</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Gupta</surname><given-names>AB</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence of acyclovir in the aquatic environment, its removal and research perspectives: a review</article-title><source>J Water Process Eng</source><year>2021</year><volume>39</volume><fpage>101855</fpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2020.101855</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>Yu</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Wang Yuanfeng Xie</surname><given-names>F</given-names></name><name><surname>Cheng</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of methylene blue by phoenix tree leaf powder in a fixed-bed column: experiments and prediction of breakthrough curves</article-title><source>Desalination</source><year>2009</year><volume>245</volume><fpage>284</fpage><lpage>297</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXovFyhtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2008.07.013</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hasnain Isa</surname><given-names>M</given-names></name><name><surname>Siew Lang</surname><given-names>L</given-names></name><name><surname>Asaari</surname><given-names>FAH</given-names></name><name><surname>Aziz</surname><given-names>HA</given-names></name><name><surname>Azam Ramli</surname><given-names>N</given-names></name><name><surname>Dhas</surname><given-names>JPA</given-names></name></person-group><article-title xml:lang=\"en\">Low cost removal of disperse dyes from aqueous solution using palm ash</article-title><source>Dye Pigment</source><year>2007</year><volume>74</volume><fpage>446</fpage><lpage>453</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXht1Ort7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.dyepig.2006.02.025</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heidary</surname><given-names>F</given-names></name><name><surname>Madani</surname><given-names>S</given-names></name><name><surname>Gharebaghi</surname><given-names>R</given-names></name><name><surname>Asadi-Amoli</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">Acyclovir as a potential add-on therapy in COVID-19 treatment regimens</article-title><source>Pharm Sci</source><year>2021</year><volume>27</volume><fpage>S68</fpage><lpage>S77</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislalsL%2FF</pub-id><pub-id pub-id-type=\"doi\">10.34172/PS.2021.38</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hosseini Nasr</surname><given-names>A</given-names></name><name><surname>Sadat</surname><given-names>AH</given-names></name><name><surname>Tayebee</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanism of different acyclovir concentrations on 1–2 nm sized magnetite nanoparticles: a molecular dynamics study</article-title><source>J Mol Liq</source><year>2018</year><volume>254</volume><fpage>64</fpage><lpage>69</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVemtro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2018.01.081</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence and removal of antiviral drugs in environment: a review</article-title><source>Water Air Soil Pollut</source><year>2013</year><pub-id pub-id-type=\"doi\">10.1007/s11270-012-1410-3</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption optimization of acyclovir on prepared activated carbon</article-title><source>Can J Chem Eng</source><year>2014</year><volume>92</volume><fpage>1627</fpage><lpage>1635</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhtlynur7J</pub-id><pub-id pub-id-type=\"doi\">10.1002/cjce.22026</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of antiviral drug, acyclovir from aqueous solution on powdered activated charcoal: kinetics, equilibrium, and thermodynamic studies</article-title><source>Desalin Water Treat</source><year>2014</year><volume>52</volume><fpage>4953</fpage><lpage>4968</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1SktL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2013.810324</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khadhri</surname><given-names>N</given-names></name><name><surname>El Khames</surname><given-names>SM</given-names></name><name><surname>Ben Mosbah</surname><given-names>M</given-names></name><name><surname>Moussaoui</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous column adsorption of indigo carmine onto activated carbon derived from date palm petiole</article-title><source>J Environ Chem Eng</source><year>2019</year><volume>7</volume><fpage>102775</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVygs7rK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jece.2018.11.020</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>U</given-names></name><name><surname>Mishra</surname><given-names>A</given-names></name><name><surname>Siddiqi</surname><given-names>H</given-names></name><name><surname>Meikap</surname><given-names>BC</given-names></name></person-group><article-title xml:lang=\"en\">Effective defluoridation of industrial wastewater by using acid modified alumina in fixed-bed adsorption column: experimental and breakthrough curves analysis</article-title><source>J Clean Prod</source><year>2021</year><volume>279</volume><fpage>123645</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslejtbvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2020.123645</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Nie</surname><given-names>X</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Can environmental pharmaceuticals be photocatalytically degraded and completely mineralized in water using g-C<sub>3</sub>N<sub>4</sub>TiO<sub>2</sub> under visible light irradiation? Implications of persistent toxic intermediates</article-title><source>Appl Catal B Environ</source><year>2016</year><volume>180</volume><fpage>726</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVOqu7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2015.07.014</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liao</surname><given-names>P</given-names></name><name><surname>Zhan</surname><given-names>Z</given-names></name><name><surname>Dai</surname><given-names>J</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>K</given-names></name><name><surname>Yuan</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of tetracycline and chloramphenicol in aqueous solutions by bamboo charcoal: a batch and fixed-bed column study</article-title><source>Chem Eng J</source><year>2013</year><volume>228</volume><fpage>496</fpage><lpage>505</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtVOrtb3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2013.04.118</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Yuan</surname><given-names>D</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yu</surname><given-names>B</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of Fe@porous carbon to be an efficient mediator for peroxymonosulfate activation for oxidation of organic contaminants: incorporation NH2-group into structure of its MOF precursor</article-title><source>Chem Eng J</source><year>2018</year><volume>354</volume><fpage>835</fpage><lpage>848</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGqsLbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2018.08.060</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name><name><surname>Liu Liyuan Tian</surname><given-names>X</given-names></name><name><surname>Liu Longyan Xia</surname><given-names>Y</given-names></name><name><surname>Liang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Tsang</surname><given-names>DCW</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>lkhlaq</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Novel carbon based Fe-Co oxides derived from Prussian blue analogues activating peroxymonosulfate: refractory drugs degradation without metal leaching</article-title><source>Chem Eng J</source><year>2020</year><volume>379</volume><fpage>122274</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVajtLjN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2019.122274</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maheshwari</surname><given-names>U</given-names></name><name><surname>Gupta</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) from wastewater using activated neem bark in a fixed-bed column: interference of other ions and kinetic modelling studies</article-title><source>Desalin Water Treat</source><year>2016</year><volume>57</volume><fpage>8514</fpage><lpage>8525</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXms1Glsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2015.1030709</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Malkoc</surname><given-names>E</given-names></name><name><surname>Nuhoglu</surname><given-names>Y</given-names></name><name><surname>Dundar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of chromium(VI) on pomace-An olive oil industry waste: Batch and column studies</article-title><source>J Hazard Mater</source><year>2006</year><volume>138</volume><fpage>142</fpage><lpage>151</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XhtVOntL3P</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2006.05.051</pub-id><pub-id pub-id-type=\"pmid\">16844293</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mascolo</surname><given-names>G</given-names></name><name><surname>Balest</surname><given-names>L</given-names></name><name><surname>Cassano</surname><given-names>D</given-names></name><name><surname>Laera</surname><given-names>G</given-names></name><name><surname>Lopez</surname><given-names>A</given-names></name><name><surname>Pollice</surname><given-names>A</given-names></name><name><surname>Salerno</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradability of pharmaceutical industrial wastewater and formation of recalcitrant organic compounds during aerobic biological treatment</article-title><source>Bioresour Technol</source><year>2010</year><volume>101</volume><fpage>2585</fpage><lpage>2591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXjvVartQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2009.10.057</pub-id><pub-id pub-id-type=\"pmid\">19939666</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Momčilović</surname><given-names>M</given-names></name><name><surname>Purenović</surname><given-names>M</given-names></name><name><surname>Bojić</surname><given-names>A</given-names></name><name><surname>Zarubica</surname><given-names>A</given-names></name><name><surname>Ranđelović</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Removal of lead(II) ions from aqueous solutions by adsorption onto pine cone activated carbon</article-title><source>Desalination</source><year>2011</year><volume>276</volume><fpage>53</fpage><lpage>59</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXnvFOltro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2011.03.013</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nannou</surname><given-names>C</given-names></name><name><surname>Ofrydopoulou</surname><given-names>A</given-names></name><name><surname>Evgenidou</surname><given-names>E</given-names></name><name><surname>Heath</surname><given-names>D</given-names></name><name><surname>Heath</surname><given-names>E</given-names></name><name><surname>Lambropoulou</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in aquatic environment and wastewater treatment plants: A review on occurrence, fate, removal and ecotoxicity</article-title><source>Sci Total Environ</source><year>2020</year><volume>699</volume><fpage>134322</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVymu7nK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2019.134322</pub-id><pub-id pub-id-type=\"pmid\">31678880</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Fixed-bed column adsorption study: a comprehensive review</article-title><source>Appl Water Sci</source><year>2019</year><volume>9</volume><fpage>1</fpage><lpage>17</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXosVegtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13201-019-0927-7</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Huang</surname><given-names>Q</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Ou</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Profile and behavior of antiviral drugs in aquatic environments of the Pearl River Delta</article-title><source>China Sci Total Environ</source><year>2014</year><volume>466–467</volume><fpage>755</fpage><lpage>761</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslOjs77M</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2013.07.062</pub-id><pub-id pub-id-type=\"pmid\">23973541</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pholosi</surname><given-names>A</given-names></name><name><surname>Naidoo</surname><given-names>EB</given-names></name><name><surname>Ofomaja</surname><given-names>AE</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous flow studies of Cr(VI) adsorption from synthetic and real wastewater by magnetic pine cone composite</article-title><source>Chem Eng Res Des</source><year>2020</year><volume>153</volume><fpage>806</fpage><lpage>818</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitFOjtLvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cherd.2019.11.004</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Schlüsener</surname><given-names>MP</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in wastewater and surface waters: a new pharmaceutical class of environmental relevance?</article-title><source>Environ Sci Technol</source><year>2010</year><volume>44</volume><fpage>1728</fpage><lpage>1735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1Sgsb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es903216p</pub-id><pub-id pub-id-type=\"pmid\">20108960</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Biotransformation of the antiviral drugs acyclovir and penciclovir in activated sludge treatment</article-title><source>Environ Sci Technol</source><year>2011</year><volume>45</volume><fpage>2761</fpage><lpage>2769</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXivFSru7o%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es103732y</pub-id><pub-id pub-id-type=\"pmid\">21388176</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Oxidation of the antiviral drug acyclovir and its biodegradation product carboxy-acyclovir with ozone: kinetics and identification of oxidation products</article-title><source>Environ Sci Technol</source><year>2012</year><volume>46</volume><fpage>2169</fpage><lpage>2178</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsFCis7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es203712z</pub-id><pub-id pub-id-type=\"pmid\">22300376</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reynel-Avila</surname><given-names>HE</given-names></name><name><surname>Mendoza-Castillo</surname><given-names>DI</given-names></name><name><surname>Bonilla-Petriciolet</surname><given-names>A</given-names></name><name><surname>Silvestre-Albero</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of naproxen adsorption on bone char in aqueous solutions using batch and fixed-bed processes</article-title><source>J Mol Liq</source><year>2015</year><volume>209</volume><fpage>187</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1yhur7F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2015.05.013</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russo</surname><given-names>D</given-names></name><name><surname>Siciliano</surname><given-names>A</given-names></name><name><surname>Guida</surname><given-names>M</given-names></name><name><surname>Galdiero</surname><given-names>E</given-names></name><name><surname>Amoresano</surname><given-names>A</given-names></name><name><surname>Andreozzi</surname><given-names>R</given-names></name><name><surname>Reis</surname><given-names>NM</given-names></name><name><surname>Li Puma</surname><given-names>G</given-names></name><name><surname>Marotta</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Photodegradation and ecotoxicology of acyclovir in water under UV254 and UV254/H<sub>2</sub>O<sub>2</sub> processes</article-title><source>Water Res</source><year>2017</year><volume>122</volume><fpage>591</fpage><lpage>602</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWltL%2FO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2017.06.020</pub-id><pub-id pub-id-type=\"pmid\">28628881</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saifi</surname><given-names>Z</given-names></name><name><surname>Rizwanullah</surname><given-names>M</given-names></name><name><surname>Mir</surname><given-names>SR</given-names></name><name><surname>Amin</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Bilosomes nanocarriers for improved oral bioavailability of acyclovir: a complete characterization through in vitro, ex-vivo and in vivo assessment</article-title><source>J Drug Deliv Sci Technol</source><year>2020</year><volume>57</volume><fpage>101634</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXkvFCgsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2020.101634</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>I</given-names></name><name><surname>Adnan</surname><given-names>R</given-names></name><name><surname>Ngah</surname><given-names>WSW</given-names></name><name><surname>Mohamed</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Iron impregnated activated carbon as an efficient adsorbent for the removal of methylene blue: regeneration and kinetics studies</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>1</fpage><lpage>23</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhvFCjsb3O</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0122603</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shetti</surname><given-names>NP</given-names></name><name><surname>Malode</surname><given-names>SJ</given-names></name><name><surname>Nayak</surname><given-names>DS</given-names></name><name><surname>Naik</surname><given-names>RR</given-names></name><name><surname>Kuchinad</surname><given-names>GT</given-names></name><name><surname>Reddy</surname><given-names>KR</given-names></name><name><surname>Shukla</surname><given-names>SS</given-names></name><name><surname>Aminabhavi</surname><given-names>TM</given-names></name></person-group><article-title xml:lang=\"en\">Hetero nanostructured iron oxide and bentonite clay composite assembly for the determination of an antiviral drug acyclovir</article-title><source>Microchem J</source><year>2020</year><volume>155</volume><fpage>104727</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ojsr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.microc.2020.104727</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thiebault</surname><given-names>T</given-names></name><name><surname>Guégan</surname><given-names>R</given-names></name><name><surname>Boussafir</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanisms of emerging micro-pollutants with a clay mineral: Case of tramadol and doxepine pharmaceutical products</article-title><source>J Colloid Interface Sci</source><year>2015</year><volume>453</volume><fpage>1</fpage><lpage>8</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnsFGmuro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2015.04.029</pub-id><pub-id pub-id-type=\"pmid\">25950945</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thommes</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Physical adsorption characterization of nanoporous materials</article-title><source>Chem-Ing-Tech</source><year>2010</year><volume>82</volume><fpage>1059</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXotFemt7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/cite.201000064</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tognarelli</surname><given-names>EI</given-names></name><name><surname>Palomino</surname><given-names>TF</given-names></name><name><surname>Corrales</surname><given-names>N</given-names></name><name><surname>Bueno</surname><given-names>SM</given-names></name><name><surname>Kalergis</surname><given-names>AM</given-names></name><name><surname>González</surname><given-names>PA</given-names></name></person-group><article-title xml:lang=\"en\">Herpes simplex virus evasion of early host antiviral responses</article-title><source>Front Cell Infect Microbiol</source><year>2019</year><pub-id pub-id-type=\"doi\">10.3389/fcimb.2019.00127</pub-id><pub-id pub-id-type=\"pmid\">31114761</pub-id><pub-id pub-id-type=\"pmcid\">6503643</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Ao</surname><given-names>Z</given-names></name><name><surname>Sun</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>An</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Ag<sub>2</sub>MoO<sub>4</sub> nanoparticles encapsulated in g-C<sub>3</sub>N<sub>4</sub> for sunlight photodegradation of pollutants</article-title><source>Catal Today</source><year>2018</year><volume>315</volume><fpage>205</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisF2ltb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cattod.2018.01.019</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>G</given-names></name><name><surname>Viraraghavan</surname><given-names>T</given-names></name><name><surname>Chen</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A new model for heavy metal removal in a biosorption column</article-title><source>Adsorpt Sci Technol</source><year>2001</year><volume>19</volume><fpage>25</fpage><lpage>43</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXks1Ghurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1260/0263617011493953</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoon</surname><given-names>YH</given-names></name><name><surname>Nelson</surname><given-names>JH</given-names></name></person-group><article-title xml:lang=\"en\">Application of gas adsorption kinetics I: a theoretical model for respirator cartridge service life</article-title><source>Am Ind Hyg Assoc J</source><year>1984</year><volume>45</volume><fpage>509</fpage><lpage>516</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaL2cXlvFOhurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/15298668491400197</pub-id><pub-id pub-id-type=\"pmid\">6475758</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Zhu</surname><given-names>H</given-names></name><name><surname>Szewzyk</surname><given-names>U</given-names></name><name><surname>Lübbecke</surname><given-names>S</given-names></name><name><surname>Uwe Geissen</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of emerging organic contaminants with a pilot-scale biofilter packed with natural manganese oxides</article-title><source>Chem Eng J</source><year>2017</year><volume>317</volume><fpage>454</fpage><lpage>460</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXjslWrtbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2017.02.095</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhou</surname><given-names>C</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Xie</surname><given-names>Q</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Fu</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Photolysis of three antiviral drugs acyclovir, zidovudine and lamivudine in surface freshwater and seawater</article-title><source>Chemosphere</source><year>2015</year><volume>138</volume><fpage>792</fpage><lpage>797</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVegtbjF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chemosphere.2015.08.033</pub-id><pub-id pub-id-type=\"pmid\">26295538</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec28\"><title>Supplementary Information</title><p id=\"Par108\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2810_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 1566 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><glossary id=\"AGS1\"><title>List of symbols</title><def-list><def-item><term>∆<italic>G</italic>°</term><def><p id=\"Par3\">Change in Gibbs free energy (kJ/moL)</p></def></def-item><def-item><term>∆<italic>H</italic>°</term><def><p id=\"Par4\">Change in heat of adsorption (kJ/moL)</p></def></def-item><def-item><term>∆pH</term><def><p id=\"Par5\">Change in pH</p></def></def-item><def-item><term>∆<italic>S</italic>°</term><def><p id=\"Par6\">Change in entropy (kJ/mol/K)</p></def></def-item><def-item><term><italic>A</italic></term><def><p id=\"Par7\">Clark constant</p></def></def-item><def-item><term><italic>a</italic></term><def><p id=\"Par8\">Yan model constant</p></def></def-item><def-item><term><italic>A</italic><sub><italic>T</italic></sub></term><def><p id=\"Par9\">Equilibrium binding constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>b</italic></term><def><p id=\"Par10\">Yan model constant</p></def></def-item><def-item><term><italic>b</italic><sub><italic>T</italic></sub></term><def><p id=\"Par11\">Constant related to heat of adsorption</p></def></def-item><def-item><term><italic>B</italic><sub><italic>T</italic></sub></term><def><p id=\"Par12\">Temkin constant related to heat of adsorption <inline-formula id=\"IEq1\"><alternatives><mml:math id=\"IEq1_Math\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>B</mml:mi><mml:mi>T</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>b</mml:mi><mml:mi>T</mml:mi></mml:msub></mml:mrow></mml:mfenced></mml:math><tex-math id=\"IEq1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\left( {B_{T} = \\left( {RT} \\right)/b_{T} } \\right)$$\\end{document}</tex-math><inline-graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_IEq1.gif\"/></alternatives></inline-formula> (J/mol)</p></def></def-item><def-item><term><italic>C</italic></term><def><p id=\"Par13\">Constant representing the thickness of boundary layer (mg/g)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>0</italic></sub></term><def><p id=\"Par14\">Initial concentration of ACVR in solution (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>e</italic></sub></term><def><p id=\"Par15\">Concentration of ACVR at equilibrium (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>f</italic></sub></term><def><p id=\"Par16\">The final concentration of ACVR (mg/L)</p></def></def-item><def-item><term><italic>F</italic></term><def><p id=\"Par17\">Linear velocity or superficial velocity (cm/min)</p></def></def-item><def-item><term><italic>K</italic><sub>0</sub></term><def><p id=\"Par18\">Equilibrium constant</p></def></def-item><def-item><term><italic>k</italic><sub>1</sub></term><def><p id=\"Par19\">Pseudo-first order rate constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>2</sub></term><def><p id=\"Par20\">Pseudo-second-order rate constant (g/mg/min)</p></def></def-item><def-item><term><italic>k</italic><sub>BA</sub></term><def><p id=\"Par21\">Adams–Bohart constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>f</sub></term><def><p id=\"Par22\">Freundlich constant ((mg/g)/(mg/L)<sup>1/n</sup>)</p></def></def-item><def-item><term><italic>K</italic><sub>L</sub></term><def><p id=\"Par23\">Langmuir adsorption constant (L/mg)</p></def></def-item><def-item><term><italic>K</italic><sub><italic>p</italic></sub></term><def><p id=\"Par24\">Rate constant for intraparticle diffusion kinetics (mg/g/min<sup>0.5</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>TH</sub></term><def><p id=\"Par25\">Thomas rate constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>y</sub></term><def><p id=\"Par26\">Yan model constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>YN</sub></term><def><p id=\"Par27\">Yoon–Nelson constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>m</italic></term><def><p id=\"Par28\">Mass of G-AC in column (g)</p></def></def-item><def-item><term><italic>n</italic></term><def><p id=\"Par29\">Adsorption intensity</p></def></def-item><def-item><term><italic>N</italic><sub>0</sub></term><def><p id=\"Par30\">Saturation concentration (mg/L)</p></def></def-item><def-item><term>pH<sub>i</sub></term><def><p id=\"Par31\">Initial pH of the solution</p></def></def-item><def-item><term><italic>Q</italic></term><def><p id=\"Par32\">Volumetric flow rate (mL/min)</p></def></def-item><def-item><term><italic>q</italic><sub>0</sub></term><def><p id=\"Par33\">Yan model maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>e</sub></term><def><p id=\"Par34\">Amount of ACVR adsorbed by adsorbent at equilibrium (mg/g)</p></def></def-item><def-item><term><italic>Q</italic><sub>m</sub></term><def><p id=\"Par35\">Maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub><italic>t</italic></sub></term><def><p id=\"Par36\">Amount of ACVR adsorbed by adsorbent at time t (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>TH</sub></term><def><p id=\"Par37\">Adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>r</italic></term><def><p id=\"Par38\">Clark constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term>RMSE</term><def><p id=\"Par39\">Root-mean-square error</p></def></def-item><def-item><term>SD%</term><def><p id=\"Par40\">Standard deviation percent</p></def></def-item><def-item><term><italic>T</italic></term><def><p id=\"Par41\">Temperature (K)</p></def></def-item><def-item><term><italic>V</italic></term><def><p id=\"Par42\">The initial volume of ACVR solution (L)</p></def></def-item><def-item><term><italic>W</italic></term><def><p id=\"Par43\">The dry mass of adsorbent (g)</p></def></def-item><def-item><term><italic>Z</italic></term><def><p id=\"Par44\">Bed height of column (cm)</p></def></def-item></def-list></glossary><glossary id=\"AGS2\"><title>Greek letters</title><def-list><def-item><term><italic>α</italic></term><def><p id=\"Par45\">Initial sorption rate (mg/g/min)</p></def></def-item><def-item><term><italic>β</italic></term><def><p id=\"Par46\">Extent of surface coverage and activation energy for chemisorption (g/mg)</p></def></def-item><def-item><term><italic>τ</italic></term><def><p id=\"Par47\">Time taken for 50% adsorbate breakthrough (min)</p></def></def-item></def-list></glossary></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02787-3</article-id><article-id pub-id-type=\"manuscript\">2787</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02787-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: In silico investigation of the therapeutic and prophylactic potential of medicinal substances bearing guanidine moieties against COVID-19</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Esam</surname><given-names>Zohreh</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Akhavan</surname><given-names>Malihe</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Lotfi</surname><given-names>Maryam</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Pourmand</surname><given-names>Saeed</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-7155-4532</contrib-id><name><surname>Bekhradnia</surname><given-names>Ahmadreza</given-names></name><address><email>abekhradnia@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027873_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411623.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 2227 0923</institution-id><institution content-type=\"org-division\">Pharmaceutical Sciences Research Centre, Department of Medicinal Chemistry</institution><institution content-type=\"org-name\">Mazandaran University of Medical Sciences</institution></institution-wrap><addr-line content-type=\"city\">Sari</addr-line><country country=\"IR\">Iran</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-division\">The Multiscale Modelling Lab</institution><institution content-type=\"org-name\">ITQB NOVA</institution></institution-wrap><addr-line content-type=\"street\">Av. da República</addr-line><addr-line content-type=\"postcode\">2780-157</addr-line><addr-line content-type=\"city\">Oeiras</addr-line><country country=\"PT\">Portugal</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412831.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 1172 3536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, Faculty of Chemical and Petroleum Engineering</institution><institution content-type=\"org-name\">University of Tabriz</institution></institution-wrap><addr-line content-type=\"city\">Tabriz</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027873_cor5\"><label>e</label><email>abekhradnia@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>30</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"52\">8</issue><fpage>4747</fpage><lpage>4747</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>30</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s11696-022-02528-y\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2787.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Correction to: Chemical Papers (2023) 77:1129–1148</bold><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link></p><p id=\"Par2\">The corresponding author asked if one of the affiliations should be deleted.</p><p id=\"Par3\">The affiliation is given below.</p><p id=\"Par4\">103 CBB, Department of Chemistry and Biochemistry, Montana State University, Bozeman, MT, 59717, USA</p><p id=\"Par5\">The Original Article is now updated.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"retraction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02818-z</article-id><article-id pub-id-type=\"manuscript\">2818</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02818-z</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Retraction Note</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Retraction Note: Nonanuclear coordination complex based on {Co<sub>9</sub>L<sub>12</sub>}<sup>6+</sup> cores: protective effect on coronary heart disease by reducing the inflammatory cytokines releasing</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Dong</surname><given-names>Fengquan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jiang</surname><given-names>Weipeng</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Jiang</surname><given-names>Yajuan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><name><surname>Li</surname><given-names>Xiaozheng</given-names></name><address><email>xiaozheng_li@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor4\">d</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><name><surname>Li</surname><given-names>Haiying</given-names></name><address><email>haiying_li666@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">Department of Cardiology</institution><institution content-type=\"org-name\">Shenzhen University General Hospital</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-name\">Shenzhen University Clinical Medical Academy</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">College of Life Science and Oceanography</institution><institution content-type=\"org-name\">Shenzhen University</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302818z_cor4\"><label>d</label><email>xiaozheng_li@126.com</email></corresp><corresp id=\"IDs1169602302818z_cor5\"><label>e</label><email>haiying_li666@126.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"53\">8</issue><fpage>4749</fpage><lpage>4749</lpage><history><date date-type=\"registration\"><day>6</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"retracted-article\" xlink:href=\"10.1007/s11696-020-01390-0\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2818.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Retraction Note to: Chemical Papers (2020) 75:1355–1362</bold></p><p><ext-link ext-link-type=\"uri\" xlink:href=\"https://doi.org/10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link></p><p id=\"Par2\">The Editor-in-Chief has retracted this article because the ethics documentation provided by the authors was inconsistent with the information in the manuscript. The authors did not provide raw data on request from the journal. In addition, Xiaozheng Li stated that their name was added to the article without their knowledge and they did not contribute to the study. The Editor has therefore lost confidence in the conclusions of this article.</p><p id=\"Par3\">Xiaozheng Li and Haiying Li agree to this retraction. Fengquan Dong, Weipeng Jiang and Yajuan Jiang have not responded to any correspondence from the editor about this retraction.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-019-00721-6</article-id><article-id pub-id-type=\"manuscript\">721</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-019-00721-6</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Meesilp</surname><given-names>Nutthawut</given-names></name><address><email>nutthawutm@hotmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10068019007216_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Mesil</surname><given-names>Nutthisha</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.443999.a</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0504 2111</institution-id><institution content-type=\"org-division\">Department of Applied Biology, Faculty of Science and Liberal Art</institution><institution content-type=\"org-name\">Rajamangala University of Technology Isan</institution></institution-wrap><addr-line content-type=\"postcode\">30000</addr-line><addr-line content-type=\"city\">Nakhon Ratchasima</addr-line><country country=\"TH\">Thailand</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.9786.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 0856</institution-id><institution content-type=\"org-division\">Srinagarind Hospital</institution><institution content-type=\"org-name\">Khon Kaen University</institution></institution-wrap><addr-line content-type=\"postcode\">40002</addr-line><addr-line content-type=\"city\">Khon Kaen</addr-line><country country=\"TH\">Thailand</country></aff></contrib-group><author-notes><corresp id=\"IDs10068019007216_cor1\"><label>a</label><email>nutthawutm@hotmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>6</day><month>12</month><year>2019</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"13\">9</issue><fpage>1297</fpage><lpage>1297</lpage><history><date date-type=\"registration\"><day>23</day><month>11</month><year>2019</year></date><date date-type=\"online\"><day>6</day><month>12</month><year>2019</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2019</copyright-statement><copyright-year>2019</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10068-018-0448-4\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2019</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>11</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>23</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2019_Article_721.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link>.</p></sec></notes></front><body><sec id=\"Sec1\"><title>Correction to: Food Sci Biotechnol (2019) 28(1):289–296 <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link></title><p id=\"Par1\">The article “Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk”, written by “Nutthawut Meesilp and Nutthisha Mesil”, was originally published electronically on the publisher’s Internet portal (currently SpringerLink) on 11 August 2018 with open access.</p><p id=\"Par2\">With the author(s)’ decision to step back from Open Choice, the copyright of the article changed on 6 December 2019 to © The Korean Society of Food Science and Technology 2019 and the article is forthwith distributed under the terms of copyright.</p><p id=\"Par3\">The original article has been corrected.</p></sec></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02795-3</article-id><article-id pub-id-type=\"manuscript\">2795</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02795-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Superior possibilities and upcoming horizons for nanoscience in COVID-19: noteworthy approach for effective diagnostics and management of SARS-CoV-2 outbreak</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Bhattacharjee</surname><given-names>Bedanta</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Ikbal</surname><given-names>Abu Md Ashif</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Farooqui</surname><given-names>Atika</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-5671-6591</contrib-id><name><surname>Sahu</surname><given-names>Ram Kumar</given-names></name><address><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff4\">4</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027953_cor4\">d</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Ruhi</surname><given-names>Sakina</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Syed</surname><given-names>Ayesha</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Miatmoko</surname><given-names>Andang</given-names></name><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Khan</surname><given-names>Danish</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au9\"><name><surname>Khan</surname><given-names>Jiyauddin</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Girijananda Chowdhury Institute of Pharmaceutical Science</institution></institution-wrap><addr-line content-type=\"postcode\">784501</addr-line><addr-line content-type=\"city\">Tezpur</addr-line><addr-line content-type=\"state\">Assam</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411460.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 4538</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Assam University (A Central University)</institution></institution-wrap><addr-line content-type=\"postcode\">788011</addr-line><addr-line content-type=\"city\">Silchar</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.413565.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 2452</institution-id><institution content-type=\"org-name\">The Deccan College of Medical Sciences</institution></institution-wrap><addr-line content-type=\"street\">Kanchan Bagh</addr-line><addr-line content-type=\"postcode\">500058</addr-line><addr-line content-type=\"city\">Hyderabad</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412161.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0681 6439</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Hemvati Nandan Bahuguna Garhwal University (A Central University)</institution></institution-wrap><addr-line content-type=\"street\">Chauras Campus</addr-line><addr-line content-type=\"postcode\">249161</addr-line><addr-line content-type=\"city\">Tehri Garhwal</addr-line><addr-line content-type=\"state\">Uttarakhand</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">Department of Biochemistry</institution><institution content-type=\"org-name\">IMS, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-name\">International Medical School, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440745.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0152 762X</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences, Faculty of Pharmacy</institution><institution content-type=\"org-name\">Universitas Airlangga</institution></institution-wrap><addr-line content-type=\"postcode\">60115</addr-line><addr-line content-type=\"city\">Surabaya</addr-line><addr-line content-type=\"state\">East Java</addr-line><country country=\"ID\">Indonesia</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Panineeya Institute of Dental Science and Research Centre</institution><institution content-type=\"org-name\">Kalonji Narayana Rao University of Health Sciences</institution></institution-wrap><addr-line content-type=\"postcode\">506007</addr-line><addr-line content-type=\"city\">Warangal</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">School of Pharmacy</institution><institution content-type=\"org-name\">Management and Science University</institution></institution-wrap><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027953_cor4\"><label>d</label><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>4</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"3\">8</issue><fpage>4107</fpage><lpage>4130</lpage><history><date date-type=\"registration\"><day>20</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>21</day><month>1</month><year>2023</year></date><date date-type=\"accepted\"><day>18</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>4</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">The outbreak of COVID-19 has caused great havoc and affected many parts of the world. It has imposed a great challenge to the medical and health fraternity with its ability to continue mutating and increasing the transmission rate. Some challenges include the availability of current knowledge of active drugs against the virus, mode of delivery of the medicaments, its diagnosis, which are relatively limited and do not suffice for further prognosis. One recently developed drug delivery system called nanoparticles is currently being utilized in combating COVID-19. This article highlights the existing methods for diagnosis of COVID-19 such as computed tomography scan, reverse transcription-polymerase chain reaction, nucleic acid sequencing, immunoassay, point-of-care test, detection from breath, nanotechnology-based bio-sensors, viral antigen detection, microfluidic device, magnetic nanosensor, magnetic resonance platform and internet-of-things biosensors. The latest detection strategy based on nanotechnology, biosensor, is said to produce satisfactory results in recognizing SARS-CoV-2 virus. It also highlights the successes in the research and development of COVID-19 treatments and vaccines that are already in use. In addition, there are a number of nanovaccines and nanomedicines currently in clinical trials that have the potential to target COVID-19.\n",
       "</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>COVID-19</kwd><kwd>Computer tomography</kwd><kwd>Polymerase chain reaction</kwd><kwd>Biosensor</kwd><kwd>SARS-CoV-2 virus</kwd><kwd>Nanovaccines</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>20</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2795.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Viruses like Middle East Respiratory Syndrome (MERS), Zika, Ebola, Influenza A (H1N1), and Severe Acute Respiratory Syndrome (SARS) have spread around the world in the last 20 years. Health policy and economic relations have been profoundly affected by these pandemics (Boopathi et al. <xref ref-type=\"bibr\" rid=\"CR21\">2021</xref>). The most recent form of the virus is the novel coronavirus, which has been recognized globally as a thoroughly transmissible disease since 2019. It is commonly known as novel COVID-19 (SARS-CoV-2) (Lu and XZ <xref ref-type=\"bibr\" rid=\"CR112\">2020</xref>; Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). The virus was firstly discovered in the Wuhan city of China. The infected patients exhibited unusual pneumonia-like symptoms that later spread globally (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). Coronavirus has caused great concerns due to its rapid diversity and a high number of new cases reported that resulted in huge economic loss worldwide. The structure of coronavirus consists of an envelope of spikes in its outer layer. Its total height is approximately 32 kb, with a single-stranded RNA (Benzigar et al. <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>). At the time of replication of polyprotein translations, it would act on the mRNA. Human SARS-CoV-2 has a spherical morphology with a dimension of around 125 nm (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>) (Daemi et al. <xref ref-type=\"bibr\" rid=\"CR43\">2021</xref>). The coronaviridae family and the nidovirales order comprise the virus evolutionary lineage (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Su et al. <xref ref-type=\"bibr\" rid=\"CR177\">2016</xref>). Coronavirus is subdivided into four main categories, referred to as: alpha (<italic>α</italic>), beta (<italic>β</italic>), gamma (<italic>γ</italic>), and delta (<italic>δ</italic>) and all of which have been recognized to date. Some of the common coronavirus for human are HKU1, NL63, MERS-CoV, OC43, SARS-CoV, and 229E. Alpha coronavirus consists of NL63 and 229E, whereas the SARS-CoV, MERS-CoV, OC43, and HKU1 families are examples of a beta coronavirus (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>). Four crucial structural proteins are encoded by the coronaviral genome, namely spike (S) protein, nucleocapsid (N) protein, membrane (M) protein, and the envelope (E) protein, that collectively constitute a complete viral entity (Masters <xref ref-type=\"bibr\" rid=\"CR120\">2006</xref>). Every protein plays a specialized function in the construction of the viral genome. These proteins have a central position in the replication process as well. The greatest predominant antigen in the viruses is the S glycoprotein, which is called the spikes of the virus (Jaimes et al. <xref ref-type=\"bibr\" rid=\"CR75\">2020</xref>). It has the ability to interact with the receptor and then fuse with the host membranes. The fused viral protein has branches in two different ways: first, it uses the S1 protein of the primary subunit to attach to the receptor. By reserving the cell membranes of the receptor, the S2 subunit would promote infusion. After targeting the virus, it would be structurally modified by the proteolytic enzymes. The peptide would then be incorporated into the cell membrane. Finally, irreversible cyclization would stress the S2 subunit of the fusion protein, which in turn would be mediated by ACE -2 (angiotensin-converting enzyme 2), allowing the viral genome to enter the cytoplasm (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Shafiee et al. <xref ref-type=\"bibr\" rid=\"CR170\">2021</xref>; Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>).<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The SARS-CoV-2 virus impacting humans is depicted in this illustration</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig1_HTML.png\"/></fig><fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Invasion of SARS-CoV-2 via ACE-2. micelles: TPGSM</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig2_HTML.png\"/></fig></p><p id=\"Par3\">COVID-19 is considered a pandemic of the world’s most significant humanitarian catastrophes due to its uncontrollable spread worldwide, huge loss of life, and impact on the world’s economy. Huge efforts have been done to overcome the disease and prevail over the pandemic (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>; Ziegler et al. <xref ref-type=\"bibr\" rid=\"CR223\">2020</xref>; Comentale et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). The most common symptoms among infected people are fever, loss of taste and smell, fatigue, and cough. In severe cases, several other symptoms are also reported, including difficulty breathing, pneumonia, and severe darkening of the lungs. Patients who present with such signs are evaluated by chest radiographs or computed tomography (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>; Yang et al. <xref ref-type=\"bibr\" rid=\"CR211\">2020</xref>; Weiss et al. <xref ref-type=\"bibr\" rid=\"CR200\">2020</xref>). Officially, the coronavirus was termed as SARS-CoV-2 or 2019- nCoV (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Munster et al. <xref ref-type=\"bibr\" rid=\"CR128\">2020</xref>; Liu et al. <xref ref-type=\"bibr\" rid=\"CR109\">2020</xref>; Habibi et al. <xref ref-type=\"bibr\" rid=\"CR59\">2020</xref>; Heymann and Shindo <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR198\">2020</xref>).</p><p id=\"Par4\">Nanotechnology is pertinent in managing and treating a wide range of disorders. Applications of nanomedicine include immune engineering, drug delivery, nanosensors, cancer nanovaccines, and platform technologies (Chakravarty and Vora <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>). Since the existing antiviral agents have truncated bioavailability and a narrow spectrum, nanomaterials are crucial for antiviral treatment. ENMs (engineered nanomaterials) have easy tunable physicochemical, low cost, high efficiency, and efficacy of drug delivery with lesser side effects (Chen et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>). The cell–virus interactions can be adjusted, particularly their interaction with cell receptors or virus particles, by adjusting the shape, size, and surface of ENMs (Ye et al. <xref ref-type=\"bibr\" rid=\"CR212\">2015</xref>). Nanotechnologies can be used for rapid detection of COVID-19 to further improve the efficiency of diagnostic tests (Rabiee et al. <xref ref-type=\"bibr\" rid=\"CR153\">2020</xref>). Organisms with a higher incidence of susceptibility can also be diagnosed using nanomedicines, which act as a prophylactic and alternative means of diagnosing infectious diseases (Vahedifard and Chakravarthy <xref ref-type=\"bibr\" rid=\"CR190\">2021</xref>). Magnetic nanoparticles and quantum dots are the nanoparticles that are commonly employed to detect coronavirus. The protein corona sensor array technology, which would aid in identifying protein/biomolecule sequences at an early stage, is at the forefront of catastrophic COVID-19 (Misra et al. <xref ref-type=\"bibr\" rid=\"CR123\">2021</xref>). The design and implementation of nanotechnology facilitate the production of new nanovaccines with a higher efficacy. One such efficient delivery strategy for treating SARS-CoV-2 is a nanoloaded formulation that is currently being explored. In addition, novel nanovaccines and nanotherapeutics are being developed to address SARS-CoV-2 infection (Jan et al. <xref ref-type=\"bibr\" rid=\"CR76\">2022</xref>). Therefore, researchers are making tremendous efforts to discover novel nano-based technologies and develop relevant nanovaccines and create effective therapies for SARS-CoV-2 infections. This article focuses on the unique and innovative nanotechnology strategies, detection, and prevention against COVID-19 and the potential for future research.</p></sec><sec id=\"Sec2\"><title>Diagnosis of SARS-CoV-2 infestation</title><sec id=\"Sec3\"><title>Computed tomography scan</title><p id=\"Par5\">A computed tomography (CT) scan is used to examine the internal organs. Chest radiographs and 3D cross-sectional images are utilized to identify the symptoms of COVID-19 (Shah et al. <xref ref-type=\"bibr\" rid=\"CR171\">2021</xref>). Patchy lung lesions of small size are seen in the early stages. One of the reports stated that CT scans are more likely to be tactful in the first stage of infection than RT-PCR (Law et al. <xref ref-type=\"bibr\" rid=\"CR100\">2015</xref>). CT scan helps to better understand the condition of the lung and the severity of the disease. It helps the physician to treat the patient appropriately depending on the severity (Bernheim et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). However, the usage of CT scans for COVID-19 has some challenges, including low specificity, which is only about 25%. As with asymptomatic patients, there are also opportunities for uncertainty with CT scans (Alafif et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021</xref>).</p><p id=\"Par6\">Furthermore, the diagnostic images of CT scans could overlap with other diseases like pneumonia. Repeated radiation exposure also would inflict long-term negative effects (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>). Additionally, it is an expensive tool to utilize.</p><p id=\"Par7\">On chest X-rays, it is common to see recurring mottling, subsegmental, or segmental morning opacities in both lungs, and atypical findings are becoming more common (Kaur et al. <xref ref-type=\"bibr\" rid=\"CR83\">2020</xref>). However, CT radiation is associated with minimal carcinogenic risk. It is currently unclear how important CT is for the management of patients with COVID-19. Presently, no antiviral therapy is available for 2019-nCoV, which is not the case in pulmonary TB patients. Drug treatment for tuberculosis can be started without a positive pathogenic Mycobacterium tuberculosis smear or tuberculosis culture if a clinical diagnosis is made that is supported by imaging. In cases of COVID-19, CT may show symptoms similar to those of viral pneumonia, supporting the clinical diagnosis (Zhao et al. <xref ref-type=\"bibr\" rid=\"CR221\">2020</xref>). In contrast, a change from a suspicious to a clinically diagnosed COVID-19 case classification has no impact on clinical therapy (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>).</p></sec><sec id=\"Sec4\"><title>Reverse transcription-polymerase chain reaction (RT-PCR)</title><p id=\"Par8\">RT-PCR would establish the criteria for COVID-19 diagnosis. Although COVID-19 is based on the extent of chronic respiratory complication from viruses, the practical detection approach appears to be less reliable. The SARS-CoV-2 virus is a single-stranded RNA virus with a positive direction of infection. Therefore, the most accurate approach for identification is RT-PCR. This approach requires the conversion of RNA to its supporting DNA, followed by amplification for detection (Harvey and O’Regan <xref ref-type=\"bibr\" rid=\"CR64\">2020</xref>). The E, S, N protein, and the RNA-dependent RNA polymerase gene (RdRp) portions of the entire genome, and ORF1ab are the most common targets for RT-PCR-based detection. These have shown good sensitivity, with the exception of RdRp, as it has lower sensitivity due to reverse primer-template mismatches than the rest (Huang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2020</xref>). The product of PCR amplification analysis is based upon CT (cycle threshold) value, <italic>i.e., </italic>the threshold value found in the duration of the amplification of the viral genome and control, <italic>i.e., </italic>negative control, standard control, and positive control. The positivity criteria of COVID-19 are analyzed based on the measurement of CT score. The CT score is a number that shows how long it took for the total number of PCR sessions to exceed the baseline fluorescence emission value. In the case of infection, the CT score cut-off is between 35 and 40 with a higher CT score indicating a lower viral load in the specimen. Thus, patients with a CT score of less than 35 would be classified as COVID-19 favorable, whereas patients with a CT score greater than 40 would be classified as clinically negative (Lai et al. <xref ref-type=\"bibr\" rid=\"CR97\">2020</xref>; Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>).</p><p id=\"Par9\">The virus accumulates in the nasopharynx, so a swab is taken from these body sites. Various biochemical reagents are used to remove impurities such as lipids and proteins to isolate only the RNA from the sample. The genetic information of the individual and, if applicable, the RNA of the virus are combined to form this RNA. A special enzyme is needed to convert the RNA into DNA. Then, researchers insert micro-DNA fragments that complement specific zones of replicated virulent DNA. When the virus is identified in the sample, these fragments bind to the target sites of the virulent DNA. During the amplification process, some parts of the genome are added to design the DNA sequence. On the other hand, DNA fragments are produced and attached to the sequences with labels to identify the virus. The device RT-PCR is then employed to test the sample (Udugama et al. <xref ref-type=\"bibr\" rid=\"CR188\">2020</xref>). The apparatus heats and cools the samples simultaneously, activating biochemical reactions that result in additional duplicated versions of the virulent DNA target regions (Craw and Balachandran <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). The process is repeated infinitely to copy the viral DNA target regions. A conventional real-time system RT-PCR runs for 35 sessions, which means that each viral strand in the material will have produced approximately 35 billion additional versions of the viral DNA at the end of the process. As additional versions of the virulent DNA fragments are produced, the tags combine with the DNA sequences. The result is the release of a fluorescent dye that is tracked by the computer device and displayed on the monitor in real time. The computer device logs the amount of fluorescence in the sample at the end of each session. The prevalence of the virus is determined when a predetermined fluorescence threshold is exceeded (Rao et al. <xref ref-type=\"bibr\" rid=\"CR158\">2022</xref>). To assess the severity of the disease, experts also monitor the number of sessions required to do so: The fewer sessions required, the more severe the viral contamination (Mohammadniaei et al. <xref ref-type=\"bibr\" rid=\"CR125\">2021</xref>). The real-time PCR method is precise and accurate and can provide a reliable diagnosis within 3 h, whereas laboratory methods typically require 6–8 h. Real-time PCR (RT-PCR) is much faster and has less potential for inaccuracy than other known viral extraction methods because the entire process can be performed in a closed container (Wu et al. <xref ref-type=\"bibr\" rid=\"CR202\">2013</xref>).</p></sec><sec id=\"Sec5\"><title>Nucleic acid sequencing</title><p id=\"Par10\">Detection of SARS-CoV-2 can be sequence-specific based on nucleic acid sequencing. It is considered one of the most precise scientific methods for detecting COVID-19 as it can examine the whole genomic makeup of the virus (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>). Sanger sequencing is a type of sequencing in which a patient's sample is recovered to recover viral RNA and spike-in control RNA that is different from viral RNA. Most of the diagnoses of COVID-19 are based on next-generation sequencing such as oxford nanopore sequencing and illumine sequencing. Unlike Sanger sequencing, the next-generation sequencing would concurrently target billions of target sequences (Quick et al. <xref ref-type=\"bibr\" rid=\"CR152\">2017</xref>). On the other hand, the amount of virus remaining in the specimen is measured and normalized based on the spiked RNA (Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>; Yu et al. <xref ref-type=\"bibr\" rid=\"CR214\">2021</xref>).</p><p id=\"Par11\">Assays based on nucleic acids usually take 12 to 24 h or longer and are thought to be the best way to identify microorganisms. On the other hand, nucleic acid-based assays offer a much shorter reaction time and excellent sensitivity and specificity, bringing them to the forefront of laboratory and, in some circumstances, field analysis. In the laboratory, nucleic acid sequencing has been effectively used to detect both nonpathogenic and pathogenic microorganisms. Ribosomal RNA techniques have been used to conduct similar studies of microbial diversity (Chandler-Brown et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>).</p><p id=\"Par12\">Sequence-based ID with nucleic acid testing is a key part of any defense against biological agent attacks. It helps find out whether a biological agent attack has happened and, if so, what kinds of biological agents were used. Even if the test response time is slow—on the order of 15–30 min—this confirmation function is critical (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p><p id=\"Par13\">The following is the sequence of processes that must occur within this response time: (1) collect the sample: prepare the sample for an assay. This includes the removal of assay inhibitors and lysis of target cells. This can be a serious problem with contaminated samples or when analyzing spore nucleic acids. (2) Performance of the actual assay: analyzing and reporting the assay results (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p></sec><sec id=\"Sec6\"><title>Immunoassay</title><p id=\"Par14\">COVID-19 can be diagnosed indirectly by looking at the patients’ immunological response. Recent research has been conducted to determine serologic surveillance of COVID-19 by collecting various individual fluids (Rizzo and Buck <xref ref-type=\"bibr\" rid=\"CR162\">2012</xref>). The target for serological diagnosis is mainly blood samples instead of the nasopharyngeal swab. Detection of an immune response is the most important requirement for verifying transmission of the virus. Immunoassay is a bioanalytical method based on the interaction between antibody and antigen that allows rapid detection of COVID-19. Two prime serological detections based on immunoassay are enzyme-linked immunosorbent assay (ELISA) and lateral flow assay (LFA). Targeting specific antibodies and antigens is done for the diagnosis of COVID-19. According to studies, a minimum of 5 days is required for sufficient viral load. For antibodies, however, it usually takes at least 7 days, and the concentration decreases after the infection has progressed for 7 days (Alcoba-Florez et al. <xref ref-type=\"bibr\" rid=\"CR8\">2020</xref>). Thus, if the viral load is minimal, this will lead to misleading negative results.</p></sec><sec id=\"Sec7\"><title>Point-of-care test</title><p id=\"Par15\">Point-of-care (POC) testing is essential for disease management during a pandemic. For this reason, it is important to understand the causative agent of a virus before developing tools for diagnosis. It allows for early diagnosis by detecting the analyte, which could optimize the effectiveness of the therapy (Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR84\">2020</xref>). The tools developed for the POC screening of COVID-19 are based on the detection of immunological components such as antibodies and antigen or viral RNA (Ma et al. <xref ref-type=\"bibr\" rid=\"CR115\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> highlights the existing approaches for identifying SARS-CoV-2.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>A list of the existing strategies for SARS-CoV-2 confirmation</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>SARS-CoV-2 confirmation approach</p></th><th align=\"left\"><p>Methodology</p></th><th align=\"left\"><p>Salient characteristics</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Enzymatic sensor system based on regularly interspaced short palindromic repeats for assay-specific great specificity</p></td><td align=\"left\"><p>When SARS-COV-2-specific guide RNA is activated, reporter RNA sequences are removed</p></td><td align=\"left\"><p>Execution time 1 h; inexpensive; simple instrumentation</p></td><td align=\"left\"><p>To et al. (<xref ref-type=\"bibr\" rid=\"CR185\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Enzyme-Linked immunosorbent assay</p></td><td align=\"left\"><p>Antibodies against viral proteins coated on microwell plates bound; Secondary antibody produces Fluorimetric/ Colorimetric signal</p></td><td align=\"left\"><p>High-throughput screening; quick; point of care test</p></td><td align=\"left\"><p>Zhang et al. (<xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>), Chen et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Transcription-mediated amplification</p></td><td align=\"left\"><p>Amplification of particular RNA/DNA segments</p></td><td align=\"left\"><p>Within 15–30 min, there is a ten-billion-fold rise; generates RNA amplicon; rapid kinetics</p></td><td align=\"left\"><p>Phan et al. (<xref ref-type=\"bibr\" rid=\"CR146\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Metagenomic sequencing</p></td><td align=\"left\"><p>Amplification using a single primer that is not sequence-dependent</p></td><td align=\"left\"><p>The SARS-COV-2 mutation rate is analyzed</p></td><td align=\"left\"><p>Carter et al. (<xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Chest Computer Tomography Scans</p></td><td align=\"left\"><p>X-rays of the patient's chest from multiple angles</p></td><td align=\"left\"><p>The imaging characteristics of viral pneumonia are similar. Indeterminate for SARS-CoV-2; non-invasive</p></td><td align=\"left\"><p>Hardick et al. (<xref ref-type=\"bibr\" rid=\"CR63\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Lateral Flow Immunoassay</p></td><td align=\"left\"><p>A sample is passed through an immobilized viral antigen. Color develops in the presence of anti-CoV antibodies, as do marker antibodies</p></td><td align=\"left\"><p>Does not require a professional staff; expensive; quick (10–30 min)</p></td><td align=\"left\"><p>Arslan et al. (<xref ref-type=\"bibr\" rid=\"CR10\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription loop-mediated isothermal amplification</p></td><td align=\"left\"><p>At the same temperature, amplification using four different primer systems</p></td><td align=\"left\"><p>In place of a heating cycler, a heating bath is provided; expeditious (&lt; 13 h); expensive</p></td><td align=\"left\"><p>Calucho et al. (<xref ref-type=\"bibr\" rid=\"CR26\">2020</xref>), Jiang et al. (<xref ref-type=\"bibr\" rid=\"CR78\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription-polymerase chain reaction</p></td><td align=\"left\"><p>Virus genomic RNA amplification</p></td><td align=\"left\"><p>Gold standard; sample requirement is less</p></td><td align=\"left\"><p>Park et al. (<xref ref-type=\"bibr\" rid=\"CR140\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par16\">POC tests are procedures performed with a test strip or kit at home or at the healthcare facility where the person is being treated. The biosensor is the most important component of POC screening because it is required to perform molecular analysis to identify the virus. POC screening has the following advantages: (i) fewer storage and experimental facilities are needed; (ii) it is adaptable to a variety of medical problems; (iii) it is a significant test; and (iv) multiple sites can be used for evaluation (Asdaq et al. <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par17\">Each POC test is approved for use with a specific type of specimen and can only be used with those specimens. Appropriate specimen procurement and handling are required for all COVID-19 assays, especially those performed in POC clinics. An erroneous or inaccurate test result may result from improper specimen submission or collection. Employees who collect specimens within 6 feet of persons must wear the required personal protective equipment (PPE). It consists of gloves, safety glasses, N95 protection, and a lab coat or robe. All personnel collecting specimens not actively involved in specimen collection should follow standard safety guidelines and not be more than one meter away from the patient (Dincer et al. <xref ref-type=\"bibr\" rid=\"CR47\">2017</xref>).</p></sec><sec id=\"Sec8\"><title>Detection from breath</title><p id=\"Par18\">For the objective of monitoring COVID-19, screening of expelled breath may be a less intrusive form of analysis (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Broza et al. <xref ref-type=\"bibr\" rid=\"CR25\">2018</xref>). However, screening SARS-CoV-2 from exhaled air has proven to be quite complicated. These findings show that COVID-19 individuals breathed billions of RNA molecules of severe acute respiratory syndrome coronavirus every extra few minutes (Broza et al. <xref ref-type=\"bibr\" rid=\"CR24\">2015</xref>). According to scientific interpretation, exhaled air had a greater affirmative incidence (26.7%) than surface samples (5.4%) and air (3.8%). In addition, exhaled breath condensate (EBC), a unique technique for quickly identifying viruses from exhaled breath, was required to collect the sample for a longer period of time..</p><p id=\"Par19\">Furthermore, recent studies have shown that non-volatile components such as viruses, bacteria, DNA, and RNA, can be easily detected and observed by extracting and analyzing the aqueous fraction of expelled breath (Expelled Breath Aerosol (EBA) or EBC) (Ma et al. <xref ref-type=\"bibr\" rid=\"CR116\">2021</xref>). Depending on these two factors, an EBC system can effectively detect different droplets: (1) the percentage of the virus still alive after extraction and (2) the ratio between the percentage of extracted particles and the total percentage of particles in the air. For example, Haick and colleagues conducted a scientific investigation in March 2020 in Wuhan city of China, featuring screening with a breath scanner that follows the principle of chemoreceptive devices formed of gold nanoparticles combined with artificial intelligence algorithms (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Lamote et al. <xref ref-type=\"bibr\" rid=\"CR99\">2020</xref>). In another experiment, researchers investigated preliminary markers of enhanced reactive oxygen species formation in swab samples. In this approach, swab samples were exposed to a biosensor nanostructured with carbon nanofibers, which provided 97% accurate confirmatory readings in about 30 s (Shan et al. <xref ref-type=\"bibr\" rid=\"CR172\">2020</xref>).</p></sec><sec id=\"Sec9\"><title>Nanotechnology-based biosensors for COVID-19</title><p id=\"Par20\">Biosensors are devices containing a physical transducer and a biorecognition element that can be used to diagnose the analytes present in solutions, liquids, and bodily fluids. A physical sensor is used to produce a response after an analyte coincides with a biological factor and transforms it into a measurable quantity. Biorecognition elements are glycoproteins (antibodies) or enzymes, nucleic acids such as RNA, DNA, biological receptors, organic receptors, tissues, and whole cells (Miripour et al. <xref ref-type=\"bibr\" rid=\"CR122\">2020</xref>). As per the specifications, enormous numbers of biosensors can be fabricated. Nanomaterials are beneficial as they provide the surface chemistry that would aid in molecular bioconjugation, powerful expanding effect on signals, and large surface energy. Metallic nanoparticles such as silver and gold nanoparticles, nanogels, carbon nanomaterials, <italic>i.e.,</italic> graphene and nanotubes, microgels, and phonic crystals, are utilized in the application of biosensors. Meanwhile, a piezoelectric and electrochemical method based on nanocarbon particles has been developed for the detection of bacteria (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>).</p><p id=\"Par21\">A biosensor detects solutes in bodily fluids, liquids, and by combining a biological sensing device with a mechanical detector. When a solute interferes with a biological component, an output signal is generated, in which the physical detector then converts into a scientifically and quantitatively detectable component. Proteins such as antibodies or enzymes, nucleic acids such as RNA or DNA, physiological receptors, whole tissues, and cells are among the biological detector elements, while electrochemical, photonic, and piezoelectric devices are often used as physical detectors (Bandodkar and Wang <xref ref-type=\"bibr\" rid=\"CR15\">2014</xref>). Biosensor elements can be manufactured in large quantities to meet different requirements. Nanomaterials can be used as detectors due to their excellent resistance in a wide range of environments and bio-compatibility with biological systems (Sharma et al. <xref ref-type=\"bibr\" rid=\"CR174\">2020</xref>). Additionally, nanomaterials have high interfacial potential, robust signal amplification capability, and surface chemistry that can help bioconjugate biomolecules. The properties of photonic crystals, microgels, and magnetic nanostructures including gold nanostructures and silver, carbon-based nanoparticles (nanotubes and graphene), and nanocomposites have contributed to their use in biosensor technologies. The use of nanocarbon-based particles has facilitated microfluidic systems. These have already proven to be platforms for bacterial detection, using a range of strategies such as electrochemistry and piezoelectricity (Olvera and Monaghan <xref ref-type=\"bibr\" rid=\"CR138\">2021</xref>).</p></sec><sec id=\"Sec10\"><title>Development in nanotechnology-based COVID-19 diagnosis</title><p id=\"Par22\">When an infection occurs, immune globulin M (IgM) and immunoglobulin G (IgG) are produced. Initially, IgM antibodies appear, which are reduced to a minimum in general infection. Thereafter, IgG antibodies would develop and displace the IgM antibodies in the body, which remain in high concentration for a considerable time until they are identified after 12 weeks. In SARS-CoV-2 transmission, there is no latency between IgM and IgG confirmation, as both occur within 5–7 days of the onset of manifestations (Tarim et al. <xref ref-type=\"bibr\" rid=\"CR182\">2021</xref>). Several laboratory-based immunoassay methods, such as chemiluminescence immunoassay, can find antibodies in blood serum to the SARS-CoV-2 viral protein. The most commonly utilized method for detection is enzyme-linked immunosorbent assay. However, to identify contamination and conduct an on-site investigation, rapid and portable immunoassay detection is required. One promising, quick and portable platform for the point of care is lateral flow immunoassay, which uses colloidal gold nanoparticles (Deeks et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>). Gold nanostructures are coupled to the SARS-CoV-2 protein and immobilized in the intracellular membrane in a lateral flow immunoassay. The testing for SARS-CoV-2 virus detection would take roughly 15 to 20 min. Due to this, cross-reactivity with other antibodies may result in false-positive outcomes (Han et al. <xref ref-type=\"bibr\" rid=\"CR62\">2019</xref>).</p><p id=\"Par23\">The rate at which antibodies to SARS-CoV-2 are produced is the most important immunologic factor determining susceptibility to infection. RT-PCR can be replaced with immunologic screening to validate SARS-CoV-2. The rate of unequivocal virus detection is significantly increased when immunologic testing and real-time PCR are coupled (Lustig et al. <xref ref-type=\"bibr\" rid=\"CR114\">2021</xref>). In most individuals, IgM levels rise in the first week resulting from exposed to SARS-CoV-2, continue to rise until levels reach a maximum after two weeks, and then begin to decline until levels are near reference values. After one week, IgG is visible and it remains in significant concentration for a longer period of time.</p><p id=\"Par24\">IgG, on the other hand, can be seen after only one week. Its levels stay high for a longer time, probably up to 48 days, and it may help prevent a second infection. Immunoglobulin A (IgA) responses begin to manifest about 4 and 10 days after infection. IgM, IgA, and IgG expression in blood plasma is a diagnostic factor. Different targets can describe the variability of antibodies to SARS-CoV-2. After exposure to infection, antibody titers may decrease for up to 7 days. Specific antibodies to SARS-CoV-2 were recently detected in saliva. Experts investigated whether there are differences in antibody concentrations between serum and saliva using multiplex antibody immunoassays for SARS-CoV-2. The antibodies indicate parallel compartmental humoral immune responses in saliva that correspond to those in serum (Pan et al. <xref ref-type=\"bibr\" rid=\"CR139\">2020</xref>).</p></sec><sec id=\"Sec11\"><title>Viral antigen detection</title><p id=\"Par25\">Prior biosensing strategy would utilize complementary DNA (cDNA) or antibodies to obtain viral RNA or antigen. The existence of SARS-CoV-2 infection is identified by utilizing graphene field-effect transistors (GFET) as a biosensor. It helps in determining the seriousness of the infection (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>). The graphite film serves as the reading region of the FET-based biosensor, which is then transferred to the Si/SiO<sub>2</sub> substrate to successfully modify with the antibody of COVID-19. It is immobilized well on the surface of the graphene sheet by drop-casting. This instrument can detect COVID-19 antigen concentrations as low as 1 fg/ml in a phosphate buffer, which is substantially lower than the PCR/ELISA approach (Nguyen-Contant et al. <xref ref-type=\"bibr\" rid=\"CR134\">2020</xref>). The medium used is universal transport medium (UTM), with the assay used for biosensors for nasopharyngeal swab suspension.</p><p id=\"Par26\">A rapid assay method was developed to confirm the expression of the viral genome released by SARS-CoV-2 in samples from an infected person’s respiratory tract. In this experiment, the protein in the sample adheres to antibodies coupled with a sheet of paper in a glass container. Within half an hour, this reaction produces a detectable signal. The test could indicate intense or initial infection because the proteins found are expressed only when the pathogen is continuously replicating (Drobysh et al. <xref ref-type=\"bibr\" rid=\"CR49\">2022</xref>). In addition, Abbott has introduced a more powerful version of the COVID-19 instant diagnostic test, which screens for antibodies in infected human blood. Abbott's diagnostic devices can detect antibodies to SARS-CoV-2 and are available on the ARCHITECT i1000SR and i2000SR scientific instruments, which can perform 100–200 tests per hour. Antibodies to SARS-CoV-2 are produced after one week of infection. Age, nutritional status, the severity of the disease, concomitant diseases, and medications can influence the intensity of the antibody response (Seo et al. <xref ref-type=\"bibr\" rid=\"CR169\">2020</xref>).</p></sec><sec id=\"Sec12\"><title>Microfluidic device</title><p id=\"Par27\">Another possibility for experiments with real evidence is microfluidic devices. These microfluidic crisps are imprinted with micrometer-sized circuits and reactivity units made of polydimethyl sulfoxide and paper and offer advantages such as reduced sample size, compact dimensions, and rapid detection time. These microfluidic crisps work by blending and separating fluid materials employing electrokinetics and capillary pressure, which is their core principle (Ramachandran et al. <xref ref-type=\"bibr\" rid=\"CR156\">2021</xref>).</p><p id=\"Par28\">Mass screening of people is recognized as a crucial step in limiting the virus proliferation. Some countries in America and Africa reported fewer cases due to the shortage of tests done. Nowadays, quantitative reverse transcriptase polymerase chain reaction (qRTPCR) is applied and endorsed by WHO for case confirmation. qRTPCR (quantitative reverse transcriptase polymerase chain reaction) primer, thermocyclers, specific reagent, and qualified personnel are the most important requirements (Sen et al. <xref ref-type=\"bibr\" rid=\"CR168\">2022</xref>). Microfluidics can be used as an alternative to the time-consuming assay. The emergence of microfluid devices is enabled due to the microelectronics and micro-electromechanical systems as they can manipulate a small number of fluids.</p><p id=\"Par29\">Moreover, they can also retrieve information from the small sample volume. Microfluidic devices are used at the early stage of diagnosis as recent researchers have found that they can detect viruses (Kalita et al. <xref ref-type=\"bibr\" rid=\"CR80\">2021</xref>). The genetic code that forms virus consists of an envelope of proteins and a lipid ligand. Proteins are detected by Western blot, PCR, and amplified genetic material. Many approaches based on benchtop assays are transferred to the microfluidic device to minimize the amount of reagents needed and to speed up the reaction. The microfluidic device would elaborate a structure that can detect the biological samples. The detector that uses the amplification of genetic code requires a specific DNA probe design called primer to find a target sequence in the virus's genetic material (Maggi et al. <xref ref-type=\"bibr\" rid=\"CR117\">2019</xref>).</p></sec><sec id=\"Sec13\"><title>Magnetic nanosensor</title><p id=\"Par30\">Gold nanoislands (AuNIs) were utilized to build plasmonic photothermal (PT) biosensors that target the ORF1ab, RdRp, and E genes to pinpoint SARS-CoV-2 RNA sequences. Nanosensors based on magnetic nanoparticles (MNPs) generate heat as a result of their optical properties when aggregate vibrations in energy concentration take place at the interface of nanomaterial via conjugating to magnetic radiation. The increased frequency of collisions with lattice atoms caused by the enhanced motion of conduction electrons leads to the generation of PT-induced heat (Hofmann et al. <xref ref-type=\"bibr\" rid=\"CR67\">2008</xref>; Moabelo et al. <xref ref-type=\"bibr\" rid=\"CR124\">2021</xref>; Esbin et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>).</p><p id=\"Par31\">Nanosensors based on magnetic nanoparticles (MNPs) are advantageous for viral pathogen identification (Islam and Ahsan <xref ref-type=\"bibr\" rid=\"CR72\">2020</xref>; Barnett et al. <xref ref-type=\"bibr\" rid=\"CR16\">2019</xref>). Tian et al. claim that nanosensors based on MNPs can be used for screening nucleic acids by combining uniform circle-to-circle amplification (UC2CA) and iron oxide NPs (IONPs). In this approach, the effect of an ambient magnetism allows both single and coupled IONPs to exhibit different optical properties (dispersion or absorption). When the recognition markers of the IONPs are paired with UC2CA, the coupled IONPs that produce the end amplicons (single-stranded DNA) are formed. Consequently, the configurations of the IONPs can be used to study the opto-magnetic parameters. This method has been shown to be accurate in distinguishing the genetic variants of SARS-CoV and SARS-CoV-2. In addition, it has been used to detect RdRp motifs for SARS-CoV-2 (Tian et al. <xref ref-type=\"bibr\" rid=\"CR183\">2020</xref>).</p></sec></sec><sec id=\"Sec14\"><title>Nanomedicine</title><p id=\"Par32\">The discipline of medicine known as \"nanomedicine\" exploits nanotechnology to treat and prevent disease. It is utilized in various applications of living organisms, including diagnostics, delivery, and actuation. It is described as using manmade nanodevices and nanostructures to monitor, create, and regulate human biological systems at the molecular level (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR219\">2022</xref>). The unique qualities of nanomedicines may hold the key to solving many of today's problems in cancer, cardiovascular and neurological illnesses, and sickness. The nanomedicine formulation imparts various beneficial properties such as lower side effects, molecular targeting by nanoengineered devices, better absorption with higher intestinal permeability, improved bioavailability, and prolonged and targeted medication delivery. The main benefit of nanomedicines in several current applications is improving drug delivery for therapy. The main mechanism of drug delivery based on nanomedicine called increased bioavailability, and retention is the example of selective medication tailoring (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>). It helps to increase absorbability and can be removed from the body waste before treatment, which can be effective. Nanomedicine is vital in improved efficacy, bioavailability, and targeting ability and is safer than conventional medicines. Nanomedicine influences all medicine sectors and is regarded as a significant tool for vaccinations, medical imaging, innovative diagnostics, nanotherapeutics, and the development of biomaterials for restorative medicine (Ledet and Mandal <xref ref-type=\"bibr\" rid=\"CR102\">2012</xref>). Surfactants (liquid crystals, nanoemulsions, and microemulsions), polymers (polymeric nanoparticles), proteins (protein nanoparticles), and lipids (liposomes, lipid–solid nanoparticles, and nanostructured lipid carriers) have all been employed in nanomedicine. The strength of interactions between biological/tissue molecules and nanoparticles is the foundation of their use in various medicinal applications (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Soft nanomaterials are primarily used to enhance the pharmacokinetic, pharmacodynamics, and biopharmaceutical aspects of drug loading. The performance and tolerability of the therapy may be impacted by the use of nanoparticles to enhance selective therapeutic localization (active or passive) and optimized the release rate of the drug. Due to their antimicrobial qualities, metallic, and polymer-based nanoparticles were also applied in nanomedicine (antiviral, antifungal, antibacterial, and antiparasitic) (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>).</p></sec><sec id=\"Sec15\"><title>Nanomedicine-based drug delivery system</title><p id=\"Par33\">Numerous conditions, including diabetes, cancer, cerebrovascular disease, and neurodegenerative disorder, can be treated with drug delivery systems based on nanotechnology. Some nanomedicines are available in the market through parenteral or oral administration. Novel nanomedicines can be prepared through several preclinical and clinical trials, which can be administered through different routes such as nasal, ocular, vaginal, dermal and pulmonary. Several developed formulations based on nanomaterials are discussed below.</p><sec id=\"Sec16\"><title>Liposomes</title><p id=\"Par34\">Alec Bangham developed liposomes in the year 1960. Cosmetics and pharmaceutical industries have utilized liposomes to transport drugs and other diverse molecules. Liposomes are considered a fixed formulation method to improve the drug delivery system. Liposomes are spherical-shaped vesicles, usually in the 50–450 nm, composed of steroids and phospholipids. They are regarded as a good vehicle for drug delivery due to their ability to incorporate drugs and possess an analogous membrane structure (Kupferschmidt and Cohen <xref ref-type=\"bibr\" rid=\"CR96\">2020</xref>). Liposomes would improve biodistribution and build stable compounds that allow them to utilize hydrophobic and hydrophilic drugs. There are four types of liposomes. The first is a lipid bilayer in which aqueous hydrophilic and hydrophobic substances are surrounded by cationic, anionic, or neutral phospholipids and cholesterol. The second variety is PEGylated liposomes, which consist of PEG (polyethylene glycol) inserted onto their surface to achieve steric equilibrium. The ligand-targeted type is the third kind in which ligands such as carbohydrates, antibodies, and peptides are attached to the surface of liposomes. The fourth form is the theranostic, a combination of the first three types addressed prior that consists of a nanoparticle with imaging, targeting, and therapeutic elements (Bozzuto and Molinari <xref ref-type=\"bibr\" rid=\"CR23\">2015</xref>).</p><p id=\"Par35\">Li et al<italic>.</italic> prepared and characterized a liposomal remdesivir inhalation solution for targeted pulmonary delivery for COVID-19 therapy. The results indicate that liposomal remdesivir shows rapid conversion of remdesivir to active nucleoside triphosphate and its Tmax is 1 h. This observation might be due to the fact that liposomes have high drug loading and are biocompatible with cell membranes. This would lead to faster uptake of remdesivir by cells and in larger amounts (Li et al. <xref ref-type=\"bibr\" rid=\"CR106\">2021</xref>). The central composite design was used to optimize the ritonavir proliposomes, and the results of the physicochemical properties were appropriate. The rats treated with the liposomal formulation exhibited a higher amount of drug in the lymphoid tissues compared with the pure drug. The results suggest that the targeting potential of biotin-coated proliposomes of ritonavir is present (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>). Mice infected with either dead or live SARS-CoV-2 developed acute respiratory distress syndrome (ARDS). Treatment with tripterine liposomes significantly reduced the severity of pulmonary pathological changes and viral load. It was observed that tripterine liposomes reduced SARS-CoV-2 replication and hyperinflammation in mice and cells infected with the virus. Therefore, it is a potential option for a drug that could be used to treat ARDS caused by SARS-CoV-2 (Que et al. <xref ref-type=\"bibr\" rid=\"CR151\">2022</xref>).</p></sec><sec id=\"Sec17\"><title>Polymeric micelle</title><p id=\"Par36\">Micellar nanostructures known as polymeric micelles are composed of an amphiphilic block copolymer that has the ability to self-assemble into a core with a shell when introduced into an aquatic system. For example, docetaxel, camptothecin, and paclitaxel are hydrophobic drugs that can be loaded into the hydrophobic cavity. The hydrophilic coating would stabilize the core and make the system soluble in water when used. To avoid rapid excretion through the kidneys, polymeric micelles have a diameter of about 100 nm and have only a small volume of distribution. Therefore, they can be collected in tumor tissue due to their increased permeability and retention effect.</p><p id=\"Par37\">On the other hand, the hydrophobic medication can be integrated into the core structure, enhancing the drug's bioavailability and stability (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>; Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>). These micelles can be synthesized by precipitating one block and adding a solvent and direct dissolution of polymer in a convenient solvent-based followed by dialysis (Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR208\">2013</xref>). Micelle formation is affected by amphiphilic molecules in the hydrophobic chain, the concentration of amphiphiles, temperature, and solvent system (Kulthe et al. <xref ref-type=\"bibr\" rid=\"CR91\">2012</xref>).</p><p id=\"Par38\">Targeted drug delivery, nanoscopic core structure, minimal biodegradability, micellar coupling, biocompatibility, and much-improved longevity are all attributes of polymeric micelles that make them ideal for drug therapy (Chamundeeswari et al. <xref ref-type=\"bibr\" rid=\"CR29\">2019</xref>). Chaudhari and Handge reported the development, characterization, and validation of lopinavir-encapsulated polymeric micelles using co-solvent (Tween 80) and polymeric material (Pluronic F127 and Pluronic F68) (Chaudhari and Handge <xref ref-type=\"bibr\" rid=\"CR32\">2020</xref>). The novel drug formulation increased drug loading capacity, bioavailability, and encapsulation performance. Mahajan and Patil developed polymeric vitamin E-tocopheryl polyethylene glycol succinate micelles encapsulating lopinavir for oral therapy. The developed lopinavir polymeric micelles had a better dissolution rate. An in vivo pharmacokinetic study in New Zealand rabbits showed that the formulated drug had much higher bioavailability (3.17%) than lopinavir suspension (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>).</p></sec><sec id=\"Sec18\"><title>Dendrimers</title><p id=\"Par39\">Dendrimers are monodispersed 3-D entities that may be cleaved into two halves. They have a spherical shape whose surface can be uniformly functionalized, making them exceptional for drug delivery (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>; Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR86\">2015</xref>). The dendrimers can be synthesized in two ways: the dendrimers would start forming from the core and then outside of the dendrimers. The other method is convergent in which the formation would start from outside the dendrimers (Zhu and Shi <xref ref-type=\"bibr\" rid=\"CR222\">2013</xref>). There are various types of dendrimers based on their functionalization moieties such as polypropylene imine (PPI), polyamidoamine (PAMAM), core–shell, peptide, glycodendrimers, liquid crystalline, chiral and polyamidoamine-organosilicon (PAMAMOS). Since PAMAM is water-soluble and can penetrate epithelial tissues, it is mainly explored for oral drug delivery (Cheng et al. <xref ref-type=\"bibr\" rid=\"CR36\">2008</xref>). However, the applications of dendrimers are limited due to the presence of amino groups'. This group is cationic or positively charged, which makes them toxic. Therefore, the utilization of dendrimers requires some modifications to reduce their toxicities. Drugs are loaded into the dendrimers through electrostatic interaction, simple encapsulation, and covalent conjugation (Noriega-Luna et al. <xref ref-type=\"bibr\" rid=\"CR136\">2014</xref>). Drugs can be delivered in two ways. The first is in vivo delivery, where the degradation of the covalent bond of the dendrimers depends on the favorable environment and the enzymes available to cleave the bond. Ethynylene–perylene (EPer) would discharge the drug due to environmental changes such as temperature and pH (Tripathy and Das <xref ref-type=\"bibr\" rid=\"CR187\">2013</xref>). Drugs can be delivered orally, pulmonary, transdermally, ocularly, and targeted through dendrimers (Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR85\">2014</xref>).\n",
       "</p><p id=\"Par40\">Among the extensively documented studies on the application of dendrimer systems for therapeutic purposes COVID-19, numerous findings can be summarized. According to Itani et al<italic>.,</italic> dendrimers can connect robustly with viruses and COVID-19 suppress invasion in target cells. This comprehensive analysis explored the prospective importance of nanoparticles such as theranostic dendrimers as suitable transporters for immune stimulators or cures to help combat COVID-19 (Prusty and Swain <xref ref-type=\"bibr\" rid=\"CR224\">2018</xref>). In an experiment comparable to Middle East coronavirus, Kandeel and co-workers investigated the antiviral potential of 16 different polyamidoamine dendrimers against SARS-CoV-2. In these intriguing experiments, polycationic dendrimers comprising the primary amine (-NH2-) class (grade 2/3/4 and/or 5) and three different categories of polyanionic dendrimers comprising the amido ethanol hydroxyl class (grade 2/3/4 and/or 5), the sodium carboxylate end group class (grade 1.5/2.5/3.5 and/or 4.5), succinic acid class (grade 2/3/4 and/or 5) were structured and assumed to have calcification inhibitory activity in SARS-CoV-2. The grade 1.5 CO2-Na polyamidoamine dendrimer inhibited SARS-CoV-2 calcification growth most strongly (inhibitory effect 40%), followed by the grade 5 polyamidoamine dendrimer with succinic acid (inhibitory effect 39.7%), which contains terminal grades 16 and 128. Vero cells showed a strong toxic effect in the presence of cationic dendrimers. The authors suggested that antiviral agents could be incorporated into these dendrimers for use as nanomedicines (Itani et al. <xref ref-type=\"bibr\" rid=\"CR73\">2020</xref>).</p></sec><sec id=\"Sec19\"><title>Inorganic nanoparticles</title><p id=\"Par41\">Inorganic nanoparticles, including gold, silver, silica, and iron oxide, possess exceptional potential applications. Gold and silver-like nanoparticles have properties like surface plasmon resonance (SPR), which is not possessed by dendrimers, liposomes, or micelles. They also have several advantages that include versatility and biocompatibility. Drug conjugation with gold nanoparticles can occur through covalent or ionic bonds. In addition, silver nanoparticles can exhibit antimicrobial activities. For example, a spongy and interconnected dextran/polyacrylamide nanohydrogel matrix covalently coupled with AuNPs could release the ornidazole with a 98.5% in vitro (Kandeel et al. <xref ref-type=\"bibr\" rid=\"CR81\">2020</xref>).</p><p id=\"Par42\">Experimental research with iron oxide (IO) nanoparticles targeting SARS-CoV-2 was explored. IO and metal oxide nanoparticles with the versatile properties of zinc oxide, titanium dioxide, and copper oxide are effective in antiviral diagnosis and treatment. Titanium dioxide nanoparticles destroy pathogens by generating reactive oxygen species. This is according to a recent paper that investigated the photocatalytic and antiviral properties of titanium dioxide. It tends to inhibit viruses, structurally destroy DNA, and degrade the phospholipid bilayer (Choudhury et al. <xref ref-type=\"bibr\" rid=\"CR38\">2005</xref>). Silver nanoparticles bind to viral proteins in two ways: (i) they attach to the outer membrane of the virus, preventing it from attaching to the cell surface receptor; and (ii) they attach to the viral nucleic acid (RNA or DNA), limiting transfection and viral replication within the host cell (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>). In addition, the entry of the antiviral transcription factors interferon regulatory factors (IRF)-7 into the nucleus of lung epithelial cells is suppressed by silver nanoparticles and mitochondrial cycling is disrupted (Halbus et al. <xref ref-type=\"bibr\" rid=\"CR60\">2017</xref>). According to a recent finding, SARS-CoV-2 may resist the antiviral effect of silver nanoparticles (Salleh et al. <xref ref-type=\"bibr\" rid=\"CR167\">2020</xref>).</p></sec><sec id=\"Sec20\"><title>Nanoemulsions</title><p id=\"Par43\">Nanoemulsions do not irritate or affect the skin. Therefore, they can effectively improve drug bioavailability (Villeret et al. <xref ref-type=\"bibr\" rid=\"CR194\">2018</xref>). In addition, nanoemulsions consist of small particles with a large surface area, which promotes absorbability (Abd Elkodous et al. <xref ref-type=\"bibr\" rid=\"CR1\">2021</xref>). To understand how some pathogenic viruses are affected by nanoemulsions of different essential oils, several experiments have been carried out. The majority of published studies assume that the nanoemulsions of various essential oils would interfere with the assembly of the virion envelope, thus blocking the attachment of viruses and their penetration into host cells (Naseema et al. <xref ref-type=\"bibr\" rid=\"CR131\">2021</xref>).</p><p id=\"Par44\">Nanoemulsions can compromise virion membrane integrity or obscure viral assembly, preventing SARS-CoV-2 penetration and adhesion into host cells. Furthermore, the encapsulated favipiravir nanoemulsion can suppress the viral polymerase enzyme of SARS-CoV-2, i.e., RNA-dependent RNA polymerase activity in infected cells. In conclusion, we predict that encapsulated favipiravir nanoemulsions could successfully prevent the adhesion of SARS-CoV-2 while disrupting its morphology (Franklyne et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>).</p></sec><sec id=\"Sec21\"><title>Nanosuspensions</title><p id=\"Par45\">Nanosuspensions are submicron dispersions of hydrophobic drug particulates that have been stabilized with surfactants and possess a high affinity for target receptors (de M Ribeiro and Fonseca <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Nanosuspensions have high bioavailability and dissolution rate due to their small size and greater surface area (Jacob et al. <xref ref-type=\"bibr\" rid=\"CR74\">2020</xref>). In the recipient tissues, nanosuspensions can drastically limit viral replication and inhibit the virus from infiltrating host cells (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Due to their extensive surface area and capacity to adhere various antigens to their interface, they are very often used in research. Furthermore, nanomaterials such as carbon quantum dots, gold nanoparticles, and carbon quantum dots have been characterized as both stimulating and preventing virus interaction in the host cells (Noyes and Whitney <xref ref-type=\"bibr\" rid=\"CR137\">1897</xref>).</p><p id=\"Par46\">Nanosuspensions are effective in delivering therapeutics that are lipid and water-insoluble (Szunerits et al. <xref ref-type=\"bibr\" rid=\"CR180\">2015</xref>). Additionally, they increase the drug solubility and rate of dissolution (Murtaza et al. <xref ref-type=\"bibr\" rid=\"CR129\">2020</xref>). Fucoxanthin (a carotenoid antioxidant) is readily soluble in organic solvents, i.e., acetone, and insoluble in water and exhibits excellent consistency and oxidative stability over a wide pH range. Acute toxicity was studied at a dose of 2000 mg/kg (LD50) and confirmed to be non-toxic. Conventional therapeutic administration is limited by the high dosage of fucoxanthin extract and its solubility properties. However, nanosuspension materials could easily overcome these limitations (Yousaf et al. <xref ref-type=\"bibr\" rid=\"CR213\">2021</xref>).</p></sec><sec id=\"Sec22\"><title>Quantum dots</title><p id=\"Par47\">Quantum dots have a size of 2–10 nm and are known as semiconductor nanocrystals. Their optical properties such as photoluminescence and absorption depend on their size (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>). Nowadays, quantum dots have become very important in the field of nanomedicine. The emission of quantum dots is in the infrared range, i.e., below 650 nm. This is due to the lower scattering of light and low absorption in tissue, which makes them desirable for biomedical imaging (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>; Volkov <xref ref-type=\"bibr\" rid=\"CR195\">2015</xref>).</p><p id=\"Par48\">Additionally, different quantum dot sizes and compositions can excite a similar source of light, culminating in different of emitting colors throughout a wide range of spectra (Liu et al. <xref ref-type=\"bibr\" rid=\"CR108\">2010</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR207\">2016</xref>). In this manner, quantum dots are highly captivating for multiplex imaging. Quantum dots are studied as sensors, target drug delivery, and bioimaging in the medicinal field. In literature, the in vivo imaging of the quantum dots is available for its application (Shi et al. <xref ref-type=\"bibr\" rid=\"CR175\">2015</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR61\">2015</xref>).</p><p id=\"Par49\">Carbon-based quantum dots with a cationic external potential could be exploited to inactivate the SARS-CoV-2 S protein (Balaji et al. <xref ref-type=\"bibr\" rid=\"CR14\">2018</xref>). Furthermore, SARS-CoV-2 releases reactive oxygen species due to the binding of quantum dots with positive extrinsic characteristics to the virus-negative RNA motif (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>). In a study conducted by Du et al<italic>.,</italic> it was shown that carbon dots have antiviral properties against respiratory syndrome and pseudorabies virus (Dong et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). In addition, carbon dots trigger interferon-stimulated signaling pathways, including interferon-α formation, which limits viral replication. Integration of targeted substituents into quantum dots could also efficiently attach to the SARS-CoV-2 entry receptor and alter cellular reproduction (Du et al. <xref ref-type=\"bibr\" rid=\"CR50\">2016</xref>).</p></sec><sec id=\"Sec23\"><title>Protein and polysaccharide nanoparticles</title><p id=\"Par50\">Proteins and polysaccharides are natural biopolymers extracted from animals, plants, marine sources, and microorganisms (Iannazzo et al. <xref ref-type=\"bibr\" rid=\"CR70\">2018</xref>). Protein-based nanoparticles are usually metabolized, decomposable and facile for drug and other targeted ligand attachment. There are two ways to produce these nanoparticles. The first method is derived from water-soluble proteins such as human serum albumin and bovine serum albumin. The other option is to use insoluble proteins such as gliadin and zein. The usual synthesis is by desolvation or coacervation, complex coacervation, solvent/emulsion extraction, and electrospraying. These nanoparticles were chemically modified to integrate targeted compounds and localize tissues and cells to support the targeting mechanism (Bassas-Galia et al. <xref ref-type=\"bibr\" rid=\"CR18\">2017</xref>).</p><p id=\"Par51\">Polyguluronate sulfate, carrageenan, chitosan, and some derivatives of marine polysaccharides have potent inhibitory antagonistic potential against several viruses and provide a platform for conducting a study on coronaviruses (Lohcharoenkal et al. <xref ref-type=\"bibr\" rid=\"CR110\">2014</xref>). The most common cause of severe infectious diseases of the upper respiratory tract sometimes referred to as the common cold are respiratory viruses such as influenza viruses, coronaviruses, and rhinoviruses (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>; Tomas et al. <xref ref-type=\"bibr\" rid=\"CR186\">2022</xref>). Clinical studies have shown that using an iota carrageenan nasal spray to relieve the common cold can shorten the severity of the infection. The antiviral efficacy of carrageenan nasal spray was found to be significant for three categories of viruses: influenza A virus, human coronavirus, and human rhinovirus, with the greatest success observed in individuals with human coronavirus. In individuals with coronavirus treated with Carra-Gene, the severity of infection was 3 days shorter (<italic>p</italic> &lt; 0.01), and the frequency of exacerbations was threefold lower (<italic>p</italic> &lt; 0.01) than in untreated individuals (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>). In addition, sulfated polysaccharides such as sulfated rham-nan and fucoidan may impair the development and function of the epidermal growth factor receptor or decrease its signal transduction, which could improve the silencing function of coronavirus (Monto et al. <xref ref-type=\"bibr\" rid=\"CR126\">2001</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR196\">2017</xref>).</p></sec></sec><sec id=\"Sec24\"><title>Role of nanomedicine in the management of COVID-19</title><p id=\"Par52\">One of the most pressing issues over the past pandemics is the absence of a viable therapy for viral diseases such as SARS-CoV-1, MERS-1, and SARS-CoV-2. As a result, repurposed therapeutics are more beneficial than newer drugs developed in treating COVID-19 outbreaks. A new vaccine typically takes a long time to produce, requiring multiple rounds of guidelines and trials. One of the major challenges in successfully treating COVID-19 is insufficient cellular uptake of the newly developed antiviral drugs. Their usage is also limited due to several factors such as low bioavailability, poor water solubility, increased toxicity, lower biological potential, and a high risk-to-benefit ratio (Wang et al. <xref ref-type=\"bibr\" rid=\"CR197\">2018</xref>; Ansari et al. <xref ref-type=\"bibr\" rid=\"CR9\">2020</xref>).</p><p id=\"Par53\">Consequently, these limitations would limit the prospects of new techniques to increase the efficacy of repurposed drugs and their better absorption into the body. To overcome this limitation, a variety of nanomedicine-based systems have been developed to enhance the therapeutic efficacy of newly developed antiviral drugs (Lembo et al. <xref ref-type=\"bibr\" rid=\"CR103\">2018</xref>). Antiviral drugs based on nanomedicine offer several opportunities to overcome the drawbacks of current antiviral therapeutics (Zazo et al. <xref ref-type=\"bibr\" rid=\"CR215\">2016</xref>). Several fundamental obstacles, such as low bioavailability and poor aqueous solubility, can be addressed through nanomedicine design. This involves modifying the pharmacokinetic and pharmacodynamic properties of drugs to achieve lower toxicity and lower dose, improve drug bioavailability, and inhibit viral spread (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>). Additionally, nanomedicine can penetrate cellular membranes and concentrate therapeutic effects in viral persistence sites (Akhter et al. <xref ref-type=\"bibr\" rid=\"CR6\">2012</xref>). One of the most attractive approaches to minimize the adverse effects of viral rebound and patient compliance during the course of therapy is nanomedicine with intrinsically regulated or prolonged release capabilities (Yadav et al. <xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>). Nanocarrier technology could thus be used to repurpose drugs and optimize treatment efficacy COVID-19 (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>).</p><p id=\"Par54\">The upsides of nanocarriers include their high drug loading effectiveness, larger surface area, smaller in size, prolonged drug releasing pattern, and enhanced optimum performance of the loaded pharmaceuticals in such a novel drug delivery system (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>) (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>; Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>). Nanotechnology has a specific physicochemical advantage in medicine due to its unique hydrophobic or hydrophilic nature, nanosize, nanoshape, and the possibility of ligand conjugation on its membrane. These nanoscale physicochemical features would improve pharmaceutical applications' efficiency, specificity, functionality, and sensitivity (Kobayashi et al. <xref ref-type=\"bibr\" rid=\"CR89\">2014</xref>). Several nanomedicine groups could be repurposed to treat COVID-19 therapy, and they are further summarized below. Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> outlines the findings of nanomedicine as a strategy to augment the pharmacological output of several repurposed medications with the potential for COVID-19 treatment.<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Role of nanomedicine drug delivery in COVID-19. Nanomedicine offers numerous characteristics that may be effectively used to transport therapeutic compounds to infected cells, and the tailored ligand-coupled nanoparticle selectively interacts with viral residues, resulting in virus destruction and inability to penetrate the cells </p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig3_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Overview of nanomedicine for improving the pharmacological efficacy of a repurposed medication with COVID-19 management potential</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Drug</p></th><th align=\"left\"><p>Nanocarrier</p></th><th align=\"left\"><p>Model (<italic>In vitro/In vivo</italic>)</p></th><th align=\"left\"><p>Results</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Improved target specificity; bioavailability is significantly elevated in the thymus and spleen</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR141\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased drug transport in the lymphatic system; oral bioavailability is significantly improved; the drug release is regulated through diffusion</p></td><td align=\"left\"><p>Freitas et al. (<xref ref-type=\"bibr\" rid=\"CR55\">2005</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Oral bioavailability is significantly improved</p></td><td align=\"left\"><p>Sercombe et al. (<xref ref-type=\"bibr\" rid=\"CR01\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Enhanced lymphatic targeting specificity in the intestine; bioavailability in the spleen and thymus is significantly higher than in the plasma</p></td><td align=\"left\"><p>Negi et al. (<xref ref-type=\"bibr\" rid=\"CR133\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased bioavailability and <italic>in-vitro</italic> release in the spleen and thymus; improved target specificity</p></td><td align=\"left\"><p>Maniyar and Kokare (<xref ref-type=\"bibr\" rid=\"CR02\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>The drug peak plasma concentration is improved</p></td><td align=\"left\"><p>Kumar et al. (<xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Bioavailability is increased by 6.98 times when compared to a drug suspension</p></td><td align=\"left\"><p>Garg et al. (<xref ref-type=\"bibr\" rid=\"CR56\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Improved Tissue Distribution; improved oral bioavailability by a significant extent; drug degradation protection</p></td><td align=\"left\"><p>Khan et al. (<xref ref-type=\"bibr\" rid=\"CR88\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Ensure that viral replication is actively suppressed; entrapment efficiency has increased; biphasic controlled release</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR159\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PM</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Entrapment efficiency has increased; improved stability</p></td><td align=\"left\"><p>Yadav et al. (<xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ribavirin</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p>C57BL/6N mice</p></td><td align=\"left\"><p>Longer residence time in the mice liver; improved cellular internalization</p></td><td align=\"left\"><p>Javan et al. (<xref ref-type=\"bibr\" rid=\"CR77\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Enhanced drug release pattern; Entrapment efficiency has increased</p></td><td align=\"left\"><p>Ishihara et al. (<xref ref-type=\"bibr\" rid=\"CR71\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Significant build-up in organs such as the spleen and liver; biphasic controlled release; bioavailability are enhanced by 4 times</p></td><td align=\"left\"><p>Katata et al. (<xref ref-type=\"bibr\" rid=\"CR82\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Entrapment efficiency has increased; bioavailability is considerably enhanced</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR160\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>TPGSM</p></td><td align=\"left\"><p>New Zealand rabbits</p></td><td align=\"left\"><p>Relative bioavailability is enhanced by 3.17 times; entrapment efficiency has increased; improved drug dissolution</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR03\">2016</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>Nanostructured lipid carriers: NLC, Solid lipid nanoparticles: SLN, Polymeric micelles: PM, Polymeric nanoparticles: PN, Self-nanoemulsifying drug delivery system: SNEDDS, D-α-Tocopheryl polyethylene glycol succinate micelles: TPGSM</p></table-wrap-foot></table-wrap></p><sec id=\"Sec25\"><title>Liposomes targeted drug delivery system for COVID-19</title><p id=\"Par55\">The liposomes are the bilayers of lipid vesicles and are currently utilized to deliver aquaphobic drugs (Puri et al. <xref ref-type=\"bibr\" rid=\"CR150\">2009</xref>). The liposomes would function as stealth liposomes whereby the systemic circulation is long, targeted, and stimulus-responsive for drug delivery. As liposome is complex and biologically degraded in nature, it is one of the methods of choice for the carrier of drug formulation. Moreover, it is also a productive platform for developing a novel formulation of COVID-19 diseases. Studies have found that synthetic peptides loaded on the liposomes are potential venues for treating SARS coronavirus infection (Lokhande <xref ref-type=\"bibr\" rid=\"CR111\">2018</xref>). Liposomes that are chemically conjugated peptides have been used in the treatment of SARS because they are suitable for the cytotoxic T-lymphocyte strategy that reduces viral load. A novel hydroxychloroquine formulation administered to Sprague–Dawley rats had shown efficacy against COVID-19. Unlike the IV injection, the formulation is breathed and has a greater concentration of medication in the lungs, a longer half-life, and less exposure to other organs such as the heart. Liposome as a carrier system for the delivery of drugs in the treatment of COVID-19 has many advantages. Forty percentage of small molecule drugs used to treat cancer are poorly soluble in water. Therefore, a similar drug delivery method is needed immediately to encapsulate the drug and increase its water solubility (Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Cyclodextrin combination targeted drug delivery system for COVID-19</title><p id=\"Par56\">Cyclodextrin (a biocompatible natural polysaccharide) has α, β, and γ as essential components. According to reports, cyclodextrin nanoparticles are gaining importance due to numerous cyclodextrin components that absorb a greater amount of therapeutics than normal cyclodextrin (Sun et al. <xref ref-type=\"bibr\" rid=\"CR178\">2014</xref>). Cyclodextrin-based remdesivir nanoformulation has just received Food Drug Administration (FDA) approval for the treatment of COVID-19 (Khalaj‐Hedayati et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>). In the clinical context, it has shown encouraging results in various viruses, including Middle East respiratory syndrome coronavirus and Nipah virus (Jones et al. <xref ref-type=\"bibr\" rid=\"CR79\">2020</xref>). In contrast to its function as a medical dispersant for therapeutics, which are primarily hydrophobic, cyclodextrin also possesses antiviral activities (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>). Therefore, cyclodextrin could be used as a delivery strategy for respiratory sprays for therapeutics that require targeted selectively to the pulmonary system (<italic>Khalaj‐Hedayati</italic> et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>).</p></sec><sec id=\"Sec27\"><title>Inorganic and metallic-based nanocomposites targeted drug delivery system for COVID-19</title><p id=\"Par57\">Inorganic nanocomposites for therapeutics delivery can be developed using a variety of metallic nanoparticles, including platinum, silica, silver, gadolinium, gold, and its nanocrystal conglomerates (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR94\">2017</xref>). Silver nanoparticles are known for their extensive antibacterial activity and are an essential ingredient in pharmaceutical formulations, e.g., silver sulfasalazine (a sulfa antibiotic), where silver particles support the antibacterial activity of the sulfasalazine. Silver nanoparticles are also being studied for their possible virucidal properties. It has been observed that SARS-CoV-2 attaches to cell membrane and persists there for up to 12 h, while the silver nanoparticle-based antibacterial film can eradicate the virus (Chaturvedi et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>). This film can be sprayed on metal, glass, and ceramic surfaces to hamper the proliferation of SARS-CoV-2 in busy places such as universities, hospitals, and marketplaces (Prasad et al. <xref ref-type=\"bibr\" rid=\"CR148\">2015</xref>).</p></sec><sec id=\"Sec28\"><title>Polymeric solid colloidal nanoparticles targeted drug delivery system for COVID-19</title><p id=\"Par58\">Due to their unique properties, including targeted drug delivery, controlled drug release, improved tolerability, and efficiency, PN promotes stable <italic>in vivo</italic> delivery of therapeutic agents (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>; Balagna et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Due to their nanosize, they also prevent large dose-dependent side effects and penetration through biological membranes. This favorable property of polymeric nanoparticles can be exploited for drug delivery to revive currently accessible therapeutics for the treatment of the SARS-CoV-2 outbreak. Zhang et al<italic>.</italic> formulated nanoparticles based on PLGA [poly (lactic-co-glycolic acid)] polymers to combat the infection rate of SARS-CoV-2. PLGA was used as the internal base component in a membrane-enveloped nanoparticle derived from human macrophages and human lung epithelial cells of the II type, which exhibit the same physiological processes as SARS-CoV-2 invading recipient cells (Verma et al. <xref ref-type=\"bibr\" rid=\"CR192\">2017</xref>). During the incubation period, these biological nanoparticles serve as a receptive target for SARS-CoV-2, where it is destroyed and can no longer enter recipient cells (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>). Therefore, the polymer-based nanoparticles may provide a nanocarrier platform for the delivery of therapeutics against SARS-CoV-2 contamination.</p></sec><sec id=\"Sec29\"><title>Nano-based vaccines</title><p id=\"Par59\">The sudden appearance of COVID-19 has necessitated the immediate use of drugs to combat the virus. Health professionals have attempted various methods to manage the symptoms. Still, due to certain factors including low solubility, permeability, and the limitation of targeting competence, they could not be used as they were unable to provide the required therapeutic effect. To deliver the drug to the appropriate venue with the fewest adverse effects, nanotechnology has been essential (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR95\">2018</xref>). They could transport the therapeutic molecule to the infected cells, and the targeted liganded nanoparticles would bind with the viral epitopes, deactivating the virus and preventing it from entering the cell. According to the WHO report on COVID-19 candidate vaccines, 52 possible vaccines are undergoing clinical trials, while 162 are in the preclinical stage of development throughout December 8, 2020 (Raj et al. <xref ref-type=\"bibr\" rid=\"CR155\">2021</xref>). The platforms include virus-like particles, DNA, protein subunits, RNA, inactivated viruses, and non-replicating viral vectors. With a pandemic like COVID-19, a huge amount of vaccines is needed, which is not made possible with the existing production of vaccines (Ashraf et al. <xref ref-type=\"bibr\" rid=\"CR12\">2021</xref>). In this case, the noble way of drug delivery has assisted, including reverse vaccinology, nanomedicine, next-generation sequencing, and human challenge studies. The nanoparticle-based vaccination has elicited a strong immunological response that is well-tolerated. Gold, silver sulfide, titanium oxide, zirconium, and graphene are some of the materials that have been considered (Haslberger et al. <xref ref-type=\"bibr\" rid=\"CR65\">2020</xref>). Drug delivery, cancer nanovaccines, immunological engineering, nanosensors, and platform technologies are some techniques employed by nanoresearchers.</p><p id=\"Par60\">In the case of the increasing number of pathogenic bacteria that are resistant to antimicrobial agents, there are several studies that have demonstrated the efficacy of nanotechnology-based antimicrobial therapy (Nikaeen et al. <xref ref-type=\"bibr\" rid=\"CR135\">2020</xref>). Targeted nanotechnology is a new resource for the treatment of antiviral diseases. The employment of nanoparticles in the battle against SARS-CoV-2 has created a mechanism that inhibits the virus's entry into the host cell until it is inactivated. The targeted nanoparticles specific to virus-expressed protein would minimize the internalization of the virus. Metal nanoparticles can prevent the virus from attaching to the surface of the cell and impair the virus's ability to replicate once it enters the cell (Kumar et al. <xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>). The size, shape, and surface charge of nanoparticles would determine how effective the treatment is. However, several safety precautions need to be taken to reduce host cell cytotoxicity. Well-established nanotechnology drug delivery would improve traditional therapies in medicine, as seen in the SARS-CoV-2 outbreak, but its use in viral diseases is still relatively unknown. By shortening PCR assays, reducing detection time, improving sensitivity, amplifying signals, and using them as adjuvants in vaccines, nanostructure technology could help diagnose disease. Nanotechnology has the potential to assist in the pathogenesis of COVID-19 in a variety of ways, including attenuation, possible membrane attachment and fusion upon viral entry, and protein fusion in infected cells. Activating intracellular pathways for reversible virus damage and deactivating viral transcription, translation, and replication may be more effective with nanocapsulated vaccines (Maji et al. <xref ref-type=\"bibr\" rid=\"CR119\">2020</xref>).</p><p id=\"Par61\">To increase half-life and longevity and to elicit adequate cellular and antibody responses, nanoparticles can serve as biologics or cargo for vaccination, sometimes referred to as nanovaccines, among other factors (Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>). The basic principle of this approach is to integrate an epitope that triggers protective immunity into nanoparticles of 1–100 nm and establish an immunological storage that protects the body from emerging harmful microorganisms. Moreover, suitable protein nanoparticles from the architectural proteins of SARS-CoV-2, <italic>i.e.,</italic> envelope (E), membrane (M), nucleocapsid (N), and spike (S), can be transported to target immune systems that would trigger immune feedback and confer durable resistance to the virus. Protein nanoparticles and biologics have already been used for Middle East Coronavirus 1 and Severe Acute Respiratory Syndrome, and they could also be developed for SARS-CoV-2 (Coleman et al. <xref ref-type=\"bibr\" rid=\"CR39\">2014</xref>; Xiang et al. <xref ref-type=\"bibr\" rid=\"CR206\">2006</xref>). Pimentel et al<italic>.</italic> developed a nanovaccine using a repetitive SARS-CoV-1 S protein antigen of around 25 nm to design a targeted antibody that can prevent contamination with SARS-CoV-1 (Pimentel et al. <xref ref-type=\"bibr\" rid=\"CR147\">2009</xref>). Matsuura et al<italic>.</italic> designed virus-like nanoparticles that stimulate a viral protein. We hypothesize that these nanoparticles can also be used to produce SARS-CoV-2 proteins for vaccination applications (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Microparticles (25 nm) depending on epitope selectivity and adjuvant strategies have been utilized to enhance the transport of peptide/protein epitopes to dendritic cells to trigger both cellular and humoral immunological responses in rats (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Numerous nanoparticles, including nanobioceramics, liposomes, cationic polymers, polymer-based particles, gold nanoparticles, virus-like nanoparticles, and peptide nanoparticles, can be used to stimulate lymphatic circulation and produce B lymphocytes (B cells) and T lymphocytes (T cells) for efficient clearance of SARS-CoV-2 (Reddy et al. <xref ref-type=\"bibr\" rid=\"CR161\">2008</xref>; Gregory et al. <xref ref-type=\"bibr\" rid=\"CR58\">2013</xref>; Wilson et al. <xref ref-type=\"bibr\" rid=\"CR201\">2018</xref>; Koshy et al. <xref ref-type=\"bibr\" rid=\"CR90\">2017</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR33\">2016</xref>). Stimulating CD8 + (cytotoxic) T lymphocyte cells or CD4 + (helper) T lymphocyte cells is essential for any proposed vaccination. In the case of coronavirus, T lymphocyte cells have shown the potential to prevent relapse and provide immunologic maintenance for up to 11 years after exposure to SARS-CoV-1 in recovered patients (Goodwin and Huang <xref ref-type=\"bibr\" rid=\"CR57\">2017</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR181\">2011</xref>).</p><p id=\"Par62\">Research shows that DNA vaccination also induces neutralization and preventive resistance to SARS-CoV-1 through upregulation of neutralizing antibodies, clusters of differentiation 4 (CD4 +), and clusters of differentiation 8 (CD8 +), suggesting that DNA vaccination can be delivered with appropriate nanoparticles. In addition, the antigens of B lymphocyte cells and T lymphocyte cells in the immunological domains of SARS-CoV-1 and SARS-CoV-2 were evaluated in a random analysis and were shown to be equivalent for both infections (Peng et al. <xref ref-type=\"bibr\" rid=\"CR145\">2006</xref>). This implies that SARS-CoV-1 vaccinations may be more effective at getting rid of SARS-CoV-2. Numerous properties may be considered for optimizing a SARS-CoV-1 vaccine against SARS-CoV-2, namely enhancing epitope integrity, cytokine production, specific target allocation, prolonged release of biologics and/or epitopes, sequestration of SARS-CoV-2 infections, and prevention of peripheral manifestations of infection (Ahmed et al. <xref ref-type=\"bibr\" rid=\"CR5\">2020</xref>; Rao et al. <xref ref-type=\"bibr\" rid=\"CR157\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> provides an overview of some of the most promising COVID-19 vaccine alternatives that have been formally clinically evaluated (Vijayan et al. <xref ref-type=\"bibr\" rid=\"CR193\">2019</xref>).<table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Currently existing COVID-19 prospective vaccines and their stage of clinical advancement</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Vaccine candidate</p></th><th align=\"left\"><p>Inventors</p></th><th align=\"left\"><p>Summary</p></th><th align=\"left\"><p>Status</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>SARS-CoV-2 recombinant glycoprotein/Matrix M1 nanoparticle vaccine</p></td><td align=\"left\"><p>Novavax United Kingdom</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Russell et al. (<xref ref-type=\"bibr\" rid=\"CR165\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>FINLAY-FR-2 vaccination against SARS-CoV-2</p></td><td align=\"left\"><p>Instituto Finlay de Vacunas</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Puga-Gómez et al. (<xref ref-type=\"bibr\" rid=\"CR149\">2023</xref>)</p></td></tr><tr><td align=\"left\"><p>nCov vaccine</p></td><td align=\"left\"><p>Zydus Cadila</p></td><td align=\"left\"><p>DNA-based vaccine</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Barrett et al. (<xref ref-type=\"bibr\" rid=\"CR17\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>SARS-CoV-2 vaccine (Vero cells)</p></td><td align=\"left\"><p>Chinese Academy of Medical Sciences + Institute of Medical Biology</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Huang et al. (<xref ref-type=\"bibr\" rid=\"CR69\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>mRNA- 1273 and mRNA- 1283</p></td><td align=\"left\"><p>National Institute of Allergy and Infectious Diseases + Moderna</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Patone et al. (<xref ref-type=\"bibr\" rid=\"CR142\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>VAT00002: SARS-CoV-2 spike (S) protein with a biologics</p></td><td align=\"left\"><p>GlaxoSmithKline + Sanofi pasteur</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ndwandwe and Wiysonge (<xref ref-type=\"bibr\" rid=\"CR132\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BNT 162b1 and BNT 162b2</p></td><td align=\"left\"><p>Fosun Pharma + Biontech/Pfizer</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Lamb (<xref ref-type=\"bibr\" rid=\"CR98\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Adeno-based Gam-COVID-Vac (rAd26-S + rAd5-S)</p></td><td align=\"left\"><p>Gamaleya Research Institute, Health Ministry of the Russian Federation</p></td><td align=\"left\"><p>Viral vector (non-proliferating)</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Vaezi and Meysamie (<xref ref-type=\"bibr\" rid=\"CR189\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Recombinant SARS-CoV-2 vaccine (CHO cells)</p></td><td align=\"left\"><p>Institute of Microbiology, Chinese Academy of Sciences + Anhui Zhifei Longcom Biopharmaceutical</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Sun et al. (<xref ref-type=\"bibr\" rid=\"CR179\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BBV152</p></td><td align=\"left\"><p>Bharat Biotech International Limited</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ella et al. (<xref ref-type=\"bibr\" rid=\"CR51\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>ChAdOx1</p></td><td align=\"left\"><p>University of Oxford + AstraZeneca</p></td><td align=\"left\"><p>Vector-based vaccines</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Agrawal et al. (<xref ref-type=\"bibr\" rid=\"CR2\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Ad26.C0V2.S</p></td><td align=\"left\"><p>Janssen Pharmaceuticals</p></td><td align=\"left\"><p>Vector-based vaccines encoding S protein</p></td><td align=\"left\"><p>Phase 3</p></td><td align=\"left\"><p>Bos et al. (<xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p></sec></sec><sec id=\"Sec30\"><title>Challenges in treatment with nano-based medicines</title><p id=\"Par63\">Nanovaccines and nanomedicine that have been utilized in novel medicines to fight against the continuous pandemic, like all current medications, must demonstrate their safety and be studied extensively before their release on the market. As the interaction between engineered nanomaterials (ENMs) with viral targets (envelope, capsid, and nucleic acid) is comprehended by many, the prediction and screening of ENM–virus interactions require a robust analytical framework. Toxicity relevant to inorganic nanoparticles is the limiting factor of their usage in various cases. Other beneficial biological properties can be made by changing the shape, size, and surface chemistry. Moreover, the physicochemical properties must be modified to reduce the side effects of the nanoparticles. ENMs should have a variable dose. Antiviral stability and bioavailability should be considered in many therapeutic applications. The main challenges of nanodelivery come in the form of physiological barriers, which are the mucosa and the alveolar fluid as they are utilized for intranasal as well as the reticuloendothelial structure and used for systemic delivery (Le et al. <xref ref-type=\"bibr\" rid=\"CR101\">2020</xref>). In the cell, the nanoparticle must be able to reject the endocytic degradation component. Certain studies also have shown that nanoparticles could cause damage to respiratory sites and affect lung function. The four key pathobiological aspects of nanoparticles, which include inflammation, genotoxicity, oxidative stress, and fibrosis, must be examined further. Certain studies have yet to prove whether a nasal or oral nanovaccine is safer and provides longer mucosal immunity and viral protection than others. That being said, the ability to provide adequate protection when administered to different age groups by different routes has yet to be confirmed. There are various nanomaterials ranging from polymers to dendrimers to nanoparticles, oligomers, liposomes and tiny molecules. However, once the virus chemical complex dissociates and the viruses are released, they lose their effectiveness and the cycle begins again. The reactive oxygen species formed by nanoparticles would lead to oxidative stress as it can cause a disturbance in mitochondrial respiration. This would cause harmful pathological processes in the cell that cause toxicity. In various studies, it was found that titanium dioxide could enter mice's lungs, causing nanoparticle accumulation to increase the level of lipid peroxidation and minimize the number of antioxidant cells treated by nanoparticles. Due to this, the level of the AP-1 transcription factor increased and affected the transcriptional process. DNA would interact with the carbon nanotubes (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>). Although nanoparticles have gained tremendous importance in research because they are very effective, their toxicological aspects need to be considered. The way nanoparticles act in the human body is of critical importance. When deposited to the maximum, they can cause harmful side effects, especially with platinum-containing drugs (cisplatin, carbapenem) used in the treatment of lungs, colorectal, ovarian, head, neck, and bladder. Due to this, its use is limited as the prominent side effect of it is nephrotoxicity. The major challenge in translating nanodrugs is the first clearance by the RES that minimizes the efficacy and the increasing toxicity in the organs such as the spleen, liver, and kidney (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR210\">2020</xref>). Iron oxide particles have harmful factors both <italic>in vivo</italic> and <italic>in vitro</italic> as they generate reactive oxygen species (ROS).</p><p id=\"Par64\">Nevertheless, the iron oxide is coated with a polymer that would maximize cell viability. When hydroxyl group is added to the gadolinium fullerene particle, it prevents the formation of ROS and excessive release of pro-inflammatory cytokines, which can cause mortality and severe infection. Macrophages and monocyte lineage significantly control the cytokine-facilitated inflammatory response and innate and adaptive response (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>; Shanley et al. <xref ref-type=\"bibr\" rid=\"CR173\">2021</xref>).</p></sec><sec id=\"Sec31\"><title>Future prospects</title><p id=\"Par65\">COVID-19 is a major global threat and the greatest challenge to humanity since World War II. The pandemic has caused many calamities that disrupted health and caused human casualties and other issues. Despite numerous attempts by all countries across the world, including extensive research and numerous clinical trials, no medication has yet been demonstrated to control the virus. This study has highlighted several traditional treatment strategies as well as the crucial function of nanomedicine in the new SARS-CoV-2. Drug substitution, such as remdesivir and chloroquine, is a quick way to achieve safe therapies. Furthermore, several relevant experimental types of research have revealed positive evidence against COVID-19. Nanometal vaccines for example, which are based on the antigenic properties of protein S, have been utilized to reduce viral load. However, a substantial number of studies that support the use of NP-based diagnostic and treatment techniques for SARS-CoV-2 have been published. As a result, all researchers across the world are encouraged to extend their studies in the field of NP-based therapy. There is a great requirement for adequate information on the virus's virulence and transmission to prevent and cure COVID-19 or any other viral pandemic. In the future, this will lead to the discovery of numerous unstructured encoded proteins, enzymes, and functional processes and enable us to understand viral transmission between species. As a result, we could use nanoparticles at the functional level to identify and establish therapeutic targets. As a result, a clinical trial to develop an effective nanovaccine can be conducted with the appropriate knowledge of the viral life cycle and host response. A universal NP-based vaccine with sufficient immunogenicity is the ultimate goal in overcoming the pandemic, as endorsed by researchers worldwide. With factors such as fast detection and mobility, microfluidics would play an essential role in the diagnosis of CoV.</p></sec><sec id=\"Sec32\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par66\">Other than vaccines, there are no approved SARS-CoV-2 therapies being explored in clinical trials. In addition, the nano-based vaccine to treat SARS-CoV-2 has not been adequately studied. All efforts to combat the virus should be explored, and nanotechnology approaches should be prioritized because they may offer new perspectives, particularly in preventing or treating viral internalization compared with standard therapy. To develop a rational design for targeted therapies, further research is also required to comprehend the relationship between CoV and nanoparticles. As the pathophysiology of SARS-CoV-2 is unclear, nanotechnology might be a beneficial technique apart from other methods to produce excellent COVID-19 therapy results.</p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author contributions</title><p>BB and RKS performed conceptualization; BB and AMAI did writing—original draft preparation; AF, SR, AM, AS, DK and JK contributed to writing—review and editing; RKS done supervision. All authors have read and agreed to the published version of the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par67\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References\n",
       "</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abd Elkodous</surname><given-names>M</given-names></name><name><surname>Olojede</surname><given-names>S</given-names></name><name><surname>Morsi</surname><given-names>M</given-names></name><name><surname>El-Sayyad</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Nanomaterial-based drug delivery systems as promising carriers for patients with COVID-19</article-title><source>RSC Adv</source><year>2021</year><volume>11</volume><issue>43</issue><fpage>26463</fpage><lpage>26480</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1ynt7fP</pub-id><pub-id pub-id-type=\"pmid\">35480012</pub-id><pub-id pub-id-type=\"pmcid\">9037715</pub-id><pub-id pub-id-type=\"doi\">10.1039/D1RA04835J</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Agrawal</surname><given-names>U</given-names></name><name><surname>Katikireddi</surname><given-names>SV</given-names></name><name><surname>McCowan</surname><given-names>C</given-names></name><name><surname>Mulholland</surname><given-names>RH</given-names></name><name><surname>Azcoaga-Lorenzo</surname><given-names>A</given-names></name><name><surname>Amele</surname><given-names>S</given-names></name><name><surname>Fagbamigbe</surname><given-names>AF</given-names></name><name><surname>Vasileiou</surname><given-names>E</given-names></name><name><surname>Grange</surname><given-names>Z</given-names></name><name><surname>Shi</surname><given-names>T</given-names></name><name><surname>Kerr</surname><given-names>S</given-names></name><name><surname>Moore</surname><given-names>E</given-names></name><name><surname>Murray</surname><given-names>JLK</given-names></name><name><surname>Shah</surname><given-names>SA</given-names></name><name><surname>Ritchie</surname><given-names>L</given-names></name><name><surname>O'Reilly</surname><given-names>D</given-names></name><name><surname>Stock</surname><given-names>SJ</given-names></name><name><surname>Beggs</surname><given-names>J</given-names></name><name><surname>Chuter</surname><given-names>A</given-names></name><name><surname>Torabi</surname><given-names>F</given-names></name><name><surname>Akbari</surname><given-names>A</given-names></name><name><surname>Bedston</surname><given-names>S</given-names></name><name><surname>McMenamin</surname><given-names>J</given-names></name><name><surname>Wood</surname><given-names>R</given-names></name><name><surname>Tang</surname><given-names>RSM</given-names></name><name><surname>de Lusignan</surname><given-names>S</given-names></name><name><surname>Hobbs</surname><given-names>FDR</given-names></name><name><surname>Woolhouse</surname><given-names>M</given-names></name><name><surname>Simpson</surname><given-names>CR</given-names></name><name><surname>Robertson</surname><given-names>C</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 hospital admissions and deaths after BNT162b2 and ChAdOx1 nCoV-19 vaccinations in 2·57 million people in Scotland (EAVE II): a prospective cohort study</article-title><source>Lancet Respir Med</source><year>2021</year><volume>9</volume><issue>12</issue><fpage>1439</fpage><lpage>1449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFOlsLvJ</pub-id><pub-id pub-id-type=\"pmid\">34599903</pub-id><pub-id pub-id-type=\"pmcid\">8480963</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(21)00380-5</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Paul</surname><given-names>J</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Shavi</surname><given-names>GV</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development and in vivo evaluation of functionalized ritonavir proliposomes for lymphatic targeting</article-title><source>Life Sci</source><year>2017</year><volume>183</volume><fpage>11</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtFSisb7P</pub-id><pub-id pub-id-type=\"pmid\">28647214</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.lfs.2017.06.022</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Alkahtani</surname><given-names>SA</given-names></name><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Ahmad</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>MS</given-names></name><name><surname>Mohsin</surname><given-names>N</given-names></name><name><surname>Abdel-Wahab</surname><given-names>BA</given-names></name></person-group><article-title xml:lang=\"en\">Progress in nanotechnology-based drug carrier in designing of curcumin nanomedicines for cancer therapy: current state-of-the-art</article-title><source>J Drug Target</source><year>2016</year><volume>24</volume><issue>4</issue><fpage>273</fpage><lpage>293</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xhs12qurs%3D</pub-id><pub-id pub-id-type=\"pmid\">26066739</pub-id><pub-id pub-id-type=\"doi\">10.3109/1061186X.2015.1055570</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmed</surname><given-names>SF</given-names></name><name><surname>Quadeer</surname><given-names>AA</given-names></name><name><surname>McKay</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>3</issue><fpage>254</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnvVKlsLk%3D</pub-id><pub-id pub-id-type=\"pmid\">32106567</pub-id><pub-id pub-id-type=\"pmcid\">7150947</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12030254</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Storm</surname><given-names>G</given-names></name><name><surname>Kok</surname><given-names>RJ</given-names></name></person-group><article-title xml:lang=\"en\">Gold nanoparticles in theranostic oncology: current state-of-the-art</article-title><source>Expert Opin Drug Deliv</source><year>2012</year><volume>9</volume><issue>10</issue><fpage>1225</fpage><lpage>1243</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhtl2ltr7N</pub-id><pub-id pub-id-type=\"pmid\">22897613</pub-id><pub-id pub-id-type=\"doi\">10.1517/17425247.2012.716824</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alafif</surname><given-names>T</given-names></name><name><surname>Tehame</surname><given-names>AM</given-names></name><name><surname>Bajaba</surname><given-names>S</given-names></name><name><surname>Barnawi</surname><given-names>A</given-names></name><name><surname>Zia</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Machine and deep learning towards COVID-19 diagnosis and treatment: survey, challenges, and future directions</article-title><source>Int J Environ Res Public Health</source><year>2021</year><volume>18</volume><issue>3</issue><fpage>1117</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlslCjtrw%3D</pub-id><pub-id pub-id-type=\"pmid\">33513984</pub-id><pub-id pub-id-type=\"pmcid\">7908539</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijerph18031117</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alcoba-Florez</surname><given-names>J</given-names></name><name><surname>González-Montelongo</surname><given-names>R</given-names></name><name><surname>Íñigo-Campos</surname><given-names>A</given-names></name><name><surname>de Artola</surname><given-names>DGM</given-names></name><name><surname>Gil-Campesino</surname><given-names>H</given-names></name><name><surname>Team</surname><given-names>TMTS</given-names></name><name><surname>Ciuffreda</surname><given-names>L</given-names></name><name><surname>Valenzuela-Fernández</surname><given-names>A</given-names></name><name><surname>Flores</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Fast SARS-CoV-2 detection by RT-qPCR in preheated nasopharyngeal swab samples</article-title><source>Int J Infect Dis</source><year>2020</year><volume>97</volume><fpage>66</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylurfN</pub-id><pub-id pub-id-type=\"pmid\">32492531</pub-id><pub-id pub-id-type=\"pmcid\">7833505</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2020.05.099</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ansari</surname><given-names>MA</given-names></name><name><surname>Almatroudi</surname><given-names>A</given-names></name><name><surname>Alzohairy</surname><given-names>MA</given-names></name><name><surname>AlYahya</surname><given-names>S</given-names></name><name><surname>Alomary</surname><given-names>MN</given-names></name><name><surname>Al-Dossary</surname><given-names>HA</given-names></name><name><surname>Alghamdi</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Lipid-based nano delivery of Tat-peptide conjugated drug or vaccine–promising therapeutic strategy for SARS-CoV-2 treatment</article-title><source>Expert Opin Drug Deliv</source><year>2020</year><volume>17</volume><issue>12</issue><fpage>1671</fpage><lpage>1674</lpage><pub-id pub-id-type=\"pmid\">32820694</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2020.1813712</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"other\">Arslan S, Delice O, Kahraman M, Yilmaz S, Aslan M (2021) Correlation of chest CT and RT-PCR testing in coronavirus disease 2019 (COVID-19) in Turkey. Ann Clin Anal Med. pp 483–487.</mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Asdaq</surname><given-names>SMB</given-names></name><name><surname>Ikbal</surname><given-names>AMA</given-names></name><name><surname>Sahu</surname><given-names>RK</given-names></name><name><surname>Bhattacharjee</surname><given-names>B</given-names></name><name><surname>Paul</surname><given-names>T</given-names></name><name><surname>Deka</surname><given-names>B</given-names></name><name><surname>Fattepur</surname><given-names>S</given-names></name><name><surname>Widyowati</surname><given-names>R</given-names></name><name><surname>Vijaya</surname><given-names>J</given-names></name><name><surname>Al Mohaini</surname><given-names>M</given-names></name><name><surname>Alsalman</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology integration for SARS-CoV-2 diagnosis and treatment: an approach to preventing pandemic</article-title><source>Nanomaterials</source><year>2021</year><volume>11</volume><issue>7</issue><fpage>1841</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWktrrF</pub-id><pub-id pub-id-type=\"pmid\">34361227</pub-id><pub-id pub-id-type=\"pmcid\">8308419</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano11071841</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ashraf</surname><given-names>MU</given-names></name><name><surname>Kim</surname><given-names>Y</given-names></name><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Seo</surname><given-names>D</given-names></name><name><surname>Ashraf</surname><given-names>M</given-names></name><name><surname>Bae</surname><given-names>YS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines (revisited) and oral-mucosal vector system as a potential vaccine platform</article-title><source>Vaccines</source><year>2021</year><volume>9</volume><issue>2</issue><fpage>171</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVSlsrrO</pub-id><pub-id pub-id-type=\"pmid\">33670630</pub-id><pub-id pub-id-type=\"pmcid\">7922043</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines9020171</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Balagna</surname><given-names>C</given-names></name><name><surname>Perero</surname><given-names>S</given-names></name><name><surname>Percivalle</surname><given-names>E</given-names></name><name><surname>Nepita</surname><given-names>EV</given-names></name><name><surname>Ferraris</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Virucidal effect against coronavirus SARS-CoV-2 of a silver nanocluster/silica composite sputtered coating</article-title><source>Open Ceram</source><year>2020</year><volume>1</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XisFegsL3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.oceram.2020.100006</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Balaji</surname><given-names>AB</given-names></name><name><surname>Pakalapati</surname><given-names>H</given-names></name><name><surname>Khalid</surname><given-names>M</given-names></name><name><surname>Walvekar</surname><given-names>R</given-names></name><name><surname>Siddiqui</surname><given-names>H</given-names></name></person-group><source>Natural and synthetic biocompatible and biodegradable polymers. Biodegradable and biocompatible polymer composites</source><year>2018</year><publisher-loc>Duxford</publisher-loc><publisher-name>Woodhead Publishing</publisher-name><fpage>3</fpage><lpage>32</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-08-100970-3.00001-8</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bandodkar</surname><given-names>AJ</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Non-invasive wearable electrochemical sensors: a review</article-title><source>Trends Biotechnol</source><year>2014</year><volume>32</volume><issue>7</issue><fpage>363</fpage><lpage>371</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXotlyjsrw%3D</pub-id><pub-id pub-id-type=\"pmid\">24853270</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2014.04.005</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barnett</surname><given-names>JM</given-names></name><name><surname>Monnier</surname><given-names>BM</given-names></name><name><surname>Tyler</surname><given-names>S</given-names></name><name><surname>West</surname><given-names>D</given-names></name><name><surname>Ballantine-Dykes</surname><given-names>H</given-names></name><name><surname>Regan</surname><given-names>E</given-names></name><name><surname>Wraith</surname><given-names>P</given-names></name><name><surname>Kiely</surname><given-names>J</given-names></name><name><surname>Luxton</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Initial trail results of a magnetic biosensor for the rapid detection of Porcine Reproductive and Respiratory Virus (PRRSV) infection</article-title><source>Sens Bio-Sens Res</source><year>2019</year><volume>27</volume><pub-id pub-id-type=\"doi\">10.1016/j.sbsr.2019.100315</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barrett</surname><given-names>ADT</given-names></name><name><surname>Titball</surname><given-names>RW</given-names></name><name><surname>MacAry</surname><given-names>PA</given-names></name><name><surname>Rupp</surname><given-names>RE</given-names></name><name><surname>von Messling</surname><given-names>V</given-names></name><name><surname>Walker</surname><given-names>DH</given-names></name><name><surname>Fanget</surname><given-names>NVJ</given-names></name></person-group><article-title xml:lang=\"en\">The rapid progress in COVID vaccine development and implementation</article-title><source>NPJ Vaccines</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>20</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCgsL0%3D</pub-id><pub-id pub-id-type=\"pmid\">35145102</pub-id><pub-id pub-id-type=\"pmcid\">8831501</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-022-00442-8</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"other\">Bassas-Galia M, Follonier, S, Pusnik M, Zinn M (2017) Natural polymers: a source of inspiration. In Bioresorbable polymers for biomedical applications. Elsevier: pp 31–64</mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Benzigar</surname><given-names>MR</given-names></name><name><surname>Bhattacharjee</surname><given-names>R</given-names></name><name><surname>Baharfar</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Current methods for diagnosis of human coronaviruses: pros and cons</article-title><source>Anal Bioanal Chem</source><year>2021</year><volume>413</volume><issue>9</issue><fpage>2311</fpage><lpage>2330</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitl2qs7%2FE</pub-id><pub-id pub-id-type=\"pmid\">33219449</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00216-020-03046-0</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bernheim</surname><given-names>A</given-names></name><name><surname>Mei</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Fayad</surname><given-names>ZA</given-names></name><name><surname>Zhang</surname><given-names>N</given-names></name><name><surname>Diao</surname><given-names>K</given-names></name><name><surname>Lin</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Chest CT findings in coronavirus disease-19 (COVID-19): relationship to duration of infection</article-title><source>Radiology</source><year>2020</year><volume>295</volume><issue>3</issue><fpage>685</fpage><lpage>691</lpage><pub-id pub-id-type=\"doi\">10.1148/radiol.2020200463</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Boopathi</surname><given-names>S</given-names></name><name><surname>Poma</surname><given-names>AB</given-names></name><name><surname>Kolandaivel</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</article-title><source>J Biomol Struct Dyn</source><year>2021</year><volume>39</volume><issue>9</issue><fpage>3409</fpage><lpage>3418</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpvVSgtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">32306836</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bos</surname><given-names>R</given-names></name><name><surname>Rutten</surname><given-names>L</given-names></name><name><surname>van der Lubbe</surname><given-names>JE</given-names></name><name><surname>Bakkers</surname><given-names>MJ</given-names></name><name><surname>Hardenberg</surname><given-names>G</given-names></name><name><surname>Wegmann</surname><given-names>F</given-names></name><name><surname>Zuijdgeest</surname><given-names>D</given-names></name><name><surname>de Wilde</surname><given-names>AH</given-names></name><name><surname>Koornneef</surname><given-names>A</given-names></name><name><surname>Verwilligen</surname><given-names>A</given-names></name><name><surname>van Manen</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Ad26 vector-based COVID-19 vaccine encoding a prefusion-stabilized SARS-CoV-2 Spike immunogen induces potent humoral and cellular immune responses</article-title><source>npj Vaccines</source><year>2020</year><volume>5</volume><issue>1</issue><fpage>91</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFKntr7O</pub-id><pub-id pub-id-type=\"pmid\">33083026</pub-id><pub-id pub-id-type=\"pmcid\">7522255</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-020-00243-x</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bozzuto</surname><given-names>G</given-names></name><name><surname>Molinari</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Liposomes as nanomedical devices</article-title><source>Int J Nanomed</source><year>2015</year><volume>10</volume><fpage>975</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXktl2qs7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJN.S68861</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Mochalski</surname><given-names>P</given-names></name><name><surname>Ruzsanyi</surname><given-names>V</given-names></name><name><surname>Amann</surname><given-names>A</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Hybrid volatolomics and disease detection</article-title><source>Angew Chem Int Ed</source><year>2015</year><volume>54</volume><issue>38</issue><fpage>11036</fpage><lpage>11048</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1KrtrzJ</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201500153</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Vishinkin</surname><given-names>R</given-names></name><name><surname>Barash</surname><given-names>O</given-names></name><name><surname>Nakhleh</surname><given-names>MK</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Synergy between nanomaterials and volatile organic compounds for non-invasive medical evaluation</article-title><source>Chem Soc Rev</source><year>2018</year><volume>47</volume><issue>13</issue><fpage>4781</fpage><lpage>4859</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhtFSgtbvK</pub-id><pub-id pub-id-type=\"pmid\">29888356</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8CS00317C</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"other\">Calucho E (2020) Lateral flow devices for COVID-19-related biomarkers. Biosens Pandemics. Vol. 29</mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Carter</surname><given-names>LJ</given-names></name><name><surname>Garner</surname><given-names>LV</given-names></name><name><surname>Smoot</surname><given-names>JW</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Zhou</surname><given-names>Q</given-names></name><name><surname>Saveson</surname><given-names>CJ</given-names></name><name><surname>Sasso</surname><given-names>JM</given-names></name><name><surname>Gregg</surname><given-names>AC</given-names></name><name><surname>Soares</surname><given-names>DJ</given-names></name><name><surname>Beskid</surname><given-names>TR</given-names></name></person-group><source>Assay techniques and test development for COVID-19 diagnosis</source><year>2020</year><publisher-loc>Washington</publisher-loc><publisher-name>ACS Publications</publisher-name><pub-id pub-id-type=\"doi\">10.1021/acscentsci.0c00501</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chakravarty</surname><given-names>M</given-names></name><name><surname>Vora</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based antiviral therapeutics</article-title><source>Drug Deliv Transl Res</source><year>2021</year><volume>11</volume><issue>3</issue><fpage>748</fpage><lpage>787</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Sgs7%2FN</pub-id><pub-id pub-id-type=\"pmid\">32748035</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13346-020-00818-0</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chamundeeswari</surname><given-names>M</given-names></name><name><surname>Jeslin</surname><given-names>J</given-names></name><name><surname>Verma</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Nanocarriers for drug delivery applications</article-title><source>Environ Chem Lett</source><year>2019</year><volume>17</volume><issue>2</issue><fpage>849</fpage><lpage>865</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXit12ltb7I</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10311-018-00841-1</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"other\">Chandler-Brown D, Bueno AM, Atay O, Tsao DS (2020) A highly scalable and rapidly deployable RNA extraction-free COVID-19 assay by quantitative Sanger sequencing. Biorxiv</mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Yadav</surname><given-names>N</given-names></name><name><surname>Rai</surname><given-names>NK</given-names></name><name><surname>Ellah</surname><given-names>NHA</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name><name><surname>Rehan</surname><given-names>IF</given-names></name><name><surname>Marraiki</surname><given-names>N</given-names></name><name><surname>Batiha</surname><given-names>GES</given-names></name><name><surname>Hetta</surname><given-names>HF</given-names></name><name><surname>Singh</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pleurotus sajor-caju-mediated synthesis of silver and gold nanoparticles active against colon cancer cell lines: a new era of herbonanoceutics</article-title><source>Molecules</source><year>2020</year><volume>25</volume><issue>13</issue><fpage>3091</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsV2ksr3K</pub-id><pub-id pub-id-type=\"pmid\">32645899</pub-id><pub-id pub-id-type=\"pmcid\">7412267</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules25133091</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaudhari</surname><given-names>SP</given-names></name><name><surname>Handge</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Formulation, development and evaluation of lopinavir loaded polymeric micelles</article-title><source>J Sci Technol</source><year>2020</year><volume>5</volume><fpage>173</fpage><lpage>187</lpage></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>HW</given-names></name><name><surname>Huang</surname><given-names>CY</given-names></name><name><surname>Lin</surname><given-names>SY</given-names></name><name><surname>Fang</surname><given-names>ZS</given-names></name><name><surname>Hsu</surname><given-names>CH</given-names></name><name><surname>Lin</surname><given-names>JC</given-names></name><name><surname>Chen</surname><given-names>YI</given-names></name><name><surname>Yao</surname><given-names>BY</given-names></name><name><surname>Hu</surname><given-names>CMJ</given-names></name></person-group><article-title xml:lang=\"en\">Synthetic virus-like particles prepared via protein corona formation enable effective vaccination in an avian model of coronavirus infection</article-title><source>Biomaterials</source><year>2016</year><volume>106</volume><fpage>111</fpage><lpage>118</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtlCqsLvN</pub-id><pub-id pub-id-type=\"pmid\">27552321</pub-id><pub-id pub-id-type=\"pmcid\">7112462</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biomaterials.2016.08.018</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Xu</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral nanoagents: more attention and effort needed?</article-title><source>Nano Today</source><year>2020</year><volume>35</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjtbrP</pub-id><pub-id pub-id-type=\"pmid\">32963581</pub-id><pub-id pub-id-type=\"pmcid\">7498411</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nantod.2020.100976</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Zhai</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>H</given-names></name><name><surname>Bian</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>P</given-names></name><name><surname>Yu</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Rapid and sensitive detection of anti-SARS-CoV-2 IgG, using lanthanide-doped nanoparticles-based lateral flow immunoassay</article-title><source>Anal Chem</source><year>2020</year><volume>92</volume><issue>10</issue><fpage>7226</fpage><lpage>7231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsleju7o%3D</pub-id><pub-id pub-id-type=\"pmid\">32323974</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.analchem.0c00784</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheng</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>Z</given-names></name><name><surname>Ma</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers as drug carriers: applications in different routes of drug administration</article-title><source>J Pharm Sci</source><year>2008</year><volume>97</volume><issue>1</issue><fpage>123</fpage><lpage>143</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXkslOrtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">17721949</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.21079</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cho</surname><given-names>H</given-names></name><name><surname>Lai</surname><given-names>TC</given-names></name><name><surname>Tomoda</surname><given-names>K</given-names></name><name><surname>Kwon</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles for multi-drug delivery in cancer</article-title><source>AAPS Pharm Sci Tech</source><year>2015</year><volume>16</volume><issue>1</issue><fpage>10</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitFait7nO</pub-id><pub-id pub-id-type=\"doi\">10.1208/s12249-014-0251-3</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choudhury</surname><given-names>KR</given-names></name><name><surname>Sahoo</surname><given-names>Y</given-names></name><name><surname>Ohulchanskyy</surname><given-names>T</given-names></name><name><surname>Prasad</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Efficient photoconductive devices at infrared wavelengths using quantum dot-polymer nanocomposites</article-title><source>Appl Phys Lett</source><year>2005</year><volume>87</volume><issue>7</issue><pub-id pub-id-type=\"doi\">10.1063/1.2011768</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Coleman</surname><given-names>CM</given-names></name><name><surname>Liu</surname><given-names>YV</given-names></name><name><surname>Mu</surname><given-names>H</given-names></name><name><surname>Taylor</surname><given-names>JK</given-names></name><name><surname>Massare</surname><given-names>M</given-names></name><name><surname>Flyer</surname><given-names>DC</given-names></name><name><surname>Glenn</surname><given-names>GM</given-names></name><name><surname>Smith</surname><given-names>GE</given-names></name><name><surname>Frieman</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</article-title><source>Vaccine</source><year>2014</year><volume>32</volume><issue>26</issue><fpage>3169</fpage><lpage>3174</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXnsVWltbc%3D</pub-id><pub-id pub-id-type=\"pmid\">24736006</pub-id><pub-id pub-id-type=\"pmcid\">4058772</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2014.04.016</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Comentale</surname><given-names>G</given-names></name><name><surname>Manzo</surname><given-names>R</given-names></name><name><surname>Pilato</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sars-Cov-2 interference in HEME production: is it the time for an early predictive biomarker?</article-title><source>J Mol Med</source><year>2020</year><volume>98</volume><issue>8</issue><fpage>1053</fpage><lpage>1054</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtleks7bO</pub-id><pub-id pub-id-type=\"pmid\">32601795</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00109-020-01945-4</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Corman</surname><given-names>VM</given-names></name><name><surname>Landt</surname><given-names>O</given-names></name><name><surname>Kaiser</surname><given-names>M</given-names></name><name><surname>Molenkamp</surname><given-names>R</given-names></name><name><surname>Meijer</surname><given-names>A</given-names></name><name><surname>Chu</surname><given-names>DK</given-names></name><name><surname>Bleicker</surname><given-names>T</given-names></name><name><surname>Brünink</surname><given-names>S</given-names></name><name><surname>Schneider</surname><given-names>J</given-names></name><name><surname>Schmidt</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</article-title><source>Eurosurveillance</source><year>2020</year><volume>25</volume><issue>3</issue><fpage>2000045</fpage><pub-id pub-id-type=\"pmid\">31992387</pub-id><pub-id pub-id-type=\"pmcid\">6988269</pub-id><pub-id pub-id-type=\"doi\">10.2807/1560-7917.ES.2020.25.3.2000045</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Craw</surname><given-names>P</given-names></name><name><surname>Balachandran</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Isothermal nucleic acid amplification technologies for point-of-care diagnostics: a critical review</article-title><source>Lab Chip</source><year>2012</year><volume>12</volume><issue>14</issue><fpage>2469</fpage><lpage>2486</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XovVylt7k%3D</pub-id><pub-id pub-id-type=\"pmid\">22592150</pub-id><pub-id pub-id-type=\"doi\">10.1039/c2lc40100b</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Daemi</surname><given-names>HB</given-names></name><name><surname>Kulyar</surname><given-names>MF</given-names></name><name><surname>He</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Karimpour</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>X</given-names></name><name><surname>Zou</surname><given-names>Z</given-names></name><name><surname>Jin</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Progression and trends in virus from influenza A to COVID-19: an overview of recent studies</article-title><source>Viruses</source><year>2021</year><volume>13</volume><issue>6</issue><fpage>1145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1OnurrF</pub-id><pub-id pub-id-type=\"pmid\">34203647</pub-id><pub-id pub-id-type=\"pmcid\">8232279</pub-id><pub-id pub-id-type=\"doi\">10.3390/v13061145</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Ribeiro</surname><given-names>LNM</given-names></name><name><surname>Fonseca</surname><given-names>BB</given-names></name></person-group><article-title xml:lang=\"en\">The role of pharmaceutical nanotechnology in the time of COVID-19 pandemic</article-title><source>Future Microbiol</source><year>2020</year><volume>15</volume><issue>16</issue><fpage>1571</fpage><lpage>1582</lpage><pub-id pub-id-type=\"doi\">10.2217/fmb-2020-0118</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Wit</surname><given-names>E</given-names></name><name><surname>Feldmann</surname><given-names>F</given-names></name><name><surname>Cronin</surname><given-names>J</given-names></name><name><surname>Jordan</surname><given-names>R</given-names></name><name><surname>Okumura</surname><given-names>A</given-names></name><name><surname>Thomas</surname><given-names>T</given-names></name><name><surname>Scott</surname><given-names>D</given-names></name><name><surname>Cihlar</surname><given-names>T</given-names></name><name><surname>Feldmann</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Prophylactic and therapeutic remdesivir (GS-5734) treatment in the rhesus macaque model of MERS-CoV infection</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>12</issue><fpage>6771</fpage><lpage>6776</lpage><pub-id pub-id-type=\"pmid\">32054787</pub-id><pub-id pub-id-type=\"pmcid\">7104368</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1922083117</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Deeks</surname><given-names>JJ</given-names></name><name><surname>Dinnes</surname><given-names>J</given-names></name><name><surname>Takwoingi</surname><given-names>Y</given-names></name><name><surname>Davenport</surname><given-names>C</given-names></name><name><surname>Spijker</surname><given-names>R</given-names></name><name><surname>Taylor-Phillips</surname><given-names>S</given-names></name><name><surname>Adriano</surname><given-names>A</given-names></name><name><surname>Beese</surname><given-names>S</given-names></name><name><surname>Dretzke</surname><given-names>J</given-names></name><name><surname>di Ruffano</surname><given-names>LF</given-names></name></person-group><article-title xml:lang=\"en\">Antibody tests for identification of current and past infection with SARS-CoV-2</article-title><source>Cochrane Database Syst Rev</source><year>2020</year><volume>2020</volume><fpage>CD013652</fpage><pub-id pub-id-type=\"pmcid\">7387103</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dincer</surname><given-names>C</given-names></name><name><surname>Bruch</surname><given-names>R</given-names></name><name><surname>Kling</surname><given-names>A</given-names></name><name><surname>Dittrich</surname><given-names>PS</given-names></name><name><surname>Urban</surname><given-names>GA</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed point-of-care testing–xPOCT</article-title><source>Trends Biotechnol</source><year>2017</year><volume>35</volume><issue>8</issue><fpage>728</fpage><lpage>742</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmtFGns70%3D</pub-id><pub-id pub-id-type=\"pmid\">28456344</pub-id><pub-id pub-id-type=\"pmcid\">5538621</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2017.03.013</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dong</surname><given-names>X</given-names></name><name><surname>Moyer</surname><given-names>MM</given-names></name><name><surname>Yang</surname><given-names>F</given-names></name><name><surname>Sun</surname><given-names>YP</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots’ antiviral functions against noroviruses</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>10</lpage></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Drobysh</surname><given-names>M</given-names></name><name><surname>Ramanaviciene</surname><given-names>A</given-names></name><name><surname>Viter</surname><given-names>R</given-names></name><name><surname>Chen</surname><given-names>CF</given-names></name><name><surname>Samukaite-Bubniene</surname><given-names>U</given-names></name><name><surname>Ratautaite</surname><given-names>V</given-names></name><name><surname>Ramanavicius</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Biosensors for the determination of SARS-CoV-2 virus and diagnosis of COVID-19 infection</article-title><source>Int J Mol Sci</source><year>2022</year><volume>23</volume><issue>2</issue><fpage>666</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitFertr4%3D</pub-id><pub-id pub-id-type=\"pmid\">35054850</pub-id><pub-id pub-id-type=\"pmcid\">8776074</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms23020666</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Liang</surname><given-names>J</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Liu</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots as inhibitors of virus by activation of type I interferon response</article-title><source>Carbon</source><year>2016</year><volume>110</volume><fpage>278</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFCmtLnF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbon.2016.09.032</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ella</surname><given-names>R</given-names></name><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>Blackwelder</surname><given-names>W</given-names></name><name><surname>Potdar</surname><given-names>V</given-names></name><name><surname>Yadav</surname><given-names>P</given-names></name><name><surname>Sarangi</surname><given-names>V</given-names></name><name><surname>Aileni</surname><given-names>VK</given-names></name><name><surname>Kanungo</surname><given-names>S</given-names></name><name><surname>Rai</surname><given-names>S</given-names></name><name><surname>Reddy</surname><given-names>P</given-names></name><name><surname>Verma</surname><given-names>S</given-names></name><name><surname>Singh</surname><given-names>C</given-names></name><name><surname>Redkar</surname><given-names>S</given-names></name><name><surname>Mohapatra</surname><given-names>S</given-names></name><name><surname>Pandey</surname><given-names>A</given-names></name><name><surname>Ranganadin</surname><given-names>P</given-names></name><name><surname>Gumashta</surname><given-names>R</given-names></name><name><surname>Multani</surname><given-names>M</given-names></name><name><surname>Mohammad</surname><given-names>S</given-names></name><name><surname>Bhatt</surname><given-names>P</given-names></name><name><surname>Kumari</surname><given-names>L</given-names></name><name><surname>Sapkal</surname><given-names>G</given-names></name><name><surname>Gupta</surname><given-names>N</given-names></name><name><surname>Abraham</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>S</given-names></name><name><surname>Prasad</surname><given-names>S</given-names></name><name><surname>Bhargava</surname><given-names>B</given-names></name><name><surname>Ella</surname><given-names>K</given-names></name><name><surname>Vadrevu</surname><given-names>KM</given-names></name><collab>COVAXIN Study Group</collab></person-group><article-title xml:lang=\"en\">Efficacy, safety, and lot-to-lot immunogenicity of an inactivated SARS-CoV-2 vaccine (BBV152): interim results of a randomised, double-blind, controlled, phase 3 trial</article-title><source>Lancet (London, England)</source><year>2021</year><volume>398</volume><issue>10317</issue><fpage>2173</fpage><lpage>2184</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1eqsrfE</pub-id><pub-id pub-id-type=\"pmid\">34774196</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(21)02000-6</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Esbin</surname><given-names>MN</given-names></name><name><surname>Whitney</surname><given-names>ON</given-names></name><name><surname>Chong</surname><given-names>S</given-names></name><name><surname>Maurer</surname><given-names>A</given-names></name><name><surname>Darzacq</surname><given-names>X</given-names></name><name><surname>Tjian</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Overcoming the bottleneck to widespread testing: a rapid review of nucleic acid testing approaches for COVID-19 detection</article-title><source>RNA</source><year>2020</year><volume>26</volume><issue>7</issue><fpage>771</fpage><lpage>783</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Klt7rP</pub-id><pub-id pub-id-type=\"pmid\">32358057</pub-id><pub-id pub-id-type=\"pmcid\">7297120</pub-id><pub-id pub-id-type=\"doi\">10.1261/rna.076232.120</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Flühmann</surname><given-names>B</given-names></name><name><surname>Ntai</surname><given-names>I</given-names></name><name><surname>Borchard</surname><given-names>G</given-names></name><name><surname>Simoens</surname><given-names>S</given-names></name><name><surname>Mühlebach</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicines: the magic bullets reaching their target?</article-title><source>Eur J Pharm Sci</source><year>2019</year><volume>128</volume><fpage>73</fpage><lpage>80</lpage><pub-id pub-id-type=\"pmid\">30465818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2018.11.019</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Franklyne</surname><given-names>JS</given-names></name><name><surname>Gopinath</surname><given-names>PM</given-names></name><name><surname>Mukherjee</surname><given-names>A</given-names></name><name><surname>Chandrasekaran</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Nanoemulsions: the rising star of antiviral therapeutics and nanodelivery system—Current status and prospects</article-title><source>Curr Opin Colloid Interface Sci</source><year>2021</year><volume>54</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXptlansro%3D</pub-id><pub-id pub-id-type=\"pmid\">33814954</pub-id><pub-id pub-id-type=\"pmcid\">8007535</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cocis.2021.101458</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Freitas</surname><given-names>RA</given-names><suffix>Jr</suffix></name></person-group><article-title xml:lang=\"en\">What is nanomedicine? Nanomedicine: nanotechnology</article-title><source>Biology Med</source><year>2005</year><volume>1</volume><issue>1</issue><fpage>2</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXlvVSrurs%3D</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Garg</surname><given-names>B</given-names></name><name><surname>Beg</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>R</given-names></name><name><surname>Katara</surname><given-names>O</given-names></name><name><surname>Singh</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructured lipidic carriers of lopinavir for effective management of HIV-associated neurocognitive disorder</article-title><source>J Drug Deliv Sci Technol</source><year>2019</year><volume>53</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhs1artbfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2019.101220</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Goodwin</surname><given-names>TJ</given-names></name><name><surname>Huang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of phosphorylated adjuvants co-encapsulated with a model cancer peptide antigen for the treatment of colorectal cancer and liver metastasis</article-title><source>Vaccine</source><year>2017</year><volume>35</volume><issue>19</issue><fpage>2550</fpage><lpage>2557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsFagt78%3D</pub-id><pub-id pub-id-type=\"pmid\">28385609</pub-id><pub-id pub-id-type=\"pmcid\">5448460</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2017.03.067</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gregory</surname><given-names>AE</given-names></name><name><surname>Titball</surname><given-names>R</given-names></name><name><surname>Williamson</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Vaccine delivery using nanoparticles</article-title><source>Front Incell Infect Microbiol</source><year>2013</year><volume>3</volume><fpage>13</fpage></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Habibi</surname><given-names>R</given-names></name><name><surname>Burci</surname><given-names>GL</given-names></name><name><surname>De Campos</surname><given-names>TC</given-names></name><name><surname>Chirwa</surname><given-names>D</given-names></name><name><surname>Cinà</surname><given-names>M</given-names></name><name><surname>Dagron</surname><given-names>S</given-names></name><name><surname>Eccleston-Turner</surname><given-names>M</given-names></name><name><surname>Forman</surname><given-names>L</given-names></name><name><surname>Gostin</surname><given-names>LO</given-names></name><name><surname>Mier</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">Do not violate the international health regulations during the COVID-19 outbreak</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10225</issue><fpage>664</fpage><lpage>666</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXksVaisLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30373-1</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halbus</surname><given-names>AF</given-names></name><name><surname>Horozov</surname><given-names>TS</given-names></name><name><surname>Paunov</surname><given-names>VN</given-names></name></person-group><article-title xml:lang=\"en\">Colloid particle formulations for antimicrobial applications</article-title><source>Adv Coll Interface Sci</source><year>2017</year><volume>249</volume><fpage>134</fpage><lpage>148</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXotVarsrY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2017.05.012</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>HS</given-names></name><name><surname>Niemeyer</surname><given-names>E</given-names></name><name><surname>Huang</surname><given-names>Y</given-names></name><name><surname>Kamoun</surname><given-names>WS</given-names></name><name><surname>Martin</surname><given-names>JD</given-names></name><name><surname>Bhaumik</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Roberge</surname><given-names>S</given-names></name><name><surname>Cui</surname><given-names>J</given-names></name><name><surname>Martin</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dot/antibody conjugates for in vivo cytometric imaging in mice</article-title><source>Proc Natl Acad Sci</source><year>2015</year><volume>112</volume><issue>5</issue><fpage>1350</fpage><lpage>1355</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtFKhtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">25605916</pub-id><pub-id pub-id-type=\"pmcid\">4321304</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1421632111</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>GR</given-names></name><name><surname>Ki</surname><given-names>H</given-names></name><name><surname>Kim</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Automated, universal, and mass-producible paper-based lateral flow biosensing platform for high-performance point-of-care testing</article-title><source>ACS Appl Mater Interfaces</source><year>2019</year><volume>12</volume><issue>1</issue><fpage>1885</fpage><lpage>1894</lpage><pub-id pub-id-type=\"pmid\">31813220</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.9b17888</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hardick</surname><given-names>J</given-names></name><name><surname>Metzgar</surname><given-names>D</given-names></name><name><surname>Risen</surname><given-names>L</given-names></name><name><surname>Myers</surname><given-names>C</given-names></name><name><surname>Balansay</surname><given-names>M</given-names></name><name><surname>Malcom</surname><given-names>T</given-names></name><name><surname>Rothman</surname><given-names>R</given-names></name><name><surname>Gaydos</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Initial performance evaluation of a spotted array Mobile Analysis Platform (MAP) for the detection of influenza A/B, RSV, and MERS coronavirus</article-title><source>Diagn Microbiol Infect Dis</source><year>2018</year><volume>91</volume><issue>3</issue><fpage>245</fpage><lpage>247</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXlt1SisLg%3D</pub-id><pub-id pub-id-type=\"pmid\">29550057</pub-id><pub-id pub-id-type=\"pmcid\">6013072</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.diagmicrobio.2018.02.011</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Harvey</surname><given-names>CJ</given-names></name><name><surname>O’Regan</surname><given-names>DP</given-names></name></person-group><article-title xml:lang=\"en\">Radiological investigations and applications</article-title><source>Med MRCP</source><year>2020</year><volume>17</volume><fpage>111</fpage><pub-id pub-id-type=\"doi\">10.1093/med/9780198779506.003.0005</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Haslberger</surname><given-names>A</given-names></name><name><surname>Jacob</surname><given-names>U</given-names></name><name><surname>Hippe</surname><given-names>B</given-names></name><name><surname>Karlic</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Mechanisms of selected functional foods against viral infections with a view on COVID-19: Mini review</article-title><source>Funct Foods Health Dis</source><year>2020</year><volume>10</volume><issue>5</issue><fpage>195</fpage><lpage>209</lpage><pub-id pub-id-type=\"doi\">10.31989/ffhd.v10i5.707</pub-id></mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heymann</surname><given-names>DL</given-names></name><name><surname>Shindo</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19: what is next for public health?</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>542</fpage><lpage>545</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30374-3</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hofmann</surname><given-names>MA</given-names></name><name><surname>Renzullo</surname><given-names>S</given-names></name><name><surname>Mader</surname><given-names>M</given-names></name><name><surname>Chaignat</surname><given-names>V</given-names></name><name><surname>Worwa</surname><given-names>G</given-names></name><name><surname>Thuer</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Genetic characterization of toggenburg orbivirus a new bluetongue virus, from goats. Switzerland</article-title><source>Emerg Infect Dis</source><year>2008</year><volume>14</volume><issue>12</issue><fpage>1855</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXjvVKltA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19046507</pub-id><pub-id pub-id-type=\"pmcid\">2634640</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid1412.080818</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Fan</surname><given-names>G</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Gu</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>497</fpage><lpage>506</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Kqu7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30183-5</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>Z</given-names></name><name><surname>Jiang</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Yi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>Z</given-names></name><name><surname>Fan</surname><given-names>S</given-names></name><name><surname>Liao</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Jiang</surname><given-names>G</given-names></name><name><surname>Tang</surname><given-names>D</given-names></name><name><surname>Ye</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Ma</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 inactivated vaccine (Vero cells) shows good safety in repeated administration toxicity test of Sprague Dawley rats</article-title><source>Food Chem Toxicol</source><year>2021</year><volume>152</volume><fpage>112239</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXpvVSmtLY%3D</pub-id><pub-id pub-id-type=\"pmid\">33901607</pub-id><pub-id pub-id-type=\"pmcid\">8064818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fct.2021.112239</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Iannazzo</surname><given-names>D</given-names></name><name><surname>Pistone</surname><given-names>A</given-names></name><name><surname>Ferro</surname><given-names>S</given-names></name><name><surname>De Luca</surname><given-names>L</given-names></name><name><surname>Monforte</surname><given-names>AM</given-names></name><name><surname>Romeo</surname><given-names>R</given-names></name><name><surname>Buemi</surname><given-names>MR</given-names></name><name><surname>Pannecouque</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Graphene quantum dots based systems as HIV inhibitors</article-title><source>Bioconjug Chem</source><year>2018</year><volume>29</volume><issue>9</issue><fpage>3084</fpage><lpage>3093</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsV2lsLjK</pub-id><pub-id pub-id-type=\"pmid\">30106563</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.bioconjchem.8b00448</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Kaneko</surname><given-names>K</given-names></name><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Mizushima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Development of biodegradable nanoparticles for liver-specific ribavirin delivery</article-title><source>J Pharm Sci</source><year>2014</year><volume>103</volume><issue>12</issue><fpage>4005</fpage><lpage>4011</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVaktrjF</pub-id><pub-id pub-id-type=\"pmid\">25335768</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.24219</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Islam</surname><given-names>A</given-names></name><name><surname>Ahsan</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Plausible approach for rapid detection of SARS-CoV-2 virus by magnetic nanoparticle based biosensors</article-title><source>Am J Nanosci</source><year>2020</year><volume>6</volume><fpage>6</fpage><pub-id pub-id-type=\"doi\">10.11648/j.ajn.20200602.11</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Itani</surname><given-names>R</given-names></name><name><surname>Tobaiqy</surname><given-names>M</given-names></name><name><surname>Al Faraj</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Optimizing use of theranostic nanoparticles as a life-saving strategy for treating COVID-19 patients</article-title><source>Theranostics</source><year>2020</year><volume>10</volume><issue>13</issue><fpage>5932</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFGgsrzI</pub-id><pub-id pub-id-type=\"pmid\">32483428</pub-id><pub-id pub-id-type=\"pmcid\">7254986</pub-id><pub-id pub-id-type=\"doi\">10.7150/thno.46691</pub-id></mixed-citation></ref><ref id=\"CR74\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jacob</surname><given-names>S</given-names></name><name><surname>Nair</surname><given-names>AB</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Emerging role of nanosuspensions in drug delivery systems</article-title><source>Biomater Res</source><year>2020</year><volume>24</volume><issue>1</issue><fpage>1</fpage><lpage>16</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlOnurs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s40824-020-0184-8</pub-id></mixed-citation></ref><ref id=\"CR75\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaimes</surname><given-names>JA</given-names></name><name><surname>Millet</surname><given-names>JK</given-names></name><name><surname>Stout</surname><given-names>AE</given-names></name><name><surname>André</surname><given-names>NM</given-names></name><name><surname>Whittaker</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">A tale of two viruses: the distinct spike glycoproteins of feline coronaviruses</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>1</issue><fpage>83</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsVCquro%3D</pub-id><pub-id pub-id-type=\"pmid\">31936749</pub-id><pub-id pub-id-type=\"pmcid\">7019228</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12010083</pub-id></mixed-citation></ref><ref id=\"CR76\"><mixed-citation publication-type=\"other\">Jan N, Madni A, Khan S, Shah H, Akram F, Khan A, Ertas D, Bostanudin MF, Contag CH, Ashammakhi N, Ertas YN (2022) Biomimetic cell membrane‐coated poly (lactic‐co‐glycolic acid) nanoparticles for biomedical applications. Bioeng Transl Med. p e10441</mixed-citation></ref><ref id=\"CR77\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Javan</surname><given-names>F</given-names></name><name><surname>Vatanara</surname><given-names>A</given-names></name><name><surname>Azadmanesh</surname><given-names>K</given-names></name><name><surname>Nabi-Meibodi</surname><given-names>M</given-names></name><name><surname>Shakouri</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Encapsulation of ritonavir in solid lipid nanoparticles: in-vitro anti-HIV-1 activity using lentiviral particles</article-title><source>J Pharm Pharmacol</source><year>2017</year><volume>69</volume><issue>8</issue><fpage>1002</fpage><lpage>1009</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXntVKhsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">28471000</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12737</pub-id></mixed-citation></ref><ref id=\"CR78\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jiang</surname><given-names>M</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Arasthfer</surname><given-names>A</given-names></name><name><surname>Fang</surname><given-names>W</given-names></name><name><surname>Ling</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>H</given-names></name><name><surname>Daneshnia</surname><given-names>F</given-names></name><name><surname>Yu</surname><given-names>J</given-names></name><name><surname>Liao</surname><given-names>W</given-names></name><name><surname>Pei</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Development and validation of a apid, single-step reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) system potentially to be used for reliable and high-throughput screening of COVID-19</article-title><source>Front Cell Infect Microbiol</source><year>2020</year><volume>10</volume><fpage>331</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaktbbE</pub-id><pub-id pub-id-type=\"pmid\">32626666</pub-id><pub-id pub-id-type=\"pmcid\">7313420</pub-id><pub-id pub-id-type=\"doi\">10.3389/fcimb.2020.00331</pub-id></mixed-citation></ref><ref id=\"CR79\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jones</surname><given-names>ST</given-names></name><name><surname>Cagno</surname><given-names>V</given-names></name><name><surname>Janeček</surname><given-names>M</given-names></name><name><surname>Ortiz</surname><given-names>D</given-names></name><name><surname>Gasilova</surname><given-names>N</given-names></name><name><surname>Piret</surname><given-names>J</given-names></name><name><surname>Gasbarri</surname><given-names>M</given-names></name><name><surname>Constant</surname><given-names>DA</given-names></name><name><surname>Han</surname><given-names>Y</given-names></name><name><surname>Vuković</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Modified cyclodextrins as broad-spectrum antivirals</article-title><source>Sci Adv</source><year>2020</year><volume>6</volume><issue>5</issue><fpage>eaax9318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGgsr%2FL</pub-id><pub-id pub-id-type=\"pmid\">32064341</pub-id><pub-id pub-id-type=\"pmcid\">6989148</pub-id><pub-id pub-id-type=\"doi\">10.1126/sciadv.aax9318</pub-id></mixed-citation></ref><ref id=\"CR80\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kalita</surname><given-names>MJ</given-names></name><name><surname>Dutta</surname><given-names>K</given-names></name><name><surname>Hazarika</surname><given-names>G</given-names></name><name><surname>Dutta</surname><given-names>R</given-names></name><name><surname>Kalita</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>PP</given-names></name><name><surname>Sarma</surname><given-names>MP</given-names></name><name><surname>Banu</surname><given-names>S</given-names></name><name><surname>Idris</surname><given-names>M</given-names></name><name><surname>Talukdar</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">In-house reverse transcriptase polymerase chain reaction for detection of SARS-CoV-2 with increased sensitivity</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>12</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-97502-1</pub-id></mixed-citation></ref><ref id=\"CR81\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kandeel</surname><given-names>M</given-names></name><name><surname>Al-Taher</surname><given-names>A</given-names></name><name><surname>Park</surname><given-names>BK</given-names></name><name><surname>Kwon</surname><given-names>HJ</given-names></name><name><surname>Al-Nazawi</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A pilot study of the antiviral activity of anionic and cationic polyamidoamine dendrimers against the Middle East respiratory syndrome coronavirus</article-title><source>J Med Virol</source><year>2020</year><volume>92</volume><issue>9</issue><fpage>1665</fpage><lpage>1670</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVyhsb8%3D</pub-id><pub-id pub-id-type=\"pmid\">32330296</pub-id><pub-id pub-id-type=\"pmcid\">7264540</pub-id><pub-id pub-id-type=\"doi\">10.1002/jmv.25928</pub-id></mixed-citation></ref><ref id=\"CR82\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Katata-Seru</surname><given-names>L</given-names></name><name><surname>Ojo</surname><given-names>BM</given-names></name><name><surname>Okubanjo</surname><given-names>O</given-names></name><name><surname>Soremekun</surname><given-names>R</given-names></name><name><surname>Aremu</surname><given-names>OS</given-names></name></person-group><article-title xml:lang=\"en\">Nanoformulated Eudragit lopinavir and preliminary release of its loaded suppositories</article-title><source>Heliyon</source><year>2020</year><volume>6</volume><issue>5</issue><pub-id pub-id-type=\"pmid\">32420478</pub-id><pub-id pub-id-type=\"pmcid\">7218025</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.heliyon.2020.e03890</pub-id></mixed-citation></ref><ref id=\"CR83\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaur</surname><given-names>I</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Jakhar</surname><given-names>D</given-names></name><name><surname>Das</surname><given-names>A</given-names></name><name><surname>Aradhya</surname><given-names>SS</given-names></name><name><surname>Sharma</surname><given-names>R</given-names></name><name><surname>Jindal</surname><given-names>V</given-names></name><name><surname>Mhatre</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease (COVID-19): An updated review based on current knowledge and existing literature for dermatologists</article-title><source>Dermatol Ther</source><year>2020</year><volume>33</volume><issue>4</issue><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXps12jsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">32447820</pub-id><pub-id pub-id-type=\"pmcid\">7283871</pub-id><pub-id pub-id-type=\"doi\">10.1111/dth.13677</pub-id></mixed-citation></ref><ref id=\"CR84\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>AK</given-names></name><name><surname>Dhau</surname><given-names>JS</given-names></name><name><surname>Gohel</surname><given-names>H</given-names></name><name><surname>Mishra</surname><given-names>YK</given-names></name><name><surname>Kateb</surname><given-names>B</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Goswami</surname><given-names>DY</given-names></name></person-group><article-title xml:lang=\"en\">Electrochemical SARS-CoV-2 sensing at point-of-care and artificial intelligence for intelligent COVID-19 management</article-title><source>ACS Appl Bio Mater</source><year>2020</year><volume>3</volume><issue>11</issue><fpage>7306</fpage><lpage>7325</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGltr3I</pub-id><pub-id pub-id-type=\"pmid\">35019473</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsabm.0c01004</pub-id></mixed-citation></ref><ref id=\"CR85\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Jain</surname><given-names>K</given-names></name><name><surname>Jain</surname><given-names>NK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer as nanocarrier for drug delivery</article-title><source>Prog Polym Sci</source><year>2014</year><volume>39</volume><issue>2</issue><fpage>268</fpage><lpage>307</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1Wrur7O</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.progpolymsci.2013.07.005</pub-id></mixed-citation></ref><ref id=\"CR86\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Xie</surname><given-names>L</given-names></name><name><surname>Banerjee</surname><given-names>S</given-names></name><name><surname>Mao</surname><given-names>G</given-names></name><name><surname>Padhye</surname><given-names>S</given-names></name><name><surname>Sarkar</surname><given-names>FH</given-names></name><name><surname>Iyer</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Hyaluronic acid-conjugated polyamidoamine dendrimers for targeted delivery of 3, 4-difluorobenzylidene curcumin to CD44 overexpressing pancreatic cancer cells</article-title><source>Colloids Surf, B</source><year>2015</year><volume>136</volume><fpage>413</fpage><lpage>423</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhs1aktr3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2015.09.043</pub-id></mixed-citation></ref><ref id=\"CR87\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalaj-Hedayati</surname><given-names>A</given-names></name><name><surname>Chua</surname><given-names>CLL</given-names></name><name><surname>Smooker</surname><given-names>P</given-names></name><name><surname>Lee</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Nanoparticles in influenza subunit vaccine development: immunogenicity enhancement</article-title><source>Influenza Other Respir Viruses</source><year>2020</year><volume>14</volume><issue>1</issue><fpage>92</fpage><lpage>101</lpage><pub-id pub-id-type=\"pmid\">31774251</pub-id><pub-id pub-id-type=\"doi\">10.1111/irv.12697</pub-id></mixed-citation></ref><ref id=\"CR88\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khan</surname><given-names>AA</given-names></name><name><surname>Mudassir</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>S</given-names></name><name><surname>Murugaiyah</surname><given-names>V</given-names></name><name><surname>Darwis</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Freeze-dried lopinavir-loaded nanostructured lipid carriers for enhanced cellular uptake and bioavailability: statistical optimization, in vitro and in vivo evaluations</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>2</issue><fpage>97</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlWhtLrP</pub-id><pub-id pub-id-type=\"pmid\">30823545</pub-id><pub-id pub-id-type=\"pmcid\">6410192</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11020097</pub-id></mixed-citation></ref><ref id=\"CR89\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kobayashi</surname><given-names>K</given-names></name><name><surname>Wei</surname><given-names>J</given-names></name><name><surname>Iida</surname><given-names>R</given-names></name><name><surname>Ijiro</surname><given-names>K</given-names></name><name><surname>Niikura</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Surface engineering of nanoparticles for therapeutic applications</article-title><source>Polym J</source><year>2014</year><volume>46</volume><issue>8</issue><fpage>460</fpage><lpage>468</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1KksLbK</pub-id><pub-id pub-id-type=\"doi\">10.1038/pj.2014.40</pub-id></mixed-citation></ref><ref id=\"CR90\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Koshy</surname><given-names>ST</given-names></name><name><surname>Cheung</surname><given-names>AS</given-names></name><name><surname>Gu</surname><given-names>L</given-names></name><name><surname>Graveline</surname><given-names>AR</given-names></name><name><surname>Mooney</surname><given-names>DJ</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal delivery enhances immune activation by STING agonists for cancer immunotherapy</article-title><source>Adv Biosyst</source><year>2017</year><volume>1</volume><issue>1–2</issue><fpage>1600013</fpage><pub-id pub-id-type=\"pmid\">30258983</pub-id><pub-id pub-id-type=\"pmcid\">6152940</pub-id><pub-id pub-id-type=\"doi\">10.1002/adbi.201600013</pub-id></mixed-citation></ref><ref id=\"CR91\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kulthe</surname><given-names>SS</given-names></name><name><surname>Choudhari</surname><given-names>YM</given-names></name><name><surname>Inamdar</surname><given-names>NN</given-names></name><name><surname>Mourya</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles: authoritative aspects for drug delivery</article-title><source>Des Monomers Polym</source><year>2012</year><volume>15</volume><issue>5</issue><fpage>465</fpage><lpage>521</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsVSqsL%2FP</pub-id><pub-id pub-id-type=\"doi\">10.1080/1385772X.2012.688328</pub-id></mixed-citation></ref><ref id=\"CR92\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>M</given-names></name><name><surname>Curtis</surname><given-names>A</given-names></name><name><surname>Hoskins</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanoparticle technologies in the combat against anti-microbial resistance</article-title><source>Pharmaceutics</source><year>2018</year><volume>10</volume><issue>1</issue><fpage>11</fpage><pub-id pub-id-type=\"pmid\">29342903</pub-id><pub-id pub-id-type=\"pmcid\">5874824</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics10010011</pub-id></mixed-citation></ref><ref id=\"CR93\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Naha</surname><given-names>A</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development of ritonavir solid lipid nanoparticles by Box Behnken design for intestinal lymphatic targeting</article-title><source>J Drug Deliv Sci Technol</source><year>2018</year><volume>44</volume><fpage>181</fpage><lpage>189</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXis1Clsg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2017.12.014</pub-id></mixed-citation></ref><ref id=\"CR94\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight to ROS and apoptosis regulated cytotoxicity inferred by green synthesized CuO nanoparticles from Calotropis gigantea to embryonic zebrafish</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>17</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-017-16581-1</pub-id></mixed-citation></ref><ref id=\"CR95\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Pramanik</surname><given-names>N</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Soni</surname><given-names>N</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Molecular insight to in vitro biocompatibility of phytofabricated copper oxide nanoparticles with human embryonic kidney cells</article-title><source>Nanomedicine</source><year>2018</year><volume>13</volume><issue>19</issue><fpage>2415</fpage><lpage>2433</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitV2isL%2FN</pub-id><pub-id pub-id-type=\"pmid\">30251920</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2018-0175</pub-id></mixed-citation></ref><ref id=\"CR96\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupferschmidt</surname><given-names>K</given-names></name><name><surname>Cohen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Race to find COVID-19 treatments accelerates</article-title><source>Am Assoc Adv Sci</source><year>2020</year><volume>367</volume><fpage>6485</fpage></mixed-citation></ref><ref id=\"CR97\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>S</given-names></name><name><surname>Ruktanonchai</surname><given-names>NW</given-names></name><name><surname>Zhou</surname><given-names>L</given-names></name><name><surname>Prosper</surname><given-names>O</given-names></name><name><surname>Luo</surname><given-names>W</given-names></name><name><surname>Floyd</surname><given-names>JR</given-names></name><name><surname>Wesolowski</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Du</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak: an observational and modelling study</article-title><source>MedRxiv</source><year>2020</year><volume>202003</volume><issue>03</issue><fpage>20029843</fpage></mixed-citation></ref><ref id=\"CR98\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamb</surname><given-names>YN</given-names></name></person-group><article-title xml:lang=\"en\">BNT162b2 mRNA COVID-19 vaccine: first approval</article-title><source>Drugs</source><year>2021</year><volume>81</volume><fpage>495</fpage><lpage>501</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1yquro%3D</pub-id><pub-id pub-id-type=\"pmid\">33683637</pub-id><pub-id pub-id-type=\"pmcid\">7938284</pub-id><pub-id pub-id-type=\"doi\">10.1007/s40265-021-01480-7</pub-id></mixed-citation></ref><ref id=\"CR99\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamote</surname><given-names>K</given-names></name><name><surname>Janssens</surname><given-names>E</given-names></name><name><surname>Schillebeeckx</surname><given-names>E</given-names></name><name><surname>Lapperre</surname><given-names>TS</given-names></name><name><surname>De Winter</surname><given-names>BY</given-names></name><name><surname>Van Meerbeeck</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">The scent of COVID-19: viral (semi-) volatiles as fast diagnostic biomarkers?</article-title><source>J Breath Res</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>042001</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1egsrvJ</pub-id><pub-id pub-id-type=\"pmid\">32599571</pub-id><pub-id pub-id-type=\"doi\">10.1088/1752-7163/aba105</pub-id></mixed-citation></ref><ref id=\"CR100\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Law</surname><given-names>JWF</given-names></name><name><surname>Ab Mutalib</surname><given-names>NS</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Lee</surname><given-names>LH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid methods for the detection of foodborne bacterial pathogens: principles, applications, advantages and limitations</article-title><source>Front Microbiol</source><year>2015</year><volume>5</volume><fpage>770</fpage><pub-id pub-id-type=\"pmid\">25628612</pub-id><pub-id pub-id-type=\"pmcid\">4290631</pub-id><pub-id pub-id-type=\"doi\">10.3389/fmicb.2014.00770</pub-id></mixed-citation></ref><ref id=\"CR101\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Le</surname><given-names>TT</given-names></name><name><surname>Andreadakis</surname><given-names>Z</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Román</surname><given-names>RG</given-names></name><name><surname>Tollefsen</surname><given-names>S</given-names></name><name><surname>Saville</surname><given-names>M</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><issue>5</issue><fpage>305</fpage><lpage>306</lpage><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00073-5</pub-id></mixed-citation></ref><ref id=\"CR102\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ledet</surname><given-names>G</given-names></name><name><surname>Mandal</surname><given-names>TK</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine: emerging therapeutics for the 21st century</article-title><source>US Pharm</source><year>2012</year><volume>37</volume><issue>3</issue><fpage>7</fpage><lpage>11</lpage></mixed-citation></ref><ref id=\"CR103\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lembo</surname><given-names>D</given-names></name><name><surname>Donalisio</surname><given-names>M</given-names></name><name><surname>Civra</surname><given-names>A</given-names></name><name><surname>Argenziano</surname><given-names>M</given-names></name><name><surname>Cavalli</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine formulations for the delivery of antiviral drugs: a promising solution for the treatment of viral infections</article-title><source>Expert Opin Drug Deliv</source><year>2018</year><volume>15</volume><issue>1</issue><fpage>93</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXht12ltr%2FO</pub-id><pub-id pub-id-type=\"pmid\">28749739</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2017.1360863</pub-id></mixed-citation></ref><ref id=\"CR104\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>SM</given-names></name><name><surname>Yu</surname><given-names>XH</given-names></name><name><surname>Tang</surname><given-names>SL</given-names></name><name><surname>Tang</surname><given-names>CK</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 (COVID-19): current status and future perspectives</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><issue>5</issue><fpage>105951</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSksLs%3D</pub-id><pub-id pub-id-type=\"pmid\">32234466</pub-id><pub-id pub-id-type=\"pmcid\">7139247</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105951</pub-id></mixed-citation></ref><ref id=\"CR105\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Geng</surname><given-names>M</given-names></name><name><surname>Peng</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Molecular immune pathogenesis and diagnosis of COVID-19</article-title><source>J Pharm Anal</source><year>2020</year><volume>10</volume><issue>2</issue><fpage>102</fpage><lpage>108</lpage><pub-id pub-id-type=\"pmid\">32282863</pub-id><pub-id pub-id-type=\"pmcid\">7104082</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jpha.2020.03.001</pub-id></mixed-citation></ref><ref id=\"CR106\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wu</surname><given-names>D</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Chu</surname><given-names>X</given-names></name><name><surname>Qin</surname><given-names>C</given-names></name><name><surname>Han</surname><given-names>X</given-names></name><name><surname>Hang</surname><given-names>T</given-names></name><name><surname>Xu</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name><name><surname>Yin</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal remdesivir inhalation solution for targeted lung delivery as a novel therapeutic approach for COVID-19</article-title><source>Asian J Pharm Sci</source><year>2021</year><volume>16</volume><issue>6</issue><fpage>772</fpage><lpage>783</lpage><pub-id pub-id-type=\"pmid\">34703490</pub-id><pub-id pub-id-type=\"pmcid\">8529908</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ajps.2021.09.002</pub-id></mixed-citation></ref><ref id=\"CR107\"><mixed-citation publication-type=\"other\">Lim ES, Wang D. Pathogen discovery (2016) Molecular microbiology: diagnostic principles and practice. pp 80–91</mixed-citation></ref><ref id=\"CR108\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lau</surname><given-names>SK</given-names></name><name><surname>Varma</surname><given-names>VA</given-names></name><name><surname>Moffitt</surname><given-names>RA</given-names></name><name><surname>Caldwell</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>T</given-names></name><name><surname>Young</surname><given-names>AN</given-names></name><name><surname>Petros</surname><given-names>JA</given-names></name><name><surname>Osunkoya</surname><given-names>AO</given-names></name><name><surname>Krogstad</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Molecular mapping of tumor heterogeneity on clinical tissue specimens with multiplexed quantum dots</article-title><source>ACS Nano</source><year>2010</year><volume>4</volume><issue>5</issue><fpage>2755</fpage><lpage>2765</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXksVKltLc%3D</pub-id><pub-id pub-id-type=\"pmid\">20377268</pub-id><pub-id pub-id-type=\"pmcid\">2923482</pub-id><pub-id pub-id-type=\"doi\">10.1021/nn100213v</pub-id></mixed-citation></ref><ref id=\"CR109\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>YC</given-names></name><name><surname>Liao</surname><given-names>CH</given-names></name><name><surname>Chang</surname><given-names>CF</given-names></name><name><surname>Chou</surname><given-names>CC</given-names></name><name><surname>Lin</surname><given-names>YR</given-names></name></person-group><article-title xml:lang=\"en\">A locally transmitted case of SARS-CoV-2 infection in Taiwan</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>11</issue><fpage>1070</fpage><lpage>1072</lpage><pub-id pub-id-type=\"pmid\">32050059</pub-id><pub-id pub-id-type=\"pmcid\">7121202</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMc2001573</pub-id></mixed-citation></ref><ref id=\"CR110\"><mixed-citation publication-type=\"other\">Lohcharoenkal W, Wang L, Chen YC, Rojanasakul Y (2014) Protein nanoparticles as drug delivery carriers for cancer therapy. BioMed Res Int </mixed-citation></ref><ref id=\"CR111\"><mixed-citation publication-type=\"other\">Lokhande SS (2018) Liposome drug delivery: an update review. Pharm Sci Monit 9(1)</mixed-citation></ref><ref id=\"CR112\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Caracterización genómica y epidemiología del nuevo coronavirus 2019: implicaciones para los orígenes del virus y la unión al receptor</article-title><source>Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>566</fpage><lpage>568</lpage></mixed-citation></ref><ref id=\"CR113\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Niu</surname><given-names>P</given-names></name><name><surname>Yang</surname><given-names>B</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>565</fpage><lpage>574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOmsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30251-8</pub-id></mixed-citation></ref><ref id=\"CR114\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lustig</surname><given-names>Y</given-names></name><name><surname>Keler</surname><given-names>S</given-names></name><name><surname>Kolodny</surname><given-names>R</given-names></name><name><surname>Ben-Tal</surname><given-names>N</given-names></name><name><surname>Atias-Varon</surname><given-names>D</given-names></name><name><surname>Shlush</surname><given-names>E</given-names></name><name><surname>Gerlic</surname><given-names>M</given-names></name><name><surname>Munitz</surname><given-names>A</given-names></name><name><surname>Doolman</surname><given-names>R</given-names></name><name><surname>Asraf</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Potential antigenic cross-reactivity between severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and dengue viruses</article-title><source>Clin Infect Dis</source><year>2021</year><volume>73</volume><issue>7</issue><fpage>e2444</fpage><lpage>e2449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1Wrt7%2FE</pub-id><pub-id pub-id-type=\"pmid\">32797228</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1207</pub-id></mixed-citation></ref><ref id=\"CR115\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>Q</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Xie</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Jiang</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Liu Shen capsule shows antiviral and anti-inflammatory abilities against novel coronavirus SARS-CoV-2 via suppression of NF-κB signaling pathway</article-title><source>Pharmacol Res</source><year>2020</year><volume>158</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVeltLrK</pub-id><pub-id pub-id-type=\"pmid\">32360580</pub-id><pub-id pub-id-type=\"pmcid\">7192119</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.phrs.2020.104850</pub-id></mixed-citation></ref><ref id=\"CR116\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Guo</surname><given-names>J</given-names></name><name><surname>Morawska</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 patients in earlier stages exhaled millions of severe acute respiratory syndrome coronavirus 2 per hour</article-title><source>Clin Infect Dis</source><year>2021</year><volume>72</volume><issue>10</issue><fpage>e652</fpage><lpage>e654</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFyitrvM</pub-id><pub-id pub-id-type=\"pmid\">32857833</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1283</pub-id></mixed-citation></ref><ref id=\"CR117\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maggi</surname><given-names>F</given-names></name><name><surname>Pistello</surname><given-names>M</given-names></name><name><surname>Antonelli</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Future management of viral diseases: role of new technologies and new approaches in microbial interactions</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>2</issue><fpage>136</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BB3crntlyrug%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">30502490</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.11.015</pub-id></mixed-citation></ref><ref id=\"CR118\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahajan</surname><given-names>HS</given-names></name><name><surname>Patil</surname><given-names>PH</given-names></name></person-group><article-title xml:lang=\"en\">Central composite design-based optimization of lopinavir vitamin E-TPGS micelle: In vitro characterization and in vivo pharmacokinetic study</article-title><source>Colloids Surf, B</source><year>2020</year><volume>194</volume><fpage>111149</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXht1Kis7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2020.111149</pub-id></mixed-citation></ref><ref id=\"CR119\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maji</surname><given-names>S</given-names></name><name><surname>Biswas</surname><given-names>A</given-names></name><name><surname>Mandal</surname><given-names>S</given-names></name><name><surname>Sen</surname><given-names>DJ</given-names></name><name><surname>Mahanti</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Silver and copper nano particles in mask for Corona Virus-19 protection</article-title><source>Glob J Endrocrinol Metab</source><year>2020</year><volume>3</volume><fpage>000556</fpage></mixed-citation></ref><ref id=\"CR02\"><mixed-citation publication-type=\"other\">Maniyar MG, Kokare CR (2019) Formulation and evaluation of spray-dried liposomes of lopinavir for topical application. J Pharm Investig 49:259–270.</mixed-citation></ref><ref id=\"CR120\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Masters</surname><given-names>PS</given-names></name></person-group><article-title xml:lang=\"en\">The molecular biology of coronaviruses</article-title><source>Adv Virus Res</source><year>2006</year><volume>66</volume><fpage>193</fpage><lpage>292</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhslOhtLw%3D</pub-id><pub-id pub-id-type=\"pmid\">16877062</pub-id><pub-id pub-id-type=\"pmcid\">7112330</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0065-3527(06)66005-3</pub-id></mixed-citation></ref><ref id=\"CR121\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Matsuura</surname><given-names>K</given-names></name><name><surname>Watanabe</surname><given-names>K</given-names></name><name><surname>Matsuzaki</surname><given-names>T</given-names></name><name><surname>Sakurai</surname><given-names>K</given-names></name><name><surname>Kimizuka</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Self-assembled synthetic viral capsids from a 24-mer viral peptide fragment</article-title><source>Angew Chem Int Ed</source><year>2010</year><volume>49</volume><issue>50</issue><fpage>9662</fpage><lpage>9665</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXhsFaqs7jF</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201004606</pub-id></mixed-citation></ref><ref id=\"CR122\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Miripour</surname><given-names>ZS</given-names></name><name><surname>Sarrami-Forooshani</surname><given-names>R</given-names></name><name><surname>Sanati</surname><given-names>H</given-names></name><name><surname>Makarem</surname><given-names>J</given-names></name><name><surname>Taheri</surname><given-names>MS</given-names></name><name><surname>Shojaeian</surname><given-names>F</given-names></name><name><surname>Eskafi</surname><given-names>AH</given-names></name><name><surname>Abbasvandi</surname><given-names>F</given-names></name><name><surname>Namdar</surname><given-names>N</given-names></name><name><surname>Ghafari</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Real-time diagnosis of reactive oxygen species (ROS) in fresh sputum by electrochemical tracing; correlation between COVID-19 and viral-induced ROS in lung/respiratory epithelium during this pandemic</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><fpage>112435</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlyjtL%2FI</pub-id><pub-id pub-id-type=\"pmid\">32729548</pub-id><pub-id pub-id-type=\"pmcid\">7341050</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112435</pub-id></mixed-citation></ref><ref id=\"CR123\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Misra</surname><given-names>N</given-names></name><name><surname>Bhatt</surname><given-names>S</given-names></name><name><surname>Arefi-Khonsari</surname><given-names>F</given-names></name><name><surname>Kumar</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">State of the art in nonthermal plasma processing for biomedical applications: Can it help fight viral pandemics like COVID-19?</article-title><source>Plasma Process Polym</source><year>2021</year><volume>18</volume><issue>7</issue><fpage>2000215</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFKktbnF</pub-id><pub-id pub-id-type=\"pmid\">34220401</pub-id><pub-id pub-id-type=\"pmcid\">8237024</pub-id><pub-id pub-id-type=\"doi\">10.1002/ppap.202000215</pub-id></mixed-citation></ref><ref id=\"CR124\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Moabelo</surname><given-names>KL</given-names></name><name><surname>Martin</surname><given-names>DR</given-names></name><name><surname>Fadaka</surname><given-names>AO</given-names></name><name><surname>Sibuyi</surname><given-names>NR</given-names></name><name><surname>Meyer</surname><given-names>M</given-names></name><name><surname>Madiehe</surname><given-names>AM</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based strategies for effective and rapid detection of SARS-CoV-2</article-title><source>Materials</source><year>2021</year><volume>14</volume><issue>24</issue><fpage>7851</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXivVaqs7jJ</pub-id><pub-id pub-id-type=\"pmid\">34947447</pub-id><pub-id pub-id-type=\"pmcid\">8703409</pub-id><pub-id pub-id-type=\"doi\">10.3390/ma14247851</pub-id></mixed-citation></ref><ref id=\"CR125\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mohammadniaei</surname><given-names>M</given-names></name><name><surname>Zhang</surname><given-names>M</given-names></name><name><surname>Ashley</surname><given-names>J</given-names></name><name><surname>Christensen</surname><given-names>UB</given-names></name><name><surname>Friis-Hansen</surname><given-names>LJ</given-names></name><name><surname>Gregersen</surname><given-names>R</given-names></name><name><surname>Lisby</surname><given-names>JG</given-names></name><name><surname>Benfield</surname><given-names>TL</given-names></name><name><surname>Nielsen</surname><given-names>FE</given-names></name><name><surname>Henning Rasmussen</surname><given-names>J</given-names></name><name><surname>Pedersen</surname><given-names>EB</given-names></name></person-group><article-title xml:lang=\"en\">A non-enzymatic, isothermal strand displacement and amplification assay for rapid detection of SARS-CoV-2 RNA</article-title><source>Nat Commun</source><year>2021</year><volume>12</volume><issue>1</issue><fpage>1</fpage><lpage>2</lpage><pub-id pub-id-type=\"doi\">10.1038/s41467-021-25387-9</pub-id></mixed-citation></ref><ref id=\"CR126\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Monto</surname><given-names>AS</given-names></name><name><surname>Fendrick</surname><given-names>AM</given-names></name><name><surname>Sarnes</surname><given-names>MW</given-names></name></person-group><article-title xml:lang=\"en\">Respiratory illness caused by picornavirus infection: a review of clinical outcomes</article-title><source>Clin Ther</source><year>2001</year><volume>23</volume><issue>10</issue><fpage>1615</fpage><lpage>1627</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BD3MnnvFCjsQ%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">11726001</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0149-2918(01)80133-8</pub-id></mixed-citation></ref><ref id=\"CR127\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morokutti-Kurz</surname><given-names>M</given-names></name><name><surname>Graf</surname><given-names>C</given-names></name><name><surname>Prieschl-Grassauer</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Amylmetacresol/2, 4-dichlorobenzyl alcohol, hexylresorcinol, or carrageenan lozenges as active treatments for sore throat</article-title><source>Int J General Med</source><year>2017</year><volume>10</volume><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFamtL7E</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJGM.S120665</pub-id></mixed-citation></ref><ref id=\"CR128\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Munster</surname><given-names>VJ</given-names></name><name><surname>Koopmans</surname><given-names>M</given-names></name><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>van Riel</surname><given-names>D</given-names></name><name><surname>de Wit</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus emerging in China—key questions for impact assessment</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>8</issue><fpage>692</fpage><lpage>694</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjslGltrk%3D</pub-id><pub-id pub-id-type=\"pmid\">31978293</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMp2000929</pub-id></mixed-citation></ref><ref id=\"CR129\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Murtaza</surname><given-names>A</given-names></name><name><surname>Rehman</surname><given-names>T</given-names></name><name><surname>Zafar</surname><given-names>Z</given-names></name><name><surname>Noreen</surname><given-names>S</given-names></name><name><surname>Iqbal</surname><given-names>M</given-names></name><name><surname>Hassan</surname><given-names>S</given-names></name><name><surname>Khawaish</surname><given-names>I</given-names></name><name><surname>Tarar</surname><given-names>MHA</given-names></name></person-group><article-title xml:lang=\"en\">Synthesis and characterization of herbal nano-suspensions and evaluation of their In-vivo antihypertensive potential with especial focus on piperine</article-title><source>ACE</source><year>2020</year><volume>1</volume><fpage>23</fpage></mixed-citation></ref><ref id=\"CR130\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muthuirulappan</surname><given-names>S</given-names></name><name><surname>Francis</surname><given-names>SP</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer mechanism and possibility of nano-suspension formulation for a marine algae product fucoxanthin</article-title><source>Asian Pac J Cancer Prev</source><year>2013</year><volume>14</volume><issue>4</issue><fpage>2213</fpage><lpage>2216</lpage><pub-id pub-id-type=\"pmid\">23725114</pub-id><pub-id pub-id-type=\"doi\">10.7314/APJCP.2013.14.4.2213</pub-id></mixed-citation></ref><ref id=\"CR131\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Naseema</surname><given-names>A</given-names></name><name><surname>Kovooru</surname><given-names>L</given-names></name><name><surname>Behera</surname><given-names>AK</given-names></name><name><surname>Kumar</surname><given-names>KP</given-names></name><name><surname>Srivastava</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">A critical review of synthesis procedures, applications and future potential of nanoemulsions</article-title><source>Adv Coll Interface Sci</source><year>2021</year><volume>287</volume><fpage>102318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisVSmtrvN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2020.102318</pub-id></mixed-citation></ref><ref id=\"CR132\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndwandwe</surname><given-names>D</given-names></name><name><surname>Wiysonge</surname><given-names>CS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines</article-title><source>Curr Opin Immunol</source><year>2021</year><volume>71</volume><fpage>111</fpage><lpage>116</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsF2gtrrK</pub-id><pub-id pub-id-type=\"pmid\">34330017</pub-id><pub-id pub-id-type=\"pmcid\">8272971</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.coi.2021.07.003</pub-id></mixed-citation></ref><ref id=\"CR133\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Negi</surname><given-names>JS</given-names></name><name><surname>Chattopadhyay</surname><given-names>P</given-names></name><name><surname>Sharma</surname><given-names>AK</given-names></name><name><surname>Ram</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Development and evaluation of glyceryl behenate based solid lipid nanoparticles (SLNs) using hot self-nanoemulsification (SNE) technique</article-title><source>Arch Pharmacal Res</source><year>2014</year><volume>37</volume><issue>3</issue><fpage>361</fpage><lpage>370</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXot1Kns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12272-013-0154-y</pub-id></mixed-citation></ref><ref id=\"CR134\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nguyen-Contant</surname><given-names>P</given-names></name><name><surname>Embong</surname><given-names>AK</given-names></name><name><surname>Kanagaiah</surname><given-names>P</given-names></name><name><surname>Chaves</surname><given-names>FA</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Branche</surname><given-names>AR</given-names></name><name><surname>Topham</surname><given-names>DJ</given-names></name><name><surname>Sangster</surname><given-names>MY</given-names></name></person-group><article-title xml:lang=\"en\">S protein-reactive IgG and memory B cell production after human SARS-CoV-2 infection includes broad reactivity to the S2 subunit</article-title><source>Mbio</source><year>2020</year><volume>11</volume><issue>5</issue><fpage>e01991</fpage><lpage>e2020</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitl2kt7g%3D</pub-id><pub-id pub-id-type=\"pmid\">32978311</pub-id><pub-id pub-id-type=\"pmcid\">7520599</pub-id><pub-id pub-id-type=\"doi\">10.1128/mBio.01991-20</pub-id></mixed-citation></ref><ref id=\"CR135\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nikaeen</surname><given-names>G</given-names></name><name><surname>Abbaszadeh</surname><given-names>S</given-names></name><name><surname>Yousefinejad</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanomaterials in treatment, anti-infection and detection of coronaviruses</article-title><source>Nanomedicine</source><year>2020</year><volume>15</volume><issue>15</issue><fpage>1501</fpage><lpage>1512</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlWqtrvF</pub-id><pub-id pub-id-type=\"pmid\">32378459</pub-id><pub-id pub-id-type=\"pmcid\">7373208</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2020-0117</pub-id></mixed-citation></ref><ref id=\"CR136\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noriega-Luna</surname><given-names>B</given-names></name><name><surname>Godínez</surname><given-names>LA</given-names></name><name><surname>Rodríguez</surname><given-names>FJ</given-names></name><name><surname>Rodríguez</surname><given-names>A</given-names></name><name><surname>Larrea</surname><given-names>GZLD</given-names></name><name><surname>Sosa-Ferreyra</surname><given-names>CF</given-names></name><name><surname>Mercado-Curiel</surname><given-names>RF</given-names></name><name><surname>Manríquez</surname><given-names>J</given-names></name><name><surname>Bustos</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Applications of dendrimers in drug delivery agents, diagnosis, therapy, and detection</article-title><source>J Nanomater</source><year>2014</year><volume>2014</volume><fpage>39</fpage><lpage>39</lpage><pub-id pub-id-type=\"doi\">10.1155/2014/507273</pub-id></mixed-citation></ref><ref id=\"CR137\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noyes</surname><given-names>AA</given-names></name><name><surname>Whitney</surname><given-names>WR</given-names></name></person-group><article-title xml:lang=\"en\">The rate of solution of solid substances in their own solutions</article-title><source>J Am Chem Soc</source><year>1897</year><volume>19</volume><issue>12</issue><fpage>930</fpage><lpage>934</lpage><pub-id pub-id-type=\"doi\">10.1021/ja02086a003</pub-id></mixed-citation></ref><ref id=\"CR138\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olvera</surname><given-names>D</given-names></name><name><surname>Monaghan</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Electroactive material-based biosensors for detection and drug delivery</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>170</volume><fpage>396</fpage><lpage>424</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvF2jsLnE</pub-id><pub-id pub-id-type=\"pmid\">32987096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2020.09.011</pub-id></mixed-citation></ref><ref id=\"CR139\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>G</given-names></name><name><surname>Fan</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>Y</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Long</surname><given-names>X</given-names></name><name><surname>Guo</surname><given-names>S</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Hu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Serological immunochromatographic approach in diagnosis with SARS-CoV-2 infected COVID-19 patients</article-title><source>J Infect</source><year>2020</year><volume>81</volume><issue>1</issue><fpage>e28</fpage><lpage>32</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCjtb7P</pub-id><pub-id pub-id-type=\"pmid\">32283141</pub-id><pub-id pub-id-type=\"pmcid\">7195339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jinf.2020.03.051</pub-id></mixed-citation></ref><ref id=\"CR140\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Park</surname><given-names>GS</given-names></name><name><surname>Ku</surname><given-names>K</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SJ</given-names></name><name><surname>Kim</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>BT</given-names></name><name><surname>Maeng</surname><given-names>JS</given-names></name></person-group><article-title xml:lang=\"en\">Development of reverse transcription loop-mediated isothermal amplification assays targeting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)</article-title><source>J Mol Diagn</source><year>2020</year><volume>22</volume><issue>6</issue><fpage>729</fpage><lpage>735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXns1Wrsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">32276051</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jmoldx.2020.03.006</pub-id></mixed-citation></ref><ref id=\"CR03\"><mixed-citation publication-type=\"other\">Patel G, Shelat P, Lalwani A (2016) Statistical modeling, optimization and characterization of solid self-nanoemulsifying drug delivery system of lopinavir using design of experiment. Drug delivery 23:3027–3042</mixed-citation></ref><ref id=\"CR141\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>GM</given-names></name><name><surname>Shelat</surname><given-names>PK</given-names></name><name><surname>Lalwani</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">QbD based development of proliposome of lopinavir for improved oral bioavailability</article-title><source>Eur J Pharm Sci</source><year>2017</year><volume>108</volume><fpage>50</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsVOqu7rO</pub-id><pub-id pub-id-type=\"pmid\">27586019</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2016.08.057</pub-id></mixed-citation></ref><ref id=\"CR142\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patone</surname><given-names>M</given-names></name><name><surname>Mei</surname><given-names>XW</given-names></name><name><surname>Handunnetthi</surname><given-names>L</given-names></name><name><surname>Dixon</surname><given-names>S</given-names></name><name><surname>Zaccardi</surname><given-names>F</given-names></name><name><surname>Shankar-Hari</surname><given-names>M</given-names></name><name><surname>Watkinson</surname><given-names>P</given-names></name><name><surname>Khunti</surname><given-names>K</given-names></name><name><surname>Harnden</surname><given-names>A</given-names></name><name><surname>Coupland</surname><given-names>CAC</given-names></name><name><surname>Channon</surname><given-names>KM</given-names></name><name><surname>Mills</surname><given-names>NL</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name><name><surname>Hippisley-Cox</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Risks of myocarditis, pericarditis, and cardiac arrhythmias associated with COVID-19 vaccination or SARS-CoV-2 infection</article-title><source>Nat Med</source><year>2022</year><volume>28</volume><issue>2</issue><fpage>410</fpage><lpage>422</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislCjsbnJ</pub-id><pub-id pub-id-type=\"pmid\">34907393</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41591-021-01630-0</pub-id></mixed-citation></ref><ref id=\"CR143\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Fraceto</surname><given-names>LF</given-names></name><name><surname>Campos</surname><given-names>EVR</given-names></name><name><surname>Rodriguez-Torres</surname><given-names>MdP</given-names></name><name><surname>Acosta-Torres</surname><given-names>LS</given-names></name><name><surname>Diaz-Torres</surname><given-names>LA</given-names></name><name><surname>Grillo</surname><given-names>R</given-names></name><name><surname>Swamy</surname><given-names>MK</given-names></name><name><surname>Sharma</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nano based drug delivery systems: recent developments and future prospects</article-title><source>J Nanobiotechnol</source><year>2018</year><volume>16</volume><issue>1</issue><fpage>1</fpage><lpage>33</lpage><pub-id pub-id-type=\"doi\">10.1186/s12951-018-0392-8</pub-id></mixed-citation></ref><ref id=\"CR144\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Ansari</surname><given-names>A</given-names></name><name><surname>Kim</surname><given-names>H</given-names></name><name><surname>Shin</surname><given-names>HS</given-names></name></person-group><article-title xml:lang=\"en\">Photo-mediated biosynthesis of silver nanoparticles using the non-edible accrescent fruiting calyx of physalis peruviana L. fruits and investigation of its radical scavenging potential and cytotoxicity activities</article-title><source>J Photochem Photobiol, B</source><year>2018</year><volume>188</volume><fpage>116</fpage><lpage>125</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslyqs7vO</pub-id><pub-id pub-id-type=\"pmid\">30266015</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jphotobiol.2018.08.004</pub-id></mixed-citation></ref><ref id=\"CR145\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>H</given-names></name><name><surname>Lt</surname><given-names>Y</given-names></name><name><surname>Ly</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name><name><surname>Zq</surname><given-names>Lu</given-names></name><name><surname>Koup</surname><given-names>RA</given-names></name><name><surname>Bailer</surname><given-names>RT</given-names></name><name><surname>Cy</surname><given-names>Wu</given-names></name></person-group><article-title xml:lang=\"en\">Long-lived memory T lymphocyte responses against SARS coronavirus nucleocapsid protein in SARS-recovered patients</article-title><source>Virology</source><year>2006</year><volume>351</volume><issue>2</issue><fpage>466</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28Xns1Gks7c%3D</pub-id><pub-id pub-id-type=\"pmid\">16690096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.virol.2006.03.036</pub-id></mixed-citation></ref><ref id=\"CR146\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Phan</surname><given-names>IQ</given-names></name><name><surname>Subramanian</surname><given-names>S</given-names></name><name><surname>Kim</surname><given-names>D</given-names></name><name><surname>Murphy</surname><given-names>M</given-names></name><name><surname>Pettie</surname><given-names>D</given-names></name><name><surname>Carter</surname><given-names>L</given-names></name><name><surname>Anishchenko</surname><given-names>I</given-names></name><name><surname>Barrett</surname><given-names>LK</given-names></name><name><surname>Craig</surname><given-names>J</given-names></name><name><surname>Tillery</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">In silico detection of SARS-CoV-2 specific B-cell epitopes and validation in ELISA for serological diagnosis of COVID-19</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>11</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-83730-y</pub-id></mixed-citation></ref><ref id=\"CR147\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pimentel</surname><given-names>TA</given-names></name><name><surname>Yan</surname><given-names>Z</given-names></name><name><surname>Jeffers</surname><given-names>SA</given-names></name><name><surname>Holmes</surname><given-names>KV</given-names></name><name><surname>Hodges</surname><given-names>RS</given-names></name><name><surname>Burkhard</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Peptide nanoparticles as novel immunogens: design and analysis of a prototypic severe acute respiratory syndrome vaccine</article-title><source>Chem Biol Drug Des</source><year>2009</year><volume>73</volume><issue>1</issue><fpage>53</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptFCnsw%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19152635</pub-id><pub-id pub-id-type=\"doi\">10.1111/j.1747-0285.2008.00746.x</pub-id></mixed-citation></ref><ref id=\"CR148\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasad</surname><given-names>R</given-names></name><name><surname>Pandey</surname><given-names>R</given-names></name><name><surname>Barman</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Engineering tailored nanoparticles with microbes: quo vadis</article-title><source>Wires Nanomed Nanobiotechnol</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1002/wnan.1363</pub-id></mixed-citation></ref><ref id=\"CR224\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prusty</surname><given-names>K</given-names></name><name><surname>Swain</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Nano silver decorated polyacrylamide/dextran nanohydrogels hybrid composites for drug delivery applications</article-title><source>Mater Sci Eng, C</source><year>2018</year><volume>85</volume><fpage>130</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitVejtLfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.msec.2017.11.028</pub-id></mixed-citation></ref><ref id=\"CR149\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Puga-Gómez</surname><given-names>R</given-names></name><name><surname>Ricardo-Delgado</surname><given-names>Y</given-names></name><name><surname>Rojas-Iriarte</surname><given-names>C</given-names></name><name><surname>Céspedes-Henriquez</surname><given-names>L</given-names></name><name><surname>Piedra-Bello</surname><given-names>M</given-names></name><name><surname>Vega-Mendoza</surname><given-names>D</given-names></name><name><surname>Pérez</surname><given-names>NP</given-names></name><name><surname>Paredes-Moreno</surname><given-names>B</given-names></name><name><surname>Rodríguez-González</surname><given-names>M</given-names></name><name><surname>Valenzuela-Silva</surname><given-names>C</given-names></name><name><surname>Sánchez-Ramírez</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Open-label phase I/II clinical trial of SARS-CoV-2 receptor binding domain-tetanus toxoid conjugate vaccine (FINLAY-FR-2) in combination with receptor binding domain-protein vaccine (FINLAY-FR-1A) in children</article-title><source>Int J Infect Dis</source><year>2023</year><volume>126</volume><fpage>164</fpage><lpage>173</lpage><pub-id pub-id-type=\"pmid\">36403819</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2022.11.016</pub-id></mixed-citation></ref><ref id=\"CR150\"><mixed-citation publication-type=\"other\">Puri A, Loomis K, Smith B, Lee JH, Yavlovich A, Heldman E, Blumenthal, R (2009) Lipid-based nanoparticles as pharmaceutical drug carriers: from concepts to clinic. Crit Rev™ Ther Drug Carr Syst 26(6)</mixed-citation></ref><ref id=\"CR151\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Que</surname><given-names>H</given-names></name><name><surname>Hong</surname><given-names>W</given-names></name><name><surname>Lan</surname><given-names>T</given-names></name><name><surname>Zeng</surname><given-names>H</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>Wan</surname><given-names>D</given-names></name><name><surname>Bi</surname><given-names>Z</given-names></name><name><surname>Ren</surname><given-names>W</given-names></name><name><surname>Luo</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>He</surname><given-names>C</given-names></name><name><surname>Zhong</surname><given-names>A</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Tripterin liposome relieves severe acute respiratory syndrome as a potent COVID-19 treatment</article-title><source>Signal Transduct Target Ther</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>399</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjtF2rsrzL</pub-id><pub-id pub-id-type=\"pmid\">36566328</pub-id><pub-id pub-id-type=\"pmcid\">9789731</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41392-022-01283-6</pub-id></mixed-citation></ref><ref id=\"CR152\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quick</surname><given-names>J</given-names></name><name><surname>Grubaugh</surname><given-names>ND</given-names></name><name><surname>Pullan</surname><given-names>ST</given-names></name><name><surname>Claro</surname><given-names>IM</given-names></name><name><surname>Smith</surname><given-names>AD</given-names></name><name><surname>Gangavarapu</surname><given-names>K</given-names></name><name><surname>Oliveira</surname><given-names>G</given-names></name><name><surname>Robles-Sikisaka</surname><given-names>R</given-names></name><name><surname>Rogers</surname><given-names>TF</given-names></name><name><surname>Beutler</surname><given-names>NA</given-names></name><name><surname>Burton</surname><given-names>DR</given-names></name></person-group><article-title xml:lang=\"en\">Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples</article-title><source>Nat Protoc</source><year>2017</year><volume>12</volume><issue>6</issue><fpage>1261</fpage><lpage>1276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXosVahu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">28538739</pub-id><pub-id pub-id-type=\"pmcid\">5902022</pub-id><pub-id pub-id-type=\"doi\">10.1038/nprot.2017.066</pub-id></mixed-citation></ref><ref id=\"CR153\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rabiee</surname><given-names>N</given-names></name><name><surname>Bagherzadeh</surname><given-names>M</given-names></name><name><surname>Ghasemi</surname><given-names>A</given-names></name><name><surname>Zare</surname><given-names>H</given-names></name><name><surname>Ahmadi</surname><given-names>S</given-names></name><name><surname>Fatahi</surname><given-names>Y</given-names></name><name><surname>Dinarvand</surname><given-names>R</given-names></name><name><surname>Rabiee</surname><given-names>M</given-names></name><name><surname>Ramakrishna</surname><given-names>S</given-names></name><name><surname>Shokouhimehr</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Point-of-use rapid detection of SARS-CoV-2: nanotechnology-enabled solutions for the COVID-19 pandemic</article-title><source>Int J Mol Sci</source><year>2020</year><volume>21</volume><issue>14</issue><fpage>5126</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVChtLfN</pub-id><pub-id pub-id-type=\"pmid\">32698479</pub-id><pub-id pub-id-type=\"pmcid\">7404277</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms21145126</pub-id></mixed-citation></ref><ref id=\"CR155\"><mixed-citation publication-type=\"other\">Raj S, Khurana S, Choudhari R, Kesari KK, Kamal MA, Garg N, Ruokolainen J, Das BC, Kumar D (2021) In Specific targeting cancer cells with nanoparticles and drug delivery in cancer therapy, Seminars in cancer biology. Elsevier pp 166–177</mixed-citation></ref><ref id=\"CR156\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ramachandran</surname><given-names>AK</given-names></name><name><surname>Das</surname><given-names>S</given-names></name><name><surname>Joseph</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Crosstalk between COVID-19 and associated neurological disorders: a review</article-title><source>Curr Neuropharmacol</source><year>2021</year><volume>19</volume><issue>10</issue><fpage>1688</fpage><lpage>1700</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlyksrnO</pub-id><pub-id pub-id-type=\"pmid\">33441073</pub-id><pub-id pub-id-type=\"doi\">10.2174/1570159X19666210113154342</pub-id></mixed-citation></ref><ref id=\"CR157\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>L</given-names></name><name><surname>Xia</surname><given-names>S</given-names></name><name><surname>Xu</surname><given-names>W</given-names></name><name><surname>Tian</surname><given-names>R</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name><name><surname>Gu</surname><given-names>C</given-names></name><name><surname>Pan</surname><given-names>P</given-names></name><name><surname>Meng</surname><given-names>QF</given-names></name><name><surname>Ca</surname><given-names>X</given-names></name><name><surname>Qu</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Decoy nanoparticles protect against COVID-19 by concurrently adsorbing viruses and inflammatory cytokines</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>44</issue><fpage>27141</fpage><lpage>27147</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Ghu73J</pub-id><pub-id pub-id-type=\"pmid\">33024017</pub-id><pub-id pub-id-type=\"pmcid\">7959535</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.2014352117</pub-id></mixed-citation></ref><ref id=\"CR158\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>PV</given-names></name><name><surname>Krishna</surname><given-names>BS</given-names></name><name><surname>Jeffree</surname><given-names>MS</given-names></name></person-group><source>Coronaviruses: transmission, frontliners, nanotechnology and economy</source><year>2022</year><publisher-loc> Kota Kinabalu</publisher-loc><publisher-name>Universiti Malaysia Sabah Press</publisher-name></mixed-citation></ref><ref id=\"CR159\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Murthy</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">A hybrid design to optimize preparation of lopinavir loaded solid lipid nanoparticles and comparative pharmacokinetic evaluation with marketed lopinavir/ritonavir coformulation</article-title><source>J Pharm Pharmacol</source><year>2014</year><volume>66</volume><issue>7</issue><fpage>912</fpage><lpage>926</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXps1Ohtro%3D</pub-id><pub-id pub-id-type=\"pmid\">24697749</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12217</pub-id></mixed-citation></ref><ref id=\"CR160\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Gadekar</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Design, optimization and evaluation of poly-ɛ-caprolactone (PCL) based polymeric nanoparticles for oral delivery of lopinavir</article-title><source>Drug Dev Ind Pharm</source><year>2015</year><volume>41</volume><issue>1</issue><fpage>131</fpage><lpage>140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVylurrM</pub-id><pub-id pub-id-type=\"pmid\">24180260</pub-id><pub-id pub-id-type=\"doi\">10.3109/03639045.2013.850710</pub-id></mixed-citation></ref><ref id=\"CR161\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>van der Vlies</surname><given-names>A</given-names></name><name><surname>Simeoni</surname><given-names>E</given-names></name><name><surname>Oneil</surname><given-names>C</given-names></name><name><surname>Swartz</surname><given-names>M</given-names></name><name><surname>Hubbell</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Exploiting lymphatic transport and complement activation in nanoparticle vaccines</article-title><source>Tissue Eng Part A</source><year>2008</year><volume>14</volume><fpage>734</fpage><lpage>735</lpage></mixed-citation></ref><ref id=\"CR162\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rizzo</surname><given-names>JM</given-names></name><name><surname>Buck</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">All things ChIP: ChIP-Chip, ChIP-Seq, ChIP-PCR</article-title><source>Epigenet Regul Epigenomics</source><year>2012</year><volume>2</volume><fpage>41</fpage></mixed-citation></ref><ref id=\"CR163\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rocchitta</surname><given-names>G</given-names></name><name><surname>Spanu</surname><given-names>A</given-names></name><name><surname>Babudieri</surname><given-names>S</given-names></name><name><surname>Latte</surname><given-names>G</given-names></name><name><surname>Madeddu</surname><given-names>G</given-names></name><name><surname>Galleri</surname><given-names>G</given-names></name><name><surname>Nuvoli</surname><given-names>S</given-names></name><name><surname>Bagella</surname><given-names>P</given-names></name><name><surname>Demartis</surname><given-names>MI</given-names></name><name><surname>Fiore</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Enzyme biosensors for biomedical applications: strategies for safeguarding analytical performances in biological fluids</article-title><source>Sensors</source><year>2016</year><volume>16</volume><issue>6</issue><fpage>780</fpage><pub-id pub-id-type=\"pmid\">27249001</pub-id><pub-id pub-id-type=\"pmcid\">4934206</pub-id><pub-id pub-id-type=\"doi\">10.3390/s16060780</pub-id></mixed-citation></ref><ref id=\"CR164\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>CD</given-names></name><name><surname>Millar</surname><given-names>JE</given-names></name><name><surname>Baillie</surname><given-names>JK</given-names></name></person-group><article-title xml:lang=\"en\">Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>473</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaksr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30317-2</pub-id></mixed-citation></ref><ref id=\"CR165\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>RL</given-names></name><name><surname>Pelka</surname><given-names>P</given-names></name><name><surname>Mark</surname><given-names>BL</given-names></name></person-group><article-title xml:lang=\"en\">Frontrunners in the race to develop a SARS-CoV-2 vaccine</article-title><source>Can J Microbiol</source><year>2021</year><volume>67</volume><issue>3</issue><fpage>189</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktF2mtbs%3D</pub-id><pub-id pub-id-type=\"pmid\">33264067</pub-id><pub-id pub-id-type=\"doi\">10.1139/cjm-2020-0465</pub-id></mixed-citation></ref><ref id=\"CR166\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sahakijpijarn</surname><given-names>S</given-names></name><name><surname>Moon</surname><given-names>C</given-names></name><name><surname>Koleng</surname><given-names>JJ</given-names></name><name><surname>Christensen</surname><given-names>DJ</given-names></name><name><surname>Williams</surname><given-names>RO</given-names></name></person-group><article-title xml:lang=\"en\">Development of remdesivir as a dry powder for inhalation by thin film freezing</article-title><source>Pharmaceutics</source><year>2020</year><volume>12</volume><issue>11</issue><fpage>1002</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlsFCktr4%3D</pub-id><pub-id pub-id-type=\"pmid\">33105618</pub-id><pub-id pub-id-type=\"pmcid\">7690377</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics12111002</pub-id></mixed-citation></ref><ref id=\"CR167\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Salleh</surname><given-names>A</given-names></name><name><surname>Naomi</surname><given-names>R</given-names></name><name><surname>Utami</surname><given-names>ND</given-names></name><name><surname>Mohammad</surname><given-names>AW</given-names></name><name><surname>Mahmoudi</surname><given-names>E</given-names></name><name><surname>Mustafa</surname><given-names>N</given-names></name><name><surname>Fauzi</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">The potential of silver nanoparticles for antiviral and antibacterial applications: a mechanism of action</article-title><source>Nanomaterials</source><year>2020</year><volume>10</volume><issue>8</issue><fpage>1566</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsl2gu7bL</pub-id><pub-id pub-id-type=\"pmid\">32784939</pub-id><pub-id pub-id-type=\"pmcid\">7466543</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano10081566</pub-id></mixed-citation></ref><ref id=\"CR168\"><mixed-citation publication-type=\"other\">Sen AK, Nath A, Sudeepthi A, Jain SK, Banerjee U (2022) Microfluidics-Based Point-of-Care Diagnostic Devices. In: Advanced microfluidics-based point-of-care diagnostics. CRC Press pp. 99–120</mixed-citation></ref><ref id=\"CR169\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Seo</surname><given-names>G</given-names></name><name><surname>Lee</surname><given-names>G</given-names></name><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Choi</surname><given-names>M</given-names></name><name><surname>Ku</surname><given-names>KB</given-names></name><name><surname>Lee</surname><given-names>CS</given-names></name><name><surname>Jun</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>D</given-names></name><name><surname>Kim</surname><given-names>HG</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of COVID-19 causative virus (SARS-CoV-2) in human nasopharyngeal swab specimens using field-effect transistor-based biosensor</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>5135</fpage><lpage>5142</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnt1SrtrY%3D</pub-id><pub-id pub-id-type=\"pmid\">32293168</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02823</pub-id></mixed-citation></ref><ref id=\"CR01\"><mixed-citation publication-type=\"other\">Sercombe L, Veerati T, Moheimani F, Wu SY, Sood AK, Hua S (2015)\n",
       "Advances and challenges of liposome assisted drug delivery. Front Pharmacol. 6:286</mixed-citation></ref><ref id=\"CR170\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shafiee</surname><given-names>S</given-names></name><name><surname>Cegolon</surname><given-names>L</given-names></name><name><surname>Khafaei</surname><given-names>M</given-names></name><name><surname>Gholami</surname><given-names>N</given-names></name><name><surname>Zhao</surname><given-names>S</given-names></name><name><surname>Khalesi</surname><given-names>N</given-names></name><name><surname>Moosavian</surname><given-names>H</given-names></name><name><surname>Fathi</surname><given-names>S</given-names></name><name><surname>Izadi</surname><given-names>M</given-names></name><name><surname>Ghadian</surname><given-names>A</given-names></name><name><surname>Javanbakht</surname><given-names>M</given-names></name><name><surname>Javanbakht</surname><given-names>A</given-names></name><name><surname>Akhavan-Sigari</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Gastrointestinal cancers, ACE-2/TMPRSS2 expression and susceptibility to COVID-19</article-title><source>Cancer Cell Int</source><year>2021</year><volume>21</volume><issue>1</issue><fpage>431</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVemurzO</pub-id><pub-id pub-id-type=\"pmid\">34399734</pub-id><pub-id pub-id-type=\"pmcid\">8365127</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12935-021-02129-x</pub-id></mixed-citation></ref><ref id=\"CR171\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>V</given-names></name><name><surname>Keniya</surname><given-names>R</given-names></name><name><surname>Shridharani</surname><given-names>A</given-names></name><name><surname>Punjabi</surname><given-names>M</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name><name><surname>Mehendale</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 using CT scan images and deep learning techniques</article-title><source>Emerg Radiol</source><year>2021</year><volume>28</volume><issue>3</issue><fpage>497</fpage><lpage>505</lpage><pub-id pub-id-type=\"pmid\">33523309</pub-id><pub-id pub-id-type=\"pmcid\">7848247</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10140-020-01886-y</pub-id></mixed-citation></ref><ref id=\"CR172\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shan</surname><given-names>B</given-names></name><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Li</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>S</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name><name><surname>Gui</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed nanomaterial-based sensor array for detection of COVID-19 in exhaled breath</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>9</issue><fpage>12125</fpage><lpage>12132</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1aqsLfK</pub-id><pub-id pub-id-type=\"pmid\">32808759</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c05657</pub-id></mixed-citation></ref><ref id=\"CR173\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shanley</surname><given-names>LC</given-names></name><name><surname>Mahon</surname><given-names>OR</given-names></name><name><surname>Kelly</surname><given-names>DJ</given-names></name><name><surname>Dunne</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Harnessing the innate and adaptive immune system for tissue repair and regeneration: Considering more than macrophages</article-title><source>Acta Biomater</source><year>2021</year><volume>133</volume><fpage>208</fpage><lpage>221</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1yns7%2FK</pub-id><pub-id pub-id-type=\"pmid\">33657453</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.actbio.2021.02.023</pub-id></mixed-citation></ref><ref id=\"CR174\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Sharma</surname><given-names>N</given-names></name><name><surname>Kumari</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>T</given-names></name><name><surname>Tripathi</surname><given-names>KM</given-names></name></person-group><article-title xml:lang=\"en\">Nano-carbon based sensors for bacterial detection and discrimination in clinical diagnosis: a junction between material science and biology</article-title><source>Appl Mater Today</source><year>2020</year><volume>18</volume><pub-id pub-id-type=\"doi\">10.1016/j.apmt.2019.100467</pub-id></mixed-citation></ref><ref id=\"CR175\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Pramanik</surname><given-names>A</given-names></name><name><surname>Tchounwou</surname><given-names>C</given-names></name><name><surname>Pedraza</surname><given-names>F</given-names></name><name><surname>Crouch</surname><given-names>RA</given-names></name><name><surname>Chavva</surname><given-names>SR</given-names></name><name><surname>Vangara</surname><given-names>A</given-names></name><name><surname>Sinha</surname><given-names>SS</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Sardar</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Multifunctional biocompatible graphene oxide quantum dots decorated magnetic nanoplatform for efficient capture and two-photon imaging of rare tumor cells</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>20</issue><fpage>10935</fpage><lpage>10943</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnslCqu7o%3D</pub-id><pub-id pub-id-type=\"pmid\">25939643</pub-id><pub-id pub-id-type=\"pmcid\">4570252</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b02199</pub-id></mixed-citation></ref><ref id=\"CR176\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Singh</surname><given-names>B</given-names></name><name><surname>Datta</surname><given-names>B</given-names></name><name><surname>Ashish</surname><given-names>A</given-names></name><name><surname>Dutta</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">A comprehensive review on current COVID-19 detection methods: From lab care to point of care diagnosis</article-title><source>Sens Int</source><year>2021</year><volume>2</volume><pub-id pub-id-type=\"pmid\">34766062</pub-id><pub-id pub-id-type=\"pmcid\">8302821</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.sintl.2021.100119</pub-id></mixed-citation></ref><ref id=\"CR177\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Su</surname><given-names>S</given-names></name><name><surname>Wong</surname><given-names>G</given-names></name><name><surname>Shi</surname><given-names>W</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lai</surname><given-names>AC</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name><name><surname>Bi</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</article-title><source>Trends Microbiol</source><year>2016</year><volume>24</volume><issue>6</issue><fpage>490</fpage><lpage>502</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XksVWmsbc%3D</pub-id><pub-id pub-id-type=\"pmid\">27012512</pub-id><pub-id pub-id-type=\"pmcid\">7125511</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tim.2016.03.003</pub-id></mixed-citation></ref><ref id=\"CR178\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>T</given-names></name><name><surname>Zhang</surname><given-names>YS</given-names></name><name><surname>Pang</surname><given-names>B</given-names></name><name><surname>Hyun</surname><given-names>DC</given-names></name><name><surname>Yang</surname><given-names>M</given-names></name><name><surname>Xia</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Engineered nanoparticles for drug delivery in cancer therapy</article-title><source>Angew Chem Int Ed</source><year>2014</year><volume>53</volume><issue>46</issue><fpage>12320</fpage><lpage>12364</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGksLzO</pub-id></mixed-citation></ref><ref id=\"CR179\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Gu</surname><given-names>H</given-names></name><name><surname>Fang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Deng</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>P</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Zhao</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>C</given-names></name><name><surname>Lang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Fei</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Geng</surname><given-names>S</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>Wei</surname><given-names>W</given-names></name><name><surname>Hu</surname><given-names>Z</given-names></name><name><surname>Han</surname><given-names>G</given-names></name><name><surname>Sun</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Recombinant vaccine containing an RBD-Fc fusion induced protection against SARS-CoV-2 in nonhuman primates and mice</article-title><source>Cell Mol Immunol</source><year>2021</year><volume>18</volume><issue>4</issue><fpage>1070</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1OlurY%3D</pub-id><pub-id pub-id-type=\"pmid\">33731916</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41423-021-00658-z</pub-id></mixed-citation></ref><ref id=\"CR180\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Szunerits</surname><given-names>S</given-names></name><name><surname>Barras</surname><given-names>A</given-names></name><name><surname>Khanal</surname><given-names>M</given-names></name><name><surname>Pagneux</surname><given-names>Q</given-names></name><name><surname>Boukherroub</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructures for the inhibition of viral infections</article-title><source>Molecules</source><year>2015</year><volume>20</volume><issue>8</issue><fpage>14051</fpage><lpage>14081</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtlKrtr7F</pub-id><pub-id pub-id-type=\"pmid\">26247927</pub-id><pub-id pub-id-type=\"pmcid\">6332336</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules200814051</pub-id></mixed-citation></ref><ref id=\"CR181\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>F</given-names></name><name><surname>Quan</surname><given-names>Y</given-names></name><name><surname>Xin</surname><given-names>ZT</given-names></name><name><surname>Wrammert</surname><given-names>J</given-names></name><name><surname>Ma</surname><given-names>MJ</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>TB</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Richardus</surname><given-names>JH</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Lack of peripheral memory B cell responses in recovered patients with severe acute respiratory syndrome: a six-year follow-up study</article-title><source>J Immunol</source><year>2011</year><volume>186</volume><issue>12</issue><fpage>7264</fpage><lpage>7268</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXntVGmsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">21576510</pub-id><pub-id pub-id-type=\"doi\">10.4049/jimmunol.0903490</pub-id></mixed-citation></ref><ref id=\"CR182\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tarim</surname><given-names>EA</given-names></name><name><surname>Karakuzu</surname><given-names>B</given-names></name><name><surname>Oksuz</surname><given-names>C</given-names></name><name><surname>Sarigil</surname><given-names>O</given-names></name><name><surname>Kizilkaya</surname><given-names>M</given-names></name><name><surname>Al-Ruweidi</surname><given-names>MK</given-names></name><name><surname>Yalcin</surname><given-names>HC</given-names></name><name><surname>Ozcivici</surname><given-names>E</given-names></name><name><surname>Tekin</surname><given-names>HC</given-names></name></person-group><article-title xml:lang=\"en\">Microfluidic-based virus detection methods for respiratory diseases</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>143</fpage><lpage>168</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFeisLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00169-7</pub-id></mixed-citation></ref><ref id=\"CR183\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tian</surname><given-names>B</given-names></name><name><surname>Gao</surname><given-names>F</given-names></name><name><surname>Fock</surname><given-names>J</given-names></name><name><surname>Dufva</surname><given-names>M</given-names></name><name><surname>Hansen</surname><given-names>MF</given-names></name></person-group><article-title xml:lang=\"en\">Homogeneous circle-to-circle amplification for real-time optomagnetic detection of SARS-CoV-2 RdRp coding sequence</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFegtbnF</pub-id><pub-id pub-id-type=\"pmid\">32510339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112356</pub-id></mixed-citation></ref><ref id=\"CR184\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ting</surname><given-names>D</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>J</given-names></name><name><surname>Bi</surname><given-names>J</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Multisite inhibitors for enteric coronavirus: antiviral cationic carbon dots based on curcumin</article-title><source>ACS Appl Nano Mater</source><year>2018</year><volume>1</volume><issue>10</issue><fpage>5451</fpage><lpage>5459</lpage><pub-id pub-id-type=\"doi\">10.1021/acsanm.8b00779</pub-id></mixed-citation></ref><ref id=\"CR185\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>To</surname><given-names>KK</given-names></name><name><surname>Yip</surname><given-names>CC</given-names></name><name><surname>Lai</surname><given-names>CY</given-names></name><name><surname>Wong</surname><given-names>CK</given-names></name><name><surname>Ho</surname><given-names>DT</given-names></name><name><surname>Pang</surname><given-names>PK</given-names></name><name><surname>Ng</surname><given-names>AC</given-names></name><name><surname>Leung</surname><given-names>KH</given-names></name><name><surname>Poon</surname><given-names>RW</given-names></name><name><surname>Chan</surname><given-names>KH</given-names></name><name><surname>Cheng</surname><given-names>VC</given-names></name></person-group><article-title xml:lang=\"en\">Saliva as a diagnostic specimen for testing respiratory virus by a point-of-care molecular assay: a diagnostic validity study</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>3</issue><fpage>372</fpage><lpage>378</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1yksbnE</pub-id><pub-id pub-id-type=\"pmid\">29906597</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.06.009</pub-id></mixed-citation></ref><ref id=\"CR186\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tomas</surname><given-names>M</given-names></name><name><surname>Capanoglu</surname><given-names>E</given-names></name><name><surname>Bahrami</surname><given-names>A</given-names></name><name><surname>Hosseini</surname><given-names>H</given-names></name><name><surname>Akbari-Alavijeh</surname><given-names>S</given-names></name><name><surname>Shaddel</surname><given-names>R</given-names></name><name><surname>Rehman</surname><given-names>A</given-names></name><name><surname>Rezaei</surname><given-names>A</given-names></name><name><surname>Rashidinejad</surname><given-names>A</given-names></name><name><surname>Garavand</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">The direct and indirect effects of bioactive compounds against coronavirus</article-title><source>Food Front</source><year>2022</year><volume>3</volume><issue>1</issue><fpage>96</fpage><lpage>123</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitlCgtLc%3D</pub-id><pub-id pub-id-type=\"pmid\">35462942</pub-id><pub-id pub-id-type=\"doi\">10.1002/fft2.119</pub-id></mixed-citation></ref><ref id=\"CR187\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tripathy</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>MK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers and their applications as novel drug delivery carriers</article-title><source>J Appl Pharm Sci</source><year>2013</year><volume>3</volume><issue>09</issue><fpage>142</fpage><lpage>149</lpage></mixed-citation></ref><ref id=\"CR188\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Udugama</surname><given-names>B</given-names></name><name><surname>Kadhiresan</surname><given-names>P</given-names></name><name><surname>Kozlowski</surname><given-names>HN</given-names></name><name><surname>Malekjahani</surname><given-names>A</given-names></name><name><surname>Osborne</surname><given-names>M</given-names></name><name><surname>Li</surname><given-names>VY</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Mubareka</surname><given-names>S</given-names></name><name><surname>Gubbay</surname><given-names>JB</given-names></name><name><surname>Chan</surname><given-names>WC</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosing COVID-19: the disease and tools for detection</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>3822</fpage><lpage>3835</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXlvFChtrk%3D</pub-id><pub-id pub-id-type=\"pmid\">32223179</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02624</pub-id></mixed-citation></ref><ref id=\"CR189\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vaezi</surname><given-names>A</given-names></name><name><surname>Meysamie</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines cost-effectiveness analysis: a scenario for Iran</article-title><source>Vaccines</source><year>2021</year><volume>10</volume><issue>1</issue><fpage>37</fpage><pub-id pub-id-type=\"pmid\">35062698</pub-id><pub-id pub-id-type=\"pmcid\">8777749</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines10010037</pub-id></mixed-citation></ref><ref id=\"CR190\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vahedifard</surname><given-names>F</given-names></name><name><surname>Chakravarthy</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for COVID-19: the role of nanotechnology in the treatment and diagnosis of COVID-19</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>75</fpage><lpage>99</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktlajsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00168-8</pub-id></mixed-citation></ref><ref id=\"CR191\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Varahachalam</surname><given-names>SP</given-names></name><name><surname>Lahooti</surname><given-names>B</given-names></name><name><surname>Chamaneh</surname><given-names>M</given-names></name><name><surname>Bagchi</surname><given-names>S</given-names></name><name><surname>Chhibber</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>K</given-names></name><name><surname>Bolanos</surname><given-names>JF</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Kaushik</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for the SARS-CoV-2: state-of-the-art and future prospects</article-title><source>Int J Nanomed</source><year>2021</year><volume>16</volume><fpage>539</fpage><pub-id pub-id-type=\"doi\">10.2147/IJN.S283686</pub-id></mixed-citation></ref><ref id=\"CR192\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Verma</surname><given-names>SK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Sahoo</surname><given-names>B</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Thirumurugan</surname><given-names>A</given-names></name><name><surname>Parashar</surname><given-names>S</given-names></name><name><surname>Suar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight into the rapid one-step facile biofabrication of antibacterial silver nanoparticles from bacterial release and their biogenicity and concentration-dependent in vitro cytotoxicity to colon cells</article-title><source>RSC Adv</source><year>2017</year><volume>7</volume><issue>64</issue><fpage>40034</fpage><lpage>40045</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVantLrF</pub-id><pub-id pub-id-type=\"doi\">10.1039/C7RA05943D</pub-id></mixed-citation></ref><ref id=\"CR193\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vijayan</surname><given-names>V</given-names></name><name><surname>Mohapatra</surname><given-names>A</given-names></name><name><surname>Uthaman</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>IK</given-names></name></person-group><article-title xml:lang=\"en\">Recent advances in nanovaccines using biomimetic immunomodulatory materials</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>10</issue><fpage>534</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXotFKkt7o%3D</pub-id><pub-id pub-id-type=\"pmid\">31615112</pub-id><pub-id pub-id-type=\"pmcid\">6835828</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11100534</pub-id></mixed-citation></ref><ref id=\"CR194\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Villeret</surname><given-names>B</given-names></name><name><surname>Dieu</surname><given-names>A</given-names></name><name><surname>Straube</surname><given-names>M</given-names></name><name><surname>Solhonne</surname><given-names>B</given-names></name><name><surname>Miklavc</surname><given-names>P</given-names></name><name><surname>Hamadi</surname><given-names>S</given-names></name><name><surname>Le Borgne</surname><given-names>R</given-names></name><name><surname>Mailleux</surname><given-names>A</given-names></name><name><surname>Norel</surname><given-names>X</given-names></name><name><surname>Aerts</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Silver nanoparticles impair retinoic acid-inducible gene I-mediated mitochondrial antiviral immunity by blocking the autophagic flux in lung epithelial cells</article-title><source>ACS Nano</source><year>2018</year><volume>12</volume><issue>2</issue><fpage>1188</fpage><lpage>1202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVart7c%3D</pub-id><pub-id pub-id-type=\"pmid\">29357226</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.7b06934</pub-id></mixed-citation></ref><ref id=\"CR195\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Volkov</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dots in nanomedicine: recent trends, advances and unresolved issues</article-title><source>Biochem Biophys Res Commun</source><year>2015</year><volume>468</volume><issue>3</issue><fpage>419</fpage><lpage>427</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1SqtL3L</pub-id><pub-id pub-id-type=\"pmid\">26168726</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bbrc.2015.07.039</pub-id></mixed-citation></ref><ref id=\"CR196\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Jiao</surname><given-names>G</given-names></name><name><surname>Shan</surname><given-names>X</given-names></name><name><surname>Tai</surname><given-names>W</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Inhibition of influenza A virus infection by fucoidan targeting viral neuraminidase and cellular EGFR pathway</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>14</lpage><pub-id pub-id-type=\"pmid\">28127051</pub-id><pub-id pub-id-type=\"pmcid\">5428335</pub-id></mixed-citation></ref><ref id=\"CR197\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Yunjia</surname><given-names>Y</given-names></name><name><surname>Qin</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>M</given-names></name><name><surname>Mao</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity against enterovirus 71 of sulfated rhamnan isolated from the green alga Monostroma latissimum</article-title><source>Carbohyd Polym</source><year>2018</year><volume>200</volume><fpage>43</fpage><lpage>53</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVSqur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbpol.2018.07.067</pub-id></mixed-citation></ref><ref id=\"CR198\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Horby</surname><given-names>PW</given-names></name><name><surname>Hayden</surname><given-names>FG</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus outbreak of global health concern</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>470</fpage><lpage>473</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30185-9</pub-id></mixed-citation></ref><ref id=\"CR199\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wankar</surname><given-names>JN</given-names></name><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Bohara</surname><given-names>C</given-names></name><name><surname>Singh</surname><given-names>MP</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name></person-group><article-title xml:lang=\"en\">Role of nanomedicine in management and prevention of COVID-19</article-title><source>Front Nanotechnol</source><year>2020</year><volume>2</volume><pub-id pub-id-type=\"doi\">10.3389/fnano.2020.589541</pub-id></mixed-citation></ref><ref id=\"CR200\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Weiss</surname><given-names>C</given-names></name><name><surname>Carriere</surname><given-names>M</given-names></name><name><surname>Fusco</surname><given-names>L</given-names></name><name><surname>Capua</surname><given-names>I</given-names></name><name><surname>Regla-Nava</surname><given-names>JA</given-names></name><name><surname>Pasquali</surname><given-names>M</given-names></name><name><surname>Scott</surname><given-names>JA</given-names></name><name><surname>Vitale</surname><given-names>F</given-names></name><name><surname>Unal</surname><given-names>MA</given-names></name><name><surname>Mattevi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Toward nanotechnology-enabled approaches against the COVID-19 pandemic</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>6</issue><fpage>6383</fpage><lpage>6406</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFWjsLnI</pub-id><pub-id pub-id-type=\"pmid\">32519842</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c03697</pub-id></mixed-citation></ref><ref id=\"CR201\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wilson</surname><given-names>DR</given-names></name><name><surname>Sen</surname><given-names>R</given-names></name><name><surname>Sunshine</surname><given-names>JC</given-names></name><name><surname>Pardoll</surname><given-names>DM</given-names></name><name><surname>Green</surname><given-names>JJ</given-names></name><name><surname>Kim</surname><given-names>YJ</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradable STING agonist nanoparticles for enhanced cancer immunotherapy</article-title><source>Nanomed Nanotechnol Biol Med</source><year>2018</year><volume>14</volume><issue>2</issue><fpage>237</fpage><lpage>246</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhvVOrsLjK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nano.2017.10.013</pub-id></mixed-citation></ref><ref id=\"CR202\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Ye</surname><given-names>F</given-names></name><name><surname>Cui</surname><given-names>M</given-names></name><name><surname>Shibata</surname><given-names>R</given-names></name><name><surname>Xu</surname><given-names>R</given-names></name><name><surname>Cheng</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>DY</given-names></name></person-group><source>Diagnostic methodology and technology in molecular genetic pathology. Molecular genetic pathology</source><year>2013</year><publisher-loc>New York</publisher-loc><publisher-name>Springer</publisher-name><fpage>211</fpage><lpage>300</lpage><pub-id pub-id-type=\"doi\">10.1007/978-1-4614-4800-6_9</pub-id></mixed-citation></ref><ref id=\"CR203\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>S</given-names></name><name><surname>Peng</surname><given-names>Z</given-names></name><name><surname>Xiao</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Yan</surname><given-names>R</given-names></name><name><surname>Luo</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Detection and analysis of nucleic acid in various biological samples of COVID-19 patients</article-title><source>Travel Med Infect Dis</source><year>2020</year><volume>37</volume><pub-id pub-id-type=\"pmid\">32311437</pub-id><pub-id pub-id-type=\"pmcid\">7165102</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tmaid.2020.101673</pub-id></mixed-citation></ref><ref id=\"CR204\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>K</given-names></name><name><surname>Saha</surname><given-names>R</given-names></name><name><surname>Su</surname><given-names>D</given-names></name><name><surname>Krishna</surname><given-names>VD</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Cheeran</surname><given-names>MCJ</given-names></name><name><surname>Wang</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">Magnetic-nanosensor-based virus and pathogen detection strategies before and during COVID-19</article-title><source>ACS Appl Nano Mater</source><year>2020</year><volume>3</volume><issue>10</issue><fpage>9560</fpage><lpage>9580</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjs7vN</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsanm.0c02048</pub-id></mixed-citation></ref><ref id=\"CR206\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xiang</surname><given-names>SD</given-names></name><name><surname>Scholzen</surname><given-names>A</given-names></name><name><surname>Minigo</surname><given-names>G</given-names></name><name><surname>David</surname><given-names>C</given-names></name><name><surname>Apostolopoulos</surname><given-names>V</given-names></name><name><surname>Mottram</surname><given-names>PL</given-names></name><name><surname>Plebanski</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pathogen recognition and development of particulate vaccines: does size matter?</article-title><source>Methods</source><year>2006</year><volume>40</volume><issue>1</issue><fpage>1</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XpvFGiu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">16997708</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ymeth.2006.05.016</pub-id></mixed-citation></ref><ref id=\"CR207\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>G</given-names></name><name><surname>Zeng</surname><given-names>S</given-names></name><name><surname>Zhang</surname><given-names>B</given-names></name><name><surname>Swihart</surname><given-names>MT</given-names></name><name><surname>Yong</surname><given-names>KT</given-names></name><name><surname>Prasad</surname><given-names>PN</given-names></name></person-group><article-title xml:lang=\"en\">New generation cadmium-free quantum dots for biophotonics and nanomedicine</article-title><source>Chem Rev</source><year>2016</year><volume>116</volume><issue>19</issue><fpage>12234</fpage><lpage>12327</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFemurjF</pub-id><pub-id pub-id-type=\"pmid\">27657177</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.chemrev.6b00290</pub-id></mixed-citation></ref><ref id=\"CR208\"><mixed-citation publication-type=\"other\">Xu W, Ling P, Zhang T (2013) Polymeric micelles, a promising drug delivery system to enhance bioavailability of poorly water-soluble drugs. J Drug Deliv</mixed-citation></ref><ref id=\"CR209\"><mixed-citation publication-type=\"other\">Yadav HK, Almokdad AA, Sumia I, Debe MS (2019) Polymer-based nanomaterials for drug-delivery carriers. In Nanocarriers for drug delivery. Elsevier pp 531–556</mixed-citation></ref><ref id=\"CR210\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>X</given-names></name><name><surname>Zhou</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name><name><surname>Jin</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">An overview of biodegradable nanomaterials and applications in vaccines</article-title><source>Vaccine</source><year>2020</year><volume>38</volume><issue>5</issue><fpage>1096</fpage><lpage>1104</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlehurfF</pub-id><pub-id pub-id-type=\"pmid\">31813649</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2019.11.031</pub-id></mixed-citation></ref><ref id=\"CR211\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Shu</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Yu</surname><given-names>Z</given-names></name><name><surname>Fang</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</article-title><source>Lancet Respir Med</source><year>2020</year><volume>8</volume><issue>5</issue><fpage>475</fpage><lpage>481</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjvVWqt70%3D</pub-id><pub-id pub-id-type=\"pmid\">32105632</pub-id><pub-id pub-id-type=\"pmcid\">7102538</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(20)30079-5</pub-id></mixed-citation></ref><ref id=\"CR212\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ye</surname><given-names>S</given-names></name><name><surname>Shao</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Guo</surname><given-names>N</given-names></name><name><surname>Zuo</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Lu</surname><given-names>Z</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>Q</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity of graphene oxide: how sharp-edged structure and charge matter</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>38</issue><fpage>21571</fpage><lpage>21579</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsFSitrbM</pub-id><pub-id pub-id-type=\"pmid\">26370151</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b06876</pub-id></mixed-citation></ref><ref id=\"CR213\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yousaf</surname><given-names>SS</given-names></name><name><surname>Isreb</surname><given-names>A</given-names></name><name><surname>Khan</surname><given-names>I</given-names></name><name><surname>Mewsiga</surname><given-names>E</given-names></name><name><surname>Elhissi</surname><given-names>A</given-names></name><name><surname>Ahmed</surname><given-names>W</given-names></name><name><surname>Alhnan</surname><given-names>MA</given-names></name></person-group><article-title xml:lang=\"en\">Impact of nanosizing on the formation and characteristics of polymethacrylate films: micro-versus nano-suspensions</article-title><source>Pharm Dev Technol</source><year>2021</year><volume>26</volume><issue>7</issue><fpage>729</fpage><lpage>739</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Ggtr7L</pub-id><pub-id pub-id-type=\"pmid\">34034606</pub-id><pub-id pub-id-type=\"doi\">10.1080/10837450.2021.1931886</pub-id></mixed-citation></ref><ref id=\"CR214\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yu</surname><given-names>CY</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Yean</surname><given-names>CY</given-names></name><name><surname>Ang</surname><given-names>GY</given-names></name></person-group><article-title xml:lang=\"en\">Nucleic acid-based diagnostic tests for the detection SARS-CoV-2: an update</article-title><source>Diagnostics</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1CjtLzK</pub-id><pub-id pub-id-type=\"pmid\">33401392</pub-id><pub-id pub-id-type=\"pmcid\">7823986</pub-id><pub-id pub-id-type=\"doi\">10.3390/diagnostics11010053</pub-id></mixed-citation></ref><ref id=\"CR215\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zazo</surname><given-names>H</given-names></name><name><surname>Colino</surname><given-names>CI</given-names></name><name><surname>Lanao</surname><given-names>JM</given-names></name></person-group><article-title xml:lang=\"en\">Current applications of nanoparticles in infectious diseases</article-title><source>J Control Release</source><year>2016</year><volume>224</volume><fpage>86</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XovFCgtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">26772877</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jconrel.2016.01.008</pub-id></mixed-citation></ref><ref id=\"CR216\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Q</given-names></name><name><surname>Honko</surname><given-names>A</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Gong</surname><given-names>H</given-names></name><name><surname>Downs</surname><given-names>SN</given-names></name><name><surname>Vasquez</surname><given-names>JH</given-names></name><name><surname>Fang</surname><given-names>RH</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Griffiths</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Cellular nanosponges inhibit SARS-CoV-2 infectivity</article-title><source>Nano Lett</source><year>2020</year><volume>20</volume><issue>7</issue><fpage>5570</fpage><lpage>5574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFKiu7rM</pub-id><pub-id pub-id-type=\"pmid\">32551679</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.nanolett.0c02278</pub-id></mixed-citation></ref><ref id=\"CR217\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>F</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Vijver</surname><given-names>MG</given-names></name><name><surname>Peijnenburg</surname><given-names>WJ</given-names></name></person-group><article-title xml:lang=\"en\">Probing nano-QSAR to assess the interactions between carbon nanoparticles and a SARS-CoV-2 RNA fragment</article-title><source>Ecotoxicol Environ Saf</source><year>2021</year><volume>219</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFynurvJ</pub-id><pub-id pub-id-type=\"pmid\">34044308</pub-id><pub-id pub-id-type=\"pmcid\">8133531</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2021.112357</pub-id></mixed-citation></ref><ref id=\"CR218\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>K</given-names></name><name><surname>Di</surname><given-names>L</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Yue</surname><given-names>P</given-names></name><name><surname>Song</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Traditional herbal medicine and nanomedicine: Converging disciplines to improve therapeutic efficacy and human health</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>178</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVelsrzI</pub-id><pub-id pub-id-type=\"pmid\">34499982</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2021.113964</pub-id></mixed-citation></ref><ref id=\"CR219\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>SH</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>SJ</given-names></name><name><surname>Yao</surname><given-names>L</given-names></name><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>YD</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 pneumonia via a novel deep learning architecture</article-title><source>J Comput Sci Technol</source><year>2022</year><volume>37</volume><issue>2</issue><fpage>330</fpage><lpage>343</lpage><pub-id pub-id-type=\"pmid\">35496726</pub-id><pub-id pub-id-type=\"pmcid\">9035772</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11390-020-0679-8</pub-id></mixed-citation></ref><ref id=\"CR220\"><mixed-citation publication-type=\"other\">Zhang F, Abudayyeh OO, Gootenberg JS (2020a) A protocol for detection of COVID-19 using CRISPR diagnostics. A protocol for detection of COVID-19 using CRISPR diagnostics 8.</mixed-citation></ref><ref id=\"CR221\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhao</surname><given-names>W</given-names></name><name><surname>Zhong</surname><given-names>Z</given-names></name><name><surname>Xie</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Q</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Relation between chest CT findings and clinical conditions of coronavirus disease (COVID-19) pneumonia: a multicenter study</article-title><source>Ajr Am J Roentgenol</source><year>2020</year><volume>214</volume><issue>5</issue><fpage>1072</fpage><lpage>1077</lpage><pub-id pub-id-type=\"pmid\">32125873</pub-id><pub-id pub-id-type=\"doi\">10.2214/AJR.20.22976</pub-id></mixed-citation></ref><ref id=\"CR222\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhu</surname><given-names>J</given-names></name><name><surname>Shi</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer-based nanodevices for targeted drug delivery applications</article-title><source>J Mater Chem B</source><year>2013</year><volume>1</volume><issue>34</issue><fpage>4199</fpage><lpage>4211</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1entrfP</pub-id><pub-id pub-id-type=\"pmid\">32261015</pub-id><pub-id pub-id-type=\"doi\">10.1039/c3tb20724b</pub-id></mixed-citation></ref><ref id=\"CR223\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ziegler</surname><given-names>CG</given-names></name><name><surname>Allon</surname><given-names>SJ</given-names></name><name><surname>Nyquist</surname><given-names>SK</given-names></name><name><surname>Mbano</surname><given-names>IM</given-names></name><name><surname>Miao</surname><given-names>VN</given-names></name><name><surname>Tzouanas</surname><given-names>CN</given-names></name><name><surname>Cao</surname><given-names>Y</given-names></name><name><surname>Yousif</surname><given-names>AS</given-names></name><name><surname>Bals</surname><given-names>J</given-names></name><name><surname>Hauser</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues</article-title><source>Cell</source><year>2020</year><volume>181</volume><issue>5</issue><fpage>1016</fpage><lpage>1035</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpsVKkuro%3D</pub-id><pub-id pub-id-type=\"pmid\">32413319</pub-id><pub-id pub-id-type=\"pmcid\">7252096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cell.2020.04.035</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">13197</journal-id><journal-id journal-id-type=\"doi\">10.1007/13197.0975-8402</journal-id><journal-title-group><journal-title>Journal of Food Science and Technology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Food Sci Technol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0022-1155</issn><issn pub-type=\"epub\">0975-8402</issn><publisher><publisher-name>Springer India</publisher-name><publisher-loc>New Delhi</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s13197-022-05528-8</article-id><article-id pub-id-type=\"manuscript\">5528</article-id><article-id pub-id-type=\"doi\">10.1007/s13197-022-05528-8</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Bioactive metabolites in functional and fermented foods and their role as immunity booster and anti-viral innate mechanisms</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Varsha</surname><given-names>Kontham Kulangara</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Narisetty</surname><given-names>Vivek</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Brar</surname><given-names>Kamalpreet Kaur</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"aff\" rid=\"Aff4\">4</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Madhavan</surname><given-names>Aravind</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Alphy</surname><given-names>Maria Paul</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Sindhu</surname><given-names>Raveendran</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Awasthi</surname><given-names>Mukesh Kumar</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Varjani</surname><given-names>Sunita</given-names></name><xref ref-type=\"aff\" rid=\"Aff10\">10</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au9\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-4191-8048</contrib-id><name><surname>Binod</surname><given-names>Parameswaran</given-names></name><address><email>binodkannur@niist.res.in</email></address><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref><xref ref-type=\"corresp\" rid=\"IDs13197022055288_cor9\">j</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.254567.7</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 9075 106X</institution-id><institution content-type=\"org-division\">School of Medicine</institution><institution content-type=\"org-name\">University of South Carolina</institution></institution-wrap><addr-line content-type=\"postcode\">29209</addr-line><addr-line content-type=\"city\">Columbia</addr-line><addr-line content-type=\"state\">SC</addr-line><country country=\"US\">USA</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Moolec Science, Innovation Centre</institution></institution-wrap><addr-line content-type=\"street\">Gallows Hill, CV34 6UW</addr-line><addr-line content-type=\"city\">Warwick</addr-line><country country=\"GB\">UK</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.21100.32</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1936 9430</institution-id><institution content-type=\"org-division\">Department of Civil Engineering, Lassonde School of Engineering</institution><institution content-type=\"org-name\">York University</institution></institution-wrap><addr-line content-type=\"postcode\">M3J 1P3</addr-line><addr-line content-type=\"city\">North York, Toronto</addr-line><addr-line content-type=\"state\">Ontario</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.468122.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 5906 3081</institution-id><institution content-type=\"org-name\">Centre Technologique des Résidus Industriels en Abitibi Témiscamingue</institution></institution-wrap><addr-line content-type=\"street\">433 Boulevard du collège, J9X0E1</addr-line><addr-line content-type=\"city\">Rouyn-Noranda</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.418917.2</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0177 8509</institution-id><institution content-type=\"org-name\">Rajiv Gandhi Centre for Biotechnology</institution></institution-wrap><addr-line content-type=\"postcode\">695 014</addr-line><addr-line content-type=\"city\">Thiruvananthapuram</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.419023.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1808 3107</institution-id><institution content-type=\"org-division\">Microbial Processes and Technology Division</institution><institution content-type=\"org-name\">CSIR-National Institute for Interdisciplinary Science and Technology</institution></institution-wrap><addr-line content-type=\"postcode\">695 019</addr-line><addr-line content-type=\"city\">Trivandrum</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.469887.c</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 7744 2771</institution-id><institution content-type=\"org-name\">Academy of Scientific and Innovative Research (AcSIR)</institution></institution-wrap><addr-line content-type=\"postcode\">201002</addr-line><addr-line content-type=\"city\">Ghaziabad</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Department of Food Technology</institution><institution content-type=\"org-name\">T K M Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">691505</addr-line><addr-line content-type=\"city\">Kollam</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.144022.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1760 4150</institution-id><institution content-type=\"org-division\">College of Natural Resources and Environment</institution><institution content-type=\"org-name\">Northwest A &amp; F University</institution></institution-wrap><addr-line content-type=\"postcode\">712 100</addr-line><addr-line content-type=\"city\">Yangling</addr-line><addr-line content-type=\"state\">Shaanxi</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff10\"><label>10</label><institution-wrap><institution content-type=\"org-name\">Gujarat Pollution Control Board</institution></institution-wrap><addr-line content-type=\"street\">Paryavaran Bhavan, CHH Road, Sector 10 A</addr-line><addr-line content-type=\"postcode\">382010</addr-line><addr-line content-type=\"city\">Gandhinagar</addr-line><addr-line content-type=\"state\">Gujarat</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs13197022055288_cor9\"><label>j</label><email>binodkannur@niist.res.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>24</day><month>6</month><year>2022</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>9</month><year>2023</year></pub-date><volume>60</volume><issue seq=\"2\">9</issue><fpage>2309</fpage><lpage>2318</lpage><history><date date-type=\"registration\"><day>1</day><month>6</month><year>2022</year></date><date date-type=\"rev-recd\"><day>19</day><month>4</month><year>2022</year></date><date date-type=\"accepted\"><day>18</day><month>5</month><year>2022</year></date><date date-type=\"online\"><day>24</day><month>6</month><year>2022</year></date></history><permissions><copyright-statement content-type=\"compact\">© Association of Food Scientists &amp; Technologists (India) 2022</copyright-statement><copyright-year>2022</copyright-year><copyright-holder>Association of Food Scientists &amp; Technologists (India)</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par15\">Live microorganisms in the fermented foods termed probiotics and their secondary metabolites with bioactive potential were considered as potential anti-viral capabilities through various mechanisms. Given the importance of functional and fermented foods in disease prevention, there is a need to discuss the contextualization and deep understanding of the mechanism of action of these foods, particularly considering the appearance of coronavirus (COVID-19) pandemic, which is causing health concerns and increased social services globally. The mechanism of probiotic strains or their bioactive metabolites is due to stimulation of immune response through boosting T-lymphocytes, cytokines, and cell toxicity of natural killer cells. Proper consumption of these functional and fermented foods may provide additional antiviral approaches for public benefit by modulating the immune functions in the hosts.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Anti-viral metabolites</kwd><kwd>Immunity</kwd><kwd>Fermented foods</kwd><kwd>COVID-19</kwd><kwd>Functional foods</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>19</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Association of Food Scientists &amp; Technologists (India)</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2022</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/13197_2022_Article_5528.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Kontham Kulangara Varsha and Vivek Narisetty these authors contributed equally and Co-first authors.</p></notes><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s13197-022-05528-8\">https://doi.org/10.1007/s13197-022-05528-8</ext-link>.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par42\">The process of fermentation has long been used to increase the shelf life, flavour and functional properties of food. In addition to help food last longer, fermentation increases the nutritional value, and the probiotic bacteria present in fermented food confer health benefits that comprise reducing the risk of type 2 diabetes and cardiovascular diseases by decreasing total and LDL cholesterol (Marco et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). Multiple clinical trials investigated the advantages of fermented food and exposed the ability of kimchi and yogurt to downsize the risk of type 2 diabetes (Chen et al. <xref ref-type=\"bibr\" rid=\"CR7\">2014</xref>), Chungkookjang to decrease obesity (Byun et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>), and the capacity of fermented milk and rye bread to control infection and irritable bowel syndrome (Laatikainen et al. <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>). Among this health promoting and infectivity demoting effects include the antiviral activity of certain fermented foods owing to the presence of live bacteria in it, and the examples are Chr. Hansen (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.chr-hansen.com/\">http://www.chr-hansen.com/</ext-link>); Kingdom Supercultures (<ext-link ext-link-type=\"uri\" xlink:href=\"https://kingdomsupercultures.com/\">https://kingdomsupercultures.com/</ext-link>); Probitat (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.probitat.eu/\">http://www.probitat.eu/</ext-link>); 3FBIO Ltd (ENOUGH) (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.enough-food.com/\">https://www.enough-food.com/</ext-link>); Fermbiotics (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fermbiotics.com/\">https://www.fermbiotics.com/</ext-link>).</p><p id=\"Par43\">Functional foods (FFs) are distinguished by their nutraceuticals. Nutraceuticals are either whole foods or food ingredients that provide health benefits, such as disease prevention and/or treatment. This is generally associated with their microvascular, anti-inflammatory, and anti-oxidation properties in highly affected individuals. Active FFs are rich in polyphenols, terpenoids, flavonoids, and unsaturated fatty acids ingredients are among the widely active functional foods to be consumed (Alkhatib et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018</xref>). Recently, Acquah et al. (<xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>) reported that some bioactive peptides presented many similar hormonal and neurological activities of human system (Acquah et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>). On the other side, fermented food products contain probiotics (García-Burgos et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Probiotics are “live microorganisms which when administered in adequate amounts confer a health benefit on the host”. Covid-19 was observed to be a severe acute respiratory syndrome (SARS) which is named as SARS corona virus 2 or SARS-CoV-2 (Lai et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). Broadly, coronaviruses are large and enveloped mainly found in humans and mammals and known to cause respiratory, gastrointestinal, and neurological disease. Through genetic recombination and mutation, corona viruses can be more infectious. Stringent measures were taken by various countries based on their resource limitations, geography, population, and political factors. Although these severe interventions, since Feb. 2020 and as of March 17, 2022, the outbreak has infected almost 464 million people and killed over 6.06 million. Previous studies have shown that 65% of airborne MERS-CoV virus remains viable in the air and infectious after 60 min (Pyankov et al. <xref ref-type=\"bibr\" rid=\"CR61\">2018</xref>). The presence of SARS-CoV-2 in hospitals and entrance to department stores in Wuhan, China (Liu et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>), air outlet fans in a COVID-19 outbreak center in Singapore (Ong et al. <xref ref-type=\"bibr\" rid=\"CR57\">2020</xref>) and hospital isolation rooms in Nebraska (Santarpia et al. <xref ref-type=\"bibr\" rid=\"CR63\">2020</xref>) were observed. Recently, some preliminarily results have shown that the virus can survive up to 3 h as aerosol and infect cells throughout this period (van Doremalen et al. <xref ref-type=\"bibr\" rid=\"CR67\">2020</xref>). A ferret model of SARS-CoV-2 infection that reiterates aspects of human disease has also confirmed the potential of virus airborne transmission (Kim et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020</xref>). A recent study reported that SARS-CoV-2 can be viable for 4 h on copper, 24 h on cardboard, 2–3 days on plastic and stainless steel (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Generally, SARS-CoV-2 is transmitted via respiratory pathways, but they may spread via multiple dominant routes. An acceptor individual must receive an infectious dose of the virus from a donor, either directly through the air or indirectly through deposits of the virus on various surfaces. Molecular based detection techniques, like plaque assay, Enzyme-Linked Immunosorbent Assay (ELISA), Lateral-Flow (immuno) Assay (LFA), Polymerase Chain Reaction (PCR) and Surface Plasmon Resonance (SPR) assay are identified quality and quantitative techniques for virus detection. Approximately 120 candidates are hardly working to formulate vaccines based on either nucleic acids, inactivated or live attenuated virus, and recombinant proteins (Le et al. 2020). Other approaches based on monoclonal antibodies, hyperimmune globulin will be another option. The antiviral role of FFs and fermented foods for the defence of COVID-19 is lacking and has not yet been established. However, recent reports showed that diabetes is considered a risk factor for the development and diagnosis of COVID-19 (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Fermented foods generally contain single or multiple genera of live probiotic microorganisms that has positive impact on the host beneficially preserving the intestinal microbiota that in return has modulatory role in immune responses and human health (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). Herein, we present antiviral role of fermented foods containing of probiotic and bioactive compounds; mechanism to stimulate immunity; case studies of viral infections and their prevention or treatment and different types of fermented foods available worldwide. This review leads to inculcating the knowledge of benefits of probiotics and fermented foods in reducing the risk of COVID-19 or other viral infections.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic representation of applications of probiotics and bioactive metabolites</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec2\"><title>Antiviral activity of fermented foods containing probiotics and bioactive compounds</title><p id=\"Par45\">The plethora of fermented foods is derived from plants and animal resources. Plant-based fermented foods are tempeh, <italic>kimchi</italic>, <italic>tempoyak</italic>, and <italic>tapai</italic>. While animal-based fermented foods include kefir, cheese, yogurt, sauces, other dairy and meat products (Tamang et al., 2016; Raji et al. <xref ref-type=\"bibr\" rid=\"CR62\">2017</xref>; Khalil et al. <xref ref-type=\"bibr\" rid=\"CR32\">2018</xref>; Lee et al. <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Alcoholic beverages like fermented porridge made of sorghum and maize in South American countries (Tamang et al. <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>). Fermented foods are enriched with anti-microbial end products such as various organic acids, ethanol and peptides or bacteriocins and several studies report the antiviral potential of fermented foods <italic>in-vitro</italic> and <italic>in-vivo</italic>. The probiotic bacteria and bioactive compounds in fermented foods possess antiviral activities against gut and respiratory and viruses. These active foods stimulate immune system function by increasing the synthesis of pro-inflammatory cytokines and T lymphocytes (CD3<sup>+</sup>, CD16<sup>+</sup>, CD56<sup>+</sup>) (Muhialdin et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>). <italic>Lactobacillus plantarum</italic> LBP-K10 isolated from kimchi synthesised cyclic di-peptides that inhibited the growth of the influenza A (H3N2) virus (Kwak et al. <xref ref-type=\"bibr\" rid=\"CR38\">2013</xref>), while another study reported declined survival of feline calicivirus and murine norovirus proliferation during <italic>Dongchimi</italic> fermentation along with an increase in lactic acid bacteria (LAB) (Lee et al. <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). Likewise, soy extracts fermented with <italic>Aspergillus fumigatus</italic> F-993 or <italic>A. awamori</italic> FB-133 showed therapeutic potential by decreasing hepatitis A virus titers <italic>in-vitro</italic> (Ghanem et al. <xref ref-type=\"bibr\" rid=\"CR14\">2020</xref>).</p><p id=\"Par46\">The cell free supernatant of yogurt has antiviral activity for RNA viruses such as enterovirus 71 and influenza, porcine epidemic diarrhoea virus and Coxsackie A and B viruses (Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>). Polyphenols, bioactive peptides, exopolysaccharides, linoleic acid, and vitamins are among the bioactive compounds found in fermented foods (Hayes and García-Vaquero <xref ref-type=\"bibr\" rid=\"CR24\">2016</xref>). Spanish sausage release angiotensin-converting enzyme inhibitor (ACE-I) when <italic>L. pentosus</italic> and <italic>S. carnosus</italic> used as a inoculum for the fermentation (Mora et al. <xref ref-type=\"bibr\" rid=\"CR51\">2015</xref>). The fermentation of <italic>Ruditapes philippinarum</italic> clams with <italic>Bacillus natto</italic> stimulate hyper-production of ACE-I peptide synthesis exerts anti-cancer property (Chen et al. <xref ref-type=\"bibr\" rid=\"CR8\">2018</xref>). However, further studies assessing the full potential of the probiotics to combat COVID-19 should be carried out (Olaimat et al. <xref ref-type=\"bibr\" rid=\"CR55\">2020</xref>) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>).</p></sec><sec id=\"Sec3\"><title>Clinical impact of probiotics and bioactive compounds against viral infections in humans</title><p id=\"Par47\">High mutation rates of RNA viruses lead to their rapid evolution and better environmental adaptability (Carrasco-Hernandez et al. <xref ref-type=\"bibr\" rid=\"CR6\">2017</xref>). Various case studies report the ability of fermented and probiotic food to reduce respiratory tract infections. In one case study, some COVID-19 patients exhibited <italic>Bifidobacterium</italic> and <italic>Lactobacillus</italic> dysbiosis. The aged patients who suffered severely from COVID-19 had poor gut microbiota diversity (Dhar et al., 2020). Probiotics <italic>L. acidophilus, Bifidobacterium</italic>, and <italic>Saccharomyces boulardii</italic>, as well as minerals and vitamins, were found to reduce the complications of massive antibiotic-associated diarrhoea and <italic>Clostridium difficile</italic> infections. (Horowitz et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). The simultaneous intake of probiotics with azithromycin decreases the severity of <italic>Candida albicans</italic> infection. In another study, COVID-19-like symptoms in a young boy disappeared after 2 days of probiotic administration. (Ji et al. <xref ref-type=\"bibr\" rid=\"CR28\">2020</xref>). The administration of probiotics as an adjunct protected 97% patients against SARS-CoV2 infection. According to the recent trail, patients with severe disease recovered more easily by probiotics therapy (87.5% vs. 40.4%, p = 0.037) compared to non-severe ones. A human clinical trial describes oral intake of the probiotic <italic>L. fermentum</italic> CECT5716 increased NK cells proliferation after vaccination when compared to the group without probiotic consumption (Olivares et al. <xref ref-type=\"bibr\" rid=\"CR56\">2007</xref>). A study conducted on elderly people found to be having a significant increase in NK cell activity (Makino et al. <xref ref-type=\"bibr\" rid=\"CR46\">2010</xref>). At the same time, a randomized control clinical trial among women consumed 1073R-1-yogurt yogurt (n = 479) demonstrated IFN-γ production with no increase in NK cell activity (Kinoshita et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>).</p><p id=\"Par48\">A control trial revealed that the probiotic strain <italic>Lactobacillus</italic> GG holds an adjuvant potential (Davidson et al. <xref ref-type=\"bibr\" rid=\"CR11\">2011</xref>). In children, for acute rotavirus diarrhoea, probiotic administration can relieve symptoms (Grandy et al. <xref ref-type=\"bibr\" rid=\"CR17\">2010</xref>). Multiple clinical trials are underway where the adeptness of probiotic and other dietary supplements is being investigated to alleviate the symptoms of COVID-19 infection (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>).</p><p id=\"Par49\">In the last two years of CoviD-19 pandemic, priority was to prevent the spreading, reduce the infections, save the lives, inculcate the knowledge of vaccination and healthy diet. Frontline health teams would benefit from the development of cutting-edge technology and the collection of available evidence. Sufficient nutrition improves health and boosts immunity, which aids in the prevention and treatment of infections. In this review we have discussed the role of probiotics in combating COVID-19 based on recent evidences, as well as their role as immune-modulators and antiviral agents. Further investigations on impact of probiotic strains and their bioactive compounds on COVID-19 affected individuals are awaiting (Kurian et al. <xref ref-type=\"bibr\" rid=\"CR37\">2021</xref>). However, more research needs to be done to study the production titres of interleukins, interferons, antibodies, and viral count due to probiotic administration during viral infection especially SARS-CoV-2.</p></sec><sec id=\"Sec4\"><title>Immune modulatory mechanisms by probiotics and fermented food</title><p id=\"Par50\">The health benefits conferred by fermented food are mainly attributed to the presence of live probiotic bacteria present which colonize the intestine and impart favourable effects. Lymphocytes, NK cells, macrophages and neutrophils are capable of mediating antibody dependent cell mediated cytotoxicity (ADCC) against virus infected cells and various probiotic and fermented food are reported to induce these immune cells. Multiple clinical trials have shown the ability of fermented food products with probiotics to improve NK cell activity and <italic>L. casei</italic> strain Shirota is a principal probiotic strain with this ability (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Additionally, consumption of dairy yogurt containing probiotics increased NK cell, IL-12, IFNγ and IgG1 levels in a randomized, open-label, placebo-controlled study conducted in 200 healthy volunteers (Lee et al. <xref ref-type=\"bibr\" rid=\"CR43\">2017</xref>). Probiotics play critical role in resistance against viral infections particularly in elderly people with age-related decline in lymphoid cell activity, by modulating the immune responses of hosts. Randomized control trials show that probiotic consumption significantly increased NK cell activity and CD56-positive lymphocytes in peripheral circulation in healthy elderly individuals (Gui et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). Probiotic intakes enhance cellular immunity in elderly who had poor pretreatment immune responses. Dietary supplementation of probiotic <italic>B. lactis</italic> HN019 increased helper (CD4 (+)) and activated (CD25 (+)) T lymphocytes, polymorphonuclear cells and natural killer cells in healthy elderly volunteers (Gill et al. <xref ref-type=\"bibr\" rid=\"CR16\">2001</xref>). Probiotic consumption found to increase natural and acquired immunity in mice and significantly improved serum antibody responses to antigens administered orally and systemically (Gill et al. <xref ref-type=\"bibr\" rid=\"CR15\">2000</xref>). Oral administration of heat killed probiotics from Mongolian dairy products augmented IFN -α, IL-12, and IFN -γ productions and increased NK cell activity leading to alleviated influenza symptoms in mice (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Similarly, oral administration of <italic>L. rhamnoses</italic> CRL1505 improved resistance to RSV infection in infant mice via IFN-γ and IL-10 secretion, which resulted in the activation of CD103<sup>+</sup> and CD11b<sup>high</sup> dendritic cells and the generation of CD3<sup>+</sup>CD4<sup>+</sup>IFN-γ<sup>+</sup> Th1 cells, with subsequent attenuation of strong Th2 reactions associated with RSV challenge (Chiba et al. <xref ref-type=\"bibr\" rid=\"CR9\">2013</xref>). An in vitro model study demonstrated the ability of probiotic bacteria to decrease vesicular stomatitis viral infection by production of nitric oxide and inflammatory cytokines such as IL-6 and IFN-γ (Ivec et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). When <italic>L. casei</italic> DK128 was administered into a mice intranasally, reduction in the weight and viral loads was observed, which might elicit the protection against different subtypes of influenza viruses, and mice are observed to be immune to primary infection and subsequently developed heterosubtypic secondary virus infection. The protective effect was linked to an increase in alveolar macrophage cells in the lungs and airways, the early induction of virus-specific antibodies, and lower levels of pro-inflammatory cytokines and innate immune cells (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Jung et al. <xref ref-type=\"bibr\" rid=\"CR29\">2017</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Possible mechanism of anti-viral activity by probiotics</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig2_HTML.png\"/></fig></p><p id=\"Par52\">Exopolysaccharides (EPS) secreted by probiotic microrganisms in fermented food contribute towards their immunomodulatory ability and antiviral potential <italic>in-vivo</italic>. Consumption of yogurt consisting of the starter culture <italic>L. delbrueckii</italic> ssp. <italic>bulgaricus</italic> OLL1073R-1 and secreted EPS reduced influenza virus titer in mice. The prognosis observed significant increase of anti-influenza virus antibodies such as IgA and IgG<sub>1</sub> along with augmented NK cell activity. In knockout mice, the presence of myeloid differentiation factor 88, EPS produced by this probiotic strain activated NK cells through IL-12- and IL-18-mediated IFN-γ production. The same probiotic strain demonstrated resistance flu virus by inducing NK cell activity in human subjects (Makino et al. <xref ref-type=\"bibr\" rid=\"CR47\">2016</xref>). EPS produced by <italic>L. delbrueckii</italic> TUA4408L improved the resistance to rotaviral infection by preventing the viral replication, activation of Toll-like receptor 3 rendering antiviral innate immune response in porcine intestinal epithelial cells. The study also reports <italic>L. delbrueckii</italic> TUA4408L and its EPS activated interferon regulatory factor (IRF)-3 and nuclear factor κB (NF-κB) signalling pathways leading to improved expression of the antiviral factors IFN-β, Myxovirus resistance gene A (MxA) and RNaseL (Kanmani et al. <xref ref-type=\"bibr\" rid=\"CR30\">2018</xref>). In addition to EPS, short chain fatty acids (SCFAs) produced by these microorganisms are observed to regulate immune responses (Parada Venegas et al. <xref ref-type=\"bibr\" rid=\"CR58\">2019</xref>). Indirectly D-phenyl lactate produced by various lactic acid bacteria regulates the immune reactions stimulated by G-protein coupled receptors by activating regulatory hydroxycarboxylic acids (Peters et al. <xref ref-type=\"bibr\" rid=\"CR59\">2019</xref>). Lactate and pyruvate also contribute to enhanced immune responses in mice models by inducing GPR31-mediated dendrite protrusion of intestinal CX3CR1<sup>+</sup> cells (Morita et al. <xref ref-type=\"bibr\" rid=\"CR52\">2019</xref>). The modulation of gut microbiota seems to be a one of the approaches to combat viral infections including COVID-19, but it needs to be further confirmed through animal models (Akour <xref ref-type=\"bibr\" rid=\"CR2\">2020</xref>). Overall, research indicates that fermented foods and probiotics contain high or low molecular weight bioactive metabolites that elicit modulations in the immune system rendering health benefits.</p></sec><sec id=\"Sec5\"><title>Future outlook</title><p id=\"Par53\">Fermentation by microbes applies various unconventional and uncharacterised enzymes to produce hydrolysates of protein with varying compositions of peptides. By utilizing highly efficient bacteria and by optimizing conditions of fermentation, a wide array of proteases could be produced to hydrolyse peptides and proteins with varying amino acid composition and different chain length could be synthesised in accordance to different enzyme specificity, possibly producing unique sequences of peptide with novel anti-viral bioactive properties.</p><p id=\"Par54\">It is predictable that the greatest perplexing part for researchers dealing with functional food bioactive peptides is to warrant the full bioavailability of the bioactive component after their consumption. The bioavailability of bioactive peptide is based on the capability of the peptides to resist proteolytic activity in the intestine and serum, and their potential to enter into the blood stream and consequently, exercising their biological activity. Hence, it is critical to improve the processing settings in order to retain their biological activities. Encapsulation is one of the well-characterised technologies to improve the bioavailability of functionally active peptides. This technique has been extensively applied in the nutraceutical, and food industries to encapsulate biologically active components. Encapsulation of bioactive is economically viable as this technique could avoid the interaction of the peptides with environment and/or other components in the food matrix, decrease the consequence from processing, intensify peptide stability, protect peptides against digestive enzymes and improve bioavailability.</p><p id=\"Par55\">Since functionally active foods are graded as a normal or enriched with bioactive, it has certain side effects, although the research work in this perspective is not adequate to support this statement. Even though much cell line based, and animal model studies have been widely explored to examine the bioactivity of food derived bioactive, inadequate data on human experiments is available. More research data on human clinical trials are required to validate the efficiency of food derived bioactivities. Along with this other safety parameters, like cytotoxicity, allergic response of the functional food should be evaluated before commercialization.</p><p id=\"Par56\">Viral diseases are considered as an immune compromised state resulting from poor consumption of micronutrients, vitamins and other trace elements. Several research previously indicated enhanced function of the immune system by taking those many fermented and functional foods including essential fatty and amino acids, and the above-mentioned minerals and vitamins (Calder and Kew <xref ref-type=\"bibr\" rid=\"CR5\">2002</xref>). Satisfactory diet-based consumption, and supplementation of such functional foods, contribute to sustaining optimal levels in the humans, which improves several characteristics of the immune function, and provides an important antiviral prevention of COVID-19. On the other hand, less strong immune activation has been proved to be the primary threat factor for COVID-19, which makes it appropriate to define the defensive role of functional food particles benefits in the perspective of preventing COVID-19 and other viral diseases (Grant et al. <xref ref-type=\"bibr\" rid=\"CR18\">2020</xref>).</p><p id=\"Par57\">Examination of outcome of viral disease management in high-risk populations and aged people is very central. Large number of viral and COVID 19 infection rate is reported in older adults and persons with other co-morbidities. The prevalence of COVID-19 in people with diabetes is high and now considered a hazard factor for the progression COVID-19 (McGurnaghan et al. <xref ref-type=\"bibr\" rid=\"CR49\">2021</xref>). Therefore, best “immune-modulating” functional foods could provide the finest prevention and progression of the viral diseases. The use of functional foods to combat viral diseases would especially benefit the elderly, which has become a growing sector of the world population.</p></sec><sec id=\"Sec6\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par58\">Fermentation by microbe’s functions as a potent technique to supplement foods with biologically active peptides from various animal or plant sources. These bioactive compounds produced in the fermented foods tends to boost the immune response either directly or indirectly against the viral infections by modulating the lymphocytes, NK cells, macrophages and neutrophils that are capable of mediating antibody dependent cell mediated cytotoxicity. Several works have been documented to unveil novel bioactive components from fermented foods and other edible plant by products. It is possible that the varieties of biopeptides from various fermented food will continue to develop in the coming years. Further awareness of these health beneficial fermented foods and their compositions could help in improving the health and wellness of the society.</p><p id=\"Par59\"><table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Antiviral bioactives from functional and fermented food</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"2\"><p>Probiotic microbes or Fermented foods</p></th><th align=\"left\"><p>Salient Feature</p></th><th align=\"left\"><p>Mode of action</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\" colspan=\"2\"><p><italic>Bifidobacterium animalis</italic></p></td><td align=\"left\"><p>Prevent</p><p>upper respiratory tract infection</p></td><td align=\"left\"><p>Inhibit viral replication</p></td><td align=\"left\"><p>(Smith et al. <xref ref-type=\"bibr\" rid=\"CR64\">2013</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Prevent gastroenteritis COVID virus</p></td><td align=\"left\"><p>Reduces granulocyte, diminishes virus recovery</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. lactis</italic></p></td><td align=\"left\"><p>Prevent respiratory tract infection</p></td><td align=\"left\"><p>Activates plasmacytoid dendrtitic cell</p></td><td align=\"left\"><p>(Kokubo et al. <xref ref-type=\"bibr\" rid=\"CR36\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Gastroenteritis coronaviruses (TGEV)</p></td><td align=\"left\"><p>Reduces inflammation and tissue injury</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>Lactobacillus casei</italic> DN-114 001; Dan Active/Actimel</p></td><td align=\"left\"><p>Reduced incidence and duration of RTIs</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Guillemard et al. <xref ref-type=\"bibr\" rid=\"CR21\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Kefir</p></td><td align=\"left\"><p>Zika, hepatitis C, influenza, rotaviruses</p></td><td align=\"left\"><p>Enhanced macrophage synthesis, increases phagocytosis, enhanced synthesis of (CD4+), CD8 + cells, immunoglobulins, neutrophilss, and various cytokines (IL-2, IL-12, INF-γ).</p></td><td align=\"left\"><p>(Hamida et al. <xref ref-type=\"bibr\" rid=\"CR23\">2021</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Yoghurt</p></td><td align=\"left\"><p>Inhibit Enterovirus</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Fermented ginseng extracts</p></td><td align=\"left\"><p>Inhibit influenza virus H1N1, H3N2, H5N1, and H7N9 strains</p></td><td align=\"left\"><p>Viral inoculation with extract of ginseng formed better immune responses against the 2<sup>0</sup> infection with homologous and heterosubtypic virus.</p></td><td align=\"left\"><p>(Wang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2018</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Black ginseng</p></td><td align=\"left\"><p>Inhibit influenza virus</p></td><td align=\"left\"><p>Black ginseng improved the levels of GM-CSF and IL-10 at the time of infection</p></td><td align=\"left\"><p>(Kim et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Dietary xylitol</p></td><td align=\"left\"><p>Inhibit influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Yin et al. <xref ref-type=\"bibr\" rid=\"CR73\">2014</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Chongkukjang (Traditional Korean feremented food)</p></td><td align=\"left\"><p>Influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Wei et al. <xref ref-type=\"bibr\" rid=\"CR69\">2015b</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Resveratrol (From red grapes)</p></td><td align=\"left\"><p>Inhibit Epstein-Barr virus</p></td><td align=\"left\"><p>Downregulation of antiapoptotic proteins</p></td><td align=\"left\"><p>(De Leo et al. <xref ref-type=\"bibr\" rid=\"CR12\">2012</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Zingiberofficinale (Ginger)</p></td><td align=\"left\"><p>Anti-chikungunya activity</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Curcumin</p></td><td align=\"left\"><p>Inhibit Zika and chikungunya viruses</p></td><td align=\"left\"><p>Curcumin interferes with virus-cell binding.</p></td><td align=\"left\"><p>(Mounce et al. <xref ref-type=\"bibr\" rid=\"CR53\">2017</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par60\"><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Ongoing clinical trials where probiotics and dietary supplements are used against COVID-19</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>No.</p></th><th align=\"left\"><p>Clinical trial identifier</p></th><th align=\"left\"><p>Intervention</p></th><th align=\"left\"><p>Aim</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>NCT04621071</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Evaluation of the efficacy of probiotics to dcrease the duration and symptoms of COVID-19</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>NCT04458519</p></td><td align=\"left\"><p>Probiorinse</p></td><td align=\"left\"><p>Reduction of severity of COVID-19 symptoms</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>NCT04390477</p></td><td align=\"left\"><p>Probiotic</p></td><td align=\"left\"><p>Effect of probiotic on COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>NCT04366180</p></td><td align=\"left\"><p><italic>L. coryniformis</italic> K8</p></td><td align=\"left\"><p>Effect of probiotic in the incidence and prevention of COVID-19 infection in health workers</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>NCT04734886</p></td><td align=\"left\"><p><italic>L. reuteri</italic> DSM 17,938</p></td><td align=\"left\"><p>Impact of probiotic supplementation on SARS-CoV-2 specific antibody response following COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>NCT04666116</p></td><td align=\"left\"><p>Dietary supplements including probiotics</p></td><td align=\"left\"><p>To check the changes in viral load in COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>NCT04847349</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Efficacy of probiotic intervention to boost the immunity in unvaccinated people infected previously with SARS-CoV-2</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>NCT04420676</p></td><td align=\"left\"><p>Synbiotic (Omnibiotic AAD)</p></td><td align=\"left\"><p>To check the ability to reduce gastrointestinal problems in COVID-19 patients</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>NCT04813718</p></td><td align=\"left\"><p>Omni-Biotic Pro Vi 5</p></td><td align=\"left\"><p>Analysis of post-Covid syndrome</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>NCT04399252</p></td><td align=\"left\"><p><italic>L. rhamnosus</italic> GG</p></td><td align=\"left\"><p>In order to study the consequence of microbiome in COVID-19 exposed household contacts</p></td></tr></tbody></table></table-wrap></p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author's contribution</title><p>KKV, VN, AM, KKB, and MPA: Collecting articles and Writing original draft. RS: Conceptualization, Methodology, Writing - original draft. MKA, SV: Writing, Reviewing and Editing, Formal analysis. PB: Project Administration, Conceptualization and Visualization.</p></sec><sec><title>Funding</title><p>(Information that explains whether and by whom the research was supported): No funding to mention and no funding received for the particular work mentioned in the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec><p>The authors declare that the work described has not been published before (except in the form of an abstract, a published lecture or academic thesis),  it is not under consideration for publication elsewhere, its submission to JFST publication has been approved by all authors as well as the responsible authorities – tacitly or explicitly – at the institute where the work has been carried out, if accepted, it will not be published elsewhere in the same form, in English or in any other language, including electronically without the written consent of the copyright holder, and  JFST will not be held legally responsible should there be any claims for compensation or dispute on authorship.</p></sec><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Acquah</surname><given-names>C</given-names></name><name><surname>Agyei</surname><given-names>D</given-names></name><name><surname>Obeng</surname><given-names>EM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aptamers: an emerging class of bioaffinity ligands in bioactive peptide applications</article-title><source>Crit Rev Food Sci Nutr</source><year>2020</year><volume>60</volume><fpage>1195</fpage><lpage>1206</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVCms7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10408398.2018.1564234</pub-id><pub-id pub-id-type=\"pmid\">30714390</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akour</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Probiotics and COVID-19: is there any link?</article-title><source>Lett Appl Microbiol</source><year>2020</year><volume>71</volume><fpage>229</fpage><lpage>234</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1ShtLrJ</pub-id><pub-id pub-id-type=\"doi\">10.1111/lam.13334</pub-id><pub-id pub-id-type=\"pmid\">32495940</pub-id><pub-id pub-id-type=\"pmcid\">7300613</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alkhatib</surname><given-names>A</given-names></name><name><surname>Tsang</surname><given-names>C</given-names></name><name><surname>Tuomilehto</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Olive oil nutraceuticals in the prevention and management of diabetes: from molecules to lifestyle</article-title><source>Int J Mol Sci</source><year>2018</year><volume>19</volume><fpage>2021</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisFKltb7J</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms19072024</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Byun</surname><given-names>M-S</given-names></name><name><surname>Yu</surname><given-names>O-K</given-names></name><name><surname>Cha</surname><given-names>Y-S</given-names></name><name><surname>Park</surname><given-names>T-S</given-names></name></person-group><article-title xml:lang=\"en\">Korean traditional Chungkookjang improves body composition, lipid profiles and atherogenic indices in overweight/obese subjects: a double-blind, randomized, crossover, placebo-controlled clinical trial</article-title><source>Eur J Clin Nutr</source><year>2016</year><volume>70</volume><fpage>1116</fpage><lpage>1122</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtVWgu77K</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2016.77</pub-id><pub-id pub-id-type=\"pmid\">27302672</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Calder</surname><given-names>PC</given-names></name><name><surname>Kew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The immune system: a target for functional foods?</article-title><source>Br J Nutr</source><year>2002</year><volume>88</volume><fpage>S165</fpage><lpage>S176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XovFOlsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1079/BJN2002682</pub-id><pub-id pub-id-type=\"pmid\">12495459</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Carrasco-Hernandez</surname><given-names>R</given-names></name><name><surname>Jácome</surname><given-names>R</given-names></name><name><surname>López Vidal</surname><given-names>Y</given-names></name><name><surname>Ponce de León</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Are RNA viruses candidate agents for the next global pandemic? a review</article-title><source>ILAR J</source><year>2017</year><volume>58</volume><fpage>343</fpage><lpage>358</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVCmtbjK</pub-id><pub-id pub-id-type=\"doi\">10.1093/ilar/ilx026</pub-id><pub-id pub-id-type=\"pmid\">28985316</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>Q</given-names></name><name><surname>Giovannucci</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Dairy consumption and risk of type 2 diabetes: 3 cohorts of US adults and an updated meta-analysis</article-title><source>BMC Med</source><year>2014</year><volume>12</volume><fpage>215</fpage><pub-id pub-id-type=\"doi\">10.1186/s12916-014-0215-1</pub-id><pub-id pub-id-type=\"pmid\">25420418</pub-id><pub-id pub-id-type=\"pmcid\">4243376</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>X</given-names></name><name><surname>Wei</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Isolation, purification and the anti-hypertensive effect of a novel angiotensin I-converting enzyme (ACE) inhibitory peptide from <italic>Ruditapes philippinarum</italic> fermented with <italic>Bacillus natto</italic></article-title><source>Food Funct</source><year>2018</year><volume>9</volume><fpage>5230</fpage><lpage>5237</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFOgsLnL</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8FO01146J</pub-id><pub-id pub-id-type=\"pmid\">30206615</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chiba</surname><given-names>E</given-names></name><name><surname>Tomosada</surname><given-names>Y</given-names></name><name><surname>Vizoso-Pinto</surname><given-names>MG</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Immunobiotic Lactobacillus rhamnosus improves resistance of infant mice against respiratory syncytial virus infection</article-title><source>Int Immunopharmacol</source><year>2013</year><volume>17</volume><fpage>373</fpage><lpage>382</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtlGqtrbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.intimp.2013.06.024</pub-id><pub-id pub-id-type=\"pmid\">23838113</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choi</surname><given-names>H-J</given-names></name><name><surname>Song</surname><given-names>J-H</given-names></name><name><surname>Park</surname><given-names>K-S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of yogurt against enterovirus 71 in vero cells</article-title><source>Food Sci Biotechnol</source><year>2010</year><volume>19</volume><fpage>289</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGisLzE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-010-0042-x</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Davidson</surname><given-names>LE</given-names></name><name><surname>Fiorino</surname><given-names>A-M</given-names></name><name><surname>Snydman</surname><given-names>DR</given-names></name><name><surname>Hibberd</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Lactobacillus GG as an immune adjuvant for live-attenuated influenza vaccine in healthy adults: a randomized double-blind placebo-controlled trial</article-title><source>Eur J Clin Nutr</source><year>2011</year><volume>65</volume><fpage>501</fpage><lpage>507</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXkt1Wlt7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2010.289</pub-id><pub-id pub-id-type=\"pmid\">21285968</pub-id><pub-id pub-id-type=\"pmcid\">3071884</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Leo</surname><given-names>A</given-names></name><name><surname>Arena</surname><given-names>G</given-names></name><name><surname>Lacanna</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Resveratrol inhibits Epstein Barr virus lytic cycle in Burkitt’s lymphoma cells by affecting multiple molecular targets</article-title><source>Antiviral Res</source><year>2012</year><volume>96</volume><fpage>196</fpage><lpage>202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhs12gtLfE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2012.09.003</pub-id><pub-id pub-id-type=\"pmid\">22985630</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>García-Burgos</surname><given-names>M</given-names></name><name><surname>Moreno-Fernández</surname><given-names>J</given-names></name><name><surname>Alférez</surname><given-names>MJM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">New perspectives in fermented dairy products and their health relevance</article-title><source>J Funct Foods</source><year>2020</year><volume>72</volume><fpage>104059</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylu7vM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jff.2020.104059</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ghanem</surname><given-names>KZ</given-names></name><name><surname>Mahran</surname><given-names>MZ</given-names></name><name><surname>Ramadan</surname><given-names>MM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">A comparative study on flavour components and therapeutic properties of unfermented and fermented defatted soybean meal extract</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>5998</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSlur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-62907-x</pub-id><pub-id pub-id-type=\"pmid\">32265465</pub-id><pub-id pub-id-type=\"pmcid\">7138831</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Prasad</surname><given-names>J</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of natural and acquired immunity by Lactobacillus rhamnosus (HN001), Lactobacillus acidophilus (HN017) and Bifidobacterium lactis (HN019)</article-title><source>Br J Nutr</source><year>2000</year><volume>83</volume><fpage>167</fpage><lpage>176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXhsVymur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114500000210</pub-id><pub-id pub-id-type=\"pmid\">10743496</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Cross</surname><given-names>ML</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of immunity in the elderly by dietary supplementation with the probiotic Bifidobacterium lactis HN019</article-title><source>Am J Clin Nutr</source><year>2001</year><volume>74</volume><fpage>833</fpage><lpage>839</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXovVSgtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1093/ajcn/74.6.833</pub-id><pub-id pub-id-type=\"pmid\">11722966</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grandy</surname><given-names>G</given-names></name><name><surname>Medina</surname><given-names>M</given-names></name><name><surname>Soria</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in the treatment of acute rotavirus diarrhoea. a randomized, double-blind, controlled trial using two different probiotic preparations in Bolivian children</article-title><source>BMC Infect Dis</source><year>2010</year><volume>10</volume><fpage>253</fpage><pub-id pub-id-type=\"doi\">10.1186/1471-2334-10-253</pub-id><pub-id pub-id-type=\"pmid\">20735858</pub-id><pub-id pub-id-type=\"pmcid\">2940902</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grant</surname><given-names>W</given-names></name><name><surname>Lahore</surname><given-names>H</given-names></name><name><surname>McDonnell</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Evidence that vitamin D supplementation could reduce risk of influenza and COVID-19 infections and deaths</article-title><source>Nutrients</source><year>2020</year><volume>12</volume><fpage>988</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVemsbnK</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu12040988</pub-id><pub-id pub-id-type=\"pmid\">32252338</pub-id><pub-id pub-id-type=\"pmcid\">7231123</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guan</surname><given-names>W</given-names></name><name><surname>Ni</surname><given-names>Z</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical characteristics of coronavirus disease 2019 in China</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1708</fpage><lpage>1720</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSjsb8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMoa2002032</pub-id><pub-id pub-id-type=\"pmid\">32109013</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gui</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>A</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effects of probiotic supplementation on natural killer cell function in healthy elderly individuals: a meta-analysis of randomized controlled trials</article-title><source>Eur J Clin Nutr</source><year>2020</year><volume>74</volume><fpage>1630</fpage><lpage>1637</lpage><pub-id pub-id-type=\"doi\">10.1038/s41430-020-0670-z</pub-id><pub-id pub-id-type=\"pmid\">32514029</pub-id><pub-id pub-id-type=\"pmcid\">7279433</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guillemard</surname><given-names>E</given-names></name><name><surname>Tondu</surname><given-names>F</given-names></name><name><surname>Lacoin</surname><given-names>F</given-names></name><name><surname>Schrezenmeir</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Consumption of a fermented dairy product containing the probiotic Lactobacillus casei DN-114 001 reduces the duration of respiratory infections in the elderly in a randomised controlled trial</article-title><source>Br J Nutr</source><year>2010</year><volume>103</volume><fpage>58</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhs1Smt73N</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114509991395</pub-id><pub-id pub-id-type=\"pmid\">19747410</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z-D</given-names></name><name><surname>Wang</surname><given-names>Z-Y</given-names></name><name><surname>Zhang</surname><given-names>S-F</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface distribution of severe acute respiratory syndrome coronavirus 2 in hospital wards, Wuhan, China, 2020</article-title><source>Emerg Infect Dis</source><year>2020</year><volume>26</volume><fpage>1583</fpage><lpage>1591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlOlsbrF</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid2607.200885</pub-id><pub-id pub-id-type=\"pmid\">32275497</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hamida</surname><given-names>RS</given-names></name><name><surname>Shami</surname><given-names>A</given-names></name><name><surname>Ali</surname><given-names>MA</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Kefir: a protective dietary supplementation against viral infection</article-title><source>Biomed Pharmacother</source><year>2021</year><volume>133</volume><fpage>110974</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlynsrbL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biopha.2020.110974</pub-id><pub-id pub-id-type=\"pmid\">33186795</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"other\">Hayes M, García-Vaquero M (2016) Bioactive Compounds from Fermented Food Products. pp&amp;nbsp;293–310</mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Horowitz</surname><given-names>RI</given-names></name><name><surname>Freeman</surname><given-names>PR</given-names></name><name><surname>Bruzzese</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Efficacy of glutathione therapy in relieving dyspnea associated with COVID-19 pneumonia: a report of 2 cases</article-title><source>Respiratory Med Case Rep</source><year>2020</year><volume>30</volume><fpage>101063</fpage><pub-id pub-id-type=\"doi\">10.1016/j.rmcr.2020.101063</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Lin</surname><given-names>W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Review article: probiotics, prebiotics and dietary approaches during COVID-19 pandemic</article-title><source>Trends Food Sci Technol</source><year>2021</year><volume>108</volume><fpage>187</fpage><lpage>196</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVyrsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2020.12.009</pub-id><pub-id pub-id-type=\"pmid\">33519087</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ivec</surname><given-names>M</given-names></name><name><surname>Botić</surname><given-names>T</given-names></name><name><surname>Koren</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Interactions of macrophages with probiotic bacteria lead to increased antiviral response against vesicular stomatitis virus</article-title><source>Antiviral Res</source><year>2007</year><volume>75</volume><fpage>266</fpage><lpage>274</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXmsFKht7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2007.03.013</pub-id><pub-id pub-id-type=\"pmid\">17512614</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ji</surname><given-names>L-N</given-names></name><name><surname>Chao</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>Y-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical features of pediatric patients with COVID-19: a report of two family cluster cases</article-title><source>World J Pediatr</source><year>2020</year><volume>16</volume><fpage>267</fpage><lpage>270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXltVSisbc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12519-020-00356-2</pub-id><pub-id pub-id-type=\"pmid\">32180140</pub-id><pub-id pub-id-type=\"pmcid\">7091281</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Lee</surname><given-names>Y-T</given-names></name><name><surname>Ngo</surname><given-names>V</given-names></name><name><surname>Le</surname></name><etal/></person-group><article-title xml:lang=\"en\">Heat-killed Lactobacillus casei confers broad protection against influenza A virus primary infection and develops heterosubtypic immunity against future secondary infection</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><fpage>17360</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGitLbF</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-017-17487-8</pub-id><pub-id pub-id-type=\"pmid\">29234060</pub-id><pub-id pub-id-type=\"pmcid\">5727132</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kanmani</surname><given-names>P</given-names></name><name><surname>Albarracin</surname><given-names>L</given-names></name><name><surname>Kobayashi</surname><given-names>H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">AGenomic characterization of Lactobacillus delbrueckii TUA4408L and evaluation of the antiviral activities of its extracellular polysaccharides in porcine intestinal epithelial cells</article-title><source>Front Immunol</source><year>2018</year><volume>9</volume><fpage>27</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXnslyrsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2018.02178</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>S</given-names></name><name><surname>Jangra</surname><given-names>G</given-names></name><name><surname>Kundu</surname><given-names>V</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Anti-viral activity of Zingiber officinale (Ginger) ingredients against the Chikungunya virus</article-title><source>VirusDisease</source><year>2020</year><volume>31</volume><fpage>270</fpage><lpage>276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38Xit1yqurjO</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13337-020-00584-0</pub-id><pub-id pub-id-type=\"pmid\">32420412</pub-id><pub-id pub-id-type=\"pmcid\">7223110</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalil</surname><given-names>E</given-names></name><name><surname>Abd Manap</surname><given-names>M</given-names></name><name><surname>Mustafa</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotic properties of exopolysaccharide-producing lactobacillus strains isolated from tempoyak</article-title><source>Molecules</source><year>2018</year><volume>23</volume><fpage>398</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhs1Kjt7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules23020398</pub-id><pub-id pub-id-type=\"pmid\">29438288</pub-id><pub-id pub-id-type=\"pmcid\">6017292</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>E-H</given-names></name><name><surname>Kim</surname><given-names>S-W</given-names></name><name><surname>Park</surname><given-names>S-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Greater efficacy of black ginseng (CJ EnerG) over red ginseng against lethal Influenza A virus infection</article-title><source>Nutrients</source><year>2019</year><volume>11</volume><fpage>1879</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVGmtbjP</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu11081879</pub-id><pub-id pub-id-type=\"pmid\">31412594</pub-id><pub-id pub-id-type=\"pmcid\">6723933</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>Y-I</given-names></name><name><surname>Kim</surname><given-names>S-G</given-names></name><name><surname>Kim</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Infection and rapid transmission of SARS-CoV-2 in Ferrets</article-title><source>Cell Host Microbe</source><year>2020</year><volume>27</volume><fpage>704</fpage><lpage>709e2</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsl2qsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chom.2020.03.023</pub-id><pub-id pub-id-type=\"pmid\">32259477</pub-id><pub-id pub-id-type=\"pmcid\">7144857</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kinoshita</surname><given-names>T</given-names></name><name><surname>Maruyama</surname><given-names>K</given-names></name><name><surname>Suyama</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of OLL1073R-1 yogurt intake on influenza incidence and immunological markers among women healthcare workers: a randomized controlled trial</article-title><source>Food Funct</source><year>2019</year><volume>10</volume><fpage>8129</fpage><lpage>8136</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVaksLrI</pub-id><pub-id pub-id-type=\"doi\">10.1039/C9FO02128K</pub-id><pub-id pub-id-type=\"pmid\">31738351</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kokubo</surname><given-names>T</given-names></name><name><surname>Komano</surname><given-names>Y</given-names></name><name><surname>Tsuji</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of plasmacytoid dendritic cell-Stimulative lactic acid bacteria, lactococcus lactis strain plasma, on exercise-Induced fatigue and recovery via immunomodulatory action</article-title><source>Int J Sport Nutr Exerc Metab</source><year>2019</year><volume>25</volume><fpage>1</fpage><lpage>5</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmt1art74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1123/ijsnem.2018-0377</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kurian</surname><given-names>SJ</given-names></name><name><surname>Unnikrishnan</surname><given-names>MK</given-names></name><name><surname>Miraj</surname><given-names>SS</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in prevention and treatment of COVID-19: current perspective and future prospects</article-title><source>Arch Med Res</source><year>2021</year><volume>52</volume><fpage>582</fpage><lpage>594</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFehu74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.arcmed.2021.03.002</pub-id><pub-id pub-id-type=\"pmid\">33785208</pub-id><pub-id pub-id-type=\"pmcid\">7972717</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kwak</surname><given-names>M-K</given-names></name><name><surname>Liu</surname><given-names>R</given-names></name><name><surname>Kwon</surname><given-names>J-O</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Cyclic dipeptides from lactic acid bacteria inhibit proliferation of the influenza a virus</article-title><source>J Microbiol</source><year>2013</year><volume>51</volume><fpage>836</fpage><lpage>843</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXksVajsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12275-013-3521-y</pub-id><pub-id pub-id-type=\"pmid\">24385362</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Laatikainen</surname><given-names>R</given-names></name><name><surname>Koskenpato</surname><given-names>J</given-names></name><name><surname>Hongisto</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Randomised clinical trial: low-FODMAP rye bread vs. regular rye bread to relieve the symptoms of irritable bowel syndrome</article-title><source>Aliment Pharmacol Ther</source><year>2016</year><volume>44</volume><fpage>460</fpage><lpage>470</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KktbjL</pub-id><pub-id pub-id-type=\"doi\">10.1111/apt.13726</pub-id><pub-id pub-id-type=\"pmid\">27417338</pub-id><pub-id pub-id-type=\"pmcid\">5113694</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>C-C</given-names></name><name><surname>Shih</surname><given-names>T-P</given-names></name><name><surname>Ko</surname><given-names>W-C</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronavirus disease-2019 (COVID-19): The epidemic and the challenges</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><fpage>105924</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ogtr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105924</pub-id><pub-id pub-id-type=\"pmid\">32081636</pub-id><pub-id pub-id-type=\"pmcid\">7127800</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lam</surname><given-names>TT-Y</given-names></name><name><surname>Jia</surname><given-names>N</given-names></name><name><surname>Zhang</surname><given-names>Y-W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Identifying SARS-CoV-2-related coronaviruses in Malayan pangolins</article-title><source>Nature</source><year>2020</year><volume>583</volume><fpage>282</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCnu77L</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-020-2169-0</pub-id><pub-id pub-id-type=\"pmid\">32218527</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>MH</given-names></name><name><surname>Yoo</surname><given-names>S-H</given-names></name><name><surname>Ha</surname><given-names>S-D</given-names></name><name><surname>Choi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Inactivation of feline calicivirus and murine norovirus during Dongchimi fermentation</article-title><source>Food Microbiol</source><year>2012</year><volume>31</volume><fpage>210</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFKgsLw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fm.2012.04.002</pub-id><pub-id pub-id-type=\"pmid\">22608225</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>YJ</given-names></name><name><surname>Yoo</surname><given-names>HJ</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Consumption of dairy yogurt containing Lactobacillus paracasei ssp. paracasei, Bifidobacterium animalis ssp. lactis and heat-treated Lactobacillus plantarum improves immune function including natural killer cell activity</article-title><source>Nutrients</source><year>2017</year><volume>9</volume><fpage>558</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVersbjI</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu9060558</pub-id><pub-id pub-id-type=\"pmid\">28561762</pub-id><pub-id pub-id-type=\"pmcid\">5490537</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Whon</surname><given-names>TW</given-names></name><name><surname>Roh</surname><given-names>SW</given-names></name><name><surname>Jeon</surname><given-names>CO</given-names></name></person-group><article-title xml:lang=\"en\">Unraveling microbial fermentation features in kimchi: from classical to meta-omics approaches</article-title><source>Appl Microbiol Biotechnol</source><year>2020</year><volume>104</volume><fpage>7731</fpage><lpage>7744</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsFejurbJ</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00253-020-10804-8</pub-id><pub-id pub-id-type=\"pmid\">32749526</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Gayle</surname><given-names>AA</given-names></name><name><surname>Wilder-Smith</surname><given-names>A</given-names></name><name><surname>Rocklöv</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The reproductive number of COVID-19 is higher compared to SARS coronavirus</article-title><source>J Travel Med</source><year>2020</year><volume>27</volume><fpage>98</fpage><lpage>1006</lpage><pub-id pub-id-type=\"doi\">10.1093/jtm/taaa021</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Ikegami</surname><given-names>S</given-names></name><name><surname>Kume</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Reducing the risk of infection in the elderly by dietary intake of yoghurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>Br J Nutr</source><year>2010</year><volume>104</volume><fpage>998</fpage><lpage>1006</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1GjsLfO</pub-id><pub-id pub-id-type=\"doi\">10.1017/S000711451000173X</pub-id><pub-id pub-id-type=\"pmid\">20487575</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Sato</surname><given-names>A</given-names></name><name><surname>Goto</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Enhanced natural killer cell activation by exopolysaccharides derived from yogurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>J Dairy Sci</source><year>2016</year><volume>99</volume><fpage>915</fpage><lpage>923</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXitVSksLbE</pub-id><pub-id pub-id-type=\"doi\">10.3168/jds.2015-10376</pub-id><pub-id pub-id-type=\"pmid\">26686726</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marco</surname><given-names>ML</given-names></name><name><surname>Heeney</surname><given-names>D</given-names></name><name><surname>Binda</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Health benefits of fermented foods: microbiota and beyond</article-title><source>Curr Opin Biotechnol</source><year>2017</year><volume>44</volume><fpage>94</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhvFKjs7zF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.copbio.2016.11.010</pub-id><pub-id pub-id-type=\"pmid\">27998788</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McGurnaghan</surname><given-names>SJ</given-names></name><name><surname>Weir</surname><given-names>A</given-names></name><name><surname>Bishop</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Risks of and risk factors for COVID-19 disease in people with diabetes: a cohort study of the total population of Scotland</article-title><source>Lancet Diabetes Endocrinol</source><year>2021</year><volume>9</volume><fpage>82</fpage><lpage>93</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXks1aqsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-8587(20)30405-8</pub-id><pub-id pub-id-type=\"pmid\">33357491</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mirashrafi</surname><given-names>S</given-names></name><name><surname>Moravejolahkami</surname><given-names>AR</given-names></name><name><surname>Balouch Zehi</surname><given-names>Z</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The efficacy of probiotics on virus titres and antibody production in virus diseases: a systematic review on recent evidence for COVID-19 treatment</article-title><source>Clin Nutr ESPEN</source><year>2021</year><volume>46</volume><fpage>1</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.1016/j.clnesp.2021.10.016</pub-id><pub-id pub-id-type=\"pmid\">34857182</pub-id><pub-id pub-id-type=\"pmcid\">8539817</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mora</surname><given-names>L</given-names></name><name><surname>Escudero</surname><given-names>E</given-names></name><name><surname>Aristoy</surname><given-names>M-C</given-names></name><name><surname>Toldrá</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">A peptidomic approach to study the contribution of added casein proteins to the peptide profile in Spanish dry-fermented sausages</article-title><source>Int J Food Microbiol</source><year>2015</year><volume>212</volume><fpage>41</fpage><lpage>48</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtVKhtbjM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijfoodmicro.2015.05.022</pub-id><pub-id pub-id-type=\"pmid\">26116420</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morita</surname><given-names>N</given-names></name><name><surname>Umemoto</surname><given-names>E</given-names></name><name><surname>Fujita</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">GPR31-dependent dendrite protrusion of intestinal CX3CR1 + cells by bacterial metabolites</article-title><source>Nature</source><year>2019</year><volume>566</volume><fpage>110</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmtVyjtL0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-019-0884-1</pub-id><pub-id pub-id-type=\"pmid\">30675063</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mounce</surname><given-names>BC</given-names></name><name><surname>Cesaro</surname><given-names>T</given-names></name><name><surname>Carrau</surname><given-names>L</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Curcumin inhibits Zika and chikungunya virus infection by inhibiting cell binding</article-title><source>Antiviral Res</source><year>2017</year><volume>142</volume><fpage>148</fpage><lpage>157</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsV2jtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2017.03.014</pub-id><pub-id pub-id-type=\"pmid\">28343845</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muhialdin</surname><given-names>BJ</given-names></name><name><surname>Zawawi</surname><given-names>N</given-names></name><name><surname>Abdull Razis</surname><given-names>AF</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented foods and their probiotics bacteria towards respiratory and alimentary tracts viruses</article-title><source>Food Control</source><year>2021</year><volume>127</volume><fpage>108140</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVSmtLfF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2021.108140</pub-id><pub-id pub-id-type=\"pmid\">33867696</pub-id><pub-id pub-id-type=\"pmcid\">8036130</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olaimat</surname><given-names>AN</given-names></name><name><surname>Aolymat</surname><given-names>I</given-names></name><name><surname>Al-Holy</surname><given-names>M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The potential application of probiotics and prebiotics for the prevention and treatment of COVID-19</article-title><source>npj Sci Food</source><year>2020</year><volume>4</volume><fpage>17</fpage><pub-id pub-id-type=\"doi\">10.1038/s41538-020-00078-9</pub-id><pub-id pub-id-type=\"pmid\">33083549</pub-id><pub-id pub-id-type=\"pmcid\">7536434</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivares</surname><given-names>M</given-names></name><name><surname>Díaz-Ropero</surname><given-names>MP</given-names></name><name><surname>Sierra</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Oral intake of Lactobacillus fermentum CECT5716 enhances the effects of influenza vaccination</article-title><source>Nutrition</source><year>2007</year><volume>23</volume><fpage>254</fpage><lpage>260</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXislWgurg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nut.2007.01.004</pub-id><pub-id pub-id-type=\"pmid\">17352961</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ong</surname><given-names>SWX</given-names></name><name><surname>Tan</surname><given-names>YK</given-names></name><name><surname>Chia</surname><given-names>PY</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Air, surface environmental, and personal protective equipment contamination by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from a symptomatic patient</article-title><source>JAMA</source><year>2020</year><volume>323</volume><fpage>1610</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSrtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.3227</pub-id><pub-id pub-id-type=\"pmid\">32129805</pub-id><pub-id pub-id-type=\"pmcid\">7057172</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parada Venegas</surname><given-names>D</given-names></name><name><surname>De la Fuente</surname><given-names>MK</given-names></name><name><surname>Landskron</surname><given-names>G</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Short chain fatty acids (SCFAs)-mediated gut epithelial and immune regulation and its relevance for inflammatory bowel diseases</article-title><source>Front Immunol</source><year>2019</year><volume>10</volume><fpage>107</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVWrtL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2019.00277</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peters</surname><given-names>A</given-names></name><name><surname>Krumbholz</surname><given-names>P</given-names></name><name><surname>Jäger</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Metabolites of lactic acid bacteria present in fermented foods are highly potent agonists of human hydroxycarboxylic acid receptor 3</article-title><source>PLOS Genet</source><year>2019</year><volume>15</volume><fpage>e1008145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhvFaqsb%2FN</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pgen.1008145</pub-id><pub-id pub-id-type=\"pmid\">31120900</pub-id><pub-id pub-id-type=\"pmcid\">6532841</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>przeLe</surname><given-names>TT</given-names></name><name><surname>Cramer</surname><given-names>JP</given-names></name><name><surname>Chen</surname><given-names>R</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Evolution of the COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><fpage>667</fpage><lpage>668</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVylsb3M</pub-id><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00151-8</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pyankov</surname><given-names>OV</given-names></name><name><surname>Bodnev</surname><given-names>SA</given-names></name><name><surname>Pyankova</surname><given-names>OG</given-names></name><name><surname>Agranovski</surname><given-names>IE</given-names></name></person-group><article-title xml:lang=\"en\">Survival of aerosolized coronavirus in the ambient air</article-title><source>J Aerosol Sci</source><year>2018</year><volume>115</volume><fpage>158</fpage><lpage>163</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVyit7jN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jaerosci.2017.09.009</pub-id><pub-id pub-id-type=\"pmid\">32226116</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Raji</surname><given-names>MNA</given-names></name><name><surname>Ab Karim</surname><given-names>S</given-names></name><name><surname>Ishak</surname><given-names>FAC</given-names></name><name><surname>Arshad</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Past and present practices of the Malay food heritage and culture in Malaysia</article-title><source>J Ethnic Foods</source><year>2017</year><volume>4</volume><fpage>221</fpage><lpage>231</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jef.2017.11.001</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Santarpia</surname><given-names>JL</given-names></name><name><surname>Rivera</surname><given-names>DN</given-names></name><name><surname>Herrera</surname><given-names>VL</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface contamination of SARS-CoV-2 observed in quarantine and isolation care</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>12732</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVyku7rM</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-69286-3</pub-id><pub-id pub-id-type=\"pmid\">32728118</pub-id><pub-id pub-id-type=\"pmcid\">7391640</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Smith</surname><given-names>TJ</given-names></name><name><surname>Rigassio-Radler</surname><given-names>D</given-names></name><name><surname>Denmark</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effect of Lactobacillus rhamnosus LGG ® and Bifidobacterium animalis ssp. lactis BB-12 ® on health-related quality of life in college students affected by upper respiratory infections</article-title><source>Br J Nutr</source><year>2013</year><volume>109</volume><fpage>1999</fpage><lpage>2007</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXnvVGhsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114512004138</pub-id><pub-id pub-id-type=\"pmid\">23020819</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"other\">Takeda K, Okumura K (2007) Effects of a Fermented Milk Drink Containing Lactobacillus casei Strain Shirota on the Human NK-Cell Activity. J Nutr 137. <ext-link ext-link-type=\"doi\" xlink:href=\"10.1093/jn/137.3.791S\">https://doi.org/10.1093/jn/137.3.791S</ext-link>. :791S-793S</mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tamang</surname><given-names>JP</given-names></name><name><surname>Cotter</surname><given-names>PD</given-names></name><name><surname>Endo</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Fermented foods in a global age: east meets West</article-title><source>Compr Rev Food Sci Food Saf</source><year>2020</year><volume>19</volume><fpage>184</fpage><lpage>217</lpage><pub-id pub-id-type=\"doi\">10.1111/1541-4337.12520</pub-id><pub-id pub-id-type=\"pmid\">33319517</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>Bushmaker</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>DH</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface stability of SARS-CoV-2 as compared with SARS-CoV-1</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1564</fpage><lpage>1567</lpage><pub-id pub-id-type=\"doi\">10.1056/NEJMc2004973</pub-id><pub-id pub-id-type=\"pmid\">32182409</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Kim</surname><given-names>K-H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented ginseng extracts against a broad range of Influenza viruses</article-title><source>Viruses</source><year>2018</year><volume>10</volume><fpage>471</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXntF2gsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/v10090471</pub-id><pub-id pub-id-type=\"pmid\">30200514</pub-id><pub-id pub-id-type=\"pmcid\">6164659</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wei</surname><given-names>L</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>H</given-names></name><name><surname>Wen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Engineering Scheffersomyces stipitis for fumaric acid production from xylose</article-title><source>Bioresour Technol</source><year>2015</year><volume>187</volume><fpage>246</fpage><lpage>254</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXlvVems74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2015.03.122</pub-id><pub-id pub-id-type=\"pmid\">25863201</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wiersinga</surname><given-names>WJ</given-names></name><name><surname>Rhodes</surname><given-names>A</given-names></name><name><surname>Cheng</surname><given-names>AC</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Pathophysiology, transmission, diagnosis, and treatment of coronavirus disease 2019 (COVID-19)</article-title><source>JAMA</source><year>2020</year><volume>324</volume><fpage>782</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslamu73N</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.12839</pub-id><pub-id pub-id-type=\"pmid\">32648899</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>N</given-names></name><name><surname>Qiao</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Application of metabolic controls for the maximization of lipid production in semicontinuous fermentation</article-title><source>Proc Natl Acad Sci USA</source><year>2017</year><volume>114</volume><fpage>E5308</fpage><lpage>E5316</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWmtLvI</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1703321114</pub-id><pub-id pub-id-type=\"pmid\">28630295</pub-id><pub-id pub-id-type=\"pmcid\">5502627</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral effects of a probiotic metabolic products against transmissible gastroenteritis coronavirus</article-title><source>J Probiotics Heal</source><year>2017</year><volume>5</volume><fpage>543</fpage><pub-id pub-id-type=\"doi\">10.4172/2329-8901.1000184</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yin</surname><given-names>SY</given-names></name><name><surname>Kim</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>H-J</given-names></name></person-group><article-title xml:lang=\"en\">Protective effect of dietary xylitol on Influenza A virus infection</article-title><source>PLoS One</source><year>2014</year><volume>9</volume><fpage>e84633</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXltlOit78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0084633</pub-id><pub-id pub-id-type=\"pmid\">24392148</pub-id><pub-id pub-id-type=\"pmcid\">3879333</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec14\"><title>Supplementary Information</title><p id=\"Par35\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/13197_2022_5528_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 556 kb)</p></caption></media></supplementary-material></p></sec></app></app-group><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-023-01258-5</article-id><article-id pub-id-type=\"manuscript\">1258</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-023-01258-5</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Detection method for genetically modified potato using an ultra-fast PCR system</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-0608-6354</contrib-id><name><surname>Shin</surname><given-names>Min Ki</given-names></name><address><email>mkhj1212@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs10068023012585_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jeon</surname><given-names>Seon Min</given-names></name><address><email>jsm0065@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Koo</surname><given-names>Yong Eui</given-names></name><address><email>kye2020@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.31501.36</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 5905</institution-id><institution content-type=\"org-division\">Interdisciplinary Program in Bioengineering</institution><institution content-type=\"org-name\">Seoul National University</institution></institution-wrap><addr-line content-type=\"postcode\">08826</addr-line><addr-line content-type=\"city\">Seoul</addr-line><country country=\"KR\">Republic of Korea</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.467691.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1773 0675</institution-id><institution content-type=\"org-division\">Food Safety Risk Assessment Department</institution><institution content-type=\"org-name\">National Institute of Food and Drug Safety Evaluation</institution></institution-wrap><addr-line content-type=\"postcode\">28159</addr-line><addr-line content-type=\"city\">Cheongju</addr-line><country country=\"KR\">Republic of Korea</country></aff></contrib-group><author-notes><corresp id=\"IDs10068023012585_cor1\"><label>a</label><email>mkhj1212@korea.kr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>1</day><month>2</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"4\">9</issue><fpage>1185</fpage><lpage>1191</lpage><history><date date-type=\"registration\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"received\"><day>24</day><month>10</month><year>2022</year></date><date date-type=\"rev-recd\"><day>27</day><month>12</month><year>2022</year></date><date date-type=\"accepted\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"online\"><day>1</day><month>2</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Genetically modified (GM) potatoes having resistance to insects and viral diseases, low reducing sugar contents, and black spots for high quality continue to be developed. However, no GM potato has been approved as food or feed in the Republic of Korea as the country adheres to a zero-tolerance policy to unauthorized genetically modified organisms (GMOs). When the self-sufficiency rate is low, a detection method to assess GMOs in crops or other products is necessary. Therefore, a rapid method for two GM potato events (SPS-Y9 and EH92-527-1) using an ultra-fast PCR (UF-PCR) system has been developed, and its specificity, sensitivity, and applicability were demonstrated. UF-PCR can decrease the runtime of PCR by more than half of that needed in conventional methods. However, UF-PCR is not a common method for GMO analysis. This rapid detection method may be useful for GMO analyses in field conditions.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Solanum tuberosum</italic></kwd><kwd>Event specific qualification</kwd><kwd>GM potato</kwd><kwd>UF-PCR</kwd><kwd>Monitoring</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution>National Institute of Food and Drug Safety Evaluation</institution><institution-id institution-id-type=\"doi\" vocab=\"open-funder-registry\">http://dx.doi.org/10.13039/501100003655</institution-id></institution-wrap></funding-source><award-id award-type=\"FundRef grant\">19161MFDS051</award-id><principal-award-recipient><name><surname>Koo</surname><given-names>Yong Eui</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>10</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2023_Article_1258.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Potato (<italic>Solanum tuberosum</italic>) is widely used as a food and snack ingredient and is one of the four major food crops, along with rice, wheat, and corn. As of 2017, approximately 390 million ton of potatoes had been produced worldwide in an area of ​​19.3 million hectares. The area for cultivation decreased by 3% compared to that in 2000; however, production increased by 20.3% during the same period owing to the increase in yield (FAOSTAT, <xref ref-type=\"bibr\" rid=\"CR8\">2019</xref>). The productivity is steadily improving with the development of potato varieties and production technology.</p><p id=\"Par3\">Genetically modified (GM) crops also contribute to the increase in productivity. Although commercialization has ceased, the first GM potato, the NewLeaf potato having a pest-resistance trait, was approved as food in 1995 (Jaccaud et al., <xref ref-type=\"bibr\" rid=\"CR13\">2003</xref>). Subsequently, various traits, such as resistance against insects, diseases, and modified product qualities, have been introduced into potato (Martinez-Prada et al., <xref ref-type=\"bibr\" rid=\"CR17\">2021</xref>). In 2022, according to the GM Approval Database (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.isaaa.org/gmapprovaldatabase/default.asp\">www.isaaa.org/gmapprovaldatabase/default.asp</ext-link>) of the International Service for the Acquisition of Agri-biotech Applications, fifty-one GM potato events have been developed and commercialized.</p><p id=\"Par4\">Potato production in Republic of Korea was approximately 0.63 million ton, importing 0.24 million ton in 2019, and it has continued to increase since 2010 (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/</ext-link>). Under these circumstances, the urge for use of GM potato is increasing; however, no GM potato varieties are currently approved as food and feed in Republic of Korea (MFDS, <xref ref-type=\"bibr\" rid=\"CR20\">2022</xref>).</p><p id=\"Par5\">In several countries, unauthorized genetically modified organisms (GMOs) are managed under zero-tolerance or low level presence polices (Fraiture et al., <xref ref-type=\"bibr\" rid=\"CR9\">2016</xref>), which necessitates the development of rapid and accurate detection methods to manage unauthorized GMOs.</p><p id=\"Par6\">Several studies have been conducted to develop methods to identify GM potatoes. Detection methods for NewLeaf, NewLeaf Y, NewLeaf Plus, PH05-026-0048, EH92-527-1, AM04-1020, SPS-E12, AV43-6-G7, and insertion <italic>Cry3A</italic> have been developed using conventional PCR (PCR), real-time PCR (qPCR) (Cote et al., <xref ref-type=\"bibr\" rid=\"CR5\">2005</xref>; Roh et al., <xref ref-type=\"bibr\" rid=\"CR22\">2004</xref>; Song et al., <xref ref-type=\"bibr\" rid=\"CR25\">2017</xref>; Yoo et al., <xref ref-type=\"bibr\" rid=\"CR29\">2013</xref>), loop-mediated isothermal amplification (Tu et al., <xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>), and droplet digital PCR (Gao et al., <xref ref-type=\"bibr\" rid=\"CR10\">2016</xref>). Recent gene detection assays are directed towards simpler and more cost-effective methods.</p><p id=\"Par7\">Most GMO detection methods using PCR or qPCR take approximately 40 min to 2 h or more including data analysis. However, the ultra-fast PCR (UF-PCR) system can decrease the run time of PCR. In addition, the use of UF-PCR system can reduce reagent usage by half. Owing to these advantages, the UF-PCR system is being considered for application in various analyses, such as forensic genotyping (Aboud et al., <xref ref-type=\"bibr\" rid=\"CR1\">2013</xref>), pathogen screening (Das et al., <xref ref-type=\"bibr\" rid=\"CR6\">2022</xref>; Kim et al., <xref ref-type=\"bibr\" rid=\"CR15\">2018</xref>; Lee et al., <xref ref-type=\"bibr\" rid=\"CR16\">2014</xref>), allergen analysis (Kim et al., <xref ref-type=\"bibr\" rid=\"CR14\">2019</xref>), detection of epidemic viruses like SARS CoV-2 (Chen et al., <xref ref-type=\"bibr\" rid=\"CR4\">2022</xref>; Milosevic et al., <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>), and authenticity determination (Hong et al., <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par8\">However, little research has been conducted in the field of GMO detection using the UF-PCR system (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). Here, we propose a reliable and rapid detection method for GM potatoes.</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Samples</title><p id=\"Par9\">Reference materials of GM soybean (A2704-12, A5547-127, CV127, DAS-44406-6, DAS-68416-4, DAS-81419-2, DP-305423-1, DP-356043-5, FG72, GMB151, RRS, MON87701, MON87705, MON87708, MON87751, MON87769, MON89788, and SYHT0H2), GM maize (3272, 5307, Bt10, Bt11, Bt176, DAS-40278-9, DAS-59122-7, DLL25, DP-004114-3, DP-023211-2, DP-098140-6, DP-202216-6, DP-915635-4, GA21, MIR162, MIR604, MZHG0JG, MZIR098, NK603, T25, TC1507, MON810, MON863, MON87403, MON87411, MON87419, MON87427, MON87429, MON87460, MON88017, MON89034, MON95379, and VCO-01981-5), GM canola (DP-073496-4, GT73, LBFLFK, MON88302, MON94100, MS8, MS11, NS-B50027-4, RF3, and T45), GM cotton (271/3006, COT102, DAS-81910-7, GHB119, GHB614, GHB811, LLcotton25, MON531, MON1445, MON15985, MON88701, MON88702, MON88913, and T304xGHB119), and GM potato (AM04-1020, EH92-527-1, PH05-026-0048, SPS-E12, and SPS-Y9) were kindly provided by the developers or purchased from the Institute for Reference Materials and Measurements (IRMM, Geel, Belgium) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Non-GM (NGM) seeds of soybean, maize, canola, cotton, rice, wheat, barley, and potato tissues were also used.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>GM reference materials used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event name</p></th><th align=\"left\"><p>Developer</p></th><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event Name</p></th><th align=\"left\"><p>Developer</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>RRS</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87411</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON89788</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>VCO-01981-5</p></td><td align=\"left\"><p>Genective</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A2704-12</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87403</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-356043-5</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87419</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-305423-1</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZHG0JG</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A5547-127</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZIR098</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87701</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-202216-6</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>CV127</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87429</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87769</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-023211-2</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87705</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON95379</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87708</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-915635-4</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>FG72</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON531</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>SYHT0H2</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON1445</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-44406-6</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON15985</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-68416-4</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>281/3006</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-81419-2</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>LLcotton25</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87751</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88913</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>GMB151</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB614</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON810</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>T304-40 × GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>GA21</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>COT102</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>TC1507</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88701</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>NK603</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON863</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>DAS-81910-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt11</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB811</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>T25</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88702</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt176</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>GT73</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DLL25</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>T45</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-59122-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON88302</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON88017</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS8</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR604</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>RF3</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt10</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>DP-073496-4</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON89034</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS11</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR162</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>LBFLFK</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-098140-6</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>NS-B50027-4</p></td><td align=\"left\"><p>NUSEED</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>3272</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON94100</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87460</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>5307</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>EH92-527-1</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87427</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-E12</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-40278-9</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>AM04-1020</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-004114-3</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>PH05-026-0048</p></td><td align=\"left\"><p>BASF</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec4\"><title>DNA extraction</title><p id=\"Par10\">NGM crops, GM soybean, GM maize, GM canola, GM cotton seeds, freeze-dried GM potato tissue, and processed products were ground separately. Samples of GM soybean mix (18 events), maize mix (33 events), canola mix (10 events), and cotton mix (14 events) were prepared by mixing 0.1 g of ground powder from each event. Genomic DNA (gDNA) was extracted from the powder samples using a Maxwell RCS PureFood GMO and Authentication Kit (Promega, Madison, WI, USA) according to the manufacturer’s instructions. DNA concentrations were determined using a QUBIT 3.0 Quantitation Kit (Thermo Fisher Scientific, Wilmington, DE, USA). DNA samples were diluted in nuclease-free water to a final concentration of 10 ng/µL and stored at − 20 °C until use.</p></sec><sec id=\"Sec5\"><title>Primers</title><p id=\"Par11\">Event-specific primers for the two GM potato events were used in PCR assays. <italic>APRT</italic> (GenBank accession number CK270447), which encodes adenine phosphoribosyltransferase, was selected as an endogenous reference gene. A previously reported APRT F/R primer set for <italic>APRT</italic> detection was used (MHLW, <xref ref-type=\"bibr\" rid=\"CR21\">2020</xref>). The SPS-Y9 mF1/mR1, and Event527-bf1/St527-R1 primer pairs were based on the 5' and 3' integration flanking sequences of T-DNA and the host genome of GM potato, respectively (Shin et al., <xref ref-type=\"bibr\" rid=\"CR23\">2020</xref>; JRC-GMO-Matrix, Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). All primers were synthesized by Bioneer (Daejeon, Republic of Korea), and the oligonucleotide sequences are listed in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The alignment sites of event-specific primers for GM potato SPS-Y9 and EH92-527-1. The arrows indicate the location of the primers. RB, right border; LB, left border; P-Vnt1, promoter for the native Rpi-vnt1 gene of <italic>Solanum venturii</italic>; Vnt1, Rpi-vnt1 gene encoding sequence of <italic>Solanum venturii</italic>; T-Vnt1, termination sequence of Rpi-vnt1 gene of <italic>Solanum venturii</italic>; P-Agp, promoter for the ADP glucose pyrophosphorylase gene of <italic>Solanum tuberosum</italic> var. Ranger Russet; VInv, vacuolar invertase gene fragment of <italic>Solanum tuberosum</italic> var. Ranger Russet; P-Gbss, promoter for the granule-bound starch synthase gene of <italic>Solanum tuberosum</italic>; P-nos, constitutive promoter of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; nptII, sequence encoding the NPTII enzyme (neomycin phosphotranspherase) from Tn5 transposon of E. coli; T-nos, termination sequence of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; Anti-gbss, antisense sequence encoding the gbss enzyme, involved in the starch synthase of <italic>Solanum tuberosum</italic></p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig1_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Oligonucleotide primers used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Target</p></th><th align=\"left\"><p>Name</p></th><th align=\"left\"><p>Sequence (5′ → 3′)</p></th><th align=\"left\"><p>Length (bp)</p></th><th align=\"left\"><p>Source</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>APRT F</p><p>APRT R</p></td><td align=\"left\"><p>TGAAAACGATCCCGATCG</p><p>CAATCCCAGCGATACGTTC</p></td><td align=\"char\" char=\".\"><p>90</p></td><td align=\"left\"><p>MHLW (Japan)</p></td></tr><tr><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Y9-mF1</p><p>Y9-mR1</p></td><td align=\"left\"><p>GAAGACAGCTATCAATATACTTTAGTAA</p><p>AATCTTTTGCCATACTCATACAGAG</p></td><td align=\"char\" char=\".\"><p>129</p></td><td align=\"left\"><p>MFDS (Korea)</p></td></tr><tr><td align=\"left\"><p>EH92</p><p>-527-1</p></td><td align=\"left\"><p>Event527-bf1</p><p>St527-R1</p></td><td align=\"left\"><p>GTGTCAAAACACAATTTACAGCA</p><p>TCCCTTAATTCTCCGCTCATGA</p></td><td align=\"char\" char=\".\"><p>93</p></td><td align=\"left\"><p>JRC (EU)</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec6\"><title>UF-PCR system</title><p id=\"Par12\">UF-PCR, which is based on the same principle as that of SYBR Green qPCR, was used by replacing SYBR Green with EvaGreen as an intercalating dye (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). PCR analysis was performed using a GENECHECKER UF-150 Ultra-fast Real-Time PCR system (Genesystem, Daejeon, Republic of Korea) with Rapi:chip. The final reaction volume was 10 µL consisting of 5 µL SSO Fast EvaGreen Supermixes (Bio-Rad Laboratories, Berkeley, CA, USA), 1 µL each of forward and reverse primers (0.8 µM), and 2 µL DNA template (50 ng). All PCR reactions were performed under the following conditions: one cycle of initial denaturation at 95 °C for 1 min, 40 cycles of denaturation at 95 °C for 5 s, annealing at 54 °C for 5 s, extension at 72 °C for 5 s, and one cycle of elongation at 72 °C for 5 s. No template control (NTC) was used as a negative control. The amplicons were analyzed using a microchip electrophoresis system, MCE-202 MultiNA (Shimazu, Nishinokyo, Kyoto, Japan) with MultiNA software v1.12 (Shimazu).</p></sec><sec id=\"Sec7\"><title>Specificity, sensitivity, and application test</title><p id=\"Par13\">The specificity of UF-PCR detection method was evaluated using various crops and GM events. To determine the detection level, gDNA samples of NGM potato and each GM potato event were mixed in concentrations ranging from 10 to 0.005% (w/w), and a sensitivity test was conducted. In addition, to check whether the method can be used for processed foods, a test was conducted on foods containing potato available in the market.</p></sec></sec><sec id=\"Sec8\"><title>Results and discussion</title><sec id=\"Sec9\"><title>Specificity</title><p id=\"Par14\">The UF-PCR assay was performed to assess the specificity of this detection method by analyzing NGM crops (soybean, maize, canola, cotton, wheat, rice, barley, and potato), GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048).</p><p id=\"Par15\">Results showed that only a single peak was detected for each target of GM potatoes, without specific amplification of non-target crops, GM events, and NTC (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>). In addition, the melting temperature (Tm) values of amplicons were 77.5 °C and 76.84 °C for SPS-Y9 and EH92-527-1, respectively (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Because amplicons have specific Tm values and do not cross-react, the two GM potato events could be selectively distinguished from each other by their Tm values.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Primer specificity for two GM potato events using Ultra-fast PCR. (<bold>A</bold>, <bold>C</bold>) Specificity results for Non-GM crops (eight crops; soybean, maize, canola, cotton, rice, wheat, potato, and barley) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (seven crops; soybean, maize, canola, rice, wheat, potato, and barley), lane 18: no template; (<bold>B</bold>, <bold>D</bold>) Specificity results for GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (eighty GM events; soybean, maize, canola, cotton, and potato), lane 18: no template</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig2_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Sensitivity results of the ultra-fast PCR method</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Events</p></th><th align=\"left\" rowspan=\"2\"><p>Conc</p></th><th align=\"left\" colspan=\"2\"><p>Ct value</p></th><th align=\"left\" colspan=\"2\"><p>Tm (°C) value</p></th></tr><tr><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>24.86</p></td><td align=\"char\" char=\".\"><p>0.25</p></td><td align=\"char\" char=\".\"><p>79.52</p></td><td align=\"char\" char=\".\"><p>0.21</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>SPS-Y9</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>26.34</p></td><td align=\"char\" char=\".\"><p>0.42</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>77.45</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.21</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>30.62</p></td><td align=\"char\" char=\".\"><p>0.40</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.12</p></td><td align=\"char\" char=\".\"><p>0.85</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>EH92-527-1</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>23.80</p></td><td align=\"char\" char=\".\"><p>0.40</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>76.77</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.27</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>29.75</p></td><td align=\"char\" char=\".\"><p>0.30</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.18</p></td><td align=\"char\" char=\".\"><p>0.69</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr></tbody></table><table-wrap-foot><p><italic>SD</italic> Standard deviation</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec10\"><title>Sensitivity</title><p id=\"Par16\">To evaluate the sensitivity of UF-PCR method for GM potato events, PCR assays were performed with gDNA samples that were serially diluted to concentrations ranging from 10 to 0.005% (wt/wt) per reaction. The Ct and Tm values of each event were analyzed. For all GM potato events, the assay was sufficiently identifiable even at 0.5% (0.25 ng) level of the target DNA. Based on the copy number of potato (one copy number for haploid genome of potato is considered as 1.8 pg (Arumuganathan and Earle, <xref ref-type=\"bibr\" rid=\"CR2\">1991</xref>)), 0.25 ng is equivalent to approximately 139 copies.</p><p id=\"Par17\">These results demonstrate that the rapid detection system used in this study can accurately detect GM incorporation as low as 0.5%. This level of detection is consistent with the low-level presence (LLP) of GMOs, that are not approved in their countries, but are already authorized in another countries (Stein and Rodríguez-Cerezo, <xref ref-type=\"bibr\" rid=\"CR26\">2010</xref>).</p></sec><sec id=\"Sec11\"><title>Application of UF-PCR in commercial processed products</title><p id=\"Par18\">Nine potato-containing processed products were tested to confirm the applicability of the UF-PCR method to GM potato, SPS-Y9 and EH92-527-1. The selected products included chips, snacks, ready-made meals, French fries, salads, and other items. Extracted gDNA samples were used for UF-PCR analysis, and the results are shown in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. In all the products, the APRT F/R primer set resulted in amplification, and the Tm values of the amplicons were approximately 79.3 °C. The primer sets for SPS-Y9 and EH92-527-1 did not produce amplicons for any of these samples. This suggests that the domestic distribution of processed potato products selected for this study did not contain two unapproved GM potato events.<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>The ultra-fast PCR results for potato DNA in processed products</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Products</p></th><th align=\"left\"><p>APRT</p></th><th align=\"left\"><p>SPS-Y9</p></th><th align=\"left\"><p>EH92-527-1</p></th></tr><tr><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Creamer potatoes</p></td><td align=\"left\"><p>28.15 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Baked potatoes</p></td><td align=\"left\"><p>30.09 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato salad</p></td><td align=\"left\"><p>31.10 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Frozen raw potatoes</p></td><td align=\"left\"><p>29.15 / 78.82</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Stick potatoes</p></td><td align=\"left\"><p>28.59 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Half cooked potatoes</p></td><td align=\"left\"><p>31.04 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Crinklecut potatoes</p></td><td align=\"left\"><p>30.11 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato chips</p></td><td align=\"left\"><p>24.65 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Deep fried potatoes</p></td><td align=\"left\"><p>30.85 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (SPS-Y9)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>35.04 / 77.50</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (EH92-527-1)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>34.22 / 76.51</p></td></tr></tbody></table><table-wrap-foot><p><italic>ND</italic> not determined</p></table-wrap-foot></table-wrap></p><p id=\"Par19\">A major challenge of this study was that obtaining products containing unauthorized GM potato was difficult. To address this issue, GM potato-spiked samples were tested together and confirmed to be correctly detected. Therefore, this assay method can be applied to analyze processed products based on the positive results for endogenous <italic>APRT</italic> of potato and GM spiked samples. The rapid detection method of GM potato using the UF-PCR system could be analyzed by selecting specific primers and identifying a specific amplification product with a unique melting temperature. In addition, this method was highly sensitive in detecting low copy numbers of DNA in GM potato samples. Recently, with the development of genetic analysis technology and equipment, high-speed analysis methods have been proposed to reflect the high-throughput demand for processing various samples. This reduces the analysis time, which normally takes 2 h, to as little as 40 min, thereby allowing more samples to be processed, faster results, and repetition of the test within the allotted time if case of any issues.</p><p id=\"Par20\">This study proposes a GM potato detection method using an UF-PCR system, which is a shortened test method. The method uses a very thin Rapi:chip and has the advantage of being able to finish the PCR reaction within a short time because of the high efficiency of heat transfer of the reaction mixtures and short duration between cycles.</p><p id=\"Par21\">The UF-PCR analysis method reviewed in this study is specific, sensitive, and reliable for identifying GM potato events. Moreover, the assay is eco-friendly because it uses a small amount of reagents, and if necessary, it can perform analysis using a small portable analyzer in field conditions rather than in the laboratory.</p><p id=\"Par22\">In conclusion, the reaction time and usage of reagents of the UF-PCR system are less than half of those for the conventional method, and the equipment is portable; therefore it can be utilized for on-site sample analysis. Therefore, we propose the use of this method that can perform a quick test within approximately 20 min to inspect and monitor GM potato events.</p></sec></sec></body><back><ack><title>Acknowledgements</title><p>This work was supported by the Ministry of Food and Drug Safety of Korea.</p></ack><sec><title>Funding</title><p>This study was funded by the Ministry of Food and Drug Safety of Korea (Grant Number 19161MFDS051).</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par23\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Aboud</surname><given-names>M</given-names></name><name><surname>Oh</surname><given-names>HH</given-names></name><name><surname>McCord</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Rapid direct PCR for forensic genotyping in under 25 min</article-title><source>Electrophoresis.</source><year>2013</year><volume>34</volume><fpage>1539</fpage><lpage>1547</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXos12qsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/elps.201200570</pub-id><pub-id pub-id-type=\"pmid\">23529977</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Arumuganathan</surname><given-names>K</given-names></name><name><surname>Earle</surname><given-names>ED</given-names></name></person-group><article-title xml:lang=\"en\">Nuclear DNA content of some important plant species</article-title><source>Plant Molecular Biology Reporter.</source><year>1991</year><volume>9</volume><fpage>208</fpage><lpage>218</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaK38XhslSjurY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/BF02672069</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"other\">Chen XJ, Liu YT, Zhan XA, Gao YB, Sun ZY, Wen WJ, Zheng WD. Ultrafast PCR detection of COVID-19 by using a microfluidic chip-based system. Bioengineering (Basel). 9: 548 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.3390/bioengineering9100548\">https://doi.org/10.3390/bioengineering9100548</ext-link></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cote</surname><given-names>MJ</given-names></name><name><surname>Meldrum</surname><given-names>AJ</given-names></name><name><surname>Raymond</surname><given-names>P</given-names></name><name><surname>Dollard</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Identification of genetically modified potato cultivars using event specific polymerase chain reaction</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2005</year><volume>53</volume><fpage>6691</fpage><lpage>6696</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXmsFGjs74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf050591i</pub-id><pub-id pub-id-type=\"pmid\">16104786</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"other\">Das D, Hsieh HC, Chen CS, Chen WL, Chuang HS. Ultrafast and sensitive screening of pathogens by functionalized janus microbeads-enabled rotational diffusometry in combination with isothermal amplification. Small Science. 2: 202200010 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.1002/smsc.202200010\">https://doi.org/10.1002/smsc.202200010</ext-link></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"other\">Food and Agriculture Organization of the United Nations. FAOSTAT Statistical Database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fao.org/faostat\">https://www.fao.org/faostat</ext-link>. Accessed Aug. 27, 2022.</mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fraiture</surname><given-names>MA</given-names></name><name><surname>Roosens</surname><given-names>NHC</given-names></name><name><surname>Taverniers</surname><given-names>I</given-names></name><name><surname>Loose</surname><given-names>MD</given-names></name><name><surname>Deforce</surname><given-names>D</given-names></name><name><surname>Herman</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Biotech Rice: Current developments and future detection challenges in food and feed chain</article-title><source>Trends in Food Science &amp; Technology.</source><year>2016</year><volume>52</volume><fpage>66</fpage><lpage>79</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XmtFCntr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2016.03.011</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gao</surname><given-names>H</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Deng</surname><given-names>T</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name><name><surname>Xiao</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Event-specific detection of transgenic potato AV43-6-G7 using real-time and digital PCR methods</article-title><source>BMC Biotechnology.</source><year>2016</year><volume>16</volume><fpage>74</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlvV2gt7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12896-016-0303-8</pub-id><pub-id pub-id-type=\"pmid\">27784303</pub-id><pub-id pub-id-type=\"pmcid\">5081928</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hong</surname><given-names>YW</given-names></name><name><surname>Kim</surname><given-names>JJ</given-names></name><name><surname>Yu</surname><given-names>YC</given-names></name><name><surname>Kim</surname><given-names>HS</given-names></name><name><surname>Moon</surname><given-names>GI</given-names></name><name><surname>Park</surname><given-names>EM</given-names></name></person-group><article-title xml:lang=\"en\">Ultra-fast PCR method for the distinguishing between Miichthys miiuy and Sciaenops ocellatus</article-title><source>Food Science and Biotechnology.</source><year>2021</year><volume>30</volume><issue>9</issue><fpage>1225</fpage><lpage>1231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXit1Ggs7rE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-00954-4</pub-id><pub-id pub-id-type=\"pmid\">34603821</pub-id><pub-id pub-id-type=\"pmcid\">8423894</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"other\">International Service for the Acquisition of Agri-Biotech Applications (ISAAA). GM approval database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.isaaa.org/gmapprovaldatabase\">https://www.isaaa.org/gmapprovaldatabase</ext-link>. Accessed Sep. 18, 2022.</mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaccaud</surname><given-names>E</given-names></name><name><surname>Hohne</surname><given-names>M</given-names></name><name><surname>Meyer</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of screening methods for the identification of genetically modified potatoes in raw materials and finished products</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2003</year><volume>51</volume><fpage>550</fpage><lpage>557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XpsFCnsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf0208031</pub-id><pub-id pub-id-type=\"pmid\">12537422</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Suh</surname><given-names>SM</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Rapid on-site detection of shrimp allergen tropomyosin using a novel ultrafast PCR system</article-title><source>Food Science and Biotechnology</source><year>2019</year><volume>28</volume><issue>2</issue><fpage>591</fpage><lpage>597</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhvVerurfP</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-018-0479-x</pub-id><pub-id pub-id-type=\"pmid\">30956872</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"other\">Kim SH, Lee YS, Joo IS, Kwak HS, Chung GT, Kim SH. Rapid detection for salmonella spp. by ultrafast real-time PCR assay. J. Food Hyg. Saf. 33: 50–57 (2018)</mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SW</given-names></name><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>ES</given-names></name><name><surname>Kim</surname><given-names>DJ</given-names></name><name><surname>Park</surname><given-names>SH</given-names></name><name><surname>Lee</surname><given-names>EJ</given-names></name><name><surname>Lee</surname><given-names>SY</given-names></name><name><surname>Lee</surname><given-names>JS</given-names></name><name><surname>Lim</surname><given-names>CS</given-names></name><name><surname>Kim</surname><given-names>WK</given-names></name><name><surname>In</surname><given-names>KH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of mycobacterium tuberculosis using a novel ultrafast chip-type real-time polymerase chain reaction system</article-title><source>Chest.</source><year>2014</year><volume>146</volume><issue>5</issue><fpage>1319</fpage><lpage>1326</lpage><pub-id pub-id-type=\"doi\">10.1378/chest.14-0626</pub-id><pub-id pub-id-type=\"pmid\">24967657</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Martinez-Prada</surname><given-names>MM</given-names></name><name><surname>Curtin</surname><given-names>SJ</given-names></name><name><surname>Gutierrez-Gonzalez</surname><given-names>JJ</given-names></name></person-group><article-title xml:lang=\"en\">Potato improvement through genetic engineering</article-title><source>GM Crops &amp; Food.</source><year>2021</year><volume>12</volume><fpage>479</fpage><lpage>496</lpage><pub-id pub-id-type=\"doi\">10.1080/21645698.2021.1993688</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Milosevic</surname><given-names>J</given-names></name><name><surname>Lu</surname><given-names>M</given-names></name><name><surname>Greene</surname><given-names>W</given-names></name><name><surname>He</surname><given-names>HZ</given-names></name><name><surname>Zheng</surname><given-names>SY</given-names></name></person-group><article-title xml:lang=\"en\">An ultrafast one-step quantitative reverse transcription-polymerase chain reaction assay for detection of SARS-CoV-2</article-title><source>Frontiers in Microbiology.</source><year>2021</year><volume>12</volume><pub-id pub-id-type=\"doi\">10.3389/fmicb.2021.749783</pub-id><pub-id pub-id-type=\"pmid\">34803970</pub-id><pub-id pub-id-type=\"pmcid\">8600229</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"other\">Ministry of Food and Drug Safety (MFDS). GMO status database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.foodsafetykorea.go.kr/portal/board/board.do\">https://www.foodsafetykorea.go.kr/portal/board/board.do</ext-link>. Accessed July. 15, 2022.</mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"other\">Ministry of Health, Labour and Welfare (MHLW). Detection method for genetically modified DNA (2020). Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf\">https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf</ext-link>. Accessed Feb. 16, 2022.</mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Roh</surname><given-names>JK</given-names></name><name><surname>Lee</surname><given-names>T</given-names></name><name><surname>Jung</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>TA</given-names></name><name><surname>Park</surname><given-names>YH</given-names></name><name><surname>Kim</surname><given-names>YM</given-names></name></person-group><article-title xml:lang=\"en\">Qualitative and Quantitative PCR methods for detection of three lines of genetically modified potatoes</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2004</year><volume>52</volume><fpage>3269</fpage><lpage>3274</lpage><pub-id pub-id-type=\"doi\">10.1021/jf0499020</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"other\">Shin MK, Jeon SM, Hong SJ, Choi CH, Kim HA, Park JS. Research on the establishment of test method for labeling and management of GMOs. Ministry of Food and Drug Safety (2020)</mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shin</surname><given-names>MK</given-names></name><name><surname>Jeon</surname><given-names>SM</given-names></name><name><surname>Koo</surname><given-names>YE</given-names></name></person-group><article-title xml:lang=\"en\">Development of a rapid detection method for genetically modified rice using the ultra-fast PCR system</article-title><source>Food Science and Biotechnology</source><year>2022</year><volume>31</volume><issue>2</issue><fpage>175</fpage><lpage>182</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XmsF2gs78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-01025-4</pub-id><pub-id pub-id-type=\"pmid\">35095287</pub-id><pub-id pub-id-type=\"pmcid\">8783574</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Song</surname><given-names>JY</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Detection of unapproved genetically modified potatoes in Korea using multiplex polymerase chain reaction</article-title><source>Food Control.</source><year>2017</year><volume>80</volume><fpage>19</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmsFejtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2017.04.026</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Stein</surname><given-names>AJ</given-names></name><name><surname>Rodríguez-Cerezo</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Low-level presence of new GM crops: An issue on the rise for countries where they lack approval</article-title><source>AgBioForum.</source><year>2010</year><volume>13</volume><issue>2</issue><fpage>173</fpage><lpage>182</lpage></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tu</surname><given-names>YK</given-names></name><name><surname>Lin</surname><given-names>YC</given-names></name><name><surname>Feng</surname><given-names>YW</given-names></name><name><surname>Tseng</surname><given-names>YY</given-names></name><name><surname>Chen</surname><given-names>HW</given-names></name></person-group><article-title xml:lang=\"en\">Visual, sensitive and rapid event-specific detection of genetically modified potato EH92-527-1 by loop-mediated isothermal amplification method</article-title><source>Bioscience, Biotechnology, and Biochemistry.</source><year>2020</year><volume>84</volume><fpage>43</fpage><lpage>52</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsleitrjO</pub-id><pub-id pub-id-type=\"doi\">10.1080/09168451.2019.1661766</pub-id><pub-id pub-id-type=\"pmid\">31495297</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"other\">Visual Capitalist. Mapped: Food production around the world. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world</ext-link>. Accessed Aug. 18, 2022.</mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoo</surname><given-names>MR</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Yea</surname><given-names>MC</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Development of detection method of unapproved genetically modified potato (EH92-527-1) in Korea using duplex polymerase chain reaction</article-title><source>Korean Journal of Food Science and Technology.</source><year>2013</year><volume>45</volume><fpage>156</fpage><lpage>160</lpage><pub-id pub-id-type=\"doi\">10.9721/KJFST.2013.45.2.156</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"other\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10953</journal-id><journal-id journal-id-type=\"doi\">10.1007/10953.1572-8927</journal-id><journal-title-group><journal-title>Journal of Solution Chemistry</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Solution Chem</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0095-9782</issn><issn pub-type=\"epub\">1572-8927</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10953-023-01308-w</article-id><article-id pub-id-type=\"manuscript\">1308</article-id><article-id pub-id-type=\"doi\">10.1007/s10953-023-01308-w</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">In Memoriam: Jacques Desnoyers (1935–2023)</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Quirion</surname><given-names>Francois</given-names></name><address><email>francois.quirion@physicochimie.fr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs1095302301308w_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Perron</surname><given-names>Gérald</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Desnoyers</surname><given-names>Christine</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Desnoyers</surname><given-names>Marie</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Desnoyers</surname><given-names>Nichola</given-names></name></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Consultant R&amp;D Physicochimie</institution></institution-wrap><addr-line content-type=\"street\">261 Boulevard Pereire</addr-line><addr-line content-type=\"postcode\">75017</addr-line><addr-line content-type=\"city\">Paris</addr-line><country country=\"FR\">France</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Retired from INRS-Énergie Matériaux et Télécommunications</institution></institution-wrap><addr-line content-type=\"city\">Varennes</addr-line><addr-line content-type=\"state\">QC</addr-line><country country=\"CA\">Canada</country></aff></contrib-group><author-notes><corresp id=\"IDs1095302301308w_cor1\"><label>a</label><email>francois.quirion@physicochimie.fr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>20</day><month>6</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>52</volume><issue seq=\"1\">8</issue><fpage>859</fpage><lpage>860</lpage><history><date date-type=\"registration\"><day>31</day><month>5</month><year>2023</year></date><date date-type=\"online\"><day>20</day><month>6</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>9</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>5</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10953_2023_Article_1308.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>EditorialNotes</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Geochemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Oceanography</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Inorganic Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Condensed Matter Physics</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Christine Desnoyers, Marie Desnoyers and Nichola Desnoyers are son and daughters of Professor Desnoyers.</p></notes></front><body><p id=\"Par1\">It is with great sadness that we report the death of Jacques Desnoyers who passed away peacefully in Quebec City on January 12th at the age of 87, surrounded by his loved ones.<fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"JPEG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/10953_2023_1308_Figa_HTML.jpg\"/></fig></p><p id=\"Par2\">Jacques Desnoyers was born on 28th January 1935 in Ottawa where he obtained his Ph.D. in 1961 at the University of Ottawa. After a year of post-doctorate studies in England (University of London and of Manchester), he joined, in 1962, the chemistry department at the <italic>Université de Sherbrooke</italic> as full professor and later as director. In 1983, he became the scientific director of <italic>Institut national de la recherche scientifique</italic> (INRS) where he also acted as interim general director (3 March 1987 to 31 July 1988). In 1991, he returned to research as full professor at <italic>INRS</italic>. During the same period, he was appointed as Ombudsman at the <italic>Université du Québec</italic>. In 2002, he retired from INRS. Meanwhile, he accepted the position of scientific advisor for the <italic>Gouvernement du Québec</italic> until 2006. He finally retired fully in 2007.</p><p id=\"Par3\">He developed and promoted flow microcalorimetry with Patrick Picker. He investigated the thermodynamic properties of electrolytes and non-electrolytes in aqueous, non-aqueous and mixed solvents, surfactant solutions, swollen micelles and microemulsions, and polymer-surfactant systems. His research found applications in the fields of low-temperature batteries, Li-ion batteries, enhanced oil recovery and soil remediation.</p><p id=\"Par4\">His work was internationally recognized and he frequently collaborated with other Canadian groups but also with groups around the world, including the USA, France and Italy. After his retirement, he received the 2007 Setaram Award from the <italic>Associazione italiana di calorimetria e analisi termica</italic> (AICAT) for his outstanding scientific achievements in the field of calorimetry.</p><p id=\"Par5\">Throughout his career, Jacques Desnoyers strongly believed in the importance of transmitting knowledge which he did through teaching, directing thesis, writing scientific articles and book chapters and reviewing manuscripts for many scientific journals.</p><p id=\"Par6\">He will be remembered for his <italic>joie de vivre</italic>, his human qualities, his professional ethical principles and for his continuous effort to promote the use of thermodynamic properties as powerful tools for the investigation of simple and complex systems.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02771-x</article-id><article-id pub-id-type=\"manuscript\">2771</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02771-x</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Optimization of microwave-enhanced extraction parameters to recover phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-5114-2100</contrib-id><name><surname>Alara</surname><given-names>Oluwaseun Ruth</given-names></name><address><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302771x_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Abdurahman</surname><given-names>Nour Hamid</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Ali</surname><given-names>Hassan Alsaggaf</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Centre for Research in Advanced Fluid and Processes (Fluid Centre)</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, College of Engineering</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution content-type=\"org-name\">Eastern Unity Technology</institution></institution-wrap><addr-line content-type=\"street\">Suite 01, 12th Floor Plaza, 138 Annex Hotel Maya, Jalan Ampang</addr-line><addr-line content-type=\"postcode\">50450</addr-line><addr-line content-type=\"city\">Kuala Lumpur</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302771x_cor1\"><label>a</label><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>10</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"9\">8</issue><fpage>4217</fpage><lpage>4233</lpage><history><date date-type=\"registration\"><day>6</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>23</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>4</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>10</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Vegetables are industrial crops endowed with both nutritional and medicinal values. The overwhelming contributions of vegetables to human living in the form of nutrients and medicine cannot be under emphasised. Thus, this study examined the recoveries of phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves using a microwave-enhanced extraction technique. Furthermore, the phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were comprehensively identified using liquid chromatography-mass spectrometry quadrupole of flight (LC-QToF-MS). At the optimized conditions of microwave-enhanced extraction (extraction time of 131 s, microwave power 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%), total phenolic content (TPC) of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for 2,2-diphenyl-1-picrylhydrazyl (DPPH) and 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS) assays, respectively, were achieved. Furthermore, an aggregate of fourteen phenolic compounds that include 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose was tentatively identified in the leaf extract of <italic>C. olitorius</italic>. In general, this study has established <italic>C. olitorius</italic> leaves as sources of phenolic compounds and natural antioxidants. Thus, the intake can continue to be promoted as a way forward in solving the problem of food insecurity.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Corchorus olitorius</italic></kwd><kwd>Response surface methodology</kwd><kwd>Total phenols</kwd><kwd>Antioxidant</kwd><kwd>Liquid chromatography quadrupole time-of-flight mass spectrometry</kwd><kwd>Microwave-enhanced extraction</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2771.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02771-x\">https://doi.org/10.1007/s11696-023-02771-x</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par12\">Vegetables have been categorised as industrial crops with valuable means of nutrients and a source of income. Other than the valuable nutrients in vegetables, some of them possess diverse medicinal characteristics that are good for human and animal health. Although indigenous vegetables are long been employed in day-to-day diets in different communities of the world, several of these vegetables are underutilised as most have embodiments of unknown and underestimated pharmacological features. Most attention is based on the consumption of maize, rice, and cassava. It is not disputed these crops are endowed with great nutritional values; nevertheless, vegetables can be prioritised due to their functional properties (food and medicine). Additionally, United Nation Sustainable Development Goals (SDG 2) established the importance of figuring out sustainable means of producing foods for humans, especially natural sources. This is highly needed in this time of post-Covid-19 recovery as the world is being faced with hunger, food insecurity and consumption of low-quality foods that cause micronutrient deficiencies (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>)</p><p id=\"Par13\"><italic>Corchorus olitorius</italic> (Jute mallow) is a leafy vegetable mostly found in Tropical regions including the Middle East, Africa and some parts of Asia (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). It belongs to the family of Malvaceae (Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). This is one of the vegetables consumed mainly in different African countries including Nigeria, Tanzania, South Africa, and Ethiopia (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). Consuming this vegetable has been encouraged to curb malnutrition and food insecurity, especially in developing nations such as African countries. The consumption of <italic>C. olitorius</italic> leaves is much encouraged as it is believed to overcome nutritional and food insecurity among children under the ages of five, lactating mothers and pregnant women in sub-Saharan Africa (Andersen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2003</xref>; Kamga et al. <xref ref-type=\"bibr\" rid=\"CR17\">2013</xref>). This vegetable is mostly consumed in the form of soup or sauce alongside solid foods from rice, yam or cassava in African cooking cuisine. It is endowed with different vitamins, enrich in fibre, and antioxidants (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). A brief description of <italic>C. olitorius</italic>: it is an annual vegetable that can reach a height of about 2.4 m; the leaves are finely lobed or serrated margin, simple, lanceolate, and alternate; it has only a few branches and can be unbranched; seeds are used when planting; the flowers are yellowish small with five petals (2–3 cm in diameters) and can survive in any kinds of soil (loamy, sandy or clay) (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). The leaves of <italic>C. olitorius</italic> are enriched in beta carotene, vitamins A and C, folic acid, calcium, iron, and diverse phenolic compounds (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). The leaves of <italic>C. olitorius</italic> were reported to contain nutritional values more than stems and fruits. This is the reason why it is generally used as food. The leaves had been revealed to be made up of about seventeen active nutrients compounds such as protein, carbohydrate, fat, ash, fibre, potassium, calcium, sodium, phosphorous, iron, beta-carotene, riboflavin, thiamine, ascorbic acid, niacin, and so on. Moreover, the leaves had been reported to possess hepatoprotective, antidiabetic, antioxidant, diuretic, antipyretic, analgesic, antitumor, antimicrobial, anti-obesity, hypoglycemic, and gastroprotective properties (Abir et al. <xref ref-type=\"bibr\" rid=\"CR1\">2019</xref>; Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>; Saliu et al. <xref ref-type=\"bibr\" rid=\"CR28\">2019</xref>).</p><p id=\"Par14\">The presence of phenolic compounds in plant materials is continually studied due to their important link with the occurrence of natural antioxidants that possess outstanding applications in human health (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Cheynier <xref ref-type=\"bibr\" rid=\"CR13\">2012</xref>). The antioxidants from leafy vegetables such as <italic>C. olitorius</italic> can act in place of synthetic antioxidants readily available in the market. These synthetic antioxidants are being reported to pose side effects on consumers (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). However, natural antioxidants can act as protection against degenerative diseases and alter the DNA, proteins and lipids (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). Besides, plants are surviving the exposure to ultraviolet radiation in the environment; this has positioned them as a perfect antioxidant source. Likewise, the consumptions of plant-sourced antioxidants are relatively safe and widely accepted; this has propelled the introduction of plant polyphenols into cosmeceutical and pharmaceutical products in place of synthetic antioxidants (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>; Lobo et al. <xref ref-type=\"bibr\" rid=\"CR20\">2010</xref>). The process of extraction is important when targeting to achieve significant phenolic compounds and antioxidants from plant materials. This process can either be conventional or non-conventional; the latter is currently being used due to its ability to recover a tangible quantity of phenolic compounds and antioxidants from plant materials (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Herrero et al. <xref ref-type=\"bibr\" rid=\"CR16\">2012</xref>). Out of several unconventional techniques that are being used, the microwave-enhanced method has been generally accepted due to its ability to leach a significant quantity of phenolic compounds from the considered sample within a reduced duration of time and not alter the bioactive elements in the extracts (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>).</p><p id=\"Par15\">Several parameters can alter the efficiency of recovered phenolic compounds and antioxidants from plant matrices when using microwave-enhanced technique; these are temperature, extraction time, solvent/sample ratio, solvent concentration, microwave power, and solvent selection. Out of different available solvents used in the extraction processes, ethanol and water are the most environment-friendly. It had been explained that pure ethanol cannot extract the totality of phenolic compounds especially those that are hydrophilic (water-soluble) (Thoo et al. <xref ref-type=\"bibr\" rid=\"CR32\">2010</xref>). Previous studies had suggested that the use of aqueous ethanol or binary mixture contributes to the recovery of antioxidant from plant sample (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>). Thus, mixing these two solvents (water and ethanol) in rightful proportion tends to improve the recovery of bioactive materials from plant matrices (Khoddami et al. <xref ref-type=\"bibr\" rid=\"CR18\">2013</xref>). As the selection of extraction is important in achieving significant recovery, process optimization is as well essential to achieve this goal.</p><p id=\"Par16\">There are two common techniques used for process optimization (one leading to the other); these are one-factor-at-a-time (OFAT) experimental trials and response surface methodology (RSM). Although OFAT suffers drawbacks such as no room for interaction between the factors; nonetheless, OFAT is widely employed to determine the range of values used for the response surface methodology. RSM is utilised to settle the limitations of OFAT experimental trials which include inadequate data evaluation, being expensive and time-consuming (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par17\">Although diverse studies had been carried out on <italic>C. olitorius</italic> leaves concerning the proximate constituents, total phenols and antioxidants (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). Nevertheless, there is yet a report on the optimization studies involving microwave-enhanced extraction of the total phenols and antioxidants from <italic>C. olitorius</italic> leaves and comprehensive identifications of the phenolic compounds. Hence, this study focuses on unveiling the optimised microwave-enhanced extraction conditions to achieve maxima phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. At the optimized conditions, the comprehensive identifications of different phenolic compounds in the extracts were carried out using a fast liquid chromatography quadrupole time-of-flight mass spectrometry (LC-QToF-MS).</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Cultivation of <italic>C. olitorius</italic> seeds and sample</title><p id=\"Par18\">The seeds of <italic>C. olitorius</italic> were cultivated in a small garden in the Universiti Malaysia Pahang (UMP) for two months (from February and April 2019); every agricultural practice such as watering, weed control and thinning was adequately observed to achieve productive results. Sandy-loam soil was used for the cultivation. In the maturity stage, the leaves of <italic>C. olitorius</italic> were harvested and washed to evacuate dirt under running water. Then, the leaves were allowed to drain off the water and dried at room temperature until a constant weight was reached. The dried leaves were milled using a blender and sieved using a laboratory sieve (400/425 µm pore size). Afterwards, the ground sample was kept in a zip-lock bag and kept in a freezer at − 20 °C before the extraction process.</p></sec><sec id=\"Sec4\"><title>Chemicals and reagents used</title><p id=\"Par19\">Ethanol, gallic acid, methanol, 2,2-diphenyl-1-picrylhydrazyl (DPPH), sodium carbonate, 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS), potassium persulfate, and Folin–Ciocalteu reagent were procured from Sigma-Aldrich (Selangor, Malaysia).</p></sec><sec id=\"Sec5\"><title>Microwave-enhanced extraction through experimental design</title><p id=\"Par20\">The solvents used in this study were water and ethanol. An Ethos microwave oven (Milestone, Italy) was employed, and this extractor is configured using a control compartment with power, time and temperature control. Since the microwave-enhanced extraction technique is influenced by microwave power, temperature, time, solvent concentration, and solvent/sample ratio; thus, these factors were first studied by carrying out OFAT experimental trials to evaluate the effect of an individual factor on the recovered phenolic compounds and antioxidants as well as the range for individual factor. These ranges of values were employed for the optimization studies. For the extraction, 5 g of <italic>C. olitorius</italic> leaf sample was adequate mixed together with an estimated proportion of ethanol–water solvent (experimental design). The mixture was filtered, and a rotary evaporator set at 50 °C was utilised to concentrate the extract.</p><p id=\"Par21\">After the determination of preliminary ranges for the five considered factors through OFAT experimental trials, a face-centred central composite design (FCCCD) was used to optimise the recovery of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Four factors were considered during the optimization study since temperature (kept at 40 °C) showed an insignificant effect on the dependent factors (total phenolic compounds (TPC) and antioxidant activity (DPPH and ABTS)). Thus, the independent factors were extraction time (<italic>X</italic><sub>1</sub>: 60–180 s), microwave power (<italic>X</italic><sub>2</sub>: 200–400 W), solvent/sample (<italic>X</italic><sub>3</sub>: 8–16 mL/1 g), and ethanol concentration (<italic>X</italic><sub>4</sub>: 30–50%), while the dependent factors were TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>). Thirty randomised runs of experiments with six centre points were obtained using FCCCD (Table S1 in supplementary materials). The regression analysis for this study was initiated using Eq. (<xref ref-type=\"disp-formula\" rid=\"Equ1\">1</xref>).<disp-formula id=\"Equ1\"><label>1</label><alternatives><mml:math display=\"block\" id=\"Equ1_Math\"><mml:mrow><mml:mi>Y</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>β</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mrow><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn><mml:mo>,</mml:mo><mml:mi>i</mml:mi><mml:mo>&lt;</mml:mo><mml:mi>j</mml:mi></mml:mrow><mml:mrow><mml:mi>k</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:munderover><mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>j</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ij</mml:mi></mml:mrow></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>i</mml:mi></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>j</mml:mi></mml:msub><mml:mo>+</mml:mo></mml:mrow></mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ii</mml:mi></mml:mrow></mml:msub><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mi>i</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mrow></mml:math><tex-math id=\"Equ1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$Y = \\beta_{0} + \\sum\\limits_{i = 1}^{k} {\\beta_{1} X_{1} } + \\sum\\limits_{i = 1,i &lt; j}^{k = 1} {\\sum\\limits_{j = 2}^{k} {\\beta_{ij} X_{i} X_{j} + } } \\sum\\limits_{i = 2}^{k} {\\beta_{ii} X_{i}^{2} }$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ1.gif\"/></alternatives></disp-formula>where <italic>Y</italic> indicates the dependent factors which are TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>), <italic>β</italic><sub>0</sub> is a constant, <italic>β</italic><sub><italic>i</italic></sub> indicates the linear coefficient (actual effect), <italic>β</italic><sub><italic>ij</italic></sub> represents the interaction coefficients between two considered extraction factors, <italic>β</italic><sub><italic>ii</italic></sub> is the second-order polynomial coefficient, and <italic>k</italic> is the number of dependent factors (<italic>k</italic> = 4). The significant values were removed, while only the insignificant values were used to construct a reduced model (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par22\">Having achieved the predicted yields of the considered responses, the adequacy of the constructed models was tested. These were carried out by comparing the actual and predicted values. A pair <italic>t</italic>-test analysis was employed to verify the postulated models.</p></sec><sec id=\"Sec6\"><title>Determination of TPC in the <italic>C. olitorius</italic> leaf extract</title><p id=\"Par23\">The TPC in extract from <italic>C. olitorius</italic> leaves was estimated following the previously described procedures (Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>). Briefly, 200 µL of Folin–Ciocalteu reagent was added to 100 µL of leaf extract of <italic>C. olitorius</italic>; then, 0.2 mM Na<sub>2</sub>CO<sub>3</sub> using the volume of 600 µL was added and the mixture was allowed to stay 2 h in the dark at room temperature for adequate reactions. After this, the absorbance of the reacted mixture was read at 765 nm.</p></sec><sec id=\"Sec7\"><title>Determination of antioxidant activity</title><sec id=\"Sec8\"><title>DPPH analysis</title><p id=\"Par24\">The free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, the DPPH solution was prepared to achieve 0.1 mM. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves was presented as IC<sub>50</sub> (this is the concentration of the <italic>C. olitorius</italic> leaf extract needed to inhibit 50% formation of DPPH radicals). These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 200 µL of the extract and 0.1 mM DPPH solution (2 mL volume) were thoroughly mixed. The absorbance was read at 517 nm after the mixture was left to adequate react for 30 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1478<italic>x</italic> + 19.781, <italic>R</italic><sup>2</sup> = 0.9986).</p></sec><sec id=\"Sec9\"><title>ABTS analysis</title><p id=\"Par25\">For the ABTS analysis, the free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, an equal ratio of 2.45 mM potassium persulfate and 7 mM ABTS<sup>+</sup> solution was dissolved in 120 mL of methanol to achieve the stock solution of ABTS. This stock solution was allowed to react for 12 h before its usage. Then, 2 mL of stock solution was further diluted in 120 mL of methanol to obtain a fresh solution with an absorbance of 1.10 ± 0.02 at 734 nm. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves using ABTS was presented as IC<sub>50</sub>. These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 2.85 mL of a fresh solution of ABTS and 0.15 mL of extract were thoroughly mixed. At 734 nm, the absorbance was recorded after the mixture had been left to react for 120 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1008<italic>x</italic> + 46.858, <italic>R</italic><sup>2</sup> = 0.9977).</p></sec></sec><sec id=\"Sec10\"><title>LC-QToF-MS analysis of phenolic compounds</title><p id=\"Par26\">Every phenolic compound in the leaf extract of <italic>C. olitorius</italic> at the optimized microwave-enhanced extraction conditions was identified using LC-QToF-MS (Waters Vion IMS, USA). A C<sub>18</sub> column with the dimension of 100 mm × 2.10 mm × 1.80 µm was used. There are two mobile phases used in this analysis; they are water + 0.1% formic acid (A) and 100% acetonitrile (B). The operating conditions were a seal wash period of 5 min, a pre-injector volume of 0 µL, low pressure of 0 psi, high-pressure limit of 18,000 psi, gradient elution was 90% A and 10% B (1.25 min), 45% A and 55% B (4.17 min), 10% A and 90% B (6.25 min), 90% A and 10% B (8.34 min), with the injection volume of 20 µL, the flow rate of 0.50 mL/min, spray voltage of 4 keV, mass range of 100–1000 m/z, sample temperature of 15 °C, column temperature of 40 °C, gas flow of 0.50 mL/min, desolvation gas flow rate of 800 L/h, desolvation temperature of 550 °C, low collision energy of 4 eV, high collision energy ramp between 10 and 45 eV, source temperature of 120 °C. The identifications of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were performed by utilising SYNAPT mass spectrometer (Waters) coupled with an electrospray ionization operated in negative ion mode. Waters® UNIFY Software 1.0.0 installed in equipment was used to identify the phenolic compounds.</p></sec><sec id=\"Sec11\"><title>Data analysis</title><p id=\"Par27\">Each experimental trial was repeated thrice, and the data were recorded as mean values ± standard deviation. The Design Expert® software (Version 7, Stat. Ease Inc., Minneapolis, USA) was employed to carry out the optimization study. The model significance was based on a <italic>p</italic> value less than 0.05 and vice versa for the insignificant model. Three-dimensional plots were used to understand the interactions between two factors. A Chi-squared test was used to verify the validity of the predicted models.</p></sec></sec><sec id=\"Sec12\"><title>Results and discussions</title><sec id=\"Sec13\"><title>One-factor-at-a-time experimental trials for the extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</title><p id=\"Par28\">From the beginning of this study, the preliminary experimental trials were to study the effects of the factors influencing the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves using a microwave-enhanced extraction technique. These factors were extraction time, microwave power, solvent/sample ration, ethanol concentration, and temperature. Likewise, the experimental ranges for the subsequent optimization study were determined using OFAT. OFAT experimental trials involve varying a factor, while others are kept constant.</p><sec id=\"Sec14\"><title>Contributing impact of extraction time</title><p id=\"Par29\">Extraction time is an essential factor when considering solid–liquid extraction; this is due to its impact on mass transfer and solubility. Figure <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ai and Aii showed the impact of extraction time on the recoveries of phenolics and antioxidants from <italic>C. olitorius</italic>. This was done by keeping solvent/sample ratio, microwave power, ethanol concentration, and temperature at 8 mL/g, 200 W, 20%, and 20 °C, respectively. It can be observed that TPC recovery increased with extraction time to achieve a higher yield at 120 s before the decline in the yield set in. This might be because the extended time of extraction can be dangerous to phenolic compounds in the plant matrix; leading to degradation when expose to microwave radiation (Alara et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021b</xref>; Azwanida <xref ref-type=\"bibr\" rid=\"CR10\">2015</xref>). On contrary, the recoveries of antioxidant through DPPH and ABTS declined with increasing time; the minima yield of antioxidants was achieved at 120 s prior to the further increase in the yield with increasing extraction time. This decrease occurs because the antioxidant activity is inversely proportional to IC<sub>50</sub> values (linear regression between the percentage of inhibition and extract concentration) (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>). Thus, a lesser IC<sub>50</sub> value indicates higher antioxidant potential. The maxima scavenging potentials of <italic>C. olitorius</italic> leaves for both DPPH and ABTS were reached at 120 s. For the optimization study, 60–180 s was the range of values. However, 120 s was considered as the fixed extraction time when studying the impacts of other factors.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Multiple parameters that influence the recoveries from phenolics and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec15\"><title>Contributing impact of solvent/sample ratio</title><p id=\"Par30\">During the solid–liquid extraction, the solvent/sample ratio can impact the recoveries of phenolic compounds and antioxidants. This is because this factor has an impact on the concentration gradient between the solutes in the plant sample and the solvent (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). In this study, the weight of the <italic>C. olitorius</italic> leaves was kept constant, while the volume of the solvent was varied. The effect of solvent/sample ratio was studied by keeping extraction time, microwave power, ethanol concentration, and temperature at 120 s, 200 W, 20%, and 20 °C, respectively. As seen in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bi, the recovery of TPC tends to increase as the solvent/sample ratio improves until the maximum recovery of TPC was obtained when the solvent/sample ratio reached 12 mL/g. Beyond the solvent/sample ratio of 12 mL/g, the occurrence of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> declined. For the maxima, antioxidant potentials through DPPH and ABTS (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bii); the lowest IC<sub>50</sub> values (strong antioxidant) were reached at a solvent/sample ratio of 12 mL/g. Beyond 12 mL/g, the antioxidant potentials declined with increasing solvent volume. This is industrial valuable because reducing solvent usage that achieves higher recovery favour the production cost (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Bouras et al. <xref ref-type=\"bibr\" rid=\"CR11\">2015</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Moreover, the higher solvent volume tends to increase the concentration gradient; this can then lead to an increased diffusion rate of solute into the solvent. Therefore, 8–16 mL/g was chosen as a range of values for the optimization. However, 12 mL/g was considered as the fixed solvent/sample ratio when studying the impacts of other factors.</p></sec><sec id=\"Sec16\"><title>Contributing impact of microwave power</title><p id=\"Par31\">In microwave-enhanced extraction, the plant sample absorbs microwave energy, this energy changes to heat and evaporation kick-starts from the left-over moisture content in the plant matrix. The evaporated water then causes the pressure body-up within the cell wall, leading to cell rupture in the plant sample. Thus, the bioactive compounds within the plant matrix will leach out into the surrounding solvent after cell rupture (Mandal et al. <xref ref-type=\"bibr\" rid=\"CR23\">2007</xref>).</p><p id=\"Par32\">In this, the impact of microwave power on the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves was investigated by fixing the following factors: extraction time (120 s), solvent/sample ratio (12 mL/g), ethanol concentration (20%), and temperature (20 °C) (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ci and Cii). There was a slight increase in the TPC recovery as the power increased. At microwave power of 300 W, maximum TPC was achieved, after which the recovery of phenolic compounds tends to decrease against increasing microwave power. In contrast, the antioxidants (IC<sub>50</sub> values) for both DPPH and ABTS declined with increasing microwave power. This is justifiable because a lower IC<sub>50</sub> value indicates higher antioxidant potential. The minima IC<sub>50</sub> values for DPPH and ABTS were obtained at 300 W of microwave power. Beyond 300 W, the phenolic compounds and antioxidants declined with increasing power. Increasing the microwave power can propel increasing temperature. At higher temperatures, the viscosity and surface tension of extracting solvent will be reduced as the diffusion coefficient and solubility increase. Nonetheless, overexposure of the bioactive compounds already leached out into the solvent to higher microwave power propels the degradation of these phenolic compounds and antioxidants (thermal-sensitive phytochemicals) (Quoc <xref ref-type=\"bibr\" rid=\"CR26\">2021</xref>). Thus, 200–400 W was selected as a range of values for the optimization. Nevertheless, 300 W was considered as the fixed microwave power when studying the remaining factors.</p></sec><sec id=\"Sec17\"><title>Contributing impact of ethanol concentration</title><p id=\"Par33\">Organic solvents are mostly used in the extraction of phenolic compounds and antioxidants from plant materials. These solvents are mostly mixed with water because it enhances the water-soluble phytochemicals in the plant sample. Moreover, the mixture of organic solvents with water tends to create a moderate polar medium. Out of several organic solvents used in extraction, ethanol has been considered safe because it is environment-friendly (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>).</p><p id=\"Par34\">The influence of ethanol concentration was investigated by fixing extraction time, microwave power, solvent/sample ratio, and temperature at 120 s, 300 W, 12 mL/g, and 20 °C, respectively. As provided in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Di and Dii, the recovery of TPC tends to increase and IC<sub>50</sub> values (showing higher antioxidants) declined as ethanol concentration in water increased until the maximum recoveries of TPC and antioxidants (both DPPH and ABTS) were achieved ethanol concentration of 40%. Beyond this concentration, the phenolic compounds and antioxidants in the leaf extract of <italic>C. olitorius</italic> declined. Hence, the ethanol concentration ranged between 30 and 50% for the optimization. However, 40% was considered the fixed concentration for the next factor.</p></sec><sec id=\"Sec18\"><title>Contributing impact of temperature</title><p id=\"Par35\">In solid–liquid extraction, the higher temperature tends to upsurge the release of phenolic compounds and antioxidants from the plant matrix. This is due to its influence on mass transfer (diffusion coefficient), equilibrium (solubility) and stability of phytochemicals (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Although, heating can soften the plant tissue and weaken the interactions of phenolic compounds; hence, it promotes the migration of bioactive compounds into the extracting solvent. Nonetheless, a higher temperature can cause degradation and oxidation of phenolic compounds and antioxidants in the plant sample.</p><p id=\"Par36\">At fixed extraction time (120 s), microwave power (300 W), solvent/sample ratio (12 mL/g), and an ethanol concentration of 40%, the impact of the temperature was as well studied. In Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ei and Eii, it can be observed that temperature did not influence the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. This might be because the increasing temperature beyond a certain limit can profuse solvent loss through augment and vaporization; leading to the destruction of the phenolic compounds and antioxidants (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>).</p></sec></sec><sec id=\"Sec19\"><title>Optimization study and modelling of microwave-enhanced extraction of <italic>C. olitorius</italic> leaves</title><sec id=\"Sec20\"><title>Model fittings</title><p id=\"Par37\">The results from OFAT experimental trials have established the impacts of extraction time, solvent/sample ratio, microwave power, and ethanol concentration. These four factors were optimized through microwave-enhanced extraction to recover higher yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. In this study, FCCCD was employed to optimize this process. This was used because it is the simplest form of RSM. A total of 30 experimental trials were generated (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). The obtained outcomes from the randomized experimental trials are presented in Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>. These results were further analysed using analysis of variance to investigate the contributing impacts of individual factors considered and their interactions. The effectiveness of the RSM model is best explained through the satisfaction of model performance by measuring the determination coefficients (<italic>R</italic><sup>2</sup>), <italic>F</italic>-values, lack of fit, coefficient of variation, adjusted <italic>R</italic><sup>2</sup>, adequate precision, and <italic>p</italic> values (the value less than 0.05 indicates significant and vice versa).<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Design of experiments for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Std</p></th><th align=\"left\"><p>Extraction time, <italic>X</italic><sub>1</sub> (s)</p></th><th align=\"left\"><p>Microwave power, <italic>X</italic><sub>2</sub> (W)</p></th><th align=\"left\"><p>Solvent/sample, <italic>X</italic><sub>3</sub> (mL/g)</p></th><th align=\"left\"><p>Ethanol concentration, <italic>X</italic><sub>4</sub> (%)</p></th><th align=\"left\"><p>TPC, <italic>Y</italic><sub>1</sub> (mg GAE/10 g d.b.)</p></th><th align=\"left\"><p>DPPH, <italic>Y</italic><sub>2</sub> (µg/mL)</p></th><th align=\"left\"><p>ABTS, <italic>Y</italic><sub>3</sub> (µg/mL)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.69 ± 4.07</p></td><td align=\"char\" char=\"±\"><p>113.79 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>74.13 ± 1.08</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.00 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>109.47 ± 2.12</p></td><td align=\"char\" char=\"±\"><p>70.24 ± 1.05</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.42 ± 4.29</p></td><td align=\"char\" char=\"±\"><p>125.98 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>86.68 ± 2.43</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.72 ± 3.00</p></td><td align=\"char\" char=\"±\"><p>108.50 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>69.22 ± 1.54</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.50 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>120.00 ± 1.06</p></td><td align=\"char\" char=\"±\"><p>80.37 ± 1.58</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.33 ± 1.09</p></td><td align=\"char\" char=\"±\"><p>119.60 ± 2.14</p></td><td align=\"char\" char=\"±\"><p>80.47 ± 1.99</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>329.92 ± 4.76</p></td><td align=\"char\" char=\"±\"><p>117.00 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>77.79 ± 2.19</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.14 ± 4.11</p></td><td align=\"char\" char=\"±\"><p>103.72 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>64.50 ± 1.56</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>342.17 ± 2.33</p></td><td align=\"char\" char=\"±\"><p>120.02 ± 2.28</p></td><td align=\"char\" char=\"±\"><p>80.83 ± 1.05</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>341.89 ± 2.36</p></td><td align=\"char\" char=\"±\"><p>99.92 ± 3.05</p></td><td align=\"char\" char=\"±\"><p>60.60 ± 2.11</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>336.78 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>94.48 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>55.25 ± 1.55</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>330.63 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>70.67 ± 1.65</p></td><td align=\"char\" char=\"±\"><p>31.36 ± 1.00</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 2.11</p></td><td align=\"char\" char=\"±\"><p>120.15 ± 2.04</p></td><td align=\"char\" char=\"±\"><p>81.00 ± 1.05</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>340.18 ± 2.23</p></td><td align=\"char\" char=\"±\"><p>107.99 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.66 ± 1.00</p></td></tr><tr><td align=\"left\"><p>15</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>332.82 ± 2.05</p></td><td align=\"char\" char=\"±\"><p>90.45 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>51.22 ± 1.02</p></td></tr><tr><td align=\"left\"><p>16</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>325.52 ± 2.99</p></td><td align=\"char\" char=\"±\"><p>70.29 ± 1.77</p></td><td align=\"char\" char=\"±\"><p>31.08 ± 1.03</p></td></tr><tr><td align=\"left\"><p>17</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>343.47 ± 4.22</p></td><td align=\"char\" char=\"±\"><p>101.04 ± 1.22</p></td><td align=\"char\" char=\"±\"><p>61.81 ± 1.06</p></td></tr><tr><td align=\"left\"><p>18</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.78 ± 3.08</p></td><td align=\"char\" char=\"±\"><p>85.28 ± 2.06</p></td><td align=\"char\" char=\"±\"><p>46.00 ± 1.08</p></td></tr><tr><td align=\"left\"><p>19</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>346.89 ± 2.17</p></td><td align=\"char\" char=\"±\"><p>102.02 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>62.82 ± 1.00</p></td></tr><tr><td align=\"left\"><p>20</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.51 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>85.20 ± 1.18</p></td><td align=\"char\" char=\"±\"><p>45.96 ± 0.99</p></td></tr><tr><td align=\"left\"><p>21</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.70 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>102.39 ± 2.35</p></td><td align=\"char\" char=\"±\"><p>63.10 ± 1.69</p></td></tr><tr><td align=\"left\"><p>22</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>339.54 ± 1.98</p></td><td align=\"char\" char=\"±\"><p>104.14 ± 1.23</p></td><td align=\"char\" char=\"±\"><p>64.93 ± 2.01</p></td></tr><tr><td align=\"left\"><p>23</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>333.32 ± 4.03</p></td><td align=\"char\" char=\"±\"><p>90.82 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>51.59 ± 1.03</p></td></tr><tr><td align=\"left\"><p>24</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.84 ± 1.34</p></td><td align=\"char\" char=\"±\"><p>29.58 ± 1.03</p></td></tr><tr><td align=\"left\"><p>25</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>344.00 ± 3.11</p></td><td align=\"char\" char=\"±\"><p>86.40 ± 2.07</p></td><td align=\"char\" char=\"±\"><p>47.30 ± 1.26</p></td></tr><tr><td align=\"left\"><p>26</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.74 ± 2.16</p></td><td align=\"char\" char=\"±\"><p>90.08 ± 2.02</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 1.00</p></td></tr><tr><td align=\"left\"><p>27</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.91 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>90.26 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>51.01 ± 1.00</p></td></tr><tr><td align=\"left\"><p>28</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.68 ± 3.27</p></td><td align=\"char\" char=\"±\"><p>90.01 ± 1.15</p></td><td align=\"char\" char=\"±\"><p>50.78 ± 1.22</p></td></tr><tr><td align=\"left\"><p>29</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.77 ± 1.97</p></td><td align=\"char\" char=\"±\"><p>90.09 ± 1.04</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 2.14</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.92 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>90.23 ± 2.22</p></td><td align=\"char\" char=\"±\"><p>51.00 ± 1.14</p></td></tr></tbody></table></table-wrap></p><p id=\"Par38\">Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> presents the results obtained for the analysis of variance in recovering phenolic compounds and antioxidants (DPPH and ABTS assays). It can be observed that the three models of the responses (TPC, DPPH and ABTS assays) are significant. Out of the four considered factors for the optimization study, microwave power and ethanol concentration are the most significant with a <italic>p</italic> value &lt; 0.0001, showing model significance at a 99.9% confidence level. Extraction time is also significant, whereas solvent/sample ratio was insignificant for the three responses. Moreover, the lack of fit for the models was far greater than 0.05; this shows that the models are significant. The <italic>R</italic><sup>2</sup> values of the TPC, DPPH and ABTS assays ranged between 0.98 and 0.99; these are indications that the predictive models can explain at least 98% of the result variations. The adjusted and predicted <italic>R</italic><sup>2</sup> values for the three predictive models were closer to 1, indicating significant correlations between the actual and predicted results (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>). The lower value of the coefficient of variation below 10% shows that the experimental results were reproducible and reliable. The obtained results indicate for CV values for TPC, and antioxidant assays (DPPH and ABTS) were 0.33, 2.04 and 3.44, respectively. It should be noted that the contributions of the quadratic models are notable. Thus, the predictive models for the phenolic compounds and antioxidant assays are well suitable for the prediction.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Analysis of variance for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"6\"><p>TPC</p></th><th align=\"left\" colspan=\"6\"><p>DPPH</p></th><th align=\"left\" colspan=\"6\"><p>ABTS</p></th></tr><tr><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th></tr></thead><tbody><tr><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>1856.45</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>132.60</p></td><td align=\"char\" char=\".\"><p>105.49</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6946.43</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>496.17</p></td><td align=\"char\" char=\".\"><p>121.69</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6922.90</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>494.49</p></td><td align=\"char\" char=\".\"><p>117.32</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"char\" char=\".\"><p>21.64</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"char\" char=\".\"><p>221.39</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"char\" char=\".\"><p>212.43</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"char\" char=\".\"><p>88.47</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"char\" char=\".\"><p>293.10</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"char\" char=\".\"><p>281.19</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"char\" char=\".\"><p>3.99</p></td><td align=\"left\"><p>0.0643</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"char\" char=\".\"><p>0.90</p></td><td align=\"left\"><p>0.3582</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"char\" char=\".\"><p>0.98</p></td><td align=\"left\"><p>0.3386</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"char\" char=\".\"><p>358.38</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"char\" char=\".\"><p>375.77</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"char\" char=\".\"><p>360.63</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"char\" char=\".\"><p>7.67</p></td><td align=\"left\"><p>0.0143</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"char\" char=\".\"><p>21.84</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"char\" char=\".\"><p>21.89</p></td><td align=\"left\"><p>0.0003</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"char\" char=\".\"><p>9.47</p></td><td align=\"left\"><p>0.0077</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"left\"><p>0.0276</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"char\" char=\".\"><p>5.81</p></td><td align=\"left\"><p>0.0292</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"char\" char=\".\"><p>10.83</p></td><td align=\"left\"><p>0.0049</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"char\" char=\".\"><p>25.45</p></td><td align=\"left\"><p>0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"char\" char=\".\"><p>26.23</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"char\" char=\".\"><p>4.93</p></td><td align=\"left\"><p>0.0422</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"char\" char=\".\"><p>27.96</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"char\" char=\".\"><p>26.94</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"char\" char=\".\"><p>103.93</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"char\" char=\".\"><p>201.06</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"char\" char=\".\"><p>196.65</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"char\" char=\".\"><p>5.44</p></td><td align=\"left\"><p>0.0340</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"char\" char=\".\"><p>0.022</p></td><td align=\"left\"><p>0.8829</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"char\" char=\".\"><p>0.017</p></td><td align=\"left\"><p>0.8990</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"char\" char=\".\"><p>10.27</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"char\" char=\".\"><p>15.19</p></td><td align=\"left\"><p>0.0014</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"char\" char=\".\"><p>14.52</p></td><td align=\"left\"><p>0.0017</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"char\" char=\".\"><p>0.89</p></td><td align=\"left\"><p>0.3605</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"char\" char=\".\"><p>18.12</p></td><td align=\"left\"><p>0.0007</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"char\" char=\".\"><p>17.56</p></td><td align=\"left\"><p>0.0008</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"char\" char=\".\"><p>37.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"char\" char=\".\"><p>142.87</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"char\" char=\".\"><p>137.75</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"char\" char=\".\"><p>85.01</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"char\" char=\".\"><p>45.27</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"char\" char=\".\"><p>44.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>18.86</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>1.26</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>61.16</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.08</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>63.22</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.21</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>16.10</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>1.61</p></td><td align=\"char\" char=\".\"><p>2.92</p></td><td align=\"left\"><p>0.1243</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>49.50</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>4.95</p></td><td align=\"char\" char=\".\"><p>2.12</p></td><td align=\"left\"><p>0.2103</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>52.42</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>5.24</p></td><td align=\"char\" char=\".\"><p>2.42</p></td><td align=\"left\"><p>0.1701</p></td></tr><tr><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>2.76</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>0.55</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>11.66</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.33</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>10.81</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.16</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>1875.30</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>7007.59</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>6986.12</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>0.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>2.04</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>3.44</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>118.47</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>368.81</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>389.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9899</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9913</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9910</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9806</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9831</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9825</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9368</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9474</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9443</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>28.799</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>39.281</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>38.612</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr></tbody></table><table-wrap-foot><p><italic>DF</italic> degree of freedom</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec21\"><title>Effects of microwave-enhanced extraction factors on TPC and antioxidant assays (DPPH and ABTS) in <italic>C. olitorius</italic> leaves</title><p id=\"Par39\">The influences of microwave-enhanced extraction factors on the phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. The phenolic compounds in the extract of <italic>C. olitorius</italic> leaves varied from 325.52 ± 2.99 to 346.89 ± 2.17 mg GAE/10 g d.b. (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>). The least TPC was obtained at extraction time of 180 s, microwave power of 400 W, solvent/sample ratio of 16 mL/g, and ethanol concentration of 50%, whereas the highest TPC was obtained at extraction time of 120 s, microwave power of 200 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 40%. The antioxidant assays (DPPH and ABTS) from the leaf extract of <italic>C. olitorius</italic> ranged from 68.84 ± 1.34 to 125.98 ± 1.16 µg/mL and 29.58 ± 1.03 to 86.68 ± 2.43 µg/mL, respectively. The least (extraction time of 60 s, microwave power of 400 W, solvent/sample ratio of 8 mL/g, and ethanol concentration of 30%) and highest (extraction time of 120 s, microwave power of 300 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%) yields of antioxidants from <italic>C. olitorius</italic> leaves were obtained. Analysis of variance analysis (ANOVA) results indicate that extraction time, microwave power and ethanol concentration significantly influenced the recoveries of phenolic compounds from <italic>C. olitorius</italic> leaves; however, the solvent/sample ratio was insignificant. The quadratic effects of all the factors except microwave power are significant (<italic>p</italic> value &lt; 0.05). Besides, the interactions between all the considered factors significantly impact the microwave-enhanced extraction of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves except for the interactions between solvent/sample ratio and ethanol concentration (<italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub>) on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves which were insignificant (<italic>p</italic> value &gt; 0.05). The quadratic model for the coded values of TPC, DPPH and ABTS assays is estimated from Eqs. (<xref ref-type=\"disp-formula\" rid=\"Equ2\">2</xref>–<xref ref-type=\"disp-formula\" rid=\"Equ4\">4</xref>).<disp-formula id=\"Equ2\"><label>2</label><alternatives><mml:math display=\"block\" id=\"Equ2_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>345.43</mml:mn><mml:mo>-</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.49</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.53</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>5</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.78</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.92</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>0.62</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.65</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.23</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>0.66</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>4.24</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>6.42</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ2_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{1} &amp; = &amp; 345.43 - 1.23X_{1} - 2.49X_{2} - 0.53X_{3} + 5X_{4} - 0.78X_{1} X_{2} - 0.86X_{1} X_{3} - 0.92X_{1} X_{4} \\\\ \\quad - 0.62X_{2} X_{3} - 2.86X_{2} X_{4} - 0.65X_{3} X_{4} - 2.23X_{1}^{2} - 0.66X_{2}^{2} - 4.24X_{3}^{2} - 6.42X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ2.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ3\"><label>3</label><alternatives><mml:math display=\"block\" id=\"Equ3_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>88.89</mml:mn><mml:mo>-</mml:mo><mml:mn>7.08</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.15</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.45</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.36</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.55</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.67</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.16</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.076</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.89</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.99</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.44</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ3_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{2} &amp; = &amp; 88.89 - 7.08X_{1} - 8.15X_{2} + 0.45X_{3} - 9.23X_{4} - 2.36X_{1} X_{2} + 1.23X_{1} X_{3} - 2.55X_{1} X_{4} \\\\ \\quad - 2.67X_{2} X_{3} - 7.16X_{2} X_{4} + 0.076X_{3} X_{4} + 4.89X_{1}^{2} + 5.34X_{2}^{2} + 14.99X_{3}^{2} - 8.44X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ3.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ4\"><label>4</label><alternatives><mml:math display=\"block\" id=\"Equ4_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>49.67</mml:mn><mml:mo>-</mml:mo><mml:mn>7.05</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.11</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.48</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.19</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.40</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.24</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.63</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.66</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.20</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.066</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.86</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.97</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.46</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ4_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{3} &amp; = &amp; 49.67 - 7.05X_{1} - 8.11X_{2} + 0.48X_{3} - 9.19X_{4} - 2.40X_{1} X_{2} + 1.24X_{1} X_{3} - 2.63X_{1} X_{4} \\\\ \\quad - 2.66X_{2} X_{3} - 7.20X_{2} X_{4} + 0.066X_{3} X_{4} + 4.86X_{1}^{2} + 5.34X_{2}^{2} + 14.97X_{3}^{2} - 8.46X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ4.gif\"/></alternatives></disp-formula>where <italic>X</italic><sub>1</sub> represents extraction time (s), <italic>X</italic><sub>2</sub> indicates microwave power (W), <italic>X</italic><sub>3</sub> represents solvent/sample ratio (mL/g), <italic>X</italic><sub>4</sub> is the ethanol concentration, <italic>Y</italic><sub>1</sub>, <italic>Y</italic><sub>2</sub> and <italic>Y</italic><sub>3</sub> are the responses TPC, DPPH and ABTS assays, accordingly.</p><p id=\"Par40\">The three-dimensional plot is the representation of regression equations to show the interaction between independent and dependent factors. This plot is presented in different shapes; this illustrates whether the factor or its mutual interaction is significant or not (Halee et al. <xref ref-type=\"bibr\" rid=\"CR15\">2020</xref>). The contributing impacts of individual factors and their mutual interactions on the phenolic compounds and antioxidants are presented using three-dimensional graphical representations in Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a–f, <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a–f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a–f. It should be noted that antioxidant increases with reduced IC<sub>50</sub> values (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the phenolic compounds from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig2_HTML.png\"/></fig><fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the DPPH assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig3_HTML.png\"/></fig><fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the ABTS assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig4_HTML.png\"/></fig></p><p id=\"Par41\">The contributing effect of interactions between microwave power and extraction time on TPC at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is shown in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a. A gradual increase in extraction time and a slight decline in microwave power slowly increase the yield of TPC. Nonetheless, beyond 131.37 s and 305.08 W, TPC started to decline. For Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b, increasing both solvent/sample and extraction time at a fixed microwave power of 300 W and ethanol concentration of 40% gradually increased the recovery of TPC from <italic>C. olitorius</italic> leaves. After the certain limits of microwave power of 305.08 W and extraction time of 131.37 s, the phenolic compounds declined. In Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c, TPC from <italic>C. olitorius</italic> leaves increases with increasing ethanol concentration and reduces the extraction time. After these factors reached 45.07% and 69.85 s, respectively, TPC gradually decreases. The interaction between solvent/sample ratio and microwave power is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d; increasing solvent/sample ratio, and reducing microwave power increased the recovery of phenolic compounds. At solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recovery of TPC decreased. For the interaction between ethanol concentration and extraction time (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c), increasing ethanol concentration and reducing microwave power favour increasing phenolic compounds (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>e). At the ethanol concentration and microwave power of 45.07% and 257.62 W, optimal yields of TPC and antioxidants were achieved; beyond these values, the recoveries declined. Finally, the effects of interactions between ethanol concentration and solvent/sample ratio on the recoveries of TPC from <italic>C. olitorius</italic> leaves (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>f). Reducing the solvent/sample ratio and increasing the ethanol concentration favour the recovery of optimal TPC from the considered sample. Hence, the optimal predicted TPC yield of 343.327 mg GAE/10 g d.b. at the microwave-enhanced extraction conditions including extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p><p id=\"Par42\">Moreover, the influence of interactions between microwave power and extraction time on antioxidants (DPPH and ABTS assays) in the leaf extract of <italic>C. olitorius</italic> at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a. Increasing the extraction time and a slight decrease in microwave power gradually increases the antioxidant yields. The antioxidants in leaf extract declined as the microwave power and extraction time reached 131.37 s and 305.08 W, respectively. Besides, the interaction between solvent/sample ratio and extraction time is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>b and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b showed that increasing solvent/sample ratio and declining extraction time favoured antioxidant recoveries from <italic>C. olitorius</italic> leaf at a fixed microwave power of 300 W and ethanol concentration of 40%. Beyond microwave power of 305.08 W and extraction time of 131.37 s, the antioxidants declined. For Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, the antioxidants increase with increasing ethanol concentration and extraction time; beyond 45.07% and 69.85 s, respectively, gradual declination of antioxidants was observed. Increasing solvent/sample ratio and reducing microwave power increased the recoveries of phenolic compounds (Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d and <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>d). On reaching solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recoveries of antioxidants declined. Similarly, in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, increasing ethanol concentration and reducing microwave power increase the recoveries of antioxidants from leaf extract of <italic>C. olitorius</italic> (Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>e and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>e). Beyond the ethanol concentration and microwave power of 45.07% and 257.62 W, respectively, the recoveries of antioxidants declined. In conclusion, the interactions between ethanol concentration and solvent/sample ratio on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves as presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>f reflected that increasing solvent/sample ratio and reducing ethanol concentration yielded increasing antioxidants. Thus, the optimal predicted antioxidant yields (both DPPH and ABTS assays) were 68.8396 and 29.6220 µg/mL, respectively, at the microwave-enhanced extraction conditions of extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p></sec><sec id=\"Sec22\"><title>Validations of the generated models</title><p id=\"Par43\">Based on the predicted values from the Design Expert® software, the optimal conditions to achieve phenolic compounds (TPC of 343.327 mg GAE/10 g d.b.) and antioxidants (IC<sub>50</sub> values of 68.8396 and 29.6220 µg/mL for DPPH and ABTS assays, respectively) from extracts from <italic>C. olitorius</italic> leaves were found at extraction time of 131.37 s, microwave power of 305.08 W, solvent/sample ratio of 11.67 mL/g, and ethanol concentration of 50% with the desirability of 0.942. The validation experiments were carried out using approximated values of the predicted values such as extraction time of 131 s, microwave power of 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%. Three replicate experiments were carried out using these conditions, and the mean was determined to affirm the generated outcomes from the software. The experimental trials amounted to TPC of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for DPPH and ABTS assays, respectively, at 131 s, 305 W, 12 mL/g, and 50% of extraction time, microwave power, solvent/sample ratio, and ethanol concentration, respectively. Using Chi-squared analysis, the results obtained indicated no significant difference between the experimental and predicted values from the software.</p><p id=\"Par44\">As earlier reported, some previous studies had evaluated the contents of total phenols and antioxidants in the leaf extract of <italic>C. olitorius</italic>; however, no study was reported on recovering phenolic compounds and antioxidants from this plant through microwave-enhanced technique. A recent finding reported the recoveries of total phenols (0.10–9.81 µ GA/g), DPPH inhibition (1.4–70.3%) and ABTS inhibition (83–93%) (Obeng et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). In another recent study, a total antioxidant capacity of 165.66 ± 1.30 mg ascorbic acid/g dry extract was reported in the leaf extract of <italic>C. olitorius</italic> (Ali et al. <xref ref-type=\"bibr\" rid=\"CR8\">2021</xref>). The TPC of 2.5–32.0 Mg TE/g was reported in a recent study; it was established that phenolic compounds and antioxidants from this plant are unaffected by cultivation treatments. Moreover, it had been reported that the mode of extraction can alter the phenolic compounds in the <italic>C. olitorius</italic> leaves (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). The antioxidant from <italic>C. olitorius</italic> leaves was reported to be 91.65 ± 0.15% using DPPH inhibition with the maceration extraction technique (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). The results from the current study established the importance of using the microwave-enhanced extraction method to recover optimal yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Other than this, this plant can further be used in combating food insecurity because it is rich in phenolic compounds.</p></sec></sec><sec id=\"Sec23\"><title>LC-QToF-MS analysis of the optimized extract from <italic>C. olitorius</italic> leaves</title><p id=\"Par45\">The use of LC-QToF-MS analysis to characterize phenolic compounds has been a current trend in natural product chemistry; this analysis has proven to be efficient in tentatively identifying phenolic compounds in diverse plant samples (Alara et al. <xref ref-type=\"bibr\" rid=\"CR5\">2018c</xref>, <xref ref-type=\"bibr\" rid=\"CR4\">2018b</xref>; Tan et al. <xref ref-type=\"bibr\" rid=\"CR29\">2020</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). Unifi software processed the data. Thus, the phenolic profile of the leaf extract of <italic>C. olitorius</italic> at optimized microwave-enhanced extraction conditions was characterized using the LC-QToF-MS system (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). The characterization and identification of phytochemicals in the leaf extract of <italic>C. olitorius</italic> were done by comparing retention time (RT), mass spectrometric (MS), and mass error between mass observed (which must be less than 10 ppm for viable identified compound) (Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). A total of fourteen phenolic compounds were identified in the extract of <italic>C. olitorius</italic> leaves as presented in Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>. The identified phytochemicals were 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose as presented in the Supplementary materials (Figs. S1–S14).<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Negative ionization plots for the identified chemical compounds in the extract from <italic>C. olitorius</italic> leaf</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig5_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>The tentative assignments of chemical compounds in the leaf extract of <italic>C. olitorius</italic></p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>S/N</p></th><th align=\"left\"><p>Chemical compound</p></th><th align=\"left\"><p>RT (min)</p></th><th align=\"left\"><p>[M-H]- (m/z)</p></th><th align=\"left\"><p>Mass error (ppm)</p></th><th align=\"left\"><p>Molecular formula</p></th><th align=\"left\"><p>Response</p></th><th align=\"left\"><p>Adducts</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>1-Galloyl-glucose</p></td><td align=\"char\" char=\".\"><p>0.58</p></td><td align=\"char\" char=\".\"><p>331.0673</p></td><td align=\"char\" char=\".\"><p>0.8</p></td><td align=\"left\"><p>C<sub>13</sub>H<sub>16</sub>O<sub>10</sub></p></td><td align=\"left\"><p>6533</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>1,3,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>0.59</p></td><td align=\"char\" char=\".\"><p>677.1499</p></td><td align=\"char\" char=\".\"><p>−1.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>3235</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>Procyanidin C-1</p></td><td align=\"char\" char=\".\"><p>0.61</p></td><td align=\"char\" char=\".\"><p>865.1981</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>45</sub>H<sub>38</sub>O<sub>18</sub></p></td><td align=\"left\"><p>105,272</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>4,4′,5,6-Tetrahydroxystilbene</p></td><td align=\"char\" char=\".\"><p>0.75</p></td><td align=\"char\" char=\".\"><p>289.0725</p></td><td align=\"char\" char=\".\"><p>2.5</p></td><td align=\"left\"><p>C<sub>14</sub>H<sub>12</sub>O<sub>4</sub></p></td><td align=\"left\"><p>6478</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>3,4,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>3.11</p></td><td align=\"char\" char=\".\"><p>677.1517</p></td><td align=\"char\" char=\".\"><p>0.7</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>7900</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>5-Desgalloylstachyurin</p></td><td align=\"char\" char=\".\"><p>4.00</p></td><td align=\"char\" char=\".\"><p>783.0674</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>3123</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>Sanguiin H-4</p></td><td align=\"char\" char=\".\"><p>5.31</p></td><td align=\"char\" char=\".\"><p>633.0731</p></td><td align=\"char\" char=\".\"><p>−0.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2263</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>Corilagin_1</p></td><td align=\"char\" char=\".\"><p>5.85</p></td><td align=\"char\" char=\".\"><p>679.0783</p></td><td align=\"char\" char=\".\"><p>−0.8</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2547</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>1-O-Galloylpedunculagin</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"char\" char=\".\"><p>981.0849</p></td><td align=\"char\" char=\".\"><p>−0.1</p></td><td align=\"left\"><p>C<sub>41</sub>H<sub>28</sub>O<sub>26</sub></p></td><td align=\"left\"><p>3356</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>Laevigatin A</p></td><td align=\"char\" char=\".\"><p>6.47</p></td><td align=\"char\" char=\".\"><p>847.0839</p></td><td align=\"char\" char=\".\"><p>−0.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>26</sub>O<sub>23</sub></p></td><td align=\"left\"><p>2292</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>Pedunculagin</p></td><td align=\"char\" char=\".\"><p>6.94</p></td><td align=\"char\" char=\".\"><p>829.0728</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>6165</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>2,4,6-Tri-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.40</p></td><td align=\"char\" char=\".\"><p>681.0937</p></td><td align=\"char\" char=\".\"><p>−1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>3128</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>1,3,6-Trigalloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.91</p></td><td align=\"char\" char=\".\"><p>681.0954</p></td><td align=\"char\" char=\".\"><p>1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2265</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>1,2,3,6-Tetra-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>8.14</p></td><td align=\"char\" char=\".\"><p>833.1060</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>28</sub>O<sub>22</sub></p></td><td align=\"left\"><p>5153</p></td><td align=\"left\"><p>–H</p></td></tr></tbody></table></table-wrap></p><p id=\"Par46\">In simplicity, identified 1-galloyl-glucose, 1-O-galloylpedunculagin, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose belong to tannins (naturally occurring polyphenols). Another identified tannin was 5-desgalloylstachyurin. These compounds are rich in antioxidants and can alleviate different forms of cancer and free radicals (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR37\">2009</xref>). Other identified compounds were 1,3,5-O-tricaffeoylquinic acid and 3,4,5-O-tricaffeoylquinic acid; these compounds were reported as polyphenols that are rich in antioxidants (Wang et al. <xref ref-type=\"bibr\" rid=\"CR33\">2017</xref>). Procyanidin C-1 belongs to polyflavonoids and bioflavonoids; this compound has been reported natural occurring antioxidant from a plant source (Xu et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>). The identified stilbene monomer in the leaf extract of <italic>C. olitorius</italic> was 4,4′,5,6-tetrahydroxystilbene. This phytochemical has been reported to possess diverse pharmacological features which antioxidants (Zhang and Chen <xref ref-type=\"bibr\" rid=\"CR36\">2018</xref>). Sanguiin H-4 had been reported to possess antioxidant properties (Lachowicz et al. <xref ref-type=\"bibr\" rid=\"CR19\">2020</xref>). Corilagin_1 and laevigatin A are compounds that are rich in antioxidants as well. Therefore, extract from <italic>C. olitorius</italic> leaf has been established as a source of phenolic compounds and natural antioxidants.</p></sec></sec><sec id=\"Sec24\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par47\">Seeking alternative sources of antioxidants, especially plant-based, will continue to be studied. Given this, this study investigated <italic>C. olitorius</italic> leaves as a source of phenolic compounds and antioxidants by considering a technique that can recover yields (microwave-enhanced extraction) using ethanol/water as extracting solvent. The optimized process established optimal recoveries of phenolic compounds and antioxidants which were higher compared to previously reported results on <italic>C. olitorius</italic> leaves. In addition to this, the phenolic compounds that were comprehensively identified in the extract (fourteen in total) ascertain <italic>C. olitorius</italic> leaves as great sources of antioxidants. This outcome is an indication that <italic>C. olitorius</italic> leaves can continue to be encouraged in daily consumption, a way of combating food insecurity in the world. Thus, future study can focus on isolation and investigation of the tentatively identified phenolic compounds. These can further explain the pharmacological importance of this plant sample.</p></sec></body><back><ack><title>Acknowledgements</title><p>The authors thankfully acknowledge Eastern Unity Technology, Malaysia for the financial support through the grant (UIC190806).</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par48\">We declare none.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abir</surname><given-names>RR</given-names></name><name><surname>Marjia</surname><given-names>M</given-names></name><name><surname>Rakhi</surname><given-names>NN</given-names></name><name><surname>Saha</surname><given-names>O</given-names></name><name><surname>Hossain</surname><given-names>MA</given-names></name><name><surname>Rahaman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">In vitro comparative analysis of antibacterial activity of different fractions of <italic>Corchorus capsularis</italic> and <italic>Corchorus olitorius</italic> leaves extracts</article-title><source>Bangladesh J Microbiol</source><year>2019</year><volume>36</volume><fpage>69</fpage><lpage>73</lpage><pub-id pub-id-type=\"doi\">10.3329/bjm.v36i2.45530</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolics from Hibiscus sabdariffa calyces: Kinetic modelling and process intensification</article-title><source>Ind Crops Prod</source><year>2019</year><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2019.05.053</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\"><italic>Vernonia cinerea</italic> leaves as the source of phenolic compounds, antioxidants, and anti-diabetic activity using microwave-assisted extraction technique</article-title><source>Ind Crop Prod</source><year>2018</year><volume>122</volume><fpage>533</fpage><lpage>544</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1Wru7nJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.06.034</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Metabolic profiling of flavonoids, saponins, alkaloids, and terpenoids in the extract from <italic>Vernonia cinerea</italic> leaf using LC-Q-TOF-MS</article-title><source>J Liq Chromatogr Relat Technol</source><year>2018</year><pub-id pub-id-type=\"doi\">10.1080/10826076.2018.1511995</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Hassan</surname><given-names>Z</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Dataset on LC-Q-TOF/MS tentative identification of phytochemicals in the extract of <italic>Vernonia amygdalina</italic> leaf through positive ionization</article-title><source>Data Br</source><year>2018</year><volume>21</volume><fpage>1686</fpage><lpage>1689</lpage><pub-id pub-id-type=\"doi\">10.1016/j.dib.2018.10.159</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ali</surname><given-names>HA</given-names></name><name><surname>Zain</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from Carica papaya leaves: an optimization study and LC-QTOF-MS analysis</article-title><source>Futur Foods</source><year>2021</year><volume>3</volume><fpage>100035</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XhtlSmurnO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fufo.2021.100035</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name></person-group><article-title xml:lang=\"en\">Extraction of phenolic compounds: a review</article-title><source>Curr Res Food Sci</source><year>2021</year><volume>4</volume><fpage>200</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Cjsb%2FK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.crfs.2021.03.011</pub-id><pub-id pub-id-type=\"pmid\">33899007</pub-id><pub-id pub-id-type=\"pmcid\">8058613</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ali</surname><given-names>MM</given-names></name><name><surname>Ahmed</surname><given-names>KS</given-names></name><name><surname>Hossain</surname><given-names>H</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Rokeya</surname><given-names>B</given-names></name><name><surname>Rahman</surname><given-names>MT</given-names></name><name><surname>Jahan</surname><given-names>IA</given-names></name><name><surname>Rahman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Total antioxidant capacity and profiling of polyphenolic compounds in jute leaves by hplc-dad</article-title><source>Food Res</source><year>2021</year><volume>5</volume><fpage>343</fpage><lpage>348</lpage><pub-id pub-id-type=\"doi\">10.26656/fr.2017.5(1).358</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Andersen</surname><given-names>LT</given-names></name><name><surname>Thilsted</surname><given-names>SH</given-names></name><name><surname>Nielsen</surname><given-names>BB</given-names></name><name><surname>Rangasamy</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Food and nutrient intakes among pregnant women in rural Tamil Nadu</article-title><source>South India Public Heal Nutr</source><year>2003</year><volume>6</volume><fpage>131</fpage><lpage>137</lpage><pub-id pub-id-type=\"doi\">10.1079/PHN2002367</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azwanida</surname><given-names>NN</given-names></name></person-group><article-title xml:lang=\"en\">A review on the extraction methods use in medicinal plants, principle</article-title><source>Strength Limit Med Aromat Plants</source><year>2015</year><volume>04</volume><fpage>3</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.4172/2167-0412.1000196</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bouras</surname><given-names>M</given-names></name><name><surname>Chadni</surname><given-names>M</given-names></name><name><surname>Barba</surname><given-names>FJ</given-names></name><name><surname>Grimi</surname><given-names>N</given-names></name><name><surname>Bals</surname><given-names>O</given-names></name><name><surname>Vorobiev</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Optimization of microwave-assisted extraction of polyphenols from Quercus bark</article-title><source>Ind Crops Prod</source><year>2015</year><volume>77</volume><fpage>590</fpage><lpage>601</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsF2gt73F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2015.09.018</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Che Sulaiman</surname><given-names>IS</given-names></name><name><surname>Basri</surname><given-names>M</given-names></name><name><surname>Fard Masoumi</surname><given-names>HR</given-names></name><name><surname>Chee</surname><given-names>WJ</given-names></name><name><surname>Ashari</surname><given-names>SE</given-names></name><name><surname>Ismail</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Effects of temperature, time, and solvent ratio on the extraction of phenolic compounds and the anti-radical activity of <italic>Clinacanthus nutans</italic> Lindau leaves by response surface methodology</article-title><source>Chem Cent J</source><year>2017</year><volume>11</volume><fpage>1</fpage><lpage>11</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVShtLzE</pub-id><pub-id pub-id-type=\"doi\">10.1186/s13065-017-0285-1</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheynier</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Phenolic compounds: from plants to foods</article-title><source>Phytochem Rev</source><year>2012</year><volume>11</volume><fpage>153</fpage><lpage>177</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXktFeqsro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11101-012-9242-8</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guzzetti</surname><given-names>L</given-names></name><name><surname>Panzeri</surname><given-names>D</given-names></name><name><surname>Ulaszewska</surname><given-names>M</given-names></name><name><surname>Sacco</surname><given-names>G</given-names></name><name><surname>Forcella</surname><given-names>M</given-names></name><name><surname>Fusi</surname><given-names>P</given-names></name><name><surname>Tommasi</surname><given-names>N</given-names></name><name><surname>Fiorini</surname><given-names>A</given-names></name><name><surname>Campone</surname><given-names>L</given-names></name><name><surname>Labra</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of dietary bioactive phenolic compounds and agricultural sustainability of an African leafy vegetable <italic>Corchorus olitorius</italic> L</article-title><source>Front Nutr</source><year>2021</year><volume>8</volume><fpage>1</fpage><lpage>10</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWqsrzI</pub-id><pub-id pub-id-type=\"doi\">10.3389/fnut.2021.667812</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halee</surname><given-names>A</given-names></name><name><surname>Supavititpatana</surname><given-names>P</given-names></name><name><surname>Ruttarattanamongkol</surname><given-names>K</given-names></name><name><surname>Jittrepotch</surname><given-names>N</given-names></name><name><surname>Rojsuntornkitti</surname><given-names>K</given-names></name><name><surname>Kongbangkerd</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Optimisation of the microwave-assisted extraction of natural antioxidants from defatted black rice bran of <italic>oryza sativa</italic> L. CV. homnin</article-title><source>J Microbiol Biotechnol Food Sci</source><year>2020</year><volume>9</volume><fpage>1134</fpage><lpage>1140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOksrrP</pub-id><pub-id pub-id-type=\"doi\">10.15414/JMBFS.2020.9.6.1134-1140</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Herrero</surname><given-names>M</given-names></name><name><surname>Plaza</surname><given-names>M</given-names></name><name><surname>Cifuentes</surname><given-names>A</given-names></name><name><surname>Ibáñez</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Extraction techniques for the determination of phenolic compounds in food</article-title><source>Compr Sampl Sample Prep</source><year>2012</year><volume>4</volume><fpage>159</fpage><lpage>180</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-12-381373-2.00132-0</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kamga</surname><given-names>RT</given-names></name><name><surname>Kouamé</surname><given-names>C</given-names></name><name><surname>Atangana</surname><given-names>AR</given-names></name><name><surname>Chagomoka</surname><given-names>T</given-names></name><name><surname>Ndango</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nutritional evaluation of five African indigenous vegetables</article-title><source>J Hortic Res</source><year>2013</year><volume>21</volume><fpage>99</fpage><lpage>106</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1SqsL%2FL</pub-id><pub-id pub-id-type=\"doi\">10.2478/johr-2013-0014</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khoddami</surname><given-names>A</given-names></name><name><surname>Wilkes</surname><given-names>MA</given-names></name><name><surname>Roberts</surname><given-names>TH</given-names></name></person-group><article-title xml:lang=\"en\">Techniques for analysis of plant phenolic compounds</article-title><source>Molecules</source><year>2013</year><volume>18</volume><fpage>2328</fpage><lpage>2375</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXjs12htrw%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules18022328</pub-id><pub-id pub-id-type=\"pmid\">23429347</pub-id><pub-id pub-id-type=\"pmcid\">6270361</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lachowicz</surname><given-names>S</given-names></name><name><surname>Oszmiański</surname><given-names>J</given-names></name><name><surname>Rapak</surname><given-names>A</given-names></name><name><surname>Ochmian</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Profile and content of phenolic compounds in leaves, flowers, roots, and stalks of <italic>Sanguisorba officinalis</italic> L. Determined with the LC-DAD-ESI- QTOF-MS/MS analysis and their in vitro antioxidant, antidiabetic, antiproliferative potency</article-title><source>Pharmaceuticals</source><year>2020</year><volume>13</volume><fpage>1</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVCgurrP</pub-id><pub-id pub-id-type=\"doi\">10.3390/ph13080191</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lobo</surname><given-names>V</given-names></name><name><surname>Patil</surname><given-names>A</given-names></name><name><surname>Phatak</surname><given-names>A</given-names></name><name><surname>Chandra</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Free radicals, antioxidants and functional foods: impact on human health</article-title><source>Pharmacogn Rev</source><year>2010</year><volume>4</volume><fpage>118</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BC387ivVOgtA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.4103/0973-7847.70902</pub-id><pub-id pub-id-type=\"pmid\">22228951</pub-id><pub-id pub-id-type=\"pmcid\">3249911</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Loumerem</surname><given-names>M</given-names></name><name><surname>Alercia</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Descriptors for jute (<italic>Corchorus olitorius</italic> L.)</article-title><source>Genet Resour Crop Evol</source><year>2016</year><volume>63</volume><fpage>1103</fpage><lpage>1111</lpage><pub-id pub-id-type=\"doi\">10.1007/s10722-016-0415-y</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahmud</surname><given-names>IA</given-names></name><name><surname>Mirghani</surname><given-names>SME</given-names></name><name><surname>Yusof</surname><given-names>F</given-names></name><name><surname>Al-khatib</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Effects of time, temperature, and solvent ratio on the extraction of non-extractable polyphenols with anticancer activity of barhi date palm kernels extracts using response surface methodology</article-title><source>J. Biotechnol. Eng.</source><year>2019</year><volume>1</volume><fpage>1</fpage><lpage>36</lpage><pub-id pub-id-type=\"doi\">10.20944/preprints201907.0055.v1</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mandal</surname><given-names>V</given-names></name><name><surname>Mohan</surname><given-names>Y</given-names></name><name><surname>Hemalath</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Microwave assisted extraction-an innovative and promising extraction tool for medicinal plant research</article-title><source>Pharmacogn Rev</source><year>2007</year><volume>1</volume><fpage>7</fpage><lpage>18</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXhtVSit7zI</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndlovu</surname><given-names>S</given-names></name><name><surname>Pullabhotla</surname><given-names>RVSR</given-names></name><name><surname>Ntuli</surname><given-names>NR</given-names></name></person-group><article-title xml:lang=\"en\">Response of <italic>Corchorus olitorius</italic> leafy vegetable to cadmium in the soil</article-title><source>Plants</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>13</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGnsLbE</pub-id><pub-id pub-id-type=\"doi\">10.3390/plants9091200</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Obeng</surname><given-names>E</given-names></name><name><surname>Kpodo</surname><given-names>FM</given-names></name><name><surname>Tettey</surname><given-names>CO</given-names></name><name><surname>Essuman</surname><given-names>EK</given-names></name><name><surname>Adzinyo</surname><given-names>OA</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant total phenols and proximate constituents of four tropical leafy vegetables</article-title><source>Sci African</source><year>2020</year><volume>7</volume><fpage>e00227</fpage><pub-id pub-id-type=\"doi\">10.1016/j.sciaf.2019.e00227</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quoc</surname><given-names>LPT</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from coffee (<italic>Coffea robusta</italic> L. Linden) bee pollen</article-title><source>Herb Pol</source><year>2021</year><volume>67</volume><fpage>37</fpage><lpage>44</lpage><pub-id pub-id-type=\"doi\">10.2478/hepo-2021-0015</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Racha</surname><given-names>A</given-names></name><name><surname>Yakoub</surname><given-names>O</given-names></name><name><surname>Abdehedi</surname><given-names>O</given-names></name></person-group><article-title xml:lang=\"en\">Flavonoids, phenols, antioxidant, and antimicrobial activities in various extracts from Tossa jute leave (<italic>Corchorus olitorus</italic> L.)</article-title><source>Ind Crops Prod</source><year>2018</year><volume>118</volume><fpage>206</fpage><lpage>213</lpage><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.03.047</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saliu</surname><given-names>JA</given-names></name><name><surname>Ademiluyi</surname><given-names>AO</given-names></name><name><surname>Boligon</surname><given-names>AA</given-names></name><name><surname>Oboh</surname><given-names>G</given-names></name><name><surname>Schetinger</surname><given-names>MRC</given-names></name><name><surname>Rocha</surname><given-names>JBT</given-names></name></person-group><article-title xml:lang=\"en\">Dietary supplementation of jute leaf (<italic>Corchorus olitorius</italic>) modulates hepatic delta-aminolevulinic acid dehydratase (δ-ALAD) activity and oxidative status in high-fat fed/low streptozotocin-induced diabetic rats</article-title><source>J Food Biochem</source><year>2019</year><volume>43</volume><fpage>e12949</fpage><pub-id pub-id-type=\"doi\">10.1111/jfbc.12949</pub-id><pub-id pub-id-type=\"pmid\">31368580</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tan</surname><given-names>HM</given-names></name><name><surname>Leong</surname><given-names>KH</given-names></name><name><surname>Song</surname><given-names>J</given-names></name><name><surname>Mohd Sufian</surname><given-names>NSF</given-names></name><name><surname>Mohd Hazli</surname><given-names>UHA</given-names></name><name><surname>Chew</surname><given-names>LY</given-names></name><name><surname>Kong</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant and LC-QToF-MS/MS analysis of polyphenols in polar and non-polar extracts from Strobilanthes crispus and Clinacanthus nutans</article-title><source>Int Food Res J</source><year>2020</year><volume>27</volume><fpage>903</fpage><lpage>914</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlynu7s%3D</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>J</given-names></name><name><surname>Dunshea</surname><given-names>FR</given-names></name><name><surname>Suleria</surname><given-names>HAR</given-names></name></person-group><article-title xml:lang=\"en\">LC-ESI-QTOF/MS characterization of phenolic compounds from medicinal plants (Hops and Juniper Berries) and their antioxidant activity</article-title><source>Foods</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>25</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVektLnE</pub-id><pub-id pub-id-type=\"doi\">10.3390/foods9010007</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tchabo</surname><given-names>W</given-names></name><name><surname>Ma</surname><given-names>Y</given-names></name><name><surname>Kwaw</surname><given-names>E</given-names></name><name><surname>Xiao</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Apaliya</surname><given-names>MT</given-names></name></person-group><article-title xml:lang=\"en\">Impact of extraction parameters and their optimization on the nutraceuticals and antioxidant properties of aqueous extract mulberry leaf</article-title><source>Int J Food Prop</source><year>2018</year><volume>21</volume><fpage>717</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXovFyrsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10942912.2018.1446025</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thoo</surname><given-names>YY</given-names></name><name><surname>Ho</surname><given-names>SK</given-names></name><name><surname>Liang</surname><given-names>JY</given-names></name><name><surname>Ho</surname><given-names>CW</given-names></name><name><surname>Tan</surname><given-names>CP</given-names></name></person-group><article-title xml:lang=\"en\">Effects of binary solvent extraction system, extraction time and extraction temperature on phenolic antioxidants and antioxidant capacity from mengkudu (<italic>Morinda citrifolia</italic>)</article-title><source>Food Chem</source><year>2010</year><volume>120</volume><fpage>290</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhsFClu7zO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2009.09.064</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Hwang</surname><given-names>SH</given-names></name><name><surname>Quispe</surname><given-names>YNG</given-names></name><name><surname>Arce</surname><given-names>PHG</given-names></name><name><surname>Lim</surname><given-names>SS</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of the antioxidant and aldose reductase inhibitory activities of extracts from Peruvian tea plant infusions</article-title><source>Food Chem</source><year>2017</year><volume>231</volume><fpage>222</fpage><lpage>230</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXltlOnurw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2017.03.107</pub-id><pub-id pub-id-type=\"pmid\">28450000</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>Q</given-names></name><name><surname>Fu</surname><given-names>Q</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>R</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Ju</surname><given-names>Z</given-names></name><name><surname>Campisi</surname><given-names>J</given-names></name><name><surname>Kirkland</surname><given-names>JL</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The flavonoid procyanidin C1 has senotherapeutic activity and increases lifespan in mice</article-title><source>Nat Metab</source><year>2021</year><volume>3</volume><fpage>1706</fpage><lpage>1726</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XotFKmsLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s42255-021-00491-8</pub-id><pub-id pub-id-type=\"pmid\">34873338</pub-id><pub-id pub-id-type=\"pmcid\">8688144</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Youssef</surname><given-names>AF</given-names></name><name><surname>Younes</surname><given-names>NA</given-names></name><name><surname>Youssef</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Genetic diversity in <italic>Corchorus olitorius</italic> L. revealed by morphophysiological and molecular analyses</article-title><source>Mol Biol Rep</source><year>2019</year><volume>46</volume><fpage>2933</fpage><lpage>2940</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmsFCnu7g%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11033-019-04754-2</pub-id><pub-id pub-id-type=\"pmid\">30887258</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Biological effects of tetrahydroxystilbene glucoside: an active component of a rhizome extracted from <italic>Polygonum multiflorum</italic></article-title><source>Oxid Med Cell Longev</source><year>2018</year><volume>2018</volume><fpage>3641960</fpage><pub-id pub-id-type=\"doi\">10.1155/2018/3641960</pub-id><pub-id pub-id-type=\"pmid\">30524653</pub-id><pub-id pub-id-type=\"pmcid\">6247474</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Kim</surname><given-names>SH</given-names></name><name><surname>Hagerman</surname><given-names>AE</given-names></name><name><surname>Lü</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer, anti-diabetic and other pharmacologic and biological activities of penta-galloyl-glucose</article-title><source>Pharm Res</source><year>2009</year><volume>26</volume><fpage>2066</fpage><lpage>2080</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptlaku7w%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11095-009-9932-0</pub-id><pub-id pub-id-type=\"pmid\">19575286</pub-id><pub-id pub-id-type=\"pmcid\">2822717</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec25\"><title>Supplementary Information</title><p id=\"Par49\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2771_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 934 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>ABTS</term><def><p id=\"Par2\">2,2′-Azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid)</p></def></def-item><def-item><term>ANOVA</term><def><p id=\"Par3\">Analysis of variance analysis</p></def></def-item><def-item><term>DNA</term><def><p id=\"Par4\">Deoxyribonucleic acid</p></def></def-item><def-item><term>DPPH</term><def><p id=\"Par5\">2,2-Diphenyl-1-picrylhydrazyl</p></def></def-item><def-item><term>FCCCD</term><def><p id=\"Par6\">Face-centred central composite design</p></def></def-item><def-item><term>LC-QToF-MS</term><def><p id=\"Par7\">Liquid chromatography-mass spectrometry quadrupole of flight</p></def></def-item><def-item><term>OFAT</term><def><p id=\"Par8\">One-factor-at-a-time</p></def></def-item><def-item><term>TPC</term><def><p id=\"Par9\">Total phenolic content</p></def></def-item><def-item><term>RSM</term><def><p id=\"Par10\">Response surface methodology</p></def></def-item><def-item><term>SDG 2</term><def><p id=\"Par11\">Sustainable development goals</p></def></def-item></def-list></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10934</journal-id><journal-id journal-id-type=\"doi\">10.1007/10934.1573-4854</journal-id><journal-title-group><journal-title>Journal of Porous Materials</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Porous Mater</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1380-2224</issn><issn pub-type=\"epub\">1573-4854</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10934-023-01452-0</article-id><article-id pub-id-type=\"manuscript\">1452</article-id><article-id pub-id-type=\"doi\">10.1007/s10934-023-01452-0</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Synthesis and characterization of polyurethane rigid foam by using feedstocks received from renewable and recyclable resources</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Sarim</surname><given-names>Muntajab</given-names></name><address><email>muntajabsarim@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><name><surname>Nikje</surname><given-names>Mir Mohammad Alavi</given-names></name><address><email>alavi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10934023014520_cor2\">b</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Dargahi</surname><given-names>Maryam</given-names></name><address><email>dargahi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411537.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 8608 1112</institution-id><institution content-type=\"org-division\">Department of Chemistry, Faculty of Science</institution><institution content-type=\"org-name\">Imam Khomeini International University (IKIU)</institution></institution-wrap><addr-line content-type=\"city\">Qazvin</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs10934023014520_cor2\"><label>b</label><email>alavi@sci.ikiu.ac.ir</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>30</volume><issue seq=\"25\">4</issue><fpage>1357</fpage><lpage>1357</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10934-023-01425-3\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10934_2023_Article_1452.pdf</meta-value></custom-meta><custom-meta><meta-name>output-medium</meta-name><meta-value>All</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Catalysis</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Characterization and Evaluation of Materials</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>The online version of the original article can be found at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p></notes></front><body><p id=\"Par1\">Correction to: <bold>Journal of Porous Materials</bold></p><p id=\"Par2\"><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p><p id=\"Par6\">In this article, the author name Maryam Dargah has been corrected to Maryam Dargahi.</p><p id=\"Par7\">The original article has been corrected.</p></body><back><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article></records><facets><facet name=\"subject\"><facet-value count=\"62325\">Chemistry</facet-value><facet-value count=\"18407\">Biotechnology</facet-value><facet-value count=\"16573\">Chemistry/Food Science, general</facet-value><facet-value count=\"14684\">Analytical Chemistry</facet-value><facet-value count=\"13061\">Physical Chemistry</facet-value><facet-value count=\"11941\">Polymer Sciences</facet-value><facet-value count=\"9755\">Organic Chemistry</facet-value><facet-value count=\"9616\">Bioorganic Chemistry</facet-value><facet-value count=\"8700\">Biomaterials</facet-value><facet-value count=\"8278\">Biochemistry, general</facet-value><facet-value count=\"7729\">Food Science</facet-value><facet-value count=\"7329\">Surfaces and Interfaces, Thin Films</facet-value><facet-value count=\"7293\">Characterization and Evaluation of Materials</facet-value><facet-value count=\"6410\">Inorganic Chemistry</facet-value><facet-value count=\"6305\">Microbiology</facet-value><facet-value count=\"5608\">Bioinformatics</facet-value><facet-value count=\"5262\">Applied Microbiology</facet-value><facet-value count=\"5254\">Proteomics</facet-value><facet-value count=\"4744\">Genetic Engineering</facet-value><facet-value count=\"4494\">Industrial Chemistry/Chemical Engineering</facet-value></facet><facet name=\"keyword\"><facet-value count=\"854\">Diffusion Coefficient</facet-value><facet-value count=\"698\">Liquid Mixtures</facet-value><facet-value count=\"608\">Electrochemistry</facet-value><facet-value count=\"540\">Static Dielectric Constant</facet-value><facet-value count=\"487\">Mass spectrometry</facet-value><facet-value count=\"450\">Infinite Dilution</facet-value><facet-value count=\"414\">Metabolic engineering</facet-value><facet-value count=\"413\">Binary Mixtures</facet-value><facet-value count=\"365\">Taylor Dispersion Technique</facet-value><facet-value count=\"357\">Binary Liquid Mixtures</facet-value><facet-value count=\"334\">Condensed Matter Physics</facet-value><facet-value count=\"323\">Cellulose</facet-value><facet-value count=\"307\">Adsorption</facet-value><facet-value count=\"275\">Nanoparticles</facet-value><facet-value count=\"263\">Fermentation</facet-value><facet-value count=\"239\">Cellulase</facet-value><facet-value count=\"239\">COVID-19</facet-value><facet-value count=\"222\">\n",
       "                Escherichia coli\n",
       "              </facet-value><facet-value count=\"219\">Kinetics</facet-value><facet-value count=\"214\">SARS-CoV-2</facet-value></facet><facet name=\"pub\"><facet-value count=\"6998\">Polymer Journal</facet-value><facet-value count=\"3049\">Microbial Cell Factories</facet-value><facet-value count=\"2562\">Journal of the American Society for Mass Spectrometry</facet-value><facet-value count=\"2500\">Biotechnology for Biofuels</facet-value><facet-value count=\"2489\">Diffusion in Gases, Liquids and Electrolytes</facet-value><facet-value count=\"2225\">Journal of Nanobiotechnology</facet-value><facet-value count=\"2155\">Analytical and Bioanalytical Chemistry</facet-value><facet-value count=\"2137\">Journal of The American Society for Mass Spectrometry</facet-value><facet-value count=\"1818\">Electrochemistry</facet-value><facet-value count=\"1682\">BMC Biotechnology</facet-value><facet-value count=\"1323\">Journal of Cheminformatics</facet-value><facet-value count=\"1223\">Chemistry Central Journal</facet-value><facet-value count=\"1215\">Nature Chemical Biology</facet-value><facet-value count=\"1195\">Journal of Thermal Analysis and Calorimetry</facet-value><facet-value count=\"1132\">Nature Chemistry</facet-value><facet-value count=\"939\">Communications Chemistry</facet-value><facet-value count=\"901\">3 Biotech</facet-value><facet-value count=\"894\">European Journal of Nutrition</facet-value><facet-value count=\"690\">Cellulose</facet-value><facet-value count=\"666\">Static Dielectric Constants of Pure Liquids and Binary Liquid Mixtures</facet-value></facet><facet name=\"year\"><facet-value count=\"3443\">2023</facet-value><facet-value count=\"4831\">2022</facet-value><facet-value count=\"4105\">2021</facet-value><facet-value count=\"3047\">2020</facet-value><facet-value count=\"3345\">2019</facet-value><facet-value count=\"5583\">2018</facet-value><facet-value count=\"3010\">2017</facet-value><facet-value count=\"4988\">2016</facet-value><facet-value count=\"3441\">2015</facet-value><facet-value count=\"2421\">2014</facet-value><facet-value count=\"2727\">2013</facet-value><facet-value count=\"2324\">2012</facet-value><facet-value count=\"1936\">2011</facet-value><facet-value count=\"1577\">2010</facet-value><facet-value count=\"1599\">2009</facet-value><facet-value count=\"1125\">2008</facet-value><facet-value count=\"908\">2007</facet-value><facet-value count=\"895\">2006</facet-value><facet-value count=\"540\">2005</facet-value><facet-value count=\"427\">2004</facet-value></facet><facet name=\"country\"><facet-value count=\"10038\">Germany</facet-value><facet-value count=\"8110\">United States</facet-value><facet-value count=\"7241\">China</facet-value><facet-value count=\"5990\">Japan</facet-value><facet-value count=\"3688\">Poland</facet-value><facet-value count=\"3077\">United Kingdom</facet-value><facet-value count=\"2795\">India</facet-value><facet-value count=\"1931\">South Korea</facet-value><facet-value count=\"1527\">Italy</facet-value><facet-value count=\"1441\">Spain</facet-value><facet-value count=\"1413\">Netherlands</facet-value><facet-value count=\"1402\">France</facet-value><facet-value count=\"1293\">Canada</facet-value><facet-value count=\"1225\">Sweden</facet-value><facet-value count=\"1167\">Russia</facet-value><facet-value count=\"1124\">Austria</facet-value><facet-value count=\"1024\">Iran</facet-value><facet-value count=\"1021\">Egypt</facet-value><facet-value count=\"813\">Switzerland</facet-value><facet-value count=\"780\">Australia</facet-value></facet><facet name=\"type\"><facet-value count=\"56083\">Journal</facet-value><facet-value count=\"6242\">Book</facet-value></facet></facets></response>"
      ]
     },
     "execution_count": 65,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "start = 1\n",
    "# url = f\"https://api.springernature.com/openaccess/jats?api_key={SPRINGER_API_KEY}&q=language%3Aen%20openaccess%3Atrue&s={start}&p=5\"\n",
    "url = f\"https://api.springernature.com/openaccess/jats?api_key=865eda9aa6b27bd84ff8832558fce532&q=subject%3AChemistry&s=1&p=10\"  \n",
    "\n",
    "r = requests.get(url, timeout=300)\n",
    "soup = BeautifulSoup(r.content, \"xml\")\n",
    "\n",
    "soup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "subject = soup.find('facet', {'name': 'subject'})\n",
    "subject = subject.find_all(\"facet-value\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<records><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02810-7</article-id><article-id pub-id-type=\"manuscript\">2810</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02810-7</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Evaluation of acyclovir adsorption on granular activated carbon from aqueous solutions: batch and fixed-bed parametric studies</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Gupta</surname><given-names>Anju</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-9330-1539</contrib-id><name><surname>Vyas</surname><given-names>Raj K.</given-names></name><address><email>rkvyas.chem@mnit.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023028107_cor2\">b</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444471.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1764 2536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering</institution><institution content-type=\"org-name\">Malaviya National Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">302017</addr-line><addr-line content-type=\"city\">Jaipur</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023028107_cor2\"><label>b</label><email>rkvyas.chem@mnit.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>13</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"40\">8</issue><fpage>4599</fpage><lpage>4612</lpage><history><date date-type=\"registration\"><day>3</day><month>4</month><year>2023</year></date><date date-type=\"received\"><day>13</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>1</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>13</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><sec><p id=\"Par1\">The present study is aimed to assess the adsorptive potential of carbonaceous material for the acyclovir (ACVR) removal from the aquatic environment using batch and fixed-bed processes. In batch mode, the impact of various process conditions (contact time, pH, adsorbent dose, initial ACVR concentration, and temperature) on ACVR adsorption was investigated. Experimental results revealed that Langmuir isotherm and the pseudo-second-order kinetic model adequately represent the ACVR adsorption mechanism, indicating homogeneous adsorption. The process was found exothermic and spontaneous. Thermodynamic studies concluded that adsorption is a result of both physisorption and chemisorption. To understand the dynamic regime for the design of large-scale column studies, experimental data obtained from breakthrough curve were fitted to various analytical kinetic models. Yan model followed by Thomas model demonstrated a greater correlation of breakthrough data, confirming that the results are significant and are in line with Langmuir isotherm and pseudo-second-order kinetic. G-AC exhibits sufficient adsorption capacity for ACVR. Hence, it is concluded that it can be used in a fixed-bed column in continuous mode for the treatment of ACVR-contaminated wastewater.</p></sec><sec id=\"ASec2\"><title>Graphical abstract</title><p id=\"Par2\"><fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"PNG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Figa_HTML.png\"/></fig></p></sec></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Granular material</kwd><kwd>Kinetics</kwd><kwd>Isotherms</kwd><kwd>Fixed-bed</kwd><kwd>Breakthrough curve</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2810.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02810-7\">https://doi.org/10.1007/s11696-023-02810-7</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par48\">Over the past few decades, the new strains of life-threatening pandemic viruses, including Nipah virus, severe acute respiratory syndrome (SARS) virus, Middle East respiratory syndrome (MERS), and the most recent one, coronavirus disease (COVID-19), have hit the world. Therefore, the viral infections are of the global concern. Researchers are exploring new promising antibodies, antiviral treatments, and ways to improve the efficacy of existing medications on a laboratory scale (Al-Otaibi et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>; Heidary et al. <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>; Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). Antiviral drugs have received a lot of attention recently (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR30\">2013</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). Acyclovir (ACVR) (Fig. S1), a synthetically prepared purine-based nucleoside analog, shows antiviral activities for the treatment of herpes simplex virus (HSV), hepatitis B, varicella-zoster virus, Epstein-Barr virus, chickenpox, etc. (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). ACVR is usually favored over other drugs because of its low cytotoxicity and high selectivity towards infected cells (Shetti et al. <xref ref-type=\"bibr\" rid=\"CR54\">2020</xref>). More than two-thirds of the population is infected by α-HSV (Tognarelli et al. <xref ref-type=\"bibr\" rid=\"CR57\">2019</xref>). ACVR reaches drinking water via several routes from domestic/clinic waste, bulk and formulation drug industries, etc. (Gupta et al. <xref ref-type=\"bibr\" rid=\"CR25\">2021</xref>). The simultaneous presence of a pertinent virus in living beings and the modest amount of ACVR can render the virus drug-resistant (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Its presence has been found in industrial effluents, surface water, drinking water, and wastewater influent/effluent stream(s) (Funke et al. <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>; Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR47\">2010</xref>; Mascolo et al. <xref ref-type=\"bibr\" rid=\"CR41\">2010</xref>). Even though ACVR is present at a low concentration ranging from ng/L to μg/L (Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>), its persistent nature and harmful effects on the aquatic environment necessitate the treatment of contaminated water (Russo et al. <xref ref-type=\"bibr\" rid=\"CR51\">2017</xref>). A high therapeutic dose is mainly recommended as it only lasts a brief time in the human body (Saifi et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>). As a result, water contamination and possible environmental issues with the widespread usage of ACVR should be seriously considered. Remediation of industrial wastewater before its discharge is more economical than treating low concentrated large volume of water among preventive routes. Consequently, to prevent the adverse effects of ACVR on the ecosystem, ACVR polluted wastewater must be treated before being discharged.</p><p id=\"Par49\">Adsorption (Hosseini Nasr et al. <xref ref-type=\"bibr\" rid=\"CR29\">2018</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>, <xref ref-type=\"bibr\" rid=\"CR32\">2014b</xref>), UV/H<sub>2</sub>O<sub>2</sub> (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>), catalytic ozonation (Bing et al. <xref ref-type=\"bibr\" rid=\"CR5\">2017</xref>), photocatalysis (Wu et al. <xref ref-type=\"bibr\" rid=\"CR58\">2018</xref>; Li et al. <xref ref-type=\"bibr\" rid=\"CR35\">2016</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR10\">2014</xref>), photolysis (Zhou et al. <xref ref-type=\"bibr\" rid=\"CR62\">2015</xref>), ozonation (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>), and biological treatment techniques (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR61\">2017</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>) have been attempted so far till date for the removal of ACVR from wastewater. AOP using oxone has been proven efficient (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>, <xref ref-type=\"bibr\" rid=\"CR38\">2020</xref>); however, the high cost of oxone used in the process makes it un-economical. Oxidation and biodegradation processes lead to more toxic intermediates like COFA and carboxy-ACV (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>). Bio-refractory compounds like ACVR on degradation get converted into more toxic intermediate product(s). For such compounds, adsorption appears to be the most efficient approach, as it does not degrade the compound. Most adsorption studies are carried out in batches, which do not provide precise scale-up information required for practical large-scale treatment. So far column studies on ACVR removal from wastewater have been scarcely reported.</p><p id=\"Par50\">Considering its high surface area and porous structure, granular activated carbon (G-AC) is an efficient adsorbent for ACVR. Hence, it is a viable treatment solution for eliminating ACVR from water. Additionally, G-AC is cost-effective, environmentally benign and easily available and can be used in a variety of treatment systems, including fixed-bed columns. Therefore, the present research was aimed to evaluate the performance of G-AC column for ACVR removal from the synthetically prepared aqueous solution of ACVR. The batch studies were also performed to know the effects of operating parameters on the removal efficiency of ACVR. The breakthrough curves were used to characterize the dynamic response of the fixed-bed column. Effects of initial ACVR concentration, G-AC-bed height, and flow rate on ACVR adsorption were evaluated as part of the column studies along with the analytical model studies. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p></sec><sec id=\"Sec2\"><title>Experimental</title><p id=\"Par51\">Chemicals/reagents, solution preparation, and analytical techniques used are described in supplementary file.</p><sec id=\"Sec3\"><title>Adsorbent characterization</title><p id=\"Par52\">The Brunauer–Emmett–Teller (BET) method was used to determine the specific surface area, pore volume, and pore size distribution of G-AC (Nova Touch LX2 gas sorption analyzer, Quantachrome Instruments). The scanning electron microscope (SEM) and energy-dispersive X-ray spectroscopy (EDX) were used to perform morphological characterization of the raw and spent G-AC sample (Nova Nanosem 450, FEITM).</p></sec><sec id=\"Sec4\"><title>Batch experiments</title><p id=\"Par53\">For determining the maximum removal of ACVR (dosage form, make: Ocuvir, FDC Limited, India) batch adsorption studies were conducted at varying initial concentrations of ACVR, adsorbent dose, pH of the solution, and temperature. A pre-determined amount of G-AC (make: Loba Chemie, India, LR grade) was mixed with 100 mL of synthetically prepared aqueous ACVR solution of known concentration for each group of experiments. Each batch was agitated in an orbital shaker at a fixed temperature and RPM. The solution was filtered after completion of each set, and the residual ACVR concentration was measured in a UV–Vis spectrophotometer.</p><p id=\"Par54\">The adsorption capacity of the G-AC at the time (t min) and the percent removal of ACVR were estimated using the following equations, respectively:<disp-formula id=\"Equa\"><alternatives><mml:math display=\"block\" id=\"Equa_Math\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mi>t</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mi>V</mml:mi></mml:mrow><mml:mi>W</mml:mi></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equa_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$q_{t} = \\frac{{\\left( {C_{0} - C_{f} } \\right) \\times V}}{W}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equa.gif\"/></alternatives></disp-formula><disp-formula id=\"Equb\"><alternatives><mml:math display=\"block\" id=\"Equb_Math\"><mml:mrow><mml:mo>%</mml:mo><mml:mrow><mml:mtext>Removal</mml:mtext><mml:mspace width=\"0.333333em\"/></mml:mrow><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mn>100</mml:mn></mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equb_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\% {\\text{Removal }} = \\frac{{\\left( {C_{O} - C_{f} } \\right) \\times 100}}{{C_{O} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equb.gif\"/></alternatives></disp-formula></p><p id=\"Par55\">Following equations were used to calculate standard deviation percent (SD%) and root-mean-square error (RMSE):<disp-formula id=\"Equc\"><alternatives><mml:math display=\"block\" id=\"Equc_Math\"><mml:mrow><mml:mtext>SD</mml:mtext><mml:mo>%</mml:mo><mml:mo>=</mml:mo><mml:mn>100</mml:mn><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub></mml:mrow></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equc_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{SD}}\\% = 100\\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)/q_{\\exp } } \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - 1}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equc.gif\"/></alternatives></disp-formula><disp-formula id=\"Equd\"><alternatives><mml:math display=\"block\" id=\"Equd_Math\"><mml:mrow><mml:mtext>RMSE</mml:mtext><mml:mo>=</mml:mo><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mtext>parameters</mml:mtext></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equd_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{RMSE}} = \\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)} \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - {\\text{parameters}}}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equd.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec5\"><title>Kinetic studies</title><p id=\"Par56\">Several kinetic models can test the experimental results and analyze the kinetic mechanism that governs the adsorption rate based on ACVR diffusion in adsorbent particles. The present work used pseudo-first and second-order, Elovich, and intra-particle diffusion kinetic models to evaluate experimental results. A batch size of 100 mL of 100 mg/L concentrated solution was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C.</p><p id=\"Par57\">Pseudo-first-order kinetic model deals with the adsorption potential of the adsorbent against an adsorbate. According to the pseudo-second-order kinetic model, physisorption and chemisorption contribute to adsorption (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>). Elovich model explains chemisorption (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). This model shows the declining adsorption rate with time as the vacant adsorbent sites get occupied by adsorbate molecules and as a result surface coverage increases (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The intraparticle diffusion model explains the adsorption mechanism. It is generally the last step to know about adsorption and type of diffusion within the adsorbent. Model equations used in kinetic studies are summarized in Table S1.</p><p id=\"Par58\">The low C value represents the boundary layer’s minimal influence (diffusional resistance) on adsorption (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The initial rate of adsorption h (mg/g/min) is determined using the following relation:<disp-formula id=\"Eque\"><alternatives><mml:math display=\"block\" id=\"Eque_Math\"><mml:mrow><mml:mi>h</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>k</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msubsup><mml:mi>q</mml:mi><mml:mrow><mml:mi>e</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:math><tex-math id=\"Eque_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$h = k_{2} q_{e}^{2}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Eque.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec6\"><title>Equilibrium studies</title><p id=\"Par59\">Adsorption equilibrium studies were carried out to study the adsorption phenomenon and to assess the ACVR interaction behavior with the adsorbent surface. For which solution with varying concentration was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C. Temkin, Freundlich, and Langmuir isotherm models were used to examine the equilibrium parameters. Freundlich isotherm reveals the heterogeneous nature of the sorption process (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). On the other hand, Langmuir isotherm typically shows the homogeneous nature of the adsorption process. It is assumed that adsorbents are mainly composed of fixed vacant sites, each adsorbing single adsorbate particle leading to a monolayer adsorption process (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Table S2 provides a summary of the model equations used in isotherm studies.</p><p id=\"Par60\">The nature of adsorption can be determined based on the value of ‘<italic>n</italic>’ (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Favorability of the process was checked using the separating factor (<italic>R</italic><sub><italic>L</italic></sub>):<disp-formula id=\"Equf\"><alternatives><mml:math display=\"block\" id=\"Equf_Math\"><mml:mrow><mml:msub><mml:mi>R</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mn>1</mml:mn><mml:mrow><mml:mn>1</mml:mn><mml:mo>+</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equf_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$R_{L} = \\frac{1}{{1 + K_{L} C_{0} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equf.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec7\"><title>Thermodynamic studies</title><p id=\"Par61\">To study the temperature dependence of adsorptive removal of ACVR and the spontaneity of the process, thermodynamic studies were conducted. Thermodynamic parameters, namely, Gibbs free energy and heat of adsorption, were also estimated using equations given in Table S3.</p></sec><sec id=\"Sec8\"><title>Column studies and data analysis</title><p id=\"Par62\">A glass column with an inner diameter of 1.25 cm and a height of 45 cm was used to conduct fixed-bed column studies (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). The required flow rate was maintained using a peristaltic pump. Nylon cloth was used at the bottom to stop any adsorbent loss, and glass beads were used on both sides of the column to provide mechanical support to the bed and for uniform distribution of the feed solution preventing channelling. All the experiments were performed at an ambient temperature of 30 ± 2 °C. Effects of various process parameters such as bed height, volumetric flow rate, and inlet ACVR concentration were studied. After predetermined intervals, samples were taken and examined to determine the ACVR concentration. The adsorption capacity and breakthrough time were estimated to assess column performance.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic diagram of a fixed-bed adsorption column used for ACVR removal</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig1_HTML.png\"/></fig></p><p id=\"Par63\">Fixed-bed analytical models, namely, Thomas, Adams–Bohart, Yoon–Nelson, Clark, and Yan models were used to characterize experimental breakthrough curves. Table S4 summarizes the model equations used in the study.</p></sec></sec><sec id=\"Sec9\"><title>Results and discussion</title><sec id=\"Sec10\"><title>Adsorbent characterization</title><p id=\"Par64\">BET surface area and pore volume of G-AC were found to be 470 m<sup>2</sup>/g and 0.23 cm<sup>3</sup>/g, respectively. The average pore diameter of G-AC (1.44 nm) suggests the prominent presence of supra-micropores in G-AC (0.7–2 nm) (Thommes <xref ref-type=\"bibr\" rid=\"CR56\">2010</xref>). Fig. S2 shows the nitrogen adsorption–desorption isotherm at 77 K and the G-AC pore size distribution curve.</p><p id=\"Par65\">The isotherm is a type IV isotherm, indicating the presence of mesoporous particles. Micro-mesoporous carbons usually feature H4 loops, which are closely attributed to the thin and slit-like pores (Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). The prominence of micropores causes the volume adsorbed to increase rapidly at lower relative pressure. The pore size distribution estimated by the BJH plot validated the presence of mesopores along with the micropores. Fig. S3 further summarizes the characteristics of G-AC, confirming that it is an efficacious adsorbent.</p><p id=\"Par66\">Higher magnification SEM micrograph (Fig. S3) of the G-AC sample confirmed its porous structure. Additionally, the EDX findings revealed the elemental analysis of G-AC. Fig. S3 clearly demonstrates the adsorption of ACVR on pores of G-AC.</p></sec><sec id=\"Sec11\"><title>Point of zero charge and pH effect</title><p id=\"Par67\">The liquid-phase pH significantly influences the adsorbent and their ability to adsorb, affecting the binding sites. The effect of pH on ACVR removal using G-AC was examined for 100 mg/L concentrated ACVR solution (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a). For a contact time of 75 min, q<sub>t</sub> increased from 22.75 to 24.62 mg/g on increasing pH from 3 to 7; further increasing pH to 11, q<sub>t</sub> decreased to 20.25 mg/g. From Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a, it can be seen that maximum removal was obtained at pH 7. The increased electrostatic affinity between the surface charge of G-AC and ACVR is most likely responsible for the improved adsorption capacity of the system at pH 7. As a result, possibly the G-AC and ACVR surface charges in the aqueous medium are in opposite directions. The point of zero charge has been further used to explain pH influence and to comprehend the interaction between G-AC and ACVR.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p><bold>a</bold> pH effect on ACVR removal at a contact time of 75 min, <bold>b</bold> Point of zero charge of G-AC and ACVR</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig2_HTML.png\"/></fig></p><p id=\"Par68\">The pH at which the surface of an adsorbent is believed to behave neutrally is referred to as the point of zero charge (pH<sub>ZPC</sub>). Hence, it is significant to determine the pH<sub>ZPC</sub> before conducting the adsorptive removal experiments involving ionic species adsorption from aqueous environments onto solid surfaces. The pH<sub>ZPC</sub> for G-AC and ACVR is illustrated in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b. The results show that the ACVR surface is more basic (pH<sub>ZPC</sub>: 8.2) than G-AC's surface (pH<sub>ZPC</sub>: 6.9). The ACVR and G-AC surfaces are charged positively at pH &lt; pH<sub>ZPC</sub> and negatively at pH &gt; pH<sub>ZPC</sub>. The surface of G-AC shows a negative charge at pH &gt; pH<sub>ZPC</sub>. As the initial pH of the solution is increased, the surplus hydroxide ions in the bulk liquid are generally counterbalanced by the G-AC released hydrogen ions. Furthermore, deprotonation of G-AC lowers the solution’s final pH and results in a net negative charge on the G-AC surface.</p><p id=\"Par69\">As discussed earlier, the G-AC surface is positively charged below pH<sub>ZPC</sub>, as consequence hydrogen ions compete with ACVR cations to reach the surface. The hydrogen ions will gradually surround major active sites on the G-AC surface. At pH &lt; 7, both G-AC and ACVR are positively charged which repelled from each other. As such, ACVR molecules were not effectively adsorbed on G-AC. Similar patterns for pH<sub>ZPC</sub> values for G-AC and ACVR are reported in the literature (Borah et al. <xref ref-type=\"bibr\" rid=\"CR7\">2008</xref>).</p></sec><sec id=\"Sec12\"><title>Effect of adsorbent dose</title><p id=\"Par70\">Experiments were carried out at varying doses of adsorbents ranging from 1 to 4 g/L at 25 °C for 2 h to investigate the effects of the G-AC dose on the ACVR removal efficiency (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>).<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Impact of varying adsorbent dose on ACVR removal</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig3_HTML.png\"/></fig></p><p id=\"Par71\">When adsorbent dose was increased from 1 to 4 g/L, there was a polynomial increase in the removal of ACVR from 39.7 to 95.2%. It can be seen from Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref> that there was a continuous rise in the percent removal of ACVR, but 4 g/L was taken as optimum as already 95% removal was being achieved with this dose. A reduction of 95.2% in ACVR concentration indicates that almost all of the vacant-active sites on the adsorbent surface are occupied at this point, and increasing the adsorbent dosage beyond this point will not be cost-effective.</p></sec><sec id=\"Sec13\"><title>Impact of contact time</title><p id=\"Par72\">At natural pH (~ 6.8), the impact of contact time on ACVR adsorption by G-AC was examined for 105 min. Figure <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a depicts the performance of adsorbents for the removal of ACVR with contact time.<fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p><bold>a</bold> Impact of Contact time (initial ACVR concentration: 100 mg/L) and <bold>b</bold> Impact of ACVR initial concentration on percent removal at equilibrium</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig4_HTML.png\"/></fig></p><p id=\"Par73\">The polynomial increase in percent removal was observed on increasing the contact time of adsorbent and ACVR. More than 50% of the removal was achieved within the initial 30 min using G-AC. Initially, the removal rate was rapid; however slow reduction was observed later. This could be due to more unoccupied sites on adsorbent are available at the initial stage with a high driving force which decreases with time and active sites also diminishes. Finally, for an initial ACVR concentration of 100 mg/L system reached the state of saturation at 75 min.</p></sec><sec id=\"Sec14\"><title>Adsorption kinetics</title><p id=\"Par74\">The adsorption kinetics of ACVR onto G-AC using pseudo-first- and -second-order, Elovich, and intra-particle diffusion kinetic models is depicted in Fig. S4. Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> summarizes the values of regression coefficients obtained and other kinetic parameters of ACVR adsorption.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Parameters for various kinetic models for ACVR removal</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Kinetic model</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Values</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Pseudo-first-order model</p></td><td align=\"left\"><p><italic>q</italic><sub>e,exp</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>24.62</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>k</italic><sub>1</sub> (min<sup>−1</sup>)</p></td><td align=\"char\" char=\".\"><p>0.04</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>8.98</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9899</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>8.1%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>7.82</p></td></tr><tr><td align=\"left\"><p>Pseudo-second-order model</p></td><td align=\"left\"><p><italic>k</italic><sub>2</sub> (g/mg/min)</p></td><td align=\"char\" char=\".\"><p>0.01</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>25.58</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9991</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>h</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>7.20</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>0.02%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>0.43</p></td></tr><tr><td align=\"left\"><p>Elovich kinetic model</p></td><td align=\"left\"><p><italic>α</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>296.61</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>β</italic> (g/mg)</p></td><td align=\"char\" char=\".\"><p>0.36</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9771</p></td></tr><tr><td align=\"left\"><p>Intraparticle diffusion model</p></td><td align=\"left\"><p><italic>K</italic><sub>d</sub> (mg/g/min<sup>0.5</sup>)</p></td><td align=\"char\" char=\".\"><p>1.12</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>C</italic> (mg/g)</p></td><td align=\"char\" char=\".\"><p>15.48</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9579</p></td></tr></tbody></table></table-wrap></p><p id=\"Par75\">The significance of studying the adsorption kinetics of ACVR onto G-AC using different kinetic models lies in understanding the rate and mechanism of the adsorption process. The comparison of the results obtained from different models can help determine the most suitable kinetic model for describing the adsorption process and provide insights into the mechanism of adsorption. Kinetic data obtained in the present study are significant and indicate that the adsorption process satisfies pseudo-second-order kinetic model.</p><p id=\"Par76\">The smaller difference in the equilibrium amount adsorbed between experimental and calculated values confirms the pseudo-second-order behavior of the process (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Moreover, the slightly lower correlation coefficient (<italic>R</italic><sup>2</sup>) also shows the inappropriateness of the pseudo-first-order kinetic model. The adsorption kinetics of numerous bio-refractory pollutant-adsorbent systems have exhibited comparable kinetic results (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p><p id=\"Par77\">If the value of C is not zero for any adsorption process, the intraparticle diffusion is not the sole controlling mechanism (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). Thus, it can be concluded that during ACVR sorption on G-AC, film and intraparticle diffusion occur simultaneously (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>).</p></sec><sec id=\"Sec15\"><title>Influence of ACVR initial concentration</title><p id=\"Par78\">It is vital to investigate the effect of the initial concentration of the adsorbate on the sorption rate for effective adsorption. The variation of the adsorptive removal efficiency of G-AC at different initial concentrations of ACVR (100–400 mg/L) is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b.</p><p id=\"Par79\">It is evident from Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b that as the drug concentration rise, so does the quantity of adsorbed ACVR. This could be because the difference between available adsorbent sites and the initial number of ACVR molecules is high at higher concentrations. Therefore, an initial concentration of 100 mg/L ACVR was used in the present study; moreover, further change in percent removal was significantly less. Additionally, it is common that the lower the initial adsorbate concentration, the higher removal efficiency while the other experimental conditions were kept constant.</p></sec><sec id=\"Sec16\"><title>Adsorption isotherms</title><p id=\"Par80\">Fig. S5 depicts the adsorption isotherms (Freundlich, Langmuir, and Temkin) and Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> shows the isotherm parameters for the ACVR adsorption onto G-AC.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Isotherm parameters for ACVR sorption</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Particulars</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Value</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Freundlich isotherm</p></td><td align=\"left\"><p><italic>K</italic><sub>F</sub> (mg/g)/(L/mg)<sup>n</sup></p></td><td align=\"left\"><p>24.51</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>n</italic></p></td><td align=\"left\"><p>2.37</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9293</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>12.69</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>8.72</p></td></tr><tr><td align=\"left\"><p>Langmuir isotherm</p></td><td align=\"left\"><p><italic>Q</italic><sub>m</sub> (mg/g)</p></td><td align=\"left\"><p>105.26</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>K</italic><sub>L</sub> (L/mg)</p></td><td align=\"left\"><p>0.21</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9981</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>3.46%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>2.35</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sub>L</sub></p></td><td align=\"left\"><p>0 &lt; <italic>R</italic><sub>L</sub> &lt; 1</p></td></tr><tr><td align=\"left\"><p>Temkin isotherm</p></td><td align=\"left\"><p><italic>A</italic><sub>T</sub> (L/g)</p></td><td align=\"left\"><p>2.10</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>b</italic><sub>T</sub></p></td><td align=\"left\"><p>109.93</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>B</italic><sub>T</sub> (mg/g)</p></td><td align=\"left\"><p>22.54</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9831</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>16.865</p></td></tr></tbody></table></table-wrap></p><p id=\"Par81\">The highest <italic>R</italic><sup>2</sup> value confirms the Langmuir adsorption isotherm model’s good fit to the experimental data. In the present work, the value of <italic>R</italic><sub><italic>L</italic></sub> was found 0.045 indicating the ACVR adsorption to be a favorable process. Furthermore, the low <italic>R</italic><sub><italic>L</italic></sub> value suggests that ACVR interaction with adsorbents is strong (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). The value of the heterogeneity factor also confirms the favorability of the process, and the process follows the Langmuir isotherm (0 &lt; 1/<italic>n</italic> &lt; 1) (Fytianos et al. <xref ref-type=\"bibr\" rid=\"CR22\">2000</xref>). It also signifies that monolayer chemisorption occurs with uniform adsorptive sites and energy (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>). The increase in adsorption capacity indicates weak interaction between adsorbed ACVR molecules (Hasnain Isa et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). Similar results are reported for numerous pollutant-adsorbent systems (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p></sec><sec id=\"Sec17\"><title>Thermodynamic studies</title><p id=\"Par82\">Adsorption studies at varying temperatures were conducted to examine the influence of temperature on ACVR uptake (Fig. S6). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> highlights the thermodynamic parameters for ACVR adsorption onto G-AC. Both 1/<italic>T</italic> and ln (<italic>q</italic><sub><italic>e</italic></sub>/<italic>C</italic><sub><italic>e</italic></sub>) values decrease with rise in temperature, indicating that the adsorption process is exothermic. Following equations were used to calculate change in Gibbs free energy,<disp-formula id=\"Equg\"><alternatives><mml:math display=\"block\" id=\"Equg_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mo>-</mml:mo><mml:mi>R</mml:mi><mml:mi>T</mml:mi><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:math><tex-math id=\"Equg_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = - RT\\ln K_{0}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equg.gif\"/></alternatives></disp-formula><disp-formula id=\"Equh\"><alternatives><mml:math display=\"block\" id=\"Equh_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>-</mml:mo><mml:mi>T</mml:mi><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mo>∘</mml:mo></mml:msup></mml:mrow></mml:math><tex-math id=\"Equh_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = \\Delta H^\\circ - T\\Delta S^\\circ$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equh.gif\"/></alternatives></disp-formula><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Thermodynamic parameters for ACVR adsorption onto G-AC</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Temperature (°C)</p></th><th align=\"left\"><p>∆<italic>H</italic><sup>0</sup> (kJ/mol)</p></th><th align=\"left\"><p>∆<italic>S</italic><sup>0</sup> (kJ/mol/K)</p></th><th align=\"left\"><p>∆<italic>G</italic><sup>0</sup> (kJ/mol)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>25</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 56.88</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 0.168</p></td><td align=\"char\" char=\".\"><p>− 6.93</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"char\" char=\".\"><p>− 6.04</p></td></tr><tr><td align=\"left\"><p>35</p></td><td align=\"char\" char=\".\"><p>− 4.59</p></td></tr><tr><td align=\"left\"><p>40</p></td><td align=\"char\" char=\".\"><p>− 4.19</p></td></tr><tr><td align=\"left\"><p>45</p></td><td align=\"char\" char=\".\"><p>− 3.38</p></td></tr><tr><td align=\"left\"><p>50</p></td><td align=\"char\" char=\".\"><p>− 2.64</p></td></tr><tr><td align=\"left\"><p>55</p></td><td align=\"char\" char=\".\"><p>− 1.78</p></td></tr></tbody></table></table-wrap></p><p id=\"Par83\">Combining above equations,<disp-formula id=\"Equi\"><alternatives><mml:math display=\"block\" id=\"Equi_Math\"><mml:mrow><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>=</mml:mo><mml:mo>ln</mml:mo><mml:mfrac><mml:msub><mml:mi>q</mml:mi><mml:mi>e</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mi>e</mml:mi></mml:msub></mml:mfrac><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mi>R</mml:mi></mml:mfrac><mml:mo>-</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equi_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\ln K_{0} = \\ln \\frac{{q_{e} }}{{C_{e} }} = \\frac{{\\Delta S^{0} }}{R} - \\frac{{\\Delta H^{0} }}{RT}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equi.gif\"/></alternatives></disp-formula></p><p id=\"Par84\">The heat of adsorption was negative, indicating exothermic nature for ACVR sorption onto G-AC. ACVR is soluble in water and may lose some of its hydration molecules, releasing heat. The spontaneous physisorption process is further established by the ∆<italic>G</italic>° value (0–20 kJ/mol) (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>) (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Further, as the temperature rises from 25 to 55 °C, ∆<italic>G</italic>° increases, indicating that the feasibility of the process declines with temperature rise (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>; Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The favorability of the process was also confirmed with the value of <italic>R</italic><sub><italic>L</italic></sub> reported in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>. Decrease in <italic>K</italic><sub>0</sub> value with temperature also confirms the exothermic nature of ACVR adsorption (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Moreover, the value of ∆<italic>S</italic>° was negative, showing that the ACVR adsorption was enthalpy driven (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Several pollutant-adsorbent systems have also produced similar results (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Since the Langmuir isotherm supports chemisorption while ∆<italic>G</italic>° (0–20 kJ/mol) supports physisorption. This was further clarified using ∆H° which indicates physisorption is being accompanied by chemisorption. This concludes that the adsorption of ACVR onto G-AC is in the transition phase from physisorption to chemisorption.</p></sec><sec id=\"Sec18\"><title>Effect of volumetric flow rate</title><p id=\"Par85\">The breakthrough response for ACVR adsorption was examined at various volumetric flow rates (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). As the volumetric flow rate rises from 3.1 to 3.8 mL/min, the breakthrough times at C<sub>t</sub>/C<sub>0</sub> = 0.1 shifted to 75 min from 120 min. It can be explained as at a high flow rate, the contact time between the G-AC and ACVR reduces, limiting the bed's adsorption capacity and processing times (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>). Furthermore, at a higher flow rate, the breakthrough curve shifts near the origin, become steeper, and reaches saturation more quickly.<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Effect of volumetric flow rate</p></caption><graphic id=\"MO6\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig5_HTML.png\"/></fig></p><p id=\"Par86\">Figure <xref ref-type=\"fig\" rid=\"Fig5\">5</xref> demonstrates that C<sub>t</sub>/C<sub>0</sub> increased from 0.12 to 0.32 at 3.1 and 4.5 mL/min flow rates, respectively, after 2 h. ACVR has more time to permeate through bed when the flow rate decreases, resulting in an increased column uptake. Likewise, higher volumetric flow rate diminishes the thickness of fluid film surrounding the sorbent particles, reducing mass transfer resistance and a quick increase in mass transfer (Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>). Similar trends have been reported in previous studies for the adsorptive removal of various pharmaceutical drugs (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec19\"><title>Effect of bed height</title><p id=\"Par87\">Figure <xref ref-type=\"fig\" rid=\"Fig6\">6</xref> presents breakthrough curves data at varying bed depths for a 200 mg/L concentrated solution and a volumetric flow rate of 3.1 mL/min.<fig id=\"Fig6\"><label>Fig. 6</label><caption xml:lang=\"en\"><p>Effect of bed height</p></caption><graphic id=\"MO7\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig6_HTML.png\"/></fig></p><p id=\"Par88\">The breakthrough time and exhaustion time decreased on reducing bed depth, moving the breakthrough curve towards the origin, as shown in Fig. <xref ref-type=\"fig\" rid=\"Fig6\">6</xref>. Also, C<sub>t</sub>/C<sub>0</sub> for a short-fixed bed rises fast than for a long bed. Breakthrough concentrations (C<sub>t</sub>/C<sub>0</sub> = 0.1) were attained at contact times of 2 h and 7 h for 3.5 and 12 cm of bed heights, respectively. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub>0</sub> varied from 0.59 and 0.11 at bed heights of 3.5 and 12 cm, respectively, after 7 h. With a greater bed height, more adsorbent load is used resulting in more adsorption sites and a longer contact time to attain 100% saturation (Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Furthermore, as the bed height rises, the adsorbate has considerable time for diffusion through the pores of the adsorbent, increasing the overall adsorbed amount and ACVR removal efficiency (Reynel-Avila et al. <xref ref-type=\"bibr\" rid=\"CR50\">2015</xref>). Similar results for the adsorption of several pharmacological drugs have been reported in the literature (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec20\"><title>Impact of inlet ACVR concentration</title><p id=\"Par89\">Figure <xref ref-type=\"fig\" rid=\"Fig7\">7</xref> shows that the inlet ACVR concentration considerably impacts the breakthrough curve.<fig id=\"Fig7\"><label>Fig. 7</label><caption xml:lang=\"en\"><p>Effect of inlet ACVR concentration</p></caption><graphic id=\"MO8\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig7_HTML.png\"/></fig></p><p id=\"Par90\">Time to reach breakthrough and exhaustion increased as the inlet ACVR concentration decreased. The breakthrough curve moved away from the origin at lower inlet concentration. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub><italic>0</italic></sub> varied from 0.03 to 0.10 at inlet concentrations of 200 and 300 mg/L, respectively, at a contact time of 90 min. This phenomenon may be explained by an increase in the sorption rate and the mass transfer driving force across the liquid film, which causes the fixed bed to saturate more quickly. These study results on the impact of inflow concentration on breakthrough curves concur with other bio-refractory compounds reported literature studies (Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>).</p></sec><sec id=\"Sec21\"><title>Thomas model</title><p id=\"Par91\">The Thomas model investigates how maximum column uptake and breakthrough curves are evaluated. This model emphasizes that the process follows Langmuir kinetics, and there is no axial dispersion (Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). It is also derived under the presumption that the rate of the driving force is subject to reversible 2nd order kinetics. It is advisable for the adsorption system when there is essentially no resistance to internal diffusion (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> provides more information on the obtained <italic>k</italic><sub>TH</sub>, <italic>q</italic><sub>TH</sub>, and <italic>R</italic><sup>2</sup> values at various process conditions (Fig. S7).<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>Parameters of kinetic analytical models for ACVR adsorption for different sets of process conditions</p></caption><table frame=\"hsides\" rules=\"groups\"><tbody><tr><td align=\"left\" rowspan=\"3\"><p>Process conditions</p></td><td align=\"left\"><p><italic>Z</italic> (cm)</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>8</p></td><td align=\"left\"><p>12</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td></tr><tr><td align=\"left\"><p><italic>Q</italic> (mL/min)</p></td><td align=\"left\"><p>3.8</p></td><td align=\"left\"><p>4.5</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td></tr><tr><td align=\"left\"><p><italic>C</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>300</p></td><td align=\"left\"><p>400</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Thomas model</p></td><td align=\"left\"><p>k<sub>TH</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>3.15</p></td><td align=\"left\"><p>4.45</p></td><td align=\"left\"><p>2.20</p></td><td align=\"left\"><p>1.35</p></td><td align=\"left\"><p>2.90</p></td><td align=\"left\"><p>1.97</p></td><td align=\"left\"><p>1.57</p></td></tr><tr><td align=\"left\"><p>q<sub>TH</sub> (mg/g)</p></td><td align=\"left\"><p>211.81</p></td><td align=\"left\"><p>199.54</p></td><td align=\"left\"><p>248.98</p></td><td align=\"left\"><p>242.69</p></td><td align=\"left\"><p>224.91</p></td><td align=\"left\"><p>302.20</p></td><td align=\"left\"><p>312.93</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"4\"><p>Yoon–Nelson model</p></td><td align=\"left\"><p><italic>k</italic><sub>YN</sub> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0063</p></td><td align=\"left\"><p>0.0089</p></td><td align=\"left\"><p>0.0044</p></td><td align=\"left\"><p>0.0027</p></td><td align=\"left\"><p>0.0058</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p>0.0063</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic> (min)</p></td><td align=\"left\"><p>278.70</p></td><td align=\"left\"><p>221.71</p></td><td align=\"left\"><p>803.16</p></td><td align=\"left\"><p>1174.33</p></td><td align=\"left\"><p>362.76</p></td><td align=\"left\"><p>324.95</p></td><td align=\"left\"><p>252.36</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic><sub>exp</sub> (min)</p></td><td align=\"left\"><p>270</p></td><td align=\"left\"><p>210</p></td><td align=\"left\"><p>660</p></td><td align=\"left\"><p>1200</p></td><td align=\"left\"><p>360</p></td><td align=\"left\"><p>330</p></td><td align=\"left\"><p>210</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Adams–Bohart model</p></td><td align=\"left\"><p><italic>k</italic><sub>BA</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>1.70</p></td><td align=\"left\"><p>2.25</p></td><td align=\"left\"><p>1.60</p></td><td align=\"left\"><p>0.95</p></td><td align=\"left\"><p>1.75</p></td><td align=\"left\"><p>1.13</p></td><td align=\"left\"><p>0.82</p></td></tr><tr><td align=\"left\"><p><italic>N</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>7733.02</p></td><td align=\"left\"><p>7393.39</p></td><td align=\"left\"><p>5684.98</p></td><td align=\"left\"><p>5538.09</p></td><td align=\"left\"><p>7182.78</p></td><td align=\"left\"><p>10388.09</p></td><td align=\"left\"><p>12462.19</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.81</p></td><td align=\"left\"><p>0.80</p></td><td align=\"left\"><p>0.85</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.82</p></td><td align=\"left\"><p>0.77</p></td><td align=\"left\"><p>0.77</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Clark model</p></td><td align=\"left\"><p><italic>r</italic> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0071</p></td><td align=\"left\"><p>0.01</p></td><td align=\"left\"><p>0.0054</p></td><td align=\"left\"><p>0.0032</p></td><td align=\"left\"><p>0.0067</p></td><td align=\"left\"><p>0.0068</p></td><td align=\"left\"><p>0.0072</p></td></tr><tr><td align=\"left\"><p><italic>A</italic></p></td><td align=\"left\"><p>11.89</p></td><td align=\"left\"><p>16.26</p></td><td align=\"left\"><p>115.75</p></td><td align=\"left\"><p>68.27</p></td><td align=\"left\"><p>18.30</p></td><td align=\"left\"><p>14.71</p></td><td align=\"left\"><p>9.94</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.88</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.87</p></td><td align=\"left\"><p>0.89</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Yan model</p></td><td align=\"left\"><p><italic>k</italic><sub>Y</sub> (L/mg/min)</p></td><td align=\"left\"><p>0.030</p></td><td align=\"left\"><p>0.040</p></td><td align=\"left\"><p>0.035</p></td><td align=\"left\"><p>0.032</p></td><td align=\"left\"><p>0.027</p></td><td align=\"left\"><p>0.015</p></td><td align=\"left\"><p>0.010</p></td></tr><tr><td align=\"left\"><p><italic>q</italic><sub>0</sub> (mg/g)</p></td><td align=\"left\"><p>0.756</p></td><td align=\"left\"><p>0.892</p></td><td align=\"left\"><p>0.307</p></td><td align=\"left\"><p>0.205</p></td><td align=\"left\"><p>0.617</p></td><td align=\"left\"><p>0.929</p></td><td align=\"left\"><p>1.239</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.98</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.99</p></td></tr></tbody></table></table-wrap></p><p id=\"Par92\">It indicates agreement between the Thomas model and experimental data for breakthrough curves (0.89 &gt; <italic>R</italic><sup>2</sup> &gt; 0.99), signifying the good fit of the model for ACVR adsorption on G-AC. Since the experimental data fitted the Thomas model well, film and intraparticle diffusion are not expected to be the limiting step. But, the limiting adsorption stage is the solid surface mass transfer (de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Flow resistance increases on increasing bed height, resulting in decreased mass transfer effect, causing a decrement in <italic>k</italic><sub>TH</sub> and increase in <italic>q</italic><sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>). On the other hand, increasing flow rate will result in insufficient contact time between ACVR and G-AC for fixed number of vacant sites causing decrease in the value of <italic>q</italic><sub>TH</sub> whereas increase in k<sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>). An increased driving force achieved a higher column performance at higher ACVR concentration. The results are consistent with the literature (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Pholosi et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>).</p><p id=\"Par93\">Table <xref ref-type=\"table\" rid=\"Tab5\">5</xref> compares the value of q<sub>TH</sub> of G-AC obtained in the present study with the literature. The maximum adsorption capacity of G-AC (q<sub>TH</sub>) in the present study has been found to be higher than other adsorbate-adsorbent systems.<table-wrap id=\"Tab5\"><label>Table 5</label><caption xml:lang=\"en\"><p>Comparison of q<sub>TH</sub> with values reported in literature</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Adsorbent</p></th><th align=\"left\"><p>Adsorbate</p></th><th align=\"left\"><p>Models investigated*</p></th><th align=\"left\"><p><italic>q</italic><sub>TH</sub> (mg/g)</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>G-AC</p></td><td align=\"left\"><p>ACVR</p></td><td align=\"left\"><p>TH, YN, AB, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>224.91</p></td><td align=\"left\"><p>Present study</p></td></tr><tr><td align=\"left\"><p>Thermally activated weed Salvinia Cucullata</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>159.20</p></td><td align=\"left\"><p>Baral et al. (<xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.09</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Norfloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>1.99</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Levofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.56</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon</p></td><td align=\"left\"><p>Amoxicillin</p></td><td align=\"left\"><p>TH, AB, Yan</p></td><td align=\"char\" char=\".\"><p>11.48</p></td><td align=\"left\"><p>de Franco et al. (<xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated sludge derived activated carbon</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>14.00</p></td><td align=\"left\"><p>Gupta and Garg (<xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Carboxyl-modified jute fiber</p></td><td align=\"left\"><p>Cu (II)</p></td><td align=\"left\"><p>TH, YN</p></td><td align=\"char\" char=\".\"><p>42.21</p></td><td align=\"left\"><p>Du et al. (<xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date palm petiole</p></td><td align=\"left\"><p>Indigo carmine</p></td><td align=\"left\"><p>TH, YN, AB, Yan</p></td><td align=\"char\" char=\".\"><p>48.88</p></td><td align=\"left\"><p>Khadhri et al. (<xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Bone char</p></td><td align=\"left\"><p>Crystal violet</p></td><td align=\"left\"><p>TH, Yan</p></td><td align=\"char\" char=\".\"><p>22.44</p></td><td align=\"left\"><p>Cruz et al. (<xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Magnetic pine cone composite</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>33.08</p></td><td align=\"left\"><p>Pholosi et al. (<xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Acid modified alumina</p></td><td align=\"left\"><p>Fluoride</p></td><td align=\"left\"><p>TH, YN, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>19.12</p></td><td align=\"left\"><p>Kumari et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>*<italic>TH</italic> Thomas model, <italic>YN</italic> Yoon–Nelson model, <italic>AB</italic> Adams–Bohart model</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec22\"><title>Yoon–Nelson model</title><p id=\"Par94\">This model presupposes that the decline in the adsorption rate of each adsorbate molecule is proportionally related to the adsorbate adsorption probability and probability of its breakthrough capacity onto the adsorbent (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Yoon and Nelson <xref ref-type=\"bibr\" rid=\"CR60\">1984</xref>). This model is significant in determining the nature of the adsorption process with its exhaustion time for a given concentration (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>). The estimated values of various Yoon–Nelson model parameters are summarized in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. It shows the strong correlation between experimental data and the Yoon–Nelson model (0.89 &lt; <italic>R</italic><sup>2</sup> &lt; 0.99) (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> and Fig. S8), indicating the adequacy of the model. According to the Yoon–Nelson model's fitting parameters, an increase in the initial concentration of ACVR leads to the enhancement of <italic>K</italic><sub>YN</sub> value and reduction in τ value. It might be because the G-AC in the column quickly saturates.</p><p id=\"Par95\">On the contrary, as the bed height was raised, the <italic>K</italic><sub>YN</sub> declined and the value of <italic>τ</italic> improved. This is because the amount of adsorbent increases as the height of the bed rises. Other studies have also reported consistent patterns (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Demarchi et al. <xref ref-type=\"bibr\" rid=\"CR16\">2015</xref>).</p></sec><sec id=\"Sec23\"><title>Adams–Bohart model</title><p id=\"Par96\">The Adams–Bohart model is based on the premise that equilibrium is not instantaneous and primarily concerned with the surface reaction theory. Therefore, rate of adsorption is related to adsorbate concentration in the solution and the adsorbent’s residual capacity (Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The initial segment of the breakthrough curve has been described by Adams–Bohart model in several previous studies. However, in the current study, the model has been fitted from the beginning of breakthrough curve to final point (Fig. S9). The estimated model parameters and R<sup>2</sup> values for various process conditions are provided in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> shows that the values of k<sub>BA</sub> were found to increase with increasing flow rate but deceased with increasing initial ACVR concentration and bed height. At most of the process conditions, the correlation coefficient is low (&lt; 0.86); hence, the correlation between experimental data and the Adams–Bohart model is not strong. Other investigations have also reported similar results. (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec24\"><title>Clark model</title><p id=\"Par97\">The Clark model anticipates the breakthrough curve’s behavior by studying mass transfer and adsorption equilibrium (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). It assumes that the flow in the column is plug flow type, and the Freundlich isotherm is followed by mass transfer occurring in the column (Patel <xref ref-type=\"bibr\" rid=\"CR44\">2019</xref>). Fig. S10 shows the linear curve fitting of breakthrough curves using the Clark model. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> presents the value of Clark model parameters, ‘<italic>A</italic>’ and ‘<italic>r</italic>’, at different process conditions. On increasing the bed height of the column, the value of <italic>r</italic> decreased. Further, <italic>r</italic> increases with flow rate and inlet ACVR concentration. At almost all process conditions, the value of correlation coefficient is in the range (0.87–0.92), indicating relatively poor fit of the Clark model. Similar results have been obtained earlier (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>).</p></sec><sec id=\"Sec25\"><title>Yan model</title><p id=\"Par98\">It has been reported earlier that, at absurdly short and very long operation times, the Yan model is proposed to reduce the inaccuracy that occurs from using the Thomas model (Maheshwari and Gupta <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR59\">2001</xref>). Figure <xref ref-type=\"fig\" rid=\"Fig8\">8</xref> illustrates the linear curve fitting of experimental data using the Yan model for various parameters.<fig id=\"Fig8\"><label>Fig. 8</label><caption xml:lang=\"en\"><p>Yan model for varying <bold>a</bold> flow rate, <bold>b</bold> bed height, and <bold>c</bold> initial ACVR concentration</p></caption><graphic id=\"MO9\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig8_HTML.png\"/></fig></p><p id=\"Par99\">The findings indicate a good agreement between the experimental data and the model, with R<sup>2</sup> values in the ranges of 0.94–0.99 (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>). The value of k<sub>Y</sub> increases on increasing flow rate, while decreases with concentration. According to Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>, the estimated adsorption capacity values from the Thomas and Yan model are of the same order. However, it is observed that the Yan model is better at evaluating adsorption capacity of the bed than the Thomas model which estimates inflated value (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>). Other works of literature have also reported similar overall results (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Comparison of breakthrough curve fitting</title><p id=\"Par100\">Thomas and Yan models suggest that external and intraparticle diffusion are not controlling, but the solid surface mass transfer (surface diffusion) is the limiting step in the adsorption process. (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). Adams–Bohart model demonstrated the lowest average <italic>R</italic><sup>2</sup> (0.82) signifying the weak correlation between experimental data and the model. Although, the average value of correlation coefficient for Clark model is 0.90. The poor fit of the Clark model for breakthrough data is in accordance with the batch study results, which fitted well to Langmuir isotherm. The inability of Adams–Bohart and Clark models to accurately predict the removal of ACVR by G-AC can be attributed to two factors: the existence of numerous mechanisms (possibly, adsorption and ion exchange) that contribute to ACVR adsorption onto G-AC and the significant involvement of mass transfer resistance in the removal of ACVR using G-AC (Dorado et al. <xref ref-type=\"bibr\" rid=\"CR17\">2014</xref>).</p><p id=\"Par101\">The Yan model exhibited the highest average <italic>R</italic><sup>2</sup> (0.98), indicating the best fit of the model and the experimental data. As noted in the previous section, batch ACVR adsorption onto G-AC was found to follow the pseudo-second-order kinetic model and Langmuir isotherm model. Both of these conclusions have been supported by Yan models applied to fixed-bed column studies.</p><p id=\"Par102\">Continuous column studies of ACVR adsorption on any of the adsorbent has been scarcely reported. Therefore, q<sub>TH</sub> value signifying adsorption capacity of the adsorbent obtained using Thomas model has been compared for the present study with results reported in literature for other adsorbate-adsorbent systems.</p></sec></sec><sec id=\"Sec27\" sec-type=\"conclusions\"><title>Conclusions</title><p id=\"Par103\">Antiviral pollutants, such as ACVR have been found in groundwater, surface water, and industrial wastewater. Their inadequate removal through conventional treatment is a significant challenge in water reuse. G-AC has proven to be an efficient adsorbent for the ACVR removal from aqueous solutions. The characterization showed the morphology and surface area of G-AC along with the ACVR adsorption (SEM, BET, and EDX).</p><p id=\"Par104\">Batch experiments showed that Langmuir isotherm adequately represents the equilibrium data indicating monolayer chemisorption is the predominant mechanism for ACVR adsorption onto G-AC. Pseudo-second-order kinetic model predicted the adsorption process. Negative values of heat of adsorption and Gibbs free energy indicated exothermic nature of the process and confirmed the process feasibility. The process of adsorption was found to be physisorption accompanied by chemisorption. The feasibility of adsorption decreases with increase in temperature. The adsorption process slightly decreases the randomness of molecules at the solid/liquid interface and is enthalpy driven.</p><p id=\"Par105\">The fixed-bed column experimental data were fitted to different analytical models. The Yan model fitted the data best, followed by the Thomas model confirming the pseudo-second-order kinetics and Langmuir isotherm. Adsorption capacity of the fixed bed of G-AC estimated using Thomas model indicated that the adsorption capacity is excellent for ACVR when compared to other adsorbate-adsorbent system studied in literature. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p><p id=\"Par106\">The current research's findings will be useful for small volumes of wastewater generated by ACVR manufacturing units. ACVR removal from the industrial effluents released by bulk manufacturing and formulation industries can be achieved in a continuous flow unit using a packed bed of a commercial adsorbent. This will prevent the entry of strong inhibitory substances, like ACVR, into the biological wastewater treatment units, which may inhibit the activity of the micro-organism that treat the soluble organics.</p></sec></body><back><ack><title>Acknowledgements</title><p>No specific grant was given to this research by funding organizations in the public, commercial, or not-for-profit sectors. However, one of the authors (AG) expresses her gratitude to Malaviya National Institute of Technology (MNIT) Jaipur and Ministry of Education, Govt. of India for providing the financial support in the form of Institute Fellowship for carrying out Ph. D. research work. The authors also wish to thank the Material Research Centre (MRC), MNIT Jaipur, for carrying out SEM, EDX and BET analysis of the samples.</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par107\">The authors report no declarations of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Al-Otaibi</surname><given-names>JS</given-names></name><name><surname>Mary</surname><given-names>YS</given-names></name><name><surname>Thomas</surname><given-names>R</given-names></name><name><surname>Kaya</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Detailed electronic structure, physico-chemical properties, excited state properties, virtual bioactivity screening and SERS analysis of three guanine based antiviral drugs valacyclovir HCl hydrate, acyclovir and ganciclovir</article-title><source>Polycycl Aromat Compd</source><year>2020</year><volume>42</volume><fpage>1260</fpage><lpage>1270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVKmu7jE</pub-id><pub-id pub-id-type=\"doi\">10.1080/10406638.2020.1773876</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azuma</surname><given-names>T</given-names></name><name><surname>Arima</surname><given-names>N</given-names></name><name><surname>Tsukada</surname><given-names>A</given-names></name><name><surname>Hirami</surname><given-names>S</given-names></name><name><surname>Matsuoka</surname><given-names>R</given-names></name><name><surname>Moriwake</surname><given-names>R</given-names></name><name><surname>Ishiuchi</surname><given-names>H</given-names></name><name><surname>Inoyama</surname><given-names>T</given-names></name><name><surname>Teranishi</surname><given-names>Y</given-names></name><name><surname>Yamaoka</surname><given-names>M</given-names></name><name><surname>Mino</surname><given-names>Y</given-names></name><name><surname>Hayashi</surname><given-names>T</given-names></name><name><surname>Fujita</surname><given-names>Y</given-names></name><name><surname>Masada</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Science of the Total Environment Detection of pharmaceuticals and phytochemicals together with their metabolites in hospital effluents in Japan, and their contribution to sewage treatment plant influents</article-title><source>Sci Total Environ</source><year>2016</year><volume>548–549</volume><fpage>189</fpage><lpage>197</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KqsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2015.12.157</pub-id><pub-id pub-id-type=\"pmid\">26802347</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Babas</surname><given-names>H</given-names></name><name><surname>Kaichouh</surname><given-names>G</given-names></name><name><surname>Khachani</surname><given-names>M</given-names></name><name><surname>Karbane</surname><given-names>ME</given-names></name><name><surname>Chakir</surname><given-names>A</given-names></name><name><surname>Guenbour</surname><given-names>A</given-names></name><name><surname>Bellaouchou</surname><given-names>A</given-names></name><name><surname>Warad</surname><given-names>I</given-names></name><name><surname>Zarrouk</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Equilibrium and kinetic studies for removal of antiviral sofosbuvir from aqueous solution by adsorption on expanded perlite: experimental, modelling and optimization</article-title><source>Surf Interfaces</source><year>2021</year><volume>23</volume><fpage>100962</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsFaqtr%2FF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.surfin.2021.100962</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Baral</surname><given-names>SS</given-names></name><name><surname>Das</surname><given-names>N</given-names></name><name><surname>Ramulu</surname><given-names>TS</given-names></name><name><surname>Sahoo</surname><given-names>SK</given-names></name><name><surname>Das</surname><given-names>SN</given-names></name><name><surname>Chaudhury</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) by thermally activated weed Salvinia cucullata in a fixed-bed column</article-title><source>J Hazard Mater</source><year>2009</year><volume>161</volume><fpage>1427</fpage><lpage>1435</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhsVSqsbnE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2008.04.127</pub-id><pub-id pub-id-type=\"pmid\">18571842</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bing</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Enhanced mineralization of pharmaceuticals by surface oxidation over mesoporous γ-Ti-Al<sub>2</sub>O<sub>3</sub> suspension with ozone</article-title><source>Appl Catal B Environ</source><year>2017</year><volume>202</volume><fpage>118</fpage><lpage>126</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFWnur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2016.09.019</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Biswas</surname><given-names>S</given-names></name><name><surname>Mishra</surname><given-names>U</given-names></name></person-group><article-title xml:lang=\"en\">Continuous fixed-bed column study and adsorption modeling: Removal of lead ion from aqueous solution by charcoal originated from chemical carbonization of rubber wood sawdust</article-title><source>J Chem</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1155/2015/907379</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Borah</surname><given-names>D</given-names></name><name><surname>Satokawa</surname><given-names>S</given-names></name><name><surname>Kato</surname><given-names>S</given-names></name><name><surname>Kojima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Surface-modified carbon black for As(V) removal</article-title><source>J Colloid Interface Sci</source><year>2008</year><volume>319</volume><fpage>53</fpage><lpage>62</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXnvVWntg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2007.11.019</pub-id><pub-id pub-id-type=\"pmid\">18082179</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bradley</surname><given-names>PM</given-names></name><name><surname>Barber</surname><given-names>LB</given-names></name><name><surname>Duris</surname><given-names>JW</given-names></name><name><surname>Foreman</surname><given-names>WT</given-names></name><name><surname>Furlong</surname><given-names>ET</given-names></name><name><surname>Hubbard</surname><given-names>LE</given-names></name><name><surname>Hutchinson</surname><given-names>KJ</given-names></name><name><surname>Keefe</surname><given-names>SH</given-names></name><name><surname>Kolpin</surname><given-names>DW</given-names></name></person-group><article-title xml:lang=\"en\">Riverbank filtration potential of pharmaceuticals in a wastewater-impacted stream</article-title><source>Environ Pollut</source><year>2014</year><volume>193</volume><fpage>173</fpage><lpage>180</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1ensbnK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.envpol.2014.06.028</pub-id><pub-id pub-id-type=\"pmid\">25038376</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Yue</surname><given-names>Q</given-names></name><name><surname>Gao</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Xu</surname><given-names>X</given-names></name><name><surname>Fu</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of hexavalent chromium from aqueous solution by modified corn stalk: a fixed-bed column study</article-title><source>Bioresour Technol</source><year>2012</year><volume>113</volume><fpage>114</fpage><lpage>120</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFWntrg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2011.11.110</pub-id><pub-id pub-id-type=\"pmid\">22189077</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Luo</surname><given-names>H</given-names></name><name><surname>Shi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Anatase TiO<sub>2</sub> nanoparticles-carbon nanotubes composite: optimization synthesis and the relationship of photocatalytic degradation activity of acyclovir in water</article-title><source>Appl Catal A Gen</source><year>2014</year><volume>485</volume><fpage>188</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhsVSmsLbP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcata.2014.08.004</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cruz</surname><given-names>MAP</given-names></name><name><surname>Guimarães</surname><given-names>LCM</given-names></name><name><surname>da Costa Júnior</surname><given-names>EF</given-names></name><name><surname>Rocha</surname><given-names>SDF</given-names></name><name><surname>da Mesquita</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of crystal violet from aqueous solution in continuous flow system using bone char</article-title><source>Chem Eng Commun</source><year>2020</year><volume>207</volume><fpage>372</fpage><lpage>381</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXms1amsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/00986445.2019.1596899</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Batch and fixed bed adsorption of levofloxacin on granular activated carbon from date (<italic>Phoenix dactylifera</italic> L.) stones by KOH chemical activation</article-title><source>Environ Toxicol Pharmacol</source><year>2017</year><volume>50</volume><fpage>159</fpage><lpage>166</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitlOls7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.etap.2017.02.005</pub-id><pub-id pub-id-type=\"pmid\">28189062</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of ciprofloxacin and norfloxacin from aqueous solution onto granular activated carbon in fixed bed column</article-title><source>Ecotoxicol Environ Saf</source><year>2017</year><volume>138</volume><fpage>139</fpage><lpage>145</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXislShsQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2016.12.032</pub-id><pub-id pub-id-type=\"pmid\">28040619</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Clercq</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Antivirals and antiviral strategies</article-title><source>Nat Rev Microbiol</source><year>2004</year><volume>2</volume><fpage>704</fpage><lpage>720</lpage><pub-id pub-id-type=\"doi\">10.1038/nrmicro975</pub-id><pub-id pub-id-type=\"pmid\">15372081</pub-id><pub-id pub-id-type=\"pmcid\">7097272</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Franco</surname><given-names>MAE</given-names></name><name><surname>de Carvalho</surname><given-names>CB</given-names></name><name><surname>Bonetto</surname><given-names>MM</given-names></name><name><surname>de Soares</surname><given-names>RS</given-names></name><name><surname>Féris</surname><given-names>LA</given-names></name></person-group><article-title xml:lang=\"en\">Removal of amoxicillin from water by adsorption onto activated carbon in batch process and fixed bed column: kinetics, isotherms, experimental design and breakthrough curves modelling</article-title><source>J Clean Prod</source><year>2017</year><volume>161</volume><fpage>947</fpage><lpage>956</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVant7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2017.05.197</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Demarchi</surname><given-names>CA</given-names></name><name><surname>Debrassi</surname><given-names>A</given-names></name><name><surname>Magro</surname><given-names>JD</given-names></name><name><surname>Nedelko</surname><given-names>N</given-names></name><name><surname>Ślawska-Waniewska</surname><given-names>A</given-names></name><name><surname>Dłuzewski</surname><given-names>P</given-names></name><name><surname>Greneche</surname><given-names>JM</given-names></name><name><surname>Rodrigues</surname><given-names>CA</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of Cr(VI) on crosslinked chitosan-Fe(III) complex in fixed-bed systems</article-title><source>J Water Process Eng</source><year>2015</year><volume>7</volume><fpage>141</fpage><lpage>152</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2015.05.003</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dorado</surname><given-names>AD</given-names></name><name><surname>Gamisans</surname><given-names>X</given-names></name><name><surname>Valderrama</surname><given-names>C</given-names></name><name><surname>Solé</surname><given-names>M</given-names></name><name><surname>Lao</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Cr(III) removal from aqueous solutions: a straightforward model approaching of the adsorption in a fixed-bed column</article-title><source>J Environ Sci Heal Part A Toxic/hazardous Subst Environ Eng</source><year>2014</year><volume>49</volume><fpage>179</fpage><lpage>186</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslWisbfJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/10934529.2013.838855</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>Z</given-names></name><name><surname>Zheng</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Experimental and modelling studies on fixed bed adsorption for Cu(II) removal from aqueous solution by carboxyl modified jute fiber</article-title><source>Powder Technol</source><year>2018</year><volume>338</volume><fpage>952</fpage><lpage>959</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVOltLjP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.powtec.2018.06.015</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fan</surname><given-names>L</given-names></name><name><surname>Luo</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>F</given-names></name><name><surname>Qiu</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Fabrication of novel magnetic chitosan grafted with graphene oxide to enhance adsorption properties for methyl blue</article-title><source>J Hazard Mater</source><year>2012</year><volume>215–216</volume><fpage>272</fpage><lpage>279</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XkvVWju74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2012.02.068</pub-id><pub-id pub-id-type=\"pmid\">22429622</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fountouli</surname><given-names>TV</given-names></name><name><surname>Chrysikopoulos</surname><given-names>CV</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and thermodynamics of pharmaceuticals, acyclovir and fluconazole, onto quartz sand under static and dynamic conditions</article-title><source>Environ Eng Sci</source><year>2018</year><volume>35</volume><fpage>909</fpage><lpage>917</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslSgu7jF</pub-id><pub-id pub-id-type=\"doi\">10.1089/ees.2017.0499</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Funke</surname><given-names>J</given-names></name><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Identification of transformation products of antiviral drugs formed during biological wastewater treatment and their occurrence in the urban water cycle</article-title><source>Water Res</source><year>2016</year><volume>98</volume><fpage>75</fpage><lpage>83</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XlvV2qt7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2016.03.045</pub-id><pub-id pub-id-type=\"pmid\">27082694</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fytianos</surname><given-names>K</given-names></name><name><surname>Voudrias</surname><given-names>E</given-names></name><name><surname>Kokkalis</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sorption-desorption behaviour of 2,4-dichlorophenol by marine sediments</article-title><source>Chemosphere</source><year>2000</year><volume>40</volume><fpage>3</fpage><lpage>6</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXivFSrsw%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0045-6535(99)00214-3</pub-id><pub-id pub-id-type=\"pmid\">10665437</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Gan</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>H</given-names></name><name><surname>Cai</surname><given-names>N</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Guo</surname><given-names>P</given-names></name><name><surname>Chen</surname><given-names>B</given-names></name><name><surname>Pan</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Effective degradation of COVID-19 related drugs by biochar-supported red mud catalyst activated persulfate process: Mechanism and pathway</article-title><source>J Clean Prod</source><year>2022</year><volume>340</volume><fpage>130753</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2022.130753</pub-id><pub-id pub-id-type=\"pmid\">36032562</pub-id><pub-id pub-id-type=\"pmcid\">9396784</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Garg</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and oxidation of ciprofloxacin in a fixed bed column using activated sludge derived activated carbon</article-title><source>J Environ Manag</source><year>2019</year><volume>250</volume><fpage>109474</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhslKnu7bJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jenvman.2019.109474</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Gupta</surname><given-names>AB</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence of acyclovir in the aquatic environment, its removal and research perspectives: a review</article-title><source>J Water Process Eng</source><year>2021</year><volume>39</volume><fpage>101855</fpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2020.101855</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>Yu</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Wang Yuanfeng Xie</surname><given-names>F</given-names></name><name><surname>Cheng</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of methylene blue by phoenix tree leaf powder in a fixed-bed column: experiments and prediction of breakthrough curves</article-title><source>Desalination</source><year>2009</year><volume>245</volume><fpage>284</fpage><lpage>297</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXovFyhtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2008.07.013</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hasnain Isa</surname><given-names>M</given-names></name><name><surname>Siew Lang</surname><given-names>L</given-names></name><name><surname>Asaari</surname><given-names>FAH</given-names></name><name><surname>Aziz</surname><given-names>HA</given-names></name><name><surname>Azam Ramli</surname><given-names>N</given-names></name><name><surname>Dhas</surname><given-names>JPA</given-names></name></person-group><article-title xml:lang=\"en\">Low cost removal of disperse dyes from aqueous solution using palm ash</article-title><source>Dye Pigment</source><year>2007</year><volume>74</volume><fpage>446</fpage><lpage>453</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXht1Ort7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.dyepig.2006.02.025</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heidary</surname><given-names>F</given-names></name><name><surname>Madani</surname><given-names>S</given-names></name><name><surname>Gharebaghi</surname><given-names>R</given-names></name><name><surname>Asadi-Amoli</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">Acyclovir as a potential add-on therapy in COVID-19 treatment regimens</article-title><source>Pharm Sci</source><year>2021</year><volume>27</volume><fpage>S68</fpage><lpage>S77</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislalsL%2FF</pub-id><pub-id pub-id-type=\"doi\">10.34172/PS.2021.38</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hosseini Nasr</surname><given-names>A</given-names></name><name><surname>Sadat</surname><given-names>AH</given-names></name><name><surname>Tayebee</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanism of different acyclovir concentrations on 1–2 nm sized magnetite nanoparticles: a molecular dynamics study</article-title><source>J Mol Liq</source><year>2018</year><volume>254</volume><fpage>64</fpage><lpage>69</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVemtro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2018.01.081</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence and removal of antiviral drugs in environment: a review</article-title><source>Water Air Soil Pollut</source><year>2013</year><pub-id pub-id-type=\"doi\">10.1007/s11270-012-1410-3</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption optimization of acyclovir on prepared activated carbon</article-title><source>Can J Chem Eng</source><year>2014</year><volume>92</volume><fpage>1627</fpage><lpage>1635</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhtlynur7J</pub-id><pub-id pub-id-type=\"doi\">10.1002/cjce.22026</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of antiviral drug, acyclovir from aqueous solution on powdered activated charcoal: kinetics, equilibrium, and thermodynamic studies</article-title><source>Desalin Water Treat</source><year>2014</year><volume>52</volume><fpage>4953</fpage><lpage>4968</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1SktL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2013.810324</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khadhri</surname><given-names>N</given-names></name><name><surname>El Khames</surname><given-names>SM</given-names></name><name><surname>Ben Mosbah</surname><given-names>M</given-names></name><name><surname>Moussaoui</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous column adsorption of indigo carmine onto activated carbon derived from date palm petiole</article-title><source>J Environ Chem Eng</source><year>2019</year><volume>7</volume><fpage>102775</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVygs7rK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jece.2018.11.020</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>U</given-names></name><name><surname>Mishra</surname><given-names>A</given-names></name><name><surname>Siddiqi</surname><given-names>H</given-names></name><name><surname>Meikap</surname><given-names>BC</given-names></name></person-group><article-title xml:lang=\"en\">Effective defluoridation of industrial wastewater by using acid modified alumina in fixed-bed adsorption column: experimental and breakthrough curves analysis</article-title><source>J Clean Prod</source><year>2021</year><volume>279</volume><fpage>123645</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslejtbvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2020.123645</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Nie</surname><given-names>X</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Can environmental pharmaceuticals be photocatalytically degraded and completely mineralized in water using g-C<sub>3</sub>N<sub>4</sub>TiO<sub>2</sub> under visible light irradiation? Implications of persistent toxic intermediates</article-title><source>Appl Catal B Environ</source><year>2016</year><volume>180</volume><fpage>726</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVOqu7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2015.07.014</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liao</surname><given-names>P</given-names></name><name><surname>Zhan</surname><given-names>Z</given-names></name><name><surname>Dai</surname><given-names>J</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>K</given-names></name><name><surname>Yuan</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of tetracycline and chloramphenicol in aqueous solutions by bamboo charcoal: a batch and fixed-bed column study</article-title><source>Chem Eng J</source><year>2013</year><volume>228</volume><fpage>496</fpage><lpage>505</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtVOrtb3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2013.04.118</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Yuan</surname><given-names>D</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yu</surname><given-names>B</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of Fe@porous carbon to be an efficient mediator for peroxymonosulfate activation for oxidation of organic contaminants: incorporation NH2-group into structure of its MOF precursor</article-title><source>Chem Eng J</source><year>2018</year><volume>354</volume><fpage>835</fpage><lpage>848</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGqsLbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2018.08.060</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name><name><surname>Liu Liyuan Tian</surname><given-names>X</given-names></name><name><surname>Liu Longyan Xia</surname><given-names>Y</given-names></name><name><surname>Liang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Tsang</surname><given-names>DCW</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>lkhlaq</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Novel carbon based Fe-Co oxides derived from Prussian blue analogues activating peroxymonosulfate: refractory drugs degradation without metal leaching</article-title><source>Chem Eng J</source><year>2020</year><volume>379</volume><fpage>122274</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVajtLjN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2019.122274</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maheshwari</surname><given-names>U</given-names></name><name><surname>Gupta</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) from wastewater using activated neem bark in a fixed-bed column: interference of other ions and kinetic modelling studies</article-title><source>Desalin Water Treat</source><year>2016</year><volume>57</volume><fpage>8514</fpage><lpage>8525</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXms1Glsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2015.1030709</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Malkoc</surname><given-names>E</given-names></name><name><surname>Nuhoglu</surname><given-names>Y</given-names></name><name><surname>Dundar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of chromium(VI) on pomace-An olive oil industry waste: Batch and column studies</article-title><source>J Hazard Mater</source><year>2006</year><volume>138</volume><fpage>142</fpage><lpage>151</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XhtVOntL3P</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2006.05.051</pub-id><pub-id pub-id-type=\"pmid\">16844293</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mascolo</surname><given-names>G</given-names></name><name><surname>Balest</surname><given-names>L</given-names></name><name><surname>Cassano</surname><given-names>D</given-names></name><name><surname>Laera</surname><given-names>G</given-names></name><name><surname>Lopez</surname><given-names>A</given-names></name><name><surname>Pollice</surname><given-names>A</given-names></name><name><surname>Salerno</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradability of pharmaceutical industrial wastewater and formation of recalcitrant organic compounds during aerobic biological treatment</article-title><source>Bioresour Technol</source><year>2010</year><volume>101</volume><fpage>2585</fpage><lpage>2591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXjvVartQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2009.10.057</pub-id><pub-id pub-id-type=\"pmid\">19939666</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Momčilović</surname><given-names>M</given-names></name><name><surname>Purenović</surname><given-names>M</given-names></name><name><surname>Bojić</surname><given-names>A</given-names></name><name><surname>Zarubica</surname><given-names>A</given-names></name><name><surname>Ranđelović</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Removal of lead(II) ions from aqueous solutions by adsorption onto pine cone activated carbon</article-title><source>Desalination</source><year>2011</year><volume>276</volume><fpage>53</fpage><lpage>59</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXnvFOltro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2011.03.013</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nannou</surname><given-names>C</given-names></name><name><surname>Ofrydopoulou</surname><given-names>A</given-names></name><name><surname>Evgenidou</surname><given-names>E</given-names></name><name><surname>Heath</surname><given-names>D</given-names></name><name><surname>Heath</surname><given-names>E</given-names></name><name><surname>Lambropoulou</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in aquatic environment and wastewater treatment plants: A review on occurrence, fate, removal and ecotoxicity</article-title><source>Sci Total Environ</source><year>2020</year><volume>699</volume><fpage>134322</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVymu7nK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2019.134322</pub-id><pub-id pub-id-type=\"pmid\">31678880</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Fixed-bed column adsorption study: a comprehensive review</article-title><source>Appl Water Sci</source><year>2019</year><volume>9</volume><fpage>1</fpage><lpage>17</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXosVegtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13201-019-0927-7</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Huang</surname><given-names>Q</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Ou</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Profile and behavior of antiviral drugs in aquatic environments of the Pearl River Delta</article-title><source>China Sci Total Environ</source><year>2014</year><volume>466–467</volume><fpage>755</fpage><lpage>761</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslOjs77M</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2013.07.062</pub-id><pub-id pub-id-type=\"pmid\">23973541</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pholosi</surname><given-names>A</given-names></name><name><surname>Naidoo</surname><given-names>EB</given-names></name><name><surname>Ofomaja</surname><given-names>AE</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous flow studies of Cr(VI) adsorption from synthetic and real wastewater by magnetic pine cone composite</article-title><source>Chem Eng Res Des</source><year>2020</year><volume>153</volume><fpage>806</fpage><lpage>818</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitFOjtLvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cherd.2019.11.004</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Schlüsener</surname><given-names>MP</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in wastewater and surface waters: a new pharmaceutical class of environmental relevance?</article-title><source>Environ Sci Technol</source><year>2010</year><volume>44</volume><fpage>1728</fpage><lpage>1735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1Sgsb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es903216p</pub-id><pub-id pub-id-type=\"pmid\">20108960</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Biotransformation of the antiviral drugs acyclovir and penciclovir in activated sludge treatment</article-title><source>Environ Sci Technol</source><year>2011</year><volume>45</volume><fpage>2761</fpage><lpage>2769</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXivFSru7o%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es103732y</pub-id><pub-id pub-id-type=\"pmid\">21388176</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Oxidation of the antiviral drug acyclovir and its biodegradation product carboxy-acyclovir with ozone: kinetics and identification of oxidation products</article-title><source>Environ Sci Technol</source><year>2012</year><volume>46</volume><fpage>2169</fpage><lpage>2178</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsFCis7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es203712z</pub-id><pub-id pub-id-type=\"pmid\">22300376</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reynel-Avila</surname><given-names>HE</given-names></name><name><surname>Mendoza-Castillo</surname><given-names>DI</given-names></name><name><surname>Bonilla-Petriciolet</surname><given-names>A</given-names></name><name><surname>Silvestre-Albero</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of naproxen adsorption on bone char in aqueous solutions using batch and fixed-bed processes</article-title><source>J Mol Liq</source><year>2015</year><volume>209</volume><fpage>187</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1yhur7F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2015.05.013</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russo</surname><given-names>D</given-names></name><name><surname>Siciliano</surname><given-names>A</given-names></name><name><surname>Guida</surname><given-names>M</given-names></name><name><surname>Galdiero</surname><given-names>E</given-names></name><name><surname>Amoresano</surname><given-names>A</given-names></name><name><surname>Andreozzi</surname><given-names>R</given-names></name><name><surname>Reis</surname><given-names>NM</given-names></name><name><surname>Li Puma</surname><given-names>G</given-names></name><name><surname>Marotta</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Photodegradation and ecotoxicology of acyclovir in water under UV254 and UV254/H<sub>2</sub>O<sub>2</sub> processes</article-title><source>Water Res</source><year>2017</year><volume>122</volume><fpage>591</fpage><lpage>602</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWltL%2FO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2017.06.020</pub-id><pub-id pub-id-type=\"pmid\">28628881</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saifi</surname><given-names>Z</given-names></name><name><surname>Rizwanullah</surname><given-names>M</given-names></name><name><surname>Mir</surname><given-names>SR</given-names></name><name><surname>Amin</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Bilosomes nanocarriers for improved oral bioavailability of acyclovir: a complete characterization through in vitro, ex-vivo and in vivo assessment</article-title><source>J Drug Deliv Sci Technol</source><year>2020</year><volume>57</volume><fpage>101634</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXkvFCgsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2020.101634</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>I</given-names></name><name><surname>Adnan</surname><given-names>R</given-names></name><name><surname>Ngah</surname><given-names>WSW</given-names></name><name><surname>Mohamed</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Iron impregnated activated carbon as an efficient adsorbent for the removal of methylene blue: regeneration and kinetics studies</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>1</fpage><lpage>23</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhvFCjsb3O</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0122603</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shetti</surname><given-names>NP</given-names></name><name><surname>Malode</surname><given-names>SJ</given-names></name><name><surname>Nayak</surname><given-names>DS</given-names></name><name><surname>Naik</surname><given-names>RR</given-names></name><name><surname>Kuchinad</surname><given-names>GT</given-names></name><name><surname>Reddy</surname><given-names>KR</given-names></name><name><surname>Shukla</surname><given-names>SS</given-names></name><name><surname>Aminabhavi</surname><given-names>TM</given-names></name></person-group><article-title xml:lang=\"en\">Hetero nanostructured iron oxide and bentonite clay composite assembly for the determination of an antiviral drug acyclovir</article-title><source>Microchem J</source><year>2020</year><volume>155</volume><fpage>104727</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ojsr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.microc.2020.104727</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thiebault</surname><given-names>T</given-names></name><name><surname>Guégan</surname><given-names>R</given-names></name><name><surname>Boussafir</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanisms of emerging micro-pollutants with a clay mineral: Case of tramadol and doxepine pharmaceutical products</article-title><source>J Colloid Interface Sci</source><year>2015</year><volume>453</volume><fpage>1</fpage><lpage>8</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnsFGmuro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2015.04.029</pub-id><pub-id pub-id-type=\"pmid\">25950945</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thommes</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Physical adsorption characterization of nanoporous materials</article-title><source>Chem-Ing-Tech</source><year>2010</year><volume>82</volume><fpage>1059</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXotFemt7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/cite.201000064</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tognarelli</surname><given-names>EI</given-names></name><name><surname>Palomino</surname><given-names>TF</given-names></name><name><surname>Corrales</surname><given-names>N</given-names></name><name><surname>Bueno</surname><given-names>SM</given-names></name><name><surname>Kalergis</surname><given-names>AM</given-names></name><name><surname>González</surname><given-names>PA</given-names></name></person-group><article-title xml:lang=\"en\">Herpes simplex virus evasion of early host antiviral responses</article-title><source>Front Cell Infect Microbiol</source><year>2019</year><pub-id pub-id-type=\"doi\">10.3389/fcimb.2019.00127</pub-id><pub-id pub-id-type=\"pmid\">31114761</pub-id><pub-id pub-id-type=\"pmcid\">6503643</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Ao</surname><given-names>Z</given-names></name><name><surname>Sun</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>An</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Ag<sub>2</sub>MoO<sub>4</sub> nanoparticles encapsulated in g-C<sub>3</sub>N<sub>4</sub> for sunlight photodegradation of pollutants</article-title><source>Catal Today</source><year>2018</year><volume>315</volume><fpage>205</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisF2ltb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cattod.2018.01.019</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>G</given-names></name><name><surname>Viraraghavan</surname><given-names>T</given-names></name><name><surname>Chen</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A new model for heavy metal removal in a biosorption column</article-title><source>Adsorpt Sci Technol</source><year>2001</year><volume>19</volume><fpage>25</fpage><lpage>43</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXks1Ghurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1260/0263617011493953</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoon</surname><given-names>YH</given-names></name><name><surname>Nelson</surname><given-names>JH</given-names></name></person-group><article-title xml:lang=\"en\">Application of gas adsorption kinetics I: a theoretical model for respirator cartridge service life</article-title><source>Am Ind Hyg Assoc J</source><year>1984</year><volume>45</volume><fpage>509</fpage><lpage>516</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaL2cXlvFOhurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/15298668491400197</pub-id><pub-id pub-id-type=\"pmid\">6475758</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Zhu</surname><given-names>H</given-names></name><name><surname>Szewzyk</surname><given-names>U</given-names></name><name><surname>Lübbecke</surname><given-names>S</given-names></name><name><surname>Uwe Geissen</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of emerging organic contaminants with a pilot-scale biofilter packed with natural manganese oxides</article-title><source>Chem Eng J</source><year>2017</year><volume>317</volume><fpage>454</fpage><lpage>460</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXjslWrtbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2017.02.095</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhou</surname><given-names>C</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Xie</surname><given-names>Q</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Fu</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Photolysis of three antiviral drugs acyclovir, zidovudine and lamivudine in surface freshwater and seawater</article-title><source>Chemosphere</source><year>2015</year><volume>138</volume><fpage>792</fpage><lpage>797</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVegtbjF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chemosphere.2015.08.033</pub-id><pub-id pub-id-type=\"pmid\">26295538</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec28\"><title>Supplementary Information</title><p id=\"Par108\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2810_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 1566 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><glossary id=\"AGS1\"><title>List of symbols</title><def-list><def-item><term>∆<italic>G</italic>°</term><def><p id=\"Par3\">Change in Gibbs free energy (kJ/moL)</p></def></def-item><def-item><term>∆<italic>H</italic>°</term><def><p id=\"Par4\">Change in heat of adsorption (kJ/moL)</p></def></def-item><def-item><term>∆pH</term><def><p id=\"Par5\">Change in pH</p></def></def-item><def-item><term>∆<italic>S</italic>°</term><def><p id=\"Par6\">Change in entropy (kJ/mol/K)</p></def></def-item><def-item><term><italic>A</italic></term><def><p id=\"Par7\">Clark constant</p></def></def-item><def-item><term><italic>a</italic></term><def><p id=\"Par8\">Yan model constant</p></def></def-item><def-item><term><italic>A</italic><sub><italic>T</italic></sub></term><def><p id=\"Par9\">Equilibrium binding constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>b</italic></term><def><p id=\"Par10\">Yan model constant</p></def></def-item><def-item><term><italic>b</italic><sub><italic>T</italic></sub></term><def><p id=\"Par11\">Constant related to heat of adsorption</p></def></def-item><def-item><term><italic>B</italic><sub><italic>T</italic></sub></term><def><p id=\"Par12\">Temkin constant related to heat of adsorption <inline-formula id=\"IEq1\"><alternatives><mml:math id=\"IEq1_Math\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>B</mml:mi><mml:mi>T</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>b</mml:mi><mml:mi>T</mml:mi></mml:msub></mml:mrow></mml:mfenced></mml:math><tex-math id=\"IEq1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\left( {B_{T} = \\left( {RT} \\right)/b_{T} } \\right)$$\\end{document}</tex-math><inline-graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_IEq1.gif\"/></alternatives></inline-formula> (J/mol)</p></def></def-item><def-item><term><italic>C</italic></term><def><p id=\"Par13\">Constant representing the thickness of boundary layer (mg/g)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>0</italic></sub></term><def><p id=\"Par14\">Initial concentration of ACVR in solution (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>e</italic></sub></term><def><p id=\"Par15\">Concentration of ACVR at equilibrium (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>f</italic></sub></term><def><p id=\"Par16\">The final concentration of ACVR (mg/L)</p></def></def-item><def-item><term><italic>F</italic></term><def><p id=\"Par17\">Linear velocity or superficial velocity (cm/min)</p></def></def-item><def-item><term><italic>K</italic><sub>0</sub></term><def><p id=\"Par18\">Equilibrium constant</p></def></def-item><def-item><term><italic>k</italic><sub>1</sub></term><def><p id=\"Par19\">Pseudo-first order rate constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>2</sub></term><def><p id=\"Par20\">Pseudo-second-order rate constant (g/mg/min)</p></def></def-item><def-item><term><italic>k</italic><sub>BA</sub></term><def><p id=\"Par21\">Adams–Bohart constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>f</sub></term><def><p id=\"Par22\">Freundlich constant ((mg/g)/(mg/L)<sup>1/n</sup>)</p></def></def-item><def-item><term><italic>K</italic><sub>L</sub></term><def><p id=\"Par23\">Langmuir adsorption constant (L/mg)</p></def></def-item><def-item><term><italic>K</italic><sub><italic>p</italic></sub></term><def><p id=\"Par24\">Rate constant for intraparticle diffusion kinetics (mg/g/min<sup>0.5</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>TH</sub></term><def><p id=\"Par25\">Thomas rate constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>y</sub></term><def><p id=\"Par26\">Yan model constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>YN</sub></term><def><p id=\"Par27\">Yoon–Nelson constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>m</italic></term><def><p id=\"Par28\">Mass of G-AC in column (g)</p></def></def-item><def-item><term><italic>n</italic></term><def><p id=\"Par29\">Adsorption intensity</p></def></def-item><def-item><term><italic>N</italic><sub>0</sub></term><def><p id=\"Par30\">Saturation concentration (mg/L)</p></def></def-item><def-item><term>pH<sub>i</sub></term><def><p id=\"Par31\">Initial pH of the solution</p></def></def-item><def-item><term><italic>Q</italic></term><def><p id=\"Par32\">Volumetric flow rate (mL/min)</p></def></def-item><def-item><term><italic>q</italic><sub>0</sub></term><def><p id=\"Par33\">Yan model maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>e</sub></term><def><p id=\"Par34\">Amount of ACVR adsorbed by adsorbent at equilibrium (mg/g)</p></def></def-item><def-item><term><italic>Q</italic><sub>m</sub></term><def><p id=\"Par35\">Maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub><italic>t</italic></sub></term><def><p id=\"Par36\">Amount of ACVR adsorbed by adsorbent at time t (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>TH</sub></term><def><p id=\"Par37\">Adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>r</italic></term><def><p id=\"Par38\">Clark constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term>RMSE</term><def><p id=\"Par39\">Root-mean-square error</p></def></def-item><def-item><term>SD%</term><def><p id=\"Par40\">Standard deviation percent</p></def></def-item><def-item><term><italic>T</italic></term><def><p id=\"Par41\">Temperature (K)</p></def></def-item><def-item><term><italic>V</italic></term><def><p id=\"Par42\">The initial volume of ACVR solution (L)</p></def></def-item><def-item><term><italic>W</italic></term><def><p id=\"Par43\">The dry mass of adsorbent (g)</p></def></def-item><def-item><term><italic>Z</italic></term><def><p id=\"Par44\">Bed height of column (cm)</p></def></def-item></def-list></glossary><glossary id=\"AGS2\"><title>Greek letters</title><def-list><def-item><term><italic>α</italic></term><def><p id=\"Par45\">Initial sorption rate (mg/g/min)</p></def></def-item><def-item><term><italic>β</italic></term><def><p id=\"Par46\">Extent of surface coverage and activation energy for chemisorption (g/mg)</p></def></def-item><def-item><term><italic>τ</italic></term><def><p id=\"Par47\">Time taken for 50% adsorbate breakthrough (min)</p></def></def-item></def-list></glossary></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02787-3</article-id><article-id pub-id-type=\"manuscript\">2787</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02787-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: In silico investigation of the therapeutic and prophylactic potential of medicinal substances bearing guanidine moieties against COVID-19</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Esam</surname><given-names>Zohreh</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Akhavan</surname><given-names>Malihe</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Lotfi</surname><given-names>Maryam</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Pourmand</surname><given-names>Saeed</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-7155-4532</contrib-id><name><surname>Bekhradnia</surname><given-names>Ahmadreza</given-names></name><address><email>abekhradnia@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027873_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411623.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 2227 0923</institution-id><institution content-type=\"org-division\">Pharmaceutical Sciences Research Centre, Department of Medicinal Chemistry</institution><institution content-type=\"org-name\">Mazandaran University of Medical Sciences</institution></institution-wrap><addr-line content-type=\"city\">Sari</addr-line><country country=\"IR\">Iran</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-division\">The Multiscale Modelling Lab</institution><institution content-type=\"org-name\">ITQB NOVA</institution></institution-wrap><addr-line content-type=\"street\">Av. da República</addr-line><addr-line content-type=\"postcode\">2780-157</addr-line><addr-line content-type=\"city\">Oeiras</addr-line><country country=\"PT\">Portugal</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412831.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 1172 3536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, Faculty of Chemical and Petroleum Engineering</institution><institution content-type=\"org-name\">University of Tabriz</institution></institution-wrap><addr-line content-type=\"city\">Tabriz</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027873_cor5\"><label>e</label><email>abekhradnia@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>30</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"52\">8</issue><fpage>4747</fpage><lpage>4747</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>30</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s11696-022-02528-y\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2787.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Correction to: Chemical Papers (2023) 77:1129–1148</bold><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link></p><p id=\"Par2\">The corresponding author asked if one of the affiliations should be deleted.</p><p id=\"Par3\">The affiliation is given below.</p><p id=\"Par4\">103 CBB, Department of Chemistry and Biochemistry, Montana State University, Bozeman, MT, 59717, USA</p><p id=\"Par5\">The Original Article is now updated.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"retraction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02818-z</article-id><article-id pub-id-type=\"manuscript\">2818</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02818-z</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Retraction Note</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Retraction Note: Nonanuclear coordination complex based on {Co<sub>9</sub>L<sub>12</sub>}<sup>6+</sup> cores: protective effect on coronary heart disease by reducing the inflammatory cytokines releasing</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Dong</surname><given-names>Fengquan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jiang</surname><given-names>Weipeng</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Jiang</surname><given-names>Yajuan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><name><surname>Li</surname><given-names>Xiaozheng</given-names></name><address><email>xiaozheng_li@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor4\">d</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><name><surname>Li</surname><given-names>Haiying</given-names></name><address><email>haiying_li666@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">Department of Cardiology</institution><institution content-type=\"org-name\">Shenzhen University General Hospital</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-name\">Shenzhen University Clinical Medical Academy</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">College of Life Science and Oceanography</institution><institution content-type=\"org-name\">Shenzhen University</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302818z_cor4\"><label>d</label><email>xiaozheng_li@126.com</email></corresp><corresp id=\"IDs1169602302818z_cor5\"><label>e</label><email>haiying_li666@126.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"53\">8</issue><fpage>4749</fpage><lpage>4749</lpage><history><date date-type=\"registration\"><day>6</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"retracted-article\" xlink:href=\"10.1007/s11696-020-01390-0\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2818.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Retraction Note to: Chemical Papers (2020) 75:1355–1362</bold></p><p><ext-link ext-link-type=\"uri\" xlink:href=\"https://doi.org/10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link></p><p id=\"Par2\">The Editor-in-Chief has retracted this article because the ethics documentation provided by the authors was inconsistent with the information in the manuscript. The authors did not provide raw data on request from the journal. In addition, Xiaozheng Li stated that their name was added to the article without their knowledge and they did not contribute to the study. The Editor has therefore lost confidence in the conclusions of this article.</p><p id=\"Par3\">Xiaozheng Li and Haiying Li agree to this retraction. Fengquan Dong, Weipeng Jiang and Yajuan Jiang have not responded to any correspondence from the editor about this retraction.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-019-00721-6</article-id><article-id pub-id-type=\"manuscript\">721</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-019-00721-6</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Meesilp</surname><given-names>Nutthawut</given-names></name><address><email>nutthawutm@hotmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10068019007216_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Mesil</surname><given-names>Nutthisha</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.443999.a</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0504 2111</institution-id><institution content-type=\"org-division\">Department of Applied Biology, Faculty of Science and Liberal Art</institution><institution content-type=\"org-name\">Rajamangala University of Technology Isan</institution></institution-wrap><addr-line content-type=\"postcode\">30000</addr-line><addr-line content-type=\"city\">Nakhon Ratchasima</addr-line><country country=\"TH\">Thailand</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.9786.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 0856</institution-id><institution content-type=\"org-division\">Srinagarind Hospital</institution><institution content-type=\"org-name\">Khon Kaen University</institution></institution-wrap><addr-line content-type=\"postcode\">40002</addr-line><addr-line content-type=\"city\">Khon Kaen</addr-line><country country=\"TH\">Thailand</country></aff></contrib-group><author-notes><corresp id=\"IDs10068019007216_cor1\"><label>a</label><email>nutthawutm@hotmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>6</day><month>12</month><year>2019</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"13\">9</issue><fpage>1297</fpage><lpage>1297</lpage><history><date date-type=\"registration\"><day>23</day><month>11</month><year>2019</year></date><date date-type=\"online\"><day>6</day><month>12</month><year>2019</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2019</copyright-statement><copyright-year>2019</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10068-018-0448-4\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2019</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>11</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>23</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2019_Article_721.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link>.</p></sec></notes></front><body><sec id=\"Sec1\"><title>Correction to: Food Sci Biotechnol (2019) 28(1):289–296 <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link></title><p id=\"Par1\">The article “Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk”, written by “Nutthawut Meesilp and Nutthisha Mesil”, was originally published electronically on the publisher’s Internet portal (currently SpringerLink) on 11 August 2018 with open access.</p><p id=\"Par2\">With the author(s)’ decision to step back from Open Choice, the copyright of the article changed on 6 December 2019 to © The Korean Society of Food Science and Technology 2019 and the article is forthwith distributed under the terms of copyright.</p><p id=\"Par3\">The original article has been corrected.</p></sec></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02795-3</article-id><article-id pub-id-type=\"manuscript\">2795</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02795-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Superior possibilities and upcoming horizons for nanoscience in COVID-19: noteworthy approach for effective diagnostics and management of SARS-CoV-2 outbreak</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Bhattacharjee</surname><given-names>Bedanta</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Ikbal</surname><given-names>Abu Md Ashif</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Farooqui</surname><given-names>Atika</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-5671-6591</contrib-id><name><surname>Sahu</surname><given-names>Ram Kumar</given-names></name><address><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff4\">4</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027953_cor4\">d</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Ruhi</surname><given-names>Sakina</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Syed</surname><given-names>Ayesha</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Miatmoko</surname><given-names>Andang</given-names></name><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Khan</surname><given-names>Danish</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au9\"><name><surname>Khan</surname><given-names>Jiyauddin</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Girijananda Chowdhury Institute of Pharmaceutical Science</institution></institution-wrap><addr-line content-type=\"postcode\">784501</addr-line><addr-line content-type=\"city\">Tezpur</addr-line><addr-line content-type=\"state\">Assam</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411460.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 4538</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Assam University (A Central University)</institution></institution-wrap><addr-line content-type=\"postcode\">788011</addr-line><addr-line content-type=\"city\">Silchar</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.413565.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 2452</institution-id><institution content-type=\"org-name\">The Deccan College of Medical Sciences</institution></institution-wrap><addr-line content-type=\"street\">Kanchan Bagh</addr-line><addr-line content-type=\"postcode\">500058</addr-line><addr-line content-type=\"city\">Hyderabad</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412161.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0681 6439</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Hemvati Nandan Bahuguna Garhwal University (A Central University)</institution></institution-wrap><addr-line content-type=\"street\">Chauras Campus</addr-line><addr-line content-type=\"postcode\">249161</addr-line><addr-line content-type=\"city\">Tehri Garhwal</addr-line><addr-line content-type=\"state\">Uttarakhand</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">Department of Biochemistry</institution><institution content-type=\"org-name\">IMS, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-name\">International Medical School, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440745.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0152 762X</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences, Faculty of Pharmacy</institution><institution content-type=\"org-name\">Universitas Airlangga</institution></institution-wrap><addr-line content-type=\"postcode\">60115</addr-line><addr-line content-type=\"city\">Surabaya</addr-line><addr-line content-type=\"state\">East Java</addr-line><country country=\"ID\">Indonesia</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Panineeya Institute of Dental Science and Research Centre</institution><institution content-type=\"org-name\">Kalonji Narayana Rao University of Health Sciences</institution></institution-wrap><addr-line content-type=\"postcode\">506007</addr-line><addr-line content-type=\"city\">Warangal</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">School of Pharmacy</institution><institution content-type=\"org-name\">Management and Science University</institution></institution-wrap><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027953_cor4\"><label>d</label><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>4</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"3\">8</issue><fpage>4107</fpage><lpage>4130</lpage><history><date date-type=\"registration\"><day>20</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>21</day><month>1</month><year>2023</year></date><date date-type=\"accepted\"><day>18</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>4</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">The outbreak of COVID-19 has caused great havoc and affected many parts of the world. It has imposed a great challenge to the medical and health fraternity with its ability to continue mutating and increasing the transmission rate. Some challenges include the availability of current knowledge of active drugs against the virus, mode of delivery of the medicaments, its diagnosis, which are relatively limited and do not suffice for further prognosis. One recently developed drug delivery system called nanoparticles is currently being utilized in combating COVID-19. This article highlights the existing methods for diagnosis of COVID-19 such as computed tomography scan, reverse transcription-polymerase chain reaction, nucleic acid sequencing, immunoassay, point-of-care test, detection from breath, nanotechnology-based bio-sensors, viral antigen detection, microfluidic device, magnetic nanosensor, magnetic resonance platform and internet-of-things biosensors. The latest detection strategy based on nanotechnology, biosensor, is said to produce satisfactory results in recognizing SARS-CoV-2 virus. It also highlights the successes in the research and development of COVID-19 treatments and vaccines that are already in use. In addition, there are a number of nanovaccines and nanomedicines currently in clinical trials that have the potential to target COVID-19.\n",
       "</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>COVID-19</kwd><kwd>Computer tomography</kwd><kwd>Polymerase chain reaction</kwd><kwd>Biosensor</kwd><kwd>SARS-CoV-2 virus</kwd><kwd>Nanovaccines</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>20</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2795.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Viruses like Middle East Respiratory Syndrome (MERS), Zika, Ebola, Influenza A (H1N1), and Severe Acute Respiratory Syndrome (SARS) have spread around the world in the last 20 years. Health policy and economic relations have been profoundly affected by these pandemics (Boopathi et al. <xref ref-type=\"bibr\" rid=\"CR21\">2021</xref>). The most recent form of the virus is the novel coronavirus, which has been recognized globally as a thoroughly transmissible disease since 2019. It is commonly known as novel COVID-19 (SARS-CoV-2) (Lu and XZ <xref ref-type=\"bibr\" rid=\"CR112\">2020</xref>; Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). The virus was firstly discovered in the Wuhan city of China. The infected patients exhibited unusual pneumonia-like symptoms that later spread globally (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). Coronavirus has caused great concerns due to its rapid diversity and a high number of new cases reported that resulted in huge economic loss worldwide. The structure of coronavirus consists of an envelope of spikes in its outer layer. Its total height is approximately 32 kb, with a single-stranded RNA (Benzigar et al. <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>). At the time of replication of polyprotein translations, it would act on the mRNA. Human SARS-CoV-2 has a spherical morphology with a dimension of around 125 nm (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>) (Daemi et al. <xref ref-type=\"bibr\" rid=\"CR43\">2021</xref>). The coronaviridae family and the nidovirales order comprise the virus evolutionary lineage (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Su et al. <xref ref-type=\"bibr\" rid=\"CR177\">2016</xref>). Coronavirus is subdivided into four main categories, referred to as: alpha (<italic>α</italic>), beta (<italic>β</italic>), gamma (<italic>γ</italic>), and delta (<italic>δ</italic>) and all of which have been recognized to date. Some of the common coronavirus for human are HKU1, NL63, MERS-CoV, OC43, SARS-CoV, and 229E. Alpha coronavirus consists of NL63 and 229E, whereas the SARS-CoV, MERS-CoV, OC43, and HKU1 families are examples of a beta coronavirus (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>). Four crucial structural proteins are encoded by the coronaviral genome, namely spike (S) protein, nucleocapsid (N) protein, membrane (M) protein, and the envelope (E) protein, that collectively constitute a complete viral entity (Masters <xref ref-type=\"bibr\" rid=\"CR120\">2006</xref>). Every protein plays a specialized function in the construction of the viral genome. These proteins have a central position in the replication process as well. The greatest predominant antigen in the viruses is the S glycoprotein, which is called the spikes of the virus (Jaimes et al. <xref ref-type=\"bibr\" rid=\"CR75\">2020</xref>). It has the ability to interact with the receptor and then fuse with the host membranes. The fused viral protein has branches in two different ways: first, it uses the S1 protein of the primary subunit to attach to the receptor. By reserving the cell membranes of the receptor, the S2 subunit would promote infusion. After targeting the virus, it would be structurally modified by the proteolytic enzymes. The peptide would then be incorporated into the cell membrane. Finally, irreversible cyclization would stress the S2 subunit of the fusion protein, which in turn would be mediated by ACE -2 (angiotensin-converting enzyme 2), allowing the viral genome to enter the cytoplasm (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Shafiee et al. <xref ref-type=\"bibr\" rid=\"CR170\">2021</xref>; Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>).<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The SARS-CoV-2 virus impacting humans is depicted in this illustration</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig1_HTML.png\"/></fig><fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Invasion of SARS-CoV-2 via ACE-2. micelles: TPGSM</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig2_HTML.png\"/></fig></p><p id=\"Par3\">COVID-19 is considered a pandemic of the world’s most significant humanitarian catastrophes due to its uncontrollable spread worldwide, huge loss of life, and impact on the world’s economy. Huge efforts have been done to overcome the disease and prevail over the pandemic (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>; Ziegler et al. <xref ref-type=\"bibr\" rid=\"CR223\">2020</xref>; Comentale et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). The most common symptoms among infected people are fever, loss of taste and smell, fatigue, and cough. In severe cases, several other symptoms are also reported, including difficulty breathing, pneumonia, and severe darkening of the lungs. Patients who present with such signs are evaluated by chest radiographs or computed tomography (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>; Yang et al. <xref ref-type=\"bibr\" rid=\"CR211\">2020</xref>; Weiss et al. <xref ref-type=\"bibr\" rid=\"CR200\">2020</xref>). Officially, the coronavirus was termed as SARS-CoV-2 or 2019- nCoV (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Munster et al. <xref ref-type=\"bibr\" rid=\"CR128\">2020</xref>; Liu et al. <xref ref-type=\"bibr\" rid=\"CR109\">2020</xref>; Habibi et al. <xref ref-type=\"bibr\" rid=\"CR59\">2020</xref>; Heymann and Shindo <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR198\">2020</xref>).</p><p id=\"Par4\">Nanotechnology is pertinent in managing and treating a wide range of disorders. Applications of nanomedicine include immune engineering, drug delivery, nanosensors, cancer nanovaccines, and platform technologies (Chakravarty and Vora <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>). Since the existing antiviral agents have truncated bioavailability and a narrow spectrum, nanomaterials are crucial for antiviral treatment. ENMs (engineered nanomaterials) have easy tunable physicochemical, low cost, high efficiency, and efficacy of drug delivery with lesser side effects (Chen et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>). The cell–virus interactions can be adjusted, particularly their interaction with cell receptors or virus particles, by adjusting the shape, size, and surface of ENMs (Ye et al. <xref ref-type=\"bibr\" rid=\"CR212\">2015</xref>). Nanotechnologies can be used for rapid detection of COVID-19 to further improve the efficiency of diagnostic tests (Rabiee et al. <xref ref-type=\"bibr\" rid=\"CR153\">2020</xref>). Organisms with a higher incidence of susceptibility can also be diagnosed using nanomedicines, which act as a prophylactic and alternative means of diagnosing infectious diseases (Vahedifard and Chakravarthy <xref ref-type=\"bibr\" rid=\"CR190\">2021</xref>). Magnetic nanoparticles and quantum dots are the nanoparticles that are commonly employed to detect coronavirus. The protein corona sensor array technology, which would aid in identifying protein/biomolecule sequences at an early stage, is at the forefront of catastrophic COVID-19 (Misra et al. <xref ref-type=\"bibr\" rid=\"CR123\">2021</xref>). The design and implementation of nanotechnology facilitate the production of new nanovaccines with a higher efficacy. One such efficient delivery strategy for treating SARS-CoV-2 is a nanoloaded formulation that is currently being explored. In addition, novel nanovaccines and nanotherapeutics are being developed to address SARS-CoV-2 infection (Jan et al. <xref ref-type=\"bibr\" rid=\"CR76\">2022</xref>). Therefore, researchers are making tremendous efforts to discover novel nano-based technologies and develop relevant nanovaccines and create effective therapies for SARS-CoV-2 infections. This article focuses on the unique and innovative nanotechnology strategies, detection, and prevention against COVID-19 and the potential for future research.</p></sec><sec id=\"Sec2\"><title>Diagnosis of SARS-CoV-2 infestation</title><sec id=\"Sec3\"><title>Computed tomography scan</title><p id=\"Par5\">A computed tomography (CT) scan is used to examine the internal organs. Chest radiographs and 3D cross-sectional images are utilized to identify the symptoms of COVID-19 (Shah et al. <xref ref-type=\"bibr\" rid=\"CR171\">2021</xref>). Patchy lung lesions of small size are seen in the early stages. One of the reports stated that CT scans are more likely to be tactful in the first stage of infection than RT-PCR (Law et al. <xref ref-type=\"bibr\" rid=\"CR100\">2015</xref>). CT scan helps to better understand the condition of the lung and the severity of the disease. It helps the physician to treat the patient appropriately depending on the severity (Bernheim et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). However, the usage of CT scans for COVID-19 has some challenges, including low specificity, which is only about 25%. As with asymptomatic patients, there are also opportunities for uncertainty with CT scans (Alafif et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021</xref>).</p><p id=\"Par6\">Furthermore, the diagnostic images of CT scans could overlap with other diseases like pneumonia. Repeated radiation exposure also would inflict long-term negative effects (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>). Additionally, it is an expensive tool to utilize.</p><p id=\"Par7\">On chest X-rays, it is common to see recurring mottling, subsegmental, or segmental morning opacities in both lungs, and atypical findings are becoming more common (Kaur et al. <xref ref-type=\"bibr\" rid=\"CR83\">2020</xref>). However, CT radiation is associated with minimal carcinogenic risk. It is currently unclear how important CT is for the management of patients with COVID-19. Presently, no antiviral therapy is available for 2019-nCoV, which is not the case in pulmonary TB patients. Drug treatment for tuberculosis can be started without a positive pathogenic Mycobacterium tuberculosis smear or tuberculosis culture if a clinical diagnosis is made that is supported by imaging. In cases of COVID-19, CT may show symptoms similar to those of viral pneumonia, supporting the clinical diagnosis (Zhao et al. <xref ref-type=\"bibr\" rid=\"CR221\">2020</xref>). In contrast, a change from a suspicious to a clinically diagnosed COVID-19 case classification has no impact on clinical therapy (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>).</p></sec><sec id=\"Sec4\"><title>Reverse transcription-polymerase chain reaction (RT-PCR)</title><p id=\"Par8\">RT-PCR would establish the criteria for COVID-19 diagnosis. Although COVID-19 is based on the extent of chronic respiratory complication from viruses, the practical detection approach appears to be less reliable. The SARS-CoV-2 virus is a single-stranded RNA virus with a positive direction of infection. Therefore, the most accurate approach for identification is RT-PCR. This approach requires the conversion of RNA to its supporting DNA, followed by amplification for detection (Harvey and O’Regan <xref ref-type=\"bibr\" rid=\"CR64\">2020</xref>). The E, S, N protein, and the RNA-dependent RNA polymerase gene (RdRp) portions of the entire genome, and ORF1ab are the most common targets for RT-PCR-based detection. These have shown good sensitivity, with the exception of RdRp, as it has lower sensitivity due to reverse primer-template mismatches than the rest (Huang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2020</xref>). The product of PCR amplification analysis is based upon CT (cycle threshold) value, <italic>i.e., </italic>the threshold value found in the duration of the amplification of the viral genome and control, <italic>i.e., </italic>negative control, standard control, and positive control. The positivity criteria of COVID-19 are analyzed based on the measurement of CT score. The CT score is a number that shows how long it took for the total number of PCR sessions to exceed the baseline fluorescence emission value. In the case of infection, the CT score cut-off is between 35 and 40 with a higher CT score indicating a lower viral load in the specimen. Thus, patients with a CT score of less than 35 would be classified as COVID-19 favorable, whereas patients with a CT score greater than 40 would be classified as clinically negative (Lai et al. <xref ref-type=\"bibr\" rid=\"CR97\">2020</xref>; Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>).</p><p id=\"Par9\">The virus accumulates in the nasopharynx, so a swab is taken from these body sites. Various biochemical reagents are used to remove impurities such as lipids and proteins to isolate only the RNA from the sample. The genetic information of the individual and, if applicable, the RNA of the virus are combined to form this RNA. A special enzyme is needed to convert the RNA into DNA. Then, researchers insert micro-DNA fragments that complement specific zones of replicated virulent DNA. When the virus is identified in the sample, these fragments bind to the target sites of the virulent DNA. During the amplification process, some parts of the genome are added to design the DNA sequence. On the other hand, DNA fragments are produced and attached to the sequences with labels to identify the virus. The device RT-PCR is then employed to test the sample (Udugama et al. <xref ref-type=\"bibr\" rid=\"CR188\">2020</xref>). The apparatus heats and cools the samples simultaneously, activating biochemical reactions that result in additional duplicated versions of the virulent DNA target regions (Craw and Balachandran <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). The process is repeated infinitely to copy the viral DNA target regions. A conventional real-time system RT-PCR runs for 35 sessions, which means that each viral strand in the material will have produced approximately 35 billion additional versions of the viral DNA at the end of the process. As additional versions of the virulent DNA fragments are produced, the tags combine with the DNA sequences. The result is the release of a fluorescent dye that is tracked by the computer device and displayed on the monitor in real time. The computer device logs the amount of fluorescence in the sample at the end of each session. The prevalence of the virus is determined when a predetermined fluorescence threshold is exceeded (Rao et al. <xref ref-type=\"bibr\" rid=\"CR158\">2022</xref>). To assess the severity of the disease, experts also monitor the number of sessions required to do so: The fewer sessions required, the more severe the viral contamination (Mohammadniaei et al. <xref ref-type=\"bibr\" rid=\"CR125\">2021</xref>). The real-time PCR method is precise and accurate and can provide a reliable diagnosis within 3 h, whereas laboratory methods typically require 6–8 h. Real-time PCR (RT-PCR) is much faster and has less potential for inaccuracy than other known viral extraction methods because the entire process can be performed in a closed container (Wu et al. <xref ref-type=\"bibr\" rid=\"CR202\">2013</xref>).</p></sec><sec id=\"Sec5\"><title>Nucleic acid sequencing</title><p id=\"Par10\">Detection of SARS-CoV-2 can be sequence-specific based on nucleic acid sequencing. It is considered one of the most precise scientific methods for detecting COVID-19 as it can examine the whole genomic makeup of the virus (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>). Sanger sequencing is a type of sequencing in which a patient's sample is recovered to recover viral RNA and spike-in control RNA that is different from viral RNA. Most of the diagnoses of COVID-19 are based on next-generation sequencing such as oxford nanopore sequencing and illumine sequencing. Unlike Sanger sequencing, the next-generation sequencing would concurrently target billions of target sequences (Quick et al. <xref ref-type=\"bibr\" rid=\"CR152\">2017</xref>). On the other hand, the amount of virus remaining in the specimen is measured and normalized based on the spiked RNA (Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>; Yu et al. <xref ref-type=\"bibr\" rid=\"CR214\">2021</xref>).</p><p id=\"Par11\">Assays based on nucleic acids usually take 12 to 24 h or longer and are thought to be the best way to identify microorganisms. On the other hand, nucleic acid-based assays offer a much shorter reaction time and excellent sensitivity and specificity, bringing them to the forefront of laboratory and, in some circumstances, field analysis. In the laboratory, nucleic acid sequencing has been effectively used to detect both nonpathogenic and pathogenic microorganisms. Ribosomal RNA techniques have been used to conduct similar studies of microbial diversity (Chandler-Brown et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>).</p><p id=\"Par12\">Sequence-based ID with nucleic acid testing is a key part of any defense against biological agent attacks. It helps find out whether a biological agent attack has happened and, if so, what kinds of biological agents were used. Even if the test response time is slow—on the order of 15–30 min—this confirmation function is critical (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p><p id=\"Par13\">The following is the sequence of processes that must occur within this response time: (1) collect the sample: prepare the sample for an assay. This includes the removal of assay inhibitors and lysis of target cells. This can be a serious problem with contaminated samples or when analyzing spore nucleic acids. (2) Performance of the actual assay: analyzing and reporting the assay results (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p></sec><sec id=\"Sec6\"><title>Immunoassay</title><p id=\"Par14\">COVID-19 can be diagnosed indirectly by looking at the patients’ immunological response. Recent research has been conducted to determine serologic surveillance of COVID-19 by collecting various individual fluids (Rizzo and Buck <xref ref-type=\"bibr\" rid=\"CR162\">2012</xref>). The target for serological diagnosis is mainly blood samples instead of the nasopharyngeal swab. Detection of an immune response is the most important requirement for verifying transmission of the virus. Immunoassay is a bioanalytical method based on the interaction between antibody and antigen that allows rapid detection of COVID-19. Two prime serological detections based on immunoassay are enzyme-linked immunosorbent assay (ELISA) and lateral flow assay (LFA). Targeting specific antibodies and antigens is done for the diagnosis of COVID-19. According to studies, a minimum of 5 days is required for sufficient viral load. For antibodies, however, it usually takes at least 7 days, and the concentration decreases after the infection has progressed for 7 days (Alcoba-Florez et al. <xref ref-type=\"bibr\" rid=\"CR8\">2020</xref>). Thus, if the viral load is minimal, this will lead to misleading negative results.</p></sec><sec id=\"Sec7\"><title>Point-of-care test</title><p id=\"Par15\">Point-of-care (POC) testing is essential for disease management during a pandemic. For this reason, it is important to understand the causative agent of a virus before developing tools for diagnosis. It allows for early diagnosis by detecting the analyte, which could optimize the effectiveness of the therapy (Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR84\">2020</xref>). The tools developed for the POC screening of COVID-19 are based on the detection of immunological components such as antibodies and antigen or viral RNA (Ma et al. <xref ref-type=\"bibr\" rid=\"CR115\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> highlights the existing approaches for identifying SARS-CoV-2.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>A list of the existing strategies for SARS-CoV-2 confirmation</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>SARS-CoV-2 confirmation approach</p></th><th align=\"left\"><p>Methodology</p></th><th align=\"left\"><p>Salient characteristics</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Enzymatic sensor system based on regularly interspaced short palindromic repeats for assay-specific great specificity</p></td><td align=\"left\"><p>When SARS-COV-2-specific guide RNA is activated, reporter RNA sequences are removed</p></td><td align=\"left\"><p>Execution time 1 h; inexpensive; simple instrumentation</p></td><td align=\"left\"><p>To et al. (<xref ref-type=\"bibr\" rid=\"CR185\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Enzyme-Linked immunosorbent assay</p></td><td align=\"left\"><p>Antibodies against viral proteins coated on microwell plates bound; Secondary antibody produces Fluorimetric/ Colorimetric signal</p></td><td align=\"left\"><p>High-throughput screening; quick; point of care test</p></td><td align=\"left\"><p>Zhang et al. (<xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>), Chen et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Transcription-mediated amplification</p></td><td align=\"left\"><p>Amplification of particular RNA/DNA segments</p></td><td align=\"left\"><p>Within 15–30 min, there is a ten-billion-fold rise; generates RNA amplicon; rapid kinetics</p></td><td align=\"left\"><p>Phan et al. (<xref ref-type=\"bibr\" rid=\"CR146\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Metagenomic sequencing</p></td><td align=\"left\"><p>Amplification using a single primer that is not sequence-dependent</p></td><td align=\"left\"><p>The SARS-COV-2 mutation rate is analyzed</p></td><td align=\"left\"><p>Carter et al. (<xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Chest Computer Tomography Scans</p></td><td align=\"left\"><p>X-rays of the patient's chest from multiple angles</p></td><td align=\"left\"><p>The imaging characteristics of viral pneumonia are similar. Indeterminate for SARS-CoV-2; non-invasive</p></td><td align=\"left\"><p>Hardick et al. (<xref ref-type=\"bibr\" rid=\"CR63\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Lateral Flow Immunoassay</p></td><td align=\"left\"><p>A sample is passed through an immobilized viral antigen. Color develops in the presence of anti-CoV antibodies, as do marker antibodies</p></td><td align=\"left\"><p>Does not require a professional staff; expensive; quick (10–30 min)</p></td><td align=\"left\"><p>Arslan et al. (<xref ref-type=\"bibr\" rid=\"CR10\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription loop-mediated isothermal amplification</p></td><td align=\"left\"><p>At the same temperature, amplification using four different primer systems</p></td><td align=\"left\"><p>In place of a heating cycler, a heating bath is provided; expeditious (&lt; 13 h); expensive</p></td><td align=\"left\"><p>Calucho et al. (<xref ref-type=\"bibr\" rid=\"CR26\">2020</xref>), Jiang et al. (<xref ref-type=\"bibr\" rid=\"CR78\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription-polymerase chain reaction</p></td><td align=\"left\"><p>Virus genomic RNA amplification</p></td><td align=\"left\"><p>Gold standard; sample requirement is less</p></td><td align=\"left\"><p>Park et al. (<xref ref-type=\"bibr\" rid=\"CR140\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par16\">POC tests are procedures performed with a test strip or kit at home or at the healthcare facility where the person is being treated. The biosensor is the most important component of POC screening because it is required to perform molecular analysis to identify the virus. POC screening has the following advantages: (i) fewer storage and experimental facilities are needed; (ii) it is adaptable to a variety of medical problems; (iii) it is a significant test; and (iv) multiple sites can be used for evaluation (Asdaq et al. <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par17\">Each POC test is approved for use with a specific type of specimen and can only be used with those specimens. Appropriate specimen procurement and handling are required for all COVID-19 assays, especially those performed in POC clinics. An erroneous or inaccurate test result may result from improper specimen submission or collection. Employees who collect specimens within 6 feet of persons must wear the required personal protective equipment (PPE). It consists of gloves, safety glasses, N95 protection, and a lab coat or robe. All personnel collecting specimens not actively involved in specimen collection should follow standard safety guidelines and not be more than one meter away from the patient (Dincer et al. <xref ref-type=\"bibr\" rid=\"CR47\">2017</xref>).</p></sec><sec id=\"Sec8\"><title>Detection from breath</title><p id=\"Par18\">For the objective of monitoring COVID-19, screening of expelled breath may be a less intrusive form of analysis (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Broza et al. <xref ref-type=\"bibr\" rid=\"CR25\">2018</xref>). However, screening SARS-CoV-2 from exhaled air has proven to be quite complicated. These findings show that COVID-19 individuals breathed billions of RNA molecules of severe acute respiratory syndrome coronavirus every extra few minutes (Broza et al. <xref ref-type=\"bibr\" rid=\"CR24\">2015</xref>). According to scientific interpretation, exhaled air had a greater affirmative incidence (26.7%) than surface samples (5.4%) and air (3.8%). In addition, exhaled breath condensate (EBC), a unique technique for quickly identifying viruses from exhaled breath, was required to collect the sample for a longer period of time..</p><p id=\"Par19\">Furthermore, recent studies have shown that non-volatile components such as viruses, bacteria, DNA, and RNA, can be easily detected and observed by extracting and analyzing the aqueous fraction of expelled breath (Expelled Breath Aerosol (EBA) or EBC) (Ma et al. <xref ref-type=\"bibr\" rid=\"CR116\">2021</xref>). Depending on these two factors, an EBC system can effectively detect different droplets: (1) the percentage of the virus still alive after extraction and (2) the ratio between the percentage of extracted particles and the total percentage of particles in the air. For example, Haick and colleagues conducted a scientific investigation in March 2020 in Wuhan city of China, featuring screening with a breath scanner that follows the principle of chemoreceptive devices formed of gold nanoparticles combined with artificial intelligence algorithms (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Lamote et al. <xref ref-type=\"bibr\" rid=\"CR99\">2020</xref>). In another experiment, researchers investigated preliminary markers of enhanced reactive oxygen species formation in swab samples. In this approach, swab samples were exposed to a biosensor nanostructured with carbon nanofibers, which provided 97% accurate confirmatory readings in about 30 s (Shan et al. <xref ref-type=\"bibr\" rid=\"CR172\">2020</xref>).</p></sec><sec id=\"Sec9\"><title>Nanotechnology-based biosensors for COVID-19</title><p id=\"Par20\">Biosensors are devices containing a physical transducer and a biorecognition element that can be used to diagnose the analytes present in solutions, liquids, and bodily fluids. A physical sensor is used to produce a response after an analyte coincides with a biological factor and transforms it into a measurable quantity. Biorecognition elements are glycoproteins (antibodies) or enzymes, nucleic acids such as RNA, DNA, biological receptors, organic receptors, tissues, and whole cells (Miripour et al. <xref ref-type=\"bibr\" rid=\"CR122\">2020</xref>). As per the specifications, enormous numbers of biosensors can be fabricated. Nanomaterials are beneficial as they provide the surface chemistry that would aid in molecular bioconjugation, powerful expanding effect on signals, and large surface energy. Metallic nanoparticles such as silver and gold nanoparticles, nanogels, carbon nanomaterials, <italic>i.e.,</italic> graphene and nanotubes, microgels, and phonic crystals, are utilized in the application of biosensors. Meanwhile, a piezoelectric and electrochemical method based on nanocarbon particles has been developed for the detection of bacteria (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>).</p><p id=\"Par21\">A biosensor detects solutes in bodily fluids, liquids, and by combining a biological sensing device with a mechanical detector. When a solute interferes with a biological component, an output signal is generated, in which the physical detector then converts into a scientifically and quantitatively detectable component. Proteins such as antibodies or enzymes, nucleic acids such as RNA or DNA, physiological receptors, whole tissues, and cells are among the biological detector elements, while electrochemical, photonic, and piezoelectric devices are often used as physical detectors (Bandodkar and Wang <xref ref-type=\"bibr\" rid=\"CR15\">2014</xref>). Biosensor elements can be manufactured in large quantities to meet different requirements. Nanomaterials can be used as detectors due to their excellent resistance in a wide range of environments and bio-compatibility with biological systems (Sharma et al. <xref ref-type=\"bibr\" rid=\"CR174\">2020</xref>). Additionally, nanomaterials have high interfacial potential, robust signal amplification capability, and surface chemistry that can help bioconjugate biomolecules. The properties of photonic crystals, microgels, and magnetic nanostructures including gold nanostructures and silver, carbon-based nanoparticles (nanotubes and graphene), and nanocomposites have contributed to their use in biosensor technologies. The use of nanocarbon-based particles has facilitated microfluidic systems. These have already proven to be platforms for bacterial detection, using a range of strategies such as electrochemistry and piezoelectricity (Olvera and Monaghan <xref ref-type=\"bibr\" rid=\"CR138\">2021</xref>).</p></sec><sec id=\"Sec10\"><title>Development in nanotechnology-based COVID-19 diagnosis</title><p id=\"Par22\">When an infection occurs, immune globulin M (IgM) and immunoglobulin G (IgG) are produced. Initially, IgM antibodies appear, which are reduced to a minimum in general infection. Thereafter, IgG antibodies would develop and displace the IgM antibodies in the body, which remain in high concentration for a considerable time until they are identified after 12 weeks. In SARS-CoV-2 transmission, there is no latency between IgM and IgG confirmation, as both occur within 5–7 days of the onset of manifestations (Tarim et al. <xref ref-type=\"bibr\" rid=\"CR182\">2021</xref>). Several laboratory-based immunoassay methods, such as chemiluminescence immunoassay, can find antibodies in blood serum to the SARS-CoV-2 viral protein. The most commonly utilized method for detection is enzyme-linked immunosorbent assay. However, to identify contamination and conduct an on-site investigation, rapid and portable immunoassay detection is required. One promising, quick and portable platform for the point of care is lateral flow immunoassay, which uses colloidal gold nanoparticles (Deeks et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>). Gold nanostructures are coupled to the SARS-CoV-2 protein and immobilized in the intracellular membrane in a lateral flow immunoassay. The testing for SARS-CoV-2 virus detection would take roughly 15 to 20 min. Due to this, cross-reactivity with other antibodies may result in false-positive outcomes (Han et al. <xref ref-type=\"bibr\" rid=\"CR62\">2019</xref>).</p><p id=\"Par23\">The rate at which antibodies to SARS-CoV-2 are produced is the most important immunologic factor determining susceptibility to infection. RT-PCR can be replaced with immunologic screening to validate SARS-CoV-2. The rate of unequivocal virus detection is significantly increased when immunologic testing and real-time PCR are coupled (Lustig et al. <xref ref-type=\"bibr\" rid=\"CR114\">2021</xref>). In most individuals, IgM levels rise in the first week resulting from exposed to SARS-CoV-2, continue to rise until levels reach a maximum after two weeks, and then begin to decline until levels are near reference values. After one week, IgG is visible and it remains in significant concentration for a longer period of time.</p><p id=\"Par24\">IgG, on the other hand, can be seen after only one week. Its levels stay high for a longer time, probably up to 48 days, and it may help prevent a second infection. Immunoglobulin A (IgA) responses begin to manifest about 4 and 10 days after infection. IgM, IgA, and IgG expression in blood plasma is a diagnostic factor. Different targets can describe the variability of antibodies to SARS-CoV-2. After exposure to infection, antibody titers may decrease for up to 7 days. Specific antibodies to SARS-CoV-2 were recently detected in saliva. Experts investigated whether there are differences in antibody concentrations between serum and saliva using multiplex antibody immunoassays for SARS-CoV-2. The antibodies indicate parallel compartmental humoral immune responses in saliva that correspond to those in serum (Pan et al. <xref ref-type=\"bibr\" rid=\"CR139\">2020</xref>).</p></sec><sec id=\"Sec11\"><title>Viral antigen detection</title><p id=\"Par25\">Prior biosensing strategy would utilize complementary DNA (cDNA) or antibodies to obtain viral RNA or antigen. The existence of SARS-CoV-2 infection is identified by utilizing graphene field-effect transistors (GFET) as a biosensor. It helps in determining the seriousness of the infection (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>). The graphite film serves as the reading region of the FET-based biosensor, which is then transferred to the Si/SiO<sub>2</sub> substrate to successfully modify with the antibody of COVID-19. It is immobilized well on the surface of the graphene sheet by drop-casting. This instrument can detect COVID-19 antigen concentrations as low as 1 fg/ml in a phosphate buffer, which is substantially lower than the PCR/ELISA approach (Nguyen-Contant et al. <xref ref-type=\"bibr\" rid=\"CR134\">2020</xref>). The medium used is universal transport medium (UTM), with the assay used for biosensors for nasopharyngeal swab suspension.</p><p id=\"Par26\">A rapid assay method was developed to confirm the expression of the viral genome released by SARS-CoV-2 in samples from an infected person’s respiratory tract. In this experiment, the protein in the sample adheres to antibodies coupled with a sheet of paper in a glass container. Within half an hour, this reaction produces a detectable signal. The test could indicate intense or initial infection because the proteins found are expressed only when the pathogen is continuously replicating (Drobysh et al. <xref ref-type=\"bibr\" rid=\"CR49\">2022</xref>). In addition, Abbott has introduced a more powerful version of the COVID-19 instant diagnostic test, which screens for antibodies in infected human blood. Abbott's diagnostic devices can detect antibodies to SARS-CoV-2 and are available on the ARCHITECT i1000SR and i2000SR scientific instruments, which can perform 100–200 tests per hour. Antibodies to SARS-CoV-2 are produced after one week of infection. Age, nutritional status, the severity of the disease, concomitant diseases, and medications can influence the intensity of the antibody response (Seo et al. <xref ref-type=\"bibr\" rid=\"CR169\">2020</xref>).</p></sec><sec id=\"Sec12\"><title>Microfluidic device</title><p id=\"Par27\">Another possibility for experiments with real evidence is microfluidic devices. These microfluidic crisps are imprinted with micrometer-sized circuits and reactivity units made of polydimethyl sulfoxide and paper and offer advantages such as reduced sample size, compact dimensions, and rapid detection time. These microfluidic crisps work by blending and separating fluid materials employing electrokinetics and capillary pressure, which is their core principle (Ramachandran et al. <xref ref-type=\"bibr\" rid=\"CR156\">2021</xref>).</p><p id=\"Par28\">Mass screening of people is recognized as a crucial step in limiting the virus proliferation. Some countries in America and Africa reported fewer cases due to the shortage of tests done. Nowadays, quantitative reverse transcriptase polymerase chain reaction (qRTPCR) is applied and endorsed by WHO for case confirmation. qRTPCR (quantitative reverse transcriptase polymerase chain reaction) primer, thermocyclers, specific reagent, and qualified personnel are the most important requirements (Sen et al. <xref ref-type=\"bibr\" rid=\"CR168\">2022</xref>). Microfluidics can be used as an alternative to the time-consuming assay. The emergence of microfluid devices is enabled due to the microelectronics and micro-electromechanical systems as they can manipulate a small number of fluids.</p><p id=\"Par29\">Moreover, they can also retrieve information from the small sample volume. Microfluidic devices are used at the early stage of diagnosis as recent researchers have found that they can detect viruses (Kalita et al. <xref ref-type=\"bibr\" rid=\"CR80\">2021</xref>). The genetic code that forms virus consists of an envelope of proteins and a lipid ligand. Proteins are detected by Western blot, PCR, and amplified genetic material. Many approaches based on benchtop assays are transferred to the microfluidic device to minimize the amount of reagents needed and to speed up the reaction. The microfluidic device would elaborate a structure that can detect the biological samples. The detector that uses the amplification of genetic code requires a specific DNA probe design called primer to find a target sequence in the virus's genetic material (Maggi et al. <xref ref-type=\"bibr\" rid=\"CR117\">2019</xref>).</p></sec><sec id=\"Sec13\"><title>Magnetic nanosensor</title><p id=\"Par30\">Gold nanoislands (AuNIs) were utilized to build plasmonic photothermal (PT) biosensors that target the ORF1ab, RdRp, and E genes to pinpoint SARS-CoV-2 RNA sequences. Nanosensors based on magnetic nanoparticles (MNPs) generate heat as a result of their optical properties when aggregate vibrations in energy concentration take place at the interface of nanomaterial via conjugating to magnetic radiation. The increased frequency of collisions with lattice atoms caused by the enhanced motion of conduction electrons leads to the generation of PT-induced heat (Hofmann et al. <xref ref-type=\"bibr\" rid=\"CR67\">2008</xref>; Moabelo et al. <xref ref-type=\"bibr\" rid=\"CR124\">2021</xref>; Esbin et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>).</p><p id=\"Par31\">Nanosensors based on magnetic nanoparticles (MNPs) are advantageous for viral pathogen identification (Islam and Ahsan <xref ref-type=\"bibr\" rid=\"CR72\">2020</xref>; Barnett et al. <xref ref-type=\"bibr\" rid=\"CR16\">2019</xref>). Tian et al. claim that nanosensors based on MNPs can be used for screening nucleic acids by combining uniform circle-to-circle amplification (UC2CA) and iron oxide NPs (IONPs). In this approach, the effect of an ambient magnetism allows both single and coupled IONPs to exhibit different optical properties (dispersion or absorption). When the recognition markers of the IONPs are paired with UC2CA, the coupled IONPs that produce the end amplicons (single-stranded DNA) are formed. Consequently, the configurations of the IONPs can be used to study the opto-magnetic parameters. This method has been shown to be accurate in distinguishing the genetic variants of SARS-CoV and SARS-CoV-2. In addition, it has been used to detect RdRp motifs for SARS-CoV-2 (Tian et al. <xref ref-type=\"bibr\" rid=\"CR183\">2020</xref>).</p></sec></sec><sec id=\"Sec14\"><title>Nanomedicine</title><p id=\"Par32\">The discipline of medicine known as \"nanomedicine\" exploits nanotechnology to treat and prevent disease. It is utilized in various applications of living organisms, including diagnostics, delivery, and actuation. It is described as using manmade nanodevices and nanostructures to monitor, create, and regulate human biological systems at the molecular level (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR219\">2022</xref>). The unique qualities of nanomedicines may hold the key to solving many of today's problems in cancer, cardiovascular and neurological illnesses, and sickness. The nanomedicine formulation imparts various beneficial properties such as lower side effects, molecular targeting by nanoengineered devices, better absorption with higher intestinal permeability, improved bioavailability, and prolonged and targeted medication delivery. The main benefit of nanomedicines in several current applications is improving drug delivery for therapy. The main mechanism of drug delivery based on nanomedicine called increased bioavailability, and retention is the example of selective medication tailoring (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>). It helps to increase absorbability and can be removed from the body waste before treatment, which can be effective. Nanomedicine is vital in improved efficacy, bioavailability, and targeting ability and is safer than conventional medicines. Nanomedicine influences all medicine sectors and is regarded as a significant tool for vaccinations, medical imaging, innovative diagnostics, nanotherapeutics, and the development of biomaterials for restorative medicine (Ledet and Mandal <xref ref-type=\"bibr\" rid=\"CR102\">2012</xref>). Surfactants (liquid crystals, nanoemulsions, and microemulsions), polymers (polymeric nanoparticles), proteins (protein nanoparticles), and lipids (liposomes, lipid–solid nanoparticles, and nanostructured lipid carriers) have all been employed in nanomedicine. The strength of interactions between biological/tissue molecules and nanoparticles is the foundation of their use in various medicinal applications (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Soft nanomaterials are primarily used to enhance the pharmacokinetic, pharmacodynamics, and biopharmaceutical aspects of drug loading. The performance and tolerability of the therapy may be impacted by the use of nanoparticles to enhance selective therapeutic localization (active or passive) and optimized the release rate of the drug. Due to their antimicrobial qualities, metallic, and polymer-based nanoparticles were also applied in nanomedicine (antiviral, antifungal, antibacterial, and antiparasitic) (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>).</p></sec><sec id=\"Sec15\"><title>Nanomedicine-based drug delivery system</title><p id=\"Par33\">Numerous conditions, including diabetes, cancer, cerebrovascular disease, and neurodegenerative disorder, can be treated with drug delivery systems based on nanotechnology. Some nanomedicines are available in the market through parenteral or oral administration. Novel nanomedicines can be prepared through several preclinical and clinical trials, which can be administered through different routes such as nasal, ocular, vaginal, dermal and pulmonary. Several developed formulations based on nanomaterials are discussed below.</p><sec id=\"Sec16\"><title>Liposomes</title><p id=\"Par34\">Alec Bangham developed liposomes in the year 1960. Cosmetics and pharmaceutical industries have utilized liposomes to transport drugs and other diverse molecules. Liposomes are considered a fixed formulation method to improve the drug delivery system. Liposomes are spherical-shaped vesicles, usually in the 50–450 nm, composed of steroids and phospholipids. They are regarded as a good vehicle for drug delivery due to their ability to incorporate drugs and possess an analogous membrane structure (Kupferschmidt and Cohen <xref ref-type=\"bibr\" rid=\"CR96\">2020</xref>). Liposomes would improve biodistribution and build stable compounds that allow them to utilize hydrophobic and hydrophilic drugs. There are four types of liposomes. The first is a lipid bilayer in which aqueous hydrophilic and hydrophobic substances are surrounded by cationic, anionic, or neutral phospholipids and cholesterol. The second variety is PEGylated liposomes, which consist of PEG (polyethylene glycol) inserted onto their surface to achieve steric equilibrium. The ligand-targeted type is the third kind in which ligands such as carbohydrates, antibodies, and peptides are attached to the surface of liposomes. The fourth form is the theranostic, a combination of the first three types addressed prior that consists of a nanoparticle with imaging, targeting, and therapeutic elements (Bozzuto and Molinari <xref ref-type=\"bibr\" rid=\"CR23\">2015</xref>).</p><p id=\"Par35\">Li et al<italic>.</italic> prepared and characterized a liposomal remdesivir inhalation solution for targeted pulmonary delivery for COVID-19 therapy. The results indicate that liposomal remdesivir shows rapid conversion of remdesivir to active nucleoside triphosphate and its Tmax is 1 h. This observation might be due to the fact that liposomes have high drug loading and are biocompatible with cell membranes. This would lead to faster uptake of remdesivir by cells and in larger amounts (Li et al. <xref ref-type=\"bibr\" rid=\"CR106\">2021</xref>). The central composite design was used to optimize the ritonavir proliposomes, and the results of the physicochemical properties were appropriate. The rats treated with the liposomal formulation exhibited a higher amount of drug in the lymphoid tissues compared with the pure drug. The results suggest that the targeting potential of biotin-coated proliposomes of ritonavir is present (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>). Mice infected with either dead or live SARS-CoV-2 developed acute respiratory distress syndrome (ARDS). Treatment with tripterine liposomes significantly reduced the severity of pulmonary pathological changes and viral load. It was observed that tripterine liposomes reduced SARS-CoV-2 replication and hyperinflammation in mice and cells infected with the virus. Therefore, it is a potential option for a drug that could be used to treat ARDS caused by SARS-CoV-2 (Que et al. <xref ref-type=\"bibr\" rid=\"CR151\">2022</xref>).</p></sec><sec id=\"Sec17\"><title>Polymeric micelle</title><p id=\"Par36\">Micellar nanostructures known as polymeric micelles are composed of an amphiphilic block copolymer that has the ability to self-assemble into a core with a shell when introduced into an aquatic system. For example, docetaxel, camptothecin, and paclitaxel are hydrophobic drugs that can be loaded into the hydrophobic cavity. The hydrophilic coating would stabilize the core and make the system soluble in water when used. To avoid rapid excretion through the kidneys, polymeric micelles have a diameter of about 100 nm and have only a small volume of distribution. Therefore, they can be collected in tumor tissue due to their increased permeability and retention effect.</p><p id=\"Par37\">On the other hand, the hydrophobic medication can be integrated into the core structure, enhancing the drug's bioavailability and stability (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>; Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>). These micelles can be synthesized by precipitating one block and adding a solvent and direct dissolution of polymer in a convenient solvent-based followed by dialysis (Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR208\">2013</xref>). Micelle formation is affected by amphiphilic molecules in the hydrophobic chain, the concentration of amphiphiles, temperature, and solvent system (Kulthe et al. <xref ref-type=\"bibr\" rid=\"CR91\">2012</xref>).</p><p id=\"Par38\">Targeted drug delivery, nanoscopic core structure, minimal biodegradability, micellar coupling, biocompatibility, and much-improved longevity are all attributes of polymeric micelles that make them ideal for drug therapy (Chamundeeswari et al. <xref ref-type=\"bibr\" rid=\"CR29\">2019</xref>). Chaudhari and Handge reported the development, characterization, and validation of lopinavir-encapsulated polymeric micelles using co-solvent (Tween 80) and polymeric material (Pluronic F127 and Pluronic F68) (Chaudhari and Handge <xref ref-type=\"bibr\" rid=\"CR32\">2020</xref>). The novel drug formulation increased drug loading capacity, bioavailability, and encapsulation performance. Mahajan and Patil developed polymeric vitamin E-tocopheryl polyethylene glycol succinate micelles encapsulating lopinavir for oral therapy. The developed lopinavir polymeric micelles had a better dissolution rate. An in vivo pharmacokinetic study in New Zealand rabbits showed that the formulated drug had much higher bioavailability (3.17%) than lopinavir suspension (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>).</p></sec><sec id=\"Sec18\"><title>Dendrimers</title><p id=\"Par39\">Dendrimers are monodispersed 3-D entities that may be cleaved into two halves. They have a spherical shape whose surface can be uniformly functionalized, making them exceptional for drug delivery (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>; Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR86\">2015</xref>). The dendrimers can be synthesized in two ways: the dendrimers would start forming from the core and then outside of the dendrimers. The other method is convergent in which the formation would start from outside the dendrimers (Zhu and Shi <xref ref-type=\"bibr\" rid=\"CR222\">2013</xref>). There are various types of dendrimers based on their functionalization moieties such as polypropylene imine (PPI), polyamidoamine (PAMAM), core–shell, peptide, glycodendrimers, liquid crystalline, chiral and polyamidoamine-organosilicon (PAMAMOS). Since PAMAM is water-soluble and can penetrate epithelial tissues, it is mainly explored for oral drug delivery (Cheng et al. <xref ref-type=\"bibr\" rid=\"CR36\">2008</xref>). However, the applications of dendrimers are limited due to the presence of amino groups'. This group is cationic or positively charged, which makes them toxic. Therefore, the utilization of dendrimers requires some modifications to reduce their toxicities. Drugs are loaded into the dendrimers through electrostatic interaction, simple encapsulation, and covalent conjugation (Noriega-Luna et al. <xref ref-type=\"bibr\" rid=\"CR136\">2014</xref>). Drugs can be delivered in two ways. The first is in vivo delivery, where the degradation of the covalent bond of the dendrimers depends on the favorable environment and the enzymes available to cleave the bond. Ethynylene–perylene (EPer) would discharge the drug due to environmental changes such as temperature and pH (Tripathy and Das <xref ref-type=\"bibr\" rid=\"CR187\">2013</xref>). Drugs can be delivered orally, pulmonary, transdermally, ocularly, and targeted through dendrimers (Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR85\">2014</xref>).\n",
       "</p><p id=\"Par40\">Among the extensively documented studies on the application of dendrimer systems for therapeutic purposes COVID-19, numerous findings can be summarized. According to Itani et al<italic>.,</italic> dendrimers can connect robustly with viruses and COVID-19 suppress invasion in target cells. This comprehensive analysis explored the prospective importance of nanoparticles such as theranostic dendrimers as suitable transporters for immune stimulators or cures to help combat COVID-19 (Prusty and Swain <xref ref-type=\"bibr\" rid=\"CR224\">2018</xref>). In an experiment comparable to Middle East coronavirus, Kandeel and co-workers investigated the antiviral potential of 16 different polyamidoamine dendrimers against SARS-CoV-2. In these intriguing experiments, polycationic dendrimers comprising the primary amine (-NH2-) class (grade 2/3/4 and/or 5) and three different categories of polyanionic dendrimers comprising the amido ethanol hydroxyl class (grade 2/3/4 and/or 5), the sodium carboxylate end group class (grade 1.5/2.5/3.5 and/or 4.5), succinic acid class (grade 2/3/4 and/or 5) were structured and assumed to have calcification inhibitory activity in SARS-CoV-2. The grade 1.5 CO2-Na polyamidoamine dendrimer inhibited SARS-CoV-2 calcification growth most strongly (inhibitory effect 40%), followed by the grade 5 polyamidoamine dendrimer with succinic acid (inhibitory effect 39.7%), which contains terminal grades 16 and 128. Vero cells showed a strong toxic effect in the presence of cationic dendrimers. The authors suggested that antiviral agents could be incorporated into these dendrimers for use as nanomedicines (Itani et al. <xref ref-type=\"bibr\" rid=\"CR73\">2020</xref>).</p></sec><sec id=\"Sec19\"><title>Inorganic nanoparticles</title><p id=\"Par41\">Inorganic nanoparticles, including gold, silver, silica, and iron oxide, possess exceptional potential applications. Gold and silver-like nanoparticles have properties like surface plasmon resonance (SPR), which is not possessed by dendrimers, liposomes, or micelles. They also have several advantages that include versatility and biocompatibility. Drug conjugation with gold nanoparticles can occur through covalent or ionic bonds. In addition, silver nanoparticles can exhibit antimicrobial activities. For example, a spongy and interconnected dextran/polyacrylamide nanohydrogel matrix covalently coupled with AuNPs could release the ornidazole with a 98.5% in vitro (Kandeel et al. <xref ref-type=\"bibr\" rid=\"CR81\">2020</xref>).</p><p id=\"Par42\">Experimental research with iron oxide (IO) nanoparticles targeting SARS-CoV-2 was explored. IO and metal oxide nanoparticles with the versatile properties of zinc oxide, titanium dioxide, and copper oxide are effective in antiviral diagnosis and treatment. Titanium dioxide nanoparticles destroy pathogens by generating reactive oxygen species. This is according to a recent paper that investigated the photocatalytic and antiviral properties of titanium dioxide. It tends to inhibit viruses, structurally destroy DNA, and degrade the phospholipid bilayer (Choudhury et al. <xref ref-type=\"bibr\" rid=\"CR38\">2005</xref>). Silver nanoparticles bind to viral proteins in two ways: (i) they attach to the outer membrane of the virus, preventing it from attaching to the cell surface receptor; and (ii) they attach to the viral nucleic acid (RNA or DNA), limiting transfection and viral replication within the host cell (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>). In addition, the entry of the antiviral transcription factors interferon regulatory factors (IRF)-7 into the nucleus of lung epithelial cells is suppressed by silver nanoparticles and mitochondrial cycling is disrupted (Halbus et al. <xref ref-type=\"bibr\" rid=\"CR60\">2017</xref>). According to a recent finding, SARS-CoV-2 may resist the antiviral effect of silver nanoparticles (Salleh et al. <xref ref-type=\"bibr\" rid=\"CR167\">2020</xref>).</p></sec><sec id=\"Sec20\"><title>Nanoemulsions</title><p id=\"Par43\">Nanoemulsions do not irritate or affect the skin. Therefore, they can effectively improve drug bioavailability (Villeret et al. <xref ref-type=\"bibr\" rid=\"CR194\">2018</xref>). In addition, nanoemulsions consist of small particles with a large surface area, which promotes absorbability (Abd Elkodous et al. <xref ref-type=\"bibr\" rid=\"CR1\">2021</xref>). To understand how some pathogenic viruses are affected by nanoemulsions of different essential oils, several experiments have been carried out. The majority of published studies assume that the nanoemulsions of various essential oils would interfere with the assembly of the virion envelope, thus blocking the attachment of viruses and their penetration into host cells (Naseema et al. <xref ref-type=\"bibr\" rid=\"CR131\">2021</xref>).</p><p id=\"Par44\">Nanoemulsions can compromise virion membrane integrity or obscure viral assembly, preventing SARS-CoV-2 penetration and adhesion into host cells. Furthermore, the encapsulated favipiravir nanoemulsion can suppress the viral polymerase enzyme of SARS-CoV-2, i.e., RNA-dependent RNA polymerase activity in infected cells. In conclusion, we predict that encapsulated favipiravir nanoemulsions could successfully prevent the adhesion of SARS-CoV-2 while disrupting its morphology (Franklyne et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>).</p></sec><sec id=\"Sec21\"><title>Nanosuspensions</title><p id=\"Par45\">Nanosuspensions are submicron dispersions of hydrophobic drug particulates that have been stabilized with surfactants and possess a high affinity for target receptors (de M Ribeiro and Fonseca <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Nanosuspensions have high bioavailability and dissolution rate due to their small size and greater surface area (Jacob et al. <xref ref-type=\"bibr\" rid=\"CR74\">2020</xref>). In the recipient tissues, nanosuspensions can drastically limit viral replication and inhibit the virus from infiltrating host cells (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Due to their extensive surface area and capacity to adhere various antigens to their interface, they are very often used in research. Furthermore, nanomaterials such as carbon quantum dots, gold nanoparticles, and carbon quantum dots have been characterized as both stimulating and preventing virus interaction in the host cells (Noyes and Whitney <xref ref-type=\"bibr\" rid=\"CR137\">1897</xref>).</p><p id=\"Par46\">Nanosuspensions are effective in delivering therapeutics that are lipid and water-insoluble (Szunerits et al. <xref ref-type=\"bibr\" rid=\"CR180\">2015</xref>). Additionally, they increase the drug solubility and rate of dissolution (Murtaza et al. <xref ref-type=\"bibr\" rid=\"CR129\">2020</xref>). Fucoxanthin (a carotenoid antioxidant) is readily soluble in organic solvents, i.e., acetone, and insoluble in water and exhibits excellent consistency and oxidative stability over a wide pH range. Acute toxicity was studied at a dose of 2000 mg/kg (LD50) and confirmed to be non-toxic. Conventional therapeutic administration is limited by the high dosage of fucoxanthin extract and its solubility properties. However, nanosuspension materials could easily overcome these limitations (Yousaf et al. <xref ref-type=\"bibr\" rid=\"CR213\">2021</xref>).</p></sec><sec id=\"Sec22\"><title>Quantum dots</title><p id=\"Par47\">Quantum dots have a size of 2–10 nm and are known as semiconductor nanocrystals. Their optical properties such as photoluminescence and absorption depend on their size (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>). Nowadays, quantum dots have become very important in the field of nanomedicine. The emission of quantum dots is in the infrared range, i.e., below 650 nm. This is due to the lower scattering of light and low absorption in tissue, which makes them desirable for biomedical imaging (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>; Volkov <xref ref-type=\"bibr\" rid=\"CR195\">2015</xref>).</p><p id=\"Par48\">Additionally, different quantum dot sizes and compositions can excite a similar source of light, culminating in different of emitting colors throughout a wide range of spectra (Liu et al. <xref ref-type=\"bibr\" rid=\"CR108\">2010</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR207\">2016</xref>). In this manner, quantum dots are highly captivating for multiplex imaging. Quantum dots are studied as sensors, target drug delivery, and bioimaging in the medicinal field. In literature, the in vivo imaging of the quantum dots is available for its application (Shi et al. <xref ref-type=\"bibr\" rid=\"CR175\">2015</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR61\">2015</xref>).</p><p id=\"Par49\">Carbon-based quantum dots with a cationic external potential could be exploited to inactivate the SARS-CoV-2 S protein (Balaji et al. <xref ref-type=\"bibr\" rid=\"CR14\">2018</xref>). Furthermore, SARS-CoV-2 releases reactive oxygen species due to the binding of quantum dots with positive extrinsic characteristics to the virus-negative RNA motif (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>). In a study conducted by Du et al<italic>.,</italic> it was shown that carbon dots have antiviral properties against respiratory syndrome and pseudorabies virus (Dong et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). In addition, carbon dots trigger interferon-stimulated signaling pathways, including interferon-α formation, which limits viral replication. Integration of targeted substituents into quantum dots could also efficiently attach to the SARS-CoV-2 entry receptor and alter cellular reproduction (Du et al. <xref ref-type=\"bibr\" rid=\"CR50\">2016</xref>).</p></sec><sec id=\"Sec23\"><title>Protein and polysaccharide nanoparticles</title><p id=\"Par50\">Proteins and polysaccharides are natural biopolymers extracted from animals, plants, marine sources, and microorganisms (Iannazzo et al. <xref ref-type=\"bibr\" rid=\"CR70\">2018</xref>). Protein-based nanoparticles are usually metabolized, decomposable and facile for drug and other targeted ligand attachment. There are two ways to produce these nanoparticles. The first method is derived from water-soluble proteins such as human serum albumin and bovine serum albumin. The other option is to use insoluble proteins such as gliadin and zein. The usual synthesis is by desolvation or coacervation, complex coacervation, solvent/emulsion extraction, and electrospraying. These nanoparticles were chemically modified to integrate targeted compounds and localize tissues and cells to support the targeting mechanism (Bassas-Galia et al. <xref ref-type=\"bibr\" rid=\"CR18\">2017</xref>).</p><p id=\"Par51\">Polyguluronate sulfate, carrageenan, chitosan, and some derivatives of marine polysaccharides have potent inhibitory antagonistic potential against several viruses and provide a platform for conducting a study on coronaviruses (Lohcharoenkal et al. <xref ref-type=\"bibr\" rid=\"CR110\">2014</xref>). The most common cause of severe infectious diseases of the upper respiratory tract sometimes referred to as the common cold are respiratory viruses such as influenza viruses, coronaviruses, and rhinoviruses (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>; Tomas et al. <xref ref-type=\"bibr\" rid=\"CR186\">2022</xref>). Clinical studies have shown that using an iota carrageenan nasal spray to relieve the common cold can shorten the severity of the infection. The antiviral efficacy of carrageenan nasal spray was found to be significant for three categories of viruses: influenza A virus, human coronavirus, and human rhinovirus, with the greatest success observed in individuals with human coronavirus. In individuals with coronavirus treated with Carra-Gene, the severity of infection was 3 days shorter (<italic>p</italic> &lt; 0.01), and the frequency of exacerbations was threefold lower (<italic>p</italic> &lt; 0.01) than in untreated individuals (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>). In addition, sulfated polysaccharides such as sulfated rham-nan and fucoidan may impair the development and function of the epidermal growth factor receptor or decrease its signal transduction, which could improve the silencing function of coronavirus (Monto et al. <xref ref-type=\"bibr\" rid=\"CR126\">2001</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR196\">2017</xref>).</p></sec></sec><sec id=\"Sec24\"><title>Role of nanomedicine in the management of COVID-19</title><p id=\"Par52\">One of the most pressing issues over the past pandemics is the absence of a viable therapy for viral diseases such as SARS-CoV-1, MERS-1, and SARS-CoV-2. As a result, repurposed therapeutics are more beneficial than newer drugs developed in treating COVID-19 outbreaks. A new vaccine typically takes a long time to produce, requiring multiple rounds of guidelines and trials. One of the major challenges in successfully treating COVID-19 is insufficient cellular uptake of the newly developed antiviral drugs. Their usage is also limited due to several factors such as low bioavailability, poor water solubility, increased toxicity, lower biological potential, and a high risk-to-benefit ratio (Wang et al. <xref ref-type=\"bibr\" rid=\"CR197\">2018</xref>; Ansari et al. <xref ref-type=\"bibr\" rid=\"CR9\">2020</xref>).</p><p id=\"Par53\">Consequently, these limitations would limit the prospects of new techniques to increase the efficacy of repurposed drugs and their better absorption into the body. To overcome this limitation, a variety of nanomedicine-based systems have been developed to enhance the therapeutic efficacy of newly developed antiviral drugs (Lembo et al. <xref ref-type=\"bibr\" rid=\"CR103\">2018</xref>). Antiviral drugs based on nanomedicine offer several opportunities to overcome the drawbacks of current antiviral therapeutics (Zazo et al. <xref ref-type=\"bibr\" rid=\"CR215\">2016</xref>). Several fundamental obstacles, such as low bioavailability and poor aqueous solubility, can be addressed through nanomedicine design. This involves modifying the pharmacokinetic and pharmacodynamic properties of drugs to achieve lower toxicity and lower dose, improve drug bioavailability, and inhibit viral spread (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>). Additionally, nanomedicine can penetrate cellular membranes and concentrate therapeutic effects in viral persistence sites (Akhter et al. <xref ref-type=\"bibr\" rid=\"CR6\">2012</xref>). One of the most attractive approaches to minimize the adverse effects of viral rebound and patient compliance during the course of therapy is nanomedicine with intrinsically regulated or prolonged release capabilities (Yadav et al. <xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>). Nanocarrier technology could thus be used to repurpose drugs and optimize treatment efficacy COVID-19 (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>).</p><p id=\"Par54\">The upsides of nanocarriers include their high drug loading effectiveness, larger surface area, smaller in size, prolonged drug releasing pattern, and enhanced optimum performance of the loaded pharmaceuticals in such a novel drug delivery system (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>) (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>; Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>). Nanotechnology has a specific physicochemical advantage in medicine due to its unique hydrophobic or hydrophilic nature, nanosize, nanoshape, and the possibility of ligand conjugation on its membrane. These nanoscale physicochemical features would improve pharmaceutical applications' efficiency, specificity, functionality, and sensitivity (Kobayashi et al. <xref ref-type=\"bibr\" rid=\"CR89\">2014</xref>). Several nanomedicine groups could be repurposed to treat COVID-19 therapy, and they are further summarized below. Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> outlines the findings of nanomedicine as a strategy to augment the pharmacological output of several repurposed medications with the potential for COVID-19 treatment.<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Role of nanomedicine drug delivery in COVID-19. Nanomedicine offers numerous characteristics that may be effectively used to transport therapeutic compounds to infected cells, and the tailored ligand-coupled nanoparticle selectively interacts with viral residues, resulting in virus destruction and inability to penetrate the cells </p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig3_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Overview of nanomedicine for improving the pharmacological efficacy of a repurposed medication with COVID-19 management potential</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Drug</p></th><th align=\"left\"><p>Nanocarrier</p></th><th align=\"left\"><p>Model (<italic>In vitro/In vivo</italic>)</p></th><th align=\"left\"><p>Results</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Improved target specificity; bioavailability is significantly elevated in the thymus and spleen</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR141\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased drug transport in the lymphatic system; oral bioavailability is significantly improved; the drug release is regulated through diffusion</p></td><td align=\"left\"><p>Freitas et al. (<xref ref-type=\"bibr\" rid=\"CR55\">2005</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Oral bioavailability is significantly improved</p></td><td align=\"left\"><p>Sercombe et al. (<xref ref-type=\"bibr\" rid=\"CR01\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Enhanced lymphatic targeting specificity in the intestine; bioavailability in the spleen and thymus is significantly higher than in the plasma</p></td><td align=\"left\"><p>Negi et al. (<xref ref-type=\"bibr\" rid=\"CR133\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased bioavailability and <italic>in-vitro</italic> release in the spleen and thymus; improved target specificity</p></td><td align=\"left\"><p>Maniyar and Kokare (<xref ref-type=\"bibr\" rid=\"CR02\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>The drug peak plasma concentration is improved</p></td><td align=\"left\"><p>Kumar et al. (<xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Bioavailability is increased by 6.98 times when compared to a drug suspension</p></td><td align=\"left\"><p>Garg et al. (<xref ref-type=\"bibr\" rid=\"CR56\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Improved Tissue Distribution; improved oral bioavailability by a significant extent; drug degradation protection</p></td><td align=\"left\"><p>Khan et al. (<xref ref-type=\"bibr\" rid=\"CR88\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Ensure that viral replication is actively suppressed; entrapment efficiency has increased; biphasic controlled release</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR159\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PM</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Entrapment efficiency has increased; improved stability</p></td><td align=\"left\"><p>Yadav et al. (<xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ribavirin</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p>C57BL/6N mice</p></td><td align=\"left\"><p>Longer residence time in the mice liver; improved cellular internalization</p></td><td align=\"left\"><p>Javan et al. (<xref ref-type=\"bibr\" rid=\"CR77\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Enhanced drug release pattern; Entrapment efficiency has increased</p></td><td align=\"left\"><p>Ishihara et al. (<xref ref-type=\"bibr\" rid=\"CR71\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Significant build-up in organs such as the spleen and liver; biphasic controlled release; bioavailability are enhanced by 4 times</p></td><td align=\"left\"><p>Katata et al. (<xref ref-type=\"bibr\" rid=\"CR82\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Entrapment efficiency has increased; bioavailability is considerably enhanced</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR160\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>TPGSM</p></td><td align=\"left\"><p>New Zealand rabbits</p></td><td align=\"left\"><p>Relative bioavailability is enhanced by 3.17 times; entrapment efficiency has increased; improved drug dissolution</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR03\">2016</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>Nanostructured lipid carriers: NLC, Solid lipid nanoparticles: SLN, Polymeric micelles: PM, Polymeric nanoparticles: PN, Self-nanoemulsifying drug delivery system: SNEDDS, D-α-Tocopheryl polyethylene glycol succinate micelles: TPGSM</p></table-wrap-foot></table-wrap></p><sec id=\"Sec25\"><title>Liposomes targeted drug delivery system for COVID-19</title><p id=\"Par55\">The liposomes are the bilayers of lipid vesicles and are currently utilized to deliver aquaphobic drugs (Puri et al. <xref ref-type=\"bibr\" rid=\"CR150\">2009</xref>). The liposomes would function as stealth liposomes whereby the systemic circulation is long, targeted, and stimulus-responsive for drug delivery. As liposome is complex and biologically degraded in nature, it is one of the methods of choice for the carrier of drug formulation. Moreover, it is also a productive platform for developing a novel formulation of COVID-19 diseases. Studies have found that synthetic peptides loaded on the liposomes are potential venues for treating SARS coronavirus infection (Lokhande <xref ref-type=\"bibr\" rid=\"CR111\">2018</xref>). Liposomes that are chemically conjugated peptides have been used in the treatment of SARS because they are suitable for the cytotoxic T-lymphocyte strategy that reduces viral load. A novel hydroxychloroquine formulation administered to Sprague–Dawley rats had shown efficacy against COVID-19. Unlike the IV injection, the formulation is breathed and has a greater concentration of medication in the lungs, a longer half-life, and less exposure to other organs such as the heart. Liposome as a carrier system for the delivery of drugs in the treatment of COVID-19 has many advantages. Forty percentage of small molecule drugs used to treat cancer are poorly soluble in water. Therefore, a similar drug delivery method is needed immediately to encapsulate the drug and increase its water solubility (Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Cyclodextrin combination targeted drug delivery system for COVID-19</title><p id=\"Par56\">Cyclodextrin (a biocompatible natural polysaccharide) has α, β, and γ as essential components. According to reports, cyclodextrin nanoparticles are gaining importance due to numerous cyclodextrin components that absorb a greater amount of therapeutics than normal cyclodextrin (Sun et al. <xref ref-type=\"bibr\" rid=\"CR178\">2014</xref>). Cyclodextrin-based remdesivir nanoformulation has just received Food Drug Administration (FDA) approval for the treatment of COVID-19 (Khalaj‐Hedayati et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>). In the clinical context, it has shown encouraging results in various viruses, including Middle East respiratory syndrome coronavirus and Nipah virus (Jones et al. <xref ref-type=\"bibr\" rid=\"CR79\">2020</xref>). In contrast to its function as a medical dispersant for therapeutics, which are primarily hydrophobic, cyclodextrin also possesses antiviral activities (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>). Therefore, cyclodextrin could be used as a delivery strategy for respiratory sprays for therapeutics that require targeted selectively to the pulmonary system (<italic>Khalaj‐Hedayati</italic> et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>).</p></sec><sec id=\"Sec27\"><title>Inorganic and metallic-based nanocomposites targeted drug delivery system for COVID-19</title><p id=\"Par57\">Inorganic nanocomposites for therapeutics delivery can be developed using a variety of metallic nanoparticles, including platinum, silica, silver, gadolinium, gold, and its nanocrystal conglomerates (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR94\">2017</xref>). Silver nanoparticles are known for their extensive antibacterial activity and are an essential ingredient in pharmaceutical formulations, e.g., silver sulfasalazine (a sulfa antibiotic), where silver particles support the antibacterial activity of the sulfasalazine. Silver nanoparticles are also being studied for their possible virucidal properties. It has been observed that SARS-CoV-2 attaches to cell membrane and persists there for up to 12 h, while the silver nanoparticle-based antibacterial film can eradicate the virus (Chaturvedi et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>). This film can be sprayed on metal, glass, and ceramic surfaces to hamper the proliferation of SARS-CoV-2 in busy places such as universities, hospitals, and marketplaces (Prasad et al. <xref ref-type=\"bibr\" rid=\"CR148\">2015</xref>).</p></sec><sec id=\"Sec28\"><title>Polymeric solid colloidal nanoparticles targeted drug delivery system for COVID-19</title><p id=\"Par58\">Due to their unique properties, including targeted drug delivery, controlled drug release, improved tolerability, and efficiency, PN promotes stable <italic>in vivo</italic> delivery of therapeutic agents (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>; Balagna et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Due to their nanosize, they also prevent large dose-dependent side effects and penetration through biological membranes. This favorable property of polymeric nanoparticles can be exploited for drug delivery to revive currently accessible therapeutics for the treatment of the SARS-CoV-2 outbreak. Zhang et al<italic>.</italic> formulated nanoparticles based on PLGA [poly (lactic-co-glycolic acid)] polymers to combat the infection rate of SARS-CoV-2. PLGA was used as the internal base component in a membrane-enveloped nanoparticle derived from human macrophages and human lung epithelial cells of the II type, which exhibit the same physiological processes as SARS-CoV-2 invading recipient cells (Verma et al. <xref ref-type=\"bibr\" rid=\"CR192\">2017</xref>). During the incubation period, these biological nanoparticles serve as a receptive target for SARS-CoV-2, where it is destroyed and can no longer enter recipient cells (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>). Therefore, the polymer-based nanoparticles may provide a nanocarrier platform for the delivery of therapeutics against SARS-CoV-2 contamination.</p></sec><sec id=\"Sec29\"><title>Nano-based vaccines</title><p id=\"Par59\">The sudden appearance of COVID-19 has necessitated the immediate use of drugs to combat the virus. Health professionals have attempted various methods to manage the symptoms. Still, due to certain factors including low solubility, permeability, and the limitation of targeting competence, they could not be used as they were unable to provide the required therapeutic effect. To deliver the drug to the appropriate venue with the fewest adverse effects, nanotechnology has been essential (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR95\">2018</xref>). They could transport the therapeutic molecule to the infected cells, and the targeted liganded nanoparticles would bind with the viral epitopes, deactivating the virus and preventing it from entering the cell. According to the WHO report on COVID-19 candidate vaccines, 52 possible vaccines are undergoing clinical trials, while 162 are in the preclinical stage of development throughout December 8, 2020 (Raj et al. <xref ref-type=\"bibr\" rid=\"CR155\">2021</xref>). The platforms include virus-like particles, DNA, protein subunits, RNA, inactivated viruses, and non-replicating viral vectors. With a pandemic like COVID-19, a huge amount of vaccines is needed, which is not made possible with the existing production of vaccines (Ashraf et al. <xref ref-type=\"bibr\" rid=\"CR12\">2021</xref>). In this case, the noble way of drug delivery has assisted, including reverse vaccinology, nanomedicine, next-generation sequencing, and human challenge studies. The nanoparticle-based vaccination has elicited a strong immunological response that is well-tolerated. Gold, silver sulfide, titanium oxide, zirconium, and graphene are some of the materials that have been considered (Haslberger et al. <xref ref-type=\"bibr\" rid=\"CR65\">2020</xref>). Drug delivery, cancer nanovaccines, immunological engineering, nanosensors, and platform technologies are some techniques employed by nanoresearchers.</p><p id=\"Par60\">In the case of the increasing number of pathogenic bacteria that are resistant to antimicrobial agents, there are several studies that have demonstrated the efficacy of nanotechnology-based antimicrobial therapy (Nikaeen et al. <xref ref-type=\"bibr\" rid=\"CR135\">2020</xref>). Targeted nanotechnology is a new resource for the treatment of antiviral diseases. The employment of nanoparticles in the battle against SARS-CoV-2 has created a mechanism that inhibits the virus's entry into the host cell until it is inactivated. The targeted nanoparticles specific to virus-expressed protein would minimize the internalization of the virus. Metal nanoparticles can prevent the virus from attaching to the surface of the cell and impair the virus's ability to replicate once it enters the cell (Kumar et al. <xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>). The size, shape, and surface charge of nanoparticles would determine how effective the treatment is. However, several safety precautions need to be taken to reduce host cell cytotoxicity. Well-established nanotechnology drug delivery would improve traditional therapies in medicine, as seen in the SARS-CoV-2 outbreak, but its use in viral diseases is still relatively unknown. By shortening PCR assays, reducing detection time, improving sensitivity, amplifying signals, and using them as adjuvants in vaccines, nanostructure technology could help diagnose disease. Nanotechnology has the potential to assist in the pathogenesis of COVID-19 in a variety of ways, including attenuation, possible membrane attachment and fusion upon viral entry, and protein fusion in infected cells. Activating intracellular pathways for reversible virus damage and deactivating viral transcription, translation, and replication may be more effective with nanocapsulated vaccines (Maji et al. <xref ref-type=\"bibr\" rid=\"CR119\">2020</xref>).</p><p id=\"Par61\">To increase half-life and longevity and to elicit adequate cellular and antibody responses, nanoparticles can serve as biologics or cargo for vaccination, sometimes referred to as nanovaccines, among other factors (Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>). The basic principle of this approach is to integrate an epitope that triggers protective immunity into nanoparticles of 1–100 nm and establish an immunological storage that protects the body from emerging harmful microorganisms. Moreover, suitable protein nanoparticles from the architectural proteins of SARS-CoV-2, <italic>i.e.,</italic> envelope (E), membrane (M), nucleocapsid (N), and spike (S), can be transported to target immune systems that would trigger immune feedback and confer durable resistance to the virus. Protein nanoparticles and biologics have already been used for Middle East Coronavirus 1 and Severe Acute Respiratory Syndrome, and they could also be developed for SARS-CoV-2 (Coleman et al. <xref ref-type=\"bibr\" rid=\"CR39\">2014</xref>; Xiang et al. <xref ref-type=\"bibr\" rid=\"CR206\">2006</xref>). Pimentel et al<italic>.</italic> developed a nanovaccine using a repetitive SARS-CoV-1 S protein antigen of around 25 nm to design a targeted antibody that can prevent contamination with SARS-CoV-1 (Pimentel et al. <xref ref-type=\"bibr\" rid=\"CR147\">2009</xref>). Matsuura et al<italic>.</italic> designed virus-like nanoparticles that stimulate a viral protein. We hypothesize that these nanoparticles can also be used to produce SARS-CoV-2 proteins for vaccination applications (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Microparticles (25 nm) depending on epitope selectivity and adjuvant strategies have been utilized to enhance the transport of peptide/protein epitopes to dendritic cells to trigger both cellular and humoral immunological responses in rats (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Numerous nanoparticles, including nanobioceramics, liposomes, cationic polymers, polymer-based particles, gold nanoparticles, virus-like nanoparticles, and peptide nanoparticles, can be used to stimulate lymphatic circulation and produce B lymphocytes (B cells) and T lymphocytes (T cells) for efficient clearance of SARS-CoV-2 (Reddy et al. <xref ref-type=\"bibr\" rid=\"CR161\">2008</xref>; Gregory et al. <xref ref-type=\"bibr\" rid=\"CR58\">2013</xref>; Wilson et al. <xref ref-type=\"bibr\" rid=\"CR201\">2018</xref>; Koshy et al. <xref ref-type=\"bibr\" rid=\"CR90\">2017</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR33\">2016</xref>). Stimulating CD8 + (cytotoxic) T lymphocyte cells or CD4 + (helper) T lymphocyte cells is essential for any proposed vaccination. In the case of coronavirus, T lymphocyte cells have shown the potential to prevent relapse and provide immunologic maintenance for up to 11 years after exposure to SARS-CoV-1 in recovered patients (Goodwin and Huang <xref ref-type=\"bibr\" rid=\"CR57\">2017</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR181\">2011</xref>).</p><p id=\"Par62\">Research shows that DNA vaccination also induces neutralization and preventive resistance to SARS-CoV-1 through upregulation of neutralizing antibodies, clusters of differentiation 4 (CD4 +), and clusters of differentiation 8 (CD8 +), suggesting that DNA vaccination can be delivered with appropriate nanoparticles. In addition, the antigens of B lymphocyte cells and T lymphocyte cells in the immunological domains of SARS-CoV-1 and SARS-CoV-2 were evaluated in a random analysis and were shown to be equivalent for both infections (Peng et al. <xref ref-type=\"bibr\" rid=\"CR145\">2006</xref>). This implies that SARS-CoV-1 vaccinations may be more effective at getting rid of SARS-CoV-2. Numerous properties may be considered for optimizing a SARS-CoV-1 vaccine against SARS-CoV-2, namely enhancing epitope integrity, cytokine production, specific target allocation, prolonged release of biologics and/or epitopes, sequestration of SARS-CoV-2 infections, and prevention of peripheral manifestations of infection (Ahmed et al. <xref ref-type=\"bibr\" rid=\"CR5\">2020</xref>; Rao et al. <xref ref-type=\"bibr\" rid=\"CR157\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> provides an overview of some of the most promising COVID-19 vaccine alternatives that have been formally clinically evaluated (Vijayan et al. <xref ref-type=\"bibr\" rid=\"CR193\">2019</xref>).<table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Currently existing COVID-19 prospective vaccines and their stage of clinical advancement</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Vaccine candidate</p></th><th align=\"left\"><p>Inventors</p></th><th align=\"left\"><p>Summary</p></th><th align=\"left\"><p>Status</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>SARS-CoV-2 recombinant glycoprotein/Matrix M1 nanoparticle vaccine</p></td><td align=\"left\"><p>Novavax United Kingdom</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Russell et al. (<xref ref-type=\"bibr\" rid=\"CR165\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>FINLAY-FR-2 vaccination against SARS-CoV-2</p></td><td align=\"left\"><p>Instituto Finlay de Vacunas</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Puga-Gómez et al. (<xref ref-type=\"bibr\" rid=\"CR149\">2023</xref>)</p></td></tr><tr><td align=\"left\"><p>nCov vaccine</p></td><td align=\"left\"><p>Zydus Cadila</p></td><td align=\"left\"><p>DNA-based vaccine</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Barrett et al. (<xref ref-type=\"bibr\" rid=\"CR17\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>SARS-CoV-2 vaccine (Vero cells)</p></td><td align=\"left\"><p>Chinese Academy of Medical Sciences + Institute of Medical Biology</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Huang et al. (<xref ref-type=\"bibr\" rid=\"CR69\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>mRNA- 1273 and mRNA- 1283</p></td><td align=\"left\"><p>National Institute of Allergy and Infectious Diseases + Moderna</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Patone et al. (<xref ref-type=\"bibr\" rid=\"CR142\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>VAT00002: SARS-CoV-2 spike (S) protein with a biologics</p></td><td align=\"left\"><p>GlaxoSmithKline + Sanofi pasteur</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ndwandwe and Wiysonge (<xref ref-type=\"bibr\" rid=\"CR132\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BNT 162b1 and BNT 162b2</p></td><td align=\"left\"><p>Fosun Pharma + Biontech/Pfizer</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Lamb (<xref ref-type=\"bibr\" rid=\"CR98\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Adeno-based Gam-COVID-Vac (rAd26-S + rAd5-S)</p></td><td align=\"left\"><p>Gamaleya Research Institute, Health Ministry of the Russian Federation</p></td><td align=\"left\"><p>Viral vector (non-proliferating)</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Vaezi and Meysamie (<xref ref-type=\"bibr\" rid=\"CR189\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Recombinant SARS-CoV-2 vaccine (CHO cells)</p></td><td align=\"left\"><p>Institute of Microbiology, Chinese Academy of Sciences + Anhui Zhifei Longcom Biopharmaceutical</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Sun et al. (<xref ref-type=\"bibr\" rid=\"CR179\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BBV152</p></td><td align=\"left\"><p>Bharat Biotech International Limited</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ella et al. (<xref ref-type=\"bibr\" rid=\"CR51\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>ChAdOx1</p></td><td align=\"left\"><p>University of Oxford + AstraZeneca</p></td><td align=\"left\"><p>Vector-based vaccines</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Agrawal et al. (<xref ref-type=\"bibr\" rid=\"CR2\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Ad26.C0V2.S</p></td><td align=\"left\"><p>Janssen Pharmaceuticals</p></td><td align=\"left\"><p>Vector-based vaccines encoding S protein</p></td><td align=\"left\"><p>Phase 3</p></td><td align=\"left\"><p>Bos et al. (<xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p></sec></sec><sec id=\"Sec30\"><title>Challenges in treatment with nano-based medicines</title><p id=\"Par63\">Nanovaccines and nanomedicine that have been utilized in novel medicines to fight against the continuous pandemic, like all current medications, must demonstrate their safety and be studied extensively before their release on the market. As the interaction between engineered nanomaterials (ENMs) with viral targets (envelope, capsid, and nucleic acid) is comprehended by many, the prediction and screening of ENM–virus interactions require a robust analytical framework. Toxicity relevant to inorganic nanoparticles is the limiting factor of their usage in various cases. Other beneficial biological properties can be made by changing the shape, size, and surface chemistry. Moreover, the physicochemical properties must be modified to reduce the side effects of the nanoparticles. ENMs should have a variable dose. Antiviral stability and bioavailability should be considered in many therapeutic applications. The main challenges of nanodelivery come in the form of physiological barriers, which are the mucosa and the alveolar fluid as they are utilized for intranasal as well as the reticuloendothelial structure and used for systemic delivery (Le et al. <xref ref-type=\"bibr\" rid=\"CR101\">2020</xref>). In the cell, the nanoparticle must be able to reject the endocytic degradation component. Certain studies also have shown that nanoparticles could cause damage to respiratory sites and affect lung function. The four key pathobiological aspects of nanoparticles, which include inflammation, genotoxicity, oxidative stress, and fibrosis, must be examined further. Certain studies have yet to prove whether a nasal or oral nanovaccine is safer and provides longer mucosal immunity and viral protection than others. That being said, the ability to provide adequate protection when administered to different age groups by different routes has yet to be confirmed. There are various nanomaterials ranging from polymers to dendrimers to nanoparticles, oligomers, liposomes and tiny molecules. However, once the virus chemical complex dissociates and the viruses are released, they lose their effectiveness and the cycle begins again. The reactive oxygen species formed by nanoparticles would lead to oxidative stress as it can cause a disturbance in mitochondrial respiration. This would cause harmful pathological processes in the cell that cause toxicity. In various studies, it was found that titanium dioxide could enter mice's lungs, causing nanoparticle accumulation to increase the level of lipid peroxidation and minimize the number of antioxidant cells treated by nanoparticles. Due to this, the level of the AP-1 transcription factor increased and affected the transcriptional process. DNA would interact with the carbon nanotubes (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>). Although nanoparticles have gained tremendous importance in research because they are very effective, their toxicological aspects need to be considered. The way nanoparticles act in the human body is of critical importance. When deposited to the maximum, they can cause harmful side effects, especially with platinum-containing drugs (cisplatin, carbapenem) used in the treatment of lungs, colorectal, ovarian, head, neck, and bladder. Due to this, its use is limited as the prominent side effect of it is nephrotoxicity. The major challenge in translating nanodrugs is the first clearance by the RES that minimizes the efficacy and the increasing toxicity in the organs such as the spleen, liver, and kidney (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR210\">2020</xref>). Iron oxide particles have harmful factors both <italic>in vivo</italic> and <italic>in vitro</italic> as they generate reactive oxygen species (ROS).</p><p id=\"Par64\">Nevertheless, the iron oxide is coated with a polymer that would maximize cell viability. When hydroxyl group is added to the gadolinium fullerene particle, it prevents the formation of ROS and excessive release of pro-inflammatory cytokines, which can cause mortality and severe infection. Macrophages and monocyte lineage significantly control the cytokine-facilitated inflammatory response and innate and adaptive response (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>; Shanley et al. <xref ref-type=\"bibr\" rid=\"CR173\">2021</xref>).</p></sec><sec id=\"Sec31\"><title>Future prospects</title><p id=\"Par65\">COVID-19 is a major global threat and the greatest challenge to humanity since World War II. The pandemic has caused many calamities that disrupted health and caused human casualties and other issues. Despite numerous attempts by all countries across the world, including extensive research and numerous clinical trials, no medication has yet been demonstrated to control the virus. This study has highlighted several traditional treatment strategies as well as the crucial function of nanomedicine in the new SARS-CoV-2. Drug substitution, such as remdesivir and chloroquine, is a quick way to achieve safe therapies. Furthermore, several relevant experimental types of research have revealed positive evidence against COVID-19. Nanometal vaccines for example, which are based on the antigenic properties of protein S, have been utilized to reduce viral load. However, a substantial number of studies that support the use of NP-based diagnostic and treatment techniques for SARS-CoV-2 have been published. As a result, all researchers across the world are encouraged to extend their studies in the field of NP-based therapy. There is a great requirement for adequate information on the virus's virulence and transmission to prevent and cure COVID-19 or any other viral pandemic. In the future, this will lead to the discovery of numerous unstructured encoded proteins, enzymes, and functional processes and enable us to understand viral transmission between species. As a result, we could use nanoparticles at the functional level to identify and establish therapeutic targets. As a result, a clinical trial to develop an effective nanovaccine can be conducted with the appropriate knowledge of the viral life cycle and host response. A universal NP-based vaccine with sufficient immunogenicity is the ultimate goal in overcoming the pandemic, as endorsed by researchers worldwide. With factors such as fast detection and mobility, microfluidics would play an essential role in the diagnosis of CoV.</p></sec><sec id=\"Sec32\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par66\">Other than vaccines, there are no approved SARS-CoV-2 therapies being explored in clinical trials. In addition, the nano-based vaccine to treat SARS-CoV-2 has not been adequately studied. All efforts to combat the virus should be explored, and nanotechnology approaches should be prioritized because they may offer new perspectives, particularly in preventing or treating viral internalization compared with standard therapy. To develop a rational design for targeted therapies, further research is also required to comprehend the relationship between CoV and nanoparticles. As the pathophysiology of SARS-CoV-2 is unclear, nanotechnology might be a beneficial technique apart from other methods to produce excellent COVID-19 therapy results.</p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author contributions</title><p>BB and RKS performed conceptualization; BB and AMAI did writing—original draft preparation; AF, SR, AM, AS, DK and JK contributed to writing—review and editing; RKS done supervision. All authors have read and agreed to the published version of the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par67\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References\n",
       "</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abd Elkodous</surname><given-names>M</given-names></name><name><surname>Olojede</surname><given-names>S</given-names></name><name><surname>Morsi</surname><given-names>M</given-names></name><name><surname>El-Sayyad</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Nanomaterial-based drug delivery systems as promising carriers for patients with COVID-19</article-title><source>RSC Adv</source><year>2021</year><volume>11</volume><issue>43</issue><fpage>26463</fpage><lpage>26480</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1ynt7fP</pub-id><pub-id pub-id-type=\"pmid\">35480012</pub-id><pub-id pub-id-type=\"pmcid\">9037715</pub-id><pub-id pub-id-type=\"doi\">10.1039/D1RA04835J</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Agrawal</surname><given-names>U</given-names></name><name><surname>Katikireddi</surname><given-names>SV</given-names></name><name><surname>McCowan</surname><given-names>C</given-names></name><name><surname>Mulholland</surname><given-names>RH</given-names></name><name><surname>Azcoaga-Lorenzo</surname><given-names>A</given-names></name><name><surname>Amele</surname><given-names>S</given-names></name><name><surname>Fagbamigbe</surname><given-names>AF</given-names></name><name><surname>Vasileiou</surname><given-names>E</given-names></name><name><surname>Grange</surname><given-names>Z</given-names></name><name><surname>Shi</surname><given-names>T</given-names></name><name><surname>Kerr</surname><given-names>S</given-names></name><name><surname>Moore</surname><given-names>E</given-names></name><name><surname>Murray</surname><given-names>JLK</given-names></name><name><surname>Shah</surname><given-names>SA</given-names></name><name><surname>Ritchie</surname><given-names>L</given-names></name><name><surname>O'Reilly</surname><given-names>D</given-names></name><name><surname>Stock</surname><given-names>SJ</given-names></name><name><surname>Beggs</surname><given-names>J</given-names></name><name><surname>Chuter</surname><given-names>A</given-names></name><name><surname>Torabi</surname><given-names>F</given-names></name><name><surname>Akbari</surname><given-names>A</given-names></name><name><surname>Bedston</surname><given-names>S</given-names></name><name><surname>McMenamin</surname><given-names>J</given-names></name><name><surname>Wood</surname><given-names>R</given-names></name><name><surname>Tang</surname><given-names>RSM</given-names></name><name><surname>de Lusignan</surname><given-names>S</given-names></name><name><surname>Hobbs</surname><given-names>FDR</given-names></name><name><surname>Woolhouse</surname><given-names>M</given-names></name><name><surname>Simpson</surname><given-names>CR</given-names></name><name><surname>Robertson</surname><given-names>C</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 hospital admissions and deaths after BNT162b2 and ChAdOx1 nCoV-19 vaccinations in 2·57 million people in Scotland (EAVE II): a prospective cohort study</article-title><source>Lancet Respir Med</source><year>2021</year><volume>9</volume><issue>12</issue><fpage>1439</fpage><lpage>1449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFOlsLvJ</pub-id><pub-id pub-id-type=\"pmid\">34599903</pub-id><pub-id pub-id-type=\"pmcid\">8480963</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(21)00380-5</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Paul</surname><given-names>J</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Shavi</surname><given-names>GV</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development and in vivo evaluation of functionalized ritonavir proliposomes for lymphatic targeting</article-title><source>Life Sci</source><year>2017</year><volume>183</volume><fpage>11</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtFSisb7P</pub-id><pub-id pub-id-type=\"pmid\">28647214</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.lfs.2017.06.022</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Alkahtani</surname><given-names>SA</given-names></name><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Ahmad</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>MS</given-names></name><name><surname>Mohsin</surname><given-names>N</given-names></name><name><surname>Abdel-Wahab</surname><given-names>BA</given-names></name></person-group><article-title xml:lang=\"en\">Progress in nanotechnology-based drug carrier in designing of curcumin nanomedicines for cancer therapy: current state-of-the-art</article-title><source>J Drug Target</source><year>2016</year><volume>24</volume><issue>4</issue><fpage>273</fpage><lpage>293</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xhs12qurs%3D</pub-id><pub-id pub-id-type=\"pmid\">26066739</pub-id><pub-id pub-id-type=\"doi\">10.3109/1061186X.2015.1055570</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmed</surname><given-names>SF</given-names></name><name><surname>Quadeer</surname><given-names>AA</given-names></name><name><surname>McKay</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>3</issue><fpage>254</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnvVKlsLk%3D</pub-id><pub-id pub-id-type=\"pmid\">32106567</pub-id><pub-id pub-id-type=\"pmcid\">7150947</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12030254</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Storm</surname><given-names>G</given-names></name><name><surname>Kok</surname><given-names>RJ</given-names></name></person-group><article-title xml:lang=\"en\">Gold nanoparticles in theranostic oncology: current state-of-the-art</article-title><source>Expert Opin Drug Deliv</source><year>2012</year><volume>9</volume><issue>10</issue><fpage>1225</fpage><lpage>1243</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhtl2ltr7N</pub-id><pub-id pub-id-type=\"pmid\">22897613</pub-id><pub-id pub-id-type=\"doi\">10.1517/17425247.2012.716824</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alafif</surname><given-names>T</given-names></name><name><surname>Tehame</surname><given-names>AM</given-names></name><name><surname>Bajaba</surname><given-names>S</given-names></name><name><surname>Barnawi</surname><given-names>A</given-names></name><name><surname>Zia</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Machine and deep learning towards COVID-19 diagnosis and treatment: survey, challenges, and future directions</article-title><source>Int J Environ Res Public Health</source><year>2021</year><volume>18</volume><issue>3</issue><fpage>1117</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlslCjtrw%3D</pub-id><pub-id pub-id-type=\"pmid\">33513984</pub-id><pub-id pub-id-type=\"pmcid\">7908539</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijerph18031117</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alcoba-Florez</surname><given-names>J</given-names></name><name><surname>González-Montelongo</surname><given-names>R</given-names></name><name><surname>Íñigo-Campos</surname><given-names>A</given-names></name><name><surname>de Artola</surname><given-names>DGM</given-names></name><name><surname>Gil-Campesino</surname><given-names>H</given-names></name><name><surname>Team</surname><given-names>TMTS</given-names></name><name><surname>Ciuffreda</surname><given-names>L</given-names></name><name><surname>Valenzuela-Fernández</surname><given-names>A</given-names></name><name><surname>Flores</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Fast SARS-CoV-2 detection by RT-qPCR in preheated nasopharyngeal swab samples</article-title><source>Int J Infect Dis</source><year>2020</year><volume>97</volume><fpage>66</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylurfN</pub-id><pub-id pub-id-type=\"pmid\">32492531</pub-id><pub-id pub-id-type=\"pmcid\">7833505</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2020.05.099</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ansari</surname><given-names>MA</given-names></name><name><surname>Almatroudi</surname><given-names>A</given-names></name><name><surname>Alzohairy</surname><given-names>MA</given-names></name><name><surname>AlYahya</surname><given-names>S</given-names></name><name><surname>Alomary</surname><given-names>MN</given-names></name><name><surname>Al-Dossary</surname><given-names>HA</given-names></name><name><surname>Alghamdi</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Lipid-based nano delivery of Tat-peptide conjugated drug or vaccine–promising therapeutic strategy for SARS-CoV-2 treatment</article-title><source>Expert Opin Drug Deliv</source><year>2020</year><volume>17</volume><issue>12</issue><fpage>1671</fpage><lpage>1674</lpage><pub-id pub-id-type=\"pmid\">32820694</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2020.1813712</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"other\">Arslan S, Delice O, Kahraman M, Yilmaz S, Aslan M (2021) Correlation of chest CT and RT-PCR testing in coronavirus disease 2019 (COVID-19) in Turkey. Ann Clin Anal Med. pp 483–487.</mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Asdaq</surname><given-names>SMB</given-names></name><name><surname>Ikbal</surname><given-names>AMA</given-names></name><name><surname>Sahu</surname><given-names>RK</given-names></name><name><surname>Bhattacharjee</surname><given-names>B</given-names></name><name><surname>Paul</surname><given-names>T</given-names></name><name><surname>Deka</surname><given-names>B</given-names></name><name><surname>Fattepur</surname><given-names>S</given-names></name><name><surname>Widyowati</surname><given-names>R</given-names></name><name><surname>Vijaya</surname><given-names>J</given-names></name><name><surname>Al Mohaini</surname><given-names>M</given-names></name><name><surname>Alsalman</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology integration for SARS-CoV-2 diagnosis and treatment: an approach to preventing pandemic</article-title><source>Nanomaterials</source><year>2021</year><volume>11</volume><issue>7</issue><fpage>1841</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWktrrF</pub-id><pub-id pub-id-type=\"pmid\">34361227</pub-id><pub-id pub-id-type=\"pmcid\">8308419</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano11071841</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ashraf</surname><given-names>MU</given-names></name><name><surname>Kim</surname><given-names>Y</given-names></name><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Seo</surname><given-names>D</given-names></name><name><surname>Ashraf</surname><given-names>M</given-names></name><name><surname>Bae</surname><given-names>YS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines (revisited) and oral-mucosal vector system as a potential vaccine platform</article-title><source>Vaccines</source><year>2021</year><volume>9</volume><issue>2</issue><fpage>171</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVSlsrrO</pub-id><pub-id pub-id-type=\"pmid\">33670630</pub-id><pub-id pub-id-type=\"pmcid\">7922043</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines9020171</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Balagna</surname><given-names>C</given-names></name><name><surname>Perero</surname><given-names>S</given-names></name><name><surname>Percivalle</surname><given-names>E</given-names></name><name><surname>Nepita</surname><given-names>EV</given-names></name><name><surname>Ferraris</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Virucidal effect against coronavirus SARS-CoV-2 of a silver nanocluster/silica composite sputtered coating</article-title><source>Open Ceram</source><year>2020</year><volume>1</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XisFegsL3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.oceram.2020.100006</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Balaji</surname><given-names>AB</given-names></name><name><surname>Pakalapati</surname><given-names>H</given-names></name><name><surname>Khalid</surname><given-names>M</given-names></name><name><surname>Walvekar</surname><given-names>R</given-names></name><name><surname>Siddiqui</surname><given-names>H</given-names></name></person-group><source>Natural and synthetic biocompatible and biodegradable polymers. Biodegradable and biocompatible polymer composites</source><year>2018</year><publisher-loc>Duxford</publisher-loc><publisher-name>Woodhead Publishing</publisher-name><fpage>3</fpage><lpage>32</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-08-100970-3.00001-8</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bandodkar</surname><given-names>AJ</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Non-invasive wearable electrochemical sensors: a review</article-title><source>Trends Biotechnol</source><year>2014</year><volume>32</volume><issue>7</issue><fpage>363</fpage><lpage>371</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXotlyjsrw%3D</pub-id><pub-id pub-id-type=\"pmid\">24853270</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2014.04.005</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barnett</surname><given-names>JM</given-names></name><name><surname>Monnier</surname><given-names>BM</given-names></name><name><surname>Tyler</surname><given-names>S</given-names></name><name><surname>West</surname><given-names>D</given-names></name><name><surname>Ballantine-Dykes</surname><given-names>H</given-names></name><name><surname>Regan</surname><given-names>E</given-names></name><name><surname>Wraith</surname><given-names>P</given-names></name><name><surname>Kiely</surname><given-names>J</given-names></name><name><surname>Luxton</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Initial trail results of a magnetic biosensor for the rapid detection of Porcine Reproductive and Respiratory Virus (PRRSV) infection</article-title><source>Sens Bio-Sens Res</source><year>2019</year><volume>27</volume><pub-id pub-id-type=\"doi\">10.1016/j.sbsr.2019.100315</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barrett</surname><given-names>ADT</given-names></name><name><surname>Titball</surname><given-names>RW</given-names></name><name><surname>MacAry</surname><given-names>PA</given-names></name><name><surname>Rupp</surname><given-names>RE</given-names></name><name><surname>von Messling</surname><given-names>V</given-names></name><name><surname>Walker</surname><given-names>DH</given-names></name><name><surname>Fanget</surname><given-names>NVJ</given-names></name></person-group><article-title xml:lang=\"en\">The rapid progress in COVID vaccine development and implementation</article-title><source>NPJ Vaccines</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>20</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCgsL0%3D</pub-id><pub-id pub-id-type=\"pmid\">35145102</pub-id><pub-id pub-id-type=\"pmcid\">8831501</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-022-00442-8</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"other\">Bassas-Galia M, Follonier, S, Pusnik M, Zinn M (2017) Natural polymers: a source of inspiration. In Bioresorbable polymers for biomedical applications. Elsevier: pp 31–64</mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Benzigar</surname><given-names>MR</given-names></name><name><surname>Bhattacharjee</surname><given-names>R</given-names></name><name><surname>Baharfar</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Current methods for diagnosis of human coronaviruses: pros and cons</article-title><source>Anal Bioanal Chem</source><year>2021</year><volume>413</volume><issue>9</issue><fpage>2311</fpage><lpage>2330</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitl2qs7%2FE</pub-id><pub-id pub-id-type=\"pmid\">33219449</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00216-020-03046-0</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bernheim</surname><given-names>A</given-names></name><name><surname>Mei</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Fayad</surname><given-names>ZA</given-names></name><name><surname>Zhang</surname><given-names>N</given-names></name><name><surname>Diao</surname><given-names>K</given-names></name><name><surname>Lin</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Chest CT findings in coronavirus disease-19 (COVID-19): relationship to duration of infection</article-title><source>Radiology</source><year>2020</year><volume>295</volume><issue>3</issue><fpage>685</fpage><lpage>691</lpage><pub-id pub-id-type=\"doi\">10.1148/radiol.2020200463</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Boopathi</surname><given-names>S</given-names></name><name><surname>Poma</surname><given-names>AB</given-names></name><name><surname>Kolandaivel</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</article-title><source>J Biomol Struct Dyn</source><year>2021</year><volume>39</volume><issue>9</issue><fpage>3409</fpage><lpage>3418</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpvVSgtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">32306836</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bos</surname><given-names>R</given-names></name><name><surname>Rutten</surname><given-names>L</given-names></name><name><surname>van der Lubbe</surname><given-names>JE</given-names></name><name><surname>Bakkers</surname><given-names>MJ</given-names></name><name><surname>Hardenberg</surname><given-names>G</given-names></name><name><surname>Wegmann</surname><given-names>F</given-names></name><name><surname>Zuijdgeest</surname><given-names>D</given-names></name><name><surname>de Wilde</surname><given-names>AH</given-names></name><name><surname>Koornneef</surname><given-names>A</given-names></name><name><surname>Verwilligen</surname><given-names>A</given-names></name><name><surname>van Manen</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Ad26 vector-based COVID-19 vaccine encoding a prefusion-stabilized SARS-CoV-2 Spike immunogen induces potent humoral and cellular immune responses</article-title><source>npj Vaccines</source><year>2020</year><volume>5</volume><issue>1</issue><fpage>91</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFKntr7O</pub-id><pub-id pub-id-type=\"pmid\">33083026</pub-id><pub-id pub-id-type=\"pmcid\">7522255</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-020-00243-x</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bozzuto</surname><given-names>G</given-names></name><name><surname>Molinari</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Liposomes as nanomedical devices</article-title><source>Int J Nanomed</source><year>2015</year><volume>10</volume><fpage>975</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXktl2qs7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJN.S68861</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Mochalski</surname><given-names>P</given-names></name><name><surname>Ruzsanyi</surname><given-names>V</given-names></name><name><surname>Amann</surname><given-names>A</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Hybrid volatolomics and disease detection</article-title><source>Angew Chem Int Ed</source><year>2015</year><volume>54</volume><issue>38</issue><fpage>11036</fpage><lpage>11048</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1KrtrzJ</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201500153</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Vishinkin</surname><given-names>R</given-names></name><name><surname>Barash</surname><given-names>O</given-names></name><name><surname>Nakhleh</surname><given-names>MK</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Synergy between nanomaterials and volatile organic compounds for non-invasive medical evaluation</article-title><source>Chem Soc Rev</source><year>2018</year><volume>47</volume><issue>13</issue><fpage>4781</fpage><lpage>4859</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhtFSgtbvK</pub-id><pub-id pub-id-type=\"pmid\">29888356</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8CS00317C</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"other\">Calucho E (2020) Lateral flow devices for COVID-19-related biomarkers. Biosens Pandemics. Vol. 29</mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Carter</surname><given-names>LJ</given-names></name><name><surname>Garner</surname><given-names>LV</given-names></name><name><surname>Smoot</surname><given-names>JW</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Zhou</surname><given-names>Q</given-names></name><name><surname>Saveson</surname><given-names>CJ</given-names></name><name><surname>Sasso</surname><given-names>JM</given-names></name><name><surname>Gregg</surname><given-names>AC</given-names></name><name><surname>Soares</surname><given-names>DJ</given-names></name><name><surname>Beskid</surname><given-names>TR</given-names></name></person-group><source>Assay techniques and test development for COVID-19 diagnosis</source><year>2020</year><publisher-loc>Washington</publisher-loc><publisher-name>ACS Publications</publisher-name><pub-id pub-id-type=\"doi\">10.1021/acscentsci.0c00501</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chakravarty</surname><given-names>M</given-names></name><name><surname>Vora</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based antiviral therapeutics</article-title><source>Drug Deliv Transl Res</source><year>2021</year><volume>11</volume><issue>3</issue><fpage>748</fpage><lpage>787</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Sgs7%2FN</pub-id><pub-id pub-id-type=\"pmid\">32748035</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13346-020-00818-0</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chamundeeswari</surname><given-names>M</given-names></name><name><surname>Jeslin</surname><given-names>J</given-names></name><name><surname>Verma</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Nanocarriers for drug delivery applications</article-title><source>Environ Chem Lett</source><year>2019</year><volume>17</volume><issue>2</issue><fpage>849</fpage><lpage>865</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXit12ltb7I</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10311-018-00841-1</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"other\">Chandler-Brown D, Bueno AM, Atay O, Tsao DS (2020) A highly scalable and rapidly deployable RNA extraction-free COVID-19 assay by quantitative Sanger sequencing. Biorxiv</mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Yadav</surname><given-names>N</given-names></name><name><surname>Rai</surname><given-names>NK</given-names></name><name><surname>Ellah</surname><given-names>NHA</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name><name><surname>Rehan</surname><given-names>IF</given-names></name><name><surname>Marraiki</surname><given-names>N</given-names></name><name><surname>Batiha</surname><given-names>GES</given-names></name><name><surname>Hetta</surname><given-names>HF</given-names></name><name><surname>Singh</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pleurotus sajor-caju-mediated synthesis of silver and gold nanoparticles active against colon cancer cell lines: a new era of herbonanoceutics</article-title><source>Molecules</source><year>2020</year><volume>25</volume><issue>13</issue><fpage>3091</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsV2ksr3K</pub-id><pub-id pub-id-type=\"pmid\">32645899</pub-id><pub-id pub-id-type=\"pmcid\">7412267</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules25133091</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaudhari</surname><given-names>SP</given-names></name><name><surname>Handge</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Formulation, development and evaluation of lopinavir loaded polymeric micelles</article-title><source>J Sci Technol</source><year>2020</year><volume>5</volume><fpage>173</fpage><lpage>187</lpage></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>HW</given-names></name><name><surname>Huang</surname><given-names>CY</given-names></name><name><surname>Lin</surname><given-names>SY</given-names></name><name><surname>Fang</surname><given-names>ZS</given-names></name><name><surname>Hsu</surname><given-names>CH</given-names></name><name><surname>Lin</surname><given-names>JC</given-names></name><name><surname>Chen</surname><given-names>YI</given-names></name><name><surname>Yao</surname><given-names>BY</given-names></name><name><surname>Hu</surname><given-names>CMJ</given-names></name></person-group><article-title xml:lang=\"en\">Synthetic virus-like particles prepared via protein corona formation enable effective vaccination in an avian model of coronavirus infection</article-title><source>Biomaterials</source><year>2016</year><volume>106</volume><fpage>111</fpage><lpage>118</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtlCqsLvN</pub-id><pub-id pub-id-type=\"pmid\">27552321</pub-id><pub-id pub-id-type=\"pmcid\">7112462</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biomaterials.2016.08.018</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Xu</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral nanoagents: more attention and effort needed?</article-title><source>Nano Today</source><year>2020</year><volume>35</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjtbrP</pub-id><pub-id pub-id-type=\"pmid\">32963581</pub-id><pub-id pub-id-type=\"pmcid\">7498411</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nantod.2020.100976</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Zhai</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>H</given-names></name><name><surname>Bian</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>P</given-names></name><name><surname>Yu</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Rapid and sensitive detection of anti-SARS-CoV-2 IgG, using lanthanide-doped nanoparticles-based lateral flow immunoassay</article-title><source>Anal Chem</source><year>2020</year><volume>92</volume><issue>10</issue><fpage>7226</fpage><lpage>7231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsleju7o%3D</pub-id><pub-id pub-id-type=\"pmid\">32323974</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.analchem.0c00784</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheng</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>Z</given-names></name><name><surname>Ma</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers as drug carriers: applications in different routes of drug administration</article-title><source>J Pharm Sci</source><year>2008</year><volume>97</volume><issue>1</issue><fpage>123</fpage><lpage>143</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXkslOrtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">17721949</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.21079</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cho</surname><given-names>H</given-names></name><name><surname>Lai</surname><given-names>TC</given-names></name><name><surname>Tomoda</surname><given-names>K</given-names></name><name><surname>Kwon</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles for multi-drug delivery in cancer</article-title><source>AAPS Pharm Sci Tech</source><year>2015</year><volume>16</volume><issue>1</issue><fpage>10</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitFait7nO</pub-id><pub-id pub-id-type=\"doi\">10.1208/s12249-014-0251-3</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choudhury</surname><given-names>KR</given-names></name><name><surname>Sahoo</surname><given-names>Y</given-names></name><name><surname>Ohulchanskyy</surname><given-names>T</given-names></name><name><surname>Prasad</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Efficient photoconductive devices at infrared wavelengths using quantum dot-polymer nanocomposites</article-title><source>Appl Phys Lett</source><year>2005</year><volume>87</volume><issue>7</issue><pub-id pub-id-type=\"doi\">10.1063/1.2011768</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Coleman</surname><given-names>CM</given-names></name><name><surname>Liu</surname><given-names>YV</given-names></name><name><surname>Mu</surname><given-names>H</given-names></name><name><surname>Taylor</surname><given-names>JK</given-names></name><name><surname>Massare</surname><given-names>M</given-names></name><name><surname>Flyer</surname><given-names>DC</given-names></name><name><surname>Glenn</surname><given-names>GM</given-names></name><name><surname>Smith</surname><given-names>GE</given-names></name><name><surname>Frieman</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</article-title><source>Vaccine</source><year>2014</year><volume>32</volume><issue>26</issue><fpage>3169</fpage><lpage>3174</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXnsVWltbc%3D</pub-id><pub-id pub-id-type=\"pmid\">24736006</pub-id><pub-id pub-id-type=\"pmcid\">4058772</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2014.04.016</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Comentale</surname><given-names>G</given-names></name><name><surname>Manzo</surname><given-names>R</given-names></name><name><surname>Pilato</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sars-Cov-2 interference in HEME production: is it the time for an early predictive biomarker?</article-title><source>J Mol Med</source><year>2020</year><volume>98</volume><issue>8</issue><fpage>1053</fpage><lpage>1054</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtleks7bO</pub-id><pub-id pub-id-type=\"pmid\">32601795</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00109-020-01945-4</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Corman</surname><given-names>VM</given-names></name><name><surname>Landt</surname><given-names>O</given-names></name><name><surname>Kaiser</surname><given-names>M</given-names></name><name><surname>Molenkamp</surname><given-names>R</given-names></name><name><surname>Meijer</surname><given-names>A</given-names></name><name><surname>Chu</surname><given-names>DK</given-names></name><name><surname>Bleicker</surname><given-names>T</given-names></name><name><surname>Brünink</surname><given-names>S</given-names></name><name><surname>Schneider</surname><given-names>J</given-names></name><name><surname>Schmidt</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</article-title><source>Eurosurveillance</source><year>2020</year><volume>25</volume><issue>3</issue><fpage>2000045</fpage><pub-id pub-id-type=\"pmid\">31992387</pub-id><pub-id pub-id-type=\"pmcid\">6988269</pub-id><pub-id pub-id-type=\"doi\">10.2807/1560-7917.ES.2020.25.3.2000045</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Craw</surname><given-names>P</given-names></name><name><surname>Balachandran</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Isothermal nucleic acid amplification technologies for point-of-care diagnostics: a critical review</article-title><source>Lab Chip</source><year>2012</year><volume>12</volume><issue>14</issue><fpage>2469</fpage><lpage>2486</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XovVylt7k%3D</pub-id><pub-id pub-id-type=\"pmid\">22592150</pub-id><pub-id pub-id-type=\"doi\">10.1039/c2lc40100b</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Daemi</surname><given-names>HB</given-names></name><name><surname>Kulyar</surname><given-names>MF</given-names></name><name><surname>He</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Karimpour</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>X</given-names></name><name><surname>Zou</surname><given-names>Z</given-names></name><name><surname>Jin</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Progression and trends in virus from influenza A to COVID-19: an overview of recent studies</article-title><source>Viruses</source><year>2021</year><volume>13</volume><issue>6</issue><fpage>1145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1OnurrF</pub-id><pub-id pub-id-type=\"pmid\">34203647</pub-id><pub-id pub-id-type=\"pmcid\">8232279</pub-id><pub-id pub-id-type=\"doi\">10.3390/v13061145</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Ribeiro</surname><given-names>LNM</given-names></name><name><surname>Fonseca</surname><given-names>BB</given-names></name></person-group><article-title xml:lang=\"en\">The role of pharmaceutical nanotechnology in the time of COVID-19 pandemic</article-title><source>Future Microbiol</source><year>2020</year><volume>15</volume><issue>16</issue><fpage>1571</fpage><lpage>1582</lpage><pub-id pub-id-type=\"doi\">10.2217/fmb-2020-0118</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Wit</surname><given-names>E</given-names></name><name><surname>Feldmann</surname><given-names>F</given-names></name><name><surname>Cronin</surname><given-names>J</given-names></name><name><surname>Jordan</surname><given-names>R</given-names></name><name><surname>Okumura</surname><given-names>A</given-names></name><name><surname>Thomas</surname><given-names>T</given-names></name><name><surname>Scott</surname><given-names>D</given-names></name><name><surname>Cihlar</surname><given-names>T</given-names></name><name><surname>Feldmann</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Prophylactic and therapeutic remdesivir (GS-5734) treatment in the rhesus macaque model of MERS-CoV infection</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>12</issue><fpage>6771</fpage><lpage>6776</lpage><pub-id pub-id-type=\"pmid\">32054787</pub-id><pub-id pub-id-type=\"pmcid\">7104368</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1922083117</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Deeks</surname><given-names>JJ</given-names></name><name><surname>Dinnes</surname><given-names>J</given-names></name><name><surname>Takwoingi</surname><given-names>Y</given-names></name><name><surname>Davenport</surname><given-names>C</given-names></name><name><surname>Spijker</surname><given-names>R</given-names></name><name><surname>Taylor-Phillips</surname><given-names>S</given-names></name><name><surname>Adriano</surname><given-names>A</given-names></name><name><surname>Beese</surname><given-names>S</given-names></name><name><surname>Dretzke</surname><given-names>J</given-names></name><name><surname>di Ruffano</surname><given-names>LF</given-names></name></person-group><article-title xml:lang=\"en\">Antibody tests for identification of current and past infection with SARS-CoV-2</article-title><source>Cochrane Database Syst Rev</source><year>2020</year><volume>2020</volume><fpage>CD013652</fpage><pub-id pub-id-type=\"pmcid\">7387103</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dincer</surname><given-names>C</given-names></name><name><surname>Bruch</surname><given-names>R</given-names></name><name><surname>Kling</surname><given-names>A</given-names></name><name><surname>Dittrich</surname><given-names>PS</given-names></name><name><surname>Urban</surname><given-names>GA</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed point-of-care testing–xPOCT</article-title><source>Trends Biotechnol</source><year>2017</year><volume>35</volume><issue>8</issue><fpage>728</fpage><lpage>742</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmtFGns70%3D</pub-id><pub-id pub-id-type=\"pmid\">28456344</pub-id><pub-id pub-id-type=\"pmcid\">5538621</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2017.03.013</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dong</surname><given-names>X</given-names></name><name><surname>Moyer</surname><given-names>MM</given-names></name><name><surname>Yang</surname><given-names>F</given-names></name><name><surname>Sun</surname><given-names>YP</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots’ antiviral functions against noroviruses</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>10</lpage></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Drobysh</surname><given-names>M</given-names></name><name><surname>Ramanaviciene</surname><given-names>A</given-names></name><name><surname>Viter</surname><given-names>R</given-names></name><name><surname>Chen</surname><given-names>CF</given-names></name><name><surname>Samukaite-Bubniene</surname><given-names>U</given-names></name><name><surname>Ratautaite</surname><given-names>V</given-names></name><name><surname>Ramanavicius</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Biosensors for the determination of SARS-CoV-2 virus and diagnosis of COVID-19 infection</article-title><source>Int J Mol Sci</source><year>2022</year><volume>23</volume><issue>2</issue><fpage>666</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitFertr4%3D</pub-id><pub-id pub-id-type=\"pmid\">35054850</pub-id><pub-id pub-id-type=\"pmcid\">8776074</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms23020666</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Liang</surname><given-names>J</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Liu</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots as inhibitors of virus by activation of type I interferon response</article-title><source>Carbon</source><year>2016</year><volume>110</volume><fpage>278</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFCmtLnF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbon.2016.09.032</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ella</surname><given-names>R</given-names></name><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>Blackwelder</surname><given-names>W</given-names></name><name><surname>Potdar</surname><given-names>V</given-names></name><name><surname>Yadav</surname><given-names>P</given-names></name><name><surname>Sarangi</surname><given-names>V</given-names></name><name><surname>Aileni</surname><given-names>VK</given-names></name><name><surname>Kanungo</surname><given-names>S</given-names></name><name><surname>Rai</surname><given-names>S</given-names></name><name><surname>Reddy</surname><given-names>P</given-names></name><name><surname>Verma</surname><given-names>S</given-names></name><name><surname>Singh</surname><given-names>C</given-names></name><name><surname>Redkar</surname><given-names>S</given-names></name><name><surname>Mohapatra</surname><given-names>S</given-names></name><name><surname>Pandey</surname><given-names>A</given-names></name><name><surname>Ranganadin</surname><given-names>P</given-names></name><name><surname>Gumashta</surname><given-names>R</given-names></name><name><surname>Multani</surname><given-names>M</given-names></name><name><surname>Mohammad</surname><given-names>S</given-names></name><name><surname>Bhatt</surname><given-names>P</given-names></name><name><surname>Kumari</surname><given-names>L</given-names></name><name><surname>Sapkal</surname><given-names>G</given-names></name><name><surname>Gupta</surname><given-names>N</given-names></name><name><surname>Abraham</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>S</given-names></name><name><surname>Prasad</surname><given-names>S</given-names></name><name><surname>Bhargava</surname><given-names>B</given-names></name><name><surname>Ella</surname><given-names>K</given-names></name><name><surname>Vadrevu</surname><given-names>KM</given-names></name><collab>COVAXIN Study Group</collab></person-group><article-title xml:lang=\"en\">Efficacy, safety, and lot-to-lot immunogenicity of an inactivated SARS-CoV-2 vaccine (BBV152): interim results of a randomised, double-blind, controlled, phase 3 trial</article-title><source>Lancet (London, England)</source><year>2021</year><volume>398</volume><issue>10317</issue><fpage>2173</fpage><lpage>2184</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1eqsrfE</pub-id><pub-id pub-id-type=\"pmid\">34774196</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(21)02000-6</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Esbin</surname><given-names>MN</given-names></name><name><surname>Whitney</surname><given-names>ON</given-names></name><name><surname>Chong</surname><given-names>S</given-names></name><name><surname>Maurer</surname><given-names>A</given-names></name><name><surname>Darzacq</surname><given-names>X</given-names></name><name><surname>Tjian</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Overcoming the bottleneck to widespread testing: a rapid review of nucleic acid testing approaches for COVID-19 detection</article-title><source>RNA</source><year>2020</year><volume>26</volume><issue>7</issue><fpage>771</fpage><lpage>783</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Klt7rP</pub-id><pub-id pub-id-type=\"pmid\">32358057</pub-id><pub-id pub-id-type=\"pmcid\">7297120</pub-id><pub-id pub-id-type=\"doi\">10.1261/rna.076232.120</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Flühmann</surname><given-names>B</given-names></name><name><surname>Ntai</surname><given-names>I</given-names></name><name><surname>Borchard</surname><given-names>G</given-names></name><name><surname>Simoens</surname><given-names>S</given-names></name><name><surname>Mühlebach</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicines: the magic bullets reaching their target?</article-title><source>Eur J Pharm Sci</source><year>2019</year><volume>128</volume><fpage>73</fpage><lpage>80</lpage><pub-id pub-id-type=\"pmid\">30465818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2018.11.019</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Franklyne</surname><given-names>JS</given-names></name><name><surname>Gopinath</surname><given-names>PM</given-names></name><name><surname>Mukherjee</surname><given-names>A</given-names></name><name><surname>Chandrasekaran</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Nanoemulsions: the rising star of antiviral therapeutics and nanodelivery system—Current status and prospects</article-title><source>Curr Opin Colloid Interface Sci</source><year>2021</year><volume>54</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXptlansro%3D</pub-id><pub-id pub-id-type=\"pmid\">33814954</pub-id><pub-id pub-id-type=\"pmcid\">8007535</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cocis.2021.101458</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Freitas</surname><given-names>RA</given-names><suffix>Jr</suffix></name></person-group><article-title xml:lang=\"en\">What is nanomedicine? Nanomedicine: nanotechnology</article-title><source>Biology Med</source><year>2005</year><volume>1</volume><issue>1</issue><fpage>2</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXlvVSrurs%3D</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Garg</surname><given-names>B</given-names></name><name><surname>Beg</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>R</given-names></name><name><surname>Katara</surname><given-names>O</given-names></name><name><surname>Singh</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructured lipidic carriers of lopinavir for effective management of HIV-associated neurocognitive disorder</article-title><source>J Drug Deliv Sci Technol</source><year>2019</year><volume>53</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhs1artbfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2019.101220</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Goodwin</surname><given-names>TJ</given-names></name><name><surname>Huang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of phosphorylated adjuvants co-encapsulated with a model cancer peptide antigen for the treatment of colorectal cancer and liver metastasis</article-title><source>Vaccine</source><year>2017</year><volume>35</volume><issue>19</issue><fpage>2550</fpage><lpage>2557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsFagt78%3D</pub-id><pub-id pub-id-type=\"pmid\">28385609</pub-id><pub-id pub-id-type=\"pmcid\">5448460</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2017.03.067</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gregory</surname><given-names>AE</given-names></name><name><surname>Titball</surname><given-names>R</given-names></name><name><surname>Williamson</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Vaccine delivery using nanoparticles</article-title><source>Front Incell Infect Microbiol</source><year>2013</year><volume>3</volume><fpage>13</fpage></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Habibi</surname><given-names>R</given-names></name><name><surname>Burci</surname><given-names>GL</given-names></name><name><surname>De Campos</surname><given-names>TC</given-names></name><name><surname>Chirwa</surname><given-names>D</given-names></name><name><surname>Cinà</surname><given-names>M</given-names></name><name><surname>Dagron</surname><given-names>S</given-names></name><name><surname>Eccleston-Turner</surname><given-names>M</given-names></name><name><surname>Forman</surname><given-names>L</given-names></name><name><surname>Gostin</surname><given-names>LO</given-names></name><name><surname>Mier</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">Do not violate the international health regulations during the COVID-19 outbreak</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10225</issue><fpage>664</fpage><lpage>666</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXksVaisLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30373-1</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halbus</surname><given-names>AF</given-names></name><name><surname>Horozov</surname><given-names>TS</given-names></name><name><surname>Paunov</surname><given-names>VN</given-names></name></person-group><article-title xml:lang=\"en\">Colloid particle formulations for antimicrobial applications</article-title><source>Adv Coll Interface Sci</source><year>2017</year><volume>249</volume><fpage>134</fpage><lpage>148</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXotVarsrY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2017.05.012</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>HS</given-names></name><name><surname>Niemeyer</surname><given-names>E</given-names></name><name><surname>Huang</surname><given-names>Y</given-names></name><name><surname>Kamoun</surname><given-names>WS</given-names></name><name><surname>Martin</surname><given-names>JD</given-names></name><name><surname>Bhaumik</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Roberge</surname><given-names>S</given-names></name><name><surname>Cui</surname><given-names>J</given-names></name><name><surname>Martin</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dot/antibody conjugates for in vivo cytometric imaging in mice</article-title><source>Proc Natl Acad Sci</source><year>2015</year><volume>112</volume><issue>5</issue><fpage>1350</fpage><lpage>1355</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtFKhtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">25605916</pub-id><pub-id pub-id-type=\"pmcid\">4321304</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1421632111</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>GR</given-names></name><name><surname>Ki</surname><given-names>H</given-names></name><name><surname>Kim</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Automated, universal, and mass-producible paper-based lateral flow biosensing platform for high-performance point-of-care testing</article-title><source>ACS Appl Mater Interfaces</source><year>2019</year><volume>12</volume><issue>1</issue><fpage>1885</fpage><lpage>1894</lpage><pub-id pub-id-type=\"pmid\">31813220</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.9b17888</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hardick</surname><given-names>J</given-names></name><name><surname>Metzgar</surname><given-names>D</given-names></name><name><surname>Risen</surname><given-names>L</given-names></name><name><surname>Myers</surname><given-names>C</given-names></name><name><surname>Balansay</surname><given-names>M</given-names></name><name><surname>Malcom</surname><given-names>T</given-names></name><name><surname>Rothman</surname><given-names>R</given-names></name><name><surname>Gaydos</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Initial performance evaluation of a spotted array Mobile Analysis Platform (MAP) for the detection of influenza A/B, RSV, and MERS coronavirus</article-title><source>Diagn Microbiol Infect Dis</source><year>2018</year><volume>91</volume><issue>3</issue><fpage>245</fpage><lpage>247</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXlt1SisLg%3D</pub-id><pub-id pub-id-type=\"pmid\">29550057</pub-id><pub-id pub-id-type=\"pmcid\">6013072</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.diagmicrobio.2018.02.011</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Harvey</surname><given-names>CJ</given-names></name><name><surname>O’Regan</surname><given-names>DP</given-names></name></person-group><article-title xml:lang=\"en\">Radiological investigations and applications</article-title><source>Med MRCP</source><year>2020</year><volume>17</volume><fpage>111</fpage><pub-id pub-id-type=\"doi\">10.1093/med/9780198779506.003.0005</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Haslberger</surname><given-names>A</given-names></name><name><surname>Jacob</surname><given-names>U</given-names></name><name><surname>Hippe</surname><given-names>B</given-names></name><name><surname>Karlic</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Mechanisms of selected functional foods against viral infections with a view on COVID-19: Mini review</article-title><source>Funct Foods Health Dis</source><year>2020</year><volume>10</volume><issue>5</issue><fpage>195</fpage><lpage>209</lpage><pub-id pub-id-type=\"doi\">10.31989/ffhd.v10i5.707</pub-id></mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heymann</surname><given-names>DL</given-names></name><name><surname>Shindo</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19: what is next for public health?</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>542</fpage><lpage>545</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30374-3</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hofmann</surname><given-names>MA</given-names></name><name><surname>Renzullo</surname><given-names>S</given-names></name><name><surname>Mader</surname><given-names>M</given-names></name><name><surname>Chaignat</surname><given-names>V</given-names></name><name><surname>Worwa</surname><given-names>G</given-names></name><name><surname>Thuer</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Genetic characterization of toggenburg orbivirus a new bluetongue virus, from goats. Switzerland</article-title><source>Emerg Infect Dis</source><year>2008</year><volume>14</volume><issue>12</issue><fpage>1855</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXjvVKltA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19046507</pub-id><pub-id pub-id-type=\"pmcid\">2634640</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid1412.080818</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Fan</surname><given-names>G</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Gu</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>497</fpage><lpage>506</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Kqu7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30183-5</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>Z</given-names></name><name><surname>Jiang</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Yi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>Z</given-names></name><name><surname>Fan</surname><given-names>S</given-names></name><name><surname>Liao</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Jiang</surname><given-names>G</given-names></name><name><surname>Tang</surname><given-names>D</given-names></name><name><surname>Ye</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Ma</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 inactivated vaccine (Vero cells) shows good safety in repeated administration toxicity test of Sprague Dawley rats</article-title><source>Food Chem Toxicol</source><year>2021</year><volume>152</volume><fpage>112239</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXpvVSmtLY%3D</pub-id><pub-id pub-id-type=\"pmid\">33901607</pub-id><pub-id pub-id-type=\"pmcid\">8064818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fct.2021.112239</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Iannazzo</surname><given-names>D</given-names></name><name><surname>Pistone</surname><given-names>A</given-names></name><name><surname>Ferro</surname><given-names>S</given-names></name><name><surname>De Luca</surname><given-names>L</given-names></name><name><surname>Monforte</surname><given-names>AM</given-names></name><name><surname>Romeo</surname><given-names>R</given-names></name><name><surname>Buemi</surname><given-names>MR</given-names></name><name><surname>Pannecouque</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Graphene quantum dots based systems as HIV inhibitors</article-title><source>Bioconjug Chem</source><year>2018</year><volume>29</volume><issue>9</issue><fpage>3084</fpage><lpage>3093</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsV2lsLjK</pub-id><pub-id pub-id-type=\"pmid\">30106563</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.bioconjchem.8b00448</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Kaneko</surname><given-names>K</given-names></name><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Mizushima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Development of biodegradable nanoparticles for liver-specific ribavirin delivery</article-title><source>J Pharm Sci</source><year>2014</year><volume>103</volume><issue>12</issue><fpage>4005</fpage><lpage>4011</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVaktrjF</pub-id><pub-id pub-id-type=\"pmid\">25335768</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.24219</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Islam</surname><given-names>A</given-names></name><name><surname>Ahsan</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Plausible approach for rapid detection of SARS-CoV-2 virus by magnetic nanoparticle based biosensors</article-title><source>Am J Nanosci</source><year>2020</year><volume>6</volume><fpage>6</fpage><pub-id pub-id-type=\"doi\">10.11648/j.ajn.20200602.11</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Itani</surname><given-names>R</given-names></name><name><surname>Tobaiqy</surname><given-names>M</given-names></name><name><surname>Al Faraj</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Optimizing use of theranostic nanoparticles as a life-saving strategy for treating COVID-19 patients</article-title><source>Theranostics</source><year>2020</year><volume>10</volume><issue>13</issue><fpage>5932</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFGgsrzI</pub-id><pub-id pub-id-type=\"pmid\">32483428</pub-id><pub-id pub-id-type=\"pmcid\">7254986</pub-id><pub-id pub-id-type=\"doi\">10.7150/thno.46691</pub-id></mixed-citation></ref><ref id=\"CR74\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jacob</surname><given-names>S</given-names></name><name><surname>Nair</surname><given-names>AB</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Emerging role of nanosuspensions in drug delivery systems</article-title><source>Biomater Res</source><year>2020</year><volume>24</volume><issue>1</issue><fpage>1</fpage><lpage>16</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlOnurs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s40824-020-0184-8</pub-id></mixed-citation></ref><ref id=\"CR75\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaimes</surname><given-names>JA</given-names></name><name><surname>Millet</surname><given-names>JK</given-names></name><name><surname>Stout</surname><given-names>AE</given-names></name><name><surname>André</surname><given-names>NM</given-names></name><name><surname>Whittaker</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">A tale of two viruses: the distinct spike glycoproteins of feline coronaviruses</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>1</issue><fpage>83</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsVCquro%3D</pub-id><pub-id pub-id-type=\"pmid\">31936749</pub-id><pub-id pub-id-type=\"pmcid\">7019228</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12010083</pub-id></mixed-citation></ref><ref id=\"CR76\"><mixed-citation publication-type=\"other\">Jan N, Madni A, Khan S, Shah H, Akram F, Khan A, Ertas D, Bostanudin MF, Contag CH, Ashammakhi N, Ertas YN (2022) Biomimetic cell membrane‐coated poly (lactic‐co‐glycolic acid) nanoparticles for biomedical applications. Bioeng Transl Med. p e10441</mixed-citation></ref><ref id=\"CR77\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Javan</surname><given-names>F</given-names></name><name><surname>Vatanara</surname><given-names>A</given-names></name><name><surname>Azadmanesh</surname><given-names>K</given-names></name><name><surname>Nabi-Meibodi</surname><given-names>M</given-names></name><name><surname>Shakouri</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Encapsulation of ritonavir in solid lipid nanoparticles: in-vitro anti-HIV-1 activity using lentiviral particles</article-title><source>J Pharm Pharmacol</source><year>2017</year><volume>69</volume><issue>8</issue><fpage>1002</fpage><lpage>1009</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXntVKhsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">28471000</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12737</pub-id></mixed-citation></ref><ref id=\"CR78\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jiang</surname><given-names>M</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Arasthfer</surname><given-names>A</given-names></name><name><surname>Fang</surname><given-names>W</given-names></name><name><surname>Ling</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>H</given-names></name><name><surname>Daneshnia</surname><given-names>F</given-names></name><name><surname>Yu</surname><given-names>J</given-names></name><name><surname>Liao</surname><given-names>W</given-names></name><name><surname>Pei</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Development and validation of a apid, single-step reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) system potentially to be used for reliable and high-throughput screening of COVID-19</article-title><source>Front Cell Infect Microbiol</source><year>2020</year><volume>10</volume><fpage>331</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaktbbE</pub-id><pub-id pub-id-type=\"pmid\">32626666</pub-id><pub-id pub-id-type=\"pmcid\">7313420</pub-id><pub-id pub-id-type=\"doi\">10.3389/fcimb.2020.00331</pub-id></mixed-citation></ref><ref id=\"CR79\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jones</surname><given-names>ST</given-names></name><name><surname>Cagno</surname><given-names>V</given-names></name><name><surname>Janeček</surname><given-names>M</given-names></name><name><surname>Ortiz</surname><given-names>D</given-names></name><name><surname>Gasilova</surname><given-names>N</given-names></name><name><surname>Piret</surname><given-names>J</given-names></name><name><surname>Gasbarri</surname><given-names>M</given-names></name><name><surname>Constant</surname><given-names>DA</given-names></name><name><surname>Han</surname><given-names>Y</given-names></name><name><surname>Vuković</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Modified cyclodextrins as broad-spectrum antivirals</article-title><source>Sci Adv</source><year>2020</year><volume>6</volume><issue>5</issue><fpage>eaax9318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGgsr%2FL</pub-id><pub-id pub-id-type=\"pmid\">32064341</pub-id><pub-id pub-id-type=\"pmcid\">6989148</pub-id><pub-id pub-id-type=\"doi\">10.1126/sciadv.aax9318</pub-id></mixed-citation></ref><ref id=\"CR80\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kalita</surname><given-names>MJ</given-names></name><name><surname>Dutta</surname><given-names>K</given-names></name><name><surname>Hazarika</surname><given-names>G</given-names></name><name><surname>Dutta</surname><given-names>R</given-names></name><name><surname>Kalita</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>PP</given-names></name><name><surname>Sarma</surname><given-names>MP</given-names></name><name><surname>Banu</surname><given-names>S</given-names></name><name><surname>Idris</surname><given-names>M</given-names></name><name><surname>Talukdar</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">In-house reverse transcriptase polymerase chain reaction for detection of SARS-CoV-2 with increased sensitivity</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>12</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-97502-1</pub-id></mixed-citation></ref><ref id=\"CR81\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kandeel</surname><given-names>M</given-names></name><name><surname>Al-Taher</surname><given-names>A</given-names></name><name><surname>Park</surname><given-names>BK</given-names></name><name><surname>Kwon</surname><given-names>HJ</given-names></name><name><surname>Al-Nazawi</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A pilot study of the antiviral activity of anionic and cationic polyamidoamine dendrimers against the Middle East respiratory syndrome coronavirus</article-title><source>J Med Virol</source><year>2020</year><volume>92</volume><issue>9</issue><fpage>1665</fpage><lpage>1670</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVyhsb8%3D</pub-id><pub-id pub-id-type=\"pmid\">32330296</pub-id><pub-id pub-id-type=\"pmcid\">7264540</pub-id><pub-id pub-id-type=\"doi\">10.1002/jmv.25928</pub-id></mixed-citation></ref><ref id=\"CR82\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Katata-Seru</surname><given-names>L</given-names></name><name><surname>Ojo</surname><given-names>BM</given-names></name><name><surname>Okubanjo</surname><given-names>O</given-names></name><name><surname>Soremekun</surname><given-names>R</given-names></name><name><surname>Aremu</surname><given-names>OS</given-names></name></person-group><article-title xml:lang=\"en\">Nanoformulated Eudragit lopinavir and preliminary release of its loaded suppositories</article-title><source>Heliyon</source><year>2020</year><volume>6</volume><issue>5</issue><pub-id pub-id-type=\"pmid\">32420478</pub-id><pub-id pub-id-type=\"pmcid\">7218025</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.heliyon.2020.e03890</pub-id></mixed-citation></ref><ref id=\"CR83\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaur</surname><given-names>I</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Jakhar</surname><given-names>D</given-names></name><name><surname>Das</surname><given-names>A</given-names></name><name><surname>Aradhya</surname><given-names>SS</given-names></name><name><surname>Sharma</surname><given-names>R</given-names></name><name><surname>Jindal</surname><given-names>V</given-names></name><name><surname>Mhatre</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease (COVID-19): An updated review based on current knowledge and existing literature for dermatologists</article-title><source>Dermatol Ther</source><year>2020</year><volume>33</volume><issue>4</issue><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXps12jsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">32447820</pub-id><pub-id pub-id-type=\"pmcid\">7283871</pub-id><pub-id pub-id-type=\"doi\">10.1111/dth.13677</pub-id></mixed-citation></ref><ref id=\"CR84\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>AK</given-names></name><name><surname>Dhau</surname><given-names>JS</given-names></name><name><surname>Gohel</surname><given-names>H</given-names></name><name><surname>Mishra</surname><given-names>YK</given-names></name><name><surname>Kateb</surname><given-names>B</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Goswami</surname><given-names>DY</given-names></name></person-group><article-title xml:lang=\"en\">Electrochemical SARS-CoV-2 sensing at point-of-care and artificial intelligence for intelligent COVID-19 management</article-title><source>ACS Appl Bio Mater</source><year>2020</year><volume>3</volume><issue>11</issue><fpage>7306</fpage><lpage>7325</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGltr3I</pub-id><pub-id pub-id-type=\"pmid\">35019473</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsabm.0c01004</pub-id></mixed-citation></ref><ref id=\"CR85\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Jain</surname><given-names>K</given-names></name><name><surname>Jain</surname><given-names>NK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer as nanocarrier for drug delivery</article-title><source>Prog Polym Sci</source><year>2014</year><volume>39</volume><issue>2</issue><fpage>268</fpage><lpage>307</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1Wrur7O</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.progpolymsci.2013.07.005</pub-id></mixed-citation></ref><ref id=\"CR86\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Xie</surname><given-names>L</given-names></name><name><surname>Banerjee</surname><given-names>S</given-names></name><name><surname>Mao</surname><given-names>G</given-names></name><name><surname>Padhye</surname><given-names>S</given-names></name><name><surname>Sarkar</surname><given-names>FH</given-names></name><name><surname>Iyer</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Hyaluronic acid-conjugated polyamidoamine dendrimers for targeted delivery of 3, 4-difluorobenzylidene curcumin to CD44 overexpressing pancreatic cancer cells</article-title><source>Colloids Surf, B</source><year>2015</year><volume>136</volume><fpage>413</fpage><lpage>423</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhs1aktr3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2015.09.043</pub-id></mixed-citation></ref><ref id=\"CR87\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalaj-Hedayati</surname><given-names>A</given-names></name><name><surname>Chua</surname><given-names>CLL</given-names></name><name><surname>Smooker</surname><given-names>P</given-names></name><name><surname>Lee</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Nanoparticles in influenza subunit vaccine development: immunogenicity enhancement</article-title><source>Influenza Other Respir Viruses</source><year>2020</year><volume>14</volume><issue>1</issue><fpage>92</fpage><lpage>101</lpage><pub-id pub-id-type=\"pmid\">31774251</pub-id><pub-id pub-id-type=\"doi\">10.1111/irv.12697</pub-id></mixed-citation></ref><ref id=\"CR88\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khan</surname><given-names>AA</given-names></name><name><surname>Mudassir</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>S</given-names></name><name><surname>Murugaiyah</surname><given-names>V</given-names></name><name><surname>Darwis</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Freeze-dried lopinavir-loaded nanostructured lipid carriers for enhanced cellular uptake and bioavailability: statistical optimization, in vitro and in vivo evaluations</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>2</issue><fpage>97</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlWhtLrP</pub-id><pub-id pub-id-type=\"pmid\">30823545</pub-id><pub-id pub-id-type=\"pmcid\">6410192</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11020097</pub-id></mixed-citation></ref><ref id=\"CR89\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kobayashi</surname><given-names>K</given-names></name><name><surname>Wei</surname><given-names>J</given-names></name><name><surname>Iida</surname><given-names>R</given-names></name><name><surname>Ijiro</surname><given-names>K</given-names></name><name><surname>Niikura</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Surface engineering of nanoparticles for therapeutic applications</article-title><source>Polym J</source><year>2014</year><volume>46</volume><issue>8</issue><fpage>460</fpage><lpage>468</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1KksLbK</pub-id><pub-id pub-id-type=\"doi\">10.1038/pj.2014.40</pub-id></mixed-citation></ref><ref id=\"CR90\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Koshy</surname><given-names>ST</given-names></name><name><surname>Cheung</surname><given-names>AS</given-names></name><name><surname>Gu</surname><given-names>L</given-names></name><name><surname>Graveline</surname><given-names>AR</given-names></name><name><surname>Mooney</surname><given-names>DJ</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal delivery enhances immune activation by STING agonists for cancer immunotherapy</article-title><source>Adv Biosyst</source><year>2017</year><volume>1</volume><issue>1–2</issue><fpage>1600013</fpage><pub-id pub-id-type=\"pmid\">30258983</pub-id><pub-id pub-id-type=\"pmcid\">6152940</pub-id><pub-id pub-id-type=\"doi\">10.1002/adbi.201600013</pub-id></mixed-citation></ref><ref id=\"CR91\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kulthe</surname><given-names>SS</given-names></name><name><surname>Choudhari</surname><given-names>YM</given-names></name><name><surname>Inamdar</surname><given-names>NN</given-names></name><name><surname>Mourya</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles: authoritative aspects for drug delivery</article-title><source>Des Monomers Polym</source><year>2012</year><volume>15</volume><issue>5</issue><fpage>465</fpage><lpage>521</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsVSqsL%2FP</pub-id><pub-id pub-id-type=\"doi\">10.1080/1385772X.2012.688328</pub-id></mixed-citation></ref><ref id=\"CR92\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>M</given-names></name><name><surname>Curtis</surname><given-names>A</given-names></name><name><surname>Hoskins</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanoparticle technologies in the combat against anti-microbial resistance</article-title><source>Pharmaceutics</source><year>2018</year><volume>10</volume><issue>1</issue><fpage>11</fpage><pub-id pub-id-type=\"pmid\">29342903</pub-id><pub-id pub-id-type=\"pmcid\">5874824</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics10010011</pub-id></mixed-citation></ref><ref id=\"CR93\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Naha</surname><given-names>A</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development of ritonavir solid lipid nanoparticles by Box Behnken design for intestinal lymphatic targeting</article-title><source>J Drug Deliv Sci Technol</source><year>2018</year><volume>44</volume><fpage>181</fpage><lpage>189</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXis1Clsg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2017.12.014</pub-id></mixed-citation></ref><ref id=\"CR94\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight to ROS and apoptosis regulated cytotoxicity inferred by green synthesized CuO nanoparticles from Calotropis gigantea to embryonic zebrafish</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>17</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-017-16581-1</pub-id></mixed-citation></ref><ref id=\"CR95\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Pramanik</surname><given-names>N</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Soni</surname><given-names>N</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Molecular insight to in vitro biocompatibility of phytofabricated copper oxide nanoparticles with human embryonic kidney cells</article-title><source>Nanomedicine</source><year>2018</year><volume>13</volume><issue>19</issue><fpage>2415</fpage><lpage>2433</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitV2isL%2FN</pub-id><pub-id pub-id-type=\"pmid\">30251920</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2018-0175</pub-id></mixed-citation></ref><ref id=\"CR96\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupferschmidt</surname><given-names>K</given-names></name><name><surname>Cohen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Race to find COVID-19 treatments accelerates</article-title><source>Am Assoc Adv Sci</source><year>2020</year><volume>367</volume><fpage>6485</fpage></mixed-citation></ref><ref id=\"CR97\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>S</given-names></name><name><surname>Ruktanonchai</surname><given-names>NW</given-names></name><name><surname>Zhou</surname><given-names>L</given-names></name><name><surname>Prosper</surname><given-names>O</given-names></name><name><surname>Luo</surname><given-names>W</given-names></name><name><surname>Floyd</surname><given-names>JR</given-names></name><name><surname>Wesolowski</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Du</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak: an observational and modelling study</article-title><source>MedRxiv</source><year>2020</year><volume>202003</volume><issue>03</issue><fpage>20029843</fpage></mixed-citation></ref><ref id=\"CR98\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamb</surname><given-names>YN</given-names></name></person-group><article-title xml:lang=\"en\">BNT162b2 mRNA COVID-19 vaccine: first approval</article-title><source>Drugs</source><year>2021</year><volume>81</volume><fpage>495</fpage><lpage>501</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1yquro%3D</pub-id><pub-id pub-id-type=\"pmid\">33683637</pub-id><pub-id pub-id-type=\"pmcid\">7938284</pub-id><pub-id pub-id-type=\"doi\">10.1007/s40265-021-01480-7</pub-id></mixed-citation></ref><ref id=\"CR99\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamote</surname><given-names>K</given-names></name><name><surname>Janssens</surname><given-names>E</given-names></name><name><surname>Schillebeeckx</surname><given-names>E</given-names></name><name><surname>Lapperre</surname><given-names>TS</given-names></name><name><surname>De Winter</surname><given-names>BY</given-names></name><name><surname>Van Meerbeeck</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">The scent of COVID-19: viral (semi-) volatiles as fast diagnostic biomarkers?</article-title><source>J Breath Res</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>042001</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1egsrvJ</pub-id><pub-id pub-id-type=\"pmid\">32599571</pub-id><pub-id pub-id-type=\"doi\">10.1088/1752-7163/aba105</pub-id></mixed-citation></ref><ref id=\"CR100\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Law</surname><given-names>JWF</given-names></name><name><surname>Ab Mutalib</surname><given-names>NS</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Lee</surname><given-names>LH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid methods for the detection of foodborne bacterial pathogens: principles, applications, advantages and limitations</article-title><source>Front Microbiol</source><year>2015</year><volume>5</volume><fpage>770</fpage><pub-id pub-id-type=\"pmid\">25628612</pub-id><pub-id pub-id-type=\"pmcid\">4290631</pub-id><pub-id pub-id-type=\"doi\">10.3389/fmicb.2014.00770</pub-id></mixed-citation></ref><ref id=\"CR101\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Le</surname><given-names>TT</given-names></name><name><surname>Andreadakis</surname><given-names>Z</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Román</surname><given-names>RG</given-names></name><name><surname>Tollefsen</surname><given-names>S</given-names></name><name><surname>Saville</surname><given-names>M</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><issue>5</issue><fpage>305</fpage><lpage>306</lpage><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00073-5</pub-id></mixed-citation></ref><ref id=\"CR102\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ledet</surname><given-names>G</given-names></name><name><surname>Mandal</surname><given-names>TK</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine: emerging therapeutics for the 21st century</article-title><source>US Pharm</source><year>2012</year><volume>37</volume><issue>3</issue><fpage>7</fpage><lpage>11</lpage></mixed-citation></ref><ref id=\"CR103\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lembo</surname><given-names>D</given-names></name><name><surname>Donalisio</surname><given-names>M</given-names></name><name><surname>Civra</surname><given-names>A</given-names></name><name><surname>Argenziano</surname><given-names>M</given-names></name><name><surname>Cavalli</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine formulations for the delivery of antiviral drugs: a promising solution for the treatment of viral infections</article-title><source>Expert Opin Drug Deliv</source><year>2018</year><volume>15</volume><issue>1</issue><fpage>93</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXht12ltr%2FO</pub-id><pub-id pub-id-type=\"pmid\">28749739</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2017.1360863</pub-id></mixed-citation></ref><ref id=\"CR104\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>SM</given-names></name><name><surname>Yu</surname><given-names>XH</given-names></name><name><surname>Tang</surname><given-names>SL</given-names></name><name><surname>Tang</surname><given-names>CK</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 (COVID-19): current status and future perspectives</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><issue>5</issue><fpage>105951</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSksLs%3D</pub-id><pub-id pub-id-type=\"pmid\">32234466</pub-id><pub-id pub-id-type=\"pmcid\">7139247</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105951</pub-id></mixed-citation></ref><ref id=\"CR105\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Geng</surname><given-names>M</given-names></name><name><surname>Peng</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Molecular immune pathogenesis and diagnosis of COVID-19</article-title><source>J Pharm Anal</source><year>2020</year><volume>10</volume><issue>2</issue><fpage>102</fpage><lpage>108</lpage><pub-id pub-id-type=\"pmid\">32282863</pub-id><pub-id pub-id-type=\"pmcid\">7104082</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jpha.2020.03.001</pub-id></mixed-citation></ref><ref id=\"CR106\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wu</surname><given-names>D</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Chu</surname><given-names>X</given-names></name><name><surname>Qin</surname><given-names>C</given-names></name><name><surname>Han</surname><given-names>X</given-names></name><name><surname>Hang</surname><given-names>T</given-names></name><name><surname>Xu</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name><name><surname>Yin</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal remdesivir inhalation solution for targeted lung delivery as a novel therapeutic approach for COVID-19</article-title><source>Asian J Pharm Sci</source><year>2021</year><volume>16</volume><issue>6</issue><fpage>772</fpage><lpage>783</lpage><pub-id pub-id-type=\"pmid\">34703490</pub-id><pub-id pub-id-type=\"pmcid\">8529908</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ajps.2021.09.002</pub-id></mixed-citation></ref><ref id=\"CR107\"><mixed-citation publication-type=\"other\">Lim ES, Wang D. Pathogen discovery (2016) Molecular microbiology: diagnostic principles and practice. pp 80–91</mixed-citation></ref><ref id=\"CR108\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lau</surname><given-names>SK</given-names></name><name><surname>Varma</surname><given-names>VA</given-names></name><name><surname>Moffitt</surname><given-names>RA</given-names></name><name><surname>Caldwell</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>T</given-names></name><name><surname>Young</surname><given-names>AN</given-names></name><name><surname>Petros</surname><given-names>JA</given-names></name><name><surname>Osunkoya</surname><given-names>AO</given-names></name><name><surname>Krogstad</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Molecular mapping of tumor heterogeneity on clinical tissue specimens with multiplexed quantum dots</article-title><source>ACS Nano</source><year>2010</year><volume>4</volume><issue>5</issue><fpage>2755</fpage><lpage>2765</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXksVKltLc%3D</pub-id><pub-id pub-id-type=\"pmid\">20377268</pub-id><pub-id pub-id-type=\"pmcid\">2923482</pub-id><pub-id pub-id-type=\"doi\">10.1021/nn100213v</pub-id></mixed-citation></ref><ref id=\"CR109\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>YC</given-names></name><name><surname>Liao</surname><given-names>CH</given-names></name><name><surname>Chang</surname><given-names>CF</given-names></name><name><surname>Chou</surname><given-names>CC</given-names></name><name><surname>Lin</surname><given-names>YR</given-names></name></person-group><article-title xml:lang=\"en\">A locally transmitted case of SARS-CoV-2 infection in Taiwan</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>11</issue><fpage>1070</fpage><lpage>1072</lpage><pub-id pub-id-type=\"pmid\">32050059</pub-id><pub-id pub-id-type=\"pmcid\">7121202</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMc2001573</pub-id></mixed-citation></ref><ref id=\"CR110\"><mixed-citation publication-type=\"other\">Lohcharoenkal W, Wang L, Chen YC, Rojanasakul Y (2014) Protein nanoparticles as drug delivery carriers for cancer therapy. BioMed Res Int </mixed-citation></ref><ref id=\"CR111\"><mixed-citation publication-type=\"other\">Lokhande SS (2018) Liposome drug delivery: an update review. Pharm Sci Monit 9(1)</mixed-citation></ref><ref id=\"CR112\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Caracterización genómica y epidemiología del nuevo coronavirus 2019: implicaciones para los orígenes del virus y la unión al receptor</article-title><source>Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>566</fpage><lpage>568</lpage></mixed-citation></ref><ref id=\"CR113\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Niu</surname><given-names>P</given-names></name><name><surname>Yang</surname><given-names>B</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>565</fpage><lpage>574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOmsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30251-8</pub-id></mixed-citation></ref><ref id=\"CR114\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lustig</surname><given-names>Y</given-names></name><name><surname>Keler</surname><given-names>S</given-names></name><name><surname>Kolodny</surname><given-names>R</given-names></name><name><surname>Ben-Tal</surname><given-names>N</given-names></name><name><surname>Atias-Varon</surname><given-names>D</given-names></name><name><surname>Shlush</surname><given-names>E</given-names></name><name><surname>Gerlic</surname><given-names>M</given-names></name><name><surname>Munitz</surname><given-names>A</given-names></name><name><surname>Doolman</surname><given-names>R</given-names></name><name><surname>Asraf</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Potential antigenic cross-reactivity between severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and dengue viruses</article-title><source>Clin Infect Dis</source><year>2021</year><volume>73</volume><issue>7</issue><fpage>e2444</fpage><lpage>e2449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1Wrt7%2FE</pub-id><pub-id pub-id-type=\"pmid\">32797228</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1207</pub-id></mixed-citation></ref><ref id=\"CR115\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>Q</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Xie</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Jiang</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Liu Shen capsule shows antiviral and anti-inflammatory abilities against novel coronavirus SARS-CoV-2 via suppression of NF-κB signaling pathway</article-title><source>Pharmacol Res</source><year>2020</year><volume>158</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVeltLrK</pub-id><pub-id pub-id-type=\"pmid\">32360580</pub-id><pub-id pub-id-type=\"pmcid\">7192119</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.phrs.2020.104850</pub-id></mixed-citation></ref><ref id=\"CR116\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Guo</surname><given-names>J</given-names></name><name><surname>Morawska</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 patients in earlier stages exhaled millions of severe acute respiratory syndrome coronavirus 2 per hour</article-title><source>Clin Infect Dis</source><year>2021</year><volume>72</volume><issue>10</issue><fpage>e652</fpage><lpage>e654</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFyitrvM</pub-id><pub-id pub-id-type=\"pmid\">32857833</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1283</pub-id></mixed-citation></ref><ref id=\"CR117\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maggi</surname><given-names>F</given-names></name><name><surname>Pistello</surname><given-names>M</given-names></name><name><surname>Antonelli</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Future management of viral diseases: role of new technologies and new approaches in microbial interactions</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>2</issue><fpage>136</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BB3crntlyrug%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">30502490</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.11.015</pub-id></mixed-citation></ref><ref id=\"CR118\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahajan</surname><given-names>HS</given-names></name><name><surname>Patil</surname><given-names>PH</given-names></name></person-group><article-title xml:lang=\"en\">Central composite design-based optimization of lopinavir vitamin E-TPGS micelle: In vitro characterization and in vivo pharmacokinetic study</article-title><source>Colloids Surf, B</source><year>2020</year><volume>194</volume><fpage>111149</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXht1Kis7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2020.111149</pub-id></mixed-citation></ref><ref id=\"CR119\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maji</surname><given-names>S</given-names></name><name><surname>Biswas</surname><given-names>A</given-names></name><name><surname>Mandal</surname><given-names>S</given-names></name><name><surname>Sen</surname><given-names>DJ</given-names></name><name><surname>Mahanti</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Silver and copper nano particles in mask for Corona Virus-19 protection</article-title><source>Glob J Endrocrinol Metab</source><year>2020</year><volume>3</volume><fpage>000556</fpage></mixed-citation></ref><ref id=\"CR02\"><mixed-citation publication-type=\"other\">Maniyar MG, Kokare CR (2019) Formulation and evaluation of spray-dried liposomes of lopinavir for topical application. J Pharm Investig 49:259–270.</mixed-citation></ref><ref id=\"CR120\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Masters</surname><given-names>PS</given-names></name></person-group><article-title xml:lang=\"en\">The molecular biology of coronaviruses</article-title><source>Adv Virus Res</source><year>2006</year><volume>66</volume><fpage>193</fpage><lpage>292</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhslOhtLw%3D</pub-id><pub-id pub-id-type=\"pmid\">16877062</pub-id><pub-id pub-id-type=\"pmcid\">7112330</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0065-3527(06)66005-3</pub-id></mixed-citation></ref><ref id=\"CR121\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Matsuura</surname><given-names>K</given-names></name><name><surname>Watanabe</surname><given-names>K</given-names></name><name><surname>Matsuzaki</surname><given-names>T</given-names></name><name><surname>Sakurai</surname><given-names>K</given-names></name><name><surname>Kimizuka</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Self-assembled synthetic viral capsids from a 24-mer viral peptide fragment</article-title><source>Angew Chem Int Ed</source><year>2010</year><volume>49</volume><issue>50</issue><fpage>9662</fpage><lpage>9665</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXhsFaqs7jF</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201004606</pub-id></mixed-citation></ref><ref id=\"CR122\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Miripour</surname><given-names>ZS</given-names></name><name><surname>Sarrami-Forooshani</surname><given-names>R</given-names></name><name><surname>Sanati</surname><given-names>H</given-names></name><name><surname>Makarem</surname><given-names>J</given-names></name><name><surname>Taheri</surname><given-names>MS</given-names></name><name><surname>Shojaeian</surname><given-names>F</given-names></name><name><surname>Eskafi</surname><given-names>AH</given-names></name><name><surname>Abbasvandi</surname><given-names>F</given-names></name><name><surname>Namdar</surname><given-names>N</given-names></name><name><surname>Ghafari</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Real-time diagnosis of reactive oxygen species (ROS) in fresh sputum by electrochemical tracing; correlation between COVID-19 and viral-induced ROS in lung/respiratory epithelium during this pandemic</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><fpage>112435</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlyjtL%2FI</pub-id><pub-id pub-id-type=\"pmid\">32729548</pub-id><pub-id pub-id-type=\"pmcid\">7341050</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112435</pub-id></mixed-citation></ref><ref id=\"CR123\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Misra</surname><given-names>N</given-names></name><name><surname>Bhatt</surname><given-names>S</given-names></name><name><surname>Arefi-Khonsari</surname><given-names>F</given-names></name><name><surname>Kumar</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">State of the art in nonthermal plasma processing for biomedical applications: Can it help fight viral pandemics like COVID-19?</article-title><source>Plasma Process Polym</source><year>2021</year><volume>18</volume><issue>7</issue><fpage>2000215</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFKktbnF</pub-id><pub-id pub-id-type=\"pmid\">34220401</pub-id><pub-id pub-id-type=\"pmcid\">8237024</pub-id><pub-id pub-id-type=\"doi\">10.1002/ppap.202000215</pub-id></mixed-citation></ref><ref id=\"CR124\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Moabelo</surname><given-names>KL</given-names></name><name><surname>Martin</surname><given-names>DR</given-names></name><name><surname>Fadaka</surname><given-names>AO</given-names></name><name><surname>Sibuyi</surname><given-names>NR</given-names></name><name><surname>Meyer</surname><given-names>M</given-names></name><name><surname>Madiehe</surname><given-names>AM</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based strategies for effective and rapid detection of SARS-CoV-2</article-title><source>Materials</source><year>2021</year><volume>14</volume><issue>24</issue><fpage>7851</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXivVaqs7jJ</pub-id><pub-id pub-id-type=\"pmid\">34947447</pub-id><pub-id pub-id-type=\"pmcid\">8703409</pub-id><pub-id pub-id-type=\"doi\">10.3390/ma14247851</pub-id></mixed-citation></ref><ref id=\"CR125\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mohammadniaei</surname><given-names>M</given-names></name><name><surname>Zhang</surname><given-names>M</given-names></name><name><surname>Ashley</surname><given-names>J</given-names></name><name><surname>Christensen</surname><given-names>UB</given-names></name><name><surname>Friis-Hansen</surname><given-names>LJ</given-names></name><name><surname>Gregersen</surname><given-names>R</given-names></name><name><surname>Lisby</surname><given-names>JG</given-names></name><name><surname>Benfield</surname><given-names>TL</given-names></name><name><surname>Nielsen</surname><given-names>FE</given-names></name><name><surname>Henning Rasmussen</surname><given-names>J</given-names></name><name><surname>Pedersen</surname><given-names>EB</given-names></name></person-group><article-title xml:lang=\"en\">A non-enzymatic, isothermal strand displacement and amplification assay for rapid detection of SARS-CoV-2 RNA</article-title><source>Nat Commun</source><year>2021</year><volume>12</volume><issue>1</issue><fpage>1</fpage><lpage>2</lpage><pub-id pub-id-type=\"doi\">10.1038/s41467-021-25387-9</pub-id></mixed-citation></ref><ref id=\"CR126\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Monto</surname><given-names>AS</given-names></name><name><surname>Fendrick</surname><given-names>AM</given-names></name><name><surname>Sarnes</surname><given-names>MW</given-names></name></person-group><article-title xml:lang=\"en\">Respiratory illness caused by picornavirus infection: a review of clinical outcomes</article-title><source>Clin Ther</source><year>2001</year><volume>23</volume><issue>10</issue><fpage>1615</fpage><lpage>1627</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BD3MnnvFCjsQ%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">11726001</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0149-2918(01)80133-8</pub-id></mixed-citation></ref><ref id=\"CR127\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morokutti-Kurz</surname><given-names>M</given-names></name><name><surname>Graf</surname><given-names>C</given-names></name><name><surname>Prieschl-Grassauer</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Amylmetacresol/2, 4-dichlorobenzyl alcohol, hexylresorcinol, or carrageenan lozenges as active treatments for sore throat</article-title><source>Int J General Med</source><year>2017</year><volume>10</volume><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFamtL7E</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJGM.S120665</pub-id></mixed-citation></ref><ref id=\"CR128\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Munster</surname><given-names>VJ</given-names></name><name><surname>Koopmans</surname><given-names>M</given-names></name><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>van Riel</surname><given-names>D</given-names></name><name><surname>de Wit</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus emerging in China—key questions for impact assessment</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>8</issue><fpage>692</fpage><lpage>694</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjslGltrk%3D</pub-id><pub-id pub-id-type=\"pmid\">31978293</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMp2000929</pub-id></mixed-citation></ref><ref id=\"CR129\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Murtaza</surname><given-names>A</given-names></name><name><surname>Rehman</surname><given-names>T</given-names></name><name><surname>Zafar</surname><given-names>Z</given-names></name><name><surname>Noreen</surname><given-names>S</given-names></name><name><surname>Iqbal</surname><given-names>M</given-names></name><name><surname>Hassan</surname><given-names>S</given-names></name><name><surname>Khawaish</surname><given-names>I</given-names></name><name><surname>Tarar</surname><given-names>MHA</given-names></name></person-group><article-title xml:lang=\"en\">Synthesis and characterization of herbal nano-suspensions and evaluation of their In-vivo antihypertensive potential with especial focus on piperine</article-title><source>ACE</source><year>2020</year><volume>1</volume><fpage>23</fpage></mixed-citation></ref><ref id=\"CR130\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muthuirulappan</surname><given-names>S</given-names></name><name><surname>Francis</surname><given-names>SP</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer mechanism and possibility of nano-suspension formulation for a marine algae product fucoxanthin</article-title><source>Asian Pac J Cancer Prev</source><year>2013</year><volume>14</volume><issue>4</issue><fpage>2213</fpage><lpage>2216</lpage><pub-id pub-id-type=\"pmid\">23725114</pub-id><pub-id pub-id-type=\"doi\">10.7314/APJCP.2013.14.4.2213</pub-id></mixed-citation></ref><ref id=\"CR131\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Naseema</surname><given-names>A</given-names></name><name><surname>Kovooru</surname><given-names>L</given-names></name><name><surname>Behera</surname><given-names>AK</given-names></name><name><surname>Kumar</surname><given-names>KP</given-names></name><name><surname>Srivastava</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">A critical review of synthesis procedures, applications and future potential of nanoemulsions</article-title><source>Adv Coll Interface Sci</source><year>2021</year><volume>287</volume><fpage>102318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisVSmtrvN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2020.102318</pub-id></mixed-citation></ref><ref id=\"CR132\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndwandwe</surname><given-names>D</given-names></name><name><surname>Wiysonge</surname><given-names>CS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines</article-title><source>Curr Opin Immunol</source><year>2021</year><volume>71</volume><fpage>111</fpage><lpage>116</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsF2gtrrK</pub-id><pub-id pub-id-type=\"pmid\">34330017</pub-id><pub-id pub-id-type=\"pmcid\">8272971</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.coi.2021.07.003</pub-id></mixed-citation></ref><ref id=\"CR133\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Negi</surname><given-names>JS</given-names></name><name><surname>Chattopadhyay</surname><given-names>P</given-names></name><name><surname>Sharma</surname><given-names>AK</given-names></name><name><surname>Ram</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Development and evaluation of glyceryl behenate based solid lipid nanoparticles (SLNs) using hot self-nanoemulsification (SNE) technique</article-title><source>Arch Pharmacal Res</source><year>2014</year><volume>37</volume><issue>3</issue><fpage>361</fpage><lpage>370</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXot1Kns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12272-013-0154-y</pub-id></mixed-citation></ref><ref id=\"CR134\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nguyen-Contant</surname><given-names>P</given-names></name><name><surname>Embong</surname><given-names>AK</given-names></name><name><surname>Kanagaiah</surname><given-names>P</given-names></name><name><surname>Chaves</surname><given-names>FA</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Branche</surname><given-names>AR</given-names></name><name><surname>Topham</surname><given-names>DJ</given-names></name><name><surname>Sangster</surname><given-names>MY</given-names></name></person-group><article-title xml:lang=\"en\">S protein-reactive IgG and memory B cell production after human SARS-CoV-2 infection includes broad reactivity to the S2 subunit</article-title><source>Mbio</source><year>2020</year><volume>11</volume><issue>5</issue><fpage>e01991</fpage><lpage>e2020</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitl2kt7g%3D</pub-id><pub-id pub-id-type=\"pmid\">32978311</pub-id><pub-id pub-id-type=\"pmcid\">7520599</pub-id><pub-id pub-id-type=\"doi\">10.1128/mBio.01991-20</pub-id></mixed-citation></ref><ref id=\"CR135\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nikaeen</surname><given-names>G</given-names></name><name><surname>Abbaszadeh</surname><given-names>S</given-names></name><name><surname>Yousefinejad</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanomaterials in treatment, anti-infection and detection of coronaviruses</article-title><source>Nanomedicine</source><year>2020</year><volume>15</volume><issue>15</issue><fpage>1501</fpage><lpage>1512</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlWqtrvF</pub-id><pub-id pub-id-type=\"pmid\">32378459</pub-id><pub-id pub-id-type=\"pmcid\">7373208</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2020-0117</pub-id></mixed-citation></ref><ref id=\"CR136\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noriega-Luna</surname><given-names>B</given-names></name><name><surname>Godínez</surname><given-names>LA</given-names></name><name><surname>Rodríguez</surname><given-names>FJ</given-names></name><name><surname>Rodríguez</surname><given-names>A</given-names></name><name><surname>Larrea</surname><given-names>GZLD</given-names></name><name><surname>Sosa-Ferreyra</surname><given-names>CF</given-names></name><name><surname>Mercado-Curiel</surname><given-names>RF</given-names></name><name><surname>Manríquez</surname><given-names>J</given-names></name><name><surname>Bustos</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Applications of dendrimers in drug delivery agents, diagnosis, therapy, and detection</article-title><source>J Nanomater</source><year>2014</year><volume>2014</volume><fpage>39</fpage><lpage>39</lpage><pub-id pub-id-type=\"doi\">10.1155/2014/507273</pub-id></mixed-citation></ref><ref id=\"CR137\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noyes</surname><given-names>AA</given-names></name><name><surname>Whitney</surname><given-names>WR</given-names></name></person-group><article-title xml:lang=\"en\">The rate of solution of solid substances in their own solutions</article-title><source>J Am Chem Soc</source><year>1897</year><volume>19</volume><issue>12</issue><fpage>930</fpage><lpage>934</lpage><pub-id pub-id-type=\"doi\">10.1021/ja02086a003</pub-id></mixed-citation></ref><ref id=\"CR138\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olvera</surname><given-names>D</given-names></name><name><surname>Monaghan</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Electroactive material-based biosensors for detection and drug delivery</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>170</volume><fpage>396</fpage><lpage>424</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvF2jsLnE</pub-id><pub-id pub-id-type=\"pmid\">32987096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2020.09.011</pub-id></mixed-citation></ref><ref id=\"CR139\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>G</given-names></name><name><surname>Fan</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>Y</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Long</surname><given-names>X</given-names></name><name><surname>Guo</surname><given-names>S</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Hu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Serological immunochromatographic approach in diagnosis with SARS-CoV-2 infected COVID-19 patients</article-title><source>J Infect</source><year>2020</year><volume>81</volume><issue>1</issue><fpage>e28</fpage><lpage>32</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCjtb7P</pub-id><pub-id pub-id-type=\"pmid\">32283141</pub-id><pub-id pub-id-type=\"pmcid\">7195339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jinf.2020.03.051</pub-id></mixed-citation></ref><ref id=\"CR140\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Park</surname><given-names>GS</given-names></name><name><surname>Ku</surname><given-names>K</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SJ</given-names></name><name><surname>Kim</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>BT</given-names></name><name><surname>Maeng</surname><given-names>JS</given-names></name></person-group><article-title xml:lang=\"en\">Development of reverse transcription loop-mediated isothermal amplification assays targeting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)</article-title><source>J Mol Diagn</source><year>2020</year><volume>22</volume><issue>6</issue><fpage>729</fpage><lpage>735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXns1Wrsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">32276051</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jmoldx.2020.03.006</pub-id></mixed-citation></ref><ref id=\"CR03\"><mixed-citation publication-type=\"other\">Patel G, Shelat P, Lalwani A (2016) Statistical modeling, optimization and characterization of solid self-nanoemulsifying drug delivery system of lopinavir using design of experiment. Drug delivery 23:3027–3042</mixed-citation></ref><ref id=\"CR141\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>GM</given-names></name><name><surname>Shelat</surname><given-names>PK</given-names></name><name><surname>Lalwani</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">QbD based development of proliposome of lopinavir for improved oral bioavailability</article-title><source>Eur J Pharm Sci</source><year>2017</year><volume>108</volume><fpage>50</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsVOqu7rO</pub-id><pub-id pub-id-type=\"pmid\">27586019</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2016.08.057</pub-id></mixed-citation></ref><ref id=\"CR142\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patone</surname><given-names>M</given-names></name><name><surname>Mei</surname><given-names>XW</given-names></name><name><surname>Handunnetthi</surname><given-names>L</given-names></name><name><surname>Dixon</surname><given-names>S</given-names></name><name><surname>Zaccardi</surname><given-names>F</given-names></name><name><surname>Shankar-Hari</surname><given-names>M</given-names></name><name><surname>Watkinson</surname><given-names>P</given-names></name><name><surname>Khunti</surname><given-names>K</given-names></name><name><surname>Harnden</surname><given-names>A</given-names></name><name><surname>Coupland</surname><given-names>CAC</given-names></name><name><surname>Channon</surname><given-names>KM</given-names></name><name><surname>Mills</surname><given-names>NL</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name><name><surname>Hippisley-Cox</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Risks of myocarditis, pericarditis, and cardiac arrhythmias associated with COVID-19 vaccination or SARS-CoV-2 infection</article-title><source>Nat Med</source><year>2022</year><volume>28</volume><issue>2</issue><fpage>410</fpage><lpage>422</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislCjsbnJ</pub-id><pub-id pub-id-type=\"pmid\">34907393</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41591-021-01630-0</pub-id></mixed-citation></ref><ref id=\"CR143\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Fraceto</surname><given-names>LF</given-names></name><name><surname>Campos</surname><given-names>EVR</given-names></name><name><surname>Rodriguez-Torres</surname><given-names>MdP</given-names></name><name><surname>Acosta-Torres</surname><given-names>LS</given-names></name><name><surname>Diaz-Torres</surname><given-names>LA</given-names></name><name><surname>Grillo</surname><given-names>R</given-names></name><name><surname>Swamy</surname><given-names>MK</given-names></name><name><surname>Sharma</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nano based drug delivery systems: recent developments and future prospects</article-title><source>J Nanobiotechnol</source><year>2018</year><volume>16</volume><issue>1</issue><fpage>1</fpage><lpage>33</lpage><pub-id pub-id-type=\"doi\">10.1186/s12951-018-0392-8</pub-id></mixed-citation></ref><ref id=\"CR144\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Ansari</surname><given-names>A</given-names></name><name><surname>Kim</surname><given-names>H</given-names></name><name><surname>Shin</surname><given-names>HS</given-names></name></person-group><article-title xml:lang=\"en\">Photo-mediated biosynthesis of silver nanoparticles using the non-edible accrescent fruiting calyx of physalis peruviana L. fruits and investigation of its radical scavenging potential and cytotoxicity activities</article-title><source>J Photochem Photobiol, B</source><year>2018</year><volume>188</volume><fpage>116</fpage><lpage>125</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslyqs7vO</pub-id><pub-id pub-id-type=\"pmid\">30266015</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jphotobiol.2018.08.004</pub-id></mixed-citation></ref><ref id=\"CR145\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>H</given-names></name><name><surname>Lt</surname><given-names>Y</given-names></name><name><surname>Ly</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name><name><surname>Zq</surname><given-names>Lu</given-names></name><name><surname>Koup</surname><given-names>RA</given-names></name><name><surname>Bailer</surname><given-names>RT</given-names></name><name><surname>Cy</surname><given-names>Wu</given-names></name></person-group><article-title xml:lang=\"en\">Long-lived memory T lymphocyte responses against SARS coronavirus nucleocapsid protein in SARS-recovered patients</article-title><source>Virology</source><year>2006</year><volume>351</volume><issue>2</issue><fpage>466</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28Xns1Gks7c%3D</pub-id><pub-id pub-id-type=\"pmid\">16690096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.virol.2006.03.036</pub-id></mixed-citation></ref><ref id=\"CR146\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Phan</surname><given-names>IQ</given-names></name><name><surname>Subramanian</surname><given-names>S</given-names></name><name><surname>Kim</surname><given-names>D</given-names></name><name><surname>Murphy</surname><given-names>M</given-names></name><name><surname>Pettie</surname><given-names>D</given-names></name><name><surname>Carter</surname><given-names>L</given-names></name><name><surname>Anishchenko</surname><given-names>I</given-names></name><name><surname>Barrett</surname><given-names>LK</given-names></name><name><surname>Craig</surname><given-names>J</given-names></name><name><surname>Tillery</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">In silico detection of SARS-CoV-2 specific B-cell epitopes and validation in ELISA for serological diagnosis of COVID-19</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>11</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-83730-y</pub-id></mixed-citation></ref><ref id=\"CR147\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pimentel</surname><given-names>TA</given-names></name><name><surname>Yan</surname><given-names>Z</given-names></name><name><surname>Jeffers</surname><given-names>SA</given-names></name><name><surname>Holmes</surname><given-names>KV</given-names></name><name><surname>Hodges</surname><given-names>RS</given-names></name><name><surname>Burkhard</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Peptide nanoparticles as novel immunogens: design and analysis of a prototypic severe acute respiratory syndrome vaccine</article-title><source>Chem Biol Drug Des</source><year>2009</year><volume>73</volume><issue>1</issue><fpage>53</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptFCnsw%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19152635</pub-id><pub-id pub-id-type=\"doi\">10.1111/j.1747-0285.2008.00746.x</pub-id></mixed-citation></ref><ref id=\"CR148\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasad</surname><given-names>R</given-names></name><name><surname>Pandey</surname><given-names>R</given-names></name><name><surname>Barman</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Engineering tailored nanoparticles with microbes: quo vadis</article-title><source>Wires Nanomed Nanobiotechnol</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1002/wnan.1363</pub-id></mixed-citation></ref><ref id=\"CR224\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prusty</surname><given-names>K</given-names></name><name><surname>Swain</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Nano silver decorated polyacrylamide/dextran nanohydrogels hybrid composites for drug delivery applications</article-title><source>Mater Sci Eng, C</source><year>2018</year><volume>85</volume><fpage>130</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitVejtLfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.msec.2017.11.028</pub-id></mixed-citation></ref><ref id=\"CR149\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Puga-Gómez</surname><given-names>R</given-names></name><name><surname>Ricardo-Delgado</surname><given-names>Y</given-names></name><name><surname>Rojas-Iriarte</surname><given-names>C</given-names></name><name><surname>Céspedes-Henriquez</surname><given-names>L</given-names></name><name><surname>Piedra-Bello</surname><given-names>M</given-names></name><name><surname>Vega-Mendoza</surname><given-names>D</given-names></name><name><surname>Pérez</surname><given-names>NP</given-names></name><name><surname>Paredes-Moreno</surname><given-names>B</given-names></name><name><surname>Rodríguez-González</surname><given-names>M</given-names></name><name><surname>Valenzuela-Silva</surname><given-names>C</given-names></name><name><surname>Sánchez-Ramírez</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Open-label phase I/II clinical trial of SARS-CoV-2 receptor binding domain-tetanus toxoid conjugate vaccine (FINLAY-FR-2) in combination with receptor binding domain-protein vaccine (FINLAY-FR-1A) in children</article-title><source>Int J Infect Dis</source><year>2023</year><volume>126</volume><fpage>164</fpage><lpage>173</lpage><pub-id pub-id-type=\"pmid\">36403819</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2022.11.016</pub-id></mixed-citation></ref><ref id=\"CR150\"><mixed-citation publication-type=\"other\">Puri A, Loomis K, Smith B, Lee JH, Yavlovich A, Heldman E, Blumenthal, R (2009) Lipid-based nanoparticles as pharmaceutical drug carriers: from concepts to clinic. Crit Rev™ Ther Drug Carr Syst 26(6)</mixed-citation></ref><ref id=\"CR151\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Que</surname><given-names>H</given-names></name><name><surname>Hong</surname><given-names>W</given-names></name><name><surname>Lan</surname><given-names>T</given-names></name><name><surname>Zeng</surname><given-names>H</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>Wan</surname><given-names>D</given-names></name><name><surname>Bi</surname><given-names>Z</given-names></name><name><surname>Ren</surname><given-names>W</given-names></name><name><surname>Luo</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>He</surname><given-names>C</given-names></name><name><surname>Zhong</surname><given-names>A</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Tripterin liposome relieves severe acute respiratory syndrome as a potent COVID-19 treatment</article-title><source>Signal Transduct Target Ther</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>399</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjtF2rsrzL</pub-id><pub-id pub-id-type=\"pmid\">36566328</pub-id><pub-id pub-id-type=\"pmcid\">9789731</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41392-022-01283-6</pub-id></mixed-citation></ref><ref id=\"CR152\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quick</surname><given-names>J</given-names></name><name><surname>Grubaugh</surname><given-names>ND</given-names></name><name><surname>Pullan</surname><given-names>ST</given-names></name><name><surname>Claro</surname><given-names>IM</given-names></name><name><surname>Smith</surname><given-names>AD</given-names></name><name><surname>Gangavarapu</surname><given-names>K</given-names></name><name><surname>Oliveira</surname><given-names>G</given-names></name><name><surname>Robles-Sikisaka</surname><given-names>R</given-names></name><name><surname>Rogers</surname><given-names>TF</given-names></name><name><surname>Beutler</surname><given-names>NA</given-names></name><name><surname>Burton</surname><given-names>DR</given-names></name></person-group><article-title xml:lang=\"en\">Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples</article-title><source>Nat Protoc</source><year>2017</year><volume>12</volume><issue>6</issue><fpage>1261</fpage><lpage>1276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXosVahu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">28538739</pub-id><pub-id pub-id-type=\"pmcid\">5902022</pub-id><pub-id pub-id-type=\"doi\">10.1038/nprot.2017.066</pub-id></mixed-citation></ref><ref id=\"CR153\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rabiee</surname><given-names>N</given-names></name><name><surname>Bagherzadeh</surname><given-names>M</given-names></name><name><surname>Ghasemi</surname><given-names>A</given-names></name><name><surname>Zare</surname><given-names>H</given-names></name><name><surname>Ahmadi</surname><given-names>S</given-names></name><name><surname>Fatahi</surname><given-names>Y</given-names></name><name><surname>Dinarvand</surname><given-names>R</given-names></name><name><surname>Rabiee</surname><given-names>M</given-names></name><name><surname>Ramakrishna</surname><given-names>S</given-names></name><name><surname>Shokouhimehr</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Point-of-use rapid detection of SARS-CoV-2: nanotechnology-enabled solutions for the COVID-19 pandemic</article-title><source>Int J Mol Sci</source><year>2020</year><volume>21</volume><issue>14</issue><fpage>5126</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVChtLfN</pub-id><pub-id pub-id-type=\"pmid\">32698479</pub-id><pub-id pub-id-type=\"pmcid\">7404277</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms21145126</pub-id></mixed-citation></ref><ref id=\"CR155\"><mixed-citation publication-type=\"other\">Raj S, Khurana S, Choudhari R, Kesari KK, Kamal MA, Garg N, Ruokolainen J, Das BC, Kumar D (2021) In Specific targeting cancer cells with nanoparticles and drug delivery in cancer therapy, Seminars in cancer biology. Elsevier pp 166–177</mixed-citation></ref><ref id=\"CR156\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ramachandran</surname><given-names>AK</given-names></name><name><surname>Das</surname><given-names>S</given-names></name><name><surname>Joseph</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Crosstalk between COVID-19 and associated neurological disorders: a review</article-title><source>Curr Neuropharmacol</source><year>2021</year><volume>19</volume><issue>10</issue><fpage>1688</fpage><lpage>1700</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlyksrnO</pub-id><pub-id pub-id-type=\"pmid\">33441073</pub-id><pub-id pub-id-type=\"doi\">10.2174/1570159X19666210113154342</pub-id></mixed-citation></ref><ref id=\"CR157\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>L</given-names></name><name><surname>Xia</surname><given-names>S</given-names></name><name><surname>Xu</surname><given-names>W</given-names></name><name><surname>Tian</surname><given-names>R</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name><name><surname>Gu</surname><given-names>C</given-names></name><name><surname>Pan</surname><given-names>P</given-names></name><name><surname>Meng</surname><given-names>QF</given-names></name><name><surname>Ca</surname><given-names>X</given-names></name><name><surname>Qu</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Decoy nanoparticles protect against COVID-19 by concurrently adsorbing viruses and inflammatory cytokines</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>44</issue><fpage>27141</fpage><lpage>27147</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Ghu73J</pub-id><pub-id pub-id-type=\"pmid\">33024017</pub-id><pub-id pub-id-type=\"pmcid\">7959535</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.2014352117</pub-id></mixed-citation></ref><ref id=\"CR158\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>PV</given-names></name><name><surname>Krishna</surname><given-names>BS</given-names></name><name><surname>Jeffree</surname><given-names>MS</given-names></name></person-group><source>Coronaviruses: transmission, frontliners, nanotechnology and economy</source><year>2022</year><publisher-loc> Kota Kinabalu</publisher-loc><publisher-name>Universiti Malaysia Sabah Press</publisher-name></mixed-citation></ref><ref id=\"CR159\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Murthy</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">A hybrid design to optimize preparation of lopinavir loaded solid lipid nanoparticles and comparative pharmacokinetic evaluation with marketed lopinavir/ritonavir coformulation</article-title><source>J Pharm Pharmacol</source><year>2014</year><volume>66</volume><issue>7</issue><fpage>912</fpage><lpage>926</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXps1Ohtro%3D</pub-id><pub-id pub-id-type=\"pmid\">24697749</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12217</pub-id></mixed-citation></ref><ref id=\"CR160\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Gadekar</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Design, optimization and evaluation of poly-ɛ-caprolactone (PCL) based polymeric nanoparticles for oral delivery of lopinavir</article-title><source>Drug Dev Ind Pharm</source><year>2015</year><volume>41</volume><issue>1</issue><fpage>131</fpage><lpage>140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVylurrM</pub-id><pub-id pub-id-type=\"pmid\">24180260</pub-id><pub-id pub-id-type=\"doi\">10.3109/03639045.2013.850710</pub-id></mixed-citation></ref><ref id=\"CR161\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>van der Vlies</surname><given-names>A</given-names></name><name><surname>Simeoni</surname><given-names>E</given-names></name><name><surname>Oneil</surname><given-names>C</given-names></name><name><surname>Swartz</surname><given-names>M</given-names></name><name><surname>Hubbell</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Exploiting lymphatic transport and complement activation in nanoparticle vaccines</article-title><source>Tissue Eng Part A</source><year>2008</year><volume>14</volume><fpage>734</fpage><lpage>735</lpage></mixed-citation></ref><ref id=\"CR162\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rizzo</surname><given-names>JM</given-names></name><name><surname>Buck</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">All things ChIP: ChIP-Chip, ChIP-Seq, ChIP-PCR</article-title><source>Epigenet Regul Epigenomics</source><year>2012</year><volume>2</volume><fpage>41</fpage></mixed-citation></ref><ref id=\"CR163\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rocchitta</surname><given-names>G</given-names></name><name><surname>Spanu</surname><given-names>A</given-names></name><name><surname>Babudieri</surname><given-names>S</given-names></name><name><surname>Latte</surname><given-names>G</given-names></name><name><surname>Madeddu</surname><given-names>G</given-names></name><name><surname>Galleri</surname><given-names>G</given-names></name><name><surname>Nuvoli</surname><given-names>S</given-names></name><name><surname>Bagella</surname><given-names>P</given-names></name><name><surname>Demartis</surname><given-names>MI</given-names></name><name><surname>Fiore</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Enzyme biosensors for biomedical applications: strategies for safeguarding analytical performances in biological fluids</article-title><source>Sensors</source><year>2016</year><volume>16</volume><issue>6</issue><fpage>780</fpage><pub-id pub-id-type=\"pmid\">27249001</pub-id><pub-id pub-id-type=\"pmcid\">4934206</pub-id><pub-id pub-id-type=\"doi\">10.3390/s16060780</pub-id></mixed-citation></ref><ref id=\"CR164\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>CD</given-names></name><name><surname>Millar</surname><given-names>JE</given-names></name><name><surname>Baillie</surname><given-names>JK</given-names></name></person-group><article-title xml:lang=\"en\">Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>473</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaksr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30317-2</pub-id></mixed-citation></ref><ref id=\"CR165\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>RL</given-names></name><name><surname>Pelka</surname><given-names>P</given-names></name><name><surname>Mark</surname><given-names>BL</given-names></name></person-group><article-title xml:lang=\"en\">Frontrunners in the race to develop a SARS-CoV-2 vaccine</article-title><source>Can J Microbiol</source><year>2021</year><volume>67</volume><issue>3</issue><fpage>189</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktF2mtbs%3D</pub-id><pub-id pub-id-type=\"pmid\">33264067</pub-id><pub-id pub-id-type=\"doi\">10.1139/cjm-2020-0465</pub-id></mixed-citation></ref><ref id=\"CR166\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sahakijpijarn</surname><given-names>S</given-names></name><name><surname>Moon</surname><given-names>C</given-names></name><name><surname>Koleng</surname><given-names>JJ</given-names></name><name><surname>Christensen</surname><given-names>DJ</given-names></name><name><surname>Williams</surname><given-names>RO</given-names></name></person-group><article-title xml:lang=\"en\">Development of remdesivir as a dry powder for inhalation by thin film freezing</article-title><source>Pharmaceutics</source><year>2020</year><volume>12</volume><issue>11</issue><fpage>1002</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlsFCktr4%3D</pub-id><pub-id pub-id-type=\"pmid\">33105618</pub-id><pub-id pub-id-type=\"pmcid\">7690377</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics12111002</pub-id></mixed-citation></ref><ref id=\"CR167\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Salleh</surname><given-names>A</given-names></name><name><surname>Naomi</surname><given-names>R</given-names></name><name><surname>Utami</surname><given-names>ND</given-names></name><name><surname>Mohammad</surname><given-names>AW</given-names></name><name><surname>Mahmoudi</surname><given-names>E</given-names></name><name><surname>Mustafa</surname><given-names>N</given-names></name><name><surname>Fauzi</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">The potential of silver nanoparticles for antiviral and antibacterial applications: a mechanism of action</article-title><source>Nanomaterials</source><year>2020</year><volume>10</volume><issue>8</issue><fpage>1566</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsl2gu7bL</pub-id><pub-id pub-id-type=\"pmid\">32784939</pub-id><pub-id pub-id-type=\"pmcid\">7466543</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano10081566</pub-id></mixed-citation></ref><ref id=\"CR168\"><mixed-citation publication-type=\"other\">Sen AK, Nath A, Sudeepthi A, Jain SK, Banerjee U (2022) Microfluidics-Based Point-of-Care Diagnostic Devices. In: Advanced microfluidics-based point-of-care diagnostics. CRC Press pp. 99–120</mixed-citation></ref><ref id=\"CR169\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Seo</surname><given-names>G</given-names></name><name><surname>Lee</surname><given-names>G</given-names></name><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Choi</surname><given-names>M</given-names></name><name><surname>Ku</surname><given-names>KB</given-names></name><name><surname>Lee</surname><given-names>CS</given-names></name><name><surname>Jun</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>D</given-names></name><name><surname>Kim</surname><given-names>HG</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of COVID-19 causative virus (SARS-CoV-2) in human nasopharyngeal swab specimens using field-effect transistor-based biosensor</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>5135</fpage><lpage>5142</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnt1SrtrY%3D</pub-id><pub-id pub-id-type=\"pmid\">32293168</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02823</pub-id></mixed-citation></ref><ref id=\"CR01\"><mixed-citation publication-type=\"other\">Sercombe L, Veerati T, Moheimani F, Wu SY, Sood AK, Hua S (2015)\n",
       "Advances and challenges of liposome assisted drug delivery. Front Pharmacol. 6:286</mixed-citation></ref><ref id=\"CR170\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shafiee</surname><given-names>S</given-names></name><name><surname>Cegolon</surname><given-names>L</given-names></name><name><surname>Khafaei</surname><given-names>M</given-names></name><name><surname>Gholami</surname><given-names>N</given-names></name><name><surname>Zhao</surname><given-names>S</given-names></name><name><surname>Khalesi</surname><given-names>N</given-names></name><name><surname>Moosavian</surname><given-names>H</given-names></name><name><surname>Fathi</surname><given-names>S</given-names></name><name><surname>Izadi</surname><given-names>M</given-names></name><name><surname>Ghadian</surname><given-names>A</given-names></name><name><surname>Javanbakht</surname><given-names>M</given-names></name><name><surname>Javanbakht</surname><given-names>A</given-names></name><name><surname>Akhavan-Sigari</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Gastrointestinal cancers, ACE-2/TMPRSS2 expression and susceptibility to COVID-19</article-title><source>Cancer Cell Int</source><year>2021</year><volume>21</volume><issue>1</issue><fpage>431</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVemurzO</pub-id><pub-id pub-id-type=\"pmid\">34399734</pub-id><pub-id pub-id-type=\"pmcid\">8365127</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12935-021-02129-x</pub-id></mixed-citation></ref><ref id=\"CR171\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>V</given-names></name><name><surname>Keniya</surname><given-names>R</given-names></name><name><surname>Shridharani</surname><given-names>A</given-names></name><name><surname>Punjabi</surname><given-names>M</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name><name><surname>Mehendale</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 using CT scan images and deep learning techniques</article-title><source>Emerg Radiol</source><year>2021</year><volume>28</volume><issue>3</issue><fpage>497</fpage><lpage>505</lpage><pub-id pub-id-type=\"pmid\">33523309</pub-id><pub-id pub-id-type=\"pmcid\">7848247</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10140-020-01886-y</pub-id></mixed-citation></ref><ref id=\"CR172\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shan</surname><given-names>B</given-names></name><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Li</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>S</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name><name><surname>Gui</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed nanomaterial-based sensor array for detection of COVID-19 in exhaled breath</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>9</issue><fpage>12125</fpage><lpage>12132</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1aqsLfK</pub-id><pub-id pub-id-type=\"pmid\">32808759</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c05657</pub-id></mixed-citation></ref><ref id=\"CR173\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shanley</surname><given-names>LC</given-names></name><name><surname>Mahon</surname><given-names>OR</given-names></name><name><surname>Kelly</surname><given-names>DJ</given-names></name><name><surname>Dunne</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Harnessing the innate and adaptive immune system for tissue repair and regeneration: Considering more than macrophages</article-title><source>Acta Biomater</source><year>2021</year><volume>133</volume><fpage>208</fpage><lpage>221</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1yns7%2FK</pub-id><pub-id pub-id-type=\"pmid\">33657453</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.actbio.2021.02.023</pub-id></mixed-citation></ref><ref id=\"CR174\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Sharma</surname><given-names>N</given-names></name><name><surname>Kumari</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>T</given-names></name><name><surname>Tripathi</surname><given-names>KM</given-names></name></person-group><article-title xml:lang=\"en\">Nano-carbon based sensors for bacterial detection and discrimination in clinical diagnosis: a junction between material science and biology</article-title><source>Appl Mater Today</source><year>2020</year><volume>18</volume><pub-id pub-id-type=\"doi\">10.1016/j.apmt.2019.100467</pub-id></mixed-citation></ref><ref id=\"CR175\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Pramanik</surname><given-names>A</given-names></name><name><surname>Tchounwou</surname><given-names>C</given-names></name><name><surname>Pedraza</surname><given-names>F</given-names></name><name><surname>Crouch</surname><given-names>RA</given-names></name><name><surname>Chavva</surname><given-names>SR</given-names></name><name><surname>Vangara</surname><given-names>A</given-names></name><name><surname>Sinha</surname><given-names>SS</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Sardar</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Multifunctional biocompatible graphene oxide quantum dots decorated magnetic nanoplatform for efficient capture and two-photon imaging of rare tumor cells</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>20</issue><fpage>10935</fpage><lpage>10943</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnslCqu7o%3D</pub-id><pub-id pub-id-type=\"pmid\">25939643</pub-id><pub-id pub-id-type=\"pmcid\">4570252</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b02199</pub-id></mixed-citation></ref><ref id=\"CR176\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Singh</surname><given-names>B</given-names></name><name><surname>Datta</surname><given-names>B</given-names></name><name><surname>Ashish</surname><given-names>A</given-names></name><name><surname>Dutta</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">A comprehensive review on current COVID-19 detection methods: From lab care to point of care diagnosis</article-title><source>Sens Int</source><year>2021</year><volume>2</volume><pub-id pub-id-type=\"pmid\">34766062</pub-id><pub-id pub-id-type=\"pmcid\">8302821</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.sintl.2021.100119</pub-id></mixed-citation></ref><ref id=\"CR177\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Su</surname><given-names>S</given-names></name><name><surname>Wong</surname><given-names>G</given-names></name><name><surname>Shi</surname><given-names>W</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lai</surname><given-names>AC</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name><name><surname>Bi</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</article-title><source>Trends Microbiol</source><year>2016</year><volume>24</volume><issue>6</issue><fpage>490</fpage><lpage>502</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XksVWmsbc%3D</pub-id><pub-id pub-id-type=\"pmid\">27012512</pub-id><pub-id pub-id-type=\"pmcid\">7125511</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tim.2016.03.003</pub-id></mixed-citation></ref><ref id=\"CR178\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>T</given-names></name><name><surname>Zhang</surname><given-names>YS</given-names></name><name><surname>Pang</surname><given-names>B</given-names></name><name><surname>Hyun</surname><given-names>DC</given-names></name><name><surname>Yang</surname><given-names>M</given-names></name><name><surname>Xia</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Engineered nanoparticles for drug delivery in cancer therapy</article-title><source>Angew Chem Int Ed</source><year>2014</year><volume>53</volume><issue>46</issue><fpage>12320</fpage><lpage>12364</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGksLzO</pub-id></mixed-citation></ref><ref id=\"CR179\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Gu</surname><given-names>H</given-names></name><name><surname>Fang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Deng</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>P</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Zhao</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>C</given-names></name><name><surname>Lang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Fei</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Geng</surname><given-names>S</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>Wei</surname><given-names>W</given-names></name><name><surname>Hu</surname><given-names>Z</given-names></name><name><surname>Han</surname><given-names>G</given-names></name><name><surname>Sun</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Recombinant vaccine containing an RBD-Fc fusion induced protection against SARS-CoV-2 in nonhuman primates and mice</article-title><source>Cell Mol Immunol</source><year>2021</year><volume>18</volume><issue>4</issue><fpage>1070</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1OlurY%3D</pub-id><pub-id pub-id-type=\"pmid\">33731916</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41423-021-00658-z</pub-id></mixed-citation></ref><ref id=\"CR180\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Szunerits</surname><given-names>S</given-names></name><name><surname>Barras</surname><given-names>A</given-names></name><name><surname>Khanal</surname><given-names>M</given-names></name><name><surname>Pagneux</surname><given-names>Q</given-names></name><name><surname>Boukherroub</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructures for the inhibition of viral infections</article-title><source>Molecules</source><year>2015</year><volume>20</volume><issue>8</issue><fpage>14051</fpage><lpage>14081</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtlKrtr7F</pub-id><pub-id pub-id-type=\"pmid\">26247927</pub-id><pub-id pub-id-type=\"pmcid\">6332336</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules200814051</pub-id></mixed-citation></ref><ref id=\"CR181\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>F</given-names></name><name><surname>Quan</surname><given-names>Y</given-names></name><name><surname>Xin</surname><given-names>ZT</given-names></name><name><surname>Wrammert</surname><given-names>J</given-names></name><name><surname>Ma</surname><given-names>MJ</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>TB</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Richardus</surname><given-names>JH</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Lack of peripheral memory B cell responses in recovered patients with severe acute respiratory syndrome: a six-year follow-up study</article-title><source>J Immunol</source><year>2011</year><volume>186</volume><issue>12</issue><fpage>7264</fpage><lpage>7268</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXntVGmsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">21576510</pub-id><pub-id pub-id-type=\"doi\">10.4049/jimmunol.0903490</pub-id></mixed-citation></ref><ref id=\"CR182\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tarim</surname><given-names>EA</given-names></name><name><surname>Karakuzu</surname><given-names>B</given-names></name><name><surname>Oksuz</surname><given-names>C</given-names></name><name><surname>Sarigil</surname><given-names>O</given-names></name><name><surname>Kizilkaya</surname><given-names>M</given-names></name><name><surname>Al-Ruweidi</surname><given-names>MK</given-names></name><name><surname>Yalcin</surname><given-names>HC</given-names></name><name><surname>Ozcivici</surname><given-names>E</given-names></name><name><surname>Tekin</surname><given-names>HC</given-names></name></person-group><article-title xml:lang=\"en\">Microfluidic-based virus detection methods for respiratory diseases</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>143</fpage><lpage>168</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFeisLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00169-7</pub-id></mixed-citation></ref><ref id=\"CR183\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tian</surname><given-names>B</given-names></name><name><surname>Gao</surname><given-names>F</given-names></name><name><surname>Fock</surname><given-names>J</given-names></name><name><surname>Dufva</surname><given-names>M</given-names></name><name><surname>Hansen</surname><given-names>MF</given-names></name></person-group><article-title xml:lang=\"en\">Homogeneous circle-to-circle amplification for real-time optomagnetic detection of SARS-CoV-2 RdRp coding sequence</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFegtbnF</pub-id><pub-id pub-id-type=\"pmid\">32510339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112356</pub-id></mixed-citation></ref><ref id=\"CR184\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ting</surname><given-names>D</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>J</given-names></name><name><surname>Bi</surname><given-names>J</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Multisite inhibitors for enteric coronavirus: antiviral cationic carbon dots based on curcumin</article-title><source>ACS Appl Nano Mater</source><year>2018</year><volume>1</volume><issue>10</issue><fpage>5451</fpage><lpage>5459</lpage><pub-id pub-id-type=\"doi\">10.1021/acsanm.8b00779</pub-id></mixed-citation></ref><ref id=\"CR185\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>To</surname><given-names>KK</given-names></name><name><surname>Yip</surname><given-names>CC</given-names></name><name><surname>Lai</surname><given-names>CY</given-names></name><name><surname>Wong</surname><given-names>CK</given-names></name><name><surname>Ho</surname><given-names>DT</given-names></name><name><surname>Pang</surname><given-names>PK</given-names></name><name><surname>Ng</surname><given-names>AC</given-names></name><name><surname>Leung</surname><given-names>KH</given-names></name><name><surname>Poon</surname><given-names>RW</given-names></name><name><surname>Chan</surname><given-names>KH</given-names></name><name><surname>Cheng</surname><given-names>VC</given-names></name></person-group><article-title xml:lang=\"en\">Saliva as a diagnostic specimen for testing respiratory virus by a point-of-care molecular assay: a diagnostic validity study</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>3</issue><fpage>372</fpage><lpage>378</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1yksbnE</pub-id><pub-id pub-id-type=\"pmid\">29906597</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.06.009</pub-id></mixed-citation></ref><ref id=\"CR186\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tomas</surname><given-names>M</given-names></name><name><surname>Capanoglu</surname><given-names>E</given-names></name><name><surname>Bahrami</surname><given-names>A</given-names></name><name><surname>Hosseini</surname><given-names>H</given-names></name><name><surname>Akbari-Alavijeh</surname><given-names>S</given-names></name><name><surname>Shaddel</surname><given-names>R</given-names></name><name><surname>Rehman</surname><given-names>A</given-names></name><name><surname>Rezaei</surname><given-names>A</given-names></name><name><surname>Rashidinejad</surname><given-names>A</given-names></name><name><surname>Garavand</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">The direct and indirect effects of bioactive compounds against coronavirus</article-title><source>Food Front</source><year>2022</year><volume>3</volume><issue>1</issue><fpage>96</fpage><lpage>123</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitlCgtLc%3D</pub-id><pub-id pub-id-type=\"pmid\">35462942</pub-id><pub-id pub-id-type=\"doi\">10.1002/fft2.119</pub-id></mixed-citation></ref><ref id=\"CR187\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tripathy</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>MK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers and their applications as novel drug delivery carriers</article-title><source>J Appl Pharm Sci</source><year>2013</year><volume>3</volume><issue>09</issue><fpage>142</fpage><lpage>149</lpage></mixed-citation></ref><ref id=\"CR188\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Udugama</surname><given-names>B</given-names></name><name><surname>Kadhiresan</surname><given-names>P</given-names></name><name><surname>Kozlowski</surname><given-names>HN</given-names></name><name><surname>Malekjahani</surname><given-names>A</given-names></name><name><surname>Osborne</surname><given-names>M</given-names></name><name><surname>Li</surname><given-names>VY</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Mubareka</surname><given-names>S</given-names></name><name><surname>Gubbay</surname><given-names>JB</given-names></name><name><surname>Chan</surname><given-names>WC</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosing COVID-19: the disease and tools for detection</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>3822</fpage><lpage>3835</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXlvFChtrk%3D</pub-id><pub-id pub-id-type=\"pmid\">32223179</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02624</pub-id></mixed-citation></ref><ref id=\"CR189\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vaezi</surname><given-names>A</given-names></name><name><surname>Meysamie</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines cost-effectiveness analysis: a scenario for Iran</article-title><source>Vaccines</source><year>2021</year><volume>10</volume><issue>1</issue><fpage>37</fpage><pub-id pub-id-type=\"pmid\">35062698</pub-id><pub-id pub-id-type=\"pmcid\">8777749</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines10010037</pub-id></mixed-citation></ref><ref id=\"CR190\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vahedifard</surname><given-names>F</given-names></name><name><surname>Chakravarthy</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for COVID-19: the role of nanotechnology in the treatment and diagnosis of COVID-19</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>75</fpage><lpage>99</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktlajsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00168-8</pub-id></mixed-citation></ref><ref id=\"CR191\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Varahachalam</surname><given-names>SP</given-names></name><name><surname>Lahooti</surname><given-names>B</given-names></name><name><surname>Chamaneh</surname><given-names>M</given-names></name><name><surname>Bagchi</surname><given-names>S</given-names></name><name><surname>Chhibber</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>K</given-names></name><name><surname>Bolanos</surname><given-names>JF</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Kaushik</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for the SARS-CoV-2: state-of-the-art and future prospects</article-title><source>Int J Nanomed</source><year>2021</year><volume>16</volume><fpage>539</fpage><pub-id pub-id-type=\"doi\">10.2147/IJN.S283686</pub-id></mixed-citation></ref><ref id=\"CR192\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Verma</surname><given-names>SK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Sahoo</surname><given-names>B</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Thirumurugan</surname><given-names>A</given-names></name><name><surname>Parashar</surname><given-names>S</given-names></name><name><surname>Suar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight into the rapid one-step facile biofabrication of antibacterial silver nanoparticles from bacterial release and their biogenicity and concentration-dependent in vitro cytotoxicity to colon cells</article-title><source>RSC Adv</source><year>2017</year><volume>7</volume><issue>64</issue><fpage>40034</fpage><lpage>40045</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVantLrF</pub-id><pub-id pub-id-type=\"doi\">10.1039/C7RA05943D</pub-id></mixed-citation></ref><ref id=\"CR193\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vijayan</surname><given-names>V</given-names></name><name><surname>Mohapatra</surname><given-names>A</given-names></name><name><surname>Uthaman</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>IK</given-names></name></person-group><article-title xml:lang=\"en\">Recent advances in nanovaccines using biomimetic immunomodulatory materials</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>10</issue><fpage>534</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXotFKkt7o%3D</pub-id><pub-id pub-id-type=\"pmid\">31615112</pub-id><pub-id pub-id-type=\"pmcid\">6835828</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11100534</pub-id></mixed-citation></ref><ref id=\"CR194\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Villeret</surname><given-names>B</given-names></name><name><surname>Dieu</surname><given-names>A</given-names></name><name><surname>Straube</surname><given-names>M</given-names></name><name><surname>Solhonne</surname><given-names>B</given-names></name><name><surname>Miklavc</surname><given-names>P</given-names></name><name><surname>Hamadi</surname><given-names>S</given-names></name><name><surname>Le Borgne</surname><given-names>R</given-names></name><name><surname>Mailleux</surname><given-names>A</given-names></name><name><surname>Norel</surname><given-names>X</given-names></name><name><surname>Aerts</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Silver nanoparticles impair retinoic acid-inducible gene I-mediated mitochondrial antiviral immunity by blocking the autophagic flux in lung epithelial cells</article-title><source>ACS Nano</source><year>2018</year><volume>12</volume><issue>2</issue><fpage>1188</fpage><lpage>1202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVart7c%3D</pub-id><pub-id pub-id-type=\"pmid\">29357226</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.7b06934</pub-id></mixed-citation></ref><ref id=\"CR195\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Volkov</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dots in nanomedicine: recent trends, advances and unresolved issues</article-title><source>Biochem Biophys Res Commun</source><year>2015</year><volume>468</volume><issue>3</issue><fpage>419</fpage><lpage>427</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1SqtL3L</pub-id><pub-id pub-id-type=\"pmid\">26168726</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bbrc.2015.07.039</pub-id></mixed-citation></ref><ref id=\"CR196\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Jiao</surname><given-names>G</given-names></name><name><surname>Shan</surname><given-names>X</given-names></name><name><surname>Tai</surname><given-names>W</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Inhibition of influenza A virus infection by fucoidan targeting viral neuraminidase and cellular EGFR pathway</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>14</lpage><pub-id pub-id-type=\"pmid\">28127051</pub-id><pub-id pub-id-type=\"pmcid\">5428335</pub-id></mixed-citation></ref><ref id=\"CR197\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Yunjia</surname><given-names>Y</given-names></name><name><surname>Qin</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>M</given-names></name><name><surname>Mao</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity against enterovirus 71 of sulfated rhamnan isolated from the green alga Monostroma latissimum</article-title><source>Carbohyd Polym</source><year>2018</year><volume>200</volume><fpage>43</fpage><lpage>53</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVSqur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbpol.2018.07.067</pub-id></mixed-citation></ref><ref id=\"CR198\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Horby</surname><given-names>PW</given-names></name><name><surname>Hayden</surname><given-names>FG</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus outbreak of global health concern</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>470</fpage><lpage>473</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30185-9</pub-id></mixed-citation></ref><ref id=\"CR199\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wankar</surname><given-names>JN</given-names></name><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Bohara</surname><given-names>C</given-names></name><name><surname>Singh</surname><given-names>MP</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name></person-group><article-title xml:lang=\"en\">Role of nanomedicine in management and prevention of COVID-19</article-title><source>Front Nanotechnol</source><year>2020</year><volume>2</volume><pub-id pub-id-type=\"doi\">10.3389/fnano.2020.589541</pub-id></mixed-citation></ref><ref id=\"CR200\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Weiss</surname><given-names>C</given-names></name><name><surname>Carriere</surname><given-names>M</given-names></name><name><surname>Fusco</surname><given-names>L</given-names></name><name><surname>Capua</surname><given-names>I</given-names></name><name><surname>Regla-Nava</surname><given-names>JA</given-names></name><name><surname>Pasquali</surname><given-names>M</given-names></name><name><surname>Scott</surname><given-names>JA</given-names></name><name><surname>Vitale</surname><given-names>F</given-names></name><name><surname>Unal</surname><given-names>MA</given-names></name><name><surname>Mattevi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Toward nanotechnology-enabled approaches against the COVID-19 pandemic</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>6</issue><fpage>6383</fpage><lpage>6406</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFWjsLnI</pub-id><pub-id pub-id-type=\"pmid\">32519842</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c03697</pub-id></mixed-citation></ref><ref id=\"CR201\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wilson</surname><given-names>DR</given-names></name><name><surname>Sen</surname><given-names>R</given-names></name><name><surname>Sunshine</surname><given-names>JC</given-names></name><name><surname>Pardoll</surname><given-names>DM</given-names></name><name><surname>Green</surname><given-names>JJ</given-names></name><name><surname>Kim</surname><given-names>YJ</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradable STING agonist nanoparticles for enhanced cancer immunotherapy</article-title><source>Nanomed Nanotechnol Biol Med</source><year>2018</year><volume>14</volume><issue>2</issue><fpage>237</fpage><lpage>246</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhvVOrsLjK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nano.2017.10.013</pub-id></mixed-citation></ref><ref id=\"CR202\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Ye</surname><given-names>F</given-names></name><name><surname>Cui</surname><given-names>M</given-names></name><name><surname>Shibata</surname><given-names>R</given-names></name><name><surname>Xu</surname><given-names>R</given-names></name><name><surname>Cheng</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>DY</given-names></name></person-group><source>Diagnostic methodology and technology in molecular genetic pathology. Molecular genetic pathology</source><year>2013</year><publisher-loc>New York</publisher-loc><publisher-name>Springer</publisher-name><fpage>211</fpage><lpage>300</lpage><pub-id pub-id-type=\"doi\">10.1007/978-1-4614-4800-6_9</pub-id></mixed-citation></ref><ref id=\"CR203\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>S</given-names></name><name><surname>Peng</surname><given-names>Z</given-names></name><name><surname>Xiao</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Yan</surname><given-names>R</given-names></name><name><surname>Luo</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Detection and analysis of nucleic acid in various biological samples of COVID-19 patients</article-title><source>Travel Med Infect Dis</source><year>2020</year><volume>37</volume><pub-id pub-id-type=\"pmid\">32311437</pub-id><pub-id pub-id-type=\"pmcid\">7165102</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tmaid.2020.101673</pub-id></mixed-citation></ref><ref id=\"CR204\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>K</given-names></name><name><surname>Saha</surname><given-names>R</given-names></name><name><surname>Su</surname><given-names>D</given-names></name><name><surname>Krishna</surname><given-names>VD</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Cheeran</surname><given-names>MCJ</given-names></name><name><surname>Wang</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">Magnetic-nanosensor-based virus and pathogen detection strategies before and during COVID-19</article-title><source>ACS Appl Nano Mater</source><year>2020</year><volume>3</volume><issue>10</issue><fpage>9560</fpage><lpage>9580</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjs7vN</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsanm.0c02048</pub-id></mixed-citation></ref><ref id=\"CR206\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xiang</surname><given-names>SD</given-names></name><name><surname>Scholzen</surname><given-names>A</given-names></name><name><surname>Minigo</surname><given-names>G</given-names></name><name><surname>David</surname><given-names>C</given-names></name><name><surname>Apostolopoulos</surname><given-names>V</given-names></name><name><surname>Mottram</surname><given-names>PL</given-names></name><name><surname>Plebanski</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pathogen recognition and development of particulate vaccines: does size matter?</article-title><source>Methods</source><year>2006</year><volume>40</volume><issue>1</issue><fpage>1</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XpvFGiu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">16997708</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ymeth.2006.05.016</pub-id></mixed-citation></ref><ref id=\"CR207\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>G</given-names></name><name><surname>Zeng</surname><given-names>S</given-names></name><name><surname>Zhang</surname><given-names>B</given-names></name><name><surname>Swihart</surname><given-names>MT</given-names></name><name><surname>Yong</surname><given-names>KT</given-names></name><name><surname>Prasad</surname><given-names>PN</given-names></name></person-group><article-title xml:lang=\"en\">New generation cadmium-free quantum dots for biophotonics and nanomedicine</article-title><source>Chem Rev</source><year>2016</year><volume>116</volume><issue>19</issue><fpage>12234</fpage><lpage>12327</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFemurjF</pub-id><pub-id pub-id-type=\"pmid\">27657177</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.chemrev.6b00290</pub-id></mixed-citation></ref><ref id=\"CR208\"><mixed-citation publication-type=\"other\">Xu W, Ling P, Zhang T (2013) Polymeric micelles, a promising drug delivery system to enhance bioavailability of poorly water-soluble drugs. J Drug Deliv</mixed-citation></ref><ref id=\"CR209\"><mixed-citation publication-type=\"other\">Yadav HK, Almokdad AA, Sumia I, Debe MS (2019) Polymer-based nanomaterials for drug-delivery carriers. In Nanocarriers for drug delivery. Elsevier pp 531–556</mixed-citation></ref><ref id=\"CR210\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>X</given-names></name><name><surname>Zhou</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name><name><surname>Jin</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">An overview of biodegradable nanomaterials and applications in vaccines</article-title><source>Vaccine</source><year>2020</year><volume>38</volume><issue>5</issue><fpage>1096</fpage><lpage>1104</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlehurfF</pub-id><pub-id pub-id-type=\"pmid\">31813649</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2019.11.031</pub-id></mixed-citation></ref><ref id=\"CR211\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Shu</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Yu</surname><given-names>Z</given-names></name><name><surname>Fang</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</article-title><source>Lancet Respir Med</source><year>2020</year><volume>8</volume><issue>5</issue><fpage>475</fpage><lpage>481</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjvVWqt70%3D</pub-id><pub-id pub-id-type=\"pmid\">32105632</pub-id><pub-id pub-id-type=\"pmcid\">7102538</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(20)30079-5</pub-id></mixed-citation></ref><ref id=\"CR212\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ye</surname><given-names>S</given-names></name><name><surname>Shao</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Guo</surname><given-names>N</given-names></name><name><surname>Zuo</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Lu</surname><given-names>Z</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>Q</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity of graphene oxide: how sharp-edged structure and charge matter</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>38</issue><fpage>21571</fpage><lpage>21579</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsFSitrbM</pub-id><pub-id pub-id-type=\"pmid\">26370151</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b06876</pub-id></mixed-citation></ref><ref id=\"CR213\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yousaf</surname><given-names>SS</given-names></name><name><surname>Isreb</surname><given-names>A</given-names></name><name><surname>Khan</surname><given-names>I</given-names></name><name><surname>Mewsiga</surname><given-names>E</given-names></name><name><surname>Elhissi</surname><given-names>A</given-names></name><name><surname>Ahmed</surname><given-names>W</given-names></name><name><surname>Alhnan</surname><given-names>MA</given-names></name></person-group><article-title xml:lang=\"en\">Impact of nanosizing on the formation and characteristics of polymethacrylate films: micro-versus nano-suspensions</article-title><source>Pharm Dev Technol</source><year>2021</year><volume>26</volume><issue>7</issue><fpage>729</fpage><lpage>739</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Ggtr7L</pub-id><pub-id pub-id-type=\"pmid\">34034606</pub-id><pub-id pub-id-type=\"doi\">10.1080/10837450.2021.1931886</pub-id></mixed-citation></ref><ref id=\"CR214\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yu</surname><given-names>CY</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Yean</surname><given-names>CY</given-names></name><name><surname>Ang</surname><given-names>GY</given-names></name></person-group><article-title xml:lang=\"en\">Nucleic acid-based diagnostic tests for the detection SARS-CoV-2: an update</article-title><source>Diagnostics</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1CjtLzK</pub-id><pub-id pub-id-type=\"pmid\">33401392</pub-id><pub-id pub-id-type=\"pmcid\">7823986</pub-id><pub-id pub-id-type=\"doi\">10.3390/diagnostics11010053</pub-id></mixed-citation></ref><ref id=\"CR215\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zazo</surname><given-names>H</given-names></name><name><surname>Colino</surname><given-names>CI</given-names></name><name><surname>Lanao</surname><given-names>JM</given-names></name></person-group><article-title xml:lang=\"en\">Current applications of nanoparticles in infectious diseases</article-title><source>J Control Release</source><year>2016</year><volume>224</volume><fpage>86</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XovFCgtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">26772877</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jconrel.2016.01.008</pub-id></mixed-citation></ref><ref id=\"CR216\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Q</given-names></name><name><surname>Honko</surname><given-names>A</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Gong</surname><given-names>H</given-names></name><name><surname>Downs</surname><given-names>SN</given-names></name><name><surname>Vasquez</surname><given-names>JH</given-names></name><name><surname>Fang</surname><given-names>RH</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Griffiths</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Cellular nanosponges inhibit SARS-CoV-2 infectivity</article-title><source>Nano Lett</source><year>2020</year><volume>20</volume><issue>7</issue><fpage>5570</fpage><lpage>5574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFKiu7rM</pub-id><pub-id pub-id-type=\"pmid\">32551679</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.nanolett.0c02278</pub-id></mixed-citation></ref><ref id=\"CR217\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>F</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Vijver</surname><given-names>MG</given-names></name><name><surname>Peijnenburg</surname><given-names>WJ</given-names></name></person-group><article-title xml:lang=\"en\">Probing nano-QSAR to assess the interactions between carbon nanoparticles and a SARS-CoV-2 RNA fragment</article-title><source>Ecotoxicol Environ Saf</source><year>2021</year><volume>219</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFynurvJ</pub-id><pub-id pub-id-type=\"pmid\">34044308</pub-id><pub-id pub-id-type=\"pmcid\">8133531</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2021.112357</pub-id></mixed-citation></ref><ref id=\"CR218\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>K</given-names></name><name><surname>Di</surname><given-names>L</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Yue</surname><given-names>P</given-names></name><name><surname>Song</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Traditional herbal medicine and nanomedicine: Converging disciplines to improve therapeutic efficacy and human health</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>178</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVelsrzI</pub-id><pub-id pub-id-type=\"pmid\">34499982</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2021.113964</pub-id></mixed-citation></ref><ref id=\"CR219\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>SH</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>SJ</given-names></name><name><surname>Yao</surname><given-names>L</given-names></name><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>YD</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 pneumonia via a novel deep learning architecture</article-title><source>J Comput Sci Technol</source><year>2022</year><volume>37</volume><issue>2</issue><fpage>330</fpage><lpage>343</lpage><pub-id pub-id-type=\"pmid\">35496726</pub-id><pub-id pub-id-type=\"pmcid\">9035772</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11390-020-0679-8</pub-id></mixed-citation></ref><ref id=\"CR220\"><mixed-citation publication-type=\"other\">Zhang F, Abudayyeh OO, Gootenberg JS (2020a) A protocol for detection of COVID-19 using CRISPR diagnostics. A protocol for detection of COVID-19 using CRISPR diagnostics 8.</mixed-citation></ref><ref id=\"CR221\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhao</surname><given-names>W</given-names></name><name><surname>Zhong</surname><given-names>Z</given-names></name><name><surname>Xie</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Q</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Relation between chest CT findings and clinical conditions of coronavirus disease (COVID-19) pneumonia: a multicenter study</article-title><source>Ajr Am J Roentgenol</source><year>2020</year><volume>214</volume><issue>5</issue><fpage>1072</fpage><lpage>1077</lpage><pub-id pub-id-type=\"pmid\">32125873</pub-id><pub-id pub-id-type=\"doi\">10.2214/AJR.20.22976</pub-id></mixed-citation></ref><ref id=\"CR222\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhu</surname><given-names>J</given-names></name><name><surname>Shi</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer-based nanodevices for targeted drug delivery applications</article-title><source>J Mater Chem B</source><year>2013</year><volume>1</volume><issue>34</issue><fpage>4199</fpage><lpage>4211</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1entrfP</pub-id><pub-id pub-id-type=\"pmid\">32261015</pub-id><pub-id pub-id-type=\"doi\">10.1039/c3tb20724b</pub-id></mixed-citation></ref><ref id=\"CR223\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ziegler</surname><given-names>CG</given-names></name><name><surname>Allon</surname><given-names>SJ</given-names></name><name><surname>Nyquist</surname><given-names>SK</given-names></name><name><surname>Mbano</surname><given-names>IM</given-names></name><name><surname>Miao</surname><given-names>VN</given-names></name><name><surname>Tzouanas</surname><given-names>CN</given-names></name><name><surname>Cao</surname><given-names>Y</given-names></name><name><surname>Yousif</surname><given-names>AS</given-names></name><name><surname>Bals</surname><given-names>J</given-names></name><name><surname>Hauser</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues</article-title><source>Cell</source><year>2020</year><volume>181</volume><issue>5</issue><fpage>1016</fpage><lpage>1035</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpsVKkuro%3D</pub-id><pub-id pub-id-type=\"pmid\">32413319</pub-id><pub-id pub-id-type=\"pmcid\">7252096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cell.2020.04.035</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">13197</journal-id><journal-id journal-id-type=\"doi\">10.1007/13197.0975-8402</journal-id><journal-title-group><journal-title>Journal of Food Science and Technology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Food Sci Technol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0022-1155</issn><issn pub-type=\"epub\">0975-8402</issn><publisher><publisher-name>Springer India</publisher-name><publisher-loc>New Delhi</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s13197-022-05528-8</article-id><article-id pub-id-type=\"manuscript\">5528</article-id><article-id pub-id-type=\"doi\">10.1007/s13197-022-05528-8</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Bioactive metabolites in functional and fermented foods and their role as immunity booster and anti-viral innate mechanisms</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Varsha</surname><given-names>Kontham Kulangara</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Narisetty</surname><given-names>Vivek</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Brar</surname><given-names>Kamalpreet Kaur</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"aff\" rid=\"Aff4\">4</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Madhavan</surname><given-names>Aravind</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Alphy</surname><given-names>Maria Paul</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Sindhu</surname><given-names>Raveendran</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Awasthi</surname><given-names>Mukesh Kumar</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Varjani</surname><given-names>Sunita</given-names></name><xref ref-type=\"aff\" rid=\"Aff10\">10</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au9\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-4191-8048</contrib-id><name><surname>Binod</surname><given-names>Parameswaran</given-names></name><address><email>binodkannur@niist.res.in</email></address><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref><xref ref-type=\"corresp\" rid=\"IDs13197022055288_cor9\">j</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.254567.7</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 9075 106X</institution-id><institution content-type=\"org-division\">School of Medicine</institution><institution content-type=\"org-name\">University of South Carolina</institution></institution-wrap><addr-line content-type=\"postcode\">29209</addr-line><addr-line content-type=\"city\">Columbia</addr-line><addr-line content-type=\"state\">SC</addr-line><country country=\"US\">USA</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Moolec Science, Innovation Centre</institution></institution-wrap><addr-line content-type=\"street\">Gallows Hill, CV34 6UW</addr-line><addr-line content-type=\"city\">Warwick</addr-line><country country=\"GB\">UK</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.21100.32</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1936 9430</institution-id><institution content-type=\"org-division\">Department of Civil Engineering, Lassonde School of Engineering</institution><institution content-type=\"org-name\">York University</institution></institution-wrap><addr-line content-type=\"postcode\">M3J 1P3</addr-line><addr-line content-type=\"city\">North York, Toronto</addr-line><addr-line content-type=\"state\">Ontario</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.468122.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 5906 3081</institution-id><institution content-type=\"org-name\">Centre Technologique des Résidus Industriels en Abitibi Témiscamingue</institution></institution-wrap><addr-line content-type=\"street\">433 Boulevard du collège, J9X0E1</addr-line><addr-line content-type=\"city\">Rouyn-Noranda</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.418917.2</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0177 8509</institution-id><institution content-type=\"org-name\">Rajiv Gandhi Centre for Biotechnology</institution></institution-wrap><addr-line content-type=\"postcode\">695 014</addr-line><addr-line content-type=\"city\">Thiruvananthapuram</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.419023.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1808 3107</institution-id><institution content-type=\"org-division\">Microbial Processes and Technology Division</institution><institution content-type=\"org-name\">CSIR-National Institute for Interdisciplinary Science and Technology</institution></institution-wrap><addr-line content-type=\"postcode\">695 019</addr-line><addr-line content-type=\"city\">Trivandrum</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.469887.c</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 7744 2771</institution-id><institution content-type=\"org-name\">Academy of Scientific and Innovative Research (AcSIR)</institution></institution-wrap><addr-line content-type=\"postcode\">201002</addr-line><addr-line content-type=\"city\">Ghaziabad</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Department of Food Technology</institution><institution content-type=\"org-name\">T K M Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">691505</addr-line><addr-line content-type=\"city\">Kollam</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.144022.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1760 4150</institution-id><institution content-type=\"org-division\">College of Natural Resources and Environment</institution><institution content-type=\"org-name\">Northwest A &amp; F University</institution></institution-wrap><addr-line content-type=\"postcode\">712 100</addr-line><addr-line content-type=\"city\">Yangling</addr-line><addr-line content-type=\"state\">Shaanxi</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff10\"><label>10</label><institution-wrap><institution content-type=\"org-name\">Gujarat Pollution Control Board</institution></institution-wrap><addr-line content-type=\"street\">Paryavaran Bhavan, CHH Road, Sector 10 A</addr-line><addr-line content-type=\"postcode\">382010</addr-line><addr-line content-type=\"city\">Gandhinagar</addr-line><addr-line content-type=\"state\">Gujarat</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs13197022055288_cor9\"><label>j</label><email>binodkannur@niist.res.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>24</day><month>6</month><year>2022</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>9</month><year>2023</year></pub-date><volume>60</volume><issue seq=\"2\">9</issue><fpage>2309</fpage><lpage>2318</lpage><history><date date-type=\"registration\"><day>1</day><month>6</month><year>2022</year></date><date date-type=\"rev-recd\"><day>19</day><month>4</month><year>2022</year></date><date date-type=\"accepted\"><day>18</day><month>5</month><year>2022</year></date><date date-type=\"online\"><day>24</day><month>6</month><year>2022</year></date></history><permissions><copyright-statement content-type=\"compact\">© Association of Food Scientists &amp; Technologists (India) 2022</copyright-statement><copyright-year>2022</copyright-year><copyright-holder>Association of Food Scientists &amp; Technologists (India)</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par15\">Live microorganisms in the fermented foods termed probiotics and their secondary metabolites with bioactive potential were considered as potential anti-viral capabilities through various mechanisms. Given the importance of functional and fermented foods in disease prevention, there is a need to discuss the contextualization and deep understanding of the mechanism of action of these foods, particularly considering the appearance of coronavirus (COVID-19) pandemic, which is causing health concerns and increased social services globally. The mechanism of probiotic strains or their bioactive metabolites is due to stimulation of immune response through boosting T-lymphocytes, cytokines, and cell toxicity of natural killer cells. Proper consumption of these functional and fermented foods may provide additional antiviral approaches for public benefit by modulating the immune functions in the hosts.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Anti-viral metabolites</kwd><kwd>Immunity</kwd><kwd>Fermented foods</kwd><kwd>COVID-19</kwd><kwd>Functional foods</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>19</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Association of Food Scientists &amp; Technologists (India)</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2022</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/13197_2022_Article_5528.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Kontham Kulangara Varsha and Vivek Narisetty these authors contributed equally and Co-first authors.</p></notes><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s13197-022-05528-8\">https://doi.org/10.1007/s13197-022-05528-8</ext-link>.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par42\">The process of fermentation has long been used to increase the shelf life, flavour and functional properties of food. In addition to help food last longer, fermentation increases the nutritional value, and the probiotic bacteria present in fermented food confer health benefits that comprise reducing the risk of type 2 diabetes and cardiovascular diseases by decreasing total and LDL cholesterol (Marco et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). Multiple clinical trials investigated the advantages of fermented food and exposed the ability of kimchi and yogurt to downsize the risk of type 2 diabetes (Chen et al. <xref ref-type=\"bibr\" rid=\"CR7\">2014</xref>), Chungkookjang to decrease obesity (Byun et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>), and the capacity of fermented milk and rye bread to control infection and irritable bowel syndrome (Laatikainen et al. <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>). Among this health promoting and infectivity demoting effects include the antiviral activity of certain fermented foods owing to the presence of live bacteria in it, and the examples are Chr. Hansen (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.chr-hansen.com/\">http://www.chr-hansen.com/</ext-link>); Kingdom Supercultures (<ext-link ext-link-type=\"uri\" xlink:href=\"https://kingdomsupercultures.com/\">https://kingdomsupercultures.com/</ext-link>); Probitat (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.probitat.eu/\">http://www.probitat.eu/</ext-link>); 3FBIO Ltd (ENOUGH) (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.enough-food.com/\">https://www.enough-food.com/</ext-link>); Fermbiotics (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fermbiotics.com/\">https://www.fermbiotics.com/</ext-link>).</p><p id=\"Par43\">Functional foods (FFs) are distinguished by their nutraceuticals. Nutraceuticals are either whole foods or food ingredients that provide health benefits, such as disease prevention and/or treatment. This is generally associated with their microvascular, anti-inflammatory, and anti-oxidation properties in highly affected individuals. Active FFs are rich in polyphenols, terpenoids, flavonoids, and unsaturated fatty acids ingredients are among the widely active functional foods to be consumed (Alkhatib et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018</xref>). Recently, Acquah et al. (<xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>) reported that some bioactive peptides presented many similar hormonal and neurological activities of human system (Acquah et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>). On the other side, fermented food products contain probiotics (García-Burgos et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Probiotics are “live microorganisms which when administered in adequate amounts confer a health benefit on the host”. Covid-19 was observed to be a severe acute respiratory syndrome (SARS) which is named as SARS corona virus 2 or SARS-CoV-2 (Lai et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). Broadly, coronaviruses are large and enveloped mainly found in humans and mammals and known to cause respiratory, gastrointestinal, and neurological disease. Through genetic recombination and mutation, corona viruses can be more infectious. Stringent measures were taken by various countries based on their resource limitations, geography, population, and political factors. Although these severe interventions, since Feb. 2020 and as of March 17, 2022, the outbreak has infected almost 464 million people and killed over 6.06 million. Previous studies have shown that 65% of airborne MERS-CoV virus remains viable in the air and infectious after 60 min (Pyankov et al. <xref ref-type=\"bibr\" rid=\"CR61\">2018</xref>). The presence of SARS-CoV-2 in hospitals and entrance to department stores in Wuhan, China (Liu et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>), air outlet fans in a COVID-19 outbreak center in Singapore (Ong et al. <xref ref-type=\"bibr\" rid=\"CR57\">2020</xref>) and hospital isolation rooms in Nebraska (Santarpia et al. <xref ref-type=\"bibr\" rid=\"CR63\">2020</xref>) were observed. Recently, some preliminarily results have shown that the virus can survive up to 3 h as aerosol and infect cells throughout this period (van Doremalen et al. <xref ref-type=\"bibr\" rid=\"CR67\">2020</xref>). A ferret model of SARS-CoV-2 infection that reiterates aspects of human disease has also confirmed the potential of virus airborne transmission (Kim et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020</xref>). A recent study reported that SARS-CoV-2 can be viable for 4 h on copper, 24 h on cardboard, 2–3 days on plastic and stainless steel (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Generally, SARS-CoV-2 is transmitted via respiratory pathways, but they may spread via multiple dominant routes. An acceptor individual must receive an infectious dose of the virus from a donor, either directly through the air or indirectly through deposits of the virus on various surfaces. Molecular based detection techniques, like plaque assay, Enzyme-Linked Immunosorbent Assay (ELISA), Lateral-Flow (immuno) Assay (LFA), Polymerase Chain Reaction (PCR) and Surface Plasmon Resonance (SPR) assay are identified quality and quantitative techniques for virus detection. Approximately 120 candidates are hardly working to formulate vaccines based on either nucleic acids, inactivated or live attenuated virus, and recombinant proteins (Le et al. 2020). Other approaches based on monoclonal antibodies, hyperimmune globulin will be another option. The antiviral role of FFs and fermented foods for the defence of COVID-19 is lacking and has not yet been established. However, recent reports showed that diabetes is considered a risk factor for the development and diagnosis of COVID-19 (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Fermented foods generally contain single or multiple genera of live probiotic microorganisms that has positive impact on the host beneficially preserving the intestinal microbiota that in return has modulatory role in immune responses and human health (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). Herein, we present antiviral role of fermented foods containing of probiotic and bioactive compounds; mechanism to stimulate immunity; case studies of viral infections and their prevention or treatment and different types of fermented foods available worldwide. This review leads to inculcating the knowledge of benefits of probiotics and fermented foods in reducing the risk of COVID-19 or other viral infections.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic representation of applications of probiotics and bioactive metabolites</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec2\"><title>Antiviral activity of fermented foods containing probiotics and bioactive compounds</title><p id=\"Par45\">The plethora of fermented foods is derived from plants and animal resources. Plant-based fermented foods are tempeh, <italic>kimchi</italic>, <italic>tempoyak</italic>, and <italic>tapai</italic>. While animal-based fermented foods include kefir, cheese, yogurt, sauces, other dairy and meat products (Tamang et al., 2016; Raji et al. <xref ref-type=\"bibr\" rid=\"CR62\">2017</xref>; Khalil et al. <xref ref-type=\"bibr\" rid=\"CR32\">2018</xref>; Lee et al. <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Alcoholic beverages like fermented porridge made of sorghum and maize in South American countries (Tamang et al. <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>). Fermented foods are enriched with anti-microbial end products such as various organic acids, ethanol and peptides or bacteriocins and several studies report the antiviral potential of fermented foods <italic>in-vitro</italic> and <italic>in-vivo</italic>. The probiotic bacteria and bioactive compounds in fermented foods possess antiviral activities against gut and respiratory and viruses. These active foods stimulate immune system function by increasing the synthesis of pro-inflammatory cytokines and T lymphocytes (CD3<sup>+</sup>, CD16<sup>+</sup>, CD56<sup>+</sup>) (Muhialdin et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>). <italic>Lactobacillus plantarum</italic> LBP-K10 isolated from kimchi synthesised cyclic di-peptides that inhibited the growth of the influenza A (H3N2) virus (Kwak et al. <xref ref-type=\"bibr\" rid=\"CR38\">2013</xref>), while another study reported declined survival of feline calicivirus and murine norovirus proliferation during <italic>Dongchimi</italic> fermentation along with an increase in lactic acid bacteria (LAB) (Lee et al. <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). Likewise, soy extracts fermented with <italic>Aspergillus fumigatus</italic> F-993 or <italic>A. awamori</italic> FB-133 showed therapeutic potential by decreasing hepatitis A virus titers <italic>in-vitro</italic> (Ghanem et al. <xref ref-type=\"bibr\" rid=\"CR14\">2020</xref>).</p><p id=\"Par46\">The cell free supernatant of yogurt has antiviral activity for RNA viruses such as enterovirus 71 and influenza, porcine epidemic diarrhoea virus and Coxsackie A and B viruses (Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>). Polyphenols, bioactive peptides, exopolysaccharides, linoleic acid, and vitamins are among the bioactive compounds found in fermented foods (Hayes and García-Vaquero <xref ref-type=\"bibr\" rid=\"CR24\">2016</xref>). Spanish sausage release angiotensin-converting enzyme inhibitor (ACE-I) when <italic>L. pentosus</italic> and <italic>S. carnosus</italic> used as a inoculum for the fermentation (Mora et al. <xref ref-type=\"bibr\" rid=\"CR51\">2015</xref>). The fermentation of <italic>Ruditapes philippinarum</italic> clams with <italic>Bacillus natto</italic> stimulate hyper-production of ACE-I peptide synthesis exerts anti-cancer property (Chen et al. <xref ref-type=\"bibr\" rid=\"CR8\">2018</xref>). However, further studies assessing the full potential of the probiotics to combat COVID-19 should be carried out (Olaimat et al. <xref ref-type=\"bibr\" rid=\"CR55\">2020</xref>) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>).</p></sec><sec id=\"Sec3\"><title>Clinical impact of probiotics and bioactive compounds against viral infections in humans</title><p id=\"Par47\">High mutation rates of RNA viruses lead to their rapid evolution and better environmental adaptability (Carrasco-Hernandez et al. <xref ref-type=\"bibr\" rid=\"CR6\">2017</xref>). Various case studies report the ability of fermented and probiotic food to reduce respiratory tract infections. In one case study, some COVID-19 patients exhibited <italic>Bifidobacterium</italic> and <italic>Lactobacillus</italic> dysbiosis. The aged patients who suffered severely from COVID-19 had poor gut microbiota diversity (Dhar et al., 2020). Probiotics <italic>L. acidophilus, Bifidobacterium</italic>, and <italic>Saccharomyces boulardii</italic>, as well as minerals and vitamins, were found to reduce the complications of massive antibiotic-associated diarrhoea and <italic>Clostridium difficile</italic> infections. (Horowitz et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). The simultaneous intake of probiotics with azithromycin decreases the severity of <italic>Candida albicans</italic> infection. In another study, COVID-19-like symptoms in a young boy disappeared after 2 days of probiotic administration. (Ji et al. <xref ref-type=\"bibr\" rid=\"CR28\">2020</xref>). The administration of probiotics as an adjunct protected 97% patients against SARS-CoV2 infection. According to the recent trail, patients with severe disease recovered more easily by probiotics therapy (87.5% vs. 40.4%, p = 0.037) compared to non-severe ones. A human clinical trial describes oral intake of the probiotic <italic>L. fermentum</italic> CECT5716 increased NK cells proliferation after vaccination when compared to the group without probiotic consumption (Olivares et al. <xref ref-type=\"bibr\" rid=\"CR56\">2007</xref>). A study conducted on elderly people found to be having a significant increase in NK cell activity (Makino et al. <xref ref-type=\"bibr\" rid=\"CR46\">2010</xref>). At the same time, a randomized control clinical trial among women consumed 1073R-1-yogurt yogurt (n = 479) demonstrated IFN-γ production with no increase in NK cell activity (Kinoshita et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>).</p><p id=\"Par48\">A control trial revealed that the probiotic strain <italic>Lactobacillus</italic> GG holds an adjuvant potential (Davidson et al. <xref ref-type=\"bibr\" rid=\"CR11\">2011</xref>). In children, for acute rotavirus diarrhoea, probiotic administration can relieve symptoms (Grandy et al. <xref ref-type=\"bibr\" rid=\"CR17\">2010</xref>). Multiple clinical trials are underway where the adeptness of probiotic and other dietary supplements is being investigated to alleviate the symptoms of COVID-19 infection (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>).</p><p id=\"Par49\">In the last two years of CoviD-19 pandemic, priority was to prevent the spreading, reduce the infections, save the lives, inculcate the knowledge of vaccination and healthy diet. Frontline health teams would benefit from the development of cutting-edge technology and the collection of available evidence. Sufficient nutrition improves health and boosts immunity, which aids in the prevention and treatment of infections. In this review we have discussed the role of probiotics in combating COVID-19 based on recent evidences, as well as their role as immune-modulators and antiviral agents. Further investigations on impact of probiotic strains and their bioactive compounds on COVID-19 affected individuals are awaiting (Kurian et al. <xref ref-type=\"bibr\" rid=\"CR37\">2021</xref>). However, more research needs to be done to study the production titres of interleukins, interferons, antibodies, and viral count due to probiotic administration during viral infection especially SARS-CoV-2.</p></sec><sec id=\"Sec4\"><title>Immune modulatory mechanisms by probiotics and fermented food</title><p id=\"Par50\">The health benefits conferred by fermented food are mainly attributed to the presence of live probiotic bacteria present which colonize the intestine and impart favourable effects. Lymphocytes, NK cells, macrophages and neutrophils are capable of mediating antibody dependent cell mediated cytotoxicity (ADCC) against virus infected cells and various probiotic and fermented food are reported to induce these immune cells. Multiple clinical trials have shown the ability of fermented food products with probiotics to improve NK cell activity and <italic>L. casei</italic> strain Shirota is a principal probiotic strain with this ability (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Additionally, consumption of dairy yogurt containing probiotics increased NK cell, IL-12, IFNγ and IgG1 levels in a randomized, open-label, placebo-controlled study conducted in 200 healthy volunteers (Lee et al. <xref ref-type=\"bibr\" rid=\"CR43\">2017</xref>). Probiotics play critical role in resistance against viral infections particularly in elderly people with age-related decline in lymphoid cell activity, by modulating the immune responses of hosts. Randomized control trials show that probiotic consumption significantly increased NK cell activity and CD56-positive lymphocytes in peripheral circulation in healthy elderly individuals (Gui et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). Probiotic intakes enhance cellular immunity in elderly who had poor pretreatment immune responses. Dietary supplementation of probiotic <italic>B. lactis</italic> HN019 increased helper (CD4 (+)) and activated (CD25 (+)) T lymphocytes, polymorphonuclear cells and natural killer cells in healthy elderly volunteers (Gill et al. <xref ref-type=\"bibr\" rid=\"CR16\">2001</xref>). Probiotic consumption found to increase natural and acquired immunity in mice and significantly improved serum antibody responses to antigens administered orally and systemically (Gill et al. <xref ref-type=\"bibr\" rid=\"CR15\">2000</xref>). Oral administration of heat killed probiotics from Mongolian dairy products augmented IFN -α, IL-12, and IFN -γ productions and increased NK cell activity leading to alleviated influenza symptoms in mice (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Similarly, oral administration of <italic>L. rhamnoses</italic> CRL1505 improved resistance to RSV infection in infant mice via IFN-γ and IL-10 secretion, which resulted in the activation of CD103<sup>+</sup> and CD11b<sup>high</sup> dendritic cells and the generation of CD3<sup>+</sup>CD4<sup>+</sup>IFN-γ<sup>+</sup> Th1 cells, with subsequent attenuation of strong Th2 reactions associated with RSV challenge (Chiba et al. <xref ref-type=\"bibr\" rid=\"CR9\">2013</xref>). An in vitro model study demonstrated the ability of probiotic bacteria to decrease vesicular stomatitis viral infection by production of nitric oxide and inflammatory cytokines such as IL-6 and IFN-γ (Ivec et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). When <italic>L. casei</italic> DK128 was administered into a mice intranasally, reduction in the weight and viral loads was observed, which might elicit the protection against different subtypes of influenza viruses, and mice are observed to be immune to primary infection and subsequently developed heterosubtypic secondary virus infection. The protective effect was linked to an increase in alveolar macrophage cells in the lungs and airways, the early induction of virus-specific antibodies, and lower levels of pro-inflammatory cytokines and innate immune cells (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Jung et al. <xref ref-type=\"bibr\" rid=\"CR29\">2017</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Possible mechanism of anti-viral activity by probiotics</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig2_HTML.png\"/></fig></p><p id=\"Par52\">Exopolysaccharides (EPS) secreted by probiotic microrganisms in fermented food contribute towards their immunomodulatory ability and antiviral potential <italic>in-vivo</italic>. Consumption of yogurt consisting of the starter culture <italic>L. delbrueckii</italic> ssp. <italic>bulgaricus</italic> OLL1073R-1 and secreted EPS reduced influenza virus titer in mice. The prognosis observed significant increase of anti-influenza virus antibodies such as IgA and IgG<sub>1</sub> along with augmented NK cell activity. In knockout mice, the presence of myeloid differentiation factor 88, EPS produced by this probiotic strain activated NK cells through IL-12- and IL-18-mediated IFN-γ production. The same probiotic strain demonstrated resistance flu virus by inducing NK cell activity in human subjects (Makino et al. <xref ref-type=\"bibr\" rid=\"CR47\">2016</xref>). EPS produced by <italic>L. delbrueckii</italic> TUA4408L improved the resistance to rotaviral infection by preventing the viral replication, activation of Toll-like receptor 3 rendering antiviral innate immune response in porcine intestinal epithelial cells. The study also reports <italic>L. delbrueckii</italic> TUA4408L and its EPS activated interferon regulatory factor (IRF)-3 and nuclear factor κB (NF-κB) signalling pathways leading to improved expression of the antiviral factors IFN-β, Myxovirus resistance gene A (MxA) and RNaseL (Kanmani et al. <xref ref-type=\"bibr\" rid=\"CR30\">2018</xref>). In addition to EPS, short chain fatty acids (SCFAs) produced by these microorganisms are observed to regulate immune responses (Parada Venegas et al. <xref ref-type=\"bibr\" rid=\"CR58\">2019</xref>). Indirectly D-phenyl lactate produced by various lactic acid bacteria regulates the immune reactions stimulated by G-protein coupled receptors by activating regulatory hydroxycarboxylic acids (Peters et al. <xref ref-type=\"bibr\" rid=\"CR59\">2019</xref>). Lactate and pyruvate also contribute to enhanced immune responses in mice models by inducing GPR31-mediated dendrite protrusion of intestinal CX3CR1<sup>+</sup> cells (Morita et al. <xref ref-type=\"bibr\" rid=\"CR52\">2019</xref>). The modulation of gut microbiota seems to be a one of the approaches to combat viral infections including COVID-19, but it needs to be further confirmed through animal models (Akour <xref ref-type=\"bibr\" rid=\"CR2\">2020</xref>). Overall, research indicates that fermented foods and probiotics contain high or low molecular weight bioactive metabolites that elicit modulations in the immune system rendering health benefits.</p></sec><sec id=\"Sec5\"><title>Future outlook</title><p id=\"Par53\">Fermentation by microbes applies various unconventional and uncharacterised enzymes to produce hydrolysates of protein with varying compositions of peptides. By utilizing highly efficient bacteria and by optimizing conditions of fermentation, a wide array of proteases could be produced to hydrolyse peptides and proteins with varying amino acid composition and different chain length could be synthesised in accordance to different enzyme specificity, possibly producing unique sequences of peptide with novel anti-viral bioactive properties.</p><p id=\"Par54\">It is predictable that the greatest perplexing part for researchers dealing with functional food bioactive peptides is to warrant the full bioavailability of the bioactive component after their consumption. The bioavailability of bioactive peptide is based on the capability of the peptides to resist proteolytic activity in the intestine and serum, and their potential to enter into the blood stream and consequently, exercising their biological activity. Hence, it is critical to improve the processing settings in order to retain their biological activities. Encapsulation is one of the well-characterised technologies to improve the bioavailability of functionally active peptides. This technique has been extensively applied in the nutraceutical, and food industries to encapsulate biologically active components. Encapsulation of bioactive is economically viable as this technique could avoid the interaction of the peptides with environment and/or other components in the food matrix, decrease the consequence from processing, intensify peptide stability, protect peptides against digestive enzymes and improve bioavailability.</p><p id=\"Par55\">Since functionally active foods are graded as a normal or enriched with bioactive, it has certain side effects, although the research work in this perspective is not adequate to support this statement. Even though much cell line based, and animal model studies have been widely explored to examine the bioactivity of food derived bioactive, inadequate data on human experiments is available. More research data on human clinical trials are required to validate the efficiency of food derived bioactivities. Along with this other safety parameters, like cytotoxicity, allergic response of the functional food should be evaluated before commercialization.</p><p id=\"Par56\">Viral diseases are considered as an immune compromised state resulting from poor consumption of micronutrients, vitamins and other trace elements. Several research previously indicated enhanced function of the immune system by taking those many fermented and functional foods including essential fatty and amino acids, and the above-mentioned minerals and vitamins (Calder and Kew <xref ref-type=\"bibr\" rid=\"CR5\">2002</xref>). Satisfactory diet-based consumption, and supplementation of such functional foods, contribute to sustaining optimal levels in the humans, which improves several characteristics of the immune function, and provides an important antiviral prevention of COVID-19. On the other hand, less strong immune activation has been proved to be the primary threat factor for COVID-19, which makes it appropriate to define the defensive role of functional food particles benefits in the perspective of preventing COVID-19 and other viral diseases (Grant et al. <xref ref-type=\"bibr\" rid=\"CR18\">2020</xref>).</p><p id=\"Par57\">Examination of outcome of viral disease management in high-risk populations and aged people is very central. Large number of viral and COVID 19 infection rate is reported in older adults and persons with other co-morbidities. The prevalence of COVID-19 in people with diabetes is high and now considered a hazard factor for the progression COVID-19 (McGurnaghan et al. <xref ref-type=\"bibr\" rid=\"CR49\">2021</xref>). Therefore, best “immune-modulating” functional foods could provide the finest prevention and progression of the viral diseases. The use of functional foods to combat viral diseases would especially benefit the elderly, which has become a growing sector of the world population.</p></sec><sec id=\"Sec6\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par58\">Fermentation by microbe’s functions as a potent technique to supplement foods with biologically active peptides from various animal or plant sources. These bioactive compounds produced in the fermented foods tends to boost the immune response either directly or indirectly against the viral infections by modulating the lymphocytes, NK cells, macrophages and neutrophils that are capable of mediating antibody dependent cell mediated cytotoxicity. Several works have been documented to unveil novel bioactive components from fermented foods and other edible plant by products. It is possible that the varieties of biopeptides from various fermented food will continue to develop in the coming years. Further awareness of these health beneficial fermented foods and their compositions could help in improving the health and wellness of the society.</p><p id=\"Par59\"><table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Antiviral bioactives from functional and fermented food</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"2\"><p>Probiotic microbes or Fermented foods</p></th><th align=\"left\"><p>Salient Feature</p></th><th align=\"left\"><p>Mode of action</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\" colspan=\"2\"><p><italic>Bifidobacterium animalis</italic></p></td><td align=\"left\"><p>Prevent</p><p>upper respiratory tract infection</p></td><td align=\"left\"><p>Inhibit viral replication</p></td><td align=\"left\"><p>(Smith et al. <xref ref-type=\"bibr\" rid=\"CR64\">2013</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Prevent gastroenteritis COVID virus</p></td><td align=\"left\"><p>Reduces granulocyte, diminishes virus recovery</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. lactis</italic></p></td><td align=\"left\"><p>Prevent respiratory tract infection</p></td><td align=\"left\"><p>Activates plasmacytoid dendrtitic cell</p></td><td align=\"left\"><p>(Kokubo et al. <xref ref-type=\"bibr\" rid=\"CR36\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Gastroenteritis coronaviruses (TGEV)</p></td><td align=\"left\"><p>Reduces inflammation and tissue injury</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>Lactobacillus casei</italic> DN-114 001; Dan Active/Actimel</p></td><td align=\"left\"><p>Reduced incidence and duration of RTIs</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Guillemard et al. <xref ref-type=\"bibr\" rid=\"CR21\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Kefir</p></td><td align=\"left\"><p>Zika, hepatitis C, influenza, rotaviruses</p></td><td align=\"left\"><p>Enhanced macrophage synthesis, increases phagocytosis, enhanced synthesis of (CD4+), CD8 + cells, immunoglobulins, neutrophilss, and various cytokines (IL-2, IL-12, INF-γ).</p></td><td align=\"left\"><p>(Hamida et al. <xref ref-type=\"bibr\" rid=\"CR23\">2021</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Yoghurt</p></td><td align=\"left\"><p>Inhibit Enterovirus</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Fermented ginseng extracts</p></td><td align=\"left\"><p>Inhibit influenza virus H1N1, H3N2, H5N1, and H7N9 strains</p></td><td align=\"left\"><p>Viral inoculation with extract of ginseng formed better immune responses against the 2<sup>0</sup> infection with homologous and heterosubtypic virus.</p></td><td align=\"left\"><p>(Wang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2018</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Black ginseng</p></td><td align=\"left\"><p>Inhibit influenza virus</p></td><td align=\"left\"><p>Black ginseng improved the levels of GM-CSF and IL-10 at the time of infection</p></td><td align=\"left\"><p>(Kim et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Dietary xylitol</p></td><td align=\"left\"><p>Inhibit influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Yin et al. <xref ref-type=\"bibr\" rid=\"CR73\">2014</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Chongkukjang (Traditional Korean feremented food)</p></td><td align=\"left\"><p>Influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Wei et al. <xref ref-type=\"bibr\" rid=\"CR69\">2015b</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Resveratrol (From red grapes)</p></td><td align=\"left\"><p>Inhibit Epstein-Barr virus</p></td><td align=\"left\"><p>Downregulation of antiapoptotic proteins</p></td><td align=\"left\"><p>(De Leo et al. <xref ref-type=\"bibr\" rid=\"CR12\">2012</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Zingiberofficinale (Ginger)</p></td><td align=\"left\"><p>Anti-chikungunya activity</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Curcumin</p></td><td align=\"left\"><p>Inhibit Zika and chikungunya viruses</p></td><td align=\"left\"><p>Curcumin interferes with virus-cell binding.</p></td><td align=\"left\"><p>(Mounce et al. <xref ref-type=\"bibr\" rid=\"CR53\">2017</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par60\"><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Ongoing clinical trials where probiotics and dietary supplements are used against COVID-19</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>No.</p></th><th align=\"left\"><p>Clinical trial identifier</p></th><th align=\"left\"><p>Intervention</p></th><th align=\"left\"><p>Aim</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>NCT04621071</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Evaluation of the efficacy of probiotics to dcrease the duration and symptoms of COVID-19</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>NCT04458519</p></td><td align=\"left\"><p>Probiorinse</p></td><td align=\"left\"><p>Reduction of severity of COVID-19 symptoms</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>NCT04390477</p></td><td align=\"left\"><p>Probiotic</p></td><td align=\"left\"><p>Effect of probiotic on COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>NCT04366180</p></td><td align=\"left\"><p><italic>L. coryniformis</italic> K8</p></td><td align=\"left\"><p>Effect of probiotic in the incidence and prevention of COVID-19 infection in health workers</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>NCT04734886</p></td><td align=\"left\"><p><italic>L. reuteri</italic> DSM 17,938</p></td><td align=\"left\"><p>Impact of probiotic supplementation on SARS-CoV-2 specific antibody response following COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>NCT04666116</p></td><td align=\"left\"><p>Dietary supplements including probiotics</p></td><td align=\"left\"><p>To check the changes in viral load in COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>NCT04847349</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Efficacy of probiotic intervention to boost the immunity in unvaccinated people infected previously with SARS-CoV-2</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>NCT04420676</p></td><td align=\"left\"><p>Synbiotic (Omnibiotic AAD)</p></td><td align=\"left\"><p>To check the ability to reduce gastrointestinal problems in COVID-19 patients</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>NCT04813718</p></td><td align=\"left\"><p>Omni-Biotic Pro Vi 5</p></td><td align=\"left\"><p>Analysis of post-Covid syndrome</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>NCT04399252</p></td><td align=\"left\"><p><italic>L. rhamnosus</italic> GG</p></td><td align=\"left\"><p>In order to study the consequence of microbiome in COVID-19 exposed household contacts</p></td></tr></tbody></table></table-wrap></p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author's contribution</title><p>KKV, VN, AM, KKB, and MPA: Collecting articles and Writing original draft. RS: Conceptualization, Methodology, Writing - original draft. MKA, SV: Writing, Reviewing and Editing, Formal analysis. PB: Project Administration, Conceptualization and Visualization.</p></sec><sec><title>Funding</title><p>(Information that explains whether and by whom the research was supported): No funding to mention and no funding received for the particular work mentioned in the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec><p>The authors declare that the work described has not been published before (except in the form of an abstract, a published lecture or academic thesis),  it is not under consideration for publication elsewhere, its submission to JFST publication has been approved by all authors as well as the responsible authorities – tacitly or explicitly – at the institute where the work has been carried out, if accepted, it will not be published elsewhere in the same form, in English or in any other language, including electronically without the written consent of the copyright holder, and  JFST will not be held legally responsible should there be any claims for compensation or dispute on authorship.</p></sec><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Acquah</surname><given-names>C</given-names></name><name><surname>Agyei</surname><given-names>D</given-names></name><name><surname>Obeng</surname><given-names>EM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aptamers: an emerging class of bioaffinity ligands in bioactive peptide applications</article-title><source>Crit Rev Food Sci Nutr</source><year>2020</year><volume>60</volume><fpage>1195</fpage><lpage>1206</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVCms7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10408398.2018.1564234</pub-id><pub-id pub-id-type=\"pmid\">30714390</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akour</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Probiotics and COVID-19: is there any link?</article-title><source>Lett Appl Microbiol</source><year>2020</year><volume>71</volume><fpage>229</fpage><lpage>234</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1ShtLrJ</pub-id><pub-id pub-id-type=\"doi\">10.1111/lam.13334</pub-id><pub-id pub-id-type=\"pmid\">32495940</pub-id><pub-id pub-id-type=\"pmcid\">7300613</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alkhatib</surname><given-names>A</given-names></name><name><surname>Tsang</surname><given-names>C</given-names></name><name><surname>Tuomilehto</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Olive oil nutraceuticals in the prevention and management of diabetes: from molecules to lifestyle</article-title><source>Int J Mol Sci</source><year>2018</year><volume>19</volume><fpage>2021</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisFKltb7J</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms19072024</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Byun</surname><given-names>M-S</given-names></name><name><surname>Yu</surname><given-names>O-K</given-names></name><name><surname>Cha</surname><given-names>Y-S</given-names></name><name><surname>Park</surname><given-names>T-S</given-names></name></person-group><article-title xml:lang=\"en\">Korean traditional Chungkookjang improves body composition, lipid profiles and atherogenic indices in overweight/obese subjects: a double-blind, randomized, crossover, placebo-controlled clinical trial</article-title><source>Eur J Clin Nutr</source><year>2016</year><volume>70</volume><fpage>1116</fpage><lpage>1122</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtVWgu77K</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2016.77</pub-id><pub-id pub-id-type=\"pmid\">27302672</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Calder</surname><given-names>PC</given-names></name><name><surname>Kew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The immune system: a target for functional foods?</article-title><source>Br J Nutr</source><year>2002</year><volume>88</volume><fpage>S165</fpage><lpage>S176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XovFOlsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1079/BJN2002682</pub-id><pub-id pub-id-type=\"pmid\">12495459</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Carrasco-Hernandez</surname><given-names>R</given-names></name><name><surname>Jácome</surname><given-names>R</given-names></name><name><surname>López Vidal</surname><given-names>Y</given-names></name><name><surname>Ponce de León</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Are RNA viruses candidate agents for the next global pandemic? a review</article-title><source>ILAR J</source><year>2017</year><volume>58</volume><fpage>343</fpage><lpage>358</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVCmtbjK</pub-id><pub-id pub-id-type=\"doi\">10.1093/ilar/ilx026</pub-id><pub-id pub-id-type=\"pmid\">28985316</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>Q</given-names></name><name><surname>Giovannucci</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Dairy consumption and risk of type 2 diabetes: 3 cohorts of US adults and an updated meta-analysis</article-title><source>BMC Med</source><year>2014</year><volume>12</volume><fpage>215</fpage><pub-id pub-id-type=\"doi\">10.1186/s12916-014-0215-1</pub-id><pub-id pub-id-type=\"pmid\">25420418</pub-id><pub-id pub-id-type=\"pmcid\">4243376</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>X</given-names></name><name><surname>Wei</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Isolation, purification and the anti-hypertensive effect of a novel angiotensin I-converting enzyme (ACE) inhibitory peptide from <italic>Ruditapes philippinarum</italic> fermented with <italic>Bacillus natto</italic></article-title><source>Food Funct</source><year>2018</year><volume>9</volume><fpage>5230</fpage><lpage>5237</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFOgsLnL</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8FO01146J</pub-id><pub-id pub-id-type=\"pmid\">30206615</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chiba</surname><given-names>E</given-names></name><name><surname>Tomosada</surname><given-names>Y</given-names></name><name><surname>Vizoso-Pinto</surname><given-names>MG</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Immunobiotic Lactobacillus rhamnosus improves resistance of infant mice against respiratory syncytial virus infection</article-title><source>Int Immunopharmacol</source><year>2013</year><volume>17</volume><fpage>373</fpage><lpage>382</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtlGqtrbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.intimp.2013.06.024</pub-id><pub-id pub-id-type=\"pmid\">23838113</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choi</surname><given-names>H-J</given-names></name><name><surname>Song</surname><given-names>J-H</given-names></name><name><surname>Park</surname><given-names>K-S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of yogurt against enterovirus 71 in vero cells</article-title><source>Food Sci Biotechnol</source><year>2010</year><volume>19</volume><fpage>289</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGisLzE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-010-0042-x</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Davidson</surname><given-names>LE</given-names></name><name><surname>Fiorino</surname><given-names>A-M</given-names></name><name><surname>Snydman</surname><given-names>DR</given-names></name><name><surname>Hibberd</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Lactobacillus GG as an immune adjuvant for live-attenuated influenza vaccine in healthy adults: a randomized double-blind placebo-controlled trial</article-title><source>Eur J Clin Nutr</source><year>2011</year><volume>65</volume><fpage>501</fpage><lpage>507</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXkt1Wlt7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2010.289</pub-id><pub-id pub-id-type=\"pmid\">21285968</pub-id><pub-id pub-id-type=\"pmcid\">3071884</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Leo</surname><given-names>A</given-names></name><name><surname>Arena</surname><given-names>G</given-names></name><name><surname>Lacanna</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Resveratrol inhibits Epstein Barr virus lytic cycle in Burkitt’s lymphoma cells by affecting multiple molecular targets</article-title><source>Antiviral Res</source><year>2012</year><volume>96</volume><fpage>196</fpage><lpage>202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhs12gtLfE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2012.09.003</pub-id><pub-id pub-id-type=\"pmid\">22985630</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>García-Burgos</surname><given-names>M</given-names></name><name><surname>Moreno-Fernández</surname><given-names>J</given-names></name><name><surname>Alférez</surname><given-names>MJM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">New perspectives in fermented dairy products and their health relevance</article-title><source>J Funct Foods</source><year>2020</year><volume>72</volume><fpage>104059</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylu7vM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jff.2020.104059</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ghanem</surname><given-names>KZ</given-names></name><name><surname>Mahran</surname><given-names>MZ</given-names></name><name><surname>Ramadan</surname><given-names>MM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">A comparative study on flavour components and therapeutic properties of unfermented and fermented defatted soybean meal extract</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>5998</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSlur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-62907-x</pub-id><pub-id pub-id-type=\"pmid\">32265465</pub-id><pub-id pub-id-type=\"pmcid\">7138831</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Prasad</surname><given-names>J</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of natural and acquired immunity by Lactobacillus rhamnosus (HN001), Lactobacillus acidophilus (HN017) and Bifidobacterium lactis (HN019)</article-title><source>Br J Nutr</source><year>2000</year><volume>83</volume><fpage>167</fpage><lpage>176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXhsVymur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114500000210</pub-id><pub-id pub-id-type=\"pmid\">10743496</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Cross</surname><given-names>ML</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of immunity in the elderly by dietary supplementation with the probiotic Bifidobacterium lactis HN019</article-title><source>Am J Clin Nutr</source><year>2001</year><volume>74</volume><fpage>833</fpage><lpage>839</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXovVSgtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1093/ajcn/74.6.833</pub-id><pub-id pub-id-type=\"pmid\">11722966</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grandy</surname><given-names>G</given-names></name><name><surname>Medina</surname><given-names>M</given-names></name><name><surname>Soria</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in the treatment of acute rotavirus diarrhoea. a randomized, double-blind, controlled trial using two different probiotic preparations in Bolivian children</article-title><source>BMC Infect Dis</source><year>2010</year><volume>10</volume><fpage>253</fpage><pub-id pub-id-type=\"doi\">10.1186/1471-2334-10-253</pub-id><pub-id pub-id-type=\"pmid\">20735858</pub-id><pub-id pub-id-type=\"pmcid\">2940902</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grant</surname><given-names>W</given-names></name><name><surname>Lahore</surname><given-names>H</given-names></name><name><surname>McDonnell</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Evidence that vitamin D supplementation could reduce risk of influenza and COVID-19 infections and deaths</article-title><source>Nutrients</source><year>2020</year><volume>12</volume><fpage>988</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVemsbnK</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu12040988</pub-id><pub-id pub-id-type=\"pmid\">32252338</pub-id><pub-id pub-id-type=\"pmcid\">7231123</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guan</surname><given-names>W</given-names></name><name><surname>Ni</surname><given-names>Z</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical characteristics of coronavirus disease 2019 in China</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1708</fpage><lpage>1720</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSjsb8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMoa2002032</pub-id><pub-id pub-id-type=\"pmid\">32109013</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gui</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>A</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effects of probiotic supplementation on natural killer cell function in healthy elderly individuals: a meta-analysis of randomized controlled trials</article-title><source>Eur J Clin Nutr</source><year>2020</year><volume>74</volume><fpage>1630</fpage><lpage>1637</lpage><pub-id pub-id-type=\"doi\">10.1038/s41430-020-0670-z</pub-id><pub-id pub-id-type=\"pmid\">32514029</pub-id><pub-id pub-id-type=\"pmcid\">7279433</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guillemard</surname><given-names>E</given-names></name><name><surname>Tondu</surname><given-names>F</given-names></name><name><surname>Lacoin</surname><given-names>F</given-names></name><name><surname>Schrezenmeir</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Consumption of a fermented dairy product containing the probiotic Lactobacillus casei DN-114 001 reduces the duration of respiratory infections in the elderly in a randomised controlled trial</article-title><source>Br J Nutr</source><year>2010</year><volume>103</volume><fpage>58</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhs1Smt73N</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114509991395</pub-id><pub-id pub-id-type=\"pmid\">19747410</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z-D</given-names></name><name><surname>Wang</surname><given-names>Z-Y</given-names></name><name><surname>Zhang</surname><given-names>S-F</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface distribution of severe acute respiratory syndrome coronavirus 2 in hospital wards, Wuhan, China, 2020</article-title><source>Emerg Infect Dis</source><year>2020</year><volume>26</volume><fpage>1583</fpage><lpage>1591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlOlsbrF</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid2607.200885</pub-id><pub-id pub-id-type=\"pmid\">32275497</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hamida</surname><given-names>RS</given-names></name><name><surname>Shami</surname><given-names>A</given-names></name><name><surname>Ali</surname><given-names>MA</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Kefir: a protective dietary supplementation against viral infection</article-title><source>Biomed Pharmacother</source><year>2021</year><volume>133</volume><fpage>110974</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlynsrbL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biopha.2020.110974</pub-id><pub-id pub-id-type=\"pmid\">33186795</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"other\">Hayes M, García-Vaquero M (2016) Bioactive Compounds from Fermented Food Products. pp&amp;nbsp;293–310</mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Horowitz</surname><given-names>RI</given-names></name><name><surname>Freeman</surname><given-names>PR</given-names></name><name><surname>Bruzzese</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Efficacy of glutathione therapy in relieving dyspnea associated with COVID-19 pneumonia: a report of 2 cases</article-title><source>Respiratory Med Case Rep</source><year>2020</year><volume>30</volume><fpage>101063</fpage><pub-id pub-id-type=\"doi\">10.1016/j.rmcr.2020.101063</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Lin</surname><given-names>W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Review article: probiotics, prebiotics and dietary approaches during COVID-19 pandemic</article-title><source>Trends Food Sci Technol</source><year>2021</year><volume>108</volume><fpage>187</fpage><lpage>196</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVyrsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2020.12.009</pub-id><pub-id pub-id-type=\"pmid\">33519087</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ivec</surname><given-names>M</given-names></name><name><surname>Botić</surname><given-names>T</given-names></name><name><surname>Koren</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Interactions of macrophages with probiotic bacteria lead to increased antiviral response against vesicular stomatitis virus</article-title><source>Antiviral Res</source><year>2007</year><volume>75</volume><fpage>266</fpage><lpage>274</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXmsFKht7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2007.03.013</pub-id><pub-id pub-id-type=\"pmid\">17512614</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ji</surname><given-names>L-N</given-names></name><name><surname>Chao</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>Y-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical features of pediatric patients with COVID-19: a report of two family cluster cases</article-title><source>World J Pediatr</source><year>2020</year><volume>16</volume><fpage>267</fpage><lpage>270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXltVSisbc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12519-020-00356-2</pub-id><pub-id pub-id-type=\"pmid\">32180140</pub-id><pub-id pub-id-type=\"pmcid\">7091281</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Lee</surname><given-names>Y-T</given-names></name><name><surname>Ngo</surname><given-names>V</given-names></name><name><surname>Le</surname></name><etal/></person-group><article-title xml:lang=\"en\">Heat-killed Lactobacillus casei confers broad protection against influenza A virus primary infection and develops heterosubtypic immunity against future secondary infection</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><fpage>17360</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGitLbF</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-017-17487-8</pub-id><pub-id pub-id-type=\"pmid\">29234060</pub-id><pub-id pub-id-type=\"pmcid\">5727132</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kanmani</surname><given-names>P</given-names></name><name><surname>Albarracin</surname><given-names>L</given-names></name><name><surname>Kobayashi</surname><given-names>H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">AGenomic characterization of Lactobacillus delbrueckii TUA4408L and evaluation of the antiviral activities of its extracellular polysaccharides in porcine intestinal epithelial cells</article-title><source>Front Immunol</source><year>2018</year><volume>9</volume><fpage>27</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXnslyrsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2018.02178</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>S</given-names></name><name><surname>Jangra</surname><given-names>G</given-names></name><name><surname>Kundu</surname><given-names>V</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Anti-viral activity of Zingiber officinale (Ginger) ingredients against the Chikungunya virus</article-title><source>VirusDisease</source><year>2020</year><volume>31</volume><fpage>270</fpage><lpage>276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38Xit1yqurjO</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13337-020-00584-0</pub-id><pub-id pub-id-type=\"pmid\">32420412</pub-id><pub-id pub-id-type=\"pmcid\">7223110</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalil</surname><given-names>E</given-names></name><name><surname>Abd Manap</surname><given-names>M</given-names></name><name><surname>Mustafa</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotic properties of exopolysaccharide-producing lactobacillus strains isolated from tempoyak</article-title><source>Molecules</source><year>2018</year><volume>23</volume><fpage>398</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhs1Kjt7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules23020398</pub-id><pub-id pub-id-type=\"pmid\">29438288</pub-id><pub-id pub-id-type=\"pmcid\">6017292</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>E-H</given-names></name><name><surname>Kim</surname><given-names>S-W</given-names></name><name><surname>Park</surname><given-names>S-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Greater efficacy of black ginseng (CJ EnerG) over red ginseng against lethal Influenza A virus infection</article-title><source>Nutrients</source><year>2019</year><volume>11</volume><fpage>1879</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVGmtbjP</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu11081879</pub-id><pub-id pub-id-type=\"pmid\">31412594</pub-id><pub-id pub-id-type=\"pmcid\">6723933</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>Y-I</given-names></name><name><surname>Kim</surname><given-names>S-G</given-names></name><name><surname>Kim</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Infection and rapid transmission of SARS-CoV-2 in Ferrets</article-title><source>Cell Host Microbe</source><year>2020</year><volume>27</volume><fpage>704</fpage><lpage>709e2</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsl2qsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chom.2020.03.023</pub-id><pub-id pub-id-type=\"pmid\">32259477</pub-id><pub-id pub-id-type=\"pmcid\">7144857</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kinoshita</surname><given-names>T</given-names></name><name><surname>Maruyama</surname><given-names>K</given-names></name><name><surname>Suyama</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of OLL1073R-1 yogurt intake on influenza incidence and immunological markers among women healthcare workers: a randomized controlled trial</article-title><source>Food Funct</source><year>2019</year><volume>10</volume><fpage>8129</fpage><lpage>8136</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVaksLrI</pub-id><pub-id pub-id-type=\"doi\">10.1039/C9FO02128K</pub-id><pub-id pub-id-type=\"pmid\">31738351</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kokubo</surname><given-names>T</given-names></name><name><surname>Komano</surname><given-names>Y</given-names></name><name><surname>Tsuji</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of plasmacytoid dendritic cell-Stimulative lactic acid bacteria, lactococcus lactis strain plasma, on exercise-Induced fatigue and recovery via immunomodulatory action</article-title><source>Int J Sport Nutr Exerc Metab</source><year>2019</year><volume>25</volume><fpage>1</fpage><lpage>5</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmt1art74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1123/ijsnem.2018-0377</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kurian</surname><given-names>SJ</given-names></name><name><surname>Unnikrishnan</surname><given-names>MK</given-names></name><name><surname>Miraj</surname><given-names>SS</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in prevention and treatment of COVID-19: current perspective and future prospects</article-title><source>Arch Med Res</source><year>2021</year><volume>52</volume><fpage>582</fpage><lpage>594</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFehu74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.arcmed.2021.03.002</pub-id><pub-id pub-id-type=\"pmid\">33785208</pub-id><pub-id pub-id-type=\"pmcid\">7972717</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kwak</surname><given-names>M-K</given-names></name><name><surname>Liu</surname><given-names>R</given-names></name><name><surname>Kwon</surname><given-names>J-O</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Cyclic dipeptides from lactic acid bacteria inhibit proliferation of the influenza a virus</article-title><source>J Microbiol</source><year>2013</year><volume>51</volume><fpage>836</fpage><lpage>843</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXksVajsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12275-013-3521-y</pub-id><pub-id pub-id-type=\"pmid\">24385362</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Laatikainen</surname><given-names>R</given-names></name><name><surname>Koskenpato</surname><given-names>J</given-names></name><name><surname>Hongisto</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Randomised clinical trial: low-FODMAP rye bread vs. regular rye bread to relieve the symptoms of irritable bowel syndrome</article-title><source>Aliment Pharmacol Ther</source><year>2016</year><volume>44</volume><fpage>460</fpage><lpage>470</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KktbjL</pub-id><pub-id pub-id-type=\"doi\">10.1111/apt.13726</pub-id><pub-id pub-id-type=\"pmid\">27417338</pub-id><pub-id pub-id-type=\"pmcid\">5113694</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>C-C</given-names></name><name><surname>Shih</surname><given-names>T-P</given-names></name><name><surname>Ko</surname><given-names>W-C</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronavirus disease-2019 (COVID-19): The epidemic and the challenges</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><fpage>105924</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ogtr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105924</pub-id><pub-id pub-id-type=\"pmid\">32081636</pub-id><pub-id pub-id-type=\"pmcid\">7127800</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lam</surname><given-names>TT-Y</given-names></name><name><surname>Jia</surname><given-names>N</given-names></name><name><surname>Zhang</surname><given-names>Y-W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Identifying SARS-CoV-2-related coronaviruses in Malayan pangolins</article-title><source>Nature</source><year>2020</year><volume>583</volume><fpage>282</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCnu77L</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-020-2169-0</pub-id><pub-id pub-id-type=\"pmid\">32218527</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>MH</given-names></name><name><surname>Yoo</surname><given-names>S-H</given-names></name><name><surname>Ha</surname><given-names>S-D</given-names></name><name><surname>Choi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Inactivation of feline calicivirus and murine norovirus during Dongchimi fermentation</article-title><source>Food Microbiol</source><year>2012</year><volume>31</volume><fpage>210</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFKgsLw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fm.2012.04.002</pub-id><pub-id pub-id-type=\"pmid\">22608225</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>YJ</given-names></name><name><surname>Yoo</surname><given-names>HJ</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Consumption of dairy yogurt containing Lactobacillus paracasei ssp. paracasei, Bifidobacterium animalis ssp. lactis and heat-treated Lactobacillus plantarum improves immune function including natural killer cell activity</article-title><source>Nutrients</source><year>2017</year><volume>9</volume><fpage>558</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVersbjI</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu9060558</pub-id><pub-id pub-id-type=\"pmid\">28561762</pub-id><pub-id pub-id-type=\"pmcid\">5490537</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Whon</surname><given-names>TW</given-names></name><name><surname>Roh</surname><given-names>SW</given-names></name><name><surname>Jeon</surname><given-names>CO</given-names></name></person-group><article-title xml:lang=\"en\">Unraveling microbial fermentation features in kimchi: from classical to meta-omics approaches</article-title><source>Appl Microbiol Biotechnol</source><year>2020</year><volume>104</volume><fpage>7731</fpage><lpage>7744</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsFejurbJ</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00253-020-10804-8</pub-id><pub-id pub-id-type=\"pmid\">32749526</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Gayle</surname><given-names>AA</given-names></name><name><surname>Wilder-Smith</surname><given-names>A</given-names></name><name><surname>Rocklöv</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The reproductive number of COVID-19 is higher compared to SARS coronavirus</article-title><source>J Travel Med</source><year>2020</year><volume>27</volume><fpage>98</fpage><lpage>1006</lpage><pub-id pub-id-type=\"doi\">10.1093/jtm/taaa021</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Ikegami</surname><given-names>S</given-names></name><name><surname>Kume</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Reducing the risk of infection in the elderly by dietary intake of yoghurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>Br J Nutr</source><year>2010</year><volume>104</volume><fpage>998</fpage><lpage>1006</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1GjsLfO</pub-id><pub-id pub-id-type=\"doi\">10.1017/S000711451000173X</pub-id><pub-id pub-id-type=\"pmid\">20487575</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Sato</surname><given-names>A</given-names></name><name><surname>Goto</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Enhanced natural killer cell activation by exopolysaccharides derived from yogurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>J Dairy Sci</source><year>2016</year><volume>99</volume><fpage>915</fpage><lpage>923</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXitVSksLbE</pub-id><pub-id pub-id-type=\"doi\">10.3168/jds.2015-10376</pub-id><pub-id pub-id-type=\"pmid\">26686726</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marco</surname><given-names>ML</given-names></name><name><surname>Heeney</surname><given-names>D</given-names></name><name><surname>Binda</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Health benefits of fermented foods: microbiota and beyond</article-title><source>Curr Opin Biotechnol</source><year>2017</year><volume>44</volume><fpage>94</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhvFKjs7zF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.copbio.2016.11.010</pub-id><pub-id pub-id-type=\"pmid\">27998788</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McGurnaghan</surname><given-names>SJ</given-names></name><name><surname>Weir</surname><given-names>A</given-names></name><name><surname>Bishop</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Risks of and risk factors for COVID-19 disease in people with diabetes: a cohort study of the total population of Scotland</article-title><source>Lancet Diabetes Endocrinol</source><year>2021</year><volume>9</volume><fpage>82</fpage><lpage>93</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXks1aqsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-8587(20)30405-8</pub-id><pub-id pub-id-type=\"pmid\">33357491</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mirashrafi</surname><given-names>S</given-names></name><name><surname>Moravejolahkami</surname><given-names>AR</given-names></name><name><surname>Balouch Zehi</surname><given-names>Z</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The efficacy of probiotics on virus titres and antibody production in virus diseases: a systematic review on recent evidence for COVID-19 treatment</article-title><source>Clin Nutr ESPEN</source><year>2021</year><volume>46</volume><fpage>1</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.1016/j.clnesp.2021.10.016</pub-id><pub-id pub-id-type=\"pmid\">34857182</pub-id><pub-id pub-id-type=\"pmcid\">8539817</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mora</surname><given-names>L</given-names></name><name><surname>Escudero</surname><given-names>E</given-names></name><name><surname>Aristoy</surname><given-names>M-C</given-names></name><name><surname>Toldrá</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">A peptidomic approach to study the contribution of added casein proteins to the peptide profile in Spanish dry-fermented sausages</article-title><source>Int J Food Microbiol</source><year>2015</year><volume>212</volume><fpage>41</fpage><lpage>48</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtVKhtbjM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijfoodmicro.2015.05.022</pub-id><pub-id pub-id-type=\"pmid\">26116420</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morita</surname><given-names>N</given-names></name><name><surname>Umemoto</surname><given-names>E</given-names></name><name><surname>Fujita</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">GPR31-dependent dendrite protrusion of intestinal CX3CR1 + cells by bacterial metabolites</article-title><source>Nature</source><year>2019</year><volume>566</volume><fpage>110</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmtVyjtL0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-019-0884-1</pub-id><pub-id pub-id-type=\"pmid\">30675063</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mounce</surname><given-names>BC</given-names></name><name><surname>Cesaro</surname><given-names>T</given-names></name><name><surname>Carrau</surname><given-names>L</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Curcumin inhibits Zika and chikungunya virus infection by inhibiting cell binding</article-title><source>Antiviral Res</source><year>2017</year><volume>142</volume><fpage>148</fpage><lpage>157</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsV2jtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2017.03.014</pub-id><pub-id pub-id-type=\"pmid\">28343845</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muhialdin</surname><given-names>BJ</given-names></name><name><surname>Zawawi</surname><given-names>N</given-names></name><name><surname>Abdull Razis</surname><given-names>AF</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented foods and their probiotics bacteria towards respiratory and alimentary tracts viruses</article-title><source>Food Control</source><year>2021</year><volume>127</volume><fpage>108140</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVSmtLfF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2021.108140</pub-id><pub-id pub-id-type=\"pmid\">33867696</pub-id><pub-id pub-id-type=\"pmcid\">8036130</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olaimat</surname><given-names>AN</given-names></name><name><surname>Aolymat</surname><given-names>I</given-names></name><name><surname>Al-Holy</surname><given-names>M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The potential application of probiotics and prebiotics for the prevention and treatment of COVID-19</article-title><source>npj Sci Food</source><year>2020</year><volume>4</volume><fpage>17</fpage><pub-id pub-id-type=\"doi\">10.1038/s41538-020-00078-9</pub-id><pub-id pub-id-type=\"pmid\">33083549</pub-id><pub-id pub-id-type=\"pmcid\">7536434</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivares</surname><given-names>M</given-names></name><name><surname>Díaz-Ropero</surname><given-names>MP</given-names></name><name><surname>Sierra</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Oral intake of Lactobacillus fermentum CECT5716 enhances the effects of influenza vaccination</article-title><source>Nutrition</source><year>2007</year><volume>23</volume><fpage>254</fpage><lpage>260</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXislWgurg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nut.2007.01.004</pub-id><pub-id pub-id-type=\"pmid\">17352961</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ong</surname><given-names>SWX</given-names></name><name><surname>Tan</surname><given-names>YK</given-names></name><name><surname>Chia</surname><given-names>PY</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Air, surface environmental, and personal protective equipment contamination by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from a symptomatic patient</article-title><source>JAMA</source><year>2020</year><volume>323</volume><fpage>1610</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSrtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.3227</pub-id><pub-id pub-id-type=\"pmid\">32129805</pub-id><pub-id pub-id-type=\"pmcid\">7057172</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parada Venegas</surname><given-names>D</given-names></name><name><surname>De la Fuente</surname><given-names>MK</given-names></name><name><surname>Landskron</surname><given-names>G</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Short chain fatty acids (SCFAs)-mediated gut epithelial and immune regulation and its relevance for inflammatory bowel diseases</article-title><source>Front Immunol</source><year>2019</year><volume>10</volume><fpage>107</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVWrtL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2019.00277</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peters</surname><given-names>A</given-names></name><name><surname>Krumbholz</surname><given-names>P</given-names></name><name><surname>Jäger</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Metabolites of lactic acid bacteria present in fermented foods are highly potent agonists of human hydroxycarboxylic acid receptor 3</article-title><source>PLOS Genet</source><year>2019</year><volume>15</volume><fpage>e1008145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhvFaqsb%2FN</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pgen.1008145</pub-id><pub-id pub-id-type=\"pmid\">31120900</pub-id><pub-id pub-id-type=\"pmcid\">6532841</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>przeLe</surname><given-names>TT</given-names></name><name><surname>Cramer</surname><given-names>JP</given-names></name><name><surname>Chen</surname><given-names>R</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Evolution of the COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><fpage>667</fpage><lpage>668</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVylsb3M</pub-id><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00151-8</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pyankov</surname><given-names>OV</given-names></name><name><surname>Bodnev</surname><given-names>SA</given-names></name><name><surname>Pyankova</surname><given-names>OG</given-names></name><name><surname>Agranovski</surname><given-names>IE</given-names></name></person-group><article-title xml:lang=\"en\">Survival of aerosolized coronavirus in the ambient air</article-title><source>J Aerosol Sci</source><year>2018</year><volume>115</volume><fpage>158</fpage><lpage>163</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVyit7jN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jaerosci.2017.09.009</pub-id><pub-id pub-id-type=\"pmid\">32226116</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Raji</surname><given-names>MNA</given-names></name><name><surname>Ab Karim</surname><given-names>S</given-names></name><name><surname>Ishak</surname><given-names>FAC</given-names></name><name><surname>Arshad</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Past and present practices of the Malay food heritage and culture in Malaysia</article-title><source>J Ethnic Foods</source><year>2017</year><volume>4</volume><fpage>221</fpage><lpage>231</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jef.2017.11.001</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Santarpia</surname><given-names>JL</given-names></name><name><surname>Rivera</surname><given-names>DN</given-names></name><name><surname>Herrera</surname><given-names>VL</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface contamination of SARS-CoV-2 observed in quarantine and isolation care</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>12732</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVyku7rM</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-69286-3</pub-id><pub-id pub-id-type=\"pmid\">32728118</pub-id><pub-id pub-id-type=\"pmcid\">7391640</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Smith</surname><given-names>TJ</given-names></name><name><surname>Rigassio-Radler</surname><given-names>D</given-names></name><name><surname>Denmark</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effect of Lactobacillus rhamnosus LGG ® and Bifidobacterium animalis ssp. lactis BB-12 ® on health-related quality of life in college students affected by upper respiratory infections</article-title><source>Br J Nutr</source><year>2013</year><volume>109</volume><fpage>1999</fpage><lpage>2007</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXnvVGhsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114512004138</pub-id><pub-id pub-id-type=\"pmid\">23020819</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"other\">Takeda K, Okumura K (2007) Effects of a Fermented Milk Drink Containing Lactobacillus casei Strain Shirota on the Human NK-Cell Activity. J Nutr 137. <ext-link ext-link-type=\"doi\" xlink:href=\"10.1093/jn/137.3.791S\">https://doi.org/10.1093/jn/137.3.791S</ext-link>. :791S-793S</mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tamang</surname><given-names>JP</given-names></name><name><surname>Cotter</surname><given-names>PD</given-names></name><name><surname>Endo</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Fermented foods in a global age: east meets West</article-title><source>Compr Rev Food Sci Food Saf</source><year>2020</year><volume>19</volume><fpage>184</fpage><lpage>217</lpage><pub-id pub-id-type=\"doi\">10.1111/1541-4337.12520</pub-id><pub-id pub-id-type=\"pmid\">33319517</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>Bushmaker</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>DH</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface stability of SARS-CoV-2 as compared with SARS-CoV-1</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1564</fpage><lpage>1567</lpage><pub-id pub-id-type=\"doi\">10.1056/NEJMc2004973</pub-id><pub-id pub-id-type=\"pmid\">32182409</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Kim</surname><given-names>K-H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented ginseng extracts against a broad range of Influenza viruses</article-title><source>Viruses</source><year>2018</year><volume>10</volume><fpage>471</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXntF2gsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/v10090471</pub-id><pub-id pub-id-type=\"pmid\">30200514</pub-id><pub-id pub-id-type=\"pmcid\">6164659</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wei</surname><given-names>L</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>H</given-names></name><name><surname>Wen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Engineering Scheffersomyces stipitis for fumaric acid production from xylose</article-title><source>Bioresour Technol</source><year>2015</year><volume>187</volume><fpage>246</fpage><lpage>254</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXlvVems74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2015.03.122</pub-id><pub-id pub-id-type=\"pmid\">25863201</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wiersinga</surname><given-names>WJ</given-names></name><name><surname>Rhodes</surname><given-names>A</given-names></name><name><surname>Cheng</surname><given-names>AC</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Pathophysiology, transmission, diagnosis, and treatment of coronavirus disease 2019 (COVID-19)</article-title><source>JAMA</source><year>2020</year><volume>324</volume><fpage>782</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslamu73N</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.12839</pub-id><pub-id pub-id-type=\"pmid\">32648899</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>N</given-names></name><name><surname>Qiao</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Application of metabolic controls for the maximization of lipid production in semicontinuous fermentation</article-title><source>Proc Natl Acad Sci USA</source><year>2017</year><volume>114</volume><fpage>E5308</fpage><lpage>E5316</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWmtLvI</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1703321114</pub-id><pub-id pub-id-type=\"pmid\">28630295</pub-id><pub-id pub-id-type=\"pmcid\">5502627</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral effects of a probiotic metabolic products against transmissible gastroenteritis coronavirus</article-title><source>J Probiotics Heal</source><year>2017</year><volume>5</volume><fpage>543</fpage><pub-id pub-id-type=\"doi\">10.4172/2329-8901.1000184</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yin</surname><given-names>SY</given-names></name><name><surname>Kim</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>H-J</given-names></name></person-group><article-title xml:lang=\"en\">Protective effect of dietary xylitol on Influenza A virus infection</article-title><source>PLoS One</source><year>2014</year><volume>9</volume><fpage>e84633</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXltlOit78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0084633</pub-id><pub-id pub-id-type=\"pmid\">24392148</pub-id><pub-id pub-id-type=\"pmcid\">3879333</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec14\"><title>Supplementary Information</title><p id=\"Par35\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/13197_2022_5528_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 556 kb)</p></caption></media></supplementary-material></p></sec></app></app-group><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-023-01258-5</article-id><article-id pub-id-type=\"manuscript\">1258</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-023-01258-5</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Detection method for genetically modified potato using an ultra-fast PCR system</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-0608-6354</contrib-id><name><surname>Shin</surname><given-names>Min Ki</given-names></name><address><email>mkhj1212@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs10068023012585_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jeon</surname><given-names>Seon Min</given-names></name><address><email>jsm0065@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Koo</surname><given-names>Yong Eui</given-names></name><address><email>kye2020@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.31501.36</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 5905</institution-id><institution content-type=\"org-division\">Interdisciplinary Program in Bioengineering</institution><institution content-type=\"org-name\">Seoul National University</institution></institution-wrap><addr-line content-type=\"postcode\">08826</addr-line><addr-line content-type=\"city\">Seoul</addr-line><country country=\"KR\">Republic of Korea</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.467691.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1773 0675</institution-id><institution content-type=\"org-division\">Food Safety Risk Assessment Department</institution><institution content-type=\"org-name\">National Institute of Food and Drug Safety Evaluation</institution></institution-wrap><addr-line content-type=\"postcode\">28159</addr-line><addr-line content-type=\"city\">Cheongju</addr-line><country country=\"KR\">Republic of Korea</country></aff></contrib-group><author-notes><corresp id=\"IDs10068023012585_cor1\"><label>a</label><email>mkhj1212@korea.kr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>1</day><month>2</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"4\">9</issue><fpage>1185</fpage><lpage>1191</lpage><history><date date-type=\"registration\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"received\"><day>24</day><month>10</month><year>2022</year></date><date date-type=\"rev-recd\"><day>27</day><month>12</month><year>2022</year></date><date date-type=\"accepted\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"online\"><day>1</day><month>2</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Genetically modified (GM) potatoes having resistance to insects and viral diseases, low reducing sugar contents, and black spots for high quality continue to be developed. However, no GM potato has been approved as food or feed in the Republic of Korea as the country adheres to a zero-tolerance policy to unauthorized genetically modified organisms (GMOs). When the self-sufficiency rate is low, a detection method to assess GMOs in crops or other products is necessary. Therefore, a rapid method for two GM potato events (SPS-Y9 and EH92-527-1) using an ultra-fast PCR (UF-PCR) system has been developed, and its specificity, sensitivity, and applicability were demonstrated. UF-PCR can decrease the runtime of PCR by more than half of that needed in conventional methods. However, UF-PCR is not a common method for GMO analysis. This rapid detection method may be useful for GMO analyses in field conditions.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Solanum tuberosum</italic></kwd><kwd>Event specific qualification</kwd><kwd>GM potato</kwd><kwd>UF-PCR</kwd><kwd>Monitoring</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution>National Institute of Food and Drug Safety Evaluation</institution><institution-id institution-id-type=\"doi\" vocab=\"open-funder-registry\">http://dx.doi.org/10.13039/501100003655</institution-id></institution-wrap></funding-source><award-id award-type=\"FundRef grant\">19161MFDS051</award-id><principal-award-recipient><name><surname>Koo</surname><given-names>Yong Eui</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>10</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2023_Article_1258.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Potato (<italic>Solanum tuberosum</italic>) is widely used as a food and snack ingredient and is one of the four major food crops, along with rice, wheat, and corn. As of 2017, approximately 390 million ton of potatoes had been produced worldwide in an area of ​​19.3 million hectares. The area for cultivation decreased by 3% compared to that in 2000; however, production increased by 20.3% during the same period owing to the increase in yield (FAOSTAT, <xref ref-type=\"bibr\" rid=\"CR8\">2019</xref>). The productivity is steadily improving with the development of potato varieties and production technology.</p><p id=\"Par3\">Genetically modified (GM) crops also contribute to the increase in productivity. Although commercialization has ceased, the first GM potato, the NewLeaf potato having a pest-resistance trait, was approved as food in 1995 (Jaccaud et al., <xref ref-type=\"bibr\" rid=\"CR13\">2003</xref>). Subsequently, various traits, such as resistance against insects, diseases, and modified product qualities, have been introduced into potato (Martinez-Prada et al., <xref ref-type=\"bibr\" rid=\"CR17\">2021</xref>). In 2022, according to the GM Approval Database (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.isaaa.org/gmapprovaldatabase/default.asp\">www.isaaa.org/gmapprovaldatabase/default.asp</ext-link>) of the International Service for the Acquisition of Agri-biotech Applications, fifty-one GM potato events have been developed and commercialized.</p><p id=\"Par4\">Potato production in Republic of Korea was approximately 0.63 million ton, importing 0.24 million ton in 2019, and it has continued to increase since 2010 (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/</ext-link>). Under these circumstances, the urge for use of GM potato is increasing; however, no GM potato varieties are currently approved as food and feed in Republic of Korea (MFDS, <xref ref-type=\"bibr\" rid=\"CR20\">2022</xref>).</p><p id=\"Par5\">In several countries, unauthorized genetically modified organisms (GMOs) are managed under zero-tolerance or low level presence polices (Fraiture et al., <xref ref-type=\"bibr\" rid=\"CR9\">2016</xref>), which necessitates the development of rapid and accurate detection methods to manage unauthorized GMOs.</p><p id=\"Par6\">Several studies have been conducted to develop methods to identify GM potatoes. Detection methods for NewLeaf, NewLeaf Y, NewLeaf Plus, PH05-026-0048, EH92-527-1, AM04-1020, SPS-E12, AV43-6-G7, and insertion <italic>Cry3A</italic> have been developed using conventional PCR (PCR), real-time PCR (qPCR) (Cote et al., <xref ref-type=\"bibr\" rid=\"CR5\">2005</xref>; Roh et al., <xref ref-type=\"bibr\" rid=\"CR22\">2004</xref>; Song et al., <xref ref-type=\"bibr\" rid=\"CR25\">2017</xref>; Yoo et al., <xref ref-type=\"bibr\" rid=\"CR29\">2013</xref>), loop-mediated isothermal amplification (Tu et al., <xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>), and droplet digital PCR (Gao et al., <xref ref-type=\"bibr\" rid=\"CR10\">2016</xref>). Recent gene detection assays are directed towards simpler and more cost-effective methods.</p><p id=\"Par7\">Most GMO detection methods using PCR or qPCR take approximately 40 min to 2 h or more including data analysis. However, the ultra-fast PCR (UF-PCR) system can decrease the run time of PCR. In addition, the use of UF-PCR system can reduce reagent usage by half. Owing to these advantages, the UF-PCR system is being considered for application in various analyses, such as forensic genotyping (Aboud et al., <xref ref-type=\"bibr\" rid=\"CR1\">2013</xref>), pathogen screening (Das et al., <xref ref-type=\"bibr\" rid=\"CR6\">2022</xref>; Kim et al., <xref ref-type=\"bibr\" rid=\"CR15\">2018</xref>; Lee et al., <xref ref-type=\"bibr\" rid=\"CR16\">2014</xref>), allergen analysis (Kim et al., <xref ref-type=\"bibr\" rid=\"CR14\">2019</xref>), detection of epidemic viruses like SARS CoV-2 (Chen et al., <xref ref-type=\"bibr\" rid=\"CR4\">2022</xref>; Milosevic et al., <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>), and authenticity determination (Hong et al., <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par8\">However, little research has been conducted in the field of GMO detection using the UF-PCR system (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). Here, we propose a reliable and rapid detection method for GM potatoes.</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Samples</title><p id=\"Par9\">Reference materials of GM soybean (A2704-12, A5547-127, CV127, DAS-44406-6, DAS-68416-4, DAS-81419-2, DP-305423-1, DP-356043-5, FG72, GMB151, RRS, MON87701, MON87705, MON87708, MON87751, MON87769, MON89788, and SYHT0H2), GM maize (3272, 5307, Bt10, Bt11, Bt176, DAS-40278-9, DAS-59122-7, DLL25, DP-004114-3, DP-023211-2, DP-098140-6, DP-202216-6, DP-915635-4, GA21, MIR162, MIR604, MZHG0JG, MZIR098, NK603, T25, TC1507, MON810, MON863, MON87403, MON87411, MON87419, MON87427, MON87429, MON87460, MON88017, MON89034, MON95379, and VCO-01981-5), GM canola (DP-073496-4, GT73, LBFLFK, MON88302, MON94100, MS8, MS11, NS-B50027-4, RF3, and T45), GM cotton (271/3006, COT102, DAS-81910-7, GHB119, GHB614, GHB811, LLcotton25, MON531, MON1445, MON15985, MON88701, MON88702, MON88913, and T304xGHB119), and GM potato (AM04-1020, EH92-527-1, PH05-026-0048, SPS-E12, and SPS-Y9) were kindly provided by the developers or purchased from the Institute for Reference Materials and Measurements (IRMM, Geel, Belgium) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Non-GM (NGM) seeds of soybean, maize, canola, cotton, rice, wheat, barley, and potato tissues were also used.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>GM reference materials used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event name</p></th><th align=\"left\"><p>Developer</p></th><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event Name</p></th><th align=\"left\"><p>Developer</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>RRS</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87411</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON89788</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>VCO-01981-5</p></td><td align=\"left\"><p>Genective</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A2704-12</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87403</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-356043-5</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87419</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-305423-1</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZHG0JG</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A5547-127</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZIR098</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87701</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-202216-6</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>CV127</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87429</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87769</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-023211-2</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87705</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON95379</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87708</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-915635-4</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>FG72</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON531</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>SYHT0H2</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON1445</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-44406-6</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON15985</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-68416-4</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>281/3006</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-81419-2</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>LLcotton25</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87751</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88913</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>GMB151</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB614</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON810</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>T304-40 × GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>GA21</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>COT102</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>TC1507</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88701</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>NK603</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON863</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>DAS-81910-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt11</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB811</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>T25</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88702</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt176</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>GT73</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DLL25</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>T45</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-59122-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON88302</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON88017</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS8</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR604</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>RF3</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt10</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>DP-073496-4</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON89034</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS11</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR162</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>LBFLFK</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-098140-6</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>NS-B50027-4</p></td><td align=\"left\"><p>NUSEED</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>3272</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON94100</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87460</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>5307</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>EH92-527-1</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87427</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-E12</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-40278-9</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>AM04-1020</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-004114-3</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>PH05-026-0048</p></td><td align=\"left\"><p>BASF</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec4\"><title>DNA extraction</title><p id=\"Par10\">NGM crops, GM soybean, GM maize, GM canola, GM cotton seeds, freeze-dried GM potato tissue, and processed products were ground separately. Samples of GM soybean mix (18 events), maize mix (33 events), canola mix (10 events), and cotton mix (14 events) were prepared by mixing 0.1 g of ground powder from each event. Genomic DNA (gDNA) was extracted from the powder samples using a Maxwell RCS PureFood GMO and Authentication Kit (Promega, Madison, WI, USA) according to the manufacturer’s instructions. DNA concentrations were determined using a QUBIT 3.0 Quantitation Kit (Thermo Fisher Scientific, Wilmington, DE, USA). DNA samples were diluted in nuclease-free water to a final concentration of 10 ng/µL and stored at − 20 °C until use.</p></sec><sec id=\"Sec5\"><title>Primers</title><p id=\"Par11\">Event-specific primers for the two GM potato events were used in PCR assays. <italic>APRT</italic> (GenBank accession number CK270447), which encodes adenine phosphoribosyltransferase, was selected as an endogenous reference gene. A previously reported APRT F/R primer set for <italic>APRT</italic> detection was used (MHLW, <xref ref-type=\"bibr\" rid=\"CR21\">2020</xref>). The SPS-Y9 mF1/mR1, and Event527-bf1/St527-R1 primer pairs were based on the 5' and 3' integration flanking sequences of T-DNA and the host genome of GM potato, respectively (Shin et al., <xref ref-type=\"bibr\" rid=\"CR23\">2020</xref>; JRC-GMO-Matrix, Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). All primers were synthesized by Bioneer (Daejeon, Republic of Korea), and the oligonucleotide sequences are listed in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The alignment sites of event-specific primers for GM potato SPS-Y9 and EH92-527-1. The arrows indicate the location of the primers. RB, right border; LB, left border; P-Vnt1, promoter for the native Rpi-vnt1 gene of <italic>Solanum venturii</italic>; Vnt1, Rpi-vnt1 gene encoding sequence of <italic>Solanum venturii</italic>; T-Vnt1, termination sequence of Rpi-vnt1 gene of <italic>Solanum venturii</italic>; P-Agp, promoter for the ADP glucose pyrophosphorylase gene of <italic>Solanum tuberosum</italic> var. Ranger Russet; VInv, vacuolar invertase gene fragment of <italic>Solanum tuberosum</italic> var. Ranger Russet; P-Gbss, promoter for the granule-bound starch synthase gene of <italic>Solanum tuberosum</italic>; P-nos, constitutive promoter of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; nptII, sequence encoding the NPTII enzyme (neomycin phosphotranspherase) from Tn5 transposon of E. coli; T-nos, termination sequence of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; Anti-gbss, antisense sequence encoding the gbss enzyme, involved in the starch synthase of <italic>Solanum tuberosum</italic></p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig1_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Oligonucleotide primers used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Target</p></th><th align=\"left\"><p>Name</p></th><th align=\"left\"><p>Sequence (5′ → 3′)</p></th><th align=\"left\"><p>Length (bp)</p></th><th align=\"left\"><p>Source</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>APRT F</p><p>APRT R</p></td><td align=\"left\"><p>TGAAAACGATCCCGATCG</p><p>CAATCCCAGCGATACGTTC</p></td><td align=\"char\" char=\".\"><p>90</p></td><td align=\"left\"><p>MHLW (Japan)</p></td></tr><tr><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Y9-mF1</p><p>Y9-mR1</p></td><td align=\"left\"><p>GAAGACAGCTATCAATATACTTTAGTAA</p><p>AATCTTTTGCCATACTCATACAGAG</p></td><td align=\"char\" char=\".\"><p>129</p></td><td align=\"left\"><p>MFDS (Korea)</p></td></tr><tr><td align=\"left\"><p>EH92</p><p>-527-1</p></td><td align=\"left\"><p>Event527-bf1</p><p>St527-R1</p></td><td align=\"left\"><p>GTGTCAAAACACAATTTACAGCA</p><p>TCCCTTAATTCTCCGCTCATGA</p></td><td align=\"char\" char=\".\"><p>93</p></td><td align=\"left\"><p>JRC (EU)</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec6\"><title>UF-PCR system</title><p id=\"Par12\">UF-PCR, which is based on the same principle as that of SYBR Green qPCR, was used by replacing SYBR Green with EvaGreen as an intercalating dye (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). PCR analysis was performed using a GENECHECKER UF-150 Ultra-fast Real-Time PCR system (Genesystem, Daejeon, Republic of Korea) with Rapi:chip. The final reaction volume was 10 µL consisting of 5 µL SSO Fast EvaGreen Supermixes (Bio-Rad Laboratories, Berkeley, CA, USA), 1 µL each of forward and reverse primers (0.8 µM), and 2 µL DNA template (50 ng). All PCR reactions were performed under the following conditions: one cycle of initial denaturation at 95 °C for 1 min, 40 cycles of denaturation at 95 °C for 5 s, annealing at 54 °C for 5 s, extension at 72 °C for 5 s, and one cycle of elongation at 72 °C for 5 s. No template control (NTC) was used as a negative control. The amplicons were analyzed using a microchip electrophoresis system, MCE-202 MultiNA (Shimazu, Nishinokyo, Kyoto, Japan) with MultiNA software v1.12 (Shimazu).</p></sec><sec id=\"Sec7\"><title>Specificity, sensitivity, and application test</title><p id=\"Par13\">The specificity of UF-PCR detection method was evaluated using various crops and GM events. To determine the detection level, gDNA samples of NGM potato and each GM potato event were mixed in concentrations ranging from 10 to 0.005% (w/w), and a sensitivity test was conducted. In addition, to check whether the method can be used for processed foods, a test was conducted on foods containing potato available in the market.</p></sec></sec><sec id=\"Sec8\"><title>Results and discussion</title><sec id=\"Sec9\"><title>Specificity</title><p id=\"Par14\">The UF-PCR assay was performed to assess the specificity of this detection method by analyzing NGM crops (soybean, maize, canola, cotton, wheat, rice, barley, and potato), GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048).</p><p id=\"Par15\">Results showed that only a single peak was detected for each target of GM potatoes, without specific amplification of non-target crops, GM events, and NTC (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>). In addition, the melting temperature (Tm) values of amplicons were 77.5 °C and 76.84 °C for SPS-Y9 and EH92-527-1, respectively (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Because amplicons have specific Tm values and do not cross-react, the two GM potato events could be selectively distinguished from each other by their Tm values.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Primer specificity for two GM potato events using Ultra-fast PCR. (<bold>A</bold>, <bold>C</bold>) Specificity results for Non-GM crops (eight crops; soybean, maize, canola, cotton, rice, wheat, potato, and barley) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (seven crops; soybean, maize, canola, rice, wheat, potato, and barley), lane 18: no template; (<bold>B</bold>, <bold>D</bold>) Specificity results for GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (eighty GM events; soybean, maize, canola, cotton, and potato), lane 18: no template</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig2_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Sensitivity results of the ultra-fast PCR method</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Events</p></th><th align=\"left\" rowspan=\"2\"><p>Conc</p></th><th align=\"left\" colspan=\"2\"><p>Ct value</p></th><th align=\"left\" colspan=\"2\"><p>Tm (°C) value</p></th></tr><tr><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>24.86</p></td><td align=\"char\" char=\".\"><p>0.25</p></td><td align=\"char\" char=\".\"><p>79.52</p></td><td align=\"char\" char=\".\"><p>0.21</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>SPS-Y9</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>26.34</p></td><td align=\"char\" char=\".\"><p>0.42</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>77.45</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.21</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>30.62</p></td><td align=\"char\" char=\".\"><p>0.40</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.12</p></td><td align=\"char\" char=\".\"><p>0.85</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>EH92-527-1</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>23.80</p></td><td align=\"char\" char=\".\"><p>0.40</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>76.77</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.27</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>29.75</p></td><td align=\"char\" char=\".\"><p>0.30</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.18</p></td><td align=\"char\" char=\".\"><p>0.69</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr></tbody></table><table-wrap-foot><p><italic>SD</italic> Standard deviation</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec10\"><title>Sensitivity</title><p id=\"Par16\">To evaluate the sensitivity of UF-PCR method for GM potato events, PCR assays were performed with gDNA samples that were serially diluted to concentrations ranging from 10 to 0.005% (wt/wt) per reaction. The Ct and Tm values of each event were analyzed. For all GM potato events, the assay was sufficiently identifiable even at 0.5% (0.25 ng) level of the target DNA. Based on the copy number of potato (one copy number for haploid genome of potato is considered as 1.8 pg (Arumuganathan and Earle, <xref ref-type=\"bibr\" rid=\"CR2\">1991</xref>)), 0.25 ng is equivalent to approximately 139 copies.</p><p id=\"Par17\">These results demonstrate that the rapid detection system used in this study can accurately detect GM incorporation as low as 0.5%. This level of detection is consistent with the low-level presence (LLP) of GMOs, that are not approved in their countries, but are already authorized in another countries (Stein and Rodríguez-Cerezo, <xref ref-type=\"bibr\" rid=\"CR26\">2010</xref>).</p></sec><sec id=\"Sec11\"><title>Application of UF-PCR in commercial processed products</title><p id=\"Par18\">Nine potato-containing processed products were tested to confirm the applicability of the UF-PCR method to GM potato, SPS-Y9 and EH92-527-1. The selected products included chips, snacks, ready-made meals, French fries, salads, and other items. Extracted gDNA samples were used for UF-PCR analysis, and the results are shown in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. In all the products, the APRT F/R primer set resulted in amplification, and the Tm values of the amplicons were approximately 79.3 °C. The primer sets for SPS-Y9 and EH92-527-1 did not produce amplicons for any of these samples. This suggests that the domestic distribution of processed potato products selected for this study did not contain two unapproved GM potato events.<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>The ultra-fast PCR results for potato DNA in processed products</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Products</p></th><th align=\"left\"><p>APRT</p></th><th align=\"left\"><p>SPS-Y9</p></th><th align=\"left\"><p>EH92-527-1</p></th></tr><tr><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Creamer potatoes</p></td><td align=\"left\"><p>28.15 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Baked potatoes</p></td><td align=\"left\"><p>30.09 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato salad</p></td><td align=\"left\"><p>31.10 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Frozen raw potatoes</p></td><td align=\"left\"><p>29.15 / 78.82</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Stick potatoes</p></td><td align=\"left\"><p>28.59 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Half cooked potatoes</p></td><td align=\"left\"><p>31.04 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Crinklecut potatoes</p></td><td align=\"left\"><p>30.11 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato chips</p></td><td align=\"left\"><p>24.65 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Deep fried potatoes</p></td><td align=\"left\"><p>30.85 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (SPS-Y9)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>35.04 / 77.50</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (EH92-527-1)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>34.22 / 76.51</p></td></tr></tbody></table><table-wrap-foot><p><italic>ND</italic> not determined</p></table-wrap-foot></table-wrap></p><p id=\"Par19\">A major challenge of this study was that obtaining products containing unauthorized GM potato was difficult. To address this issue, GM potato-spiked samples were tested together and confirmed to be correctly detected. Therefore, this assay method can be applied to analyze processed products based on the positive results for endogenous <italic>APRT</italic> of potato and GM spiked samples. The rapid detection method of GM potato using the UF-PCR system could be analyzed by selecting specific primers and identifying a specific amplification product with a unique melting temperature. In addition, this method was highly sensitive in detecting low copy numbers of DNA in GM potato samples. Recently, with the development of genetic analysis technology and equipment, high-speed analysis methods have been proposed to reflect the high-throughput demand for processing various samples. This reduces the analysis time, which normally takes 2 h, to as little as 40 min, thereby allowing more samples to be processed, faster results, and repetition of the test within the allotted time if case of any issues.</p><p id=\"Par20\">This study proposes a GM potato detection method using an UF-PCR system, which is a shortened test method. The method uses a very thin Rapi:chip and has the advantage of being able to finish the PCR reaction within a short time because of the high efficiency of heat transfer of the reaction mixtures and short duration between cycles.</p><p id=\"Par21\">The UF-PCR analysis method reviewed in this study is specific, sensitive, and reliable for identifying GM potato events. Moreover, the assay is eco-friendly because it uses a small amount of reagents, and if necessary, it can perform analysis using a small portable analyzer in field conditions rather than in the laboratory.</p><p id=\"Par22\">In conclusion, the reaction time and usage of reagents of the UF-PCR system are less than half of those for the conventional method, and the equipment is portable; therefore it can be utilized for on-site sample analysis. Therefore, we propose the use of this method that can perform a quick test within approximately 20 min to inspect and monitor GM potato events.</p></sec></sec></body><back><ack><title>Acknowledgements</title><p>This work was supported by the Ministry of Food and Drug Safety of Korea.</p></ack><sec><title>Funding</title><p>This study was funded by the Ministry of Food and Drug Safety of Korea (Grant Number 19161MFDS051).</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par23\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Aboud</surname><given-names>M</given-names></name><name><surname>Oh</surname><given-names>HH</given-names></name><name><surname>McCord</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Rapid direct PCR for forensic genotyping in under 25 min</article-title><source>Electrophoresis.</source><year>2013</year><volume>34</volume><fpage>1539</fpage><lpage>1547</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXos12qsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/elps.201200570</pub-id><pub-id pub-id-type=\"pmid\">23529977</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Arumuganathan</surname><given-names>K</given-names></name><name><surname>Earle</surname><given-names>ED</given-names></name></person-group><article-title xml:lang=\"en\">Nuclear DNA content of some important plant species</article-title><source>Plant Molecular Biology Reporter.</source><year>1991</year><volume>9</volume><fpage>208</fpage><lpage>218</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaK38XhslSjurY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/BF02672069</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"other\">Chen XJ, Liu YT, Zhan XA, Gao YB, Sun ZY, Wen WJ, Zheng WD. Ultrafast PCR detection of COVID-19 by using a microfluidic chip-based system. Bioengineering (Basel). 9: 548 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.3390/bioengineering9100548\">https://doi.org/10.3390/bioengineering9100548</ext-link></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cote</surname><given-names>MJ</given-names></name><name><surname>Meldrum</surname><given-names>AJ</given-names></name><name><surname>Raymond</surname><given-names>P</given-names></name><name><surname>Dollard</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Identification of genetically modified potato cultivars using event specific polymerase chain reaction</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2005</year><volume>53</volume><fpage>6691</fpage><lpage>6696</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXmsFGjs74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf050591i</pub-id><pub-id pub-id-type=\"pmid\">16104786</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"other\">Das D, Hsieh HC, Chen CS, Chen WL, Chuang HS. Ultrafast and sensitive screening of pathogens by functionalized janus microbeads-enabled rotational diffusometry in combination with isothermal amplification. Small Science. 2: 202200010 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.1002/smsc.202200010\">https://doi.org/10.1002/smsc.202200010</ext-link></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"other\">Food and Agriculture Organization of the United Nations. FAOSTAT Statistical Database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fao.org/faostat\">https://www.fao.org/faostat</ext-link>. Accessed Aug. 27, 2022.</mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fraiture</surname><given-names>MA</given-names></name><name><surname>Roosens</surname><given-names>NHC</given-names></name><name><surname>Taverniers</surname><given-names>I</given-names></name><name><surname>Loose</surname><given-names>MD</given-names></name><name><surname>Deforce</surname><given-names>D</given-names></name><name><surname>Herman</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Biotech Rice: Current developments and future detection challenges in food and feed chain</article-title><source>Trends in Food Science &amp; Technology.</source><year>2016</year><volume>52</volume><fpage>66</fpage><lpage>79</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XmtFCntr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2016.03.011</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gao</surname><given-names>H</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Deng</surname><given-names>T</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name><name><surname>Xiao</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Event-specific detection of transgenic potato AV43-6-G7 using real-time and digital PCR methods</article-title><source>BMC Biotechnology.</source><year>2016</year><volume>16</volume><fpage>74</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlvV2gt7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12896-016-0303-8</pub-id><pub-id pub-id-type=\"pmid\">27784303</pub-id><pub-id pub-id-type=\"pmcid\">5081928</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hong</surname><given-names>YW</given-names></name><name><surname>Kim</surname><given-names>JJ</given-names></name><name><surname>Yu</surname><given-names>YC</given-names></name><name><surname>Kim</surname><given-names>HS</given-names></name><name><surname>Moon</surname><given-names>GI</given-names></name><name><surname>Park</surname><given-names>EM</given-names></name></person-group><article-title xml:lang=\"en\">Ultra-fast PCR method for the distinguishing between Miichthys miiuy and Sciaenops ocellatus</article-title><source>Food Science and Biotechnology.</source><year>2021</year><volume>30</volume><issue>9</issue><fpage>1225</fpage><lpage>1231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXit1Ggs7rE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-00954-4</pub-id><pub-id pub-id-type=\"pmid\">34603821</pub-id><pub-id pub-id-type=\"pmcid\">8423894</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"other\">International Service for the Acquisition of Agri-Biotech Applications (ISAAA). GM approval database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.isaaa.org/gmapprovaldatabase\">https://www.isaaa.org/gmapprovaldatabase</ext-link>. Accessed Sep. 18, 2022.</mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaccaud</surname><given-names>E</given-names></name><name><surname>Hohne</surname><given-names>M</given-names></name><name><surname>Meyer</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of screening methods for the identification of genetically modified potatoes in raw materials and finished products</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2003</year><volume>51</volume><fpage>550</fpage><lpage>557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XpsFCnsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf0208031</pub-id><pub-id pub-id-type=\"pmid\">12537422</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Suh</surname><given-names>SM</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Rapid on-site detection of shrimp allergen tropomyosin using a novel ultrafast PCR system</article-title><source>Food Science and Biotechnology</source><year>2019</year><volume>28</volume><issue>2</issue><fpage>591</fpage><lpage>597</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhvVerurfP</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-018-0479-x</pub-id><pub-id pub-id-type=\"pmid\">30956872</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"other\">Kim SH, Lee YS, Joo IS, Kwak HS, Chung GT, Kim SH. Rapid detection for salmonella spp. by ultrafast real-time PCR assay. J. Food Hyg. Saf. 33: 50–57 (2018)</mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SW</given-names></name><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>ES</given-names></name><name><surname>Kim</surname><given-names>DJ</given-names></name><name><surname>Park</surname><given-names>SH</given-names></name><name><surname>Lee</surname><given-names>EJ</given-names></name><name><surname>Lee</surname><given-names>SY</given-names></name><name><surname>Lee</surname><given-names>JS</given-names></name><name><surname>Lim</surname><given-names>CS</given-names></name><name><surname>Kim</surname><given-names>WK</given-names></name><name><surname>In</surname><given-names>KH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of mycobacterium tuberculosis using a novel ultrafast chip-type real-time polymerase chain reaction system</article-title><source>Chest.</source><year>2014</year><volume>146</volume><issue>5</issue><fpage>1319</fpage><lpage>1326</lpage><pub-id pub-id-type=\"doi\">10.1378/chest.14-0626</pub-id><pub-id pub-id-type=\"pmid\">24967657</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Martinez-Prada</surname><given-names>MM</given-names></name><name><surname>Curtin</surname><given-names>SJ</given-names></name><name><surname>Gutierrez-Gonzalez</surname><given-names>JJ</given-names></name></person-group><article-title xml:lang=\"en\">Potato improvement through genetic engineering</article-title><source>GM Crops &amp; Food.</source><year>2021</year><volume>12</volume><fpage>479</fpage><lpage>496</lpage><pub-id pub-id-type=\"doi\">10.1080/21645698.2021.1993688</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Milosevic</surname><given-names>J</given-names></name><name><surname>Lu</surname><given-names>M</given-names></name><name><surname>Greene</surname><given-names>W</given-names></name><name><surname>He</surname><given-names>HZ</given-names></name><name><surname>Zheng</surname><given-names>SY</given-names></name></person-group><article-title xml:lang=\"en\">An ultrafast one-step quantitative reverse transcription-polymerase chain reaction assay for detection of SARS-CoV-2</article-title><source>Frontiers in Microbiology.</source><year>2021</year><volume>12</volume><pub-id pub-id-type=\"doi\">10.3389/fmicb.2021.749783</pub-id><pub-id pub-id-type=\"pmid\">34803970</pub-id><pub-id pub-id-type=\"pmcid\">8600229</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"other\">Ministry of Food and Drug Safety (MFDS). GMO status database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.foodsafetykorea.go.kr/portal/board/board.do\">https://www.foodsafetykorea.go.kr/portal/board/board.do</ext-link>. Accessed July. 15, 2022.</mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"other\">Ministry of Health, Labour and Welfare (MHLW). Detection method for genetically modified DNA (2020). Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf\">https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf</ext-link>. Accessed Feb. 16, 2022.</mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Roh</surname><given-names>JK</given-names></name><name><surname>Lee</surname><given-names>T</given-names></name><name><surname>Jung</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>TA</given-names></name><name><surname>Park</surname><given-names>YH</given-names></name><name><surname>Kim</surname><given-names>YM</given-names></name></person-group><article-title xml:lang=\"en\">Qualitative and Quantitative PCR methods for detection of three lines of genetically modified potatoes</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2004</year><volume>52</volume><fpage>3269</fpage><lpage>3274</lpage><pub-id pub-id-type=\"doi\">10.1021/jf0499020</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"other\">Shin MK, Jeon SM, Hong SJ, Choi CH, Kim HA, Park JS. Research on the establishment of test method for labeling and management of GMOs. Ministry of Food and Drug Safety (2020)</mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shin</surname><given-names>MK</given-names></name><name><surname>Jeon</surname><given-names>SM</given-names></name><name><surname>Koo</surname><given-names>YE</given-names></name></person-group><article-title xml:lang=\"en\">Development of a rapid detection method for genetically modified rice using the ultra-fast PCR system</article-title><source>Food Science and Biotechnology</source><year>2022</year><volume>31</volume><issue>2</issue><fpage>175</fpage><lpage>182</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XmsF2gs78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-01025-4</pub-id><pub-id pub-id-type=\"pmid\">35095287</pub-id><pub-id pub-id-type=\"pmcid\">8783574</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Song</surname><given-names>JY</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Detection of unapproved genetically modified potatoes in Korea using multiplex polymerase chain reaction</article-title><source>Food Control.</source><year>2017</year><volume>80</volume><fpage>19</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmsFejtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2017.04.026</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Stein</surname><given-names>AJ</given-names></name><name><surname>Rodríguez-Cerezo</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Low-level presence of new GM crops: An issue on the rise for countries where they lack approval</article-title><source>AgBioForum.</source><year>2010</year><volume>13</volume><issue>2</issue><fpage>173</fpage><lpage>182</lpage></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tu</surname><given-names>YK</given-names></name><name><surname>Lin</surname><given-names>YC</given-names></name><name><surname>Feng</surname><given-names>YW</given-names></name><name><surname>Tseng</surname><given-names>YY</given-names></name><name><surname>Chen</surname><given-names>HW</given-names></name></person-group><article-title xml:lang=\"en\">Visual, sensitive and rapid event-specific detection of genetically modified potato EH92-527-1 by loop-mediated isothermal amplification method</article-title><source>Bioscience, Biotechnology, and Biochemistry.</source><year>2020</year><volume>84</volume><fpage>43</fpage><lpage>52</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsleitrjO</pub-id><pub-id pub-id-type=\"doi\">10.1080/09168451.2019.1661766</pub-id><pub-id pub-id-type=\"pmid\">31495297</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"other\">Visual Capitalist. Mapped: Food production around the world. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world</ext-link>. Accessed Aug. 18, 2022.</mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoo</surname><given-names>MR</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Yea</surname><given-names>MC</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Development of detection method of unapproved genetically modified potato (EH92-527-1) in Korea using duplex polymerase chain reaction</article-title><source>Korean Journal of Food Science and Technology.</source><year>2013</year><volume>45</volume><fpage>156</fpage><lpage>160</lpage><pub-id pub-id-type=\"doi\">10.9721/KJFST.2013.45.2.156</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"other\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10953</journal-id><journal-id journal-id-type=\"doi\">10.1007/10953.1572-8927</journal-id><journal-title-group><journal-title>Journal of Solution Chemistry</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Solution Chem</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0095-9782</issn><issn pub-type=\"epub\">1572-8927</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10953-023-01308-w</article-id><article-id pub-id-type=\"manuscript\">1308</article-id><article-id pub-id-type=\"doi\">10.1007/s10953-023-01308-w</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">In Memoriam: Jacques Desnoyers (1935–2023)</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Quirion</surname><given-names>Francois</given-names></name><address><email>francois.quirion@physicochimie.fr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs1095302301308w_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Perron</surname><given-names>Gérald</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Desnoyers</surname><given-names>Christine</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Desnoyers</surname><given-names>Marie</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Desnoyers</surname><given-names>Nichola</given-names></name></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Consultant R&amp;D Physicochimie</institution></institution-wrap><addr-line content-type=\"street\">261 Boulevard Pereire</addr-line><addr-line content-type=\"postcode\">75017</addr-line><addr-line content-type=\"city\">Paris</addr-line><country country=\"FR\">France</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Retired from INRS-Énergie Matériaux et Télécommunications</institution></institution-wrap><addr-line content-type=\"city\">Varennes</addr-line><addr-line content-type=\"state\">QC</addr-line><country country=\"CA\">Canada</country></aff></contrib-group><author-notes><corresp id=\"IDs1095302301308w_cor1\"><label>a</label><email>francois.quirion@physicochimie.fr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>20</day><month>6</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>52</volume><issue seq=\"1\">8</issue><fpage>859</fpage><lpage>860</lpage><history><date date-type=\"registration\"><day>31</day><month>5</month><year>2023</year></date><date date-type=\"online\"><day>20</day><month>6</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>9</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>5</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10953_2023_Article_1308.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>EditorialNotes</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Geochemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Oceanography</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Inorganic Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Condensed Matter Physics</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Christine Desnoyers, Marie Desnoyers and Nichola Desnoyers are son and daughters of Professor Desnoyers.</p></notes></front><body><p id=\"Par1\">It is with great sadness that we report the death of Jacques Desnoyers who passed away peacefully in Quebec City on January 12th at the age of 87, surrounded by his loved ones.<fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"JPEG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/10953_2023_1308_Figa_HTML.jpg\"/></fig></p><p id=\"Par2\">Jacques Desnoyers was born on 28th January 1935 in Ottawa where he obtained his Ph.D. in 1961 at the University of Ottawa. After a year of post-doctorate studies in England (University of London and of Manchester), he joined, in 1962, the chemistry department at the <italic>Université de Sherbrooke</italic> as full professor and later as director. In 1983, he became the scientific director of <italic>Institut national de la recherche scientifique</italic> (INRS) where he also acted as interim general director (3 March 1987 to 31 July 1988). In 1991, he returned to research as full professor at <italic>INRS</italic>. During the same period, he was appointed as Ombudsman at the <italic>Université du Québec</italic>. In 2002, he retired from INRS. Meanwhile, he accepted the position of scientific advisor for the <italic>Gouvernement du Québec</italic> until 2006. He finally retired fully in 2007.</p><p id=\"Par3\">He developed and promoted flow microcalorimetry with Patrick Picker. He investigated the thermodynamic properties of electrolytes and non-electrolytes in aqueous, non-aqueous and mixed solvents, surfactant solutions, swollen micelles and microemulsions, and polymer-surfactant systems. His research found applications in the fields of low-temperature batteries, Li-ion batteries, enhanced oil recovery and soil remediation.</p><p id=\"Par4\">His work was internationally recognized and he frequently collaborated with other Canadian groups but also with groups around the world, including the USA, France and Italy. After his retirement, he received the 2007 Setaram Award from the <italic>Associazione italiana di calorimetria e analisi termica</italic> (AICAT) for his outstanding scientific achievements in the field of calorimetry.</p><p id=\"Par5\">Throughout his career, Jacques Desnoyers strongly believed in the importance of transmitting knowledge which he did through teaching, directing thesis, writing scientific articles and book chapters and reviewing manuscripts for many scientific journals.</p><p id=\"Par6\">He will be remembered for his <italic>joie de vivre</italic>, his human qualities, his professional ethical principles and for his continuous effort to promote the use of thermodynamic properties as powerful tools for the investigation of simple and complex systems.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02771-x</article-id><article-id pub-id-type=\"manuscript\">2771</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02771-x</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Optimization of microwave-enhanced extraction parameters to recover phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-5114-2100</contrib-id><name><surname>Alara</surname><given-names>Oluwaseun Ruth</given-names></name><address><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302771x_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Abdurahman</surname><given-names>Nour Hamid</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Ali</surname><given-names>Hassan Alsaggaf</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Centre for Research in Advanced Fluid and Processes (Fluid Centre)</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, College of Engineering</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution content-type=\"org-name\">Eastern Unity Technology</institution></institution-wrap><addr-line content-type=\"street\">Suite 01, 12th Floor Plaza, 138 Annex Hotel Maya, Jalan Ampang</addr-line><addr-line content-type=\"postcode\">50450</addr-line><addr-line content-type=\"city\">Kuala Lumpur</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302771x_cor1\"><label>a</label><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>10</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"9\">8</issue><fpage>4217</fpage><lpage>4233</lpage><history><date date-type=\"registration\"><day>6</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>23</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>4</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>10</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Vegetables are industrial crops endowed with both nutritional and medicinal values. The overwhelming contributions of vegetables to human living in the form of nutrients and medicine cannot be under emphasised. Thus, this study examined the recoveries of phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves using a microwave-enhanced extraction technique. Furthermore, the phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were comprehensively identified using liquid chromatography-mass spectrometry quadrupole of flight (LC-QToF-MS). At the optimized conditions of microwave-enhanced extraction (extraction time of 131 s, microwave power 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%), total phenolic content (TPC) of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for 2,2-diphenyl-1-picrylhydrazyl (DPPH) and 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS) assays, respectively, were achieved. Furthermore, an aggregate of fourteen phenolic compounds that include 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose was tentatively identified in the leaf extract of <italic>C. olitorius</italic>. In general, this study has established <italic>C. olitorius</italic> leaves as sources of phenolic compounds and natural antioxidants. Thus, the intake can continue to be promoted as a way forward in solving the problem of food insecurity.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Corchorus olitorius</italic></kwd><kwd>Response surface methodology</kwd><kwd>Total phenols</kwd><kwd>Antioxidant</kwd><kwd>Liquid chromatography quadrupole time-of-flight mass spectrometry</kwd><kwd>Microwave-enhanced extraction</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2771.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02771-x\">https://doi.org/10.1007/s11696-023-02771-x</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par12\">Vegetables have been categorised as industrial crops with valuable means of nutrients and a source of income. Other than the valuable nutrients in vegetables, some of them possess diverse medicinal characteristics that are good for human and animal health. Although indigenous vegetables are long been employed in day-to-day diets in different communities of the world, several of these vegetables are underutilised as most have embodiments of unknown and underestimated pharmacological features. Most attention is based on the consumption of maize, rice, and cassava. It is not disputed these crops are endowed with great nutritional values; nevertheless, vegetables can be prioritised due to their functional properties (food and medicine). Additionally, United Nation Sustainable Development Goals (SDG 2) established the importance of figuring out sustainable means of producing foods for humans, especially natural sources. This is highly needed in this time of post-Covid-19 recovery as the world is being faced with hunger, food insecurity and consumption of low-quality foods that cause micronutrient deficiencies (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>)</p><p id=\"Par13\"><italic>Corchorus olitorius</italic> (Jute mallow) is a leafy vegetable mostly found in Tropical regions including the Middle East, Africa and some parts of Asia (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). It belongs to the family of Malvaceae (Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). This is one of the vegetables consumed mainly in different African countries including Nigeria, Tanzania, South Africa, and Ethiopia (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). Consuming this vegetable has been encouraged to curb malnutrition and food insecurity, especially in developing nations such as African countries. The consumption of <italic>C. olitorius</italic> leaves is much encouraged as it is believed to overcome nutritional and food insecurity among children under the ages of five, lactating mothers and pregnant women in sub-Saharan Africa (Andersen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2003</xref>; Kamga et al. <xref ref-type=\"bibr\" rid=\"CR17\">2013</xref>). This vegetable is mostly consumed in the form of soup or sauce alongside solid foods from rice, yam or cassava in African cooking cuisine. It is endowed with different vitamins, enrich in fibre, and antioxidants (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). A brief description of <italic>C. olitorius</italic>: it is an annual vegetable that can reach a height of about 2.4 m; the leaves are finely lobed or serrated margin, simple, lanceolate, and alternate; it has only a few branches and can be unbranched; seeds are used when planting; the flowers are yellowish small with five petals (2–3 cm in diameters) and can survive in any kinds of soil (loamy, sandy or clay) (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). The leaves of <italic>C. olitorius</italic> are enriched in beta carotene, vitamins A and C, folic acid, calcium, iron, and diverse phenolic compounds (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). The leaves of <italic>C. olitorius</italic> were reported to contain nutritional values more than stems and fruits. This is the reason why it is generally used as food. The leaves had been revealed to be made up of about seventeen active nutrients compounds such as protein, carbohydrate, fat, ash, fibre, potassium, calcium, sodium, phosphorous, iron, beta-carotene, riboflavin, thiamine, ascorbic acid, niacin, and so on. Moreover, the leaves had been reported to possess hepatoprotective, antidiabetic, antioxidant, diuretic, antipyretic, analgesic, antitumor, antimicrobial, anti-obesity, hypoglycemic, and gastroprotective properties (Abir et al. <xref ref-type=\"bibr\" rid=\"CR1\">2019</xref>; Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>; Saliu et al. <xref ref-type=\"bibr\" rid=\"CR28\">2019</xref>).</p><p id=\"Par14\">The presence of phenolic compounds in plant materials is continually studied due to their important link with the occurrence of natural antioxidants that possess outstanding applications in human health (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Cheynier <xref ref-type=\"bibr\" rid=\"CR13\">2012</xref>). The antioxidants from leafy vegetables such as <italic>C. olitorius</italic> can act in place of synthetic antioxidants readily available in the market. These synthetic antioxidants are being reported to pose side effects on consumers (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). However, natural antioxidants can act as protection against degenerative diseases and alter the DNA, proteins and lipids (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). Besides, plants are surviving the exposure to ultraviolet radiation in the environment; this has positioned them as a perfect antioxidant source. Likewise, the consumptions of plant-sourced antioxidants are relatively safe and widely accepted; this has propelled the introduction of plant polyphenols into cosmeceutical and pharmaceutical products in place of synthetic antioxidants (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>; Lobo et al. <xref ref-type=\"bibr\" rid=\"CR20\">2010</xref>). The process of extraction is important when targeting to achieve significant phenolic compounds and antioxidants from plant materials. This process can either be conventional or non-conventional; the latter is currently being used due to its ability to recover a tangible quantity of phenolic compounds and antioxidants from plant materials (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Herrero et al. <xref ref-type=\"bibr\" rid=\"CR16\">2012</xref>). Out of several unconventional techniques that are being used, the microwave-enhanced method has been generally accepted due to its ability to leach a significant quantity of phenolic compounds from the considered sample within a reduced duration of time and not alter the bioactive elements in the extracts (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>).</p><p id=\"Par15\">Several parameters can alter the efficiency of recovered phenolic compounds and antioxidants from plant matrices when using microwave-enhanced technique; these are temperature, extraction time, solvent/sample ratio, solvent concentration, microwave power, and solvent selection. Out of different available solvents used in the extraction processes, ethanol and water are the most environment-friendly. It had been explained that pure ethanol cannot extract the totality of phenolic compounds especially those that are hydrophilic (water-soluble) (Thoo et al. <xref ref-type=\"bibr\" rid=\"CR32\">2010</xref>). Previous studies had suggested that the use of aqueous ethanol or binary mixture contributes to the recovery of antioxidant from plant sample (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>). Thus, mixing these two solvents (water and ethanol) in rightful proportion tends to improve the recovery of bioactive materials from plant matrices (Khoddami et al. <xref ref-type=\"bibr\" rid=\"CR18\">2013</xref>). As the selection of extraction is important in achieving significant recovery, process optimization is as well essential to achieve this goal.</p><p id=\"Par16\">There are two common techniques used for process optimization (one leading to the other); these are one-factor-at-a-time (OFAT) experimental trials and response surface methodology (RSM). Although OFAT suffers drawbacks such as no room for interaction between the factors; nonetheless, OFAT is widely employed to determine the range of values used for the response surface methodology. RSM is utilised to settle the limitations of OFAT experimental trials which include inadequate data evaluation, being expensive and time-consuming (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par17\">Although diverse studies had been carried out on <italic>C. olitorius</italic> leaves concerning the proximate constituents, total phenols and antioxidants (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). Nevertheless, there is yet a report on the optimization studies involving microwave-enhanced extraction of the total phenols and antioxidants from <italic>C. olitorius</italic> leaves and comprehensive identifications of the phenolic compounds. Hence, this study focuses on unveiling the optimised microwave-enhanced extraction conditions to achieve maxima phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. At the optimized conditions, the comprehensive identifications of different phenolic compounds in the extracts were carried out using a fast liquid chromatography quadrupole time-of-flight mass spectrometry (LC-QToF-MS).</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Cultivation of <italic>C. olitorius</italic> seeds and sample</title><p id=\"Par18\">The seeds of <italic>C. olitorius</italic> were cultivated in a small garden in the Universiti Malaysia Pahang (UMP) for two months (from February and April 2019); every agricultural practice such as watering, weed control and thinning was adequately observed to achieve productive results. Sandy-loam soil was used for the cultivation. In the maturity stage, the leaves of <italic>C. olitorius</italic> were harvested and washed to evacuate dirt under running water. Then, the leaves were allowed to drain off the water and dried at room temperature until a constant weight was reached. The dried leaves were milled using a blender and sieved using a laboratory sieve (400/425 µm pore size). Afterwards, the ground sample was kept in a zip-lock bag and kept in a freezer at − 20 °C before the extraction process.</p></sec><sec id=\"Sec4\"><title>Chemicals and reagents used</title><p id=\"Par19\">Ethanol, gallic acid, methanol, 2,2-diphenyl-1-picrylhydrazyl (DPPH), sodium carbonate, 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS), potassium persulfate, and Folin–Ciocalteu reagent were procured from Sigma-Aldrich (Selangor, Malaysia).</p></sec><sec id=\"Sec5\"><title>Microwave-enhanced extraction through experimental design</title><p id=\"Par20\">The solvents used in this study were water and ethanol. An Ethos microwave oven (Milestone, Italy) was employed, and this extractor is configured using a control compartment with power, time and temperature control. Since the microwave-enhanced extraction technique is influenced by microwave power, temperature, time, solvent concentration, and solvent/sample ratio; thus, these factors were first studied by carrying out OFAT experimental trials to evaluate the effect of an individual factor on the recovered phenolic compounds and antioxidants as well as the range for individual factor. These ranges of values were employed for the optimization studies. For the extraction, 5 g of <italic>C. olitorius</italic> leaf sample was adequate mixed together with an estimated proportion of ethanol–water solvent (experimental design). The mixture was filtered, and a rotary evaporator set at 50 °C was utilised to concentrate the extract.</p><p id=\"Par21\">After the determination of preliminary ranges for the five considered factors through OFAT experimental trials, a face-centred central composite design (FCCCD) was used to optimise the recovery of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Four factors were considered during the optimization study since temperature (kept at 40 °C) showed an insignificant effect on the dependent factors (total phenolic compounds (TPC) and antioxidant activity (DPPH and ABTS)). Thus, the independent factors were extraction time (<italic>X</italic><sub>1</sub>: 60–180 s), microwave power (<italic>X</italic><sub>2</sub>: 200–400 W), solvent/sample (<italic>X</italic><sub>3</sub>: 8–16 mL/1 g), and ethanol concentration (<italic>X</italic><sub>4</sub>: 30–50%), while the dependent factors were TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>). Thirty randomised runs of experiments with six centre points were obtained using FCCCD (Table S1 in supplementary materials). The regression analysis for this study was initiated using Eq. (<xref ref-type=\"disp-formula\" rid=\"Equ1\">1</xref>).<disp-formula id=\"Equ1\"><label>1</label><alternatives><mml:math display=\"block\" id=\"Equ1_Math\"><mml:mrow><mml:mi>Y</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>β</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mrow><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn><mml:mo>,</mml:mo><mml:mi>i</mml:mi><mml:mo>&lt;</mml:mo><mml:mi>j</mml:mi></mml:mrow><mml:mrow><mml:mi>k</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:munderover><mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>j</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ij</mml:mi></mml:mrow></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>i</mml:mi></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>j</mml:mi></mml:msub><mml:mo>+</mml:mo></mml:mrow></mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ii</mml:mi></mml:mrow></mml:msub><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mi>i</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mrow></mml:math><tex-math id=\"Equ1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$Y = \\beta_{0} + \\sum\\limits_{i = 1}^{k} {\\beta_{1} X_{1} } + \\sum\\limits_{i = 1,i &lt; j}^{k = 1} {\\sum\\limits_{j = 2}^{k} {\\beta_{ij} X_{i} X_{j} + } } \\sum\\limits_{i = 2}^{k} {\\beta_{ii} X_{i}^{2} }$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ1.gif\"/></alternatives></disp-formula>where <italic>Y</italic> indicates the dependent factors which are TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>), <italic>β</italic><sub>0</sub> is a constant, <italic>β</italic><sub><italic>i</italic></sub> indicates the linear coefficient (actual effect), <italic>β</italic><sub><italic>ij</italic></sub> represents the interaction coefficients between two considered extraction factors, <italic>β</italic><sub><italic>ii</italic></sub> is the second-order polynomial coefficient, and <italic>k</italic> is the number of dependent factors (<italic>k</italic> = 4). The significant values were removed, while only the insignificant values were used to construct a reduced model (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par22\">Having achieved the predicted yields of the considered responses, the adequacy of the constructed models was tested. These were carried out by comparing the actual and predicted values. A pair <italic>t</italic>-test analysis was employed to verify the postulated models.</p></sec><sec id=\"Sec6\"><title>Determination of TPC in the <italic>C. olitorius</italic> leaf extract</title><p id=\"Par23\">The TPC in extract from <italic>C. olitorius</italic> leaves was estimated following the previously described procedures (Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>). Briefly, 200 µL of Folin–Ciocalteu reagent was added to 100 µL of leaf extract of <italic>C. olitorius</italic>; then, 0.2 mM Na<sub>2</sub>CO<sub>3</sub> using the volume of 600 µL was added and the mixture was allowed to stay 2 h in the dark at room temperature for adequate reactions. After this, the absorbance of the reacted mixture was read at 765 nm.</p></sec><sec id=\"Sec7\"><title>Determination of antioxidant activity</title><sec id=\"Sec8\"><title>DPPH analysis</title><p id=\"Par24\">The free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, the DPPH solution was prepared to achieve 0.1 mM. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves was presented as IC<sub>50</sub> (this is the concentration of the <italic>C. olitorius</italic> leaf extract needed to inhibit 50% formation of DPPH radicals). These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 200 µL of the extract and 0.1 mM DPPH solution (2 mL volume) were thoroughly mixed. The absorbance was read at 517 nm after the mixture was left to adequate react for 30 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1478<italic>x</italic> + 19.781, <italic>R</italic><sup>2</sup> = 0.9986).</p></sec><sec id=\"Sec9\"><title>ABTS analysis</title><p id=\"Par25\">For the ABTS analysis, the free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, an equal ratio of 2.45 mM potassium persulfate and 7 mM ABTS<sup>+</sup> solution was dissolved in 120 mL of methanol to achieve the stock solution of ABTS. This stock solution was allowed to react for 12 h before its usage. Then, 2 mL of stock solution was further diluted in 120 mL of methanol to obtain a fresh solution with an absorbance of 1.10 ± 0.02 at 734 nm. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves using ABTS was presented as IC<sub>50</sub>. These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 2.85 mL of a fresh solution of ABTS and 0.15 mL of extract were thoroughly mixed. At 734 nm, the absorbance was recorded after the mixture had been left to react for 120 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1008<italic>x</italic> + 46.858, <italic>R</italic><sup>2</sup> = 0.9977).</p></sec></sec><sec id=\"Sec10\"><title>LC-QToF-MS analysis of phenolic compounds</title><p id=\"Par26\">Every phenolic compound in the leaf extract of <italic>C. olitorius</italic> at the optimized microwave-enhanced extraction conditions was identified using LC-QToF-MS (Waters Vion IMS, USA). A C<sub>18</sub> column with the dimension of 100 mm × 2.10 mm × 1.80 µm was used. There are two mobile phases used in this analysis; they are water + 0.1% formic acid (A) and 100% acetonitrile (B). The operating conditions were a seal wash period of 5 min, a pre-injector volume of 0 µL, low pressure of 0 psi, high-pressure limit of 18,000 psi, gradient elution was 90% A and 10% B (1.25 min), 45% A and 55% B (4.17 min), 10% A and 90% B (6.25 min), 90% A and 10% B (8.34 min), with the injection volume of 20 µL, the flow rate of 0.50 mL/min, spray voltage of 4 keV, mass range of 100–1000 m/z, sample temperature of 15 °C, column temperature of 40 °C, gas flow of 0.50 mL/min, desolvation gas flow rate of 800 L/h, desolvation temperature of 550 °C, low collision energy of 4 eV, high collision energy ramp between 10 and 45 eV, source temperature of 120 °C. The identifications of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were performed by utilising SYNAPT mass spectrometer (Waters) coupled with an electrospray ionization operated in negative ion mode. Waters® UNIFY Software 1.0.0 installed in equipment was used to identify the phenolic compounds.</p></sec><sec id=\"Sec11\"><title>Data analysis</title><p id=\"Par27\">Each experimental trial was repeated thrice, and the data were recorded as mean values ± standard deviation. The Design Expert® software (Version 7, Stat. Ease Inc., Minneapolis, USA) was employed to carry out the optimization study. The model significance was based on a <italic>p</italic> value less than 0.05 and vice versa for the insignificant model. Three-dimensional plots were used to understand the interactions between two factors. A Chi-squared test was used to verify the validity of the predicted models.</p></sec></sec><sec id=\"Sec12\"><title>Results and discussions</title><sec id=\"Sec13\"><title>One-factor-at-a-time experimental trials for the extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</title><p id=\"Par28\">From the beginning of this study, the preliminary experimental trials were to study the effects of the factors influencing the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves using a microwave-enhanced extraction technique. These factors were extraction time, microwave power, solvent/sample ration, ethanol concentration, and temperature. Likewise, the experimental ranges for the subsequent optimization study were determined using OFAT. OFAT experimental trials involve varying a factor, while others are kept constant.</p><sec id=\"Sec14\"><title>Contributing impact of extraction time</title><p id=\"Par29\">Extraction time is an essential factor when considering solid–liquid extraction; this is due to its impact on mass transfer and solubility. Figure <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ai and Aii showed the impact of extraction time on the recoveries of phenolics and antioxidants from <italic>C. olitorius</italic>. This was done by keeping solvent/sample ratio, microwave power, ethanol concentration, and temperature at 8 mL/g, 200 W, 20%, and 20 °C, respectively. It can be observed that TPC recovery increased with extraction time to achieve a higher yield at 120 s before the decline in the yield set in. This might be because the extended time of extraction can be dangerous to phenolic compounds in the plant matrix; leading to degradation when expose to microwave radiation (Alara et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021b</xref>; Azwanida <xref ref-type=\"bibr\" rid=\"CR10\">2015</xref>). On contrary, the recoveries of antioxidant through DPPH and ABTS declined with increasing time; the minima yield of antioxidants was achieved at 120 s prior to the further increase in the yield with increasing extraction time. This decrease occurs because the antioxidant activity is inversely proportional to IC<sub>50</sub> values (linear regression between the percentage of inhibition and extract concentration) (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>). Thus, a lesser IC<sub>50</sub> value indicates higher antioxidant potential. The maxima scavenging potentials of <italic>C. olitorius</italic> leaves for both DPPH and ABTS were reached at 120 s. For the optimization study, 60–180 s was the range of values. However, 120 s was considered as the fixed extraction time when studying the impacts of other factors.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Multiple parameters that influence the recoveries from phenolics and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec15\"><title>Contributing impact of solvent/sample ratio</title><p id=\"Par30\">During the solid–liquid extraction, the solvent/sample ratio can impact the recoveries of phenolic compounds and antioxidants. This is because this factor has an impact on the concentration gradient between the solutes in the plant sample and the solvent (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). In this study, the weight of the <italic>C. olitorius</italic> leaves was kept constant, while the volume of the solvent was varied. The effect of solvent/sample ratio was studied by keeping extraction time, microwave power, ethanol concentration, and temperature at 120 s, 200 W, 20%, and 20 °C, respectively. As seen in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bi, the recovery of TPC tends to increase as the solvent/sample ratio improves until the maximum recovery of TPC was obtained when the solvent/sample ratio reached 12 mL/g. Beyond the solvent/sample ratio of 12 mL/g, the occurrence of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> declined. For the maxima, antioxidant potentials through DPPH and ABTS (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bii); the lowest IC<sub>50</sub> values (strong antioxidant) were reached at a solvent/sample ratio of 12 mL/g. Beyond 12 mL/g, the antioxidant potentials declined with increasing solvent volume. This is industrial valuable because reducing solvent usage that achieves higher recovery favour the production cost (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Bouras et al. <xref ref-type=\"bibr\" rid=\"CR11\">2015</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Moreover, the higher solvent volume tends to increase the concentration gradient; this can then lead to an increased diffusion rate of solute into the solvent. Therefore, 8–16 mL/g was chosen as a range of values for the optimization. However, 12 mL/g was considered as the fixed solvent/sample ratio when studying the impacts of other factors.</p></sec><sec id=\"Sec16\"><title>Contributing impact of microwave power</title><p id=\"Par31\">In microwave-enhanced extraction, the plant sample absorbs microwave energy, this energy changes to heat and evaporation kick-starts from the left-over moisture content in the plant matrix. The evaporated water then causes the pressure body-up within the cell wall, leading to cell rupture in the plant sample. Thus, the bioactive compounds within the plant matrix will leach out into the surrounding solvent after cell rupture (Mandal et al. <xref ref-type=\"bibr\" rid=\"CR23\">2007</xref>).</p><p id=\"Par32\">In this, the impact of microwave power on the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves was investigated by fixing the following factors: extraction time (120 s), solvent/sample ratio (12 mL/g), ethanol concentration (20%), and temperature (20 °C) (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ci and Cii). There was a slight increase in the TPC recovery as the power increased. At microwave power of 300 W, maximum TPC was achieved, after which the recovery of phenolic compounds tends to decrease against increasing microwave power. In contrast, the antioxidants (IC<sub>50</sub> values) for both DPPH and ABTS declined with increasing microwave power. This is justifiable because a lower IC<sub>50</sub> value indicates higher antioxidant potential. The minima IC<sub>50</sub> values for DPPH and ABTS were obtained at 300 W of microwave power. Beyond 300 W, the phenolic compounds and antioxidants declined with increasing power. Increasing the microwave power can propel increasing temperature. At higher temperatures, the viscosity and surface tension of extracting solvent will be reduced as the diffusion coefficient and solubility increase. Nonetheless, overexposure of the bioactive compounds already leached out into the solvent to higher microwave power propels the degradation of these phenolic compounds and antioxidants (thermal-sensitive phytochemicals) (Quoc <xref ref-type=\"bibr\" rid=\"CR26\">2021</xref>). Thus, 200–400 W was selected as a range of values for the optimization. Nevertheless, 300 W was considered as the fixed microwave power when studying the remaining factors.</p></sec><sec id=\"Sec17\"><title>Contributing impact of ethanol concentration</title><p id=\"Par33\">Organic solvents are mostly used in the extraction of phenolic compounds and antioxidants from plant materials. These solvents are mostly mixed with water because it enhances the water-soluble phytochemicals in the plant sample. Moreover, the mixture of organic solvents with water tends to create a moderate polar medium. Out of several organic solvents used in extraction, ethanol has been considered safe because it is environment-friendly (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>).</p><p id=\"Par34\">The influence of ethanol concentration was investigated by fixing extraction time, microwave power, solvent/sample ratio, and temperature at 120 s, 300 W, 12 mL/g, and 20 °C, respectively. As provided in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Di and Dii, the recovery of TPC tends to increase and IC<sub>50</sub> values (showing higher antioxidants) declined as ethanol concentration in water increased until the maximum recoveries of TPC and antioxidants (both DPPH and ABTS) were achieved ethanol concentration of 40%. Beyond this concentration, the phenolic compounds and antioxidants in the leaf extract of <italic>C. olitorius</italic> declined. Hence, the ethanol concentration ranged between 30 and 50% for the optimization. However, 40% was considered the fixed concentration for the next factor.</p></sec><sec id=\"Sec18\"><title>Contributing impact of temperature</title><p id=\"Par35\">In solid–liquid extraction, the higher temperature tends to upsurge the release of phenolic compounds and antioxidants from the plant matrix. This is due to its influence on mass transfer (diffusion coefficient), equilibrium (solubility) and stability of phytochemicals (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Although, heating can soften the plant tissue and weaken the interactions of phenolic compounds; hence, it promotes the migration of bioactive compounds into the extracting solvent. Nonetheless, a higher temperature can cause degradation and oxidation of phenolic compounds and antioxidants in the plant sample.</p><p id=\"Par36\">At fixed extraction time (120 s), microwave power (300 W), solvent/sample ratio (12 mL/g), and an ethanol concentration of 40%, the impact of the temperature was as well studied. In Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ei and Eii, it can be observed that temperature did not influence the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. This might be because the increasing temperature beyond a certain limit can profuse solvent loss through augment and vaporization; leading to the destruction of the phenolic compounds and antioxidants (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>).</p></sec></sec><sec id=\"Sec19\"><title>Optimization study and modelling of microwave-enhanced extraction of <italic>C. olitorius</italic> leaves</title><sec id=\"Sec20\"><title>Model fittings</title><p id=\"Par37\">The results from OFAT experimental trials have established the impacts of extraction time, solvent/sample ratio, microwave power, and ethanol concentration. These four factors were optimized through microwave-enhanced extraction to recover higher yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. In this study, FCCCD was employed to optimize this process. This was used because it is the simplest form of RSM. A total of 30 experimental trials were generated (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). The obtained outcomes from the randomized experimental trials are presented in Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>. These results were further analysed using analysis of variance to investigate the contributing impacts of individual factors considered and their interactions. The effectiveness of the RSM model is best explained through the satisfaction of model performance by measuring the determination coefficients (<italic>R</italic><sup>2</sup>), <italic>F</italic>-values, lack of fit, coefficient of variation, adjusted <italic>R</italic><sup>2</sup>, adequate precision, and <italic>p</italic> values (the value less than 0.05 indicates significant and vice versa).<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Design of experiments for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Std</p></th><th align=\"left\"><p>Extraction time, <italic>X</italic><sub>1</sub> (s)</p></th><th align=\"left\"><p>Microwave power, <italic>X</italic><sub>2</sub> (W)</p></th><th align=\"left\"><p>Solvent/sample, <italic>X</italic><sub>3</sub> (mL/g)</p></th><th align=\"left\"><p>Ethanol concentration, <italic>X</italic><sub>4</sub> (%)</p></th><th align=\"left\"><p>TPC, <italic>Y</italic><sub>1</sub> (mg GAE/10 g d.b.)</p></th><th align=\"left\"><p>DPPH, <italic>Y</italic><sub>2</sub> (µg/mL)</p></th><th align=\"left\"><p>ABTS, <italic>Y</italic><sub>3</sub> (µg/mL)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.69 ± 4.07</p></td><td align=\"char\" char=\"±\"><p>113.79 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>74.13 ± 1.08</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.00 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>109.47 ± 2.12</p></td><td align=\"char\" char=\"±\"><p>70.24 ± 1.05</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.42 ± 4.29</p></td><td align=\"char\" char=\"±\"><p>125.98 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>86.68 ± 2.43</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.72 ± 3.00</p></td><td align=\"char\" char=\"±\"><p>108.50 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>69.22 ± 1.54</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.50 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>120.00 ± 1.06</p></td><td align=\"char\" char=\"±\"><p>80.37 ± 1.58</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.33 ± 1.09</p></td><td align=\"char\" char=\"±\"><p>119.60 ± 2.14</p></td><td align=\"char\" char=\"±\"><p>80.47 ± 1.99</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>329.92 ± 4.76</p></td><td align=\"char\" char=\"±\"><p>117.00 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>77.79 ± 2.19</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.14 ± 4.11</p></td><td align=\"char\" char=\"±\"><p>103.72 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>64.50 ± 1.56</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>342.17 ± 2.33</p></td><td align=\"char\" char=\"±\"><p>120.02 ± 2.28</p></td><td align=\"char\" char=\"±\"><p>80.83 ± 1.05</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>341.89 ± 2.36</p></td><td align=\"char\" char=\"±\"><p>99.92 ± 3.05</p></td><td align=\"char\" char=\"±\"><p>60.60 ± 2.11</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>336.78 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>94.48 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>55.25 ± 1.55</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>330.63 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>70.67 ± 1.65</p></td><td align=\"char\" char=\"±\"><p>31.36 ± 1.00</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 2.11</p></td><td align=\"char\" char=\"±\"><p>120.15 ± 2.04</p></td><td align=\"char\" char=\"±\"><p>81.00 ± 1.05</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>340.18 ± 2.23</p></td><td align=\"char\" char=\"±\"><p>107.99 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.66 ± 1.00</p></td></tr><tr><td align=\"left\"><p>15</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>332.82 ± 2.05</p></td><td align=\"char\" char=\"±\"><p>90.45 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>51.22 ± 1.02</p></td></tr><tr><td align=\"left\"><p>16</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>325.52 ± 2.99</p></td><td align=\"char\" char=\"±\"><p>70.29 ± 1.77</p></td><td align=\"char\" char=\"±\"><p>31.08 ± 1.03</p></td></tr><tr><td align=\"left\"><p>17</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>343.47 ± 4.22</p></td><td align=\"char\" char=\"±\"><p>101.04 ± 1.22</p></td><td align=\"char\" char=\"±\"><p>61.81 ± 1.06</p></td></tr><tr><td align=\"left\"><p>18</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.78 ± 3.08</p></td><td align=\"char\" char=\"±\"><p>85.28 ± 2.06</p></td><td align=\"char\" char=\"±\"><p>46.00 ± 1.08</p></td></tr><tr><td align=\"left\"><p>19</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>346.89 ± 2.17</p></td><td align=\"char\" char=\"±\"><p>102.02 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>62.82 ± 1.00</p></td></tr><tr><td align=\"left\"><p>20</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.51 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>85.20 ± 1.18</p></td><td align=\"char\" char=\"±\"><p>45.96 ± 0.99</p></td></tr><tr><td align=\"left\"><p>21</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.70 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>102.39 ± 2.35</p></td><td align=\"char\" char=\"±\"><p>63.10 ± 1.69</p></td></tr><tr><td align=\"left\"><p>22</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>339.54 ± 1.98</p></td><td align=\"char\" char=\"±\"><p>104.14 ± 1.23</p></td><td align=\"char\" char=\"±\"><p>64.93 ± 2.01</p></td></tr><tr><td align=\"left\"><p>23</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>333.32 ± 4.03</p></td><td align=\"char\" char=\"±\"><p>90.82 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>51.59 ± 1.03</p></td></tr><tr><td align=\"left\"><p>24</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.84 ± 1.34</p></td><td align=\"char\" char=\"±\"><p>29.58 ± 1.03</p></td></tr><tr><td align=\"left\"><p>25</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>344.00 ± 3.11</p></td><td align=\"char\" char=\"±\"><p>86.40 ± 2.07</p></td><td align=\"char\" char=\"±\"><p>47.30 ± 1.26</p></td></tr><tr><td align=\"left\"><p>26</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.74 ± 2.16</p></td><td align=\"char\" char=\"±\"><p>90.08 ± 2.02</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 1.00</p></td></tr><tr><td align=\"left\"><p>27</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.91 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>90.26 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>51.01 ± 1.00</p></td></tr><tr><td align=\"left\"><p>28</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.68 ± 3.27</p></td><td align=\"char\" char=\"±\"><p>90.01 ± 1.15</p></td><td align=\"char\" char=\"±\"><p>50.78 ± 1.22</p></td></tr><tr><td align=\"left\"><p>29</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.77 ± 1.97</p></td><td align=\"char\" char=\"±\"><p>90.09 ± 1.04</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 2.14</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.92 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>90.23 ± 2.22</p></td><td align=\"char\" char=\"±\"><p>51.00 ± 1.14</p></td></tr></tbody></table></table-wrap></p><p id=\"Par38\">Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> presents the results obtained for the analysis of variance in recovering phenolic compounds and antioxidants (DPPH and ABTS assays). It can be observed that the three models of the responses (TPC, DPPH and ABTS assays) are significant. Out of the four considered factors for the optimization study, microwave power and ethanol concentration are the most significant with a <italic>p</italic> value &lt; 0.0001, showing model significance at a 99.9% confidence level. Extraction time is also significant, whereas solvent/sample ratio was insignificant for the three responses. Moreover, the lack of fit for the models was far greater than 0.05; this shows that the models are significant. The <italic>R</italic><sup>2</sup> values of the TPC, DPPH and ABTS assays ranged between 0.98 and 0.99; these are indications that the predictive models can explain at least 98% of the result variations. The adjusted and predicted <italic>R</italic><sup>2</sup> values for the three predictive models were closer to 1, indicating significant correlations between the actual and predicted results (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>). The lower value of the coefficient of variation below 10% shows that the experimental results were reproducible and reliable. The obtained results indicate for CV values for TPC, and antioxidant assays (DPPH and ABTS) were 0.33, 2.04 and 3.44, respectively. It should be noted that the contributions of the quadratic models are notable. Thus, the predictive models for the phenolic compounds and antioxidant assays are well suitable for the prediction.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Analysis of variance for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"6\"><p>TPC</p></th><th align=\"left\" colspan=\"6\"><p>DPPH</p></th><th align=\"left\" colspan=\"6\"><p>ABTS</p></th></tr><tr><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th></tr></thead><tbody><tr><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>1856.45</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>132.60</p></td><td align=\"char\" char=\".\"><p>105.49</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6946.43</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>496.17</p></td><td align=\"char\" char=\".\"><p>121.69</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6922.90</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>494.49</p></td><td align=\"char\" char=\".\"><p>117.32</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"char\" char=\".\"><p>21.64</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"char\" char=\".\"><p>221.39</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"char\" char=\".\"><p>212.43</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"char\" char=\".\"><p>88.47</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"char\" char=\".\"><p>293.10</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"char\" char=\".\"><p>281.19</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"char\" char=\".\"><p>3.99</p></td><td align=\"left\"><p>0.0643</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"char\" char=\".\"><p>0.90</p></td><td align=\"left\"><p>0.3582</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"char\" char=\".\"><p>0.98</p></td><td align=\"left\"><p>0.3386</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"char\" char=\".\"><p>358.38</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"char\" char=\".\"><p>375.77</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"char\" char=\".\"><p>360.63</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"char\" char=\".\"><p>7.67</p></td><td align=\"left\"><p>0.0143</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"char\" char=\".\"><p>21.84</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"char\" char=\".\"><p>21.89</p></td><td align=\"left\"><p>0.0003</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"char\" char=\".\"><p>9.47</p></td><td align=\"left\"><p>0.0077</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"left\"><p>0.0276</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"char\" char=\".\"><p>5.81</p></td><td align=\"left\"><p>0.0292</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"char\" char=\".\"><p>10.83</p></td><td align=\"left\"><p>0.0049</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"char\" char=\".\"><p>25.45</p></td><td align=\"left\"><p>0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"char\" char=\".\"><p>26.23</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"char\" char=\".\"><p>4.93</p></td><td align=\"left\"><p>0.0422</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"char\" char=\".\"><p>27.96</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"char\" char=\".\"><p>26.94</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"char\" char=\".\"><p>103.93</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"char\" char=\".\"><p>201.06</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"char\" char=\".\"><p>196.65</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"char\" char=\".\"><p>5.44</p></td><td align=\"left\"><p>0.0340</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"char\" char=\".\"><p>0.022</p></td><td align=\"left\"><p>0.8829</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"char\" char=\".\"><p>0.017</p></td><td align=\"left\"><p>0.8990</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"char\" char=\".\"><p>10.27</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"char\" char=\".\"><p>15.19</p></td><td align=\"left\"><p>0.0014</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"char\" char=\".\"><p>14.52</p></td><td align=\"left\"><p>0.0017</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"char\" char=\".\"><p>0.89</p></td><td align=\"left\"><p>0.3605</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"char\" char=\".\"><p>18.12</p></td><td align=\"left\"><p>0.0007</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"char\" char=\".\"><p>17.56</p></td><td align=\"left\"><p>0.0008</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"char\" char=\".\"><p>37.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"char\" char=\".\"><p>142.87</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"char\" char=\".\"><p>137.75</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"char\" char=\".\"><p>85.01</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"char\" char=\".\"><p>45.27</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"char\" char=\".\"><p>44.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>18.86</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>1.26</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>61.16</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.08</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>63.22</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.21</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>16.10</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>1.61</p></td><td align=\"char\" char=\".\"><p>2.92</p></td><td align=\"left\"><p>0.1243</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>49.50</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>4.95</p></td><td align=\"char\" char=\".\"><p>2.12</p></td><td align=\"left\"><p>0.2103</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>52.42</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>5.24</p></td><td align=\"char\" char=\".\"><p>2.42</p></td><td align=\"left\"><p>0.1701</p></td></tr><tr><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>2.76</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>0.55</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>11.66</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.33</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>10.81</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.16</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>1875.30</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>7007.59</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>6986.12</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>0.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>2.04</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>3.44</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>118.47</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>368.81</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>389.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9899</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9913</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9910</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9806</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9831</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9825</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9368</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9474</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9443</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>28.799</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>39.281</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>38.612</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr></tbody></table><table-wrap-foot><p><italic>DF</italic> degree of freedom</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec21\"><title>Effects of microwave-enhanced extraction factors on TPC and antioxidant assays (DPPH and ABTS) in <italic>C. olitorius</italic> leaves</title><p id=\"Par39\">The influences of microwave-enhanced extraction factors on the phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. The phenolic compounds in the extract of <italic>C. olitorius</italic> leaves varied from 325.52 ± 2.99 to 346.89 ± 2.17 mg GAE/10 g d.b. (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>). The least TPC was obtained at extraction time of 180 s, microwave power of 400 W, solvent/sample ratio of 16 mL/g, and ethanol concentration of 50%, whereas the highest TPC was obtained at extraction time of 120 s, microwave power of 200 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 40%. The antioxidant assays (DPPH and ABTS) from the leaf extract of <italic>C. olitorius</italic> ranged from 68.84 ± 1.34 to 125.98 ± 1.16 µg/mL and 29.58 ± 1.03 to 86.68 ± 2.43 µg/mL, respectively. The least (extraction time of 60 s, microwave power of 400 W, solvent/sample ratio of 8 mL/g, and ethanol concentration of 30%) and highest (extraction time of 120 s, microwave power of 300 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%) yields of antioxidants from <italic>C. olitorius</italic> leaves were obtained. Analysis of variance analysis (ANOVA) results indicate that extraction time, microwave power and ethanol concentration significantly influenced the recoveries of phenolic compounds from <italic>C. olitorius</italic> leaves; however, the solvent/sample ratio was insignificant. The quadratic effects of all the factors except microwave power are significant (<italic>p</italic> value &lt; 0.05). Besides, the interactions between all the considered factors significantly impact the microwave-enhanced extraction of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves except for the interactions between solvent/sample ratio and ethanol concentration (<italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub>) on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves which were insignificant (<italic>p</italic> value &gt; 0.05). The quadratic model for the coded values of TPC, DPPH and ABTS assays is estimated from Eqs. (<xref ref-type=\"disp-formula\" rid=\"Equ2\">2</xref>–<xref ref-type=\"disp-formula\" rid=\"Equ4\">4</xref>).<disp-formula id=\"Equ2\"><label>2</label><alternatives><mml:math display=\"block\" id=\"Equ2_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>345.43</mml:mn><mml:mo>-</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.49</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.53</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>5</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.78</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.92</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>0.62</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.65</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.23</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>0.66</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>4.24</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>6.42</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ2_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{1} &amp; = &amp; 345.43 - 1.23X_{1} - 2.49X_{2} - 0.53X_{3} + 5X_{4} - 0.78X_{1} X_{2} - 0.86X_{1} X_{3} - 0.92X_{1} X_{4} \\\\ \\quad - 0.62X_{2} X_{3} - 2.86X_{2} X_{4} - 0.65X_{3} X_{4} - 2.23X_{1}^{2} - 0.66X_{2}^{2} - 4.24X_{3}^{2} - 6.42X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ2.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ3\"><label>3</label><alternatives><mml:math display=\"block\" id=\"Equ3_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>88.89</mml:mn><mml:mo>-</mml:mo><mml:mn>7.08</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.15</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.45</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.36</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.55</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.67</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.16</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.076</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.89</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.99</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.44</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ3_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{2} &amp; = &amp; 88.89 - 7.08X_{1} - 8.15X_{2} + 0.45X_{3} - 9.23X_{4} - 2.36X_{1} X_{2} + 1.23X_{1} X_{3} - 2.55X_{1} X_{4} \\\\ \\quad - 2.67X_{2} X_{3} - 7.16X_{2} X_{4} + 0.076X_{3} X_{4} + 4.89X_{1}^{2} + 5.34X_{2}^{2} + 14.99X_{3}^{2} - 8.44X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ3.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ4\"><label>4</label><alternatives><mml:math display=\"block\" id=\"Equ4_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>49.67</mml:mn><mml:mo>-</mml:mo><mml:mn>7.05</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.11</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.48</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.19</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.40</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.24</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.63</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.66</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.20</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.066</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.86</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.97</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.46</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ4_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{3} &amp; = &amp; 49.67 - 7.05X_{1} - 8.11X_{2} + 0.48X_{3} - 9.19X_{4} - 2.40X_{1} X_{2} + 1.24X_{1} X_{3} - 2.63X_{1} X_{4} \\\\ \\quad - 2.66X_{2} X_{3} - 7.20X_{2} X_{4} + 0.066X_{3} X_{4} + 4.86X_{1}^{2} + 5.34X_{2}^{2} + 14.97X_{3}^{2} - 8.46X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ4.gif\"/></alternatives></disp-formula>where <italic>X</italic><sub>1</sub> represents extraction time (s), <italic>X</italic><sub>2</sub> indicates microwave power (W), <italic>X</italic><sub>3</sub> represents solvent/sample ratio (mL/g), <italic>X</italic><sub>4</sub> is the ethanol concentration, <italic>Y</italic><sub>1</sub>, <italic>Y</italic><sub>2</sub> and <italic>Y</italic><sub>3</sub> are the responses TPC, DPPH and ABTS assays, accordingly.</p><p id=\"Par40\">The three-dimensional plot is the representation of regression equations to show the interaction between independent and dependent factors. This plot is presented in different shapes; this illustrates whether the factor or its mutual interaction is significant or not (Halee et al. <xref ref-type=\"bibr\" rid=\"CR15\">2020</xref>). The contributing impacts of individual factors and their mutual interactions on the phenolic compounds and antioxidants are presented using three-dimensional graphical representations in Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a–f, <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a–f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a–f. It should be noted that antioxidant increases with reduced IC<sub>50</sub> values (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the phenolic compounds from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig2_HTML.png\"/></fig><fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the DPPH assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig3_HTML.png\"/></fig><fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the ABTS assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig4_HTML.png\"/></fig></p><p id=\"Par41\">The contributing effect of interactions between microwave power and extraction time on TPC at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is shown in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a. A gradual increase in extraction time and a slight decline in microwave power slowly increase the yield of TPC. Nonetheless, beyond 131.37 s and 305.08 W, TPC started to decline. For Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b, increasing both solvent/sample and extraction time at a fixed microwave power of 300 W and ethanol concentration of 40% gradually increased the recovery of TPC from <italic>C. olitorius</italic> leaves. After the certain limits of microwave power of 305.08 W and extraction time of 131.37 s, the phenolic compounds declined. In Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c, TPC from <italic>C. olitorius</italic> leaves increases with increasing ethanol concentration and reduces the extraction time. After these factors reached 45.07% and 69.85 s, respectively, TPC gradually decreases. The interaction between solvent/sample ratio and microwave power is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d; increasing solvent/sample ratio, and reducing microwave power increased the recovery of phenolic compounds. At solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recovery of TPC decreased. For the interaction between ethanol concentration and extraction time (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c), increasing ethanol concentration and reducing microwave power favour increasing phenolic compounds (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>e). At the ethanol concentration and microwave power of 45.07% and 257.62 W, optimal yields of TPC and antioxidants were achieved; beyond these values, the recoveries declined. Finally, the effects of interactions between ethanol concentration and solvent/sample ratio on the recoveries of TPC from <italic>C. olitorius</italic> leaves (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>f). Reducing the solvent/sample ratio and increasing the ethanol concentration favour the recovery of optimal TPC from the considered sample. Hence, the optimal predicted TPC yield of 343.327 mg GAE/10 g d.b. at the microwave-enhanced extraction conditions including extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p><p id=\"Par42\">Moreover, the influence of interactions between microwave power and extraction time on antioxidants (DPPH and ABTS assays) in the leaf extract of <italic>C. olitorius</italic> at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a. Increasing the extraction time and a slight decrease in microwave power gradually increases the antioxidant yields. The antioxidants in leaf extract declined as the microwave power and extraction time reached 131.37 s and 305.08 W, respectively. Besides, the interaction between solvent/sample ratio and extraction time is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>b and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b showed that increasing solvent/sample ratio and declining extraction time favoured antioxidant recoveries from <italic>C. olitorius</italic> leaf at a fixed microwave power of 300 W and ethanol concentration of 40%. Beyond microwave power of 305.08 W and extraction time of 131.37 s, the antioxidants declined. For Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, the antioxidants increase with increasing ethanol concentration and extraction time; beyond 45.07% and 69.85 s, respectively, gradual declination of antioxidants was observed. Increasing solvent/sample ratio and reducing microwave power increased the recoveries of phenolic compounds (Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d and <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>d). On reaching solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recoveries of antioxidants declined. Similarly, in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, increasing ethanol concentration and reducing microwave power increase the recoveries of antioxidants from leaf extract of <italic>C. olitorius</italic> (Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>e and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>e). Beyond the ethanol concentration and microwave power of 45.07% and 257.62 W, respectively, the recoveries of antioxidants declined. In conclusion, the interactions between ethanol concentration and solvent/sample ratio on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves as presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>f reflected that increasing solvent/sample ratio and reducing ethanol concentration yielded increasing antioxidants. Thus, the optimal predicted antioxidant yields (both DPPH and ABTS assays) were 68.8396 and 29.6220 µg/mL, respectively, at the microwave-enhanced extraction conditions of extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p></sec><sec id=\"Sec22\"><title>Validations of the generated models</title><p id=\"Par43\">Based on the predicted values from the Design Expert® software, the optimal conditions to achieve phenolic compounds (TPC of 343.327 mg GAE/10 g d.b.) and antioxidants (IC<sub>50</sub> values of 68.8396 and 29.6220 µg/mL for DPPH and ABTS assays, respectively) from extracts from <italic>C. olitorius</italic> leaves were found at extraction time of 131.37 s, microwave power of 305.08 W, solvent/sample ratio of 11.67 mL/g, and ethanol concentration of 50% with the desirability of 0.942. The validation experiments were carried out using approximated values of the predicted values such as extraction time of 131 s, microwave power of 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%. Three replicate experiments were carried out using these conditions, and the mean was determined to affirm the generated outcomes from the software. The experimental trials amounted to TPC of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for DPPH and ABTS assays, respectively, at 131 s, 305 W, 12 mL/g, and 50% of extraction time, microwave power, solvent/sample ratio, and ethanol concentration, respectively. Using Chi-squared analysis, the results obtained indicated no significant difference between the experimental and predicted values from the software.</p><p id=\"Par44\">As earlier reported, some previous studies had evaluated the contents of total phenols and antioxidants in the leaf extract of <italic>C. olitorius</italic>; however, no study was reported on recovering phenolic compounds and antioxidants from this plant through microwave-enhanced technique. A recent finding reported the recoveries of total phenols (0.10–9.81 µ GA/g), DPPH inhibition (1.4–70.3%) and ABTS inhibition (83–93%) (Obeng et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). In another recent study, a total antioxidant capacity of 165.66 ± 1.30 mg ascorbic acid/g dry extract was reported in the leaf extract of <italic>C. olitorius</italic> (Ali et al. <xref ref-type=\"bibr\" rid=\"CR8\">2021</xref>). The TPC of 2.5–32.0 Mg TE/g was reported in a recent study; it was established that phenolic compounds and antioxidants from this plant are unaffected by cultivation treatments. Moreover, it had been reported that the mode of extraction can alter the phenolic compounds in the <italic>C. olitorius</italic> leaves (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). The antioxidant from <italic>C. olitorius</italic> leaves was reported to be 91.65 ± 0.15% using DPPH inhibition with the maceration extraction technique (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). The results from the current study established the importance of using the microwave-enhanced extraction method to recover optimal yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Other than this, this plant can further be used in combating food insecurity because it is rich in phenolic compounds.</p></sec></sec><sec id=\"Sec23\"><title>LC-QToF-MS analysis of the optimized extract from <italic>C. olitorius</italic> leaves</title><p id=\"Par45\">The use of LC-QToF-MS analysis to characterize phenolic compounds has been a current trend in natural product chemistry; this analysis has proven to be efficient in tentatively identifying phenolic compounds in diverse plant samples (Alara et al. <xref ref-type=\"bibr\" rid=\"CR5\">2018c</xref>, <xref ref-type=\"bibr\" rid=\"CR4\">2018b</xref>; Tan et al. <xref ref-type=\"bibr\" rid=\"CR29\">2020</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). Unifi software processed the data. Thus, the phenolic profile of the leaf extract of <italic>C. olitorius</italic> at optimized microwave-enhanced extraction conditions was characterized using the LC-QToF-MS system (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). The characterization and identification of phytochemicals in the leaf extract of <italic>C. olitorius</italic> were done by comparing retention time (RT), mass spectrometric (MS), and mass error between mass observed (which must be less than 10 ppm for viable identified compound) (Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). A total of fourteen phenolic compounds were identified in the extract of <italic>C. olitorius</italic> leaves as presented in Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>. The identified phytochemicals were 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose as presented in the Supplementary materials (Figs. S1–S14).<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Negative ionization plots for the identified chemical compounds in the extract from <italic>C. olitorius</italic> leaf</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig5_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>The tentative assignments of chemical compounds in the leaf extract of <italic>C. olitorius</italic></p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>S/N</p></th><th align=\"left\"><p>Chemical compound</p></th><th align=\"left\"><p>RT (min)</p></th><th align=\"left\"><p>[M-H]- (m/z)</p></th><th align=\"left\"><p>Mass error (ppm)</p></th><th align=\"left\"><p>Molecular formula</p></th><th align=\"left\"><p>Response</p></th><th align=\"left\"><p>Adducts</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>1-Galloyl-glucose</p></td><td align=\"char\" char=\".\"><p>0.58</p></td><td align=\"char\" char=\".\"><p>331.0673</p></td><td align=\"char\" char=\".\"><p>0.8</p></td><td align=\"left\"><p>C<sub>13</sub>H<sub>16</sub>O<sub>10</sub></p></td><td align=\"left\"><p>6533</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>1,3,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>0.59</p></td><td align=\"char\" char=\".\"><p>677.1499</p></td><td align=\"char\" char=\".\"><p>−1.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>3235</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>Procyanidin C-1</p></td><td align=\"char\" char=\".\"><p>0.61</p></td><td align=\"char\" char=\".\"><p>865.1981</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>45</sub>H<sub>38</sub>O<sub>18</sub></p></td><td align=\"left\"><p>105,272</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>4,4′,5,6-Tetrahydroxystilbene</p></td><td align=\"char\" char=\".\"><p>0.75</p></td><td align=\"char\" char=\".\"><p>289.0725</p></td><td align=\"char\" char=\".\"><p>2.5</p></td><td align=\"left\"><p>C<sub>14</sub>H<sub>12</sub>O<sub>4</sub></p></td><td align=\"left\"><p>6478</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>3,4,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>3.11</p></td><td align=\"char\" char=\".\"><p>677.1517</p></td><td align=\"char\" char=\".\"><p>0.7</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>7900</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>5-Desgalloylstachyurin</p></td><td align=\"char\" char=\".\"><p>4.00</p></td><td align=\"char\" char=\".\"><p>783.0674</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>3123</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>Sanguiin H-4</p></td><td align=\"char\" char=\".\"><p>5.31</p></td><td align=\"char\" char=\".\"><p>633.0731</p></td><td align=\"char\" char=\".\"><p>−0.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2263</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>Corilagin_1</p></td><td align=\"char\" char=\".\"><p>5.85</p></td><td align=\"char\" char=\".\"><p>679.0783</p></td><td align=\"char\" char=\".\"><p>−0.8</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2547</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>1-O-Galloylpedunculagin</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"char\" char=\".\"><p>981.0849</p></td><td align=\"char\" char=\".\"><p>−0.1</p></td><td align=\"left\"><p>C<sub>41</sub>H<sub>28</sub>O<sub>26</sub></p></td><td align=\"left\"><p>3356</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>Laevigatin A</p></td><td align=\"char\" char=\".\"><p>6.47</p></td><td align=\"char\" char=\".\"><p>847.0839</p></td><td align=\"char\" char=\".\"><p>−0.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>26</sub>O<sub>23</sub></p></td><td align=\"left\"><p>2292</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>Pedunculagin</p></td><td align=\"char\" char=\".\"><p>6.94</p></td><td align=\"char\" char=\".\"><p>829.0728</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>6165</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>2,4,6-Tri-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.40</p></td><td align=\"char\" char=\".\"><p>681.0937</p></td><td align=\"char\" char=\".\"><p>−1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>3128</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>1,3,6-Trigalloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.91</p></td><td align=\"char\" char=\".\"><p>681.0954</p></td><td align=\"char\" char=\".\"><p>1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2265</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>1,2,3,6-Tetra-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>8.14</p></td><td align=\"char\" char=\".\"><p>833.1060</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>28</sub>O<sub>22</sub></p></td><td align=\"left\"><p>5153</p></td><td align=\"left\"><p>–H</p></td></tr></tbody></table></table-wrap></p><p id=\"Par46\">In simplicity, identified 1-galloyl-glucose, 1-O-galloylpedunculagin, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose belong to tannins (naturally occurring polyphenols). Another identified tannin was 5-desgalloylstachyurin. These compounds are rich in antioxidants and can alleviate different forms of cancer and free radicals (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR37\">2009</xref>). Other identified compounds were 1,3,5-O-tricaffeoylquinic acid and 3,4,5-O-tricaffeoylquinic acid; these compounds were reported as polyphenols that are rich in antioxidants (Wang et al. <xref ref-type=\"bibr\" rid=\"CR33\">2017</xref>). Procyanidin C-1 belongs to polyflavonoids and bioflavonoids; this compound has been reported natural occurring antioxidant from a plant source (Xu et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>). The identified stilbene monomer in the leaf extract of <italic>C. olitorius</italic> was 4,4′,5,6-tetrahydroxystilbene. This phytochemical has been reported to possess diverse pharmacological features which antioxidants (Zhang and Chen <xref ref-type=\"bibr\" rid=\"CR36\">2018</xref>). Sanguiin H-4 had been reported to possess antioxidant properties (Lachowicz et al. <xref ref-type=\"bibr\" rid=\"CR19\">2020</xref>). Corilagin_1 and laevigatin A are compounds that are rich in antioxidants as well. Therefore, extract from <italic>C. olitorius</italic> leaf has been established as a source of phenolic compounds and natural antioxidants.</p></sec></sec><sec id=\"Sec24\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par47\">Seeking alternative sources of antioxidants, especially plant-based, will continue to be studied. Given this, this study investigated <italic>C. olitorius</italic> leaves as a source of phenolic compounds and antioxidants by considering a technique that can recover yields (microwave-enhanced extraction) using ethanol/water as extracting solvent. The optimized process established optimal recoveries of phenolic compounds and antioxidants which were higher compared to previously reported results on <italic>C. olitorius</italic> leaves. In addition to this, the phenolic compounds that were comprehensively identified in the extract (fourteen in total) ascertain <italic>C. olitorius</italic> leaves as great sources of antioxidants. This outcome is an indication that <italic>C. olitorius</italic> leaves can continue to be encouraged in daily consumption, a way of combating food insecurity in the world. Thus, future study can focus on isolation and investigation of the tentatively identified phenolic compounds. These can further explain the pharmacological importance of this plant sample.</p></sec></body><back><ack><title>Acknowledgements</title><p>The authors thankfully acknowledge Eastern Unity Technology, Malaysia for the financial support through the grant (UIC190806).</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par48\">We declare none.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abir</surname><given-names>RR</given-names></name><name><surname>Marjia</surname><given-names>M</given-names></name><name><surname>Rakhi</surname><given-names>NN</given-names></name><name><surname>Saha</surname><given-names>O</given-names></name><name><surname>Hossain</surname><given-names>MA</given-names></name><name><surname>Rahaman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">In vitro comparative analysis of antibacterial activity of different fractions of <italic>Corchorus capsularis</italic> and <italic>Corchorus olitorius</italic> leaves extracts</article-title><source>Bangladesh J Microbiol</source><year>2019</year><volume>36</volume><fpage>69</fpage><lpage>73</lpage><pub-id pub-id-type=\"doi\">10.3329/bjm.v36i2.45530</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolics from Hibiscus sabdariffa calyces: Kinetic modelling and process intensification</article-title><source>Ind Crops Prod</source><year>2019</year><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2019.05.053</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\"><italic>Vernonia cinerea</italic> leaves as the source of phenolic compounds, antioxidants, and anti-diabetic activity using microwave-assisted extraction technique</article-title><source>Ind Crop Prod</source><year>2018</year><volume>122</volume><fpage>533</fpage><lpage>544</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1Wru7nJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.06.034</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Metabolic profiling of flavonoids, saponins, alkaloids, and terpenoids in the extract from <italic>Vernonia cinerea</italic> leaf using LC-Q-TOF-MS</article-title><source>J Liq Chromatogr Relat Technol</source><year>2018</year><pub-id pub-id-type=\"doi\">10.1080/10826076.2018.1511995</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Hassan</surname><given-names>Z</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Dataset on LC-Q-TOF/MS tentative identification of phytochemicals in the extract of <italic>Vernonia amygdalina</italic> leaf through positive ionization</article-title><source>Data Br</source><year>2018</year><volume>21</volume><fpage>1686</fpage><lpage>1689</lpage><pub-id pub-id-type=\"doi\">10.1016/j.dib.2018.10.159</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ali</surname><given-names>HA</given-names></name><name><surname>Zain</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from Carica papaya leaves: an optimization study and LC-QTOF-MS analysis</article-title><source>Futur Foods</source><year>2021</year><volume>3</volume><fpage>100035</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XhtlSmurnO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fufo.2021.100035</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name></person-group><article-title xml:lang=\"en\">Extraction of phenolic compounds: a review</article-title><source>Curr Res Food Sci</source><year>2021</year><volume>4</volume><fpage>200</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Cjsb%2FK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.crfs.2021.03.011</pub-id><pub-id pub-id-type=\"pmid\">33899007</pub-id><pub-id pub-id-type=\"pmcid\">8058613</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ali</surname><given-names>MM</given-names></name><name><surname>Ahmed</surname><given-names>KS</given-names></name><name><surname>Hossain</surname><given-names>H</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Rokeya</surname><given-names>B</given-names></name><name><surname>Rahman</surname><given-names>MT</given-names></name><name><surname>Jahan</surname><given-names>IA</given-names></name><name><surname>Rahman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Total antioxidant capacity and profiling of polyphenolic compounds in jute leaves by hplc-dad</article-title><source>Food Res</source><year>2021</year><volume>5</volume><fpage>343</fpage><lpage>348</lpage><pub-id pub-id-type=\"doi\">10.26656/fr.2017.5(1).358</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Andersen</surname><given-names>LT</given-names></name><name><surname>Thilsted</surname><given-names>SH</given-names></name><name><surname>Nielsen</surname><given-names>BB</given-names></name><name><surname>Rangasamy</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Food and nutrient intakes among pregnant women in rural Tamil Nadu</article-title><source>South India Public Heal Nutr</source><year>2003</year><volume>6</volume><fpage>131</fpage><lpage>137</lpage><pub-id pub-id-type=\"doi\">10.1079/PHN2002367</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azwanida</surname><given-names>NN</given-names></name></person-group><article-title xml:lang=\"en\">A review on the extraction methods use in medicinal plants, principle</article-title><source>Strength Limit Med Aromat Plants</source><year>2015</year><volume>04</volume><fpage>3</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.4172/2167-0412.1000196</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bouras</surname><given-names>M</given-names></name><name><surname>Chadni</surname><given-names>M</given-names></name><name><surname>Barba</surname><given-names>FJ</given-names></name><name><surname>Grimi</surname><given-names>N</given-names></name><name><surname>Bals</surname><given-names>O</given-names></name><name><surname>Vorobiev</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Optimization of microwave-assisted extraction of polyphenols from Quercus bark</article-title><source>Ind Crops Prod</source><year>2015</year><volume>77</volume><fpage>590</fpage><lpage>601</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsF2gt73F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2015.09.018</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Che Sulaiman</surname><given-names>IS</given-names></name><name><surname>Basri</surname><given-names>M</given-names></name><name><surname>Fard Masoumi</surname><given-names>HR</given-names></name><name><surname>Chee</surname><given-names>WJ</given-names></name><name><surname>Ashari</surname><given-names>SE</given-names></name><name><surname>Ismail</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Effects of temperature, time, and solvent ratio on the extraction of phenolic compounds and the anti-radical activity of <italic>Clinacanthus nutans</italic> Lindau leaves by response surface methodology</article-title><source>Chem Cent J</source><year>2017</year><volume>11</volume><fpage>1</fpage><lpage>11</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVShtLzE</pub-id><pub-id pub-id-type=\"doi\">10.1186/s13065-017-0285-1</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheynier</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Phenolic compounds: from plants to foods</article-title><source>Phytochem Rev</source><year>2012</year><volume>11</volume><fpage>153</fpage><lpage>177</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXktFeqsro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11101-012-9242-8</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guzzetti</surname><given-names>L</given-names></name><name><surname>Panzeri</surname><given-names>D</given-names></name><name><surname>Ulaszewska</surname><given-names>M</given-names></name><name><surname>Sacco</surname><given-names>G</given-names></name><name><surname>Forcella</surname><given-names>M</given-names></name><name><surname>Fusi</surname><given-names>P</given-names></name><name><surname>Tommasi</surname><given-names>N</given-names></name><name><surname>Fiorini</surname><given-names>A</given-names></name><name><surname>Campone</surname><given-names>L</given-names></name><name><surname>Labra</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of dietary bioactive phenolic compounds and agricultural sustainability of an African leafy vegetable <italic>Corchorus olitorius</italic> L</article-title><source>Front Nutr</source><year>2021</year><volume>8</volume><fpage>1</fpage><lpage>10</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWqsrzI</pub-id><pub-id pub-id-type=\"doi\">10.3389/fnut.2021.667812</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halee</surname><given-names>A</given-names></name><name><surname>Supavititpatana</surname><given-names>P</given-names></name><name><surname>Ruttarattanamongkol</surname><given-names>K</given-names></name><name><surname>Jittrepotch</surname><given-names>N</given-names></name><name><surname>Rojsuntornkitti</surname><given-names>K</given-names></name><name><surname>Kongbangkerd</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Optimisation of the microwave-assisted extraction of natural antioxidants from defatted black rice bran of <italic>oryza sativa</italic> L. CV. homnin</article-title><source>J Microbiol Biotechnol Food Sci</source><year>2020</year><volume>9</volume><fpage>1134</fpage><lpage>1140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOksrrP</pub-id><pub-id pub-id-type=\"doi\">10.15414/JMBFS.2020.9.6.1134-1140</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Herrero</surname><given-names>M</given-names></name><name><surname>Plaza</surname><given-names>M</given-names></name><name><surname>Cifuentes</surname><given-names>A</given-names></name><name><surname>Ibáñez</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Extraction techniques for the determination of phenolic compounds in food</article-title><source>Compr Sampl Sample Prep</source><year>2012</year><volume>4</volume><fpage>159</fpage><lpage>180</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-12-381373-2.00132-0</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kamga</surname><given-names>RT</given-names></name><name><surname>Kouamé</surname><given-names>C</given-names></name><name><surname>Atangana</surname><given-names>AR</given-names></name><name><surname>Chagomoka</surname><given-names>T</given-names></name><name><surname>Ndango</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nutritional evaluation of five African indigenous vegetables</article-title><source>J Hortic Res</source><year>2013</year><volume>21</volume><fpage>99</fpage><lpage>106</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1SqsL%2FL</pub-id><pub-id pub-id-type=\"doi\">10.2478/johr-2013-0014</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khoddami</surname><given-names>A</given-names></name><name><surname>Wilkes</surname><given-names>MA</given-names></name><name><surname>Roberts</surname><given-names>TH</given-names></name></person-group><article-title xml:lang=\"en\">Techniques for analysis of plant phenolic compounds</article-title><source>Molecules</source><year>2013</year><volume>18</volume><fpage>2328</fpage><lpage>2375</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXjs12htrw%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules18022328</pub-id><pub-id pub-id-type=\"pmid\">23429347</pub-id><pub-id pub-id-type=\"pmcid\">6270361</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lachowicz</surname><given-names>S</given-names></name><name><surname>Oszmiański</surname><given-names>J</given-names></name><name><surname>Rapak</surname><given-names>A</given-names></name><name><surname>Ochmian</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Profile and content of phenolic compounds in leaves, flowers, roots, and stalks of <italic>Sanguisorba officinalis</italic> L. Determined with the LC-DAD-ESI- QTOF-MS/MS analysis and their in vitro antioxidant, antidiabetic, antiproliferative potency</article-title><source>Pharmaceuticals</source><year>2020</year><volume>13</volume><fpage>1</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVCgurrP</pub-id><pub-id pub-id-type=\"doi\">10.3390/ph13080191</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lobo</surname><given-names>V</given-names></name><name><surname>Patil</surname><given-names>A</given-names></name><name><surname>Phatak</surname><given-names>A</given-names></name><name><surname>Chandra</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Free radicals, antioxidants and functional foods: impact on human health</article-title><source>Pharmacogn Rev</source><year>2010</year><volume>4</volume><fpage>118</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BC387ivVOgtA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.4103/0973-7847.70902</pub-id><pub-id pub-id-type=\"pmid\">22228951</pub-id><pub-id pub-id-type=\"pmcid\">3249911</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Loumerem</surname><given-names>M</given-names></name><name><surname>Alercia</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Descriptors for jute (<italic>Corchorus olitorius</italic> L.)</article-title><source>Genet Resour Crop Evol</source><year>2016</year><volume>63</volume><fpage>1103</fpage><lpage>1111</lpage><pub-id pub-id-type=\"doi\">10.1007/s10722-016-0415-y</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahmud</surname><given-names>IA</given-names></name><name><surname>Mirghani</surname><given-names>SME</given-names></name><name><surname>Yusof</surname><given-names>F</given-names></name><name><surname>Al-khatib</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Effects of time, temperature, and solvent ratio on the extraction of non-extractable polyphenols with anticancer activity of barhi date palm kernels extracts using response surface methodology</article-title><source>J. Biotechnol. Eng.</source><year>2019</year><volume>1</volume><fpage>1</fpage><lpage>36</lpage><pub-id pub-id-type=\"doi\">10.20944/preprints201907.0055.v1</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mandal</surname><given-names>V</given-names></name><name><surname>Mohan</surname><given-names>Y</given-names></name><name><surname>Hemalath</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Microwave assisted extraction-an innovative and promising extraction tool for medicinal plant research</article-title><source>Pharmacogn Rev</source><year>2007</year><volume>1</volume><fpage>7</fpage><lpage>18</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXhtVSit7zI</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndlovu</surname><given-names>S</given-names></name><name><surname>Pullabhotla</surname><given-names>RVSR</given-names></name><name><surname>Ntuli</surname><given-names>NR</given-names></name></person-group><article-title xml:lang=\"en\">Response of <italic>Corchorus olitorius</italic> leafy vegetable to cadmium in the soil</article-title><source>Plants</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>13</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGnsLbE</pub-id><pub-id pub-id-type=\"doi\">10.3390/plants9091200</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Obeng</surname><given-names>E</given-names></name><name><surname>Kpodo</surname><given-names>FM</given-names></name><name><surname>Tettey</surname><given-names>CO</given-names></name><name><surname>Essuman</surname><given-names>EK</given-names></name><name><surname>Adzinyo</surname><given-names>OA</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant total phenols and proximate constituents of four tropical leafy vegetables</article-title><source>Sci African</source><year>2020</year><volume>7</volume><fpage>e00227</fpage><pub-id pub-id-type=\"doi\">10.1016/j.sciaf.2019.e00227</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quoc</surname><given-names>LPT</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from coffee (<italic>Coffea robusta</italic> L. Linden) bee pollen</article-title><source>Herb Pol</source><year>2021</year><volume>67</volume><fpage>37</fpage><lpage>44</lpage><pub-id pub-id-type=\"doi\">10.2478/hepo-2021-0015</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Racha</surname><given-names>A</given-names></name><name><surname>Yakoub</surname><given-names>O</given-names></name><name><surname>Abdehedi</surname><given-names>O</given-names></name></person-group><article-title xml:lang=\"en\">Flavonoids, phenols, antioxidant, and antimicrobial activities in various extracts from Tossa jute leave (<italic>Corchorus olitorus</italic> L.)</article-title><source>Ind Crops Prod</source><year>2018</year><volume>118</volume><fpage>206</fpage><lpage>213</lpage><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.03.047</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saliu</surname><given-names>JA</given-names></name><name><surname>Ademiluyi</surname><given-names>AO</given-names></name><name><surname>Boligon</surname><given-names>AA</given-names></name><name><surname>Oboh</surname><given-names>G</given-names></name><name><surname>Schetinger</surname><given-names>MRC</given-names></name><name><surname>Rocha</surname><given-names>JBT</given-names></name></person-group><article-title xml:lang=\"en\">Dietary supplementation of jute leaf (<italic>Corchorus olitorius</italic>) modulates hepatic delta-aminolevulinic acid dehydratase (δ-ALAD) activity and oxidative status in high-fat fed/low streptozotocin-induced diabetic rats</article-title><source>J Food Biochem</source><year>2019</year><volume>43</volume><fpage>e12949</fpage><pub-id pub-id-type=\"doi\">10.1111/jfbc.12949</pub-id><pub-id pub-id-type=\"pmid\">31368580</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tan</surname><given-names>HM</given-names></name><name><surname>Leong</surname><given-names>KH</given-names></name><name><surname>Song</surname><given-names>J</given-names></name><name><surname>Mohd Sufian</surname><given-names>NSF</given-names></name><name><surname>Mohd Hazli</surname><given-names>UHA</given-names></name><name><surname>Chew</surname><given-names>LY</given-names></name><name><surname>Kong</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant and LC-QToF-MS/MS analysis of polyphenols in polar and non-polar extracts from Strobilanthes crispus and Clinacanthus nutans</article-title><source>Int Food Res J</source><year>2020</year><volume>27</volume><fpage>903</fpage><lpage>914</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlynu7s%3D</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>J</given-names></name><name><surname>Dunshea</surname><given-names>FR</given-names></name><name><surname>Suleria</surname><given-names>HAR</given-names></name></person-group><article-title xml:lang=\"en\">LC-ESI-QTOF/MS characterization of phenolic compounds from medicinal plants (Hops and Juniper Berries) and their antioxidant activity</article-title><source>Foods</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>25</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVektLnE</pub-id><pub-id pub-id-type=\"doi\">10.3390/foods9010007</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tchabo</surname><given-names>W</given-names></name><name><surname>Ma</surname><given-names>Y</given-names></name><name><surname>Kwaw</surname><given-names>E</given-names></name><name><surname>Xiao</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Apaliya</surname><given-names>MT</given-names></name></person-group><article-title xml:lang=\"en\">Impact of extraction parameters and their optimization on the nutraceuticals and antioxidant properties of aqueous extract mulberry leaf</article-title><source>Int J Food Prop</source><year>2018</year><volume>21</volume><fpage>717</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXovFyrsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10942912.2018.1446025</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thoo</surname><given-names>YY</given-names></name><name><surname>Ho</surname><given-names>SK</given-names></name><name><surname>Liang</surname><given-names>JY</given-names></name><name><surname>Ho</surname><given-names>CW</given-names></name><name><surname>Tan</surname><given-names>CP</given-names></name></person-group><article-title xml:lang=\"en\">Effects of binary solvent extraction system, extraction time and extraction temperature on phenolic antioxidants and antioxidant capacity from mengkudu (<italic>Morinda citrifolia</italic>)</article-title><source>Food Chem</source><year>2010</year><volume>120</volume><fpage>290</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhsFClu7zO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2009.09.064</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Hwang</surname><given-names>SH</given-names></name><name><surname>Quispe</surname><given-names>YNG</given-names></name><name><surname>Arce</surname><given-names>PHG</given-names></name><name><surname>Lim</surname><given-names>SS</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of the antioxidant and aldose reductase inhibitory activities of extracts from Peruvian tea plant infusions</article-title><source>Food Chem</source><year>2017</year><volume>231</volume><fpage>222</fpage><lpage>230</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXltlOnurw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2017.03.107</pub-id><pub-id pub-id-type=\"pmid\">28450000</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>Q</given-names></name><name><surname>Fu</surname><given-names>Q</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>R</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Ju</surname><given-names>Z</given-names></name><name><surname>Campisi</surname><given-names>J</given-names></name><name><surname>Kirkland</surname><given-names>JL</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The flavonoid procyanidin C1 has senotherapeutic activity and increases lifespan in mice</article-title><source>Nat Metab</source><year>2021</year><volume>3</volume><fpage>1706</fpage><lpage>1726</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XotFKmsLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s42255-021-00491-8</pub-id><pub-id pub-id-type=\"pmid\">34873338</pub-id><pub-id pub-id-type=\"pmcid\">8688144</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Youssef</surname><given-names>AF</given-names></name><name><surname>Younes</surname><given-names>NA</given-names></name><name><surname>Youssef</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Genetic diversity in <italic>Corchorus olitorius</italic> L. revealed by morphophysiological and molecular analyses</article-title><source>Mol Biol Rep</source><year>2019</year><volume>46</volume><fpage>2933</fpage><lpage>2940</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmsFCnu7g%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11033-019-04754-2</pub-id><pub-id pub-id-type=\"pmid\">30887258</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Biological effects of tetrahydroxystilbene glucoside: an active component of a rhizome extracted from <italic>Polygonum multiflorum</italic></article-title><source>Oxid Med Cell Longev</source><year>2018</year><volume>2018</volume><fpage>3641960</fpage><pub-id pub-id-type=\"doi\">10.1155/2018/3641960</pub-id><pub-id pub-id-type=\"pmid\">30524653</pub-id><pub-id pub-id-type=\"pmcid\">6247474</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Kim</surname><given-names>SH</given-names></name><name><surname>Hagerman</surname><given-names>AE</given-names></name><name><surname>Lü</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer, anti-diabetic and other pharmacologic and biological activities of penta-galloyl-glucose</article-title><source>Pharm Res</source><year>2009</year><volume>26</volume><fpage>2066</fpage><lpage>2080</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptlaku7w%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11095-009-9932-0</pub-id><pub-id pub-id-type=\"pmid\">19575286</pub-id><pub-id pub-id-type=\"pmcid\">2822717</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec25\"><title>Supplementary Information</title><p id=\"Par49\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2771_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 934 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>ABTS</term><def><p id=\"Par2\">2,2′-Azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid)</p></def></def-item><def-item><term>ANOVA</term><def><p id=\"Par3\">Analysis of variance analysis</p></def></def-item><def-item><term>DNA</term><def><p id=\"Par4\">Deoxyribonucleic acid</p></def></def-item><def-item><term>DPPH</term><def><p id=\"Par5\">2,2-Diphenyl-1-picrylhydrazyl</p></def></def-item><def-item><term>FCCCD</term><def><p id=\"Par6\">Face-centred central composite design</p></def></def-item><def-item><term>LC-QToF-MS</term><def><p id=\"Par7\">Liquid chromatography-mass spectrometry quadrupole of flight</p></def></def-item><def-item><term>OFAT</term><def><p id=\"Par8\">One-factor-at-a-time</p></def></def-item><def-item><term>TPC</term><def><p id=\"Par9\">Total phenolic content</p></def></def-item><def-item><term>RSM</term><def><p id=\"Par10\">Response surface methodology</p></def></def-item><def-item><term>SDG 2</term><def><p id=\"Par11\">Sustainable development goals</p></def></def-item></def-list></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10934</journal-id><journal-id journal-id-type=\"doi\">10.1007/10934.1573-4854</journal-id><journal-title-group><journal-title>Journal of Porous Materials</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Porous Mater</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1380-2224</issn><issn pub-type=\"epub\">1573-4854</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10934-023-01452-0</article-id><article-id pub-id-type=\"manuscript\">1452</article-id><article-id pub-id-type=\"doi\">10.1007/s10934-023-01452-0</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Synthesis and characterization of polyurethane rigid foam by using feedstocks received from renewable and recyclable resources</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Sarim</surname><given-names>Muntajab</given-names></name><address><email>muntajabsarim@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><name><surname>Nikje</surname><given-names>Mir Mohammad Alavi</given-names></name><address><email>alavi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10934023014520_cor2\">b</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Dargahi</surname><given-names>Maryam</given-names></name><address><email>dargahi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411537.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 8608 1112</institution-id><institution content-type=\"org-division\">Department of Chemistry, Faculty of Science</institution><institution content-type=\"org-name\">Imam Khomeini International University (IKIU)</institution></institution-wrap><addr-line content-type=\"city\">Qazvin</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs10934023014520_cor2\"><label>b</label><email>alavi@sci.ikiu.ac.ir</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>30</volume><issue seq=\"25\">4</issue><fpage>1357</fpage><lpage>1357</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10934-023-01425-3\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10934_2023_Article_1452.pdf</meta-value></custom-meta><custom-meta><meta-name>output-medium</meta-name><meta-value>All</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Catalysis</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Characterization and Evaluation of Materials</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>The online version of the original article can be found at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p></notes></front><body><p id=\"Par1\">Correction to: <bold>Journal of Porous Materials</bold></p><p id=\"Par2\"><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p><p id=\"Par6\">In this article, the author name Maryam Dargah has been corrected to Maryam Dargahi.</p><p id=\"Par7\">The original article has been corrected.</p></body><back><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article></records>"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"records\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<records><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02810-7</article-id><article-id pub-id-type=\"manuscript\">2810</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02810-7</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Evaluation of acyclovir adsorption on granular activated carbon from aqueous solutions: batch and fixed-bed parametric studies</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Gupta</surname><given-names>Anju</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-9330-1539</contrib-id><name><surname>Vyas</surname><given-names>Raj K.</given-names></name><address><email>rkvyas.chem@mnit.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023028107_cor2\">b</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444471.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1764 2536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering</institution><institution content-type=\"org-name\">Malaviya National Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">302017</addr-line><addr-line content-type=\"city\">Jaipur</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023028107_cor2\"><label>b</label><email>rkvyas.chem@mnit.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>13</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"40\">8</issue><fpage>4599</fpage><lpage>4612</lpage><history><date date-type=\"registration\"><day>3</day><month>4</month><year>2023</year></date><date date-type=\"received\"><day>13</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>1</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>13</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><sec><p id=\"Par1\">The present study is aimed to assess the adsorptive potential of carbonaceous material for the acyclovir (ACVR) removal from the aquatic environment using batch and fixed-bed processes. In batch mode, the impact of various process conditions (contact time, pH, adsorbent dose, initial ACVR concentration, and temperature) on ACVR adsorption was investigated. Experimental results revealed that Langmuir isotherm and the pseudo-second-order kinetic model adequately represent the ACVR adsorption mechanism, indicating homogeneous adsorption. The process was found exothermic and spontaneous. Thermodynamic studies concluded that adsorption is a result of both physisorption and chemisorption. To understand the dynamic regime for the design of large-scale column studies, experimental data obtained from breakthrough curve were fitted to various analytical kinetic models. Yan model followed by Thomas model demonstrated a greater correlation of breakthrough data, confirming that the results are significant and are in line with Langmuir isotherm and pseudo-second-order kinetic. G-AC exhibits sufficient adsorption capacity for ACVR. Hence, it is concluded that it can be used in a fixed-bed column in continuous mode for the treatment of ACVR-contaminated wastewater.</p></sec><sec id=\"ASec2\"><title>Graphical abstract</title><p id=\"Par2\"><fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"PNG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Figa_HTML.png\"/></fig></p></sec></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Granular material</kwd><kwd>Kinetics</kwd><kwd>Isotherms</kwd><kwd>Fixed-bed</kwd><kwd>Breakthrough curve</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2810.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02810-7\">https://doi.org/10.1007/s11696-023-02810-7</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par48\">Over the past few decades, the new strains of life-threatening pandemic viruses, including Nipah virus, severe acute respiratory syndrome (SARS) virus, Middle East respiratory syndrome (MERS), and the most recent one, coronavirus disease (COVID-19), have hit the world. Therefore, the viral infections are of the global concern. Researchers are exploring new promising antibodies, antiviral treatments, and ways to improve the efficacy of existing medications on a laboratory scale (Al-Otaibi et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>; Heidary et al. <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>; Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). Antiviral drugs have received a lot of attention recently (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR30\">2013</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). Acyclovir (ACVR) (Fig. S1), a synthetically prepared purine-based nucleoside analog, shows antiviral activities for the treatment of herpes simplex virus (HSV), hepatitis B, varicella-zoster virus, Epstein-Barr virus, chickenpox, etc. (Nannou et al. <xref ref-type=\"bibr\" rid=\"CR43\">2020</xref>; De Clercq <xref ref-type=\"bibr\" rid=\"CR14\">2004</xref>). ACVR is usually favored over other drugs because of its low cytotoxicity and high selectivity towards infected cells (Shetti et al. <xref ref-type=\"bibr\" rid=\"CR54\">2020</xref>). More than two-thirds of the population is infected by α-HSV (Tognarelli et al. <xref ref-type=\"bibr\" rid=\"CR57\">2019</xref>). ACVR reaches drinking water via several routes from domestic/clinic waste, bulk and formulation drug industries, etc. (Gupta et al. <xref ref-type=\"bibr\" rid=\"CR25\">2021</xref>). The simultaneous presence of a pertinent virus in living beings and the modest amount of ACVR can render the virus drug-resistant (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Its presence has been found in industrial effluents, surface water, drinking water, and wastewater influent/effluent stream(s) (Funke et al. <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>; Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR47\">2010</xref>; Mascolo et al. <xref ref-type=\"bibr\" rid=\"CR41\">2010</xref>). Even though ACVR is present at a low concentration ranging from ng/L to μg/L (Azuma et al. <xref ref-type=\"bibr\" rid=\"CR2\">2016</xref>; Peng et al. <xref ref-type=\"bibr\" rid=\"CR45\">2014</xref>; Bradley et al. <xref ref-type=\"bibr\" rid=\"CR8\">2014</xref>), its persistent nature and harmful effects on the aquatic environment necessitate the treatment of contaminated water (Russo et al. <xref ref-type=\"bibr\" rid=\"CR51\">2017</xref>). A high therapeutic dose is mainly recommended as it only lasts a brief time in the human body (Saifi et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>). As a result, water contamination and possible environmental issues with the widespread usage of ACVR should be seriously considered. Remediation of industrial wastewater before its discharge is more economical than treating low concentrated large volume of water among preventive routes. Consequently, to prevent the adverse effects of ACVR on the ecosystem, ACVR polluted wastewater must be treated before being discharged.</p><p id=\"Par49\">Adsorption (Hosseini Nasr et al. <xref ref-type=\"bibr\" rid=\"CR29\">2018</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>, <xref ref-type=\"bibr\" rid=\"CR32\">2014b</xref>), UV/H<sub>2</sub>O<sub>2</sub> (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>), catalytic ozonation (Bing et al. <xref ref-type=\"bibr\" rid=\"CR5\">2017</xref>), photocatalysis (Wu et al. <xref ref-type=\"bibr\" rid=\"CR58\">2018</xref>; Li et al. <xref ref-type=\"bibr\" rid=\"CR35\">2016</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR10\">2014</xref>), photolysis (Zhou et al. <xref ref-type=\"bibr\" rid=\"CR62\">2015</xref>), ozonation (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>), and biological treatment techniques (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR61\">2017</xref>; Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>) have been attempted so far till date for the removal of ACVR from wastewater. AOP using oxone has been proven efficient (Liu et al. <xref ref-type=\"bibr\" rid=\"CR37\">2018</xref>, <xref ref-type=\"bibr\" rid=\"CR38\">2020</xref>); however, the high cost of oxone used in the process makes it un-economical. Oxidation and biodegradation processes lead to more toxic intermediates like COFA and carboxy-ACV (Prasse et al. <xref ref-type=\"bibr\" rid=\"CR48\">2011</xref>, <xref ref-type=\"bibr\" rid=\"CR49\">2012</xref>). Bio-refractory compounds like ACVR on degradation get converted into more toxic intermediate product(s). For such compounds, adsorption appears to be the most efficient approach, as it does not degrade the compound. Most adsorption studies are carried out in batches, which do not provide precise scale-up information required for practical large-scale treatment. So far column studies on ACVR removal from wastewater have been scarcely reported.</p><p id=\"Par50\">Considering its high surface area and porous structure, granular activated carbon (G-AC) is an efficient adsorbent for ACVR. Hence, it is a viable treatment solution for eliminating ACVR from water. Additionally, G-AC is cost-effective, environmentally benign and easily available and can be used in a variety of treatment systems, including fixed-bed columns. Therefore, the present research was aimed to evaluate the performance of G-AC column for ACVR removal from the synthetically prepared aqueous solution of ACVR. The batch studies were also performed to know the effects of operating parameters on the removal efficiency of ACVR. The breakthrough curves were used to characterize the dynamic response of the fixed-bed column. Effects of initial ACVR concentration, G-AC-bed height, and flow rate on ACVR adsorption were evaluated as part of the column studies along with the analytical model studies. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p></sec><sec id=\"Sec2\"><title>Experimental</title><p id=\"Par51\">Chemicals/reagents, solution preparation, and analytical techniques used are described in supplementary file.</p><sec id=\"Sec3\"><title>Adsorbent characterization</title><p id=\"Par52\">The Brunauer–Emmett–Teller (BET) method was used to determine the specific surface area, pore volume, and pore size distribution of G-AC (Nova Touch LX2 gas sorption analyzer, Quantachrome Instruments). The scanning electron microscope (SEM) and energy-dispersive X-ray spectroscopy (EDX) were used to perform morphological characterization of the raw and spent G-AC sample (Nova Nanosem 450, FEITM).</p></sec><sec id=\"Sec4\"><title>Batch experiments</title><p id=\"Par53\">For determining the maximum removal of ACVR (dosage form, make: Ocuvir, FDC Limited, India) batch adsorption studies were conducted at varying initial concentrations of ACVR, adsorbent dose, pH of the solution, and temperature. A pre-determined amount of G-AC (make: Loba Chemie, India, LR grade) was mixed with 100 mL of synthetically prepared aqueous ACVR solution of known concentration for each group of experiments. Each batch was agitated in an orbital shaker at a fixed temperature and RPM. The solution was filtered after completion of each set, and the residual ACVR concentration was measured in a UV–Vis spectrophotometer.</p><p id=\"Par54\">The adsorption capacity of the G-AC at the time (t min) and the percent removal of ACVR were estimated using the following equations, respectively:<disp-formula id=\"Equa\"><alternatives><mml:math display=\"block\" id=\"Equa_Math\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mi>t</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mi>V</mml:mi></mml:mrow><mml:mi>W</mml:mi></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equa_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$q_{t} = \\frac{{\\left( {C_{0} - C_{f} } \\right) \\times V}}{W}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equa.gif\"/></alternatives></disp-formula><disp-formula id=\"Equb\"><alternatives><mml:math display=\"block\" id=\"Equb_Math\"><mml:mrow><mml:mo>%</mml:mo><mml:mrow><mml:mtext>Removal</mml:mtext><mml:mspace width=\"0.333333em\"/></mml:mrow><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>C</mml:mi><mml:mi>f</mml:mi></mml:msub></mml:mrow></mml:mfenced><mml:mo>×</mml:mo><mml:mn>100</mml:mn></mml:mrow><mml:msub><mml:mi>C</mml:mi><mml:mi>O</mml:mi></mml:msub></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equb_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\% {\\text{Removal }} = \\frac{{\\left( {C_{O} - C_{f} } \\right) \\times 100}}{{C_{O} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equb.gif\"/></alternatives></disp-formula></p><p id=\"Par55\">Following equations were used to calculate standard deviation percent (SD%) and root-mean-square error (RMSE):<disp-formula id=\"Equc\"><alternatives><mml:math display=\"block\" id=\"Equc_Math\"><mml:mrow><mml:mtext>SD</mml:mtext><mml:mo>%</mml:mo><mml:mo>=</mml:mo><mml:mn>100</mml:mn><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub></mml:mrow></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equc_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{SD}}\\% = 100\\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)/q_{\\exp } } \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - 1}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equc.gif\"/></alternatives></disp-formula><disp-formula id=\"Equd\"><alternatives><mml:math display=\"block\" id=\"Equd_Math\"><mml:mrow><mml:mtext>RMSE</mml:mtext><mml:mo>=</mml:mo><mml:msqrt><mml:mfrac><mml:mrow><mml:mo>∑</mml:mo><mml:msup><mml:mfenced close=\")\" open=\"(\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>q</mml:mi><mml:mo>exp</mml:mo></mml:msub><mml:mo>-</mml:mo><mml:msub><mml:mi>q</mml:mi><mml:mtext>cal</mml:mtext></mml:msub></mml:mrow></mml:mfenced></mml:mfenced><mml:mn>2</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mtext>data</mml:mtext><mml:mspace width=\"0.166667em\"/><mml:mspace width=\"0.166667em\"/><mml:mtext>point</mml:mtext><mml:mo>-</mml:mo><mml:mtext>parameters</mml:mtext></mml:mrow></mml:mfrac></mml:msqrt></mml:mrow></mml:math><tex-math id=\"Equd_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$${\\text{RMSE}} = \\sqrt {\\frac{{\\sum \\left( {\\left( {q_{\\exp } - q_{{{\\text{cal}}}} } \\right)} \\right)^{2} }}{{{\\text{data}}\\,\\,{\\text{point}} - {\\text{parameters}}}}}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equd.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec5\"><title>Kinetic studies</title><p id=\"Par56\">Several kinetic models can test the experimental results and analyze the kinetic mechanism that governs the adsorption rate based on ACVR diffusion in adsorbent particles. The present work used pseudo-first and second-order, Elovich, and intra-particle diffusion kinetic models to evaluate experimental results. A batch size of 100 mL of 100 mg/L concentrated solution was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C.</p><p id=\"Par57\">Pseudo-first-order kinetic model deals with the adsorption potential of the adsorbent against an adsorbate. According to the pseudo-second-order kinetic model, physisorption and chemisorption contribute to adsorption (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>). Elovich model explains chemisorption (Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). This model shows the declining adsorption rate with time as the vacant adsorbent sites get occupied by adsorbate molecules and as a result surface coverage increases (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The intraparticle diffusion model explains the adsorption mechanism. It is generally the last step to know about adsorption and type of diffusion within the adsorbent. Model equations used in kinetic studies are summarized in Table S1.</p><p id=\"Par58\">The low C value represents the boundary layer’s minimal influence (diffusional resistance) on adsorption (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>). The initial rate of adsorption h (mg/g/min) is determined using the following relation:<disp-formula id=\"Eque\"><alternatives><mml:math display=\"block\" id=\"Eque_Math\"><mml:mrow><mml:mi>h</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>k</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msubsup><mml:mi>q</mml:mi><mml:mrow><mml:mi>e</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:math><tex-math id=\"Eque_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$h = k_{2} q_{e}^{2}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Eque.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec6\"><title>Equilibrium studies</title><p id=\"Par59\">Adsorption equilibrium studies were carried out to study the adsorption phenomenon and to assess the ACVR interaction behavior with the adsorbent surface. For which solution with varying concentration was treated with 4 g/L of G-AC at its natural pH (~ 6.8) at 25 °C. Temkin, Freundlich, and Langmuir isotherm models were used to examine the equilibrium parameters. Freundlich isotherm reveals the heterogeneous nature of the sorption process (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). On the other hand, Langmuir isotherm typically shows the homogeneous nature of the adsorption process. It is assumed that adsorbents are mainly composed of fixed vacant sites, each adsorbing single adsorbate particle leading to a monolayer adsorption process (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Table S2 provides a summary of the model equations used in isotherm studies.</p><p id=\"Par60\">The nature of adsorption can be determined based on the value of ‘<italic>n</italic>’ (Momčilović et al. <xref ref-type=\"bibr\" rid=\"CR42\">2011</xref>). Favorability of the process was checked using the separating factor (<italic>R</italic><sub><italic>L</italic></sub>):<disp-formula id=\"Equf\"><alternatives><mml:math display=\"block\" id=\"Equf_Math\"><mml:mrow><mml:msub><mml:mi>R</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfrac><mml:mn>1</mml:mn><mml:mrow><mml:mn>1</mml:mn><mml:mo>+</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mi>L</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equf_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$R_{L} = \\frac{1}{{1 + K_{L} C_{0} }}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equf.gif\"/></alternatives></disp-formula></p></sec><sec id=\"Sec7\"><title>Thermodynamic studies</title><p id=\"Par61\">To study the temperature dependence of adsorptive removal of ACVR and the spontaneity of the process, thermodynamic studies were conducted. Thermodynamic parameters, namely, Gibbs free energy and heat of adsorption, were also estimated using equations given in Table S3.</p></sec><sec id=\"Sec8\"><title>Column studies and data analysis</title><p id=\"Par62\">A glass column with an inner diameter of 1.25 cm and a height of 45 cm was used to conduct fixed-bed column studies (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). The required flow rate was maintained using a peristaltic pump. Nylon cloth was used at the bottom to stop any adsorbent loss, and glass beads were used on both sides of the column to provide mechanical support to the bed and for uniform distribution of the feed solution preventing channelling. All the experiments were performed at an ambient temperature of 30 ± 2 °C. Effects of various process parameters such as bed height, volumetric flow rate, and inlet ACVR concentration were studied. After predetermined intervals, samples were taken and examined to determine the ACVR concentration. The adsorption capacity and breakthrough time were estimated to assess column performance.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic diagram of a fixed-bed adsorption column used for ACVR removal</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig1_HTML.png\"/></fig></p><p id=\"Par63\">Fixed-bed analytical models, namely, Thomas, Adams–Bohart, Yoon–Nelson, Clark, and Yan models were used to characterize experimental breakthrough curves. Table S4 summarizes the model equations used in the study.</p></sec></sec><sec id=\"Sec9\"><title>Results and discussion</title><sec id=\"Sec10\"><title>Adsorbent characterization</title><p id=\"Par64\">BET surface area and pore volume of G-AC were found to be 470 m<sup>2</sup>/g and 0.23 cm<sup>3</sup>/g, respectively. The average pore diameter of G-AC (1.44 nm) suggests the prominent presence of supra-micropores in G-AC (0.7–2 nm) (Thommes <xref ref-type=\"bibr\" rid=\"CR56\">2010</xref>). Fig. S2 shows the nitrogen adsorption–desorption isotherm at 77 K and the G-AC pore size distribution curve.</p><p id=\"Par65\">The isotherm is a type IV isotherm, indicating the presence of mesoporous particles. Micro-mesoporous carbons usually feature H4 loops, which are closely attributed to the thin and slit-like pores (Guo et al. <xref ref-type=\"bibr\" rid=\"CR23\">2022</xref>). The prominence of micropores causes the volume adsorbed to increase rapidly at lower relative pressure. The pore size distribution estimated by the BJH plot validated the presence of mesopores along with the micropores. Fig. S3 further summarizes the characteristics of G-AC, confirming that it is an efficacious adsorbent.</p><p id=\"Par66\">Higher magnification SEM micrograph (Fig. S3) of the G-AC sample confirmed its porous structure. Additionally, the EDX findings revealed the elemental analysis of G-AC. Fig. S3 clearly demonstrates the adsorption of ACVR on pores of G-AC.</p></sec><sec id=\"Sec11\"><title>Point of zero charge and pH effect</title><p id=\"Par67\">The liquid-phase pH significantly influences the adsorbent and their ability to adsorb, affecting the binding sites. The effect of pH on ACVR removal using G-AC was examined for 100 mg/L concentrated ACVR solution (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a). For a contact time of 75 min, q<sub>t</sub> increased from 22.75 to 24.62 mg/g on increasing pH from 3 to 7; further increasing pH to 11, q<sub>t</sub> decreased to 20.25 mg/g. From Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a, it can be seen that maximum removal was obtained at pH 7. The increased electrostatic affinity between the surface charge of G-AC and ACVR is most likely responsible for the improved adsorption capacity of the system at pH 7. As a result, possibly the G-AC and ACVR surface charges in the aqueous medium are in opposite directions. The point of zero charge has been further used to explain pH influence and to comprehend the interaction between G-AC and ACVR.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p><bold>a</bold> pH effect on ACVR removal at a contact time of 75 min, <bold>b</bold> Point of zero charge of G-AC and ACVR</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig2_HTML.png\"/></fig></p><p id=\"Par68\">The pH at which the surface of an adsorbent is believed to behave neutrally is referred to as the point of zero charge (pH<sub>ZPC</sub>). Hence, it is significant to determine the pH<sub>ZPC</sub> before conducting the adsorptive removal experiments involving ionic species adsorption from aqueous environments onto solid surfaces. The pH<sub>ZPC</sub> for G-AC and ACVR is illustrated in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b. The results show that the ACVR surface is more basic (pH<sub>ZPC</sub>: 8.2) than G-AC's surface (pH<sub>ZPC</sub>: 6.9). The ACVR and G-AC surfaces are charged positively at pH &lt; pH<sub>ZPC</sub> and negatively at pH &gt; pH<sub>ZPC</sub>. The surface of G-AC shows a negative charge at pH &gt; pH<sub>ZPC</sub>. As the initial pH of the solution is increased, the surplus hydroxide ions in the bulk liquid are generally counterbalanced by the G-AC released hydrogen ions. Furthermore, deprotonation of G-AC lowers the solution’s final pH and results in a net negative charge on the G-AC surface.</p><p id=\"Par69\">As discussed earlier, the G-AC surface is positively charged below pH<sub>ZPC</sub>, as consequence hydrogen ions compete with ACVR cations to reach the surface. The hydrogen ions will gradually surround major active sites on the G-AC surface. At pH &lt; 7, both G-AC and ACVR are positively charged which repelled from each other. As such, ACVR molecules were not effectively adsorbed on G-AC. Similar patterns for pH<sub>ZPC</sub> values for G-AC and ACVR are reported in the literature (Borah et al. <xref ref-type=\"bibr\" rid=\"CR7\">2008</xref>).</p></sec><sec id=\"Sec12\"><title>Effect of adsorbent dose</title><p id=\"Par70\">Experiments were carried out at varying doses of adsorbents ranging from 1 to 4 g/L at 25 °C for 2 h to investigate the effects of the G-AC dose on the ACVR removal efficiency (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>).<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Impact of varying adsorbent dose on ACVR removal</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig3_HTML.png\"/></fig></p><p id=\"Par71\">When adsorbent dose was increased from 1 to 4 g/L, there was a polynomial increase in the removal of ACVR from 39.7 to 95.2%. It can be seen from Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref> that there was a continuous rise in the percent removal of ACVR, but 4 g/L was taken as optimum as already 95% removal was being achieved with this dose. A reduction of 95.2% in ACVR concentration indicates that almost all of the vacant-active sites on the adsorbent surface are occupied at this point, and increasing the adsorbent dosage beyond this point will not be cost-effective.</p></sec><sec id=\"Sec13\"><title>Impact of contact time</title><p id=\"Par72\">At natural pH (~ 6.8), the impact of contact time on ACVR adsorption by G-AC was examined for 105 min. Figure <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a depicts the performance of adsorbents for the removal of ACVR with contact time.<fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p><bold>a</bold> Impact of Contact time (initial ACVR concentration: 100 mg/L) and <bold>b</bold> Impact of ACVR initial concentration on percent removal at equilibrium</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig4_HTML.png\"/></fig></p><p id=\"Par73\">The polynomial increase in percent removal was observed on increasing the contact time of adsorbent and ACVR. More than 50% of the removal was achieved within the initial 30 min using G-AC. Initially, the removal rate was rapid; however slow reduction was observed later. This could be due to more unoccupied sites on adsorbent are available at the initial stage with a high driving force which decreases with time and active sites also diminishes. Finally, for an initial ACVR concentration of 100 mg/L system reached the state of saturation at 75 min.</p></sec><sec id=\"Sec14\"><title>Adsorption kinetics</title><p id=\"Par74\">The adsorption kinetics of ACVR onto G-AC using pseudo-first- and -second-order, Elovich, and intra-particle diffusion kinetic models is depicted in Fig. S4. Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> summarizes the values of regression coefficients obtained and other kinetic parameters of ACVR adsorption.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Parameters for various kinetic models for ACVR removal</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Kinetic model</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Values</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Pseudo-first-order model</p></td><td align=\"left\"><p><italic>q</italic><sub>e,exp</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>24.62</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>k</italic><sub>1</sub> (min<sup>−1</sup>)</p></td><td align=\"char\" char=\".\"><p>0.04</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>8.98</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9899</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>8.1%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>7.82</p></td></tr><tr><td align=\"left\"><p>Pseudo-second-order model</p></td><td align=\"left\"><p><italic>k</italic><sub>2</sub> (g/mg/min)</p></td><td align=\"char\" char=\".\"><p>0.01</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>q</italic><sub>e,cal</sub> (mg/g)</p></td><td align=\"char\" char=\".\"><p>25.58</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9991</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>h</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>7.20</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"char\" char=\".\"><p>0.02%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"char\" char=\".\"><p>0.43</p></td></tr><tr><td align=\"left\"><p>Elovich kinetic model</p></td><td align=\"left\"><p><italic>α</italic> (mg/g/min)</p></td><td align=\"char\" char=\".\"><p>296.61</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>β</italic> (g/mg)</p></td><td align=\"char\" char=\".\"><p>0.36</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9771</p></td></tr><tr><td align=\"left\"><p>Intraparticle diffusion model</p></td><td align=\"left\"><p><italic>K</italic><sub>d</sub> (mg/g/min<sup>0.5</sup>)</p></td><td align=\"char\" char=\".\"><p>1.12</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>C</italic> (mg/g)</p></td><td align=\"char\" char=\".\"><p>15.48</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>0.9579</p></td></tr></tbody></table></table-wrap></p><p id=\"Par75\">The significance of studying the adsorption kinetics of ACVR onto G-AC using different kinetic models lies in understanding the rate and mechanism of the adsorption process. The comparison of the results obtained from different models can help determine the most suitable kinetic model for describing the adsorption process and provide insights into the mechanism of adsorption. Kinetic data obtained in the present study are significant and indicate that the adsorption process satisfies pseudo-second-order kinetic model.</p><p id=\"Par76\">The smaller difference in the equilibrium amount adsorbed between experimental and calculated values confirms the pseudo-second-order behavior of the process (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Moreover, the slightly lower correlation coefficient (<italic>R</italic><sup>2</sup>) also shows the inappropriateness of the pseudo-first-order kinetic model. The adsorption kinetics of numerous bio-refractory pollutant-adsorbent systems have exhibited comparable kinetic results (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>; Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p><p id=\"Par77\">If the value of C is not zero for any adsorption process, the intraparticle diffusion is not the sole controlling mechanism (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). Thus, it can be concluded that during ACVR sorption on G-AC, film and intraparticle diffusion occur simultaneously (Shah et al. <xref ref-type=\"bibr\" rid=\"CR53\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>).</p></sec><sec id=\"Sec15\"><title>Influence of ACVR initial concentration</title><p id=\"Par78\">It is vital to investigate the effect of the initial concentration of the adsorbate on the sorption rate for effective adsorption. The variation of the adsorptive removal efficiency of G-AC at different initial concentrations of ACVR (100–400 mg/L) is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b.</p><p id=\"Par79\">It is evident from Fig. <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b that as the drug concentration rise, so does the quantity of adsorbed ACVR. This could be because the difference between available adsorbent sites and the initial number of ACVR molecules is high at higher concentrations. Therefore, an initial concentration of 100 mg/L ACVR was used in the present study; moreover, further change in percent removal was significantly less. Additionally, it is common that the lower the initial adsorbate concentration, the higher removal efficiency while the other experimental conditions were kept constant.</p></sec><sec id=\"Sec16\"><title>Adsorption isotherms</title><p id=\"Par80\">Fig. S5 depicts the adsorption isotherms (Freundlich, Langmuir, and Temkin) and Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> shows the isotherm parameters for the ACVR adsorption onto G-AC.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Isotherm parameters for ACVR sorption</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Particulars</p></th><th align=\"left\"><p>Parameter</p></th><th align=\"left\"><p>Value</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Freundlich isotherm</p></td><td align=\"left\"><p><italic>K</italic><sub>F</sub> (mg/g)/(L/mg)<sup>n</sup></p></td><td align=\"left\"><p>24.51</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>n</italic></p></td><td align=\"left\"><p>2.37</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9293</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>12.69</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>8.72</p></td></tr><tr><td align=\"left\"><p>Langmuir isotherm</p></td><td align=\"left\"><p><italic>Q</italic><sub>m</sub> (mg/g)</p></td><td align=\"left\"><p>105.26</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>K</italic><sub>L</sub> (L/mg)</p></td><td align=\"left\"><p>0.21</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9981</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>SD%</p></td><td align=\"left\"><p>3.46%</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>2.35</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sub>L</sub></p></td><td align=\"left\"><p>0 &lt; <italic>R</italic><sub>L</sub> &lt; 1</p></td></tr><tr><td align=\"left\"><p>Temkin isotherm</p></td><td align=\"left\"><p><italic>A</italic><sub>T</sub> (L/g)</p></td><td align=\"left\"><p>2.10</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>b</italic><sub>T</sub></p></td><td align=\"left\"><p>109.93</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>B</italic><sub>T</sub> (mg/g)</p></td><td align=\"left\"><p>22.54</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.9831</p></td></tr><tr><td align=\"left\"/><td align=\"left\"><p>RMSE</p></td><td align=\"left\"><p>16.865</p></td></tr></tbody></table></table-wrap></p><p id=\"Par81\">The highest <italic>R</italic><sup>2</sup> value confirms the Langmuir adsorption isotherm model’s good fit to the experimental data. In the present work, the value of <italic>R</italic><sub><italic>L</italic></sub> was found 0.045 indicating the ACVR adsorption to be a favorable process. Furthermore, the low <italic>R</italic><sub><italic>L</italic></sub> value suggests that ACVR interaction with adsorbents is strong (Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>). The value of the heterogeneity factor also confirms the favorability of the process, and the process follows the Langmuir isotherm (0 &lt; 1/<italic>n</italic> &lt; 1) (Fytianos et al. <xref ref-type=\"bibr\" rid=\"CR22\">2000</xref>). It also signifies that monolayer chemisorption occurs with uniform adsorptive sites and energy (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>). The increase in adsorption capacity indicates weak interaction between adsorbed ACVR molecules (Hasnain Isa et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). Similar results are reported for numerous pollutant-adsorbent systems (Babas et al. <xref ref-type=\"bibr\" rid=\"CR3\">2021</xref>; Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Fan et al. <xref ref-type=\"bibr\" rid=\"CR19\">2012</xref>).</p></sec><sec id=\"Sec17\"><title>Thermodynamic studies</title><p id=\"Par82\">Adsorption studies at varying temperatures were conducted to examine the influence of temperature on ACVR uptake (Fig. S6). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> highlights the thermodynamic parameters for ACVR adsorption onto G-AC. Both 1/<italic>T</italic> and ln (<italic>q</italic><sub><italic>e</italic></sub>/<italic>C</italic><sub><italic>e</italic></sub>) values decrease with rise in temperature, indicating that the adsorption process is exothermic. Following equations were used to calculate change in Gibbs free energy,<disp-formula id=\"Equg\"><alternatives><mml:math display=\"block\" id=\"Equg_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mo>-</mml:mo><mml:mi>R</mml:mi><mml:mi>T</mml:mi><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub></mml:mrow></mml:math><tex-math id=\"Equg_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = - RT\\ln K_{0}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equg.gif\"/></alternatives></disp-formula><disp-formula id=\"Equh\"><alternatives><mml:math display=\"block\" id=\"Equh_Math\"><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>G</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>=</mml:mo><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mo>∘</mml:mo></mml:msup><mml:mo>-</mml:mo><mml:mi>T</mml:mi><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mo>∘</mml:mo></mml:msup></mml:mrow></mml:math><tex-math id=\"Equh_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\Delta G^\\circ = \\Delta H^\\circ - T\\Delta S^\\circ$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equh.gif\"/></alternatives></disp-formula><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Thermodynamic parameters for ACVR adsorption onto G-AC</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Temperature (°C)</p></th><th align=\"left\"><p>∆<italic>H</italic><sup>0</sup> (kJ/mol)</p></th><th align=\"left\"><p>∆<italic>S</italic><sup>0</sup> (kJ/mol/K)</p></th><th align=\"left\"><p>∆<italic>G</italic><sup>0</sup> (kJ/mol)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>25</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 56.88</p></td><td align=\"char\" char=\".\" rowspan=\"7\"><p>− 0.168</p></td><td align=\"char\" char=\".\"><p>− 6.93</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"char\" char=\".\"><p>− 6.04</p></td></tr><tr><td align=\"left\"><p>35</p></td><td align=\"char\" char=\".\"><p>− 4.59</p></td></tr><tr><td align=\"left\"><p>40</p></td><td align=\"char\" char=\".\"><p>− 4.19</p></td></tr><tr><td align=\"left\"><p>45</p></td><td align=\"char\" char=\".\"><p>− 3.38</p></td></tr><tr><td align=\"left\"><p>50</p></td><td align=\"char\" char=\".\"><p>− 2.64</p></td></tr><tr><td align=\"left\"><p>55</p></td><td align=\"char\" char=\".\"><p>− 1.78</p></td></tr></tbody></table></table-wrap></p><p id=\"Par83\">Combining above equations,<disp-formula id=\"Equi\"><alternatives><mml:math display=\"block\" id=\"Equi_Math\"><mml:mrow><mml:mo>ln</mml:mo><mml:msub><mml:mi>K</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>=</mml:mo><mml:mo>ln</mml:mo><mml:mfrac><mml:msub><mml:mi>q</mml:mi><mml:mi>e</mml:mi></mml:msub><mml:msub><mml:mi>C</mml:mi><mml:mi>e</mml:mi></mml:msub></mml:mfrac><mml:mo>=</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>S</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mi>R</mml:mi></mml:mfrac><mml:mo>-</mml:mo><mml:mfrac><mml:mrow><mml:mi mathvariant=\"normal\">Δ</mml:mi><mml:msup><mml:mi>H</mml:mi><mml:mn>0</mml:mn></mml:msup></mml:mrow><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfrac></mml:mrow></mml:math><tex-math id=\"Equi_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\ln K_{0} = \\ln \\frac{{q_{e} }}{{C_{e} }} = \\frac{{\\Delta S^{0} }}{R} - \\frac{{\\Delta H^{0} }}{RT}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_Equi.gif\"/></alternatives></disp-formula></p><p id=\"Par84\">The heat of adsorption was negative, indicating exothermic nature for ACVR sorption onto G-AC. ACVR is soluble in water and may lose some of its hydration molecules, releasing heat. The spontaneous physisorption process is further established by the ∆<italic>G</italic>° value (0–20 kJ/mol) (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>) (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Further, as the temperature rises from 25 to 55 °C, ∆<italic>G</italic>° increases, indicating that the feasibility of the process declines with temperature rise (Khadhri et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>; Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The favorability of the process was also confirmed with the value of <italic>R</italic><sub><italic>L</italic></sub> reported in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>. Decrease in <italic>K</italic><sub>0</sub> value with temperature also confirms the exothermic nature of ACVR adsorption (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Moreover, the value of ∆<italic>S</italic>° was negative, showing that the ACVR adsorption was enthalpy driven (Fountouli and Chrysikopoulos <xref ref-type=\"bibr\" rid=\"CR20\">2018</xref>). Several pollutant-adsorbent systems have also produced similar results (Thiebault et al. <xref ref-type=\"bibr\" rid=\"CR55\">2015</xref>; Jain et al. <xref ref-type=\"bibr\" rid=\"CR31\">2014a</xref>). Since the Langmuir isotherm supports chemisorption while ∆<italic>G</italic>° (0–20 kJ/mol) supports physisorption. This was further clarified using ∆H° which indicates physisorption is being accompanied by chemisorption. This concludes that the adsorption of ACVR onto G-AC is in the transition phase from physisorption to chemisorption.</p></sec><sec id=\"Sec18\"><title>Effect of volumetric flow rate</title><p id=\"Par85\">The breakthrough response for ACVR adsorption was examined at various volumetric flow rates (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). As the volumetric flow rate rises from 3.1 to 3.8 mL/min, the breakthrough times at C<sub>t</sub>/C<sub>0</sub> = 0.1 shifted to 75 min from 120 min. It can be explained as at a high flow rate, the contact time between the G-AC and ACVR reduces, limiting the bed's adsorption capacity and processing times (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>). Furthermore, at a higher flow rate, the breakthrough curve shifts near the origin, become steeper, and reaches saturation more quickly.<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Effect of volumetric flow rate</p></caption><graphic id=\"MO6\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig5_HTML.png\"/></fig></p><p id=\"Par86\">Figure <xref ref-type=\"fig\" rid=\"Fig5\">5</xref> demonstrates that C<sub>t</sub>/C<sub>0</sub> increased from 0.12 to 0.32 at 3.1 and 4.5 mL/min flow rates, respectively, after 2 h. ACVR has more time to permeate through bed when the flow rate decreases, resulting in an increased column uptake. Likewise, higher volumetric flow rate diminishes the thickness of fluid film surrounding the sorbent particles, reducing mass transfer resistance and a quick increase in mass transfer (Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>). Similar trends have been reported in previous studies for the adsorptive removal of various pharmaceutical drugs (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec19\"><title>Effect of bed height</title><p id=\"Par87\">Figure <xref ref-type=\"fig\" rid=\"Fig6\">6</xref> presents breakthrough curves data at varying bed depths for a 200 mg/L concentrated solution and a volumetric flow rate of 3.1 mL/min.<fig id=\"Fig6\"><label>Fig. 6</label><caption xml:lang=\"en\"><p>Effect of bed height</p></caption><graphic id=\"MO7\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig6_HTML.png\"/></fig></p><p id=\"Par88\">The breakthrough time and exhaustion time decreased on reducing bed depth, moving the breakthrough curve towards the origin, as shown in Fig. <xref ref-type=\"fig\" rid=\"Fig6\">6</xref>. Also, C<sub>t</sub>/C<sub>0</sub> for a short-fixed bed rises fast than for a long bed. Breakthrough concentrations (C<sub>t</sub>/C<sub>0</sub> = 0.1) were attained at contact times of 2 h and 7 h for 3.5 and 12 cm of bed heights, respectively. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub>0</sub> varied from 0.59 and 0.11 at bed heights of 3.5 and 12 cm, respectively, after 7 h. With a greater bed height, more adsorbent load is used resulting in more adsorption sites and a longer contact time to attain 100% saturation (Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Furthermore, as the bed height rises, the adsorbate has considerable time for diffusion through the pores of the adsorbent, increasing the overall adsorbed amount and ACVR removal efficiency (Reynel-Avila et al. <xref ref-type=\"bibr\" rid=\"CR50\">2015</xref>). Similar results for the adsorption of several pharmacological drugs have been reported in the literature (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec20\"><title>Impact of inlet ACVR concentration</title><p id=\"Par89\">Figure <xref ref-type=\"fig\" rid=\"Fig7\">7</xref> shows that the inlet ACVR concentration considerably impacts the breakthrough curve.<fig id=\"Fig7\"><label>Fig. 7</label><caption xml:lang=\"en\"><p>Effect of inlet ACVR concentration</p></caption><graphic id=\"MO8\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig7_HTML.png\"/></fig></p><p id=\"Par90\">Time to reach breakthrough and exhaustion increased as the inlet ACVR concentration decreased. The breakthrough curve moved away from the origin at lower inlet concentration. <italic>C</italic><sub><italic>t</italic></sub>/<italic>C</italic><sub><italic>0</italic></sub> varied from 0.03 to 0.10 at inlet concentrations of 200 and 300 mg/L, respectively, at a contact time of 90 min. This phenomenon may be explained by an increase in the sorption rate and the mass transfer driving force across the liquid film, which causes the fixed bed to saturate more quickly. These study results on the impact of inflow concentration on breakthrough curves concur with other bio-refractory compounds reported literature studies (Darweesh and Ahmed <xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>; Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>).</p></sec><sec id=\"Sec21\"><title>Thomas model</title><p id=\"Par91\">The Thomas model investigates how maximum column uptake and breakthrough curves are evaluated. This model emphasizes that the process follows Langmuir kinetics, and there is no axial dispersion (Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). It is also derived under the presumption that the rate of the driving force is subject to reversible 2nd order kinetics. It is advisable for the adsorption system when there is essentially no resistance to internal diffusion (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> provides more information on the obtained <italic>k</italic><sub>TH</sub>, <italic>q</italic><sub>TH</sub>, and <italic>R</italic><sup>2</sup> values at various process conditions (Fig. S7).<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>Parameters of kinetic analytical models for ACVR adsorption for different sets of process conditions</p></caption><table frame=\"hsides\" rules=\"groups\"><tbody><tr><td align=\"left\" rowspan=\"3\"><p>Process conditions</p></td><td align=\"left\"><p><italic>Z</italic> (cm)</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>8</p></td><td align=\"left\"><p>12</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td><td align=\"left\"><p>3.5</p></td></tr><tr><td align=\"left\"><p><italic>Q</italic> (mL/min)</p></td><td align=\"left\"><p>3.8</p></td><td align=\"left\"><p>4.5</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td><td align=\"left\"><p>3.1</p></td></tr><tr><td align=\"left\"><p><italic>C</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>200</p></td><td align=\"left\"><p>300</p></td><td align=\"left\"><p>400</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Thomas model</p></td><td align=\"left\"><p>k<sub>TH</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>3.15</p></td><td align=\"left\"><p>4.45</p></td><td align=\"left\"><p>2.20</p></td><td align=\"left\"><p>1.35</p></td><td align=\"left\"><p>2.90</p></td><td align=\"left\"><p>1.97</p></td><td align=\"left\"><p>1.57</p></td></tr><tr><td align=\"left\"><p>q<sub>TH</sub> (mg/g)</p></td><td align=\"left\"><p>211.81</p></td><td align=\"left\"><p>199.54</p></td><td align=\"left\"><p>248.98</p></td><td align=\"left\"><p>242.69</p></td><td align=\"left\"><p>224.91</p></td><td align=\"left\"><p>302.20</p></td><td align=\"left\"><p>312.93</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"4\"><p>Yoon–Nelson model</p></td><td align=\"left\"><p><italic>k</italic><sub>YN</sub> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0063</p></td><td align=\"left\"><p>0.0089</p></td><td align=\"left\"><p>0.0044</p></td><td align=\"left\"><p>0.0027</p></td><td align=\"left\"><p>0.0058</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p>0.0063</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic> (min)</p></td><td align=\"left\"><p>278.70</p></td><td align=\"left\"><p>221.71</p></td><td align=\"left\"><p>803.16</p></td><td align=\"left\"><p>1174.33</p></td><td align=\"left\"><p>362.76</p></td><td align=\"left\"><p>324.95</p></td><td align=\"left\"><p>252.36</p></td></tr><tr><td align=\"left\"><p><italic>τ</italic><sub>exp</sub> (min)</p></td><td align=\"left\"><p>270</p></td><td align=\"left\"><p>210</p></td><td align=\"left\"><p>660</p></td><td align=\"left\"><p>1200</p></td><td align=\"left\"><p>360</p></td><td align=\"left\"><p>330</p></td><td align=\"left\"><p>210</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.91</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.90</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Adams–Bohart model</p></td><td align=\"left\"><p><italic>k</italic><sub>BA</sub> × 10<sup>−5</sup> (L/mg/min)</p></td><td align=\"left\"><p>1.70</p></td><td align=\"left\"><p>2.25</p></td><td align=\"left\"><p>1.60</p></td><td align=\"left\"><p>0.95</p></td><td align=\"left\"><p>1.75</p></td><td align=\"left\"><p>1.13</p></td><td align=\"left\"><p>0.82</p></td></tr><tr><td align=\"left\"><p><italic>N</italic><sub>0</sub> (mg/L)</p></td><td align=\"left\"><p>7733.02</p></td><td align=\"left\"><p>7393.39</p></td><td align=\"left\"><p>5684.98</p></td><td align=\"left\"><p>5538.09</p></td><td align=\"left\"><p>7182.78</p></td><td align=\"left\"><p>10388.09</p></td><td align=\"left\"><p>12462.19</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.81</p></td><td align=\"left\"><p>0.80</p></td><td align=\"left\"><p>0.85</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.82</p></td><td align=\"left\"><p>0.77</p></td><td align=\"left\"><p>0.77</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Clark model</p></td><td align=\"left\"><p><italic>r</italic> (min<sup>−1</sup>)</p></td><td align=\"left\"><p>0.0071</p></td><td align=\"left\"><p>0.01</p></td><td align=\"left\"><p>0.0054</p></td><td align=\"left\"><p>0.0032</p></td><td align=\"left\"><p>0.0067</p></td><td align=\"left\"><p>0.0068</p></td><td align=\"left\"><p>0.0072</p></td></tr><tr><td align=\"left\"><p><italic>A</italic></p></td><td align=\"left\"><p>11.89</p></td><td align=\"left\"><p>16.26</p></td><td align=\"left\"><p>115.75</p></td><td align=\"left\"><p>68.27</p></td><td align=\"left\"><p>18.30</p></td><td align=\"left\"><p>14.71</p></td><td align=\"left\"><p>9.94</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.90</p></td><td align=\"left\"><p>0.92</p></td><td align=\"left\"><p>0.88</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.89</p></td><td align=\"left\"><p>0.87</p></td><td align=\"left\"><p>0.89</p></td></tr><tr><td align=\"left\" rowspan=\"3\"><p>Yan model</p></td><td align=\"left\"><p><italic>k</italic><sub>Y</sub> (L/mg/min)</p></td><td align=\"left\"><p>0.030</p></td><td align=\"left\"><p>0.040</p></td><td align=\"left\"><p>0.035</p></td><td align=\"left\"><p>0.032</p></td><td align=\"left\"><p>0.027</p></td><td align=\"left\"><p>0.015</p></td><td align=\"left\"><p>0.010</p></td></tr><tr><td align=\"left\"><p><italic>q</italic><sub>0</sub> (mg/g)</p></td><td align=\"left\"><p>0.756</p></td><td align=\"left\"><p>0.892</p></td><td align=\"left\"><p>0.307</p></td><td align=\"left\"><p>0.205</p></td><td align=\"left\"><p>0.617</p></td><td align=\"left\"><p>0.929</p></td><td align=\"left\"><p>1.239</p></td></tr><tr><td align=\"left\"><p><italic>R</italic><sup>2</sup></p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.99</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.96</p></td><td align=\"left\"><p>0.98</p></td><td align=\"left\"><p>0.94</p></td><td align=\"left\"><p>0.99</p></td></tr></tbody></table></table-wrap></p><p id=\"Par92\">It indicates agreement between the Thomas model and experimental data for breakthrough curves (0.89 &gt; <italic>R</italic><sup>2</sup> &gt; 0.99), signifying the good fit of the model for ACVR adsorption on G-AC. Since the experimental data fitted the Thomas model well, film and intraparticle diffusion are not expected to be the limiting step. But, the limiting adsorption stage is the solid surface mass transfer (de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>). Flow resistance increases on increasing bed height, resulting in decreased mass transfer effect, causing a decrement in <italic>k</italic><sub>TH</sub> and increase in <italic>q</italic><sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>). On the other hand, increasing flow rate will result in insufficient contact time between ACVR and G-AC for fixed number of vacant sites causing decrease in the value of <italic>q</italic><sub>TH</sub> whereas increase in k<sub>TH</sub> (Du et al. <xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>). An increased driving force achieved a higher column performance at higher ACVR concentration. The results are consistent with the literature (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Pholosi et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2012</xref>; Baral et al. <xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>).</p><p id=\"Par93\">Table <xref ref-type=\"table\" rid=\"Tab5\">5</xref> compares the value of q<sub>TH</sub> of G-AC obtained in the present study with the literature. The maximum adsorption capacity of G-AC (q<sub>TH</sub>) in the present study has been found to be higher than other adsorbate-adsorbent systems.<table-wrap id=\"Tab5\"><label>Table 5</label><caption xml:lang=\"en\"><p>Comparison of q<sub>TH</sub> with values reported in literature</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Adsorbent</p></th><th align=\"left\"><p>Adsorbate</p></th><th align=\"left\"><p>Models investigated*</p></th><th align=\"left\"><p><italic>q</italic><sub>TH</sub> (mg/g)</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>G-AC</p></td><td align=\"left\"><p>ACVR</p></td><td align=\"left\"><p>TH, YN, AB, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>224.91</p></td><td align=\"left\"><p>Present study</p></td></tr><tr><td align=\"left\"><p>Thermally activated weed Salvinia Cucullata</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>159.20</p></td><td align=\"left\"><p>Baral et al. (<xref ref-type=\"bibr\" rid=\"CR4\">2009</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.09</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Norfloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>1.99</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR13\">2017b</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date stones</p></td><td align=\"left\"><p>Levofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>2.56</p></td><td align=\"left\"><p>Darweesh and Ahmed (<xref ref-type=\"bibr\" rid=\"CR12\">2017a</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon</p></td><td align=\"left\"><p>Amoxicillin</p></td><td align=\"left\"><p>TH, AB, Yan</p></td><td align=\"char\" char=\".\"><p>11.48</p></td><td align=\"left\"><p>de Franco et al. (<xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated sludge derived activated carbon</p></td><td align=\"left\"><p>Ciprofloxacin</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>14.00</p></td><td align=\"left\"><p>Gupta and Garg (<xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Carboxyl-modified jute fiber</p></td><td align=\"left\"><p>Cu (II)</p></td><td align=\"left\"><p>TH, YN</p></td><td align=\"char\" char=\".\"><p>42.21</p></td><td align=\"left\"><p>Du et al. (<xref ref-type=\"bibr\" rid=\"CR18\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Activated carbon from date palm petiole</p></td><td align=\"left\"><p>Indigo carmine</p></td><td align=\"left\"><p>TH, YN, AB, Yan</p></td><td align=\"char\" char=\".\"><p>48.88</p></td><td align=\"left\"><p>Khadhri et al. (<xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Bone char</p></td><td align=\"left\"><p>Crystal violet</p></td><td align=\"left\"><p>TH, Yan</p></td><td align=\"char\" char=\".\"><p>22.44</p></td><td align=\"left\"><p>Cruz et al. (<xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Magnetic pine cone composite</p></td><td align=\"left\"><p>Cr (VI)</p></td><td align=\"left\"><p>TH, YN, AB</p></td><td align=\"char\" char=\".\"><p>33.08</p></td><td align=\"left\"><p>Pholosi et al. (<xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Acid modified alumina</p></td><td align=\"left\"><p>Fluoride</p></td><td align=\"left\"><p>TH, YN, Clark, Yan</p></td><td align=\"char\" char=\".\"><p>19.12</p></td><td align=\"left\"><p>Kumari et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>*<italic>TH</italic> Thomas model, <italic>YN</italic> Yoon–Nelson model, <italic>AB</italic> Adams–Bohart model</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec22\"><title>Yoon–Nelson model</title><p id=\"Par94\">This model presupposes that the decline in the adsorption rate of each adsorbate molecule is proportionally related to the adsorbate adsorption probability and probability of its breakthrough capacity onto the adsorbent (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>; Yoon and Nelson <xref ref-type=\"bibr\" rid=\"CR60\">1984</xref>). This model is significant in determining the nature of the adsorption process with its exhaustion time for a given concentration (Gupta and Garg <xref ref-type=\"bibr\" rid=\"CR24\">2019</xref>). The estimated values of various Yoon–Nelson model parameters are summarized in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. It shows the strong correlation between experimental data and the Yoon–Nelson model (0.89 &lt; <italic>R</italic><sup>2</sup> &lt; 0.99) (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> and Fig. S8), indicating the adequacy of the model. According to the Yoon–Nelson model's fitting parameters, an increase in the initial concentration of ACVR leads to the enhancement of <italic>K</italic><sub>YN</sub> value and reduction in τ value. It might be because the G-AC in the column quickly saturates.</p><p id=\"Par95\">On the contrary, as the bed height was raised, the <italic>K</italic><sub>YN</sub> declined and the value of <italic>τ</italic> improved. This is because the amount of adsorbent increases as the height of the bed rises. Other studies have also reported consistent patterns (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Demarchi et al. <xref ref-type=\"bibr\" rid=\"CR16\">2015</xref>).</p></sec><sec id=\"Sec23\"><title>Adams–Bohart model</title><p id=\"Par96\">The Adams–Bohart model is based on the premise that equilibrium is not instantaneous and primarily concerned with the surface reaction theory. Therefore, rate of adsorption is related to adsorbate concentration in the solution and the adsorbent’s residual capacity (Malkoc et al. <xref ref-type=\"bibr\" rid=\"CR40\">2006</xref>). The initial segment of the breakthrough curve has been described by Adams–Bohart model in several previous studies. However, in the current study, the model has been fitted from the beginning of breakthrough curve to final point (Fig. S9). The estimated model parameters and R<sup>2</sup> values for various process conditions are provided in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> shows that the values of k<sub>BA</sub> were found to increase with increasing flow rate but deceased with increasing initial ACVR concentration and bed height. At most of the process conditions, the correlation coefficient is low (&lt; 0.86); hence, the correlation between experimental data and the Adams–Bohart model is not strong. Other investigations have also reported similar results. (Biswas and Mishra <xref ref-type=\"bibr\" rid=\"CR6\">2015</xref>; Liao et al. <xref ref-type=\"bibr\" rid=\"CR36\">2013</xref>).</p></sec><sec id=\"Sec24\"><title>Clark model</title><p id=\"Par97\">The Clark model anticipates the breakthrough curve’s behavior by studying mass transfer and adsorption equilibrium (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). It assumes that the flow in the column is plug flow type, and the Freundlich isotherm is followed by mass transfer occurring in the column (Patel <xref ref-type=\"bibr\" rid=\"CR44\">2019</xref>). Fig. S10 shows the linear curve fitting of breakthrough curves using the Clark model. Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref> presents the value of Clark model parameters, ‘<italic>A</italic>’ and ‘<italic>r</italic>’, at different process conditions. On increasing the bed height of the column, the value of <italic>r</italic> decreased. Further, <italic>r</italic> increases with flow rate and inlet ACVR concentration. At almost all process conditions, the value of correlation coefficient is in the range (0.87–0.92), indicating relatively poor fit of the Clark model. Similar results have been obtained earlier (Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>).</p></sec><sec id=\"Sec25\"><title>Yan model</title><p id=\"Par98\">It has been reported earlier that, at absurdly short and very long operation times, the Yan model is proposed to reduce the inaccuracy that occurs from using the Thomas model (Maheshwari and Gupta <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR59\">2001</xref>). Figure <xref ref-type=\"fig\" rid=\"Fig8\">8</xref> illustrates the linear curve fitting of experimental data using the Yan model for various parameters.<fig id=\"Fig8\"><label>Fig. 8</label><caption xml:lang=\"en\"><p>Yan model for varying <bold>a</bold> flow rate, <bold>b</bold> bed height, and <bold>c</bold> initial ACVR concentration</p></caption><graphic id=\"MO9\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2810_Fig8_HTML.png\"/></fig></p><p id=\"Par99\">The findings indicate a good agreement between the experimental data and the model, with R<sup>2</sup> values in the ranges of 0.94–0.99 (Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>). The value of k<sub>Y</sub> increases on increasing flow rate, while decreases with concentration. According to Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>, the estimated adsorption capacity values from the Thomas and Yan model are of the same order. However, it is observed that the Yan model is better at evaluating adsorption capacity of the bed than the Thomas model which estimates inflated value (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>). Other works of literature have also reported similar overall results (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; Cruz et al. <xref ref-type=\"bibr\" rid=\"CR11\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Comparison of breakthrough curve fitting</title><p id=\"Par100\">Thomas and Yan models suggest that external and intraparticle diffusion are not controlling, but the solid surface mass transfer (surface diffusion) is the limiting step in the adsorption process. (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>; de Franco et al. <xref ref-type=\"bibr\" rid=\"CR15\">2017</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR26\">2009</xref>). Adams–Bohart model demonstrated the lowest average <italic>R</italic><sup>2</sup> (0.82) signifying the weak correlation between experimental data and the model. Although, the average value of correlation coefficient for Clark model is 0.90. The poor fit of the Clark model for breakthrough data is in accordance with the batch study results, which fitted well to Langmuir isotherm. The inability of Adams–Bohart and Clark models to accurately predict the removal of ACVR by G-AC can be attributed to two factors: the existence of numerous mechanisms (possibly, adsorption and ion exchange) that contribute to ACVR adsorption onto G-AC and the significant involvement of mass transfer resistance in the removal of ACVR using G-AC (Dorado et al. <xref ref-type=\"bibr\" rid=\"CR17\">2014</xref>).</p><p id=\"Par101\">The Yan model exhibited the highest average <italic>R</italic><sup>2</sup> (0.98), indicating the best fit of the model and the experimental data. As noted in the previous section, batch ACVR adsorption onto G-AC was found to follow the pseudo-second-order kinetic model and Langmuir isotherm model. Both of these conclusions have been supported by Yan models applied to fixed-bed column studies.</p><p id=\"Par102\">Continuous column studies of ACVR adsorption on any of the adsorbent has been scarcely reported. Therefore, q<sub>TH</sub> value signifying adsorption capacity of the adsorbent obtained using Thomas model has been compared for the present study with results reported in literature for other adsorbate-adsorbent systems.</p></sec></sec><sec id=\"Sec27\" sec-type=\"conclusions\"><title>Conclusions</title><p id=\"Par103\">Antiviral pollutants, such as ACVR have been found in groundwater, surface water, and industrial wastewater. Their inadequate removal through conventional treatment is a significant challenge in water reuse. G-AC has proven to be an efficient adsorbent for the ACVR removal from aqueous solutions. The characterization showed the morphology and surface area of G-AC along with the ACVR adsorption (SEM, BET, and EDX).</p><p id=\"Par104\">Batch experiments showed that Langmuir isotherm adequately represents the equilibrium data indicating monolayer chemisorption is the predominant mechanism for ACVR adsorption onto G-AC. Pseudo-second-order kinetic model predicted the adsorption process. Negative values of heat of adsorption and Gibbs free energy indicated exothermic nature of the process and confirmed the process feasibility. The process of adsorption was found to be physisorption accompanied by chemisorption. The feasibility of adsorption decreases with increase in temperature. The adsorption process slightly decreases the randomness of molecules at the solid/liquid interface and is enthalpy driven.</p><p id=\"Par105\">The fixed-bed column experimental data were fitted to different analytical models. The Yan model fitted the data best, followed by the Thomas model confirming the pseudo-second-order kinetics and Langmuir isotherm. Adsorption capacity of the fixed bed of G-AC estimated using Thomas model indicated that the adsorption capacity is excellent for ACVR when compared to other adsorbate-adsorbent system studied in literature. For ACVR-contaminated wastewater, the G-AC fixed-bed column can be effectively applied in continuous operation mode.</p><p id=\"Par106\">The current research's findings will be useful for small volumes of wastewater generated by ACVR manufacturing units. ACVR removal from the industrial effluents released by bulk manufacturing and formulation industries can be achieved in a continuous flow unit using a packed bed of a commercial adsorbent. This will prevent the entry of strong inhibitory substances, like ACVR, into the biological wastewater treatment units, which may inhibit the activity of the micro-organism that treat the soluble organics.</p></sec></body><back><ack><title>Acknowledgements</title><p>No specific grant was given to this research by funding organizations in the public, commercial, or not-for-profit sectors. However, one of the authors (AG) expresses her gratitude to Malaviya National Institute of Technology (MNIT) Jaipur and Ministry of Education, Govt. of India for providing the financial support in the form of Institute Fellowship for carrying out Ph. D. research work. The authors also wish to thank the Material Research Centre (MRC), MNIT Jaipur, for carrying out SEM, EDX and BET analysis of the samples.</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par107\">The authors report no declarations of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Al-Otaibi</surname><given-names>JS</given-names></name><name><surname>Mary</surname><given-names>YS</given-names></name><name><surname>Thomas</surname><given-names>R</given-names></name><name><surname>Kaya</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Detailed electronic structure, physico-chemical properties, excited state properties, virtual bioactivity screening and SERS analysis of three guanine based antiviral drugs valacyclovir HCl hydrate, acyclovir and ganciclovir</article-title><source>Polycycl Aromat Compd</source><year>2020</year><volume>42</volume><fpage>1260</fpage><lpage>1270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVKmu7jE</pub-id><pub-id pub-id-type=\"doi\">10.1080/10406638.2020.1773876</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azuma</surname><given-names>T</given-names></name><name><surname>Arima</surname><given-names>N</given-names></name><name><surname>Tsukada</surname><given-names>A</given-names></name><name><surname>Hirami</surname><given-names>S</given-names></name><name><surname>Matsuoka</surname><given-names>R</given-names></name><name><surname>Moriwake</surname><given-names>R</given-names></name><name><surname>Ishiuchi</surname><given-names>H</given-names></name><name><surname>Inoyama</surname><given-names>T</given-names></name><name><surname>Teranishi</surname><given-names>Y</given-names></name><name><surname>Yamaoka</surname><given-names>M</given-names></name><name><surname>Mino</surname><given-names>Y</given-names></name><name><surname>Hayashi</surname><given-names>T</given-names></name><name><surname>Fujita</surname><given-names>Y</given-names></name><name><surname>Masada</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Science of the Total Environment Detection of pharmaceuticals and phytochemicals together with their metabolites in hospital effluents in Japan, and their contribution to sewage treatment plant influents</article-title><source>Sci Total Environ</source><year>2016</year><volume>548–549</volume><fpage>189</fpage><lpage>197</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KqsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2015.12.157</pub-id><pub-id pub-id-type=\"pmid\">26802347</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Babas</surname><given-names>H</given-names></name><name><surname>Kaichouh</surname><given-names>G</given-names></name><name><surname>Khachani</surname><given-names>M</given-names></name><name><surname>Karbane</surname><given-names>ME</given-names></name><name><surname>Chakir</surname><given-names>A</given-names></name><name><surname>Guenbour</surname><given-names>A</given-names></name><name><surname>Bellaouchou</surname><given-names>A</given-names></name><name><surname>Warad</surname><given-names>I</given-names></name><name><surname>Zarrouk</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Equilibrium and kinetic studies for removal of antiviral sofosbuvir from aqueous solution by adsorption on expanded perlite: experimental, modelling and optimization</article-title><source>Surf Interfaces</source><year>2021</year><volume>23</volume><fpage>100962</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsFaqtr%2FF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.surfin.2021.100962</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Baral</surname><given-names>SS</given-names></name><name><surname>Das</surname><given-names>N</given-names></name><name><surname>Ramulu</surname><given-names>TS</given-names></name><name><surname>Sahoo</surname><given-names>SK</given-names></name><name><surname>Das</surname><given-names>SN</given-names></name><name><surname>Chaudhury</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) by thermally activated weed Salvinia cucullata in a fixed-bed column</article-title><source>J Hazard Mater</source><year>2009</year><volume>161</volume><fpage>1427</fpage><lpage>1435</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhsVSqsbnE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2008.04.127</pub-id><pub-id pub-id-type=\"pmid\">18571842</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bing</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Enhanced mineralization of pharmaceuticals by surface oxidation over mesoporous γ-Ti-Al<sub>2</sub>O<sub>3</sub> suspension with ozone</article-title><source>Appl Catal B Environ</source><year>2017</year><volume>202</volume><fpage>118</fpage><lpage>126</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFWnur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2016.09.019</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Biswas</surname><given-names>S</given-names></name><name><surname>Mishra</surname><given-names>U</given-names></name></person-group><article-title xml:lang=\"en\">Continuous fixed-bed column study and adsorption modeling: Removal of lead ion from aqueous solution by charcoal originated from chemical carbonization of rubber wood sawdust</article-title><source>J Chem</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1155/2015/907379</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Borah</surname><given-names>D</given-names></name><name><surname>Satokawa</surname><given-names>S</given-names></name><name><surname>Kato</surname><given-names>S</given-names></name><name><surname>Kojima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Surface-modified carbon black for As(V) removal</article-title><source>J Colloid Interface Sci</source><year>2008</year><volume>319</volume><fpage>53</fpage><lpage>62</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXnvVWntg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2007.11.019</pub-id><pub-id pub-id-type=\"pmid\">18082179</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bradley</surname><given-names>PM</given-names></name><name><surname>Barber</surname><given-names>LB</given-names></name><name><surname>Duris</surname><given-names>JW</given-names></name><name><surname>Foreman</surname><given-names>WT</given-names></name><name><surname>Furlong</surname><given-names>ET</given-names></name><name><surname>Hubbard</surname><given-names>LE</given-names></name><name><surname>Hutchinson</surname><given-names>KJ</given-names></name><name><surname>Keefe</surname><given-names>SH</given-names></name><name><surname>Kolpin</surname><given-names>DW</given-names></name></person-group><article-title xml:lang=\"en\">Riverbank filtration potential of pharmaceuticals in a wastewater-impacted stream</article-title><source>Environ Pollut</source><year>2014</year><volume>193</volume><fpage>173</fpage><lpage>180</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1ensbnK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.envpol.2014.06.028</pub-id><pub-id pub-id-type=\"pmid\">25038376</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Yue</surname><given-names>Q</given-names></name><name><surname>Gao</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Xu</surname><given-names>X</given-names></name><name><surname>Fu</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of hexavalent chromium from aqueous solution by modified corn stalk: a fixed-bed column study</article-title><source>Bioresour Technol</source><year>2012</year><volume>113</volume><fpage>114</fpage><lpage>120</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFWntrg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2011.11.110</pub-id><pub-id pub-id-type=\"pmid\">22189077</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Luo</surname><given-names>H</given-names></name><name><surname>Shi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Anatase TiO<sub>2</sub> nanoparticles-carbon nanotubes composite: optimization synthesis and the relationship of photocatalytic degradation activity of acyclovir in water</article-title><source>Appl Catal A Gen</source><year>2014</year><volume>485</volume><fpage>188</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhsVSmsLbP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcata.2014.08.004</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cruz</surname><given-names>MAP</given-names></name><name><surname>Guimarães</surname><given-names>LCM</given-names></name><name><surname>da Costa Júnior</surname><given-names>EF</given-names></name><name><surname>Rocha</surname><given-names>SDF</given-names></name><name><surname>da Mesquita</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of crystal violet from aqueous solution in continuous flow system using bone char</article-title><source>Chem Eng Commun</source><year>2020</year><volume>207</volume><fpage>372</fpage><lpage>381</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXms1amsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/00986445.2019.1596899</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Batch and fixed bed adsorption of levofloxacin on granular activated carbon from date (<italic>Phoenix dactylifera</italic> L.) stones by KOH chemical activation</article-title><source>Environ Toxicol Pharmacol</source><year>2017</year><volume>50</volume><fpage>159</fpage><lpage>166</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitlOls7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.etap.2017.02.005</pub-id><pub-id pub-id-type=\"pmid\">28189062</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Darweesh</surname><given-names>TM</given-names></name><name><surname>Ahmed</surname><given-names>MJ</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of ciprofloxacin and norfloxacin from aqueous solution onto granular activated carbon in fixed bed column</article-title><source>Ecotoxicol Environ Saf</source><year>2017</year><volume>138</volume><fpage>139</fpage><lpage>145</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXislShsQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2016.12.032</pub-id><pub-id pub-id-type=\"pmid\">28040619</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Clercq</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Antivirals and antiviral strategies</article-title><source>Nat Rev Microbiol</source><year>2004</year><volume>2</volume><fpage>704</fpage><lpage>720</lpage><pub-id pub-id-type=\"doi\">10.1038/nrmicro975</pub-id><pub-id pub-id-type=\"pmid\">15372081</pub-id><pub-id pub-id-type=\"pmcid\">7097272</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Franco</surname><given-names>MAE</given-names></name><name><surname>de Carvalho</surname><given-names>CB</given-names></name><name><surname>Bonetto</surname><given-names>MM</given-names></name><name><surname>de Soares</surname><given-names>RS</given-names></name><name><surname>Féris</surname><given-names>LA</given-names></name></person-group><article-title xml:lang=\"en\">Removal of amoxicillin from water by adsorption onto activated carbon in batch process and fixed bed column: kinetics, isotherms, experimental design and breakthrough curves modelling</article-title><source>J Clean Prod</source><year>2017</year><volume>161</volume><fpage>947</fpage><lpage>956</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVant7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2017.05.197</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Demarchi</surname><given-names>CA</given-names></name><name><surname>Debrassi</surname><given-names>A</given-names></name><name><surname>Magro</surname><given-names>JD</given-names></name><name><surname>Nedelko</surname><given-names>N</given-names></name><name><surname>Ślawska-Waniewska</surname><given-names>A</given-names></name><name><surname>Dłuzewski</surname><given-names>P</given-names></name><name><surname>Greneche</surname><given-names>JM</given-names></name><name><surname>Rodrigues</surname><given-names>CA</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of Cr(VI) on crosslinked chitosan-Fe(III) complex in fixed-bed systems</article-title><source>J Water Process Eng</source><year>2015</year><volume>7</volume><fpage>141</fpage><lpage>152</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2015.05.003</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dorado</surname><given-names>AD</given-names></name><name><surname>Gamisans</surname><given-names>X</given-names></name><name><surname>Valderrama</surname><given-names>C</given-names></name><name><surname>Solé</surname><given-names>M</given-names></name><name><surname>Lao</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Cr(III) removal from aqueous solutions: a straightforward model approaching of the adsorption in a fixed-bed column</article-title><source>J Environ Sci Heal Part A Toxic/hazardous Subst Environ Eng</source><year>2014</year><volume>49</volume><fpage>179</fpage><lpage>186</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslWisbfJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/10934529.2013.838855</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>Z</given-names></name><name><surname>Zheng</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Experimental and modelling studies on fixed bed adsorption for Cu(II) removal from aqueous solution by carboxyl modified jute fiber</article-title><source>Powder Technol</source><year>2018</year><volume>338</volume><fpage>952</fpage><lpage>959</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVOltLjP</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.powtec.2018.06.015</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fan</surname><given-names>L</given-names></name><name><surname>Luo</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>F</given-names></name><name><surname>Qiu</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Fabrication of novel magnetic chitosan grafted with graphene oxide to enhance adsorption properties for methyl blue</article-title><source>J Hazard Mater</source><year>2012</year><volume>215–216</volume><fpage>272</fpage><lpage>279</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XkvVWju74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2012.02.068</pub-id><pub-id pub-id-type=\"pmid\">22429622</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fountouli</surname><given-names>TV</given-names></name><name><surname>Chrysikopoulos</surname><given-names>CV</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and thermodynamics of pharmaceuticals, acyclovir and fluconazole, onto quartz sand under static and dynamic conditions</article-title><source>Environ Eng Sci</source><year>2018</year><volume>35</volume><fpage>909</fpage><lpage>917</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslSgu7jF</pub-id><pub-id pub-id-type=\"doi\">10.1089/ees.2017.0499</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Funke</surname><given-names>J</given-names></name><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Identification of transformation products of antiviral drugs formed during biological wastewater treatment and their occurrence in the urban water cycle</article-title><source>Water Res</source><year>2016</year><volume>98</volume><fpage>75</fpage><lpage>83</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XlvV2qt7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2016.03.045</pub-id><pub-id pub-id-type=\"pmid\">27082694</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fytianos</surname><given-names>K</given-names></name><name><surname>Voudrias</surname><given-names>E</given-names></name><name><surname>Kokkalis</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sorption-desorption behaviour of 2,4-dichlorophenol by marine sediments</article-title><source>Chemosphere</source><year>2000</year><volume>40</volume><fpage>3</fpage><lpage>6</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXivFSrsw%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0045-6535(99)00214-3</pub-id><pub-id pub-id-type=\"pmid\">10665437</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Gan</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>H</given-names></name><name><surname>Cai</surname><given-names>N</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Guo</surname><given-names>P</given-names></name><name><surname>Chen</surname><given-names>B</given-names></name><name><surname>Pan</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Effective degradation of COVID-19 related drugs by biochar-supported red mud catalyst activated persulfate process: Mechanism and pathway</article-title><source>J Clean Prod</source><year>2022</year><volume>340</volume><fpage>130753</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2022.130753</pub-id><pub-id pub-id-type=\"pmid\">36032562</pub-id><pub-id pub-id-type=\"pmcid\">9396784</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Garg</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption and oxidation of ciprofloxacin in a fixed bed column using activated sludge derived activated carbon</article-title><source>J Environ Manag</source><year>2019</year><volume>250</volume><fpage>109474</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhslKnu7bJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jenvman.2019.109474</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gupta</surname><given-names>A</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Gupta</surname><given-names>AB</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence of acyclovir in the aquatic environment, its removal and research perspectives: a review</article-title><source>J Water Process Eng</source><year>2021</year><volume>39</volume><fpage>101855</fpage><pub-id pub-id-type=\"doi\">10.1016/j.jwpe.2020.101855</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>Yu</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Wang Yuanfeng Xie</surname><given-names>F</given-names></name><name><surname>Cheng</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of methylene blue by phoenix tree leaf powder in a fixed-bed column: experiments and prediction of breakthrough curves</article-title><source>Desalination</source><year>2009</year><volume>245</volume><fpage>284</fpage><lpage>297</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXovFyhtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2008.07.013</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hasnain Isa</surname><given-names>M</given-names></name><name><surname>Siew Lang</surname><given-names>L</given-names></name><name><surname>Asaari</surname><given-names>FAH</given-names></name><name><surname>Aziz</surname><given-names>HA</given-names></name><name><surname>Azam Ramli</surname><given-names>N</given-names></name><name><surname>Dhas</surname><given-names>JPA</given-names></name></person-group><article-title xml:lang=\"en\">Low cost removal of disperse dyes from aqueous solution using palm ash</article-title><source>Dye Pigment</source><year>2007</year><volume>74</volume><fpage>446</fpage><lpage>453</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXht1Ort7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.dyepig.2006.02.025</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heidary</surname><given-names>F</given-names></name><name><surname>Madani</surname><given-names>S</given-names></name><name><surname>Gharebaghi</surname><given-names>R</given-names></name><name><surname>Asadi-Amoli</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">Acyclovir as a potential add-on therapy in COVID-19 treatment regimens</article-title><source>Pharm Sci</source><year>2021</year><volume>27</volume><fpage>S68</fpage><lpage>S77</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislalsL%2FF</pub-id><pub-id pub-id-type=\"doi\">10.34172/PS.2021.38</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hosseini Nasr</surname><given-names>A</given-names></name><name><surname>Sadat</surname><given-names>AH</given-names></name><name><surname>Tayebee</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanism of different acyclovir concentrations on 1–2 nm sized magnetite nanoparticles: a molecular dynamics study</article-title><source>J Mol Liq</source><year>2018</year><volume>254</volume><fpage>64</fpage><lpage>69</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVemtro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2018.01.081</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Occurrence and removal of antiviral drugs in environment: a review</article-title><source>Water Air Soil Pollut</source><year>2013</year><pub-id pub-id-type=\"doi\">10.1007/s11270-012-1410-3</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>P</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption optimization of acyclovir on prepared activated carbon</article-title><source>Can J Chem Eng</source><year>2014</year><volume>92</volume><fpage>1627</fpage><lpage>1635</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhtlynur7J</pub-id><pub-id pub-id-type=\"doi\">10.1002/cjce.22026</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jain</surname><given-names>S</given-names></name><name><surname>Vyas</surname><given-names>RK</given-names></name><name><surname>Pandit</surname><given-names>P</given-names></name><name><surname>Dalai</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of antiviral drug, acyclovir from aqueous solution on powdered activated charcoal: kinetics, equilibrium, and thermodynamic studies</article-title><source>Desalin Water Treat</source><year>2014</year><volume>52</volume><fpage>4953</fpage><lpage>4968</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1SktL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2013.810324</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khadhri</surname><given-names>N</given-names></name><name><surname>El Khames</surname><given-names>SM</given-names></name><name><surname>Ben Mosbah</surname><given-names>M</given-names></name><name><surname>Moussaoui</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous column adsorption of indigo carmine onto activated carbon derived from date palm petiole</article-title><source>J Environ Chem Eng</source><year>2019</year><volume>7</volume><fpage>102775</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVygs7rK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jece.2018.11.020</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>U</given-names></name><name><surname>Mishra</surname><given-names>A</given-names></name><name><surname>Siddiqi</surname><given-names>H</given-names></name><name><surname>Meikap</surname><given-names>BC</given-names></name></person-group><article-title xml:lang=\"en\">Effective defluoridation of industrial wastewater by using acid modified alumina in fixed-bed adsorption column: experimental and breakthrough curves analysis</article-title><source>J Clean Prod</source><year>2021</year><volume>279</volume><fpage>123645</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslejtbvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jclepro.2020.123645</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Nie</surname><given-names>X</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>An</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Can environmental pharmaceuticals be photocatalytically degraded and completely mineralized in water using g-C<sub>3</sub>N<sub>4</sub>TiO<sub>2</sub> under visible light irradiation? Implications of persistent toxic intermediates</article-title><source>Appl Catal B Environ</source><year>2016</year><volume>180</volume><fpage>726</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVOqu7nL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.apcatb.2015.07.014</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liao</surname><given-names>P</given-names></name><name><surname>Zhan</surname><given-names>Z</given-names></name><name><surname>Dai</surname><given-names>J</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>K</given-names></name><name><surname>Yuan</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of tetracycline and chloramphenicol in aqueous solutions by bamboo charcoal: a batch and fixed-bed column study</article-title><source>Chem Eng J</source><year>2013</year><volume>228</volume><fpage>496</fpage><lpage>505</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtVOrtb3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2013.04.118</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Yuan</surname><given-names>D</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yu</surname><given-names>B</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of Fe@porous carbon to be an efficient mediator for peroxymonosulfate activation for oxidation of organic contaminants: incorporation NH2-group into structure of its MOF precursor</article-title><source>Chem Eng J</source><year>2018</year><volume>354</volume><fpage>835</fpage><lpage>848</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGqsLbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2018.08.060</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name><name><surname>Liu Liyuan Tian</surname><given-names>X</given-names></name><name><surname>Liu Longyan Xia</surname><given-names>Y</given-names></name><name><surname>Liang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Qi</surname><given-names>F</given-names></name><name><surname>Chu</surname><given-names>W</given-names></name><name><surname>Tsang</surname><given-names>DCW</given-names></name><name><surname>Xu</surname><given-names>B</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>lkhlaq</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Novel carbon based Fe-Co oxides derived from Prussian blue analogues activating peroxymonosulfate: refractory drugs degradation without metal leaching</article-title><source>Chem Eng J</source><year>2020</year><volume>379</volume><fpage>122274</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVajtLjN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2019.122274</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maheshwari</surname><given-names>U</given-names></name><name><surname>Gupta</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of Cr(VI) from wastewater using activated neem bark in a fixed-bed column: interference of other ions and kinetic modelling studies</article-title><source>Desalin Water Treat</source><year>2016</year><volume>57</volume><fpage>8514</fpage><lpage>8525</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXms1Glsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/19443994.2015.1030709</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Malkoc</surname><given-names>E</given-names></name><name><surname>Nuhoglu</surname><given-names>Y</given-names></name><name><surname>Dundar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption of chromium(VI) on pomace-An olive oil industry waste: Batch and column studies</article-title><source>J Hazard Mater</source><year>2006</year><volume>138</volume><fpage>142</fpage><lpage>151</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XhtVOntL3P</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jhazmat.2006.05.051</pub-id><pub-id pub-id-type=\"pmid\">16844293</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mascolo</surname><given-names>G</given-names></name><name><surname>Balest</surname><given-names>L</given-names></name><name><surname>Cassano</surname><given-names>D</given-names></name><name><surname>Laera</surname><given-names>G</given-names></name><name><surname>Lopez</surname><given-names>A</given-names></name><name><surname>Pollice</surname><given-names>A</given-names></name><name><surname>Salerno</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradability of pharmaceutical industrial wastewater and formation of recalcitrant organic compounds during aerobic biological treatment</article-title><source>Bioresour Technol</source><year>2010</year><volume>101</volume><fpage>2585</fpage><lpage>2591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXjvVartQ%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2009.10.057</pub-id><pub-id pub-id-type=\"pmid\">19939666</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Momčilović</surname><given-names>M</given-names></name><name><surname>Purenović</surname><given-names>M</given-names></name><name><surname>Bojić</surname><given-names>A</given-names></name><name><surname>Zarubica</surname><given-names>A</given-names></name><name><surname>Ranđelović</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Removal of lead(II) ions from aqueous solutions by adsorption onto pine cone activated carbon</article-title><source>Desalination</source><year>2011</year><volume>276</volume><fpage>53</fpage><lpage>59</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXnvFOltro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.desal.2011.03.013</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nannou</surname><given-names>C</given-names></name><name><surname>Ofrydopoulou</surname><given-names>A</given-names></name><name><surname>Evgenidou</surname><given-names>E</given-names></name><name><surname>Heath</surname><given-names>D</given-names></name><name><surname>Heath</surname><given-names>E</given-names></name><name><surname>Lambropoulou</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in aquatic environment and wastewater treatment plants: A review on occurrence, fate, removal and ecotoxicity</article-title><source>Sci Total Environ</source><year>2020</year><volume>699</volume><fpage>134322</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVymu7nK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2019.134322</pub-id><pub-id pub-id-type=\"pmid\">31678880</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Fixed-bed column adsorption study: a comprehensive review</article-title><source>Appl Water Sci</source><year>2019</year><volume>9</volume><fpage>1</fpage><lpage>17</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXosVegtrc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13201-019-0927-7</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Huang</surname><given-names>Q</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Ou</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Profile and behavior of antiviral drugs in aquatic environments of the Pearl River Delta</article-title><source>China Sci Total Environ</source><year>2014</year><volume>466–467</volume><fpage>755</fpage><lpage>761</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhslOjs77M</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.scitotenv.2013.07.062</pub-id><pub-id pub-id-type=\"pmid\">23973541</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pholosi</surname><given-names>A</given-names></name><name><surname>Naidoo</surname><given-names>EB</given-names></name><name><surname>Ofomaja</surname><given-names>AE</given-names></name></person-group><article-title xml:lang=\"en\">Batch and continuous flow studies of Cr(VI) adsorption from synthetic and real wastewater by magnetic pine cone composite</article-title><source>Chem Eng Res Des</source><year>2020</year><volume>153</volume><fpage>806</fpage><lpage>818</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitFOjtLvL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cherd.2019.11.004</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Schlüsener</surname><given-names>MP</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral drugs in wastewater and surface waters: a new pharmaceutical class of environmental relevance?</article-title><source>Environ Sci Technol</source><year>2010</year><volume>44</volume><fpage>1728</fpage><lpage>1735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1Sgsb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es903216p</pub-id><pub-id pub-id-type=\"pmid\">20108960</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Biotransformation of the antiviral drugs acyclovir and penciclovir in activated sludge treatment</article-title><source>Environ Sci Technol</source><year>2011</year><volume>45</volume><fpage>2761</fpage><lpage>2769</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXivFSru7o%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es103732y</pub-id><pub-id pub-id-type=\"pmid\">21388176</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasse</surname><given-names>C</given-names></name><name><surname>Wagner</surname><given-names>M</given-names></name><name><surname>Schulz</surname><given-names>R</given-names></name><name><surname>Ternes</surname><given-names>TA</given-names></name></person-group><article-title xml:lang=\"en\">Oxidation of the antiviral drug acyclovir and its biodegradation product carboxy-acyclovir with ozone: kinetics and identification of oxidation products</article-title><source>Environ Sci Technol</source><year>2012</year><volume>46</volume><fpage>2169</fpage><lpage>2178</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsFCis7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/es203712z</pub-id><pub-id pub-id-type=\"pmid\">22300376</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reynel-Avila</surname><given-names>HE</given-names></name><name><surname>Mendoza-Castillo</surname><given-names>DI</given-names></name><name><surname>Bonilla-Petriciolet</surname><given-names>A</given-names></name><name><surname>Silvestre-Albero</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of naproxen adsorption on bone char in aqueous solutions using batch and fixed-bed processes</article-title><source>J Mol Liq</source><year>2015</year><volume>209</volume><fpage>187</fpage><lpage>195</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1yhur7F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.molliq.2015.05.013</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russo</surname><given-names>D</given-names></name><name><surname>Siciliano</surname><given-names>A</given-names></name><name><surname>Guida</surname><given-names>M</given-names></name><name><surname>Galdiero</surname><given-names>E</given-names></name><name><surname>Amoresano</surname><given-names>A</given-names></name><name><surname>Andreozzi</surname><given-names>R</given-names></name><name><surname>Reis</surname><given-names>NM</given-names></name><name><surname>Li Puma</surname><given-names>G</given-names></name><name><surname>Marotta</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Photodegradation and ecotoxicology of acyclovir in water under UV254 and UV254/H<sub>2</sub>O<sub>2</sub> processes</article-title><source>Water Res</source><year>2017</year><volume>122</volume><fpage>591</fpage><lpage>602</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWltL%2FO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.watres.2017.06.020</pub-id><pub-id pub-id-type=\"pmid\">28628881</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saifi</surname><given-names>Z</given-names></name><name><surname>Rizwanullah</surname><given-names>M</given-names></name><name><surname>Mir</surname><given-names>SR</given-names></name><name><surname>Amin</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Bilosomes nanocarriers for improved oral bioavailability of acyclovir: a complete characterization through in vitro, ex-vivo and in vivo assessment</article-title><source>J Drug Deliv Sci Technol</source><year>2020</year><volume>57</volume><fpage>101634</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXkvFCgsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2020.101634</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>I</given-names></name><name><surname>Adnan</surname><given-names>R</given-names></name><name><surname>Ngah</surname><given-names>WSW</given-names></name><name><surname>Mohamed</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Iron impregnated activated carbon as an efficient adsorbent for the removal of methylene blue: regeneration and kinetics studies</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>1</fpage><lpage>23</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhvFCjsb3O</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0122603</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shetti</surname><given-names>NP</given-names></name><name><surname>Malode</surname><given-names>SJ</given-names></name><name><surname>Nayak</surname><given-names>DS</given-names></name><name><surname>Naik</surname><given-names>RR</given-names></name><name><surname>Kuchinad</surname><given-names>GT</given-names></name><name><surname>Reddy</surname><given-names>KR</given-names></name><name><surname>Shukla</surname><given-names>SS</given-names></name><name><surname>Aminabhavi</surname><given-names>TM</given-names></name></person-group><article-title xml:lang=\"en\">Hetero nanostructured iron oxide and bentonite clay composite assembly for the determination of an antiviral drug acyclovir</article-title><source>Microchem J</source><year>2020</year><volume>155</volume><fpage>104727</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ojsr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.microc.2020.104727</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thiebault</surname><given-names>T</given-names></name><name><surname>Guégan</surname><given-names>R</given-names></name><name><surname>Boussafir</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Adsorption mechanisms of emerging micro-pollutants with a clay mineral: Case of tramadol and doxepine pharmaceutical products</article-title><source>J Colloid Interface Sci</source><year>2015</year><volume>453</volume><fpage>1</fpage><lpage>8</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnsFGmuro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jcis.2015.04.029</pub-id><pub-id pub-id-type=\"pmid\">25950945</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thommes</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Physical adsorption characterization of nanoporous materials</article-title><source>Chem-Ing-Tech</source><year>2010</year><volume>82</volume><fpage>1059</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXotFemt7Y%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/cite.201000064</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tognarelli</surname><given-names>EI</given-names></name><name><surname>Palomino</surname><given-names>TF</given-names></name><name><surname>Corrales</surname><given-names>N</given-names></name><name><surname>Bueno</surname><given-names>SM</given-names></name><name><surname>Kalergis</surname><given-names>AM</given-names></name><name><surname>González</surname><given-names>PA</given-names></name></person-group><article-title xml:lang=\"en\">Herpes simplex virus evasion of early host antiviral responses</article-title><source>Front Cell Infect Microbiol</source><year>2019</year><pub-id pub-id-type=\"doi\">10.3389/fcimb.2019.00127</pub-id><pub-id pub-id-type=\"pmid\">31114761</pub-id><pub-id pub-id-type=\"pmcid\">6503643</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Ao</surname><given-names>Z</given-names></name><name><surname>Sun</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>An</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Ag<sub>2</sub>MoO<sub>4</sub> nanoparticles encapsulated in g-C<sub>3</sub>N<sub>4</sub> for sunlight photodegradation of pollutants</article-title><source>Catal Today</source><year>2018</year><volume>315</volume><fpage>205</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisF2ltb0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cattod.2018.01.019</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>G</given-names></name><name><surname>Viraraghavan</surname><given-names>T</given-names></name><name><surname>Chen</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A new model for heavy metal removal in a biosorption column</article-title><source>Adsorpt Sci Technol</source><year>2001</year><volume>19</volume><fpage>25</fpage><lpage>43</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXks1Ghurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1260/0263617011493953</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoon</surname><given-names>YH</given-names></name><name><surname>Nelson</surname><given-names>JH</given-names></name></person-group><article-title xml:lang=\"en\">Application of gas adsorption kinetics I: a theoretical model for respirator cartridge service life</article-title><source>Am Ind Hyg Assoc J</source><year>1984</year><volume>45</volume><fpage>509</fpage><lpage>516</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaL2cXlvFOhurk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/15298668491400197</pub-id><pub-id pub-id-type=\"pmid\">6475758</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Zhu</surname><given-names>H</given-names></name><name><surname>Szewzyk</surname><given-names>U</given-names></name><name><surname>Lübbecke</surname><given-names>S</given-names></name><name><surname>Uwe Geissen</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Removal of emerging organic contaminants with a pilot-scale biofilter packed with natural manganese oxides</article-title><source>Chem Eng J</source><year>2017</year><volume>317</volume><fpage>454</fpage><lpage>460</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXjslWrtbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cej.2017.02.095</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhou</surname><given-names>C</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Xie</surname><given-names>Q</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Fu</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Photolysis of three antiviral drugs acyclovir, zidovudine and lamivudine in surface freshwater and seawater</article-title><source>Chemosphere</source><year>2015</year><volume>138</volume><fpage>792</fpage><lpage>797</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsVegtbjF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chemosphere.2015.08.033</pub-id><pub-id pub-id-type=\"pmid\">26295538</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec28\"><title>Supplementary Information</title><p id=\"Par108\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2810_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 1566 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><glossary id=\"AGS1\"><title>List of symbols</title><def-list><def-item><term>∆<italic>G</italic>°</term><def><p id=\"Par3\">Change in Gibbs free energy (kJ/moL)</p></def></def-item><def-item><term>∆<italic>H</italic>°</term><def><p id=\"Par4\">Change in heat of adsorption (kJ/moL)</p></def></def-item><def-item><term>∆pH</term><def><p id=\"Par5\">Change in pH</p></def></def-item><def-item><term>∆<italic>S</italic>°</term><def><p id=\"Par6\">Change in entropy (kJ/mol/K)</p></def></def-item><def-item><term><italic>A</italic></term><def><p id=\"Par7\">Clark constant</p></def></def-item><def-item><term><italic>a</italic></term><def><p id=\"Par8\">Yan model constant</p></def></def-item><def-item><term><italic>A</italic><sub><italic>T</italic></sub></term><def><p id=\"Par9\">Equilibrium binding constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>b</italic></term><def><p id=\"Par10\">Yan model constant</p></def></def-item><def-item><term><italic>b</italic><sub><italic>T</italic></sub></term><def><p id=\"Par11\">Constant related to heat of adsorption</p></def></def-item><def-item><term><italic>B</italic><sub><italic>T</italic></sub></term><def><p id=\"Par12\">Temkin constant related to heat of adsorption <inline-formula id=\"IEq1\"><alternatives><mml:math id=\"IEq1_Math\"><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:msub><mml:mi>B</mml:mi><mml:mi>T</mml:mi></mml:msub><mml:mo>=</mml:mo><mml:mfenced close=\")\" open=\"(\"><mml:mrow><mml:mi mathvariant=\"italic\">RT</mml:mi></mml:mrow></mml:mfenced><mml:mo stretchy=\"false\">/</mml:mo><mml:msub><mml:mi>b</mml:mi><mml:mi>T</mml:mi></mml:msub></mml:mrow></mml:mfenced></mml:math><tex-math id=\"IEq1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\left( {B_{T} = \\left( {RT} \\right)/b_{T} } \\right)$$\\end{document}</tex-math><inline-graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2810_Article_IEq1.gif\"/></alternatives></inline-formula> (J/mol)</p></def></def-item><def-item><term><italic>C</italic></term><def><p id=\"Par13\">Constant representing the thickness of boundary layer (mg/g)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>0</italic></sub></term><def><p id=\"Par14\">Initial concentration of ACVR in solution (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>e</italic></sub></term><def><p id=\"Par15\">Concentration of ACVR at equilibrium (mg/L)</p></def></def-item><def-item><term><italic>C</italic><sub><italic>f</italic></sub></term><def><p id=\"Par16\">The final concentration of ACVR (mg/L)</p></def></def-item><def-item><term><italic>F</italic></term><def><p id=\"Par17\">Linear velocity or superficial velocity (cm/min)</p></def></def-item><def-item><term><italic>K</italic><sub>0</sub></term><def><p id=\"Par18\">Equilibrium constant</p></def></def-item><def-item><term><italic>k</italic><sub>1</sub></term><def><p id=\"Par19\">Pseudo-first order rate constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>2</sub></term><def><p id=\"Par20\">Pseudo-second-order rate constant (g/mg/min)</p></def></def-item><def-item><term><italic>k</italic><sub>BA</sub></term><def><p id=\"Par21\">Adams–Bohart constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>f</sub></term><def><p id=\"Par22\">Freundlich constant ((mg/g)/(mg/L)<sup>1/n</sup>)</p></def></def-item><def-item><term><italic>K</italic><sub>L</sub></term><def><p id=\"Par23\">Langmuir adsorption constant (L/mg)</p></def></def-item><def-item><term><italic>K</italic><sub><italic>p</italic></sub></term><def><p id=\"Par24\">Rate constant for intraparticle diffusion kinetics (mg/g/min<sup>0.5</sup>)</p></def></def-item><def-item><term><italic>k</italic><sub>TH</sub></term><def><p id=\"Par25\">Thomas rate constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>y</sub></term><def><p id=\"Par26\">Yan model constant (L/min/mg)</p></def></def-item><def-item><term><italic>K</italic><sub>YN</sub></term><def><p id=\"Par27\">Yoon–Nelson constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term><italic>m</italic></term><def><p id=\"Par28\">Mass of G-AC in column (g)</p></def></def-item><def-item><term><italic>n</italic></term><def><p id=\"Par29\">Adsorption intensity</p></def></def-item><def-item><term><italic>N</italic><sub>0</sub></term><def><p id=\"Par30\">Saturation concentration (mg/L)</p></def></def-item><def-item><term>pH<sub>i</sub></term><def><p id=\"Par31\">Initial pH of the solution</p></def></def-item><def-item><term><italic>Q</italic></term><def><p id=\"Par32\">Volumetric flow rate (mL/min)</p></def></def-item><def-item><term><italic>q</italic><sub>0</sub></term><def><p id=\"Par33\">Yan model maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>e</sub></term><def><p id=\"Par34\">Amount of ACVR adsorbed by adsorbent at equilibrium (mg/g)</p></def></def-item><def-item><term><italic>Q</italic><sub>m</sub></term><def><p id=\"Par35\">Maximum adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub><italic>t</italic></sub></term><def><p id=\"Par36\">Amount of ACVR adsorbed by adsorbent at time t (mg/g)</p></def></def-item><def-item><term><italic>q</italic><sub>TH</sub></term><def><p id=\"Par37\">Adsorption capacity (mg/g)</p></def></def-item><def-item><term><italic>r</italic></term><def><p id=\"Par38\">Clark constant (min<sup>−</sup><sup>1</sup>)</p></def></def-item><def-item><term>RMSE</term><def><p id=\"Par39\">Root-mean-square error</p></def></def-item><def-item><term>SD%</term><def><p id=\"Par40\">Standard deviation percent</p></def></def-item><def-item><term><italic>T</italic></term><def><p id=\"Par41\">Temperature (K)</p></def></def-item><def-item><term><italic>V</italic></term><def><p id=\"Par42\">The initial volume of ACVR solution (L)</p></def></def-item><def-item><term><italic>W</italic></term><def><p id=\"Par43\">The dry mass of adsorbent (g)</p></def></def-item><def-item><term><italic>Z</italic></term><def><p id=\"Par44\">Bed height of column (cm)</p></def></def-item></def-list></glossary><glossary id=\"AGS2\"><title>Greek letters</title><def-list><def-item><term><italic>α</italic></term><def><p id=\"Par45\">Initial sorption rate (mg/g/min)</p></def></def-item><def-item><term><italic>β</italic></term><def><p id=\"Par46\">Extent of surface coverage and activation energy for chemisorption (g/mg)</p></def></def-item><def-item><term><italic>τ</italic></term><def><p id=\"Par47\">Time taken for 50% adsorbate breakthrough (min)</p></def></def-item></def-list></glossary></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02787-3</article-id><article-id pub-id-type=\"manuscript\">2787</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02787-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: In silico investigation of the therapeutic and prophylactic potential of medicinal substances bearing guanidine moieties against COVID-19</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Esam</surname><given-names>Zohreh</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Akhavan</surname><given-names>Malihe</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Lotfi</surname><given-names>Maryam</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Pourmand</surname><given-names>Saeed</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-7155-4532</contrib-id><name><surname>Bekhradnia</surname><given-names>Ahmadreza</given-names></name><address><email>abekhradnia@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027873_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411623.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 2227 0923</institution-id><institution content-type=\"org-division\">Pharmaceutical Sciences Research Centre, Department of Medicinal Chemistry</institution><institution content-type=\"org-name\">Mazandaran University of Medical Sciences</institution></institution-wrap><addr-line content-type=\"city\">Sari</addr-line><country country=\"IR\">Iran</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-division\">The Multiscale Modelling Lab</institution><institution content-type=\"org-name\">ITQB NOVA</institution></institution-wrap><addr-line content-type=\"street\">Av. da República</addr-line><addr-line content-type=\"postcode\">2780-157</addr-line><addr-line content-type=\"city\">Oeiras</addr-line><country country=\"PT\">Portugal</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412831.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 1172 3536</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, Faculty of Chemical and Petroleum Engineering</institution><institution content-type=\"org-name\">University of Tabriz</institution></institution-wrap><addr-line content-type=\"city\">Tabriz</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027873_cor5\"><label>e</label><email>abekhradnia@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>30</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"52\">8</issue><fpage>4747</fpage><lpage>4747</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>30</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s11696-022-02528-y\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2787.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Correction to: Chemical Papers (2023) 77:1129–1148</bold><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-022-02528-y\">https://doi.org/10.1007/s11696-022-02528-y</ext-link></p><p id=\"Par2\">The corresponding author asked if one of the affiliations should be deleted.</p><p id=\"Par3\">The affiliation is given below.</p><p id=\"Par4\">103 CBB, Department of Chemistry and Biochemistry, Montana State University, Bozeman, MT, 59717, USA</p><p id=\"Par5\">The Original Article is now updated.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"retraction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02818-z</article-id><article-id pub-id-type=\"manuscript\">2818</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02818-z</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Retraction Note</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Retraction Note: Nonanuclear coordination complex based on {Co<sub>9</sub>L<sub>12</sub>}<sup>6+</sup> cores: protective effect on coronary heart disease by reducing the inflammatory cytokines releasing</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Dong</surname><given-names>Fengquan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jiang</surname><given-names>Weipeng</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Jiang</surname><given-names>Yajuan</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><name><surname>Li</surname><given-names>Xiaozheng</given-names></name><address><email>xiaozheng_li@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor4\">d</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au5\"><name><surname>Li</surname><given-names>Haiying</given-names></name><address><email>haiying_li666@126.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302818z_cor5\">e</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">Department of Cardiology</institution><institution content-type=\"org-name\">Shenzhen University General Hospital</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-name\">Shenzhen University Clinical Medical Academy</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.263488.3</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0472 9649</institution-id><institution content-type=\"org-division\">College of Life Science and Oceanography</institution><institution content-type=\"org-name\">Shenzhen University</institution></institution-wrap><addr-line content-type=\"city\">Shenzhen</addr-line><addr-line content-type=\"state\">Guangdong</addr-line><country country=\"CN\">China</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302818z_cor4\"><label>d</label><email>xiaozheng_li@126.com</email></corresp><corresp id=\"IDs1169602302818z_cor5\"><label>e</label><email>haiying_li666@126.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"53\">8</issue><fpage>4749</fpage><lpage>4749</lpage><history><date date-type=\"registration\"><day>6</day><month>4</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"retracted-article\" xlink:href=\"10.1007/s11696-020-01390-0\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>4</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2818.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link>.</p></sec></notes></front><body><p id=\"Par1\"><bold>Retraction Note to: Chemical Papers (2020) 75:1355–1362</bold></p><p><ext-link ext-link-type=\"uri\" xlink:href=\"https://doi.org/10.1007/s11696-020-01390-0\">https://doi.org/10.1007/s11696-020-01390-0</ext-link></p><p id=\"Par2\">The Editor-in-Chief has retracted this article because the ethics documentation provided by the authors was inconsistent with the information in the manuscript. The authors did not provide raw data on request from the journal. In addition, Xiaozheng Li stated that their name was added to the article without their knowledge and they did not contribute to the study. The Editor has therefore lost confidence in the conclusions of this article.</p><p id=\"Par3\">Xiaozheng Li and Haiying Li agree to this retraction. Fengquan Dong, Weipeng Jiang and Yajuan Jiang have not responded to any correspondence from the editor about this retraction.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-019-00721-6</article-id><article-id pub-id-type=\"manuscript\">721</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-019-00721-6</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Meesilp</surname><given-names>Nutthawut</given-names></name><address><email>nutthawutm@hotmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10068019007216_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Mesil</surname><given-names>Nutthisha</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.443999.a</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0504 2111</institution-id><institution content-type=\"org-division\">Department of Applied Biology, Faculty of Science and Liberal Art</institution><institution content-type=\"org-name\">Rajamangala University of Technology Isan</institution></institution-wrap><addr-line content-type=\"postcode\">30000</addr-line><addr-line content-type=\"city\">Nakhon Ratchasima</addr-line><country country=\"TH\">Thailand</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.9786.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 0856</institution-id><institution content-type=\"org-division\">Srinagarind Hospital</institution><institution content-type=\"org-name\">Khon Kaen University</institution></institution-wrap><addr-line content-type=\"postcode\">40002</addr-line><addr-line content-type=\"city\">Khon Kaen</addr-line><country country=\"TH\">Thailand</country></aff></contrib-group><author-notes><corresp id=\"IDs10068019007216_cor1\"><label>a</label><email>nutthawutm@hotmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>6</day><month>12</month><year>2019</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"13\">9</issue><fpage>1297</fpage><lpage>1297</lpage><history><date date-type=\"registration\"><day>23</day><month>11</month><year>2019</year></date><date date-type=\"online\"><day>6</day><month>12</month><year>2019</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2019</copyright-statement><copyright-year>2019</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10068-018-0448-4\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2019</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>11</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>23</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2019_Article_721.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CrossLinking\"><sec><p>The original article can be found online at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link>.</p></sec></notes></front><body><sec id=\"Sec1\"><title>Correction to: Food Sci Biotechnol (2019) 28(1):289–296 <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10068-018-0448-4\">https://doi.org/10.1007/s10068-018-0448-4</ext-link></title><p id=\"Par1\">The article “Effect of microbial sanitizers for reducing biofilm formation of <italic>Staphylococcus aureus</italic> and <italic>Pseudomonas aeruginosa</italic> on stainless steel by cultivation with UHT milk”, written by “Nutthawut Meesilp and Nutthisha Mesil”, was originally published electronically on the publisher’s Internet portal (currently SpringerLink) on 11 August 2018 with open access.</p><p id=\"Par2\">With the author(s)’ decision to step back from Open Choice, the copyright of the article changed on 6 December 2019 to © The Korean Society of Food Science and Technology 2019 and the article is forthwith distributed under the terms of copyright.</p><p id=\"Par3\">The original article has been corrected.</p></sec></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02795-3</article-id><article-id pub-id-type=\"manuscript\">2795</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02795-3</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Superior possibilities and upcoming horizons for nanoscience in COVID-19: noteworthy approach for effective diagnostics and management of SARS-CoV-2 outbreak</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Bhattacharjee</surname><given-names>Bedanta</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Ikbal</surname><given-names>Abu Md Ashif</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Farooqui</surname><given-names>Atika</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au4\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0001-5671-6591</contrib-id><name><surname>Sahu</surname><given-names>Ram Kumar</given-names></name><address><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></address><xref ref-type=\"aff\" rid=\"Aff4\">4</xref><xref ref-type=\"corresp\" rid=\"IDs11696023027953_cor4\">d</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Ruhi</surname><given-names>Sakina</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Syed</surname><given-names>Ayesha</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Miatmoko</surname><given-names>Andang</given-names></name><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Khan</surname><given-names>Danish</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au9\"><name><surname>Khan</surname><given-names>Jiyauddin</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Girijananda Chowdhury Institute of Pharmaceutical Science</institution></institution-wrap><addr-line content-type=\"postcode\">784501</addr-line><addr-line content-type=\"city\">Tezpur</addr-line><addr-line content-type=\"state\">Assam</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411460.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 4538</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Assam University (A Central University)</institution></institution-wrap><addr-line content-type=\"postcode\">788011</addr-line><addr-line content-type=\"city\">Silchar</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.413565.0</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1767 2452</institution-id><institution content-type=\"org-name\">The Deccan College of Medical Sciences</institution></institution-wrap><addr-line content-type=\"street\">Kanchan Bagh</addr-line><addr-line content-type=\"postcode\">500058</addr-line><addr-line content-type=\"city\">Hyderabad</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.412161.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0681 6439</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences</institution><institution content-type=\"org-name\">Hemvati Nandan Bahuguna Garhwal University (A Central University)</institution></institution-wrap><addr-line content-type=\"street\">Chauras Campus</addr-line><addr-line content-type=\"postcode\">249161</addr-line><addr-line content-type=\"city\">Tehri Garhwal</addr-line><addr-line content-type=\"state\">Uttarakhand</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">Department of Biochemistry</institution><institution content-type=\"org-name\">IMS, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-name\">International Medical School, Management and Science University</institution></institution-wrap><addr-line content-type=\"street\">University Drive, Off Persiaran Olahraga</addr-line><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440745.6</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0152 762X</institution-id><institution content-type=\"org-division\">Department of Pharmaceutical Sciences, Faculty of Pharmacy</institution><institution content-type=\"org-name\">Universitas Airlangga</institution></institution-wrap><addr-line content-type=\"postcode\">60115</addr-line><addr-line content-type=\"city\">Surabaya</addr-line><addr-line content-type=\"state\">East Java</addr-line><country country=\"ID\">Indonesia</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Panineeya Institute of Dental Science and Research Centre</institution><institution content-type=\"org-name\">Kalonji Narayana Rao University of Health Sciences</institution></institution-wrap><addr-line content-type=\"postcode\">506007</addr-line><addr-line content-type=\"city\">Warangal</addr-line><addr-line content-type=\"state\">Telangana</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.444504.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1772 3483</institution-id><institution content-type=\"org-division\">School of Pharmacy</institution><institution content-type=\"org-name\">Management and Science University</institution></institution-wrap><addr-line content-type=\"postcode\">40100</addr-line><addr-line content-type=\"city\">Shah Alam</addr-line><addr-line content-type=\"state\">Selangor</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs11696023027953_cor4\"><label>d</label><email>ramsahu79@gmail.com</email><email>ramkumar.sahu@hnbgu.ac.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>4</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"3\">8</issue><fpage>4107</fpage><lpage>4130</lpage><history><date date-type=\"registration\"><day>20</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>21</day><month>1</month><year>2023</year></date><date date-type=\"accepted\"><day>18</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>4</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">The outbreak of COVID-19 has caused great havoc and affected many parts of the world. It has imposed a great challenge to the medical and health fraternity with its ability to continue mutating and increasing the transmission rate. Some challenges include the availability of current knowledge of active drugs against the virus, mode of delivery of the medicaments, its diagnosis, which are relatively limited and do not suffice for further prognosis. One recently developed drug delivery system called nanoparticles is currently being utilized in combating COVID-19. This article highlights the existing methods for diagnosis of COVID-19 such as computed tomography scan, reverse transcription-polymerase chain reaction, nucleic acid sequencing, immunoassay, point-of-care test, detection from breath, nanotechnology-based bio-sensors, viral antigen detection, microfluidic device, magnetic nanosensor, magnetic resonance platform and internet-of-things biosensors. The latest detection strategy based on nanotechnology, biosensor, is said to produce satisfactory results in recognizing SARS-CoV-2 virus. It also highlights the successes in the research and development of COVID-19 treatments and vaccines that are already in use. In addition, there are a number of nanovaccines and nanomedicines currently in clinical trials that have the potential to target COVID-19.\n",
       "</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>COVID-19</kwd><kwd>Computer tomography</kwd><kwd>Polymerase chain reaction</kwd><kwd>Biosensor</kwd><kwd>SARS-CoV-2 virus</kwd><kwd>Nanovaccines</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>20</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2795.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Viruses like Middle East Respiratory Syndrome (MERS), Zika, Ebola, Influenza A (H1N1), and Severe Acute Respiratory Syndrome (SARS) have spread around the world in the last 20 years. Health policy and economic relations have been profoundly affected by these pandemics (Boopathi et al. <xref ref-type=\"bibr\" rid=\"CR21\">2021</xref>). The most recent form of the virus is the novel coronavirus, which has been recognized globally as a thoroughly transmissible disease since 2019. It is commonly known as novel COVID-19 (SARS-CoV-2) (Lu and XZ <xref ref-type=\"bibr\" rid=\"CR112\">2020</xref>; Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). The virus was firstly discovered in the Wuhan city of China. The infected patients exhibited unusual pneumonia-like symptoms that later spread globally (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>). Coronavirus has caused great concerns due to its rapid diversity and a high number of new cases reported that resulted in huge economic loss worldwide. The structure of coronavirus consists of an envelope of spikes in its outer layer. Its total height is approximately 32 kb, with a single-stranded RNA (Benzigar et al. <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>). At the time of replication of polyprotein translations, it would act on the mRNA. Human SARS-CoV-2 has a spherical morphology with a dimension of around 125 nm (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>) (Daemi et al. <xref ref-type=\"bibr\" rid=\"CR43\">2021</xref>). The coronaviridae family and the nidovirales order comprise the virus evolutionary lineage (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Su et al. <xref ref-type=\"bibr\" rid=\"CR177\">2016</xref>). Coronavirus is subdivided into four main categories, referred to as: alpha (<italic>α</italic>), beta (<italic>β</italic>), gamma (<italic>γ</italic>), and delta (<italic>δ</italic>) and all of which have been recognized to date. Some of the common coronavirus for human are HKU1, NL63, MERS-CoV, OC43, SARS-CoV, and 229E. Alpha coronavirus consists of NL63 and 229E, whereas the SARS-CoV, MERS-CoV, OC43, and HKU1 families are examples of a beta coronavirus (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>). Four crucial structural proteins are encoded by the coronaviral genome, namely spike (S) protein, nucleocapsid (N) protein, membrane (M) protein, and the envelope (E) protein, that collectively constitute a complete viral entity (Masters <xref ref-type=\"bibr\" rid=\"CR120\">2006</xref>). Every protein plays a specialized function in the construction of the viral genome. These proteins have a central position in the replication process as well. The greatest predominant antigen in the viruses is the S glycoprotein, which is called the spikes of the virus (Jaimes et al. <xref ref-type=\"bibr\" rid=\"CR75\">2020</xref>). It has the ability to interact with the receptor and then fuse with the host membranes. The fused viral protein has branches in two different ways: first, it uses the S1 protein of the primary subunit to attach to the receptor. By reserving the cell membranes of the receptor, the S2 subunit would promote infusion. After targeting the virus, it would be structurally modified by the proteolytic enzymes. The peptide would then be incorporated into the cell membrane. Finally, irreversible cyclization would stress the S2 subunit of the fusion protein, which in turn would be mediated by ACE -2 (angiotensin-converting enzyme 2), allowing the viral genome to enter the cytoplasm (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Shafiee et al. <xref ref-type=\"bibr\" rid=\"CR170\">2021</xref>; Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>).<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The SARS-CoV-2 virus impacting humans is depicted in this illustration</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig1_HTML.png\"/></fig><fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Invasion of SARS-CoV-2 via ACE-2. micelles: TPGSM</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig2_HTML.png\"/></fig></p><p id=\"Par3\">COVID-19 is considered a pandemic of the world’s most significant humanitarian catastrophes due to its uncontrollable spread worldwide, huge loss of life, and impact on the world’s economy. Huge efforts have been done to overcome the disease and prevail over the pandemic (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>; Ziegler et al. <xref ref-type=\"bibr\" rid=\"CR223\">2020</xref>; Comentale et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). The most common symptoms among infected people are fever, loss of taste and smell, fatigue, and cough. In severe cases, several other symptoms are also reported, including difficulty breathing, pneumonia, and severe darkening of the lungs. Patients who present with such signs are evaluated by chest radiographs or computed tomography (Russell et al. <xref ref-type=\"bibr\" rid=\"CR164\">2020</xref>; Yang et al. <xref ref-type=\"bibr\" rid=\"CR211\">2020</xref>; Weiss et al. <xref ref-type=\"bibr\" rid=\"CR200\">2020</xref>). Officially, the coronavirus was termed as SARS-CoV-2 or 2019- nCoV (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>; Munster et al. <xref ref-type=\"bibr\" rid=\"CR128\">2020</xref>; Liu et al. <xref ref-type=\"bibr\" rid=\"CR109\">2020</xref>; Habibi et al. <xref ref-type=\"bibr\" rid=\"CR59\">2020</xref>; Heymann and Shindo <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR198\">2020</xref>).</p><p id=\"Par4\">Nanotechnology is pertinent in managing and treating a wide range of disorders. Applications of nanomedicine include immune engineering, drug delivery, nanosensors, cancer nanovaccines, and platform technologies (Chakravarty and Vora <xref ref-type=\"bibr\" rid=\"CR28\">2021</xref>). Since the existing antiviral agents have truncated bioavailability and a narrow spectrum, nanomaterials are crucial for antiviral treatment. ENMs (engineered nanomaterials) have easy tunable physicochemical, low cost, high efficiency, and efficacy of drug delivery with lesser side effects (Chen et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>). The cell–virus interactions can be adjusted, particularly their interaction with cell receptors or virus particles, by adjusting the shape, size, and surface of ENMs (Ye et al. <xref ref-type=\"bibr\" rid=\"CR212\">2015</xref>). Nanotechnologies can be used for rapid detection of COVID-19 to further improve the efficiency of diagnostic tests (Rabiee et al. <xref ref-type=\"bibr\" rid=\"CR153\">2020</xref>). Organisms with a higher incidence of susceptibility can also be diagnosed using nanomedicines, which act as a prophylactic and alternative means of diagnosing infectious diseases (Vahedifard and Chakravarthy <xref ref-type=\"bibr\" rid=\"CR190\">2021</xref>). Magnetic nanoparticles and quantum dots are the nanoparticles that are commonly employed to detect coronavirus. The protein corona sensor array technology, which would aid in identifying protein/biomolecule sequences at an early stage, is at the forefront of catastrophic COVID-19 (Misra et al. <xref ref-type=\"bibr\" rid=\"CR123\">2021</xref>). The design and implementation of nanotechnology facilitate the production of new nanovaccines with a higher efficacy. One such efficient delivery strategy for treating SARS-CoV-2 is a nanoloaded formulation that is currently being explored. In addition, novel nanovaccines and nanotherapeutics are being developed to address SARS-CoV-2 infection (Jan et al. <xref ref-type=\"bibr\" rid=\"CR76\">2022</xref>). Therefore, researchers are making tremendous efforts to discover novel nano-based technologies and develop relevant nanovaccines and create effective therapies for SARS-CoV-2 infections. This article focuses on the unique and innovative nanotechnology strategies, detection, and prevention against COVID-19 and the potential for future research.</p></sec><sec id=\"Sec2\"><title>Diagnosis of SARS-CoV-2 infestation</title><sec id=\"Sec3\"><title>Computed tomography scan</title><p id=\"Par5\">A computed tomography (CT) scan is used to examine the internal organs. Chest radiographs and 3D cross-sectional images are utilized to identify the symptoms of COVID-19 (Shah et al. <xref ref-type=\"bibr\" rid=\"CR171\">2021</xref>). Patchy lung lesions of small size are seen in the early stages. One of the reports stated that CT scans are more likely to be tactful in the first stage of infection than RT-PCR (Law et al. <xref ref-type=\"bibr\" rid=\"CR100\">2015</xref>). CT scan helps to better understand the condition of the lung and the severity of the disease. It helps the physician to treat the patient appropriately depending on the severity (Bernheim et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). However, the usage of CT scans for COVID-19 has some challenges, including low specificity, which is only about 25%. As with asymptomatic patients, there are also opportunities for uncertainty with CT scans (Alafif et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021</xref>).</p><p id=\"Par6\">Furthermore, the diagnostic images of CT scans could overlap with other diseases like pneumonia. Repeated radiation exposure also would inflict long-term negative effects (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>). Additionally, it is an expensive tool to utilize.</p><p id=\"Par7\">On chest X-rays, it is common to see recurring mottling, subsegmental, or segmental morning opacities in both lungs, and atypical findings are becoming more common (Kaur et al. <xref ref-type=\"bibr\" rid=\"CR83\">2020</xref>). However, CT radiation is associated with minimal carcinogenic risk. It is currently unclear how important CT is for the management of patients with COVID-19. Presently, no antiviral therapy is available for 2019-nCoV, which is not the case in pulmonary TB patients. Drug treatment for tuberculosis can be started without a positive pathogenic Mycobacterium tuberculosis smear or tuberculosis culture if a clinical diagnosis is made that is supported by imaging. In cases of COVID-19, CT may show symptoms similar to those of viral pneumonia, supporting the clinical diagnosis (Zhao et al. <xref ref-type=\"bibr\" rid=\"CR221\">2020</xref>). In contrast, a change from a suspicious to a clinically diagnosed COVID-19 case classification has no impact on clinical therapy (Wu et al. <xref ref-type=\"bibr\" rid=\"CR203\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR204\">b</xref>).</p></sec><sec id=\"Sec4\"><title>Reverse transcription-polymerase chain reaction (RT-PCR)</title><p id=\"Par8\">RT-PCR would establish the criteria for COVID-19 diagnosis. Although COVID-19 is based on the extent of chronic respiratory complication from viruses, the practical detection approach appears to be less reliable. The SARS-CoV-2 virus is a single-stranded RNA virus with a positive direction of infection. Therefore, the most accurate approach for identification is RT-PCR. This approach requires the conversion of RNA to its supporting DNA, followed by amplification for detection (Harvey and O’Regan <xref ref-type=\"bibr\" rid=\"CR64\">2020</xref>). The E, S, N protein, and the RNA-dependent RNA polymerase gene (RdRp) portions of the entire genome, and ORF1ab are the most common targets for RT-PCR-based detection. These have shown good sensitivity, with the exception of RdRp, as it has lower sensitivity due to reverse primer-template mismatches than the rest (Huang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2020</xref>). The product of PCR amplification analysis is based upon CT (cycle threshold) value, <italic>i.e., </italic>the threshold value found in the duration of the amplification of the viral genome and control, <italic>i.e., </italic>negative control, standard control, and positive control. The positivity criteria of COVID-19 are analyzed based on the measurement of CT score. The CT score is a number that shows how long it took for the total number of PCR sessions to exceed the baseline fluorescence emission value. In the case of infection, the CT score cut-off is between 35 and 40 with a higher CT score indicating a lower viral load in the specimen. Thus, patients with a CT score of less than 35 would be classified as COVID-19 favorable, whereas patients with a CT score greater than 40 would be classified as clinically negative (Lai et al. <xref ref-type=\"bibr\" rid=\"CR97\">2020</xref>; Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>).</p><p id=\"Par9\">The virus accumulates in the nasopharynx, so a swab is taken from these body sites. Various biochemical reagents are used to remove impurities such as lipids and proteins to isolate only the RNA from the sample. The genetic information of the individual and, if applicable, the RNA of the virus are combined to form this RNA. A special enzyme is needed to convert the RNA into DNA. Then, researchers insert micro-DNA fragments that complement specific zones of replicated virulent DNA. When the virus is identified in the sample, these fragments bind to the target sites of the virulent DNA. During the amplification process, some parts of the genome are added to design the DNA sequence. On the other hand, DNA fragments are produced and attached to the sequences with labels to identify the virus. The device RT-PCR is then employed to test the sample (Udugama et al. <xref ref-type=\"bibr\" rid=\"CR188\">2020</xref>). The apparatus heats and cools the samples simultaneously, activating biochemical reactions that result in additional duplicated versions of the virulent DNA target regions (Craw and Balachandran <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). The process is repeated infinitely to copy the viral DNA target regions. A conventional real-time system RT-PCR runs for 35 sessions, which means that each viral strand in the material will have produced approximately 35 billion additional versions of the viral DNA at the end of the process. As additional versions of the virulent DNA fragments are produced, the tags combine with the DNA sequences. The result is the release of a fluorescent dye that is tracked by the computer device and displayed on the monitor in real time. The computer device logs the amount of fluorescence in the sample at the end of each session. The prevalence of the virus is determined when a predetermined fluorescence threshold is exceeded (Rao et al. <xref ref-type=\"bibr\" rid=\"CR158\">2022</xref>). To assess the severity of the disease, experts also monitor the number of sessions required to do so: The fewer sessions required, the more severe the viral contamination (Mohammadniaei et al. <xref ref-type=\"bibr\" rid=\"CR125\">2021</xref>). The real-time PCR method is precise and accurate and can provide a reliable diagnosis within 3 h, whereas laboratory methods typically require 6–8 h. Real-time PCR (RT-PCR) is much faster and has less potential for inaccuracy than other known viral extraction methods because the entire process can be performed in a closed container (Wu et al. <xref ref-type=\"bibr\" rid=\"CR202\">2013</xref>).</p></sec><sec id=\"Sec5\"><title>Nucleic acid sequencing</title><p id=\"Par10\">Detection of SARS-CoV-2 can be sequence-specific based on nucleic acid sequencing. It is considered one of the most precise scientific methods for detecting COVID-19 as it can examine the whole genomic makeup of the virus (Lu et al. <xref ref-type=\"bibr\" rid=\"CR113\">2020</xref>). Sanger sequencing is a type of sequencing in which a patient's sample is recovered to recover viral RNA and spike-in control RNA that is different from viral RNA. Most of the diagnoses of COVID-19 are based on next-generation sequencing such as oxford nanopore sequencing and illumine sequencing. Unlike Sanger sequencing, the next-generation sequencing would concurrently target billions of target sequences (Quick et al. <xref ref-type=\"bibr\" rid=\"CR152\">2017</xref>). On the other hand, the amount of virus remaining in the specimen is measured and normalized based on the spiked RNA (Singh et al. <xref ref-type=\"bibr\" rid=\"CR176\">2021</xref>; Yu et al. <xref ref-type=\"bibr\" rid=\"CR214\">2021</xref>).</p><p id=\"Par11\">Assays based on nucleic acids usually take 12 to 24 h or longer and are thought to be the best way to identify microorganisms. On the other hand, nucleic acid-based assays offer a much shorter reaction time and excellent sensitivity and specificity, bringing them to the forefront of laboratory and, in some circumstances, field analysis. In the laboratory, nucleic acid sequencing has been effectively used to detect both nonpathogenic and pathogenic microorganisms. Ribosomal RNA techniques have been used to conduct similar studies of microbial diversity (Chandler-Brown et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>).</p><p id=\"Par12\">Sequence-based ID with nucleic acid testing is a key part of any defense against biological agent attacks. It helps find out whether a biological agent attack has happened and, if so, what kinds of biological agents were used. Even if the test response time is slow—on the order of 15–30 min—this confirmation function is critical (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p><p id=\"Par13\">The following is the sequence of processes that must occur within this response time: (1) collect the sample: prepare the sample for an assay. This includes the removal of assay inhibitors and lysis of target cells. This can be a serious problem with contaminated samples or when analyzing spore nucleic acids. (2) Performance of the actual assay: analyzing and reporting the assay results (Lim and Wang <xref ref-type=\"bibr\" rid=\"CR107\">2016</xref>).</p></sec><sec id=\"Sec6\"><title>Immunoassay</title><p id=\"Par14\">COVID-19 can be diagnosed indirectly by looking at the patients’ immunological response. Recent research has been conducted to determine serologic surveillance of COVID-19 by collecting various individual fluids (Rizzo and Buck <xref ref-type=\"bibr\" rid=\"CR162\">2012</xref>). The target for serological diagnosis is mainly blood samples instead of the nasopharyngeal swab. Detection of an immune response is the most important requirement for verifying transmission of the virus. Immunoassay is a bioanalytical method based on the interaction between antibody and antigen that allows rapid detection of COVID-19. Two prime serological detections based on immunoassay are enzyme-linked immunosorbent assay (ELISA) and lateral flow assay (LFA). Targeting specific antibodies and antigens is done for the diagnosis of COVID-19. According to studies, a minimum of 5 days is required for sufficient viral load. For antibodies, however, it usually takes at least 7 days, and the concentration decreases after the infection has progressed for 7 days (Alcoba-Florez et al. <xref ref-type=\"bibr\" rid=\"CR8\">2020</xref>). Thus, if the viral load is minimal, this will lead to misleading negative results.</p></sec><sec id=\"Sec7\"><title>Point-of-care test</title><p id=\"Par15\">Point-of-care (POC) testing is essential for disease management during a pandemic. For this reason, it is important to understand the causative agent of a virus before developing tools for diagnosis. It allows for early diagnosis by detecting the analyte, which could optimize the effectiveness of the therapy (Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR84\">2020</xref>). The tools developed for the POC screening of COVID-19 are based on the detection of immunological components such as antibodies and antigen or viral RNA (Ma et al. <xref ref-type=\"bibr\" rid=\"CR115\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref> highlights the existing approaches for identifying SARS-CoV-2.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>A list of the existing strategies for SARS-CoV-2 confirmation</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>SARS-CoV-2 confirmation approach</p></th><th align=\"left\"><p>Methodology</p></th><th align=\"left\"><p>Salient characteristics</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Enzymatic sensor system based on regularly interspaced short palindromic repeats for assay-specific great specificity</p></td><td align=\"left\"><p>When SARS-COV-2-specific guide RNA is activated, reporter RNA sequences are removed</p></td><td align=\"left\"><p>Execution time 1 h; inexpensive; simple instrumentation</p></td><td align=\"left\"><p>To et al. (<xref ref-type=\"bibr\" rid=\"CR185\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Enzyme-Linked immunosorbent assay</p></td><td align=\"left\"><p>Antibodies against viral proteins coated on microwell plates bound; Secondary antibody produces Fluorimetric/ Colorimetric signal</p></td><td align=\"left\"><p>High-throughput screening; quick; point of care test</p></td><td align=\"left\"><p>Zhang et al. (<xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>), Chen et al. (<xref ref-type=\"bibr\" rid=\"CR34\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR35\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Transcription-mediated amplification</p></td><td align=\"left\"><p>Amplification of particular RNA/DNA segments</p></td><td align=\"left\"><p>Within 15–30 min, there is a ten-billion-fold rise; generates RNA amplicon; rapid kinetics</p></td><td align=\"left\"><p>Phan et al. (<xref ref-type=\"bibr\" rid=\"CR146\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Metagenomic sequencing</p></td><td align=\"left\"><p>Amplification using a single primer that is not sequence-dependent</p></td><td align=\"left\"><p>The SARS-COV-2 mutation rate is analyzed</p></td><td align=\"left\"><p>Carter et al. (<xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Chest Computer Tomography Scans</p></td><td align=\"left\"><p>X-rays of the patient's chest from multiple angles</p></td><td align=\"left\"><p>The imaging characteristics of viral pneumonia are similar. Indeterminate for SARS-CoV-2; non-invasive</p></td><td align=\"left\"><p>Hardick et al. (<xref ref-type=\"bibr\" rid=\"CR63\">2018</xref>)</p></td></tr><tr><td align=\"left\"><p>Lateral Flow Immunoassay</p></td><td align=\"left\"><p>A sample is passed through an immobilized viral antigen. Color develops in the presence of anti-CoV antibodies, as do marker antibodies</p></td><td align=\"left\"><p>Does not require a professional staff; expensive; quick (10–30 min)</p></td><td align=\"left\"><p>Arslan et al. (<xref ref-type=\"bibr\" rid=\"CR10\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription loop-mediated isothermal amplification</p></td><td align=\"left\"><p>At the same temperature, amplification using four different primer systems</p></td><td align=\"left\"><p>In place of a heating cycler, a heating bath is provided; expeditious (&lt; 13 h); expensive</p></td><td align=\"left\"><p>Calucho et al. (<xref ref-type=\"bibr\" rid=\"CR26\">2020</xref>), Jiang et al. (<xref ref-type=\"bibr\" rid=\"CR78\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Reverse transcription-polymerase chain reaction</p></td><td align=\"left\"><p>Virus genomic RNA amplification</p></td><td align=\"left\"><p>Gold standard; sample requirement is less</p></td><td align=\"left\"><p>Park et al. (<xref ref-type=\"bibr\" rid=\"CR140\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par16\">POC tests are procedures performed with a test strip or kit at home or at the healthcare facility where the person is being treated. The biosensor is the most important component of POC screening because it is required to perform molecular analysis to identify the virus. POC screening has the following advantages: (i) fewer storage and experimental facilities are needed; (ii) it is adaptable to a variety of medical problems; (iii) it is a significant test; and (iv) multiple sites can be used for evaluation (Asdaq et al. <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par17\">Each POC test is approved for use with a specific type of specimen and can only be used with those specimens. Appropriate specimen procurement and handling are required for all COVID-19 assays, especially those performed in POC clinics. An erroneous or inaccurate test result may result from improper specimen submission or collection. Employees who collect specimens within 6 feet of persons must wear the required personal protective equipment (PPE). It consists of gloves, safety glasses, N95 protection, and a lab coat or robe. All personnel collecting specimens not actively involved in specimen collection should follow standard safety guidelines and not be more than one meter away from the patient (Dincer et al. <xref ref-type=\"bibr\" rid=\"CR47\">2017</xref>).</p></sec><sec id=\"Sec8\"><title>Detection from breath</title><p id=\"Par18\">For the objective of monitoring COVID-19, screening of expelled breath may be a less intrusive form of analysis (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Broza et al. <xref ref-type=\"bibr\" rid=\"CR25\">2018</xref>). However, screening SARS-CoV-2 from exhaled air has proven to be quite complicated. These findings show that COVID-19 individuals breathed billions of RNA molecules of severe acute respiratory syndrome coronavirus every extra few minutes (Broza et al. <xref ref-type=\"bibr\" rid=\"CR24\">2015</xref>). According to scientific interpretation, exhaled air had a greater affirmative incidence (26.7%) than surface samples (5.4%) and air (3.8%). In addition, exhaled breath condensate (EBC), a unique technique for quickly identifying viruses from exhaled breath, was required to collect the sample for a longer period of time..</p><p id=\"Par19\">Furthermore, recent studies have shown that non-volatile components such as viruses, bacteria, DNA, and RNA, can be easily detected and observed by extracting and analyzing the aqueous fraction of expelled breath (Expelled Breath Aerosol (EBA) or EBC) (Ma et al. <xref ref-type=\"bibr\" rid=\"CR116\">2021</xref>). Depending on these two factors, an EBC system can effectively detect different droplets: (1) the percentage of the virus still alive after extraction and (2) the ratio between the percentage of extracted particles and the total percentage of particles in the air. For example, Haick and colleagues conducted a scientific investigation in March 2020 in Wuhan city of China, featuring screening with a breath scanner that follows the principle of chemoreceptive devices formed of gold nanoparticles combined with artificial intelligence algorithms (Corman et al. <xref ref-type=\"bibr\" rid=\"CR41\">2020</xref>; Lamote et al. <xref ref-type=\"bibr\" rid=\"CR99\">2020</xref>). In another experiment, researchers investigated preliminary markers of enhanced reactive oxygen species formation in swab samples. In this approach, swab samples were exposed to a biosensor nanostructured with carbon nanofibers, which provided 97% accurate confirmatory readings in about 30 s (Shan et al. <xref ref-type=\"bibr\" rid=\"CR172\">2020</xref>).</p></sec><sec id=\"Sec9\"><title>Nanotechnology-based biosensors for COVID-19</title><p id=\"Par20\">Biosensors are devices containing a physical transducer and a biorecognition element that can be used to diagnose the analytes present in solutions, liquids, and bodily fluids. A physical sensor is used to produce a response after an analyte coincides with a biological factor and transforms it into a measurable quantity. Biorecognition elements are glycoproteins (antibodies) or enzymes, nucleic acids such as RNA, DNA, biological receptors, organic receptors, tissues, and whole cells (Miripour et al. <xref ref-type=\"bibr\" rid=\"CR122\">2020</xref>). As per the specifications, enormous numbers of biosensors can be fabricated. Nanomaterials are beneficial as they provide the surface chemistry that would aid in molecular bioconjugation, powerful expanding effect on signals, and large surface energy. Metallic nanoparticles such as silver and gold nanoparticles, nanogels, carbon nanomaterials, <italic>i.e.,</italic> graphene and nanotubes, microgels, and phonic crystals, are utilized in the application of biosensors. Meanwhile, a piezoelectric and electrochemical method based on nanocarbon particles has been developed for the detection of bacteria (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>).</p><p id=\"Par21\">A biosensor detects solutes in bodily fluids, liquids, and by combining a biological sensing device with a mechanical detector. When a solute interferes with a biological component, an output signal is generated, in which the physical detector then converts into a scientifically and quantitatively detectable component. Proteins such as antibodies or enzymes, nucleic acids such as RNA or DNA, physiological receptors, whole tissues, and cells are among the biological detector elements, while electrochemical, photonic, and piezoelectric devices are often used as physical detectors (Bandodkar and Wang <xref ref-type=\"bibr\" rid=\"CR15\">2014</xref>). Biosensor elements can be manufactured in large quantities to meet different requirements. Nanomaterials can be used as detectors due to their excellent resistance in a wide range of environments and bio-compatibility with biological systems (Sharma et al. <xref ref-type=\"bibr\" rid=\"CR174\">2020</xref>). Additionally, nanomaterials have high interfacial potential, robust signal amplification capability, and surface chemistry that can help bioconjugate biomolecules. The properties of photonic crystals, microgels, and magnetic nanostructures including gold nanostructures and silver, carbon-based nanoparticles (nanotubes and graphene), and nanocomposites have contributed to their use in biosensor technologies. The use of nanocarbon-based particles has facilitated microfluidic systems. These have already proven to be platforms for bacterial detection, using a range of strategies such as electrochemistry and piezoelectricity (Olvera and Monaghan <xref ref-type=\"bibr\" rid=\"CR138\">2021</xref>).</p></sec><sec id=\"Sec10\"><title>Development in nanotechnology-based COVID-19 diagnosis</title><p id=\"Par22\">When an infection occurs, immune globulin M (IgM) and immunoglobulin G (IgG) are produced. Initially, IgM antibodies appear, which are reduced to a minimum in general infection. Thereafter, IgG antibodies would develop and displace the IgM antibodies in the body, which remain in high concentration for a considerable time until they are identified after 12 weeks. In SARS-CoV-2 transmission, there is no latency between IgM and IgG confirmation, as both occur within 5–7 days of the onset of manifestations (Tarim et al. <xref ref-type=\"bibr\" rid=\"CR182\">2021</xref>). Several laboratory-based immunoassay methods, such as chemiluminescence immunoassay, can find antibodies in blood serum to the SARS-CoV-2 viral protein. The most commonly utilized method for detection is enzyme-linked immunosorbent assay. However, to identify contamination and conduct an on-site investigation, rapid and portable immunoassay detection is required. One promising, quick and portable platform for the point of care is lateral flow immunoassay, which uses colloidal gold nanoparticles (Deeks et al. <xref ref-type=\"bibr\" rid=\"CR46\">2020</xref>). Gold nanostructures are coupled to the SARS-CoV-2 protein and immobilized in the intracellular membrane in a lateral flow immunoassay. The testing for SARS-CoV-2 virus detection would take roughly 15 to 20 min. Due to this, cross-reactivity with other antibodies may result in false-positive outcomes (Han et al. <xref ref-type=\"bibr\" rid=\"CR62\">2019</xref>).</p><p id=\"Par23\">The rate at which antibodies to SARS-CoV-2 are produced is the most important immunologic factor determining susceptibility to infection. RT-PCR can be replaced with immunologic screening to validate SARS-CoV-2. The rate of unequivocal virus detection is significantly increased when immunologic testing and real-time PCR are coupled (Lustig et al. <xref ref-type=\"bibr\" rid=\"CR114\">2021</xref>). In most individuals, IgM levels rise in the first week resulting from exposed to SARS-CoV-2, continue to rise until levels reach a maximum after two weeks, and then begin to decline until levels are near reference values. After one week, IgG is visible and it remains in significant concentration for a longer period of time.</p><p id=\"Par24\">IgG, on the other hand, can be seen after only one week. Its levels stay high for a longer time, probably up to 48 days, and it may help prevent a second infection. Immunoglobulin A (IgA) responses begin to manifest about 4 and 10 days after infection. IgM, IgA, and IgG expression in blood plasma is a diagnostic factor. Different targets can describe the variability of antibodies to SARS-CoV-2. After exposure to infection, antibody titers may decrease for up to 7 days. Specific antibodies to SARS-CoV-2 were recently detected in saliva. Experts investigated whether there are differences in antibody concentrations between serum and saliva using multiplex antibody immunoassays for SARS-CoV-2. The antibodies indicate parallel compartmental humoral immune responses in saliva that correspond to those in serum (Pan et al. <xref ref-type=\"bibr\" rid=\"CR139\">2020</xref>).</p></sec><sec id=\"Sec11\"><title>Viral antigen detection</title><p id=\"Par25\">Prior biosensing strategy would utilize complementary DNA (cDNA) or antibodies to obtain viral RNA or antigen. The existence of SARS-CoV-2 infection is identified by utilizing graphene field-effect transistors (GFET) as a biosensor. It helps in determining the seriousness of the infection (Rocchitta et al. <xref ref-type=\"bibr\" rid=\"CR163\">2016</xref>). The graphite film serves as the reading region of the FET-based biosensor, which is then transferred to the Si/SiO<sub>2</sub> substrate to successfully modify with the antibody of COVID-19. It is immobilized well on the surface of the graphene sheet by drop-casting. This instrument can detect COVID-19 antigen concentrations as low as 1 fg/ml in a phosphate buffer, which is substantially lower than the PCR/ELISA approach (Nguyen-Contant et al. <xref ref-type=\"bibr\" rid=\"CR134\">2020</xref>). The medium used is universal transport medium (UTM), with the assay used for biosensors for nasopharyngeal swab suspension.</p><p id=\"Par26\">A rapid assay method was developed to confirm the expression of the viral genome released by SARS-CoV-2 in samples from an infected person’s respiratory tract. In this experiment, the protein in the sample adheres to antibodies coupled with a sheet of paper in a glass container. Within half an hour, this reaction produces a detectable signal. The test could indicate intense or initial infection because the proteins found are expressed only when the pathogen is continuously replicating (Drobysh et al. <xref ref-type=\"bibr\" rid=\"CR49\">2022</xref>). In addition, Abbott has introduced a more powerful version of the COVID-19 instant diagnostic test, which screens for antibodies in infected human blood. Abbott's diagnostic devices can detect antibodies to SARS-CoV-2 and are available on the ARCHITECT i1000SR and i2000SR scientific instruments, which can perform 100–200 tests per hour. Antibodies to SARS-CoV-2 are produced after one week of infection. Age, nutritional status, the severity of the disease, concomitant diseases, and medications can influence the intensity of the antibody response (Seo et al. <xref ref-type=\"bibr\" rid=\"CR169\">2020</xref>).</p></sec><sec id=\"Sec12\"><title>Microfluidic device</title><p id=\"Par27\">Another possibility for experiments with real evidence is microfluidic devices. These microfluidic crisps are imprinted with micrometer-sized circuits and reactivity units made of polydimethyl sulfoxide and paper and offer advantages such as reduced sample size, compact dimensions, and rapid detection time. These microfluidic crisps work by blending and separating fluid materials employing electrokinetics and capillary pressure, which is their core principle (Ramachandran et al. <xref ref-type=\"bibr\" rid=\"CR156\">2021</xref>).</p><p id=\"Par28\">Mass screening of people is recognized as a crucial step in limiting the virus proliferation. Some countries in America and Africa reported fewer cases due to the shortage of tests done. Nowadays, quantitative reverse transcriptase polymerase chain reaction (qRTPCR) is applied and endorsed by WHO for case confirmation. qRTPCR (quantitative reverse transcriptase polymerase chain reaction) primer, thermocyclers, specific reagent, and qualified personnel are the most important requirements (Sen et al. <xref ref-type=\"bibr\" rid=\"CR168\">2022</xref>). Microfluidics can be used as an alternative to the time-consuming assay. The emergence of microfluid devices is enabled due to the microelectronics and micro-electromechanical systems as they can manipulate a small number of fluids.</p><p id=\"Par29\">Moreover, they can also retrieve information from the small sample volume. Microfluidic devices are used at the early stage of diagnosis as recent researchers have found that they can detect viruses (Kalita et al. <xref ref-type=\"bibr\" rid=\"CR80\">2021</xref>). The genetic code that forms virus consists of an envelope of proteins and a lipid ligand. Proteins are detected by Western blot, PCR, and amplified genetic material. Many approaches based on benchtop assays are transferred to the microfluidic device to minimize the amount of reagents needed and to speed up the reaction. The microfluidic device would elaborate a structure that can detect the biological samples. The detector that uses the amplification of genetic code requires a specific DNA probe design called primer to find a target sequence in the virus's genetic material (Maggi et al. <xref ref-type=\"bibr\" rid=\"CR117\">2019</xref>).</p></sec><sec id=\"Sec13\"><title>Magnetic nanosensor</title><p id=\"Par30\">Gold nanoislands (AuNIs) were utilized to build plasmonic photothermal (PT) biosensors that target the ORF1ab, RdRp, and E genes to pinpoint SARS-CoV-2 RNA sequences. Nanosensors based on magnetic nanoparticles (MNPs) generate heat as a result of their optical properties when aggregate vibrations in energy concentration take place at the interface of nanomaterial via conjugating to magnetic radiation. The increased frequency of collisions with lattice atoms caused by the enhanced motion of conduction electrons leads to the generation of PT-induced heat (Hofmann et al. <xref ref-type=\"bibr\" rid=\"CR67\">2008</xref>; Moabelo et al. <xref ref-type=\"bibr\" rid=\"CR124\">2021</xref>; Esbin et al. <xref ref-type=\"bibr\" rid=\"CR52\">2020</xref>).</p><p id=\"Par31\">Nanosensors based on magnetic nanoparticles (MNPs) are advantageous for viral pathogen identification (Islam and Ahsan <xref ref-type=\"bibr\" rid=\"CR72\">2020</xref>; Barnett et al. <xref ref-type=\"bibr\" rid=\"CR16\">2019</xref>). Tian et al. claim that nanosensors based on MNPs can be used for screening nucleic acids by combining uniform circle-to-circle amplification (UC2CA) and iron oxide NPs (IONPs). In this approach, the effect of an ambient magnetism allows both single and coupled IONPs to exhibit different optical properties (dispersion or absorption). When the recognition markers of the IONPs are paired with UC2CA, the coupled IONPs that produce the end amplicons (single-stranded DNA) are formed. Consequently, the configurations of the IONPs can be used to study the opto-magnetic parameters. This method has been shown to be accurate in distinguishing the genetic variants of SARS-CoV and SARS-CoV-2. In addition, it has been used to detect RdRp motifs for SARS-CoV-2 (Tian et al. <xref ref-type=\"bibr\" rid=\"CR183\">2020</xref>).</p></sec></sec><sec id=\"Sec14\"><title>Nanomedicine</title><p id=\"Par32\">The discipline of medicine known as \"nanomedicine\" exploits nanotechnology to treat and prevent disease. It is utilized in various applications of living organisms, including diagnostics, delivery, and actuation. It is described as using manmade nanodevices and nanostructures to monitor, create, and regulate human biological systems at the molecular level (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR219\">2022</xref>). The unique qualities of nanomedicines may hold the key to solving many of today's problems in cancer, cardiovascular and neurological illnesses, and sickness. The nanomedicine formulation imparts various beneficial properties such as lower side effects, molecular targeting by nanoengineered devices, better absorption with higher intestinal permeability, improved bioavailability, and prolonged and targeted medication delivery. The main benefit of nanomedicines in several current applications is improving drug delivery for therapy. The main mechanism of drug delivery based on nanomedicine called increased bioavailability, and retention is the example of selective medication tailoring (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>). It helps to increase absorbability and can be removed from the body waste before treatment, which can be effective. Nanomedicine is vital in improved efficacy, bioavailability, and targeting ability and is safer than conventional medicines. Nanomedicine influences all medicine sectors and is regarded as a significant tool for vaccinations, medical imaging, innovative diagnostics, nanotherapeutics, and the development of biomaterials for restorative medicine (Ledet and Mandal <xref ref-type=\"bibr\" rid=\"CR102\">2012</xref>). Surfactants (liquid crystals, nanoemulsions, and microemulsions), polymers (polymeric nanoparticles), proteins (protein nanoparticles), and lipids (liposomes, lipid–solid nanoparticles, and nanostructured lipid carriers) have all been employed in nanomedicine. The strength of interactions between biological/tissue molecules and nanoparticles is the foundation of their use in various medicinal applications (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Soft nanomaterials are primarily used to enhance the pharmacokinetic, pharmacodynamics, and biopharmaceutical aspects of drug loading. The performance and tolerability of the therapy may be impacted by the use of nanoparticles to enhance selective therapeutic localization (active or passive) and optimized the release rate of the drug. Due to their antimicrobial qualities, metallic, and polymer-based nanoparticles were also applied in nanomedicine (antiviral, antifungal, antibacterial, and antiparasitic) (Patra et al. <xref ref-type=\"bibr\" rid=\"CR143\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR144\">b</xref>).</p></sec><sec id=\"Sec15\"><title>Nanomedicine-based drug delivery system</title><p id=\"Par33\">Numerous conditions, including diabetes, cancer, cerebrovascular disease, and neurodegenerative disorder, can be treated with drug delivery systems based on nanotechnology. Some nanomedicines are available in the market through parenteral or oral administration. Novel nanomedicines can be prepared through several preclinical and clinical trials, which can be administered through different routes such as nasal, ocular, vaginal, dermal and pulmonary. Several developed formulations based on nanomaterials are discussed below.</p><sec id=\"Sec16\"><title>Liposomes</title><p id=\"Par34\">Alec Bangham developed liposomes in the year 1960. Cosmetics and pharmaceutical industries have utilized liposomes to transport drugs and other diverse molecules. Liposomes are considered a fixed formulation method to improve the drug delivery system. Liposomes are spherical-shaped vesicles, usually in the 50–450 nm, composed of steroids and phospholipids. They are regarded as a good vehicle for drug delivery due to their ability to incorporate drugs and possess an analogous membrane structure (Kupferschmidt and Cohen <xref ref-type=\"bibr\" rid=\"CR96\">2020</xref>). Liposomes would improve biodistribution and build stable compounds that allow them to utilize hydrophobic and hydrophilic drugs. There are four types of liposomes. The first is a lipid bilayer in which aqueous hydrophilic and hydrophobic substances are surrounded by cationic, anionic, or neutral phospholipids and cholesterol. The second variety is PEGylated liposomes, which consist of PEG (polyethylene glycol) inserted onto their surface to achieve steric equilibrium. The ligand-targeted type is the third kind in which ligands such as carbohydrates, antibodies, and peptides are attached to the surface of liposomes. The fourth form is the theranostic, a combination of the first three types addressed prior that consists of a nanoparticle with imaging, targeting, and therapeutic elements (Bozzuto and Molinari <xref ref-type=\"bibr\" rid=\"CR23\">2015</xref>).</p><p id=\"Par35\">Li et al<italic>.</italic> prepared and characterized a liposomal remdesivir inhalation solution for targeted pulmonary delivery for COVID-19 therapy. The results indicate that liposomal remdesivir shows rapid conversion of remdesivir to active nucleoside triphosphate and its Tmax is 1 h. This observation might be due to the fact that liposomes have high drug loading and are biocompatible with cell membranes. This would lead to faster uptake of remdesivir by cells and in larger amounts (Li et al. <xref ref-type=\"bibr\" rid=\"CR106\">2021</xref>). The central composite design was used to optimize the ritonavir proliposomes, and the results of the physicochemical properties were appropriate. The rats treated with the liposomal formulation exhibited a higher amount of drug in the lymphoid tissues compared with the pure drug. The results suggest that the targeting potential of biotin-coated proliposomes of ritonavir is present (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>). Mice infected with either dead or live SARS-CoV-2 developed acute respiratory distress syndrome (ARDS). Treatment with tripterine liposomes significantly reduced the severity of pulmonary pathological changes and viral load. It was observed that tripterine liposomes reduced SARS-CoV-2 replication and hyperinflammation in mice and cells infected with the virus. Therefore, it is a potential option for a drug that could be used to treat ARDS caused by SARS-CoV-2 (Que et al. <xref ref-type=\"bibr\" rid=\"CR151\">2022</xref>).</p></sec><sec id=\"Sec17\"><title>Polymeric micelle</title><p id=\"Par36\">Micellar nanostructures known as polymeric micelles are composed of an amphiphilic block copolymer that has the ability to self-assemble into a core with a shell when introduced into an aquatic system. For example, docetaxel, camptothecin, and paclitaxel are hydrophobic drugs that can be loaded into the hydrophobic cavity. The hydrophilic coating would stabilize the core and make the system soluble in water when used. To avoid rapid excretion through the kidneys, polymeric micelles have a diameter of about 100 nm and have only a small volume of distribution. Therefore, they can be collected in tumor tissue due to their increased permeability and retention effect.</p><p id=\"Par37\">On the other hand, the hydrophobic medication can be integrated into the core structure, enhancing the drug's bioavailability and stability (Ahammed et al. <xref ref-type=\"bibr\" rid=\"CR3\">2017</xref>; Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>). These micelles can be synthesized by precipitating one block and adding a solvent and direct dissolution of polymer in a convenient solvent-based followed by dialysis (Cho et al. <xref ref-type=\"bibr\" rid=\"CR37\">2015</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR208\">2013</xref>). Micelle formation is affected by amphiphilic molecules in the hydrophobic chain, the concentration of amphiphiles, temperature, and solvent system (Kulthe et al. <xref ref-type=\"bibr\" rid=\"CR91\">2012</xref>).</p><p id=\"Par38\">Targeted drug delivery, nanoscopic core structure, minimal biodegradability, micellar coupling, biocompatibility, and much-improved longevity are all attributes of polymeric micelles that make them ideal for drug therapy (Chamundeeswari et al. <xref ref-type=\"bibr\" rid=\"CR29\">2019</xref>). Chaudhari and Handge reported the development, characterization, and validation of lopinavir-encapsulated polymeric micelles using co-solvent (Tween 80) and polymeric material (Pluronic F127 and Pluronic F68) (Chaudhari and Handge <xref ref-type=\"bibr\" rid=\"CR32\">2020</xref>). The novel drug formulation increased drug loading capacity, bioavailability, and encapsulation performance. Mahajan and Patil developed polymeric vitamin E-tocopheryl polyethylene glycol succinate micelles encapsulating lopinavir for oral therapy. The developed lopinavir polymeric micelles had a better dissolution rate. An in vivo pharmacokinetic study in New Zealand rabbits showed that the formulated drug had much higher bioavailability (3.17%) than lopinavir suspension (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>).</p></sec><sec id=\"Sec18\"><title>Dendrimers</title><p id=\"Par39\">Dendrimers are monodispersed 3-D entities that may be cleaved into two halves. They have a spherical shape whose surface can be uniformly functionalized, making them exceptional for drug delivery (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>; Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR86\">2015</xref>). The dendrimers can be synthesized in two ways: the dendrimers would start forming from the core and then outside of the dendrimers. The other method is convergent in which the formation would start from outside the dendrimers (Zhu and Shi <xref ref-type=\"bibr\" rid=\"CR222\">2013</xref>). There are various types of dendrimers based on their functionalization moieties such as polypropylene imine (PPI), polyamidoamine (PAMAM), core–shell, peptide, glycodendrimers, liquid crystalline, chiral and polyamidoamine-organosilicon (PAMAMOS). Since PAMAM is water-soluble and can penetrate epithelial tissues, it is mainly explored for oral drug delivery (Cheng et al. <xref ref-type=\"bibr\" rid=\"CR36\">2008</xref>). However, the applications of dendrimers are limited due to the presence of amino groups'. This group is cationic or positively charged, which makes them toxic. Therefore, the utilization of dendrimers requires some modifications to reduce their toxicities. Drugs are loaded into the dendrimers through electrostatic interaction, simple encapsulation, and covalent conjugation (Noriega-Luna et al. <xref ref-type=\"bibr\" rid=\"CR136\">2014</xref>). Drugs can be delivered in two ways. The first is in vivo delivery, where the degradation of the covalent bond of the dendrimers depends on the favorable environment and the enzymes available to cleave the bond. Ethynylene–perylene (EPer) would discharge the drug due to environmental changes such as temperature and pH (Tripathy and Das <xref ref-type=\"bibr\" rid=\"CR187\">2013</xref>). Drugs can be delivered orally, pulmonary, transdermally, ocularly, and targeted through dendrimers (Kesharwani et al. <xref ref-type=\"bibr\" rid=\"CR85\">2014</xref>).\n",
       "</p><p id=\"Par40\">Among the extensively documented studies on the application of dendrimer systems for therapeutic purposes COVID-19, numerous findings can be summarized. According to Itani et al<italic>.,</italic> dendrimers can connect robustly with viruses and COVID-19 suppress invasion in target cells. This comprehensive analysis explored the prospective importance of nanoparticles such as theranostic dendrimers as suitable transporters for immune stimulators or cures to help combat COVID-19 (Prusty and Swain <xref ref-type=\"bibr\" rid=\"CR224\">2018</xref>). In an experiment comparable to Middle East coronavirus, Kandeel and co-workers investigated the antiviral potential of 16 different polyamidoamine dendrimers against SARS-CoV-2. In these intriguing experiments, polycationic dendrimers comprising the primary amine (-NH2-) class (grade 2/3/4 and/or 5) and three different categories of polyanionic dendrimers comprising the amido ethanol hydroxyl class (grade 2/3/4 and/or 5), the sodium carboxylate end group class (grade 1.5/2.5/3.5 and/or 4.5), succinic acid class (grade 2/3/4 and/or 5) were structured and assumed to have calcification inhibitory activity in SARS-CoV-2. The grade 1.5 CO2-Na polyamidoamine dendrimer inhibited SARS-CoV-2 calcification growth most strongly (inhibitory effect 40%), followed by the grade 5 polyamidoamine dendrimer with succinic acid (inhibitory effect 39.7%), which contains terminal grades 16 and 128. Vero cells showed a strong toxic effect in the presence of cationic dendrimers. The authors suggested that antiviral agents could be incorporated into these dendrimers for use as nanomedicines (Itani et al. <xref ref-type=\"bibr\" rid=\"CR73\">2020</xref>).</p></sec><sec id=\"Sec19\"><title>Inorganic nanoparticles</title><p id=\"Par41\">Inorganic nanoparticles, including gold, silver, silica, and iron oxide, possess exceptional potential applications. Gold and silver-like nanoparticles have properties like surface plasmon resonance (SPR), which is not possessed by dendrimers, liposomes, or micelles. They also have several advantages that include versatility and biocompatibility. Drug conjugation with gold nanoparticles can occur through covalent or ionic bonds. In addition, silver nanoparticles can exhibit antimicrobial activities. For example, a spongy and interconnected dextran/polyacrylamide nanohydrogel matrix covalently coupled with AuNPs could release the ornidazole with a 98.5% in vitro (Kandeel et al. <xref ref-type=\"bibr\" rid=\"CR81\">2020</xref>).</p><p id=\"Par42\">Experimental research with iron oxide (IO) nanoparticles targeting SARS-CoV-2 was explored. IO and metal oxide nanoparticles with the versatile properties of zinc oxide, titanium dioxide, and copper oxide are effective in antiviral diagnosis and treatment. Titanium dioxide nanoparticles destroy pathogens by generating reactive oxygen species. This is according to a recent paper that investigated the photocatalytic and antiviral properties of titanium dioxide. It tends to inhibit viruses, structurally destroy DNA, and degrade the phospholipid bilayer (Choudhury et al. <xref ref-type=\"bibr\" rid=\"CR38\">2005</xref>). Silver nanoparticles bind to viral proteins in two ways: (i) they attach to the outer membrane of the virus, preventing it from attaching to the cell surface receptor; and (ii) they attach to the viral nucleic acid (RNA or DNA), limiting transfection and viral replication within the host cell (Mahajan and Patil <xref ref-type=\"bibr\" rid=\"CR118\">2020</xref>). In addition, the entry of the antiviral transcription factors interferon regulatory factors (IRF)-7 into the nucleus of lung epithelial cells is suppressed by silver nanoparticles and mitochondrial cycling is disrupted (Halbus et al. <xref ref-type=\"bibr\" rid=\"CR60\">2017</xref>). According to a recent finding, SARS-CoV-2 may resist the antiviral effect of silver nanoparticles (Salleh et al. <xref ref-type=\"bibr\" rid=\"CR167\">2020</xref>).</p></sec><sec id=\"Sec20\"><title>Nanoemulsions</title><p id=\"Par43\">Nanoemulsions do not irritate or affect the skin. Therefore, they can effectively improve drug bioavailability (Villeret et al. <xref ref-type=\"bibr\" rid=\"CR194\">2018</xref>). In addition, nanoemulsions consist of small particles with a large surface area, which promotes absorbability (Abd Elkodous et al. <xref ref-type=\"bibr\" rid=\"CR1\">2021</xref>). To understand how some pathogenic viruses are affected by nanoemulsions of different essential oils, several experiments have been carried out. The majority of published studies assume that the nanoemulsions of various essential oils would interfere with the assembly of the virion envelope, thus blocking the attachment of viruses and their penetration into host cells (Naseema et al. <xref ref-type=\"bibr\" rid=\"CR131\">2021</xref>).</p><p id=\"Par44\">Nanoemulsions can compromise virion membrane integrity or obscure viral assembly, preventing SARS-CoV-2 penetration and adhesion into host cells. Furthermore, the encapsulated favipiravir nanoemulsion can suppress the viral polymerase enzyme of SARS-CoV-2, i.e., RNA-dependent RNA polymerase activity in infected cells. In conclusion, we predict that encapsulated favipiravir nanoemulsions could successfully prevent the adhesion of SARS-CoV-2 while disrupting its morphology (Franklyne et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>).</p></sec><sec id=\"Sec21\"><title>Nanosuspensions</title><p id=\"Par45\">Nanosuspensions are submicron dispersions of hydrophobic drug particulates that have been stabilized with surfactants and possess a high affinity for target receptors (de M Ribeiro and Fonseca <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Nanosuspensions have high bioavailability and dissolution rate due to their small size and greater surface area (Jacob et al. <xref ref-type=\"bibr\" rid=\"CR74\">2020</xref>). In the recipient tissues, nanosuspensions can drastically limit viral replication and inhibit the virus from infiltrating host cells (Flühmann et al. <xref ref-type=\"bibr\" rid=\"CR53\">2019</xref>). Due to their extensive surface area and capacity to adhere various antigens to their interface, they are very often used in research. Furthermore, nanomaterials such as carbon quantum dots, gold nanoparticles, and carbon quantum dots have been characterized as both stimulating and preventing virus interaction in the host cells (Noyes and Whitney <xref ref-type=\"bibr\" rid=\"CR137\">1897</xref>).</p><p id=\"Par46\">Nanosuspensions are effective in delivering therapeutics that are lipid and water-insoluble (Szunerits et al. <xref ref-type=\"bibr\" rid=\"CR180\">2015</xref>). Additionally, they increase the drug solubility and rate of dissolution (Murtaza et al. <xref ref-type=\"bibr\" rid=\"CR129\">2020</xref>). Fucoxanthin (a carotenoid antioxidant) is readily soluble in organic solvents, i.e., acetone, and insoluble in water and exhibits excellent consistency and oxidative stability over a wide pH range. Acute toxicity was studied at a dose of 2000 mg/kg (LD50) and confirmed to be non-toxic. Conventional therapeutic administration is limited by the high dosage of fucoxanthin extract and its solubility properties. However, nanosuspension materials could easily overcome these limitations (Yousaf et al. <xref ref-type=\"bibr\" rid=\"CR213\">2021</xref>).</p></sec><sec id=\"Sec22\"><title>Quantum dots</title><p id=\"Par47\">Quantum dots have a size of 2–10 nm and are known as semiconductor nanocrystals. Their optical properties such as photoluminescence and absorption depend on their size (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>). Nowadays, quantum dots have become very important in the field of nanomedicine. The emission of quantum dots is in the infrared range, i.e., below 650 nm. This is due to the lower scattering of light and low absorption in tissue, which makes them desirable for biomedical imaging (Muthuirulappan and Francis <xref ref-type=\"bibr\" rid=\"CR130\">2013</xref>; Volkov <xref ref-type=\"bibr\" rid=\"CR195\">2015</xref>).</p><p id=\"Par48\">Additionally, different quantum dot sizes and compositions can excite a similar source of light, culminating in different of emitting colors throughout a wide range of spectra (Liu et al. <xref ref-type=\"bibr\" rid=\"CR108\">2010</xref>; Xu et al. <xref ref-type=\"bibr\" rid=\"CR207\">2016</xref>). In this manner, quantum dots are highly captivating for multiplex imaging. Quantum dots are studied as sensors, target drug delivery, and bioimaging in the medicinal field. In literature, the in vivo imaging of the quantum dots is available for its application (Shi et al. <xref ref-type=\"bibr\" rid=\"CR175\">2015</xref>; Han et al. <xref ref-type=\"bibr\" rid=\"CR61\">2015</xref>).</p><p id=\"Par49\">Carbon-based quantum dots with a cationic external potential could be exploited to inactivate the SARS-CoV-2 S protein (Balaji et al. <xref ref-type=\"bibr\" rid=\"CR14\">2018</xref>). Furthermore, SARS-CoV-2 releases reactive oxygen species due to the binding of quantum dots with positive extrinsic characteristics to the virus-negative RNA motif (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>). In a study conducted by Du et al<italic>.,</italic> it was shown that carbon dots have antiviral properties against respiratory syndrome and pseudorabies virus (Dong et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). In addition, carbon dots trigger interferon-stimulated signaling pathways, including interferon-α formation, which limits viral replication. Integration of targeted substituents into quantum dots could also efficiently attach to the SARS-CoV-2 entry receptor and alter cellular reproduction (Du et al. <xref ref-type=\"bibr\" rid=\"CR50\">2016</xref>).</p></sec><sec id=\"Sec23\"><title>Protein and polysaccharide nanoparticles</title><p id=\"Par50\">Proteins and polysaccharides are natural biopolymers extracted from animals, plants, marine sources, and microorganisms (Iannazzo et al. <xref ref-type=\"bibr\" rid=\"CR70\">2018</xref>). Protein-based nanoparticles are usually metabolized, decomposable and facile for drug and other targeted ligand attachment. There are two ways to produce these nanoparticles. The first method is derived from water-soluble proteins such as human serum albumin and bovine serum albumin. The other option is to use insoluble proteins such as gliadin and zein. The usual synthesis is by desolvation or coacervation, complex coacervation, solvent/emulsion extraction, and electrospraying. These nanoparticles were chemically modified to integrate targeted compounds and localize tissues and cells to support the targeting mechanism (Bassas-Galia et al. <xref ref-type=\"bibr\" rid=\"CR18\">2017</xref>).</p><p id=\"Par51\">Polyguluronate sulfate, carrageenan, chitosan, and some derivatives of marine polysaccharides have potent inhibitory antagonistic potential against several viruses and provide a platform for conducting a study on coronaviruses (Lohcharoenkal et al. <xref ref-type=\"bibr\" rid=\"CR110\">2014</xref>). The most common cause of severe infectious diseases of the upper respiratory tract sometimes referred to as the common cold are respiratory viruses such as influenza viruses, coronaviruses, and rhinoviruses (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>; Tomas et al. <xref ref-type=\"bibr\" rid=\"CR186\">2022</xref>). Clinical studies have shown that using an iota carrageenan nasal spray to relieve the common cold can shorten the severity of the infection. The antiviral efficacy of carrageenan nasal spray was found to be significant for three categories of viruses: influenza A virus, human coronavirus, and human rhinovirus, with the greatest success observed in individuals with human coronavirus. In individuals with coronavirus treated with Carra-Gene, the severity of infection was 3 days shorter (<italic>p</italic> &lt; 0.01), and the frequency of exacerbations was threefold lower (<italic>p</italic> &lt; 0.01) than in untreated individuals (Morokutti-Kurz et al. <xref ref-type=\"bibr\" rid=\"CR127\">2017</xref>). In addition, sulfated polysaccharides such as sulfated rham-nan and fucoidan may impair the development and function of the epidermal growth factor receptor or decrease its signal transduction, which could improve the silencing function of coronavirus (Monto et al. <xref ref-type=\"bibr\" rid=\"CR126\">2001</xref>; Wang et al. <xref ref-type=\"bibr\" rid=\"CR196\">2017</xref>).</p></sec></sec><sec id=\"Sec24\"><title>Role of nanomedicine in the management of COVID-19</title><p id=\"Par52\">One of the most pressing issues over the past pandemics is the absence of a viable therapy for viral diseases such as SARS-CoV-1, MERS-1, and SARS-CoV-2. As a result, repurposed therapeutics are more beneficial than newer drugs developed in treating COVID-19 outbreaks. A new vaccine typically takes a long time to produce, requiring multiple rounds of guidelines and trials. One of the major challenges in successfully treating COVID-19 is insufficient cellular uptake of the newly developed antiviral drugs. Their usage is also limited due to several factors such as low bioavailability, poor water solubility, increased toxicity, lower biological potential, and a high risk-to-benefit ratio (Wang et al. <xref ref-type=\"bibr\" rid=\"CR197\">2018</xref>; Ansari et al. <xref ref-type=\"bibr\" rid=\"CR9\">2020</xref>).</p><p id=\"Par53\">Consequently, these limitations would limit the prospects of new techniques to increase the efficacy of repurposed drugs and their better absorption into the body. To overcome this limitation, a variety of nanomedicine-based systems have been developed to enhance the therapeutic efficacy of newly developed antiviral drugs (Lembo et al. <xref ref-type=\"bibr\" rid=\"CR103\">2018</xref>). Antiviral drugs based on nanomedicine offer several opportunities to overcome the drawbacks of current antiviral therapeutics (Zazo et al. <xref ref-type=\"bibr\" rid=\"CR215\">2016</xref>). Several fundamental obstacles, such as low bioavailability and poor aqueous solubility, can be addressed through nanomedicine design. This involves modifying the pharmacokinetic and pharmacodynamic properties of drugs to achieve lower toxicity and lower dose, improve drug bioavailability, and inhibit viral spread (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>). Additionally, nanomedicine can penetrate cellular membranes and concentrate therapeutic effects in viral persistence sites (Akhter et al. <xref ref-type=\"bibr\" rid=\"CR6\">2012</xref>). One of the most attractive approaches to minimize the adverse effects of viral rebound and patient compliance during the course of therapy is nanomedicine with intrinsically regulated or prolonged release capabilities (Yadav et al. <xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>). Nanocarrier technology could thus be used to repurpose drugs and optimize treatment efficacy COVID-19 (Li et al. <xref ref-type=\"bibr\" rid=\"CR104\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR105\">b</xref>).</p><p id=\"Par54\">The upsides of nanocarriers include their high drug loading effectiveness, larger surface area, smaller in size, prolonged drug releasing pattern, and enhanced optimum performance of the loaded pharmaceuticals in such a novel drug delivery system (Fig. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>) (Sahakijpijarn et al. <xref ref-type=\"bibr\" rid=\"CR166\">2020</xref>; Ahmad et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>; Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>). Nanotechnology has a specific physicochemical advantage in medicine due to its unique hydrophobic or hydrophilic nature, nanosize, nanoshape, and the possibility of ligand conjugation on its membrane. These nanoscale physicochemical features would improve pharmaceutical applications' efficiency, specificity, functionality, and sensitivity (Kobayashi et al. <xref ref-type=\"bibr\" rid=\"CR89\">2014</xref>). Several nanomedicine groups could be repurposed to treat COVID-19 therapy, and they are further summarized below. Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> outlines the findings of nanomedicine as a strategy to augment the pharmacological output of several repurposed medications with the potential for COVID-19 treatment.<fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Role of nanomedicine drug delivery in COVID-19. Nanomedicine offers numerous characteristics that may be effectively used to transport therapeutic compounds to infected cells, and the tailored ligand-coupled nanoparticle selectively interacts with viral residues, resulting in virus destruction and inability to penetrate the cells </p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2795_Fig3_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Overview of nanomedicine for improving the pharmacological efficacy of a repurposed medication with COVID-19 management potential</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Drug</p></th><th align=\"left\"><p>Nanocarrier</p></th><th align=\"left\"><p>Model (<italic>In vitro/In vivo</italic>)</p></th><th align=\"left\"><p>Results</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Improved target specificity; bioavailability is significantly elevated in the thymus and spleen</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR141\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased drug transport in the lymphatic system; oral bioavailability is significantly improved; the drug release is regulated through diffusion</p></td><td align=\"left\"><p>Freitas et al. (<xref ref-type=\"bibr\" rid=\"CR55\">2005</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Oral bioavailability is significantly improved</p></td><td align=\"left\"><p>Sercombe et al. (<xref ref-type=\"bibr\" rid=\"CR01\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Enhanced lymphatic targeting specificity in the intestine; bioavailability in the spleen and thymus is significantly higher than in the plasma</p></td><td align=\"left\"><p>Negi et al. (<xref ref-type=\"bibr\" rid=\"CR133\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>Liposome</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Increased bioavailability and <italic>in-vitro</italic> release in the spleen and thymus; improved target specificity</p></td><td align=\"left\"><p>Maniyar and Kokare (<xref ref-type=\"bibr\" rid=\"CR02\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>The drug peak plasma concentration is improved</p></td><td align=\"left\"><p>Kumar et al. (<xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>NLC</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Bioavailability is increased by 6.98 times when compared to a drug suspension</p></td><td align=\"left\"><p>Garg et al. (<xref ref-type=\"bibr\" rid=\"CR56\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Improved Tissue Distribution; improved oral bioavailability by a significant extent; drug degradation protection</p></td><td align=\"left\"><p>Khan et al. (<xref ref-type=\"bibr\" rid=\"CR88\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ritonavir</p></td><td align=\"left\"><p>SLN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Ensure that viral replication is actively suppressed; entrapment efficiency has increased; biphasic controlled release</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR159\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PM</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Entrapment efficiency has increased; improved stability</p></td><td align=\"left\"><p>Yadav et al. (<xref ref-type=\"bibr\" rid=\"CR209\">2019</xref>)</p></td></tr><tr><td align=\"left\"><p>Ribavirin</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p>C57BL/6N mice</p></td><td align=\"left\"><p>Longer residence time in the mice liver; improved cellular internalization</p></td><td align=\"left\"><p>Javan et al. (<xref ref-type=\"bibr\" rid=\"CR77\">2017</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>PN</p></td><td align=\"left\"><p><italic>In vitro</italic></p></td><td align=\"left\"><p>Enhanced drug release pattern; Entrapment efficiency has increased</p></td><td align=\"left\"><p>Ishihara et al. (<xref ref-type=\"bibr\" rid=\"CR71\">2014</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Significant build-up in organs such as the spleen and liver; biphasic controlled release; bioavailability are enhanced by 4 times</p></td><td align=\"left\"><p>Katata et al. (<xref ref-type=\"bibr\" rid=\"CR82\">2020</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>SNEDDS</p></td><td align=\"left\"><p>Wistar albino rats</p></td><td align=\"left\"><p>Entrapment efficiency has increased; bioavailability is considerably enhanced</p></td><td align=\"left\"><p>Ravi et al. (<xref ref-type=\"bibr\" rid=\"CR160\">2015</xref>)</p></td></tr><tr><td align=\"left\"><p>Lopinavir</p></td><td align=\"left\"><p>TPGSM</p></td><td align=\"left\"><p>New Zealand rabbits</p></td><td align=\"left\"><p>Relative bioavailability is enhanced by 3.17 times; entrapment efficiency has increased; improved drug dissolution</p></td><td align=\"left\"><p>Patel et al. (<xref ref-type=\"bibr\" rid=\"CR03\">2016</xref>)</p></td></tr></tbody></table><table-wrap-foot><p>Nanostructured lipid carriers: NLC, Solid lipid nanoparticles: SLN, Polymeric micelles: PM, Polymeric nanoparticles: PN, Self-nanoemulsifying drug delivery system: SNEDDS, D-α-Tocopheryl polyethylene glycol succinate micelles: TPGSM</p></table-wrap-foot></table-wrap></p><sec id=\"Sec25\"><title>Liposomes targeted drug delivery system for COVID-19</title><p id=\"Par55\">The liposomes are the bilayers of lipid vesicles and are currently utilized to deliver aquaphobic drugs (Puri et al. <xref ref-type=\"bibr\" rid=\"CR150\">2009</xref>). The liposomes would function as stealth liposomes whereby the systemic circulation is long, targeted, and stimulus-responsive for drug delivery. As liposome is complex and biologically degraded in nature, it is one of the methods of choice for the carrier of drug formulation. Moreover, it is also a productive platform for developing a novel formulation of COVID-19 diseases. Studies have found that synthetic peptides loaded on the liposomes are potential venues for treating SARS coronavirus infection (Lokhande <xref ref-type=\"bibr\" rid=\"CR111\">2018</xref>). Liposomes that are chemically conjugated peptides have been used in the treatment of SARS because they are suitable for the cytotoxic T-lymphocyte strategy that reduces viral load. A novel hydroxychloroquine formulation administered to Sprague–Dawley rats had shown efficacy against COVID-19. Unlike the IV injection, the formulation is breathed and has a greater concentration of medication in the lungs, a longer half-life, and less exposure to other organs such as the heart. Liposome as a carrier system for the delivery of drugs in the treatment of COVID-19 has many advantages. Forty percentage of small molecule drugs used to treat cancer are poorly soluble in water. Therefore, a similar drug delivery method is needed immediately to encapsulate the drug and increase its water solubility (Wankar et al. <xref ref-type=\"bibr\" rid=\"CR199\">2020</xref>).</p></sec><sec id=\"Sec26\"><title>Cyclodextrin combination targeted drug delivery system for COVID-19</title><p id=\"Par56\">Cyclodextrin (a biocompatible natural polysaccharide) has α, β, and γ as essential components. According to reports, cyclodextrin nanoparticles are gaining importance due to numerous cyclodextrin components that absorb a greater amount of therapeutics than normal cyclodextrin (Sun et al. <xref ref-type=\"bibr\" rid=\"CR178\">2014</xref>). Cyclodextrin-based remdesivir nanoformulation has just received Food Drug Administration (FDA) approval for the treatment of COVID-19 (Khalaj‐Hedayati et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>). In the clinical context, it has shown encouraging results in various viruses, including Middle East respiratory syndrome coronavirus and Nipah virus (Jones et al. <xref ref-type=\"bibr\" rid=\"CR79\">2020</xref>). In contrast to its function as a medical dispersant for therapeutics, which are primarily hydrophobic, cyclodextrin also possesses antiviral activities (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>). Therefore, cyclodextrin could be used as a delivery strategy for respiratory sprays for therapeutics that require targeted selectively to the pulmonary system (<italic>Khalaj‐Hedayati</italic> et al. <xref ref-type=\"bibr\" rid=\"CR87\">2020</xref>).</p></sec><sec id=\"Sec27\"><title>Inorganic and metallic-based nanocomposites targeted drug delivery system for COVID-19</title><p id=\"Par57\">Inorganic nanocomposites for therapeutics delivery can be developed using a variety of metallic nanoparticles, including platinum, silica, silver, gadolinium, gold, and its nanocrystal conglomerates (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR94\">2017</xref>). Silver nanoparticles are known for their extensive antibacterial activity and are an essential ingredient in pharmaceutical formulations, e.g., silver sulfasalazine (a sulfa antibiotic), where silver particles support the antibacterial activity of the sulfasalazine. Silver nanoparticles are also being studied for their possible virucidal properties. It has been observed that SARS-CoV-2 attaches to cell membrane and persists there for up to 12 h, while the silver nanoparticle-based antibacterial film can eradicate the virus (Chaturvedi et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>). This film can be sprayed on metal, glass, and ceramic surfaces to hamper the proliferation of SARS-CoV-2 in busy places such as universities, hospitals, and marketplaces (Prasad et al. <xref ref-type=\"bibr\" rid=\"CR148\">2015</xref>).</p></sec><sec id=\"Sec28\"><title>Polymeric solid colloidal nanoparticles targeted drug delivery system for COVID-19</title><p id=\"Par58\">Due to their unique properties, including targeted drug delivery, controlled drug release, improved tolerability, and efficiency, PN promotes stable <italic>in vivo</italic> delivery of therapeutic agents (De Wit et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>; Balagna et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Due to their nanosize, they also prevent large dose-dependent side effects and penetration through biological membranes. This favorable property of polymeric nanoparticles can be exploited for drug delivery to revive currently accessible therapeutics for the treatment of the SARS-CoV-2 outbreak. Zhang et al<italic>.</italic> formulated nanoparticles based on PLGA [poly (lactic-co-glycolic acid)] polymers to combat the infection rate of SARS-CoV-2. PLGA was used as the internal base component in a membrane-enveloped nanoparticle derived from human macrophages and human lung epithelial cells of the II type, which exhibit the same physiological processes as SARS-CoV-2 invading recipient cells (Verma et al. <xref ref-type=\"bibr\" rid=\"CR192\">2017</xref>). During the incubation period, these biological nanoparticles serve as a receptive target for SARS-CoV-2, where it is destroyed and can no longer enter recipient cells (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR220\">2020a</xref>, <xref ref-type=\"bibr\" rid=\"CR216\">b</xref>). Therefore, the polymer-based nanoparticles may provide a nanocarrier platform for the delivery of therapeutics against SARS-CoV-2 contamination.</p></sec><sec id=\"Sec29\"><title>Nano-based vaccines</title><p id=\"Par59\">The sudden appearance of COVID-19 has necessitated the immediate use of drugs to combat the virus. Health professionals have attempted various methods to manage the symptoms. Still, due to certain factors including low solubility, permeability, and the limitation of targeting competence, they could not be used as they were unable to provide the required therapeutic effect. To deliver the drug to the appropriate venue with the fewest adverse effects, nanotechnology has been essential (Kumari et al. <xref ref-type=\"bibr\" rid=\"CR95\">2018</xref>). They could transport the therapeutic molecule to the infected cells, and the targeted liganded nanoparticles would bind with the viral epitopes, deactivating the virus and preventing it from entering the cell. According to the WHO report on COVID-19 candidate vaccines, 52 possible vaccines are undergoing clinical trials, while 162 are in the preclinical stage of development throughout December 8, 2020 (Raj et al. <xref ref-type=\"bibr\" rid=\"CR155\">2021</xref>). The platforms include virus-like particles, DNA, protein subunits, RNA, inactivated viruses, and non-replicating viral vectors. With a pandemic like COVID-19, a huge amount of vaccines is needed, which is not made possible with the existing production of vaccines (Ashraf et al. <xref ref-type=\"bibr\" rid=\"CR12\">2021</xref>). In this case, the noble way of drug delivery has assisted, including reverse vaccinology, nanomedicine, next-generation sequencing, and human challenge studies. The nanoparticle-based vaccination has elicited a strong immunological response that is well-tolerated. Gold, silver sulfide, titanium oxide, zirconium, and graphene are some of the materials that have been considered (Haslberger et al. <xref ref-type=\"bibr\" rid=\"CR65\">2020</xref>). Drug delivery, cancer nanovaccines, immunological engineering, nanosensors, and platform technologies are some techniques employed by nanoresearchers.</p><p id=\"Par60\">In the case of the increasing number of pathogenic bacteria that are resistant to antimicrobial agents, there are several studies that have demonstrated the efficacy of nanotechnology-based antimicrobial therapy (Nikaeen et al. <xref ref-type=\"bibr\" rid=\"CR135\">2020</xref>). Targeted nanotechnology is a new resource for the treatment of antiviral diseases. The employment of nanoparticles in the battle against SARS-CoV-2 has created a mechanism that inhibits the virus's entry into the host cell until it is inactivated. The targeted nanoparticles specific to virus-expressed protein would minimize the internalization of the virus. Metal nanoparticles can prevent the virus from attaching to the surface of the cell and impair the virus's ability to replicate once it enters the cell (Kumar et al. <xref ref-type=\"bibr\" rid=\"CR92\">2018a</xref>, <xref ref-type=\"bibr\" rid=\"CR93\">b</xref>). The size, shape, and surface charge of nanoparticles would determine how effective the treatment is. However, several safety precautions need to be taken to reduce host cell cytotoxicity. Well-established nanotechnology drug delivery would improve traditional therapies in medicine, as seen in the SARS-CoV-2 outbreak, but its use in viral diseases is still relatively unknown. By shortening PCR assays, reducing detection time, improving sensitivity, amplifying signals, and using them as adjuvants in vaccines, nanostructure technology could help diagnose disease. Nanotechnology has the potential to assist in the pathogenesis of COVID-19 in a variety of ways, including attenuation, possible membrane attachment and fusion upon viral entry, and protein fusion in infected cells. Activating intracellular pathways for reversible virus damage and deactivating viral transcription, translation, and replication may be more effective with nanocapsulated vaccines (Maji et al. <xref ref-type=\"bibr\" rid=\"CR119\">2020</xref>).</p><p id=\"Par61\">To increase half-life and longevity and to elicit adequate cellular and antibody responses, nanoparticles can serve as biologics or cargo for vaccination, sometimes referred to as nanovaccines, among other factors (Varahachalam et al. <xref ref-type=\"bibr\" rid=\"CR191\">2021</xref>). The basic principle of this approach is to integrate an epitope that triggers protective immunity into nanoparticles of 1–100 nm and establish an immunological storage that protects the body from emerging harmful microorganisms. Moreover, suitable protein nanoparticles from the architectural proteins of SARS-CoV-2, <italic>i.e.,</italic> envelope (E), membrane (M), nucleocapsid (N), and spike (S), can be transported to target immune systems that would trigger immune feedback and confer durable resistance to the virus. Protein nanoparticles and biologics have already been used for Middle East Coronavirus 1 and Severe Acute Respiratory Syndrome, and they could also be developed for SARS-CoV-2 (Coleman et al. <xref ref-type=\"bibr\" rid=\"CR39\">2014</xref>; Xiang et al. <xref ref-type=\"bibr\" rid=\"CR206\">2006</xref>). Pimentel et al<italic>.</italic> developed a nanovaccine using a repetitive SARS-CoV-1 S protein antigen of around 25 nm to design a targeted antibody that can prevent contamination with SARS-CoV-1 (Pimentel et al. <xref ref-type=\"bibr\" rid=\"CR147\">2009</xref>). Matsuura et al<italic>.</italic> designed virus-like nanoparticles that stimulate a viral protein. We hypothesize that these nanoparticles can also be used to produce SARS-CoV-2 proteins for vaccination applications (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Microparticles (25 nm) depending on epitope selectivity and adjuvant strategies have been utilized to enhance the transport of peptide/protein epitopes to dendritic cells to trigger both cellular and humoral immunological responses in rats (Matsuura et al. <xref ref-type=\"bibr\" rid=\"CR121\">2010</xref>). Numerous nanoparticles, including nanobioceramics, liposomes, cationic polymers, polymer-based particles, gold nanoparticles, virus-like nanoparticles, and peptide nanoparticles, can be used to stimulate lymphatic circulation and produce B lymphocytes (B cells) and T lymphocytes (T cells) for efficient clearance of SARS-CoV-2 (Reddy et al. <xref ref-type=\"bibr\" rid=\"CR161\">2008</xref>; Gregory et al. <xref ref-type=\"bibr\" rid=\"CR58\">2013</xref>; Wilson et al. <xref ref-type=\"bibr\" rid=\"CR201\">2018</xref>; Koshy et al. <xref ref-type=\"bibr\" rid=\"CR90\">2017</xref>; Chen et al. <xref ref-type=\"bibr\" rid=\"CR33\">2016</xref>). Stimulating CD8 + (cytotoxic) T lymphocyte cells or CD4 + (helper) T lymphocyte cells is essential for any proposed vaccination. In the case of coronavirus, T lymphocyte cells have shown the potential to prevent relapse and provide immunologic maintenance for up to 11 years after exposure to SARS-CoV-1 in recovered patients (Goodwin and Huang <xref ref-type=\"bibr\" rid=\"CR57\">2017</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR181\">2011</xref>).</p><p id=\"Par62\">Research shows that DNA vaccination also induces neutralization and preventive resistance to SARS-CoV-1 through upregulation of neutralizing antibodies, clusters of differentiation 4 (CD4 +), and clusters of differentiation 8 (CD8 +), suggesting that DNA vaccination can be delivered with appropriate nanoparticles. In addition, the antigens of B lymphocyte cells and T lymphocyte cells in the immunological domains of SARS-CoV-1 and SARS-CoV-2 were evaluated in a random analysis and were shown to be equivalent for both infections (Peng et al. <xref ref-type=\"bibr\" rid=\"CR145\">2006</xref>). This implies that SARS-CoV-1 vaccinations may be more effective at getting rid of SARS-CoV-2. Numerous properties may be considered for optimizing a SARS-CoV-1 vaccine against SARS-CoV-2, namely enhancing epitope integrity, cytokine production, specific target allocation, prolonged release of biologics and/or epitopes, sequestration of SARS-CoV-2 infections, and prevention of peripheral manifestations of infection (Ahmed et al. <xref ref-type=\"bibr\" rid=\"CR5\">2020</xref>; Rao et al. <xref ref-type=\"bibr\" rid=\"CR157\">2020</xref>). Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref> provides an overview of some of the most promising COVID-19 vaccine alternatives that have been formally clinically evaluated (Vijayan et al. <xref ref-type=\"bibr\" rid=\"CR193\">2019</xref>).<table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Currently existing COVID-19 prospective vaccines and their stage of clinical advancement</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Vaccine candidate</p></th><th align=\"left\"><p>Inventors</p></th><th align=\"left\"><p>Summary</p></th><th align=\"left\"><p>Status</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\"><p>SARS-CoV-2 recombinant glycoprotein/Matrix M1 nanoparticle vaccine</p></td><td align=\"left\"><p>Novavax United Kingdom</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Russell et al. (<xref ref-type=\"bibr\" rid=\"CR165\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>FINLAY-FR-2 vaccination against SARS-CoV-2</p></td><td align=\"left\"><p>Instituto Finlay de Vacunas</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Puga-Gómez et al. (<xref ref-type=\"bibr\" rid=\"CR149\">2023</xref>)</p></td></tr><tr><td align=\"left\"><p>nCov vaccine</p></td><td align=\"left\"><p>Zydus Cadila</p></td><td align=\"left\"><p>DNA-based vaccine</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Barrett et al. (<xref ref-type=\"bibr\" rid=\"CR17\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>SARS-CoV-2 vaccine (Vero cells)</p></td><td align=\"left\"><p>Chinese Academy of Medical Sciences + Institute of Medical Biology</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Huang et al. (<xref ref-type=\"bibr\" rid=\"CR69\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>mRNA- 1273 and mRNA- 1283</p></td><td align=\"left\"><p>National Institute of Allergy and Infectious Diseases + Moderna</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Patone et al. (<xref ref-type=\"bibr\" rid=\"CR142\">2022</xref>)</p></td></tr><tr><td align=\"left\"><p>VAT00002: SARS-CoV-2 spike (S) protein with a biologics</p></td><td align=\"left\"><p>GlaxoSmithKline + Sanofi pasteur</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ndwandwe and Wiysonge (<xref ref-type=\"bibr\" rid=\"CR132\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BNT 162b1 and BNT 162b2</p></td><td align=\"left\"><p>Fosun Pharma + Biontech/Pfizer</p></td><td align=\"left\"><p>RNA-based vaccine</p></td><td align=\"left\"><p>Phase IV</p></td><td align=\"left\"><p>Lamb (<xref ref-type=\"bibr\" rid=\"CR98\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Adeno-based Gam-COVID-Vac (rAd26-S + rAd5-S)</p></td><td align=\"left\"><p>Gamaleya Research Institute, Health Ministry of the Russian Federation</p></td><td align=\"left\"><p>Viral vector (non-proliferating)</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Vaezi and Meysamie (<xref ref-type=\"bibr\" rid=\"CR189\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Recombinant SARS-CoV-2 vaccine (CHO cells)</p></td><td align=\"left\"><p>Institute of Microbiology, Chinese Academy of Sciences + Anhui Zhifei Longcom Biopharmaceutical</p></td><td align=\"left\"><p>Protein subunit</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Sun et al. (<xref ref-type=\"bibr\" rid=\"CR179\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>BBV152</p></td><td align=\"left\"><p>Bharat Biotech International Limited</p></td><td align=\"left\"><p>Inactivated virus</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Ella et al. (<xref ref-type=\"bibr\" rid=\"CR51\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>ChAdOx1</p></td><td align=\"left\"><p>University of Oxford + AstraZeneca</p></td><td align=\"left\"><p>Vector-based vaccines</p></td><td align=\"left\"><p>Phase III</p></td><td align=\"left\"><p>Agrawal et al. (<xref ref-type=\"bibr\" rid=\"CR2\">2021</xref>)</p></td></tr><tr><td align=\"left\"><p>Ad26.C0V2.S</p></td><td align=\"left\"><p>Janssen Pharmaceuticals</p></td><td align=\"left\"><p>Vector-based vaccines encoding S protein</p></td><td align=\"left\"><p>Phase 3</p></td><td align=\"left\"><p>Bos et al. (<xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>)</p></td></tr></tbody></table></table-wrap></p></sec></sec><sec id=\"Sec30\"><title>Challenges in treatment with nano-based medicines</title><p id=\"Par63\">Nanovaccines and nanomedicine that have been utilized in novel medicines to fight against the continuous pandemic, like all current medications, must demonstrate their safety and be studied extensively before their release on the market. As the interaction between engineered nanomaterials (ENMs) with viral targets (envelope, capsid, and nucleic acid) is comprehended by many, the prediction and screening of ENM–virus interactions require a robust analytical framework. Toxicity relevant to inorganic nanoparticles is the limiting factor of their usage in various cases. Other beneficial biological properties can be made by changing the shape, size, and surface chemistry. Moreover, the physicochemical properties must be modified to reduce the side effects of the nanoparticles. ENMs should have a variable dose. Antiviral stability and bioavailability should be considered in many therapeutic applications. The main challenges of nanodelivery come in the form of physiological barriers, which are the mucosa and the alveolar fluid as they are utilized for intranasal as well as the reticuloendothelial structure and used for systemic delivery (Le et al. <xref ref-type=\"bibr\" rid=\"CR101\">2020</xref>). In the cell, the nanoparticle must be able to reject the endocytic degradation component. Certain studies also have shown that nanoparticles could cause damage to respiratory sites and affect lung function. The four key pathobiological aspects of nanoparticles, which include inflammation, genotoxicity, oxidative stress, and fibrosis, must be examined further. Certain studies have yet to prove whether a nasal or oral nanovaccine is safer and provides longer mucosal immunity and viral protection than others. That being said, the ability to provide adequate protection when administered to different age groups by different routes has yet to be confirmed. There are various nanomaterials ranging from polymers to dendrimers to nanoparticles, oligomers, liposomes and tiny molecules. However, once the virus chemical complex dissociates and the viruses are released, they lose their effectiveness and the cycle begins again. The reactive oxygen species formed by nanoparticles would lead to oxidative stress as it can cause a disturbance in mitochondrial respiration. This would cause harmful pathological processes in the cell that cause toxicity. In various studies, it was found that titanium dioxide could enter mice's lungs, causing nanoparticle accumulation to increase the level of lipid peroxidation and minimize the number of antioxidant cells treated by nanoparticles. Due to this, the level of the AP-1 transcription factor increased and affected the transcriptional process. DNA would interact with the carbon nanotubes (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>). Although nanoparticles have gained tremendous importance in research because they are very effective, their toxicological aspects need to be considered. The way nanoparticles act in the human body is of critical importance. When deposited to the maximum, they can cause harmful side effects, especially with platinum-containing drugs (cisplatin, carbapenem) used in the treatment of lungs, colorectal, ovarian, head, neck, and bladder. Due to this, its use is limited as the prominent side effect of it is nephrotoxicity. The major challenge in translating nanodrugs is the first clearance by the RES that minimizes the efficacy and the increasing toxicity in the organs such as the spleen, liver, and kidney (Ting et al. <xref ref-type=\"bibr\" rid=\"CR184\">2018</xref>; Yan et al. <xref ref-type=\"bibr\" rid=\"CR210\">2020</xref>). Iron oxide particles have harmful factors both <italic>in vivo</italic> and <italic>in vitro</italic> as they generate reactive oxygen species (ROS).</p><p id=\"Par64\">Nevertheless, the iron oxide is coated with a polymer that would maximize cell viability. When hydroxyl group is added to the gadolinium fullerene particle, it prevents the formation of ROS and excessive release of pro-inflammatory cytokines, which can cause mortality and severe infection. Macrophages and monocyte lineage significantly control the cytokine-facilitated inflammatory response and innate and adaptive response (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR217\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR218\">b</xref>; Shanley et al. <xref ref-type=\"bibr\" rid=\"CR173\">2021</xref>).</p></sec><sec id=\"Sec31\"><title>Future prospects</title><p id=\"Par65\">COVID-19 is a major global threat and the greatest challenge to humanity since World War II. The pandemic has caused many calamities that disrupted health and caused human casualties and other issues. Despite numerous attempts by all countries across the world, including extensive research and numerous clinical trials, no medication has yet been demonstrated to control the virus. This study has highlighted several traditional treatment strategies as well as the crucial function of nanomedicine in the new SARS-CoV-2. Drug substitution, such as remdesivir and chloroquine, is a quick way to achieve safe therapies. Furthermore, several relevant experimental types of research have revealed positive evidence against COVID-19. Nanometal vaccines for example, which are based on the antigenic properties of protein S, have been utilized to reduce viral load. However, a substantial number of studies that support the use of NP-based diagnostic and treatment techniques for SARS-CoV-2 have been published. As a result, all researchers across the world are encouraged to extend their studies in the field of NP-based therapy. There is a great requirement for adequate information on the virus's virulence and transmission to prevent and cure COVID-19 or any other viral pandemic. In the future, this will lead to the discovery of numerous unstructured encoded proteins, enzymes, and functional processes and enable us to understand viral transmission between species. As a result, we could use nanoparticles at the functional level to identify and establish therapeutic targets. As a result, a clinical trial to develop an effective nanovaccine can be conducted with the appropriate knowledge of the viral life cycle and host response. A universal NP-based vaccine with sufficient immunogenicity is the ultimate goal in overcoming the pandemic, as endorsed by researchers worldwide. With factors such as fast detection and mobility, microfluidics would play an essential role in the diagnosis of CoV.</p></sec><sec id=\"Sec32\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par66\">Other than vaccines, there are no approved SARS-CoV-2 therapies being explored in clinical trials. In addition, the nano-based vaccine to treat SARS-CoV-2 has not been adequately studied. All efforts to combat the virus should be explored, and nanotechnology approaches should be prioritized because they may offer new perspectives, particularly in preventing or treating viral internalization compared with standard therapy. To develop a rational design for targeted therapies, further research is also required to comprehend the relationship between CoV and nanoparticles. As the pathophysiology of SARS-CoV-2 is unclear, nanotechnology might be a beneficial technique apart from other methods to produce excellent COVID-19 therapy results.</p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author contributions</title><p>BB and RKS performed conceptualization; BB and AMAI did writing—original draft preparation; AF, SR, AM, AS, DK and JK contributed to writing—review and editing; RKS done supervision. All authors have read and agreed to the published version of the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par67\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References\n",
       "</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abd Elkodous</surname><given-names>M</given-names></name><name><surname>Olojede</surname><given-names>S</given-names></name><name><surname>Morsi</surname><given-names>M</given-names></name><name><surname>El-Sayyad</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Nanomaterial-based drug delivery systems as promising carriers for patients with COVID-19</article-title><source>RSC Adv</source><year>2021</year><volume>11</volume><issue>43</issue><fpage>26463</fpage><lpage>26480</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1ynt7fP</pub-id><pub-id pub-id-type=\"pmid\">35480012</pub-id><pub-id pub-id-type=\"pmcid\">9037715</pub-id><pub-id pub-id-type=\"doi\">10.1039/D1RA04835J</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Agrawal</surname><given-names>U</given-names></name><name><surname>Katikireddi</surname><given-names>SV</given-names></name><name><surname>McCowan</surname><given-names>C</given-names></name><name><surname>Mulholland</surname><given-names>RH</given-names></name><name><surname>Azcoaga-Lorenzo</surname><given-names>A</given-names></name><name><surname>Amele</surname><given-names>S</given-names></name><name><surname>Fagbamigbe</surname><given-names>AF</given-names></name><name><surname>Vasileiou</surname><given-names>E</given-names></name><name><surname>Grange</surname><given-names>Z</given-names></name><name><surname>Shi</surname><given-names>T</given-names></name><name><surname>Kerr</surname><given-names>S</given-names></name><name><surname>Moore</surname><given-names>E</given-names></name><name><surname>Murray</surname><given-names>JLK</given-names></name><name><surname>Shah</surname><given-names>SA</given-names></name><name><surname>Ritchie</surname><given-names>L</given-names></name><name><surname>O'Reilly</surname><given-names>D</given-names></name><name><surname>Stock</surname><given-names>SJ</given-names></name><name><surname>Beggs</surname><given-names>J</given-names></name><name><surname>Chuter</surname><given-names>A</given-names></name><name><surname>Torabi</surname><given-names>F</given-names></name><name><surname>Akbari</surname><given-names>A</given-names></name><name><surname>Bedston</surname><given-names>S</given-names></name><name><surname>McMenamin</surname><given-names>J</given-names></name><name><surname>Wood</surname><given-names>R</given-names></name><name><surname>Tang</surname><given-names>RSM</given-names></name><name><surname>de Lusignan</surname><given-names>S</given-names></name><name><surname>Hobbs</surname><given-names>FDR</given-names></name><name><surname>Woolhouse</surname><given-names>M</given-names></name><name><surname>Simpson</surname><given-names>CR</given-names></name><name><surname>Robertson</surname><given-names>C</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 hospital admissions and deaths after BNT162b2 and ChAdOx1 nCoV-19 vaccinations in 2·57 million people in Scotland (EAVE II): a prospective cohort study</article-title><source>Lancet Respir Med</source><year>2021</year><volume>9</volume><issue>12</issue><fpage>1439</fpage><lpage>1449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFOlsLvJ</pub-id><pub-id pub-id-type=\"pmid\">34599903</pub-id><pub-id pub-id-type=\"pmcid\">8480963</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(21)00380-5</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Paul</surname><given-names>J</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Shavi</surname><given-names>GV</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development and in vivo evaluation of functionalized ritonavir proliposomes for lymphatic targeting</article-title><source>Life Sci</source><year>2017</year><volume>183</volume><fpage>11</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtFSisb7P</pub-id><pub-id pub-id-type=\"pmid\">28647214</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.lfs.2017.06.022</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Alkahtani</surname><given-names>SA</given-names></name><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Ahmad</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>MS</given-names></name><name><surname>Mohsin</surname><given-names>N</given-names></name><name><surname>Abdel-Wahab</surname><given-names>BA</given-names></name></person-group><article-title xml:lang=\"en\">Progress in nanotechnology-based drug carrier in designing of curcumin nanomedicines for cancer therapy: current state-of-the-art</article-title><source>J Drug Target</source><year>2016</year><volume>24</volume><issue>4</issue><fpage>273</fpage><lpage>293</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xhs12qurs%3D</pub-id><pub-id pub-id-type=\"pmid\">26066739</pub-id><pub-id pub-id-type=\"doi\">10.3109/1061186X.2015.1055570</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ahmed</surname><given-names>SF</given-names></name><name><surname>Quadeer</surname><given-names>AA</given-names></name><name><surname>McKay</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Preliminary identification of potential vaccine targets for the COVID-19 coronavirus (SARS-CoV-2) based on SARS-CoV immunological studies</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>3</issue><fpage>254</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnvVKlsLk%3D</pub-id><pub-id pub-id-type=\"pmid\">32106567</pub-id><pub-id pub-id-type=\"pmcid\">7150947</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12030254</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akhter</surname><given-names>S</given-names></name><name><surname>Ahmad</surname><given-names>MZ</given-names></name><name><surname>Ahmad</surname><given-names>FJ</given-names></name><name><surname>Storm</surname><given-names>G</given-names></name><name><surname>Kok</surname><given-names>RJ</given-names></name></person-group><article-title xml:lang=\"en\">Gold nanoparticles in theranostic oncology: current state-of-the-art</article-title><source>Expert Opin Drug Deliv</source><year>2012</year><volume>9</volume><issue>10</issue><fpage>1225</fpage><lpage>1243</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhtl2ltr7N</pub-id><pub-id pub-id-type=\"pmid\">22897613</pub-id><pub-id pub-id-type=\"doi\">10.1517/17425247.2012.716824</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alafif</surname><given-names>T</given-names></name><name><surname>Tehame</surname><given-names>AM</given-names></name><name><surname>Bajaba</surname><given-names>S</given-names></name><name><surname>Barnawi</surname><given-names>A</given-names></name><name><surname>Zia</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Machine and deep learning towards COVID-19 diagnosis and treatment: survey, challenges, and future directions</article-title><source>Int J Environ Res Public Health</source><year>2021</year><volume>18</volume><issue>3</issue><fpage>1117</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlslCjtrw%3D</pub-id><pub-id pub-id-type=\"pmid\">33513984</pub-id><pub-id pub-id-type=\"pmcid\">7908539</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijerph18031117</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alcoba-Florez</surname><given-names>J</given-names></name><name><surname>González-Montelongo</surname><given-names>R</given-names></name><name><surname>Íñigo-Campos</surname><given-names>A</given-names></name><name><surname>de Artola</surname><given-names>DGM</given-names></name><name><surname>Gil-Campesino</surname><given-names>H</given-names></name><name><surname>Team</surname><given-names>TMTS</given-names></name><name><surname>Ciuffreda</surname><given-names>L</given-names></name><name><surname>Valenzuela-Fernández</surname><given-names>A</given-names></name><name><surname>Flores</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Fast SARS-CoV-2 detection by RT-qPCR in preheated nasopharyngeal swab samples</article-title><source>Int J Infect Dis</source><year>2020</year><volume>97</volume><fpage>66</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylurfN</pub-id><pub-id pub-id-type=\"pmid\">32492531</pub-id><pub-id pub-id-type=\"pmcid\">7833505</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2020.05.099</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ansari</surname><given-names>MA</given-names></name><name><surname>Almatroudi</surname><given-names>A</given-names></name><name><surname>Alzohairy</surname><given-names>MA</given-names></name><name><surname>AlYahya</surname><given-names>S</given-names></name><name><surname>Alomary</surname><given-names>MN</given-names></name><name><surname>Al-Dossary</surname><given-names>HA</given-names></name><name><surname>Alghamdi</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Lipid-based nano delivery of Tat-peptide conjugated drug or vaccine–promising therapeutic strategy for SARS-CoV-2 treatment</article-title><source>Expert Opin Drug Deliv</source><year>2020</year><volume>17</volume><issue>12</issue><fpage>1671</fpage><lpage>1674</lpage><pub-id pub-id-type=\"pmid\">32820694</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2020.1813712</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"other\">Arslan S, Delice O, Kahraman M, Yilmaz S, Aslan M (2021) Correlation of chest CT and RT-PCR testing in coronavirus disease 2019 (COVID-19) in Turkey. Ann Clin Anal Med. pp 483–487.</mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Asdaq</surname><given-names>SMB</given-names></name><name><surname>Ikbal</surname><given-names>AMA</given-names></name><name><surname>Sahu</surname><given-names>RK</given-names></name><name><surname>Bhattacharjee</surname><given-names>B</given-names></name><name><surname>Paul</surname><given-names>T</given-names></name><name><surname>Deka</surname><given-names>B</given-names></name><name><surname>Fattepur</surname><given-names>S</given-names></name><name><surname>Widyowati</surname><given-names>R</given-names></name><name><surname>Vijaya</surname><given-names>J</given-names></name><name><surname>Al Mohaini</surname><given-names>M</given-names></name><name><surname>Alsalman</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology integration for SARS-CoV-2 diagnosis and treatment: an approach to preventing pandemic</article-title><source>Nanomaterials</source><year>2021</year><volume>11</volume><issue>7</issue><fpage>1841</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWktrrF</pub-id><pub-id pub-id-type=\"pmid\">34361227</pub-id><pub-id pub-id-type=\"pmcid\">8308419</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano11071841</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ashraf</surname><given-names>MU</given-names></name><name><surname>Kim</surname><given-names>Y</given-names></name><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Seo</surname><given-names>D</given-names></name><name><surname>Ashraf</surname><given-names>M</given-names></name><name><surname>Bae</surname><given-names>YS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines (revisited) and oral-mucosal vector system as a potential vaccine platform</article-title><source>Vaccines</source><year>2021</year><volume>9</volume><issue>2</issue><fpage>171</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVSlsrrO</pub-id><pub-id pub-id-type=\"pmid\">33670630</pub-id><pub-id pub-id-type=\"pmcid\">7922043</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines9020171</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Balagna</surname><given-names>C</given-names></name><name><surname>Perero</surname><given-names>S</given-names></name><name><surname>Percivalle</surname><given-names>E</given-names></name><name><surname>Nepita</surname><given-names>EV</given-names></name><name><surname>Ferraris</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Virucidal effect against coronavirus SARS-CoV-2 of a silver nanocluster/silica composite sputtered coating</article-title><source>Open Ceram</source><year>2020</year><volume>1</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XisFegsL3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.oceram.2020.100006</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Balaji</surname><given-names>AB</given-names></name><name><surname>Pakalapati</surname><given-names>H</given-names></name><name><surname>Khalid</surname><given-names>M</given-names></name><name><surname>Walvekar</surname><given-names>R</given-names></name><name><surname>Siddiqui</surname><given-names>H</given-names></name></person-group><source>Natural and synthetic biocompatible and biodegradable polymers. Biodegradable and biocompatible polymer composites</source><year>2018</year><publisher-loc>Duxford</publisher-loc><publisher-name>Woodhead Publishing</publisher-name><fpage>3</fpage><lpage>32</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-08-100970-3.00001-8</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bandodkar</surname><given-names>AJ</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Non-invasive wearable electrochemical sensors: a review</article-title><source>Trends Biotechnol</source><year>2014</year><volume>32</volume><issue>7</issue><fpage>363</fpage><lpage>371</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXotlyjsrw%3D</pub-id><pub-id pub-id-type=\"pmid\">24853270</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2014.04.005</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barnett</surname><given-names>JM</given-names></name><name><surname>Monnier</surname><given-names>BM</given-names></name><name><surname>Tyler</surname><given-names>S</given-names></name><name><surname>West</surname><given-names>D</given-names></name><name><surname>Ballantine-Dykes</surname><given-names>H</given-names></name><name><surname>Regan</surname><given-names>E</given-names></name><name><surname>Wraith</surname><given-names>P</given-names></name><name><surname>Kiely</surname><given-names>J</given-names></name><name><surname>Luxton</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Initial trail results of a magnetic biosensor for the rapid detection of Porcine Reproductive and Respiratory Virus (PRRSV) infection</article-title><source>Sens Bio-Sens Res</source><year>2019</year><volume>27</volume><pub-id pub-id-type=\"doi\">10.1016/j.sbsr.2019.100315</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Barrett</surname><given-names>ADT</given-names></name><name><surname>Titball</surname><given-names>RW</given-names></name><name><surname>MacAry</surname><given-names>PA</given-names></name><name><surname>Rupp</surname><given-names>RE</given-names></name><name><surname>von Messling</surname><given-names>V</given-names></name><name><surname>Walker</surname><given-names>DH</given-names></name><name><surname>Fanget</surname><given-names>NVJ</given-names></name></person-group><article-title xml:lang=\"en\">The rapid progress in COVID vaccine development and implementation</article-title><source>NPJ Vaccines</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>20</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjsFCgsL0%3D</pub-id><pub-id pub-id-type=\"pmid\">35145102</pub-id><pub-id pub-id-type=\"pmcid\">8831501</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-022-00442-8</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"other\">Bassas-Galia M, Follonier, S, Pusnik M, Zinn M (2017) Natural polymers: a source of inspiration. In Bioresorbable polymers for biomedical applications. Elsevier: pp 31–64</mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Benzigar</surname><given-names>MR</given-names></name><name><surname>Bhattacharjee</surname><given-names>R</given-names></name><name><surname>Baharfar</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Current methods for diagnosis of human coronaviruses: pros and cons</article-title><source>Anal Bioanal Chem</source><year>2021</year><volume>413</volume><issue>9</issue><fpage>2311</fpage><lpage>2330</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitl2qs7%2FE</pub-id><pub-id pub-id-type=\"pmid\">33219449</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00216-020-03046-0</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bernheim</surname><given-names>A</given-names></name><name><surname>Mei</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Fayad</surname><given-names>ZA</given-names></name><name><surname>Zhang</surname><given-names>N</given-names></name><name><surname>Diao</surname><given-names>K</given-names></name><name><surname>Lin</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Chest CT findings in coronavirus disease-19 (COVID-19): relationship to duration of infection</article-title><source>Radiology</source><year>2020</year><volume>295</volume><issue>3</issue><fpage>685</fpage><lpage>691</lpage><pub-id pub-id-type=\"doi\">10.1148/radiol.2020200463</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Boopathi</surname><given-names>S</given-names></name><name><surname>Poma</surname><given-names>AB</given-names></name><name><surname>Kolandaivel</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Novel 2019 coronavirus structure, mechanism of action, antiviral drug promises and rule out against its treatment</article-title><source>J Biomol Struct Dyn</source><year>2021</year><volume>39</volume><issue>9</issue><fpage>3409</fpage><lpage>3418</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpvVSgtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">32306836</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bos</surname><given-names>R</given-names></name><name><surname>Rutten</surname><given-names>L</given-names></name><name><surname>van der Lubbe</surname><given-names>JE</given-names></name><name><surname>Bakkers</surname><given-names>MJ</given-names></name><name><surname>Hardenberg</surname><given-names>G</given-names></name><name><surname>Wegmann</surname><given-names>F</given-names></name><name><surname>Zuijdgeest</surname><given-names>D</given-names></name><name><surname>de Wilde</surname><given-names>AH</given-names></name><name><surname>Koornneef</surname><given-names>A</given-names></name><name><surname>Verwilligen</surname><given-names>A</given-names></name><name><surname>van Manen</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Ad26 vector-based COVID-19 vaccine encoding a prefusion-stabilized SARS-CoV-2 Spike immunogen induces potent humoral and cellular immune responses</article-title><source>npj Vaccines</source><year>2020</year><volume>5</volume><issue>1</issue><fpage>91</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFKntr7O</pub-id><pub-id pub-id-type=\"pmid\">33083026</pub-id><pub-id pub-id-type=\"pmcid\">7522255</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41541-020-00243-x</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bozzuto</surname><given-names>G</given-names></name><name><surname>Molinari</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Liposomes as nanomedical devices</article-title><source>Int J Nanomed</source><year>2015</year><volume>10</volume><fpage>975</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXktl2qs7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJN.S68861</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Mochalski</surname><given-names>P</given-names></name><name><surname>Ruzsanyi</surname><given-names>V</given-names></name><name><surname>Amann</surname><given-names>A</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Hybrid volatolomics and disease detection</article-title><source>Angew Chem Int Ed</source><year>2015</year><volume>54</volume><issue>38</issue><fpage>11036</fpage><lpage>11048</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1KrtrzJ</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201500153</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Vishinkin</surname><given-names>R</given-names></name><name><surname>Barash</surname><given-names>O</given-names></name><name><surname>Nakhleh</surname><given-names>MK</given-names></name><name><surname>Haick</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Synergy between nanomaterials and volatile organic compounds for non-invasive medical evaluation</article-title><source>Chem Soc Rev</source><year>2018</year><volume>47</volume><issue>13</issue><fpage>4781</fpage><lpage>4859</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhtFSgtbvK</pub-id><pub-id pub-id-type=\"pmid\">29888356</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8CS00317C</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"other\">Calucho E (2020) Lateral flow devices for COVID-19-related biomarkers. Biosens Pandemics. Vol. 29</mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Carter</surname><given-names>LJ</given-names></name><name><surname>Garner</surname><given-names>LV</given-names></name><name><surname>Smoot</surname><given-names>JW</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Zhou</surname><given-names>Q</given-names></name><name><surname>Saveson</surname><given-names>CJ</given-names></name><name><surname>Sasso</surname><given-names>JM</given-names></name><name><surname>Gregg</surname><given-names>AC</given-names></name><name><surname>Soares</surname><given-names>DJ</given-names></name><name><surname>Beskid</surname><given-names>TR</given-names></name></person-group><source>Assay techniques and test development for COVID-19 diagnosis</source><year>2020</year><publisher-loc>Washington</publisher-loc><publisher-name>ACS Publications</publisher-name><pub-id pub-id-type=\"doi\">10.1021/acscentsci.0c00501</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chakravarty</surname><given-names>M</given-names></name><name><surname>Vora</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based antiviral therapeutics</article-title><source>Drug Deliv Transl Res</source><year>2021</year><volume>11</volume><issue>3</issue><fpage>748</fpage><lpage>787</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Sgs7%2FN</pub-id><pub-id pub-id-type=\"pmid\">32748035</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13346-020-00818-0</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chamundeeswari</surname><given-names>M</given-names></name><name><surname>Jeslin</surname><given-names>J</given-names></name><name><surname>Verma</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Nanocarriers for drug delivery applications</article-title><source>Environ Chem Lett</source><year>2019</year><volume>17</volume><issue>2</issue><fpage>849</fpage><lpage>865</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXit12ltb7I</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10311-018-00841-1</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"other\">Chandler-Brown D, Bueno AM, Atay O, Tsao DS (2020) A highly scalable and rapidly deployable RNA extraction-free COVID-19 assay by quantitative Sanger sequencing. Biorxiv</mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Yadav</surname><given-names>N</given-names></name><name><surname>Rai</surname><given-names>NK</given-names></name><name><surname>Ellah</surname><given-names>NHA</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name><name><surname>Rehan</surname><given-names>IF</given-names></name><name><surname>Marraiki</surname><given-names>N</given-names></name><name><surname>Batiha</surname><given-names>GES</given-names></name><name><surname>Hetta</surname><given-names>HF</given-names></name><name><surname>Singh</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pleurotus sajor-caju-mediated synthesis of silver and gold nanoparticles active against colon cancer cell lines: a new era of herbonanoceutics</article-title><source>Molecules</source><year>2020</year><volume>25</volume><issue>13</issue><fpage>3091</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsV2ksr3K</pub-id><pub-id pub-id-type=\"pmid\">32645899</pub-id><pub-id pub-id-type=\"pmcid\">7412267</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules25133091</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chaudhari</surname><given-names>SP</given-names></name><name><surname>Handge</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Formulation, development and evaluation of lopinavir loaded polymeric micelles</article-title><source>J Sci Technol</source><year>2020</year><volume>5</volume><fpage>173</fpage><lpage>187</lpage></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>HW</given-names></name><name><surname>Huang</surname><given-names>CY</given-names></name><name><surname>Lin</surname><given-names>SY</given-names></name><name><surname>Fang</surname><given-names>ZS</given-names></name><name><surname>Hsu</surname><given-names>CH</given-names></name><name><surname>Lin</surname><given-names>JC</given-names></name><name><surname>Chen</surname><given-names>YI</given-names></name><name><surname>Yao</surname><given-names>BY</given-names></name><name><surname>Hu</surname><given-names>CMJ</given-names></name></person-group><article-title xml:lang=\"en\">Synthetic virus-like particles prepared via protein corona formation enable effective vaccination in an avian model of coronavirus infection</article-title><source>Biomaterials</source><year>2016</year><volume>106</volume><fpage>111</fpage><lpage>118</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtlCqsLvN</pub-id><pub-id pub-id-type=\"pmid\">27552321</pub-id><pub-id pub-id-type=\"pmcid\">7112462</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biomaterials.2016.08.018</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Xu</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral nanoagents: more attention and effort needed?</article-title><source>Nano Today</source><year>2020</year><volume>35</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjtbrP</pub-id><pub-id pub-id-type=\"pmid\">32963581</pub-id><pub-id pub-id-type=\"pmcid\">7498411</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nantod.2020.100976</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Zhai</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>H</given-names></name><name><surname>Bian</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>P</given-names></name><name><surname>Yu</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Rapid and sensitive detection of anti-SARS-CoV-2 IgG, using lanthanide-doped nanoparticles-based lateral flow immunoassay</article-title><source>Anal Chem</source><year>2020</year><volume>92</volume><issue>10</issue><fpage>7226</fpage><lpage>7231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsleju7o%3D</pub-id><pub-id pub-id-type=\"pmid\">32323974</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.analchem.0c00784</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheng</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>Z</given-names></name><name><surname>Ma</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers as drug carriers: applications in different routes of drug administration</article-title><source>J Pharm Sci</source><year>2008</year><volume>97</volume><issue>1</issue><fpage>123</fpage><lpage>143</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXkslOrtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">17721949</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.21079</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cho</surname><given-names>H</given-names></name><name><surname>Lai</surname><given-names>TC</given-names></name><name><surname>Tomoda</surname><given-names>K</given-names></name><name><surname>Kwon</surname><given-names>GS</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles for multi-drug delivery in cancer</article-title><source>AAPS Pharm Sci Tech</source><year>2015</year><volume>16</volume><issue>1</issue><fpage>10</fpage><lpage>20</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitFait7nO</pub-id><pub-id pub-id-type=\"doi\">10.1208/s12249-014-0251-3</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choudhury</surname><given-names>KR</given-names></name><name><surname>Sahoo</surname><given-names>Y</given-names></name><name><surname>Ohulchanskyy</surname><given-names>T</given-names></name><name><surname>Prasad</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Efficient photoconductive devices at infrared wavelengths using quantum dot-polymer nanocomposites</article-title><source>Appl Phys Lett</source><year>2005</year><volume>87</volume><issue>7</issue><pub-id pub-id-type=\"doi\">10.1063/1.2011768</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Coleman</surname><given-names>CM</given-names></name><name><surname>Liu</surname><given-names>YV</given-names></name><name><surname>Mu</surname><given-names>H</given-names></name><name><surname>Taylor</surname><given-names>JK</given-names></name><name><surname>Massare</surname><given-names>M</given-names></name><name><surname>Flyer</surname><given-names>DC</given-names></name><name><surname>Glenn</surname><given-names>GM</given-names></name><name><surname>Smith</surname><given-names>GE</given-names></name><name><surname>Frieman</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">Purified coronavirus spike protein nanoparticles induce coronavirus neutralizing antibodies in mice</article-title><source>Vaccine</source><year>2014</year><volume>32</volume><issue>26</issue><fpage>3169</fpage><lpage>3174</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXnsVWltbc%3D</pub-id><pub-id pub-id-type=\"pmid\">24736006</pub-id><pub-id pub-id-type=\"pmcid\">4058772</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2014.04.016</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Comentale</surname><given-names>G</given-names></name><name><surname>Manzo</surname><given-names>R</given-names></name><name><surname>Pilato</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Sars-Cov-2 interference in HEME production: is it the time for an early predictive biomarker?</article-title><source>J Mol Med</source><year>2020</year><volume>98</volume><issue>8</issue><fpage>1053</fpage><lpage>1054</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtleks7bO</pub-id><pub-id pub-id-type=\"pmid\">32601795</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00109-020-01945-4</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Corman</surname><given-names>VM</given-names></name><name><surname>Landt</surname><given-names>O</given-names></name><name><surname>Kaiser</surname><given-names>M</given-names></name><name><surname>Molenkamp</surname><given-names>R</given-names></name><name><surname>Meijer</surname><given-names>A</given-names></name><name><surname>Chu</surname><given-names>DK</given-names></name><name><surname>Bleicker</surname><given-names>T</given-names></name><name><surname>Brünink</surname><given-names>S</given-names></name><name><surname>Schneider</surname><given-names>J</given-names></name><name><surname>Schmidt</surname><given-names>ML</given-names></name></person-group><article-title xml:lang=\"en\">Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR</article-title><source>Eurosurveillance</source><year>2020</year><volume>25</volume><issue>3</issue><fpage>2000045</fpage><pub-id pub-id-type=\"pmid\">31992387</pub-id><pub-id pub-id-type=\"pmcid\">6988269</pub-id><pub-id pub-id-type=\"doi\">10.2807/1560-7917.ES.2020.25.3.2000045</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Craw</surname><given-names>P</given-names></name><name><surname>Balachandran</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Isothermal nucleic acid amplification technologies for point-of-care diagnostics: a critical review</article-title><source>Lab Chip</source><year>2012</year><volume>12</volume><issue>14</issue><fpage>2469</fpage><lpage>2486</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XovVylt7k%3D</pub-id><pub-id pub-id-type=\"pmid\">22592150</pub-id><pub-id pub-id-type=\"doi\">10.1039/c2lc40100b</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Daemi</surname><given-names>HB</given-names></name><name><surname>Kulyar</surname><given-names>MF</given-names></name><name><surname>He</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Karimpour</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>X</given-names></name><name><surname>Zou</surname><given-names>Z</given-names></name><name><surname>Jin</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Progression and trends in virus from influenza A to COVID-19: an overview of recent studies</article-title><source>Viruses</source><year>2021</year><volume>13</volume><issue>6</issue><fpage>1145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1OnurrF</pub-id><pub-id pub-id-type=\"pmid\">34203647</pub-id><pub-id pub-id-type=\"pmcid\">8232279</pub-id><pub-id pub-id-type=\"doi\">10.3390/v13061145</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>de Ribeiro</surname><given-names>LNM</given-names></name><name><surname>Fonseca</surname><given-names>BB</given-names></name></person-group><article-title xml:lang=\"en\">The role of pharmaceutical nanotechnology in the time of COVID-19 pandemic</article-title><source>Future Microbiol</source><year>2020</year><volume>15</volume><issue>16</issue><fpage>1571</fpage><lpage>1582</lpage><pub-id pub-id-type=\"doi\">10.2217/fmb-2020-0118</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Wit</surname><given-names>E</given-names></name><name><surname>Feldmann</surname><given-names>F</given-names></name><name><surname>Cronin</surname><given-names>J</given-names></name><name><surname>Jordan</surname><given-names>R</given-names></name><name><surname>Okumura</surname><given-names>A</given-names></name><name><surname>Thomas</surname><given-names>T</given-names></name><name><surname>Scott</surname><given-names>D</given-names></name><name><surname>Cihlar</surname><given-names>T</given-names></name><name><surname>Feldmann</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Prophylactic and therapeutic remdesivir (GS-5734) treatment in the rhesus macaque model of MERS-CoV infection</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>12</issue><fpage>6771</fpage><lpage>6776</lpage><pub-id pub-id-type=\"pmid\">32054787</pub-id><pub-id pub-id-type=\"pmcid\">7104368</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1922083117</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Deeks</surname><given-names>JJ</given-names></name><name><surname>Dinnes</surname><given-names>J</given-names></name><name><surname>Takwoingi</surname><given-names>Y</given-names></name><name><surname>Davenport</surname><given-names>C</given-names></name><name><surname>Spijker</surname><given-names>R</given-names></name><name><surname>Taylor-Phillips</surname><given-names>S</given-names></name><name><surname>Adriano</surname><given-names>A</given-names></name><name><surname>Beese</surname><given-names>S</given-names></name><name><surname>Dretzke</surname><given-names>J</given-names></name><name><surname>di Ruffano</surname><given-names>LF</given-names></name></person-group><article-title xml:lang=\"en\">Antibody tests for identification of current and past infection with SARS-CoV-2</article-title><source>Cochrane Database Syst Rev</source><year>2020</year><volume>2020</volume><fpage>CD013652</fpage><pub-id pub-id-type=\"pmcid\">7387103</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dincer</surname><given-names>C</given-names></name><name><surname>Bruch</surname><given-names>R</given-names></name><name><surname>Kling</surname><given-names>A</given-names></name><name><surname>Dittrich</surname><given-names>PS</given-names></name><name><surname>Urban</surname><given-names>GA</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed point-of-care testing–xPOCT</article-title><source>Trends Biotechnol</source><year>2017</year><volume>35</volume><issue>8</issue><fpage>728</fpage><lpage>742</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmtFGns70%3D</pub-id><pub-id pub-id-type=\"pmid\">28456344</pub-id><pub-id pub-id-type=\"pmcid\">5538621</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tibtech.2017.03.013</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Dong</surname><given-names>X</given-names></name><name><surname>Moyer</surname><given-names>MM</given-names></name><name><surname>Yang</surname><given-names>F</given-names></name><name><surname>Sun</surname><given-names>YP</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots’ antiviral functions against noroviruses</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>10</lpage></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Drobysh</surname><given-names>M</given-names></name><name><surname>Ramanaviciene</surname><given-names>A</given-names></name><name><surname>Viter</surname><given-names>R</given-names></name><name><surname>Chen</surname><given-names>CF</given-names></name><name><surname>Samukaite-Bubniene</surname><given-names>U</given-names></name><name><surname>Ratautaite</surname><given-names>V</given-names></name><name><surname>Ramanavicius</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Biosensors for the determination of SARS-CoV-2 virus and diagnosis of COVID-19 infection</article-title><source>Int J Mol Sci</source><year>2022</year><volume>23</volume><issue>2</issue><fpage>666</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitFertr4%3D</pub-id><pub-id pub-id-type=\"pmid\">35054850</pub-id><pub-id pub-id-type=\"pmcid\">8776074</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms23020666</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Liang</surname><given-names>J</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Liu</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Carbon dots as inhibitors of virus by activation of type I interferon response</article-title><source>Carbon</source><year>2016</year><volume>110</volume><fpage>278</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFCmtLnF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbon.2016.09.032</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ella</surname><given-names>R</given-names></name><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>Blackwelder</surname><given-names>W</given-names></name><name><surname>Potdar</surname><given-names>V</given-names></name><name><surname>Yadav</surname><given-names>P</given-names></name><name><surname>Sarangi</surname><given-names>V</given-names></name><name><surname>Aileni</surname><given-names>VK</given-names></name><name><surname>Kanungo</surname><given-names>S</given-names></name><name><surname>Rai</surname><given-names>S</given-names></name><name><surname>Reddy</surname><given-names>P</given-names></name><name><surname>Verma</surname><given-names>S</given-names></name><name><surname>Singh</surname><given-names>C</given-names></name><name><surname>Redkar</surname><given-names>S</given-names></name><name><surname>Mohapatra</surname><given-names>S</given-names></name><name><surname>Pandey</surname><given-names>A</given-names></name><name><surname>Ranganadin</surname><given-names>P</given-names></name><name><surname>Gumashta</surname><given-names>R</given-names></name><name><surname>Multani</surname><given-names>M</given-names></name><name><surname>Mohammad</surname><given-names>S</given-names></name><name><surname>Bhatt</surname><given-names>P</given-names></name><name><surname>Kumari</surname><given-names>L</given-names></name><name><surname>Sapkal</surname><given-names>G</given-names></name><name><surname>Gupta</surname><given-names>N</given-names></name><name><surname>Abraham</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>S</given-names></name><name><surname>Prasad</surname><given-names>S</given-names></name><name><surname>Bhargava</surname><given-names>B</given-names></name><name><surname>Ella</surname><given-names>K</given-names></name><name><surname>Vadrevu</surname><given-names>KM</given-names></name><collab>COVAXIN Study Group</collab></person-group><article-title xml:lang=\"en\">Efficacy, safety, and lot-to-lot immunogenicity of an inactivated SARS-CoV-2 vaccine (BBV152): interim results of a randomised, double-blind, controlled, phase 3 trial</article-title><source>Lancet (London, England)</source><year>2021</year><volume>398</volume><issue>10317</issue><fpage>2173</fpage><lpage>2184</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1eqsrfE</pub-id><pub-id pub-id-type=\"pmid\">34774196</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(21)02000-6</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Esbin</surname><given-names>MN</given-names></name><name><surname>Whitney</surname><given-names>ON</given-names></name><name><surname>Chong</surname><given-names>S</given-names></name><name><surname>Maurer</surname><given-names>A</given-names></name><name><surname>Darzacq</surname><given-names>X</given-names></name><name><surname>Tjian</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Overcoming the bottleneck to widespread testing: a rapid review of nucleic acid testing approaches for COVID-19 detection</article-title><source>RNA</source><year>2020</year><volume>26</volume><issue>7</issue><fpage>771</fpage><lpage>783</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Klt7rP</pub-id><pub-id pub-id-type=\"pmid\">32358057</pub-id><pub-id pub-id-type=\"pmcid\">7297120</pub-id><pub-id pub-id-type=\"doi\">10.1261/rna.076232.120</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Flühmann</surname><given-names>B</given-names></name><name><surname>Ntai</surname><given-names>I</given-names></name><name><surname>Borchard</surname><given-names>G</given-names></name><name><surname>Simoens</surname><given-names>S</given-names></name><name><surname>Mühlebach</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicines: the magic bullets reaching their target?</article-title><source>Eur J Pharm Sci</source><year>2019</year><volume>128</volume><fpage>73</fpage><lpage>80</lpage><pub-id pub-id-type=\"pmid\">30465818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2018.11.019</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Franklyne</surname><given-names>JS</given-names></name><name><surname>Gopinath</surname><given-names>PM</given-names></name><name><surname>Mukherjee</surname><given-names>A</given-names></name><name><surname>Chandrasekaran</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Nanoemulsions: the rising star of antiviral therapeutics and nanodelivery system—Current status and prospects</article-title><source>Curr Opin Colloid Interface Sci</source><year>2021</year><volume>54</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXptlansro%3D</pub-id><pub-id pub-id-type=\"pmid\">33814954</pub-id><pub-id pub-id-type=\"pmcid\">8007535</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cocis.2021.101458</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Freitas</surname><given-names>RA</given-names><suffix>Jr</suffix></name></person-group><article-title xml:lang=\"en\">What is nanomedicine? Nanomedicine: nanotechnology</article-title><source>Biology Med</source><year>2005</year><volume>1</volume><issue>1</issue><fpage>2</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXlvVSrurs%3D</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Garg</surname><given-names>B</given-names></name><name><surname>Beg</surname><given-names>S</given-names></name><name><surname>Kumar</surname><given-names>R</given-names></name><name><surname>Katara</surname><given-names>O</given-names></name><name><surname>Singh</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructured lipidic carriers of lopinavir for effective management of HIV-associated neurocognitive disorder</article-title><source>J Drug Deliv Sci Technol</source><year>2019</year><volume>53</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhs1artbfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2019.101220</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Goodwin</surname><given-names>TJ</given-names></name><name><surname>Huang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of phosphorylated adjuvants co-encapsulated with a model cancer peptide antigen for the treatment of colorectal cancer and liver metastasis</article-title><source>Vaccine</source><year>2017</year><volume>35</volume><issue>19</issue><fpage>2550</fpage><lpage>2557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsFagt78%3D</pub-id><pub-id pub-id-type=\"pmid\">28385609</pub-id><pub-id pub-id-type=\"pmcid\">5448460</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2017.03.067</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gregory</surname><given-names>AE</given-names></name><name><surname>Titball</surname><given-names>R</given-names></name><name><surname>Williamson</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Vaccine delivery using nanoparticles</article-title><source>Front Incell Infect Microbiol</source><year>2013</year><volume>3</volume><fpage>13</fpage></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Habibi</surname><given-names>R</given-names></name><name><surname>Burci</surname><given-names>GL</given-names></name><name><surname>De Campos</surname><given-names>TC</given-names></name><name><surname>Chirwa</surname><given-names>D</given-names></name><name><surname>Cinà</surname><given-names>M</given-names></name><name><surname>Dagron</surname><given-names>S</given-names></name><name><surname>Eccleston-Turner</surname><given-names>M</given-names></name><name><surname>Forman</surname><given-names>L</given-names></name><name><surname>Gostin</surname><given-names>LO</given-names></name><name><surname>Mier</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">Do not violate the international health regulations during the COVID-19 outbreak</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10225</issue><fpage>664</fpage><lpage>666</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXksVaisLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30373-1</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halbus</surname><given-names>AF</given-names></name><name><surname>Horozov</surname><given-names>TS</given-names></name><name><surname>Paunov</surname><given-names>VN</given-names></name></person-group><article-title xml:lang=\"en\">Colloid particle formulations for antimicrobial applications</article-title><source>Adv Coll Interface Sci</source><year>2017</year><volume>249</volume><fpage>134</fpage><lpage>148</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXotVarsrY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2017.05.012</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>HS</given-names></name><name><surname>Niemeyer</surname><given-names>E</given-names></name><name><surname>Huang</surname><given-names>Y</given-names></name><name><surname>Kamoun</surname><given-names>WS</given-names></name><name><surname>Martin</surname><given-names>JD</given-names></name><name><surname>Bhaumik</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Roberge</surname><given-names>S</given-names></name><name><surname>Cui</surname><given-names>J</given-names></name><name><surname>Martin</surname><given-names>MR</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dot/antibody conjugates for in vivo cytometric imaging in mice</article-title><source>Proc Natl Acad Sci</source><year>2015</year><volume>112</volume><issue>5</issue><fpage>1350</fpage><lpage>1355</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtFKhtr8%3D</pub-id><pub-id pub-id-type=\"pmid\">25605916</pub-id><pub-id pub-id-type=\"pmcid\">4321304</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1421632111</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Han</surname><given-names>GR</given-names></name><name><surname>Ki</surname><given-names>H</given-names></name><name><surname>Kim</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Automated, universal, and mass-producible paper-based lateral flow biosensing platform for high-performance point-of-care testing</article-title><source>ACS Appl Mater Interfaces</source><year>2019</year><volume>12</volume><issue>1</issue><fpage>1885</fpage><lpage>1894</lpage><pub-id pub-id-type=\"pmid\">31813220</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.9b17888</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hardick</surname><given-names>J</given-names></name><name><surname>Metzgar</surname><given-names>D</given-names></name><name><surname>Risen</surname><given-names>L</given-names></name><name><surname>Myers</surname><given-names>C</given-names></name><name><surname>Balansay</surname><given-names>M</given-names></name><name><surname>Malcom</surname><given-names>T</given-names></name><name><surname>Rothman</surname><given-names>R</given-names></name><name><surname>Gaydos</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Initial performance evaluation of a spotted array Mobile Analysis Platform (MAP) for the detection of influenza A/B, RSV, and MERS coronavirus</article-title><source>Diagn Microbiol Infect Dis</source><year>2018</year><volume>91</volume><issue>3</issue><fpage>245</fpage><lpage>247</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXlt1SisLg%3D</pub-id><pub-id pub-id-type=\"pmid\">29550057</pub-id><pub-id pub-id-type=\"pmcid\">6013072</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.diagmicrobio.2018.02.011</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Harvey</surname><given-names>CJ</given-names></name><name><surname>O’Regan</surname><given-names>DP</given-names></name></person-group><article-title xml:lang=\"en\">Radiological investigations and applications</article-title><source>Med MRCP</source><year>2020</year><volume>17</volume><fpage>111</fpage><pub-id pub-id-type=\"doi\">10.1093/med/9780198779506.003.0005</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Haslberger</surname><given-names>A</given-names></name><name><surname>Jacob</surname><given-names>U</given-names></name><name><surname>Hippe</surname><given-names>B</given-names></name><name><surname>Karlic</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Mechanisms of selected functional foods against viral infections with a view on COVID-19: Mini review</article-title><source>Funct Foods Health Dis</source><year>2020</year><volume>10</volume><issue>5</issue><fpage>195</fpage><lpage>209</lpage><pub-id pub-id-type=\"doi\">10.31989/ffhd.v10i5.707</pub-id></mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Heymann</surname><given-names>DL</given-names></name><name><surname>Shindo</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19: what is next for public health?</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>542</fpage><lpage>545</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisbs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30374-3</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hofmann</surname><given-names>MA</given-names></name><name><surname>Renzullo</surname><given-names>S</given-names></name><name><surname>Mader</surname><given-names>M</given-names></name><name><surname>Chaignat</surname><given-names>V</given-names></name><name><surname>Worwa</surname><given-names>G</given-names></name><name><surname>Thuer</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Genetic characterization of toggenburg orbivirus a new bluetongue virus, from goats. Switzerland</article-title><source>Emerg Infect Dis</source><year>2008</year><volume>14</volume><issue>12</issue><fpage>1855</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXjvVKltA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19046507</pub-id><pub-id pub-id-type=\"pmcid\">2634640</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid1412.080818</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Fan</surname><given-names>G</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Gu</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>497</fpage><lpage>506</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1Kqu7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30183-5</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Huang</surname><given-names>Z</given-names></name><name><surname>Jiang</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Du</surname><given-names>T</given-names></name><name><surname>Yi</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>Z</given-names></name><name><surname>Fan</surname><given-names>S</given-names></name><name><surname>Liao</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Jiang</surname><given-names>G</given-names></name><name><surname>Tang</surname><given-names>D</given-names></name><name><surname>Ye</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Ma</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 inactivated vaccine (Vero cells) shows good safety in repeated administration toxicity test of Sprague Dawley rats</article-title><source>Food Chem Toxicol</source><year>2021</year><volume>152</volume><fpage>112239</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXpvVSmtLY%3D</pub-id><pub-id pub-id-type=\"pmid\">33901607</pub-id><pub-id pub-id-type=\"pmcid\">8064818</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fct.2021.112239</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Iannazzo</surname><given-names>D</given-names></name><name><surname>Pistone</surname><given-names>A</given-names></name><name><surname>Ferro</surname><given-names>S</given-names></name><name><surname>De Luca</surname><given-names>L</given-names></name><name><surname>Monforte</surname><given-names>AM</given-names></name><name><surname>Romeo</surname><given-names>R</given-names></name><name><surname>Buemi</surname><given-names>MR</given-names></name><name><surname>Pannecouque</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Graphene quantum dots based systems as HIV inhibitors</article-title><source>Bioconjug Chem</source><year>2018</year><volume>29</volume><issue>9</issue><fpage>3084</fpage><lpage>3093</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsV2lsLjK</pub-id><pub-id pub-id-type=\"pmid\">30106563</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.bioconjchem.8b00448</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Kaneko</surname><given-names>K</given-names></name><name><surname>Ishihara</surname><given-names>T</given-names></name><name><surname>Mizushima</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Development of biodegradable nanoparticles for liver-specific ribavirin delivery</article-title><source>J Pharm Sci</source><year>2014</year><volume>103</volume><issue>12</issue><fpage>4005</fpage><lpage>4011</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVaktrjF</pub-id><pub-id pub-id-type=\"pmid\">25335768</pub-id><pub-id pub-id-type=\"doi\">10.1002/jps.24219</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Islam</surname><given-names>A</given-names></name><name><surname>Ahsan</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Plausible approach for rapid detection of SARS-CoV-2 virus by magnetic nanoparticle based biosensors</article-title><source>Am J Nanosci</source><year>2020</year><volume>6</volume><fpage>6</fpage><pub-id pub-id-type=\"doi\">10.11648/j.ajn.20200602.11</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Itani</surname><given-names>R</given-names></name><name><surname>Tobaiqy</surname><given-names>M</given-names></name><name><surname>Al Faraj</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Optimizing use of theranostic nanoparticles as a life-saving strategy for treating COVID-19 patients</article-title><source>Theranostics</source><year>2020</year><volume>10</volume><issue>13</issue><fpage>5932</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFGgsrzI</pub-id><pub-id pub-id-type=\"pmid\">32483428</pub-id><pub-id pub-id-type=\"pmcid\">7254986</pub-id><pub-id pub-id-type=\"doi\">10.7150/thno.46691</pub-id></mixed-citation></ref><ref id=\"CR74\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jacob</surname><given-names>S</given-names></name><name><surname>Nair</surname><given-names>AB</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Emerging role of nanosuspensions in drug delivery systems</article-title><source>Biomater Res</source><year>2020</year><volume>24</volume><issue>1</issue><fpage>1</fpage><lpage>16</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlOnurs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s40824-020-0184-8</pub-id></mixed-citation></ref><ref id=\"CR75\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaimes</surname><given-names>JA</given-names></name><name><surname>Millet</surname><given-names>JK</given-names></name><name><surname>Stout</surname><given-names>AE</given-names></name><name><surname>André</surname><given-names>NM</given-names></name><name><surname>Whittaker</surname><given-names>GR</given-names></name></person-group><article-title xml:lang=\"en\">A tale of two viruses: the distinct spike glycoproteins of feline coronaviruses</article-title><source>Viruses</source><year>2020</year><volume>12</volume><issue>1</issue><fpage>83</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsVCquro%3D</pub-id><pub-id pub-id-type=\"pmid\">31936749</pub-id><pub-id pub-id-type=\"pmcid\">7019228</pub-id><pub-id pub-id-type=\"doi\">10.3390/v12010083</pub-id></mixed-citation></ref><ref id=\"CR76\"><mixed-citation publication-type=\"other\">Jan N, Madni A, Khan S, Shah H, Akram F, Khan A, Ertas D, Bostanudin MF, Contag CH, Ashammakhi N, Ertas YN (2022) Biomimetic cell membrane‐coated poly (lactic‐co‐glycolic acid) nanoparticles for biomedical applications. Bioeng Transl Med. p e10441</mixed-citation></ref><ref id=\"CR77\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Javan</surname><given-names>F</given-names></name><name><surname>Vatanara</surname><given-names>A</given-names></name><name><surname>Azadmanesh</surname><given-names>K</given-names></name><name><surname>Nabi-Meibodi</surname><given-names>M</given-names></name><name><surname>Shakouri</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Encapsulation of ritonavir in solid lipid nanoparticles: in-vitro anti-HIV-1 activity using lentiviral particles</article-title><source>J Pharm Pharmacol</source><year>2017</year><volume>69</volume><issue>8</issue><fpage>1002</fpage><lpage>1009</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXntVKhsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">28471000</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12737</pub-id></mixed-citation></ref><ref id=\"CR78\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jiang</surname><given-names>M</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Arasthfer</surname><given-names>A</given-names></name><name><surname>Fang</surname><given-names>W</given-names></name><name><surname>Ling</surname><given-names>L</given-names></name><name><surname>Fang</surname><given-names>H</given-names></name><name><surname>Daneshnia</surname><given-names>F</given-names></name><name><surname>Yu</surname><given-names>J</given-names></name><name><surname>Liao</surname><given-names>W</given-names></name><name><surname>Pei</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Development and validation of a apid, single-step reverse transcriptase loop-mediated isothermal amplification (RT-LAMP) system potentially to be used for reliable and high-throughput screening of COVID-19</article-title><source>Front Cell Infect Microbiol</source><year>2020</year><volume>10</volume><fpage>331</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaktbbE</pub-id><pub-id pub-id-type=\"pmid\">32626666</pub-id><pub-id pub-id-type=\"pmcid\">7313420</pub-id><pub-id pub-id-type=\"doi\">10.3389/fcimb.2020.00331</pub-id></mixed-citation></ref><ref id=\"CR79\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jones</surname><given-names>ST</given-names></name><name><surname>Cagno</surname><given-names>V</given-names></name><name><surname>Janeček</surname><given-names>M</given-names></name><name><surname>Ortiz</surname><given-names>D</given-names></name><name><surname>Gasilova</surname><given-names>N</given-names></name><name><surname>Piret</surname><given-names>J</given-names></name><name><surname>Gasbarri</surname><given-names>M</given-names></name><name><surname>Constant</surname><given-names>DA</given-names></name><name><surname>Han</surname><given-names>Y</given-names></name><name><surname>Vuković</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Modified cyclodextrins as broad-spectrum antivirals</article-title><source>Sci Adv</source><year>2020</year><volume>6</volume><issue>5</issue><fpage>eaax9318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGgsr%2FL</pub-id><pub-id pub-id-type=\"pmid\">32064341</pub-id><pub-id pub-id-type=\"pmcid\">6989148</pub-id><pub-id pub-id-type=\"doi\">10.1126/sciadv.aax9318</pub-id></mixed-citation></ref><ref id=\"CR80\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kalita</surname><given-names>MJ</given-names></name><name><surname>Dutta</surname><given-names>K</given-names></name><name><surname>Hazarika</surname><given-names>G</given-names></name><name><surname>Dutta</surname><given-names>R</given-names></name><name><surname>Kalita</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>PP</given-names></name><name><surname>Sarma</surname><given-names>MP</given-names></name><name><surname>Banu</surname><given-names>S</given-names></name><name><surname>Idris</surname><given-names>M</given-names></name><name><surname>Talukdar</surname><given-names>AJ</given-names></name></person-group><article-title xml:lang=\"en\">In-house reverse transcriptase polymerase chain reaction for detection of SARS-CoV-2 with increased sensitivity</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>12</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-97502-1</pub-id></mixed-citation></ref><ref id=\"CR81\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kandeel</surname><given-names>M</given-names></name><name><surname>Al-Taher</surname><given-names>A</given-names></name><name><surname>Park</surname><given-names>BK</given-names></name><name><surname>Kwon</surname><given-names>HJ</given-names></name><name><surname>Al-Nazawi</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">A pilot study of the antiviral activity of anionic and cationic polyamidoamine dendrimers against the Middle East respiratory syndrome coronavirus</article-title><source>J Med Virol</source><year>2020</year><volume>92</volume><issue>9</issue><fpage>1665</fpage><lpage>1670</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVyhsb8%3D</pub-id><pub-id pub-id-type=\"pmid\">32330296</pub-id><pub-id pub-id-type=\"pmcid\">7264540</pub-id><pub-id pub-id-type=\"doi\">10.1002/jmv.25928</pub-id></mixed-citation></ref><ref id=\"CR82\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Katata-Seru</surname><given-names>L</given-names></name><name><surname>Ojo</surname><given-names>BM</given-names></name><name><surname>Okubanjo</surname><given-names>O</given-names></name><name><surname>Soremekun</surname><given-names>R</given-names></name><name><surname>Aremu</surname><given-names>OS</given-names></name></person-group><article-title xml:lang=\"en\">Nanoformulated Eudragit lopinavir and preliminary release of its loaded suppositories</article-title><source>Heliyon</source><year>2020</year><volume>6</volume><issue>5</issue><pub-id pub-id-type=\"pmid\">32420478</pub-id><pub-id pub-id-type=\"pmcid\">7218025</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.heliyon.2020.e03890</pub-id></mixed-citation></ref><ref id=\"CR83\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaur</surname><given-names>I</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Jakhar</surname><given-names>D</given-names></name><name><surname>Das</surname><given-names>A</given-names></name><name><surname>Aradhya</surname><given-names>SS</given-names></name><name><surname>Sharma</surname><given-names>R</given-names></name><name><surname>Jindal</surname><given-names>V</given-names></name><name><surname>Mhatre</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease (COVID-19): An updated review based on current knowledge and existing literature for dermatologists</article-title><source>Dermatol Ther</source><year>2020</year><volume>33</volume><issue>4</issue><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXps12jsrs%3D</pub-id><pub-id pub-id-type=\"pmid\">32447820</pub-id><pub-id pub-id-type=\"pmcid\">7283871</pub-id><pub-id pub-id-type=\"doi\">10.1111/dth.13677</pub-id></mixed-citation></ref><ref id=\"CR84\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>AK</given-names></name><name><surname>Dhau</surname><given-names>JS</given-names></name><name><surname>Gohel</surname><given-names>H</given-names></name><name><surname>Mishra</surname><given-names>YK</given-names></name><name><surname>Kateb</surname><given-names>B</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Goswami</surname><given-names>DY</given-names></name></person-group><article-title xml:lang=\"en\">Electrochemical SARS-CoV-2 sensing at point-of-care and artificial intelligence for intelligent COVID-19 management</article-title><source>ACS Appl Bio Mater</source><year>2020</year><volume>3</volume><issue>11</issue><fpage>7306</fpage><lpage>7325</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGltr3I</pub-id><pub-id pub-id-type=\"pmid\">35019473</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsabm.0c01004</pub-id></mixed-citation></ref><ref id=\"CR85\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Jain</surname><given-names>K</given-names></name><name><surname>Jain</surname><given-names>NK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer as nanocarrier for drug delivery</article-title><source>Prog Polym Sci</source><year>2014</year><volume>39</volume><issue>2</issue><fpage>268</fpage><lpage>307</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1Wrur7O</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.progpolymsci.2013.07.005</pub-id></mixed-citation></ref><ref id=\"CR86\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kesharwani</surname><given-names>P</given-names></name><name><surname>Xie</surname><given-names>L</given-names></name><name><surname>Banerjee</surname><given-names>S</given-names></name><name><surname>Mao</surname><given-names>G</given-names></name><name><surname>Padhye</surname><given-names>S</given-names></name><name><surname>Sarkar</surname><given-names>FH</given-names></name><name><surname>Iyer</surname><given-names>AK</given-names></name></person-group><article-title xml:lang=\"en\">Hyaluronic acid-conjugated polyamidoamine dendrimers for targeted delivery of 3, 4-difluorobenzylidene curcumin to CD44 overexpressing pancreatic cancer cells</article-title><source>Colloids Surf, B</source><year>2015</year><volume>136</volume><fpage>413</fpage><lpage>423</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhs1aktr3N</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2015.09.043</pub-id></mixed-citation></ref><ref id=\"CR87\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalaj-Hedayati</surname><given-names>A</given-names></name><name><surname>Chua</surname><given-names>CLL</given-names></name><name><surname>Smooker</surname><given-names>P</given-names></name><name><surname>Lee</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Nanoparticles in influenza subunit vaccine development: immunogenicity enhancement</article-title><source>Influenza Other Respir Viruses</source><year>2020</year><volume>14</volume><issue>1</issue><fpage>92</fpage><lpage>101</lpage><pub-id pub-id-type=\"pmid\">31774251</pub-id><pub-id pub-id-type=\"doi\">10.1111/irv.12697</pub-id></mixed-citation></ref><ref id=\"CR88\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khan</surname><given-names>AA</given-names></name><name><surname>Mudassir</surname><given-names>J</given-names></name><name><surname>Akhtar</surname><given-names>S</given-names></name><name><surname>Murugaiyah</surname><given-names>V</given-names></name><name><surname>Darwis</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Freeze-dried lopinavir-loaded nanostructured lipid carriers for enhanced cellular uptake and bioavailability: statistical optimization, in vitro and in vivo evaluations</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>2</issue><fpage>97</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlWhtLrP</pub-id><pub-id pub-id-type=\"pmid\">30823545</pub-id><pub-id pub-id-type=\"pmcid\">6410192</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11020097</pub-id></mixed-citation></ref><ref id=\"CR89\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kobayashi</surname><given-names>K</given-names></name><name><surname>Wei</surname><given-names>J</given-names></name><name><surname>Iida</surname><given-names>R</given-names></name><name><surname>Ijiro</surname><given-names>K</given-names></name><name><surname>Niikura</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Surface engineering of nanoparticles for therapeutic applications</article-title><source>Polym J</source><year>2014</year><volume>46</volume><issue>8</issue><fpage>460</fpage><lpage>468</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1KksLbK</pub-id><pub-id pub-id-type=\"doi\">10.1038/pj.2014.40</pub-id></mixed-citation></ref><ref id=\"CR90\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Koshy</surname><given-names>ST</given-names></name><name><surname>Cheung</surname><given-names>AS</given-names></name><name><surname>Gu</surname><given-names>L</given-names></name><name><surname>Graveline</surname><given-names>AR</given-names></name><name><surname>Mooney</surname><given-names>DJ</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal delivery enhances immune activation by STING agonists for cancer immunotherapy</article-title><source>Adv Biosyst</source><year>2017</year><volume>1</volume><issue>1–2</issue><fpage>1600013</fpage><pub-id pub-id-type=\"pmid\">30258983</pub-id><pub-id pub-id-type=\"pmcid\">6152940</pub-id><pub-id pub-id-type=\"doi\">10.1002/adbi.201600013</pub-id></mixed-citation></ref><ref id=\"CR91\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kulthe</surname><given-names>SS</given-names></name><name><surname>Choudhari</surname><given-names>YM</given-names></name><name><surname>Inamdar</surname><given-names>NN</given-names></name><name><surname>Mourya</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Polymeric micelles: authoritative aspects for drug delivery</article-title><source>Des Monomers Polym</source><year>2012</year><volume>15</volume><issue>5</issue><fpage>465</fpage><lpage>521</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XhsVSqsL%2FP</pub-id><pub-id pub-id-type=\"doi\">10.1080/1385772X.2012.688328</pub-id></mixed-citation></ref><ref id=\"CR92\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>M</given-names></name><name><surname>Curtis</surname><given-names>A</given-names></name><name><surname>Hoskins</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanoparticle technologies in the combat against anti-microbial resistance</article-title><source>Pharmaceutics</source><year>2018</year><volume>10</volume><issue>1</issue><fpage>11</fpage><pub-id pub-id-type=\"pmid\">29342903</pub-id><pub-id pub-id-type=\"pmcid\">5874824</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics10010011</pub-id></mixed-citation></ref><ref id=\"CR93\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumar</surname><given-names>S</given-names></name><name><surname>Narayan</surname><given-names>R</given-names></name><name><surname>Ahammed</surname><given-names>V</given-names></name><name><surname>Nayak</surname><given-names>Y</given-names></name><name><surname>Naha</surname><given-names>A</given-names></name><name><surname>Nayak</surname><given-names>UY</given-names></name></person-group><article-title xml:lang=\"en\">Development of ritonavir solid lipid nanoparticles by Box Behnken design for intestinal lymphatic targeting</article-title><source>J Drug Deliv Sci Technol</source><year>2018</year><volume>44</volume><fpage>181</fpage><lpage>189</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXis1Clsg%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jddst.2017.12.014</pub-id></mixed-citation></ref><ref id=\"CR94\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight to ROS and apoptosis regulated cytotoxicity inferred by green synthesized CuO nanoparticles from Calotropis gigantea to embryonic zebrafish</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>17</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-017-16581-1</pub-id></mixed-citation></ref><ref id=\"CR95\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kumari</surname><given-names>P</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Pramanik</surname><given-names>N</given-names></name><name><surname>Nisha</surname><given-names>K</given-names></name><name><surname>Kumari</surname><given-names>K</given-names></name><name><surname>Soni</surname><given-names>N</given-names></name><name><surname>Mallick</surname><given-names>MA</given-names></name><name><surname>Verma</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Molecular insight to in vitro biocompatibility of phytofabricated copper oxide nanoparticles with human embryonic kidney cells</article-title><source>Nanomedicine</source><year>2018</year><volume>13</volume><issue>19</issue><fpage>2415</fpage><lpage>2433</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitV2isL%2FN</pub-id><pub-id pub-id-type=\"pmid\">30251920</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2018-0175</pub-id></mixed-citation></ref><ref id=\"CR96\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kupferschmidt</surname><given-names>K</given-names></name><name><surname>Cohen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Race to find COVID-19 treatments accelerates</article-title><source>Am Assoc Adv Sci</source><year>2020</year><volume>367</volume><fpage>6485</fpage></mixed-citation></ref><ref id=\"CR97\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>S</given-names></name><name><surname>Ruktanonchai</surname><given-names>NW</given-names></name><name><surname>Zhou</surname><given-names>L</given-names></name><name><surname>Prosper</surname><given-names>O</given-names></name><name><surname>Luo</surname><given-names>W</given-names></name><name><surname>Floyd</surname><given-names>JR</given-names></name><name><surname>Wesolowski</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Du</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Effect of non-pharmaceutical interventions for containing the COVID-19 outbreak: an observational and modelling study</article-title><source>MedRxiv</source><year>2020</year><volume>202003</volume><issue>03</issue><fpage>20029843</fpage></mixed-citation></ref><ref id=\"CR98\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamb</surname><given-names>YN</given-names></name></person-group><article-title xml:lang=\"en\">BNT162b2 mRNA COVID-19 vaccine: first approval</article-title><source>Drugs</source><year>2021</year><volume>81</volume><fpage>495</fpage><lpage>501</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1yquro%3D</pub-id><pub-id pub-id-type=\"pmid\">33683637</pub-id><pub-id pub-id-type=\"pmcid\">7938284</pub-id><pub-id pub-id-type=\"doi\">10.1007/s40265-021-01480-7</pub-id></mixed-citation></ref><ref id=\"CR99\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lamote</surname><given-names>K</given-names></name><name><surname>Janssens</surname><given-names>E</given-names></name><name><surname>Schillebeeckx</surname><given-names>E</given-names></name><name><surname>Lapperre</surname><given-names>TS</given-names></name><name><surname>De Winter</surname><given-names>BY</given-names></name><name><surname>Van Meerbeeck</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">The scent of COVID-19: viral (semi-) volatiles as fast diagnostic biomarkers?</article-title><source>J Breath Res</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>042001</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1egsrvJ</pub-id><pub-id pub-id-type=\"pmid\">32599571</pub-id><pub-id pub-id-type=\"doi\">10.1088/1752-7163/aba105</pub-id></mixed-citation></ref><ref id=\"CR100\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Law</surname><given-names>JWF</given-names></name><name><surname>Ab Mutalib</surname><given-names>NS</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Lee</surname><given-names>LH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid methods for the detection of foodborne bacterial pathogens: principles, applications, advantages and limitations</article-title><source>Front Microbiol</source><year>2015</year><volume>5</volume><fpage>770</fpage><pub-id pub-id-type=\"pmid\">25628612</pub-id><pub-id pub-id-type=\"pmcid\">4290631</pub-id><pub-id pub-id-type=\"doi\">10.3389/fmicb.2014.00770</pub-id></mixed-citation></ref><ref id=\"CR101\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Le</surname><given-names>TT</given-names></name><name><surname>Andreadakis</surname><given-names>Z</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Román</surname><given-names>RG</given-names></name><name><surname>Tollefsen</surname><given-names>S</given-names></name><name><surname>Saville</surname><given-names>M</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><issue>5</issue><fpage>305</fpage><lpage>306</lpage><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00073-5</pub-id></mixed-citation></ref><ref id=\"CR102\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ledet</surname><given-names>G</given-names></name><name><surname>Mandal</surname><given-names>TK</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine: emerging therapeutics for the 21st century</article-title><source>US Pharm</source><year>2012</year><volume>37</volume><issue>3</issue><fpage>7</fpage><lpage>11</lpage></mixed-citation></ref><ref id=\"CR103\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lembo</surname><given-names>D</given-names></name><name><surname>Donalisio</surname><given-names>M</given-names></name><name><surname>Civra</surname><given-names>A</given-names></name><name><surname>Argenziano</surname><given-names>M</given-names></name><name><surname>Cavalli</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine formulations for the delivery of antiviral drugs: a promising solution for the treatment of viral infections</article-title><source>Expert Opin Drug Deliv</source><year>2018</year><volume>15</volume><issue>1</issue><fpage>93</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXht12ltr%2FO</pub-id><pub-id pub-id-type=\"pmid\">28749739</pub-id><pub-id pub-id-type=\"doi\">10.1080/17425247.2017.1360863</pub-id></mixed-citation></ref><ref id=\"CR104\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>SM</given-names></name><name><surname>Yu</surname><given-names>XH</given-names></name><name><surname>Tang</surname><given-names>SL</given-names></name><name><surname>Tang</surname><given-names>CK</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 (COVID-19): current status and future perspectives</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><issue>5</issue><fpage>105951</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSksLs%3D</pub-id><pub-id pub-id-type=\"pmid\">32234466</pub-id><pub-id pub-id-type=\"pmcid\">7139247</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105951</pub-id></mixed-citation></ref><ref id=\"CR105\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Geng</surname><given-names>M</given-names></name><name><surname>Peng</surname><given-names>Y</given-names></name><name><surname>Meng</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Molecular immune pathogenesis and diagnosis of COVID-19</article-title><source>J Pharm Anal</source><year>2020</year><volume>10</volume><issue>2</issue><fpage>102</fpage><lpage>108</lpage><pub-id pub-id-type=\"pmid\">32282863</pub-id><pub-id pub-id-type=\"pmcid\">7104082</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jpha.2020.03.001</pub-id></mixed-citation></ref><ref id=\"CR106\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>K</given-names></name><name><surname>Wu</surname><given-names>D</given-names></name><name><surname>Ren</surname><given-names>L</given-names></name><name><surname>Chu</surname><given-names>X</given-names></name><name><surname>Qin</surname><given-names>C</given-names></name><name><surname>Han</surname><given-names>X</given-names></name><name><surname>Hang</surname><given-names>T</given-names></name><name><surname>Xu</surname><given-names>Y</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name><name><surname>Yin</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Liposomal remdesivir inhalation solution for targeted lung delivery as a novel therapeutic approach for COVID-19</article-title><source>Asian J Pharm Sci</source><year>2021</year><volume>16</volume><issue>6</issue><fpage>772</fpage><lpage>783</lpage><pub-id pub-id-type=\"pmid\">34703490</pub-id><pub-id pub-id-type=\"pmcid\">8529908</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ajps.2021.09.002</pub-id></mixed-citation></ref><ref id=\"CR107\"><mixed-citation publication-type=\"other\">Lim ES, Wang D. Pathogen discovery (2016) Molecular microbiology: diagnostic principles and practice. pp 80–91</mixed-citation></ref><ref id=\"CR108\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lau</surname><given-names>SK</given-names></name><name><surname>Varma</surname><given-names>VA</given-names></name><name><surname>Moffitt</surname><given-names>RA</given-names></name><name><surname>Caldwell</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>T</given-names></name><name><surname>Young</surname><given-names>AN</given-names></name><name><surname>Petros</surname><given-names>JA</given-names></name><name><surname>Osunkoya</surname><given-names>AO</given-names></name><name><surname>Krogstad</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Molecular mapping of tumor heterogeneity on clinical tissue specimens with multiplexed quantum dots</article-title><source>ACS Nano</source><year>2010</year><volume>4</volume><issue>5</issue><fpage>2755</fpage><lpage>2765</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXksVKltLc%3D</pub-id><pub-id pub-id-type=\"pmid\">20377268</pub-id><pub-id pub-id-type=\"pmcid\">2923482</pub-id><pub-id pub-id-type=\"doi\">10.1021/nn100213v</pub-id></mixed-citation></ref><ref id=\"CR109\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>YC</given-names></name><name><surname>Liao</surname><given-names>CH</given-names></name><name><surname>Chang</surname><given-names>CF</given-names></name><name><surname>Chou</surname><given-names>CC</given-names></name><name><surname>Lin</surname><given-names>YR</given-names></name></person-group><article-title xml:lang=\"en\">A locally transmitted case of SARS-CoV-2 infection in Taiwan</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>11</issue><fpage>1070</fpage><lpage>1072</lpage><pub-id pub-id-type=\"pmid\">32050059</pub-id><pub-id pub-id-type=\"pmcid\">7121202</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMc2001573</pub-id></mixed-citation></ref><ref id=\"CR110\"><mixed-citation publication-type=\"other\">Lohcharoenkal W, Wang L, Chen YC, Rojanasakul Y (2014) Protein nanoparticles as drug delivery carriers for cancer therapy. BioMed Res Int </mixed-citation></ref><ref id=\"CR111\"><mixed-citation publication-type=\"other\">Lokhande SS (2018) Liposome drug delivery: an update review. Pharm Sci Monit 9(1)</mixed-citation></ref><ref id=\"CR112\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Caracterización genómica y epidemiología del nuevo coronavirus 2019: implicaciones para los orígenes del virus y la unión al receptor</article-title><source>Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>566</fpage><lpage>568</lpage></mixed-citation></ref><ref id=\"CR113\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lu</surname><given-names>R</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Niu</surname><given-names>P</given-names></name><name><surname>Yang</surname><given-names>B</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>B</given-names></name><name><surname>Zhu</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10224</issue><fpage>565</fpage><lpage>574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOmsLY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30251-8</pub-id></mixed-citation></ref><ref id=\"CR114\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lustig</surname><given-names>Y</given-names></name><name><surname>Keler</surname><given-names>S</given-names></name><name><surname>Kolodny</surname><given-names>R</given-names></name><name><surname>Ben-Tal</surname><given-names>N</given-names></name><name><surname>Atias-Varon</surname><given-names>D</given-names></name><name><surname>Shlush</surname><given-names>E</given-names></name><name><surname>Gerlic</surname><given-names>M</given-names></name><name><surname>Munitz</surname><given-names>A</given-names></name><name><surname>Doolman</surname><given-names>R</given-names></name><name><surname>Asraf</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Potential antigenic cross-reactivity between severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and dengue viruses</article-title><source>Clin Infect Dis</source><year>2021</year><volume>73</volume><issue>7</issue><fpage>e2444</fpage><lpage>e2449</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXis1Wrt7%2FE</pub-id><pub-id pub-id-type=\"pmid\">32797228</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1207</pub-id></mixed-citation></ref><ref id=\"CR115\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>Q</given-names></name><name><surname>Pan</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>R</given-names></name><name><surname>Liu</surname><given-names>B</given-names></name><name><surname>Li</surname><given-names>C</given-names></name><name><surname>Xie</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Jiang</surname><given-names>H</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Liu Shen capsule shows antiviral and anti-inflammatory abilities against novel coronavirus SARS-CoV-2 via suppression of NF-κB signaling pathway</article-title><source>Pharmacol Res</source><year>2020</year><volume>158</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVeltLrK</pub-id><pub-id pub-id-type=\"pmid\">32360580</pub-id><pub-id pub-id-type=\"pmcid\">7192119</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.phrs.2020.104850</pub-id></mixed-citation></ref><ref id=\"CR116\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ma</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><name><surname>Sun</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Guo</surname><given-names>J</given-names></name><name><surname>Morawska</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Coronavirus disease 2019 patients in earlier stages exhaled millions of severe acute respiratory syndrome coronavirus 2 per hour</article-title><source>Clin Infect Dis</source><year>2021</year><volume>72</volume><issue>10</issue><fpage>e652</fpage><lpage>e654</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFyitrvM</pub-id><pub-id pub-id-type=\"pmid\">32857833</pub-id><pub-id pub-id-type=\"doi\">10.1093/cid/ciaa1283</pub-id></mixed-citation></ref><ref id=\"CR117\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maggi</surname><given-names>F</given-names></name><name><surname>Pistello</surname><given-names>M</given-names></name><name><surname>Antonelli</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Future management of viral diseases: role of new technologies and new approaches in microbial interactions</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>2</issue><fpage>136</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BB3crntlyrug%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">30502490</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.11.015</pub-id></mixed-citation></ref><ref id=\"CR118\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahajan</surname><given-names>HS</given-names></name><name><surname>Patil</surname><given-names>PH</given-names></name></person-group><article-title xml:lang=\"en\">Central composite design-based optimization of lopinavir vitamin E-TPGS micelle: In vitro characterization and in vivo pharmacokinetic study</article-title><source>Colloids Surf, B</source><year>2020</year><volume>194</volume><fpage>111149</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXht1Kis7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.colsurfb.2020.111149</pub-id></mixed-citation></ref><ref id=\"CR119\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Maji</surname><given-names>S</given-names></name><name><surname>Biswas</surname><given-names>A</given-names></name><name><surname>Mandal</surname><given-names>S</given-names></name><name><surname>Sen</surname><given-names>DJ</given-names></name><name><surname>Mahanti</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Silver and copper nano particles in mask for Corona Virus-19 protection</article-title><source>Glob J Endrocrinol Metab</source><year>2020</year><volume>3</volume><fpage>000556</fpage></mixed-citation></ref><ref id=\"CR02\"><mixed-citation publication-type=\"other\">Maniyar MG, Kokare CR (2019) Formulation and evaluation of spray-dried liposomes of lopinavir for topical application. J Pharm Investig 49:259–270.</mixed-citation></ref><ref id=\"CR120\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Masters</surname><given-names>PS</given-names></name></person-group><article-title xml:lang=\"en\">The molecular biology of coronaviruses</article-title><source>Adv Virus Res</source><year>2006</year><volume>66</volume><fpage>193</fpage><lpage>292</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1cXhslOhtLw%3D</pub-id><pub-id pub-id-type=\"pmid\">16877062</pub-id><pub-id pub-id-type=\"pmcid\">7112330</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0065-3527(06)66005-3</pub-id></mixed-citation></ref><ref id=\"CR121\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Matsuura</surname><given-names>K</given-names></name><name><surname>Watanabe</surname><given-names>K</given-names></name><name><surname>Matsuzaki</surname><given-names>T</given-names></name><name><surname>Sakurai</surname><given-names>K</given-names></name><name><surname>Kimizuka</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Self-assembled synthetic viral capsids from a 24-mer viral peptide fragment</article-title><source>Angew Chem Int Ed</source><year>2010</year><volume>49</volume><issue>50</issue><fpage>9662</fpage><lpage>9665</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXhsFaqs7jF</pub-id><pub-id pub-id-type=\"doi\">10.1002/anie.201004606</pub-id></mixed-citation></ref><ref id=\"CR122\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Miripour</surname><given-names>ZS</given-names></name><name><surname>Sarrami-Forooshani</surname><given-names>R</given-names></name><name><surname>Sanati</surname><given-names>H</given-names></name><name><surname>Makarem</surname><given-names>J</given-names></name><name><surname>Taheri</surname><given-names>MS</given-names></name><name><surname>Shojaeian</surname><given-names>F</given-names></name><name><surname>Eskafi</surname><given-names>AH</given-names></name><name><surname>Abbasvandi</surname><given-names>F</given-names></name><name><surname>Namdar</surname><given-names>N</given-names></name><name><surname>Ghafari</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Real-time diagnosis of reactive oxygen species (ROS) in fresh sputum by electrochemical tracing; correlation between COVID-19 and viral-induced ROS in lung/respiratory epithelium during this pandemic</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><fpage>112435</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlyjtL%2FI</pub-id><pub-id pub-id-type=\"pmid\">32729548</pub-id><pub-id pub-id-type=\"pmcid\">7341050</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112435</pub-id></mixed-citation></ref><ref id=\"CR123\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Misra</surname><given-names>N</given-names></name><name><surname>Bhatt</surname><given-names>S</given-names></name><name><surname>Arefi-Khonsari</surname><given-names>F</given-names></name><name><surname>Kumar</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">State of the art in nonthermal plasma processing for biomedical applications: Can it help fight viral pandemics like COVID-19?</article-title><source>Plasma Process Polym</source><year>2021</year><volume>18</volume><issue>7</issue><fpage>2000215</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFKktbnF</pub-id><pub-id pub-id-type=\"pmid\">34220401</pub-id><pub-id pub-id-type=\"pmcid\">8237024</pub-id><pub-id pub-id-type=\"doi\">10.1002/ppap.202000215</pub-id></mixed-citation></ref><ref id=\"CR124\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Moabelo</surname><given-names>KL</given-names></name><name><surname>Martin</surname><given-names>DR</given-names></name><name><surname>Fadaka</surname><given-names>AO</given-names></name><name><surname>Sibuyi</surname><given-names>NR</given-names></name><name><surname>Meyer</surname><given-names>M</given-names></name><name><surname>Madiehe</surname><given-names>AM</given-names></name></person-group><article-title xml:lang=\"en\">Nanotechnology-based strategies for effective and rapid detection of SARS-CoV-2</article-title><source>Materials</source><year>2021</year><volume>14</volume><issue>24</issue><fpage>7851</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXivVaqs7jJ</pub-id><pub-id pub-id-type=\"pmid\">34947447</pub-id><pub-id pub-id-type=\"pmcid\">8703409</pub-id><pub-id pub-id-type=\"doi\">10.3390/ma14247851</pub-id></mixed-citation></ref><ref id=\"CR125\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mohammadniaei</surname><given-names>M</given-names></name><name><surname>Zhang</surname><given-names>M</given-names></name><name><surname>Ashley</surname><given-names>J</given-names></name><name><surname>Christensen</surname><given-names>UB</given-names></name><name><surname>Friis-Hansen</surname><given-names>LJ</given-names></name><name><surname>Gregersen</surname><given-names>R</given-names></name><name><surname>Lisby</surname><given-names>JG</given-names></name><name><surname>Benfield</surname><given-names>TL</given-names></name><name><surname>Nielsen</surname><given-names>FE</given-names></name><name><surname>Henning Rasmussen</surname><given-names>J</given-names></name><name><surname>Pedersen</surname><given-names>EB</given-names></name></person-group><article-title xml:lang=\"en\">A non-enzymatic, isothermal strand displacement and amplification assay for rapid detection of SARS-CoV-2 RNA</article-title><source>Nat Commun</source><year>2021</year><volume>12</volume><issue>1</issue><fpage>1</fpage><lpage>2</lpage><pub-id pub-id-type=\"doi\">10.1038/s41467-021-25387-9</pub-id></mixed-citation></ref><ref id=\"CR126\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Monto</surname><given-names>AS</given-names></name><name><surname>Fendrick</surname><given-names>AM</given-names></name><name><surname>Sarnes</surname><given-names>MW</given-names></name></person-group><article-title xml:lang=\"en\">Respiratory illness caused by picornavirus infection: a review of clinical outcomes</article-title><source>Clin Ther</source><year>2001</year><volume>23</volume><issue>10</issue><fpage>1615</fpage><lpage>1627</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BD3MnnvFCjsQ%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">11726001</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0149-2918(01)80133-8</pub-id></mixed-citation></ref><ref id=\"CR127\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morokutti-Kurz</surname><given-names>M</given-names></name><name><surname>Graf</surname><given-names>C</given-names></name><name><surname>Prieschl-Grassauer</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Amylmetacresol/2, 4-dichlorobenzyl alcohol, hexylresorcinol, or carrageenan lozenges as active treatments for sore throat</article-title><source>Int J General Med</source><year>2017</year><volume>10</volume><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFamtL7E</pub-id><pub-id pub-id-type=\"doi\">10.2147/IJGM.S120665</pub-id></mixed-citation></ref><ref id=\"CR128\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Munster</surname><given-names>VJ</given-names></name><name><surname>Koopmans</surname><given-names>M</given-names></name><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>van Riel</surname><given-names>D</given-names></name><name><surname>de Wit</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus emerging in China—key questions for impact assessment</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><issue>8</issue><fpage>692</fpage><lpage>694</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjslGltrk%3D</pub-id><pub-id pub-id-type=\"pmid\">31978293</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMp2000929</pub-id></mixed-citation></ref><ref id=\"CR129\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Murtaza</surname><given-names>A</given-names></name><name><surname>Rehman</surname><given-names>T</given-names></name><name><surname>Zafar</surname><given-names>Z</given-names></name><name><surname>Noreen</surname><given-names>S</given-names></name><name><surname>Iqbal</surname><given-names>M</given-names></name><name><surname>Hassan</surname><given-names>S</given-names></name><name><surname>Khawaish</surname><given-names>I</given-names></name><name><surname>Tarar</surname><given-names>MHA</given-names></name></person-group><article-title xml:lang=\"en\">Synthesis and characterization of herbal nano-suspensions and evaluation of their In-vivo antihypertensive potential with especial focus on piperine</article-title><source>ACE</source><year>2020</year><volume>1</volume><fpage>23</fpage></mixed-citation></ref><ref id=\"CR130\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muthuirulappan</surname><given-names>S</given-names></name><name><surname>Francis</surname><given-names>SP</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer mechanism and possibility of nano-suspension formulation for a marine algae product fucoxanthin</article-title><source>Asian Pac J Cancer Prev</source><year>2013</year><volume>14</volume><issue>4</issue><fpage>2213</fpage><lpage>2216</lpage><pub-id pub-id-type=\"pmid\">23725114</pub-id><pub-id pub-id-type=\"doi\">10.7314/APJCP.2013.14.4.2213</pub-id></mixed-citation></ref><ref id=\"CR131\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Naseema</surname><given-names>A</given-names></name><name><surname>Kovooru</surname><given-names>L</given-names></name><name><surname>Behera</surname><given-names>AK</given-names></name><name><surname>Kumar</surname><given-names>KP</given-names></name><name><surname>Srivastava</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">A critical review of synthesis procedures, applications and future potential of nanoemulsions</article-title><source>Adv Coll Interface Sci</source><year>2021</year><volume>287</volume><fpage>102318</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisVSmtrvN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cis.2020.102318</pub-id></mixed-citation></ref><ref id=\"CR132\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndwandwe</surname><given-names>D</given-names></name><name><surname>Wiysonge</surname><given-names>CS</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines</article-title><source>Curr Opin Immunol</source><year>2021</year><volume>71</volume><fpage>111</fpage><lpage>116</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhsF2gtrrK</pub-id><pub-id pub-id-type=\"pmid\">34330017</pub-id><pub-id pub-id-type=\"pmcid\">8272971</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.coi.2021.07.003</pub-id></mixed-citation></ref><ref id=\"CR133\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Negi</surname><given-names>JS</given-names></name><name><surname>Chattopadhyay</surname><given-names>P</given-names></name><name><surname>Sharma</surname><given-names>AK</given-names></name><name><surname>Ram</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Development and evaluation of glyceryl behenate based solid lipid nanoparticles (SLNs) using hot self-nanoemulsification (SNE) technique</article-title><source>Arch Pharmacal Res</source><year>2014</year><volume>37</volume><issue>3</issue><fpage>361</fpage><lpage>370</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXot1Kns7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12272-013-0154-y</pub-id></mixed-citation></ref><ref id=\"CR134\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nguyen-Contant</surname><given-names>P</given-names></name><name><surname>Embong</surname><given-names>AK</given-names></name><name><surname>Kanagaiah</surname><given-names>P</given-names></name><name><surname>Chaves</surname><given-names>FA</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Branche</surname><given-names>AR</given-names></name><name><surname>Topham</surname><given-names>DJ</given-names></name><name><surname>Sangster</surname><given-names>MY</given-names></name></person-group><article-title xml:lang=\"en\">S protein-reactive IgG and memory B cell production after human SARS-CoV-2 infection includes broad reactivity to the S2 subunit</article-title><source>Mbio</source><year>2020</year><volume>11</volume><issue>5</issue><fpage>e01991</fpage><lpage>e2020</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitl2kt7g%3D</pub-id><pub-id pub-id-type=\"pmid\">32978311</pub-id><pub-id pub-id-type=\"pmcid\">7520599</pub-id><pub-id pub-id-type=\"doi\">10.1128/mBio.01991-20</pub-id></mixed-citation></ref><ref id=\"CR135\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Nikaeen</surname><given-names>G</given-names></name><name><surname>Abbaszadeh</surname><given-names>S</given-names></name><name><surname>Yousefinejad</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Application of nanomaterials in treatment, anti-infection and detection of coronaviruses</article-title><source>Nanomedicine</source><year>2020</year><volume>15</volume><issue>15</issue><fpage>1501</fpage><lpage>1512</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtlWqtrvF</pub-id><pub-id pub-id-type=\"pmid\">32378459</pub-id><pub-id pub-id-type=\"pmcid\">7373208</pub-id><pub-id pub-id-type=\"doi\">10.2217/nnm-2020-0117</pub-id></mixed-citation></ref><ref id=\"CR136\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noriega-Luna</surname><given-names>B</given-names></name><name><surname>Godínez</surname><given-names>LA</given-names></name><name><surname>Rodríguez</surname><given-names>FJ</given-names></name><name><surname>Rodríguez</surname><given-names>A</given-names></name><name><surname>Larrea</surname><given-names>GZLD</given-names></name><name><surname>Sosa-Ferreyra</surname><given-names>CF</given-names></name><name><surname>Mercado-Curiel</surname><given-names>RF</given-names></name><name><surname>Manríquez</surname><given-names>J</given-names></name><name><surname>Bustos</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Applications of dendrimers in drug delivery agents, diagnosis, therapy, and detection</article-title><source>J Nanomater</source><year>2014</year><volume>2014</volume><fpage>39</fpage><lpage>39</lpage><pub-id pub-id-type=\"doi\">10.1155/2014/507273</pub-id></mixed-citation></ref><ref id=\"CR137\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Noyes</surname><given-names>AA</given-names></name><name><surname>Whitney</surname><given-names>WR</given-names></name></person-group><article-title xml:lang=\"en\">The rate of solution of solid substances in their own solutions</article-title><source>J Am Chem Soc</source><year>1897</year><volume>19</volume><issue>12</issue><fpage>930</fpage><lpage>934</lpage><pub-id pub-id-type=\"doi\">10.1021/ja02086a003</pub-id></mixed-citation></ref><ref id=\"CR138\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olvera</surname><given-names>D</given-names></name><name><surname>Monaghan</surname><given-names>MG</given-names></name></person-group><article-title xml:lang=\"en\">Electroactive material-based biosensors for detection and drug delivery</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>170</volume><fpage>396</fpage><lpage>424</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvF2jsLnE</pub-id><pub-id pub-id-type=\"pmid\">32987096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2020.09.011</pub-id></mixed-citation></ref><ref id=\"CR139\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>G</given-names></name><name><surname>Fan</surname><given-names>J</given-names></name><name><surname>Tang</surname><given-names>Y</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Long</surname><given-names>X</given-names></name><name><surname>Guo</surname><given-names>S</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Hu</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Serological immunochromatographic approach in diagnosis with SARS-CoV-2 infected COVID-19 patients</article-title><source>J Infect</source><year>2020</year><volume>81</volume><issue>1</issue><fpage>e28</fpage><lpage>32</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCjtb7P</pub-id><pub-id pub-id-type=\"pmid\">32283141</pub-id><pub-id pub-id-type=\"pmcid\">7195339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jinf.2020.03.051</pub-id></mixed-citation></ref><ref id=\"CR140\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Park</surname><given-names>GS</given-names></name><name><surname>Ku</surname><given-names>K</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SJ</given-names></name><name><surname>Kim</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>BT</given-names></name><name><surname>Maeng</surname><given-names>JS</given-names></name></person-group><article-title xml:lang=\"en\">Development of reverse transcription loop-mediated isothermal amplification assays targeting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)</article-title><source>J Mol Diagn</source><year>2020</year><volume>22</volume><issue>6</issue><fpage>729</fpage><lpage>735</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXns1Wrsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">32276051</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jmoldx.2020.03.006</pub-id></mixed-citation></ref><ref id=\"CR03\"><mixed-citation publication-type=\"other\">Patel G, Shelat P, Lalwani A (2016) Statistical modeling, optimization and characterization of solid self-nanoemulsifying drug delivery system of lopinavir using design of experiment. Drug delivery 23:3027–3042</mixed-citation></ref><ref id=\"CR141\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patel</surname><given-names>GM</given-names></name><name><surname>Shelat</surname><given-names>PK</given-names></name><name><surname>Lalwani</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">QbD based development of proliposome of lopinavir for improved oral bioavailability</article-title><source>Eur J Pharm Sci</source><year>2017</year><volume>108</volume><fpage>50</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsVOqu7rO</pub-id><pub-id pub-id-type=\"pmid\">27586019</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ejps.2016.08.057</pub-id></mixed-citation></ref><ref id=\"CR142\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patone</surname><given-names>M</given-names></name><name><surname>Mei</surname><given-names>XW</given-names></name><name><surname>Handunnetthi</surname><given-names>L</given-names></name><name><surname>Dixon</surname><given-names>S</given-names></name><name><surname>Zaccardi</surname><given-names>F</given-names></name><name><surname>Shankar-Hari</surname><given-names>M</given-names></name><name><surname>Watkinson</surname><given-names>P</given-names></name><name><surname>Khunti</surname><given-names>K</given-names></name><name><surname>Harnden</surname><given-names>A</given-names></name><name><surname>Coupland</surname><given-names>CAC</given-names></name><name><surname>Channon</surname><given-names>KM</given-names></name><name><surname>Mills</surname><given-names>NL</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name><name><surname>Hippisley-Cox</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Risks of myocarditis, pericarditis, and cardiac arrhythmias associated with COVID-19 vaccination or SARS-CoV-2 infection</article-title><source>Nat Med</source><year>2022</year><volume>28</volume><issue>2</issue><fpage>410</fpage><lpage>422</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXislCjsbnJ</pub-id><pub-id pub-id-type=\"pmid\">34907393</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41591-021-01630-0</pub-id></mixed-citation></ref><ref id=\"CR143\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Fraceto</surname><given-names>LF</given-names></name><name><surname>Campos</surname><given-names>EVR</given-names></name><name><surname>Rodriguez-Torres</surname><given-names>MdP</given-names></name><name><surname>Acosta-Torres</surname><given-names>LS</given-names></name><name><surname>Diaz-Torres</surname><given-names>LA</given-names></name><name><surname>Grillo</surname><given-names>R</given-names></name><name><surname>Swamy</surname><given-names>MK</given-names></name><name><surname>Sharma</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Nano based drug delivery systems: recent developments and future prospects</article-title><source>J Nanobiotechnol</source><year>2018</year><volume>16</volume><issue>1</issue><fpage>1</fpage><lpage>33</lpage><pub-id pub-id-type=\"doi\">10.1186/s12951-018-0392-8</pub-id></mixed-citation></ref><ref id=\"CR144\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Patra</surname><given-names>JK</given-names></name><name><surname>Das</surname><given-names>G</given-names></name><name><surname>Kumar</surname><given-names>A</given-names></name><name><surname>Ansari</surname><given-names>A</given-names></name><name><surname>Kim</surname><given-names>H</given-names></name><name><surname>Shin</surname><given-names>HS</given-names></name></person-group><article-title xml:lang=\"en\">Photo-mediated biosynthesis of silver nanoparticles using the non-edible accrescent fruiting calyx of physalis peruviana L. fruits and investigation of its radical scavenging potential and cytotoxicity activities</article-title><source>J Photochem Photobiol, B</source><year>2018</year><volume>188</volume><fpage>116</fpage><lpage>125</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhslyqs7vO</pub-id><pub-id pub-id-type=\"pmid\">30266015</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jphotobiol.2018.08.004</pub-id></mixed-citation></ref><ref id=\"CR145\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peng</surname><given-names>H</given-names></name><name><surname>Lt</surname><given-names>Y</given-names></name><name><surname>Ly</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name><name><surname>Zq</surname><given-names>Lu</given-names></name><name><surname>Koup</surname><given-names>RA</given-names></name><name><surname>Bailer</surname><given-names>RT</given-names></name><name><surname>Cy</surname><given-names>Wu</given-names></name></person-group><article-title xml:lang=\"en\">Long-lived memory T lymphocyte responses against SARS coronavirus nucleocapsid protein in SARS-recovered patients</article-title><source>Virology</source><year>2006</year><volume>351</volume><issue>2</issue><fpage>466</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28Xns1Gks7c%3D</pub-id><pub-id pub-id-type=\"pmid\">16690096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.virol.2006.03.036</pub-id></mixed-citation></ref><ref id=\"CR146\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Phan</surname><given-names>IQ</given-names></name><name><surname>Subramanian</surname><given-names>S</given-names></name><name><surname>Kim</surname><given-names>D</given-names></name><name><surname>Murphy</surname><given-names>M</given-names></name><name><surname>Pettie</surname><given-names>D</given-names></name><name><surname>Carter</surname><given-names>L</given-names></name><name><surname>Anishchenko</surname><given-names>I</given-names></name><name><surname>Barrett</surname><given-names>LK</given-names></name><name><surname>Craig</surname><given-names>J</given-names></name><name><surname>Tillery</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">In silico detection of SARS-CoV-2 specific B-cell epitopes and validation in ELISA for serological diagnosis of COVID-19</article-title><source>Sci Rep</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>1</fpage><lpage>11</lpage><pub-id pub-id-type=\"doi\">10.1038/s41598-021-83730-y</pub-id></mixed-citation></ref><ref id=\"CR147\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pimentel</surname><given-names>TA</given-names></name><name><surname>Yan</surname><given-names>Z</given-names></name><name><surname>Jeffers</surname><given-names>SA</given-names></name><name><surname>Holmes</surname><given-names>KV</given-names></name><name><surname>Hodges</surname><given-names>RS</given-names></name><name><surname>Burkhard</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Peptide nanoparticles as novel immunogens: design and analysis of a prototypic severe acute respiratory syndrome vaccine</article-title><source>Chem Biol Drug Des</source><year>2009</year><volume>73</volume><issue>1</issue><fpage>53</fpage><lpage>61</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptFCnsw%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">19152635</pub-id><pub-id pub-id-type=\"doi\">10.1111/j.1747-0285.2008.00746.x</pub-id></mixed-citation></ref><ref id=\"CR148\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prasad</surname><given-names>R</given-names></name><name><surname>Pandey</surname><given-names>R</given-names></name><name><surname>Barman</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Engineering tailored nanoparticles with microbes: quo vadis</article-title><source>Wires Nanomed Nanobiotechnol</source><year>2015</year><pub-id pub-id-type=\"doi\">10.1002/wnan.1363</pub-id></mixed-citation></ref><ref id=\"CR224\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Prusty</surname><given-names>K</given-names></name><name><surname>Swain</surname><given-names>SK</given-names></name></person-group><article-title xml:lang=\"en\">Nano silver decorated polyacrylamide/dextran nanohydrogels hybrid composites for drug delivery applications</article-title><source>Mater Sci Eng, C</source><year>2018</year><volume>85</volume><fpage>130</fpage><lpage>141</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXitVejtLfL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.msec.2017.11.028</pub-id></mixed-citation></ref><ref id=\"CR149\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Puga-Gómez</surname><given-names>R</given-names></name><name><surname>Ricardo-Delgado</surname><given-names>Y</given-names></name><name><surname>Rojas-Iriarte</surname><given-names>C</given-names></name><name><surname>Céspedes-Henriquez</surname><given-names>L</given-names></name><name><surname>Piedra-Bello</surname><given-names>M</given-names></name><name><surname>Vega-Mendoza</surname><given-names>D</given-names></name><name><surname>Pérez</surname><given-names>NP</given-names></name><name><surname>Paredes-Moreno</surname><given-names>B</given-names></name><name><surname>Rodríguez-González</surname><given-names>M</given-names></name><name><surname>Valenzuela-Silva</surname><given-names>C</given-names></name><name><surname>Sánchez-Ramírez</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Open-label phase I/II clinical trial of SARS-CoV-2 receptor binding domain-tetanus toxoid conjugate vaccine (FINLAY-FR-2) in combination with receptor binding domain-protein vaccine (FINLAY-FR-1A) in children</article-title><source>Int J Infect Dis</source><year>2023</year><volume>126</volume><fpage>164</fpage><lpage>173</lpage><pub-id pub-id-type=\"pmid\">36403819</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijid.2022.11.016</pub-id></mixed-citation></ref><ref id=\"CR150\"><mixed-citation publication-type=\"other\">Puri A, Loomis K, Smith B, Lee JH, Yavlovich A, Heldman E, Blumenthal, R (2009) Lipid-based nanoparticles as pharmaceutical drug carriers: from concepts to clinic. Crit Rev™ Ther Drug Carr Syst 26(6)</mixed-citation></ref><ref id=\"CR151\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Que</surname><given-names>H</given-names></name><name><surname>Hong</surname><given-names>W</given-names></name><name><surname>Lan</surname><given-names>T</given-names></name><name><surname>Zeng</surname><given-names>H</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>Wan</surname><given-names>D</given-names></name><name><surname>Bi</surname><given-names>Z</given-names></name><name><surname>Ren</surname><given-names>W</given-names></name><name><surname>Luo</surname><given-names>M</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>He</surname><given-names>C</given-names></name><name><surname>Zhong</surname><given-names>A</given-names></name><name><surname>Wei</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Tripterin liposome relieves severe acute respiratory syndrome as a potent COVID-19 treatment</article-title><source>Signal Transduct Target Ther</source><year>2022</year><volume>7</volume><issue>1</issue><fpage>399</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XjtF2rsrzL</pub-id><pub-id pub-id-type=\"pmid\">36566328</pub-id><pub-id pub-id-type=\"pmcid\">9789731</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41392-022-01283-6</pub-id></mixed-citation></ref><ref id=\"CR152\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quick</surname><given-names>J</given-names></name><name><surname>Grubaugh</surname><given-names>ND</given-names></name><name><surname>Pullan</surname><given-names>ST</given-names></name><name><surname>Claro</surname><given-names>IM</given-names></name><name><surname>Smith</surname><given-names>AD</given-names></name><name><surname>Gangavarapu</surname><given-names>K</given-names></name><name><surname>Oliveira</surname><given-names>G</given-names></name><name><surname>Robles-Sikisaka</surname><given-names>R</given-names></name><name><surname>Rogers</surname><given-names>TF</given-names></name><name><surname>Beutler</surname><given-names>NA</given-names></name><name><surname>Burton</surname><given-names>DR</given-names></name></person-group><article-title xml:lang=\"en\">Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples</article-title><source>Nat Protoc</source><year>2017</year><volume>12</volume><issue>6</issue><fpage>1261</fpage><lpage>1276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXosVahu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">28538739</pub-id><pub-id pub-id-type=\"pmcid\">5902022</pub-id><pub-id pub-id-type=\"doi\">10.1038/nprot.2017.066</pub-id></mixed-citation></ref><ref id=\"CR153\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rabiee</surname><given-names>N</given-names></name><name><surname>Bagherzadeh</surname><given-names>M</given-names></name><name><surname>Ghasemi</surname><given-names>A</given-names></name><name><surname>Zare</surname><given-names>H</given-names></name><name><surname>Ahmadi</surname><given-names>S</given-names></name><name><surname>Fatahi</surname><given-names>Y</given-names></name><name><surname>Dinarvand</surname><given-names>R</given-names></name><name><surname>Rabiee</surname><given-names>M</given-names></name><name><surname>Ramakrishna</surname><given-names>S</given-names></name><name><surname>Shokouhimehr</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Point-of-use rapid detection of SARS-CoV-2: nanotechnology-enabled solutions for the COVID-19 pandemic</article-title><source>Int J Mol Sci</source><year>2020</year><volume>21</volume><issue>14</issue><fpage>5126</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVChtLfN</pub-id><pub-id pub-id-type=\"pmid\">32698479</pub-id><pub-id pub-id-type=\"pmcid\">7404277</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms21145126</pub-id></mixed-citation></ref><ref id=\"CR155\"><mixed-citation publication-type=\"other\">Raj S, Khurana S, Choudhari R, Kesari KK, Kamal MA, Garg N, Ruokolainen J, Das BC, Kumar D (2021) In Specific targeting cancer cells with nanoparticles and drug delivery in cancer therapy, Seminars in cancer biology. Elsevier pp 166–177</mixed-citation></ref><ref id=\"CR156\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ramachandran</surname><given-names>AK</given-names></name><name><surname>Das</surname><given-names>S</given-names></name><name><surname>Joseph</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Crosstalk between COVID-19 and associated neurological disorders: a review</article-title><source>Curr Neuropharmacol</source><year>2021</year><volume>19</volume><issue>10</issue><fpage>1688</fpage><lpage>1700</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlyksrnO</pub-id><pub-id pub-id-type=\"pmid\">33441073</pub-id><pub-id pub-id-type=\"doi\">10.2174/1570159X19666210113154342</pub-id></mixed-citation></ref><ref id=\"CR157\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>L</given-names></name><name><surname>Xia</surname><given-names>S</given-names></name><name><surname>Xu</surname><given-names>W</given-names></name><name><surname>Tian</surname><given-names>R</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name><name><surname>Gu</surname><given-names>C</given-names></name><name><surname>Pan</surname><given-names>P</given-names></name><name><surname>Meng</surname><given-names>QF</given-names></name><name><surname>Ca</surname><given-names>X</given-names></name><name><surname>Qu</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Decoy nanoparticles protect against COVID-19 by concurrently adsorbing viruses and inflammatory cytokines</article-title><source>Proc Natl Acad Sci</source><year>2020</year><volume>117</volume><issue>44</issue><fpage>27141</fpage><lpage>27147</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXit1Ghu73J</pub-id><pub-id pub-id-type=\"pmid\">33024017</pub-id><pub-id pub-id-type=\"pmcid\">7959535</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.2014352117</pub-id></mixed-citation></ref><ref id=\"CR158\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Rao</surname><given-names>PV</given-names></name><name><surname>Krishna</surname><given-names>BS</given-names></name><name><surname>Jeffree</surname><given-names>MS</given-names></name></person-group><source>Coronaviruses: transmission, frontliners, nanotechnology and economy</source><year>2022</year><publisher-loc> Kota Kinabalu</publisher-loc><publisher-name>Universiti Malaysia Sabah Press</publisher-name></mixed-citation></ref><ref id=\"CR159\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Murthy</surname><given-names>AN</given-names></name></person-group><article-title xml:lang=\"en\">A hybrid design to optimize preparation of lopinavir loaded solid lipid nanoparticles and comparative pharmacokinetic evaluation with marketed lopinavir/ritonavir coformulation</article-title><source>J Pharm Pharmacol</source><year>2014</year><volume>66</volume><issue>7</issue><fpage>912</fpage><lpage>926</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXps1Ohtro%3D</pub-id><pub-id pub-id-type=\"pmid\">24697749</pub-id><pub-id pub-id-type=\"doi\">10.1111/jphp.12217</pub-id></mixed-citation></ref><ref id=\"CR160\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ravi</surname><given-names>PR</given-names></name><name><surname>Vats</surname><given-names>R</given-names></name><name><surname>Dalal</surname><given-names>V</given-names></name><name><surname>Gadekar</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Design, optimization and evaluation of poly-ɛ-caprolactone (PCL) based polymeric nanoparticles for oral delivery of lopinavir</article-title><source>Drug Dev Ind Pharm</source><year>2015</year><volume>41</volume><issue>1</issue><fpage>131</fpage><lpage>140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXitVylurrM</pub-id><pub-id pub-id-type=\"pmid\">24180260</pub-id><pub-id pub-id-type=\"doi\">10.3109/03639045.2013.850710</pub-id></mixed-citation></ref><ref id=\"CR161\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Reddy</surname><given-names>S</given-names></name><name><surname>van der Vlies</surname><given-names>A</given-names></name><name><surname>Simeoni</surname><given-names>E</given-names></name><name><surname>Oneil</surname><given-names>C</given-names></name><name><surname>Swartz</surname><given-names>M</given-names></name><name><surname>Hubbell</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Exploiting lymphatic transport and complement activation in nanoparticle vaccines</article-title><source>Tissue Eng Part A</source><year>2008</year><volume>14</volume><fpage>734</fpage><lpage>735</lpage></mixed-citation></ref><ref id=\"CR162\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rizzo</surname><given-names>JM</given-names></name><name><surname>Buck</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">All things ChIP: ChIP-Chip, ChIP-Seq, ChIP-PCR</article-title><source>Epigenet Regul Epigenomics</source><year>2012</year><volume>2</volume><fpage>41</fpage></mixed-citation></ref><ref id=\"CR163\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Rocchitta</surname><given-names>G</given-names></name><name><surname>Spanu</surname><given-names>A</given-names></name><name><surname>Babudieri</surname><given-names>S</given-names></name><name><surname>Latte</surname><given-names>G</given-names></name><name><surname>Madeddu</surname><given-names>G</given-names></name><name><surname>Galleri</surname><given-names>G</given-names></name><name><surname>Nuvoli</surname><given-names>S</given-names></name><name><surname>Bagella</surname><given-names>P</given-names></name><name><surname>Demartis</surname><given-names>MI</given-names></name><name><surname>Fiore</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Enzyme biosensors for biomedical applications: strategies for safeguarding analytical performances in biological fluids</article-title><source>Sensors</source><year>2016</year><volume>16</volume><issue>6</issue><fpage>780</fpage><pub-id pub-id-type=\"pmid\">27249001</pub-id><pub-id pub-id-type=\"pmcid\">4934206</pub-id><pub-id pub-id-type=\"doi\">10.3390/s16060780</pub-id></mixed-citation></ref><ref id=\"CR164\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>CD</given-names></name><name><surname>Millar</surname><given-names>JE</given-names></name><name><surname>Baillie</surname><given-names>JK</given-names></name></person-group><article-title xml:lang=\"en\">Clinical evidence does not support corticosteroid treatment for 2019-nCoV lung injury</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>473</fpage><lpage>475</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXisFaksr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30317-2</pub-id></mixed-citation></ref><ref id=\"CR165\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Russell</surname><given-names>RL</given-names></name><name><surname>Pelka</surname><given-names>P</given-names></name><name><surname>Mark</surname><given-names>BL</given-names></name></person-group><article-title xml:lang=\"en\">Frontrunners in the race to develop a SARS-CoV-2 vaccine</article-title><source>Can J Microbiol</source><year>2021</year><volume>67</volume><issue>3</issue><fpage>189</fpage><lpage>212</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktF2mtbs%3D</pub-id><pub-id pub-id-type=\"pmid\">33264067</pub-id><pub-id pub-id-type=\"doi\">10.1139/cjm-2020-0465</pub-id></mixed-citation></ref><ref id=\"CR166\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sahakijpijarn</surname><given-names>S</given-names></name><name><surname>Moon</surname><given-names>C</given-names></name><name><surname>Koleng</surname><given-names>JJ</given-names></name><name><surname>Christensen</surname><given-names>DJ</given-names></name><name><surname>Williams</surname><given-names>RO</given-names></name></person-group><article-title xml:lang=\"en\">Development of remdesivir as a dry powder for inhalation by thin film freezing</article-title><source>Pharmaceutics</source><year>2020</year><volume>12</volume><issue>11</issue><fpage>1002</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXlsFCktr4%3D</pub-id><pub-id pub-id-type=\"pmid\">33105618</pub-id><pub-id pub-id-type=\"pmcid\">7690377</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics12111002</pub-id></mixed-citation></ref><ref id=\"CR167\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Salleh</surname><given-names>A</given-names></name><name><surname>Naomi</surname><given-names>R</given-names></name><name><surname>Utami</surname><given-names>ND</given-names></name><name><surname>Mohammad</surname><given-names>AW</given-names></name><name><surname>Mahmoudi</surname><given-names>E</given-names></name><name><surname>Mustafa</surname><given-names>N</given-names></name><name><surname>Fauzi</surname><given-names>MB</given-names></name></person-group><article-title xml:lang=\"en\">The potential of silver nanoparticles for antiviral and antibacterial applications: a mechanism of action</article-title><source>Nanomaterials</source><year>2020</year><volume>10</volume><issue>8</issue><fpage>1566</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsl2gu7bL</pub-id><pub-id pub-id-type=\"pmid\">32784939</pub-id><pub-id pub-id-type=\"pmcid\">7466543</pub-id><pub-id pub-id-type=\"doi\">10.3390/nano10081566</pub-id></mixed-citation></ref><ref id=\"CR168\"><mixed-citation publication-type=\"other\">Sen AK, Nath A, Sudeepthi A, Jain SK, Banerjee U (2022) Microfluidics-Based Point-of-Care Diagnostic Devices. In: Advanced microfluidics-based point-of-care diagnostics. CRC Press pp. 99–120</mixed-citation></ref><ref id=\"CR169\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Seo</surname><given-names>G</given-names></name><name><surname>Lee</surname><given-names>G</given-names></name><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Baek</surname><given-names>SH</given-names></name><name><surname>Choi</surname><given-names>M</given-names></name><name><surname>Ku</surname><given-names>KB</given-names></name><name><surname>Lee</surname><given-names>CS</given-names></name><name><surname>Jun</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>D</given-names></name><name><surname>Kim</surname><given-names>HG</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of COVID-19 causative virus (SARS-CoV-2) in human nasopharyngeal swab specimens using field-effect transistor-based biosensor</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>5135</fpage><lpage>5142</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnt1SrtrY%3D</pub-id><pub-id pub-id-type=\"pmid\">32293168</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02823</pub-id></mixed-citation></ref><ref id=\"CR01\"><mixed-citation publication-type=\"other\">Sercombe L, Veerati T, Moheimani F, Wu SY, Sood AK, Hua S (2015)\n",
       "Advances and challenges of liposome assisted drug delivery. Front Pharmacol. 6:286</mixed-citation></ref><ref id=\"CR170\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shafiee</surname><given-names>S</given-names></name><name><surname>Cegolon</surname><given-names>L</given-names></name><name><surname>Khafaei</surname><given-names>M</given-names></name><name><surname>Gholami</surname><given-names>N</given-names></name><name><surname>Zhao</surname><given-names>S</given-names></name><name><surname>Khalesi</surname><given-names>N</given-names></name><name><surname>Moosavian</surname><given-names>H</given-names></name><name><surname>Fathi</surname><given-names>S</given-names></name><name><surname>Izadi</surname><given-names>M</given-names></name><name><surname>Ghadian</surname><given-names>A</given-names></name><name><surname>Javanbakht</surname><given-names>M</given-names></name><name><surname>Javanbakht</surname><given-names>A</given-names></name><name><surname>Akhavan-Sigari</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Gastrointestinal cancers, ACE-2/TMPRSS2 expression and susceptibility to COVID-19</article-title><source>Cancer Cell Int</source><year>2021</year><volume>21</volume><issue>1</issue><fpage>431</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVemurzO</pub-id><pub-id pub-id-type=\"pmid\">34399734</pub-id><pub-id pub-id-type=\"pmcid\">8365127</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12935-021-02129-x</pub-id></mixed-citation></ref><ref id=\"CR171\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shah</surname><given-names>V</given-names></name><name><surname>Keniya</surname><given-names>R</given-names></name><name><surname>Shridharani</surname><given-names>A</given-names></name><name><surname>Punjabi</surname><given-names>M</given-names></name><name><surname>Shah</surname><given-names>J</given-names></name><name><surname>Mehendale</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 using CT scan images and deep learning techniques</article-title><source>Emerg Radiol</source><year>2021</year><volume>28</volume><issue>3</issue><fpage>497</fpage><lpage>505</lpage><pub-id pub-id-type=\"pmid\">33523309</pub-id><pub-id pub-id-type=\"pmcid\">7848247</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10140-020-01886-y</pub-id></mixed-citation></ref><ref id=\"CR172\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shan</surname><given-names>B</given-names></name><name><surname>Broza</surname><given-names>YY</given-names></name><name><surname>Li</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>S</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name><name><surname>Gui</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name></person-group><article-title xml:lang=\"en\">Multiplexed nanomaterial-based sensor array for detection of COVID-19 in exhaled breath</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>9</issue><fpage>12125</fpage><lpage>12132</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1aqsLfK</pub-id><pub-id pub-id-type=\"pmid\">32808759</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c05657</pub-id></mixed-citation></ref><ref id=\"CR173\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shanley</surname><given-names>LC</given-names></name><name><surname>Mahon</surname><given-names>OR</given-names></name><name><surname>Kelly</surname><given-names>DJ</given-names></name><name><surname>Dunne</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Harnessing the innate and adaptive immune system for tissue repair and regeneration: Considering more than macrophages</article-title><source>Acta Biomater</source><year>2021</year><volume>133</volume><fpage>208</fpage><lpage>221</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1yns7%2FK</pub-id><pub-id pub-id-type=\"pmid\">33657453</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.actbio.2021.02.023</pub-id></mixed-citation></ref><ref id=\"CR174\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Sharma</surname><given-names>N</given-names></name><name><surname>Kumari</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>T</given-names></name><name><surname>Tripathi</surname><given-names>KM</given-names></name></person-group><article-title xml:lang=\"en\">Nano-carbon based sensors for bacterial detection and discrimination in clinical diagnosis: a junction between material science and biology</article-title><source>Appl Mater Today</source><year>2020</year><volume>18</volume><pub-id pub-id-type=\"doi\">10.1016/j.apmt.2019.100467</pub-id></mixed-citation></ref><ref id=\"CR175\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shi</surname><given-names>Y</given-names></name><name><surname>Pramanik</surname><given-names>A</given-names></name><name><surname>Tchounwou</surname><given-names>C</given-names></name><name><surname>Pedraza</surname><given-names>F</given-names></name><name><surname>Crouch</surname><given-names>RA</given-names></name><name><surname>Chavva</surname><given-names>SR</given-names></name><name><surname>Vangara</surname><given-names>A</given-names></name><name><surname>Sinha</surname><given-names>SS</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Sardar</surname><given-names>D</given-names></name></person-group><article-title xml:lang=\"en\">Multifunctional biocompatible graphene oxide quantum dots decorated magnetic nanoplatform for efficient capture and two-photon imaging of rare tumor cells</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>20</issue><fpage>10935</fpage><lpage>10943</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXnslCqu7o%3D</pub-id><pub-id pub-id-type=\"pmid\">25939643</pub-id><pub-id pub-id-type=\"pmcid\">4570252</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b02199</pub-id></mixed-citation></ref><ref id=\"CR176\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Singh</surname><given-names>B</given-names></name><name><surname>Datta</surname><given-names>B</given-names></name><name><surname>Ashish</surname><given-names>A</given-names></name><name><surname>Dutta</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">A comprehensive review on current COVID-19 detection methods: From lab care to point of care diagnosis</article-title><source>Sens Int</source><year>2021</year><volume>2</volume><pub-id pub-id-type=\"pmid\">34766062</pub-id><pub-id pub-id-type=\"pmcid\">8302821</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.sintl.2021.100119</pub-id></mixed-citation></ref><ref id=\"CR177\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Su</surname><given-names>S</given-names></name><name><surname>Wong</surname><given-names>G</given-names></name><name><surname>Shi</surname><given-names>W</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Lai</surname><given-names>AC</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name><name><surname>Bi</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</article-title><source>Trends Microbiol</source><year>2016</year><volume>24</volume><issue>6</issue><fpage>490</fpage><lpage>502</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XksVWmsbc%3D</pub-id><pub-id pub-id-type=\"pmid\">27012512</pub-id><pub-id pub-id-type=\"pmcid\">7125511</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tim.2016.03.003</pub-id></mixed-citation></ref><ref id=\"CR178\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>T</given-names></name><name><surname>Zhang</surname><given-names>YS</given-names></name><name><surname>Pang</surname><given-names>B</given-names></name><name><surname>Hyun</surname><given-names>DC</given-names></name><name><surname>Yang</surname><given-names>M</given-names></name><name><surname>Xia</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Engineered nanoparticles for drug delivery in cancer therapy</article-title><source>Angew Chem Int Ed</source><year>2014</year><volume>53</volume><issue>46</issue><fpage>12320</fpage><lpage>12364</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGksLzO</pub-id></mixed-citation></ref><ref id=\"CR179\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Sun</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><name><surname>Gu</surname><given-names>H</given-names></name><name><surname>Fang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>T</given-names></name><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>Deng</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>P</given-names></name><name><surname>Li</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Zhao</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>C</given-names></name><name><surname>Lang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Fei</surname><given-names>G</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Geng</surname><given-names>S</given-names></name><name><surname>Gao</surname><given-names>Y</given-names></name><name><surname>Wei</surname><given-names>W</given-names></name><name><surname>Hu</surname><given-names>Z</given-names></name><name><surname>Han</surname><given-names>G</given-names></name><name><surname>Sun</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Recombinant vaccine containing an RBD-Fc fusion induced protection against SARS-CoV-2 in nonhuman primates and mice</article-title><source>Cell Mol Immunol</source><year>2021</year><volume>18</volume><issue>4</issue><fpage>1070</fpage><lpage>1073</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXms1OlurY%3D</pub-id><pub-id pub-id-type=\"pmid\">33731916</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41423-021-00658-z</pub-id></mixed-citation></ref><ref id=\"CR180\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Szunerits</surname><given-names>S</given-names></name><name><surname>Barras</surname><given-names>A</given-names></name><name><surname>Khanal</surname><given-names>M</given-names></name><name><surname>Pagneux</surname><given-names>Q</given-names></name><name><surname>Boukherroub</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nanostructures for the inhibition of viral infections</article-title><source>Molecules</source><year>2015</year><volume>20</volume><issue>8</issue><fpage>14051</fpage><lpage>14081</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtlKrtr7F</pub-id><pub-id pub-id-type=\"pmid\">26247927</pub-id><pub-id pub-id-type=\"pmcid\">6332336</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules200814051</pub-id></mixed-citation></ref><ref id=\"CR181\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>F</given-names></name><name><surname>Quan</surname><given-names>Y</given-names></name><name><surname>Xin</surname><given-names>ZT</given-names></name><name><surname>Wrammert</surname><given-names>J</given-names></name><name><surname>Ma</surname><given-names>MJ</given-names></name><name><surname>Lv</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>TB</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Richardus</surname><given-names>JH</given-names></name><name><surname>Liu</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Lack of peripheral memory B cell responses in recovered patients with severe acute respiratory syndrome: a six-year follow-up study</article-title><source>J Immunol</source><year>2011</year><volume>186</volume><issue>12</issue><fpage>7264</fpage><lpage>7268</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXntVGmsrc%3D</pub-id><pub-id pub-id-type=\"pmid\">21576510</pub-id><pub-id pub-id-type=\"doi\">10.4049/jimmunol.0903490</pub-id></mixed-citation></ref><ref id=\"CR182\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tarim</surname><given-names>EA</given-names></name><name><surname>Karakuzu</surname><given-names>B</given-names></name><name><surname>Oksuz</surname><given-names>C</given-names></name><name><surname>Sarigil</surname><given-names>O</given-names></name><name><surname>Kizilkaya</surname><given-names>M</given-names></name><name><surname>Al-Ruweidi</surname><given-names>MK</given-names></name><name><surname>Yalcin</surname><given-names>HC</given-names></name><name><surname>Ozcivici</surname><given-names>E</given-names></name><name><surname>Tekin</surname><given-names>HC</given-names></name></person-group><article-title xml:lang=\"en\">Microfluidic-based virus detection methods for respiratory diseases</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>143</fpage><lpage>168</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFeisLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00169-7</pub-id></mixed-citation></ref><ref id=\"CR183\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tian</surname><given-names>B</given-names></name><name><surname>Gao</surname><given-names>F</given-names></name><name><surname>Fock</surname><given-names>J</given-names></name><name><surname>Dufva</surname><given-names>M</given-names></name><name><surname>Hansen</surname><given-names>MF</given-names></name></person-group><article-title xml:lang=\"en\">Homogeneous circle-to-circle amplification for real-time optomagnetic detection of SARS-CoV-2 RdRp coding sequence</article-title><source>Biosens Bioelectron</source><year>2020</year><volume>165</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFegtbnF</pub-id><pub-id pub-id-type=\"pmid\">32510339</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bios.2020.112356</pub-id></mixed-citation></ref><ref id=\"CR184\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ting</surname><given-names>D</given-names></name><name><surname>Dong</surname><given-names>N</given-names></name><name><surname>Fang</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>J</given-names></name><name><surname>Bi</surname><given-names>J</given-names></name><name><surname>Xiao</surname><given-names>S</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Multisite inhibitors for enteric coronavirus: antiviral cationic carbon dots based on curcumin</article-title><source>ACS Appl Nano Mater</source><year>2018</year><volume>1</volume><issue>10</issue><fpage>5451</fpage><lpage>5459</lpage><pub-id pub-id-type=\"doi\">10.1021/acsanm.8b00779</pub-id></mixed-citation></ref><ref id=\"CR185\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>To</surname><given-names>KK</given-names></name><name><surname>Yip</surname><given-names>CC</given-names></name><name><surname>Lai</surname><given-names>CY</given-names></name><name><surname>Wong</surname><given-names>CK</given-names></name><name><surname>Ho</surname><given-names>DT</given-names></name><name><surname>Pang</surname><given-names>PK</given-names></name><name><surname>Ng</surname><given-names>AC</given-names></name><name><surname>Leung</surname><given-names>KH</given-names></name><name><surname>Poon</surname><given-names>RW</given-names></name><name><surname>Chan</surname><given-names>KH</given-names></name><name><surname>Cheng</surname><given-names>VC</given-names></name></person-group><article-title xml:lang=\"en\">Saliva as a diagnostic specimen for testing respiratory virus by a point-of-care molecular assay: a diagnostic validity study</article-title><source>Clin Microbiol Infect</source><year>2019</year><volume>25</volume><issue>3</issue><fpage>372</fpage><lpage>378</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1yksbnE</pub-id><pub-id pub-id-type=\"pmid\">29906597</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cmi.2018.06.009</pub-id></mixed-citation></ref><ref id=\"CR186\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tomas</surname><given-names>M</given-names></name><name><surname>Capanoglu</surname><given-names>E</given-names></name><name><surname>Bahrami</surname><given-names>A</given-names></name><name><surname>Hosseini</surname><given-names>H</given-names></name><name><surname>Akbari-Alavijeh</surname><given-names>S</given-names></name><name><surname>Shaddel</surname><given-names>R</given-names></name><name><surname>Rehman</surname><given-names>A</given-names></name><name><surname>Rezaei</surname><given-names>A</given-names></name><name><surname>Rashidinejad</surname><given-names>A</given-names></name><name><surname>Garavand</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">The direct and indirect effects of bioactive compounds against coronavirus</article-title><source>Food Front</source><year>2022</year><volume>3</volume><issue>1</issue><fpage>96</fpage><lpage>123</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XitlCgtLc%3D</pub-id><pub-id pub-id-type=\"pmid\">35462942</pub-id><pub-id pub-id-type=\"doi\">10.1002/fft2.119</pub-id></mixed-citation></ref><ref id=\"CR187\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tripathy</surname><given-names>S</given-names></name><name><surname>Das</surname><given-names>MK</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimers and their applications as novel drug delivery carriers</article-title><source>J Appl Pharm Sci</source><year>2013</year><volume>3</volume><issue>09</issue><fpage>142</fpage><lpage>149</lpage></mixed-citation></ref><ref id=\"CR188\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Udugama</surname><given-names>B</given-names></name><name><surname>Kadhiresan</surname><given-names>P</given-names></name><name><surname>Kozlowski</surname><given-names>HN</given-names></name><name><surname>Malekjahani</surname><given-names>A</given-names></name><name><surname>Osborne</surname><given-names>M</given-names></name><name><surname>Li</surname><given-names>VY</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Mubareka</surname><given-names>S</given-names></name><name><surname>Gubbay</surname><given-names>JB</given-names></name><name><surname>Chan</surname><given-names>WC</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosing COVID-19: the disease and tools for detection</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>4</issue><fpage>3822</fpage><lpage>3835</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXlvFChtrk%3D</pub-id><pub-id pub-id-type=\"pmid\">32223179</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c02624</pub-id></mixed-citation></ref><ref id=\"CR189\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vaezi</surname><given-names>A</given-names></name><name><surname>Meysamie</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">COVID-19 vaccines cost-effectiveness analysis: a scenario for Iran</article-title><source>Vaccines</source><year>2021</year><volume>10</volume><issue>1</issue><fpage>37</fpage><pub-id pub-id-type=\"pmid\">35062698</pub-id><pub-id pub-id-type=\"pmcid\">8777749</pub-id><pub-id pub-id-type=\"doi\">10.3390/vaccines10010037</pub-id></mixed-citation></ref><ref id=\"CR190\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vahedifard</surname><given-names>F</given-names></name><name><surname>Chakravarthy</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for COVID-19: the role of nanotechnology in the treatment and diagnosis of COVID-19</article-title><source>Emerg Mater</source><year>2021</year><volume>4</volume><issue>1</issue><fpage>75</fpage><lpage>99</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXktlajsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s42247-021-00168-8</pub-id></mixed-citation></ref><ref id=\"CR191\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Varahachalam</surname><given-names>SP</given-names></name><name><surname>Lahooti</surname><given-names>B</given-names></name><name><surname>Chamaneh</surname><given-names>M</given-names></name><name><surname>Bagchi</surname><given-names>S</given-names></name><name><surname>Chhibber</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>K</given-names></name><name><surname>Bolanos</surname><given-names>JF</given-names></name><name><surname>Kim</surname><given-names>NY</given-names></name><name><surname>Kaushik</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Nanomedicine for the SARS-CoV-2: state-of-the-art and future prospects</article-title><source>Int J Nanomed</source><year>2021</year><volume>16</volume><fpage>539</fpage><pub-id pub-id-type=\"doi\">10.2147/IJN.S283686</pub-id></mixed-citation></ref><ref id=\"CR192\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Verma</surname><given-names>SK</given-names></name><name><surname>Jha</surname><given-names>E</given-names></name><name><surname>Sahoo</surname><given-names>B</given-names></name><name><surname>Panda</surname><given-names>PK</given-names></name><name><surname>Thirumurugan</surname><given-names>A</given-names></name><name><surname>Parashar</surname><given-names>S</given-names></name><name><surname>Suar</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Mechanistic insight into the rapid one-step facile biofabrication of antibacterial silver nanoparticles from bacterial release and their biogenicity and concentration-dependent in vitro cytotoxicity to colon cells</article-title><source>RSC Adv</source><year>2017</year><volume>7</volume><issue>64</issue><fpage>40034</fpage><lpage>40045</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVantLrF</pub-id><pub-id pub-id-type=\"doi\">10.1039/C7RA05943D</pub-id></mixed-citation></ref><ref id=\"CR193\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Vijayan</surname><given-names>V</given-names></name><name><surname>Mohapatra</surname><given-names>A</given-names></name><name><surname>Uthaman</surname><given-names>S</given-names></name><name><surname>Park</surname><given-names>IK</given-names></name></person-group><article-title xml:lang=\"en\">Recent advances in nanovaccines using biomimetic immunomodulatory materials</article-title><source>Pharmaceutics</source><year>2019</year><volume>11</volume><issue>10</issue><fpage>534</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXotFKkt7o%3D</pub-id><pub-id pub-id-type=\"pmid\">31615112</pub-id><pub-id pub-id-type=\"pmcid\">6835828</pub-id><pub-id pub-id-type=\"doi\">10.3390/pharmaceutics11100534</pub-id></mixed-citation></ref><ref id=\"CR194\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Villeret</surname><given-names>B</given-names></name><name><surname>Dieu</surname><given-names>A</given-names></name><name><surname>Straube</surname><given-names>M</given-names></name><name><surname>Solhonne</surname><given-names>B</given-names></name><name><surname>Miklavc</surname><given-names>P</given-names></name><name><surname>Hamadi</surname><given-names>S</given-names></name><name><surname>Le Borgne</surname><given-names>R</given-names></name><name><surname>Mailleux</surname><given-names>A</given-names></name><name><surname>Norel</surname><given-names>X</given-names></name><name><surname>Aerts</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Silver nanoparticles impair retinoic acid-inducible gene I-mediated mitochondrial antiviral immunity by blocking the autophagic flux in lung epithelial cells</article-title><source>ACS Nano</source><year>2018</year><volume>12</volume><issue>2</issue><fpage>1188</fpage><lpage>1202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVart7c%3D</pub-id><pub-id pub-id-type=\"pmid\">29357226</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.7b06934</pub-id></mixed-citation></ref><ref id=\"CR195\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Volkov</surname><given-names>Y</given-names></name></person-group><article-title xml:lang=\"en\">Quantum dots in nanomedicine: recent trends, advances and unresolved issues</article-title><source>Biochem Biophys Res Commun</source><year>2015</year><volume>468</volume><issue>3</issue><fpage>419</fpage><lpage>427</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXht1SqtL3L</pub-id><pub-id pub-id-type=\"pmid\">26168726</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.bbrc.2015.07.039</pub-id></mixed-citation></ref><ref id=\"CR196\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><name><surname>Jiao</surname><given-names>G</given-names></name><name><surname>Shan</surname><given-names>X</given-names></name><name><surname>Tai</surname><given-names>W</given-names></name><name><surname>Yu</surname><given-names>G</given-names></name></person-group><article-title xml:lang=\"en\">Inhibition of influenza A virus infection by fucoidan targeting viral neuraminidase and cellular EGFR pathway</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><issue>1</issue><fpage>1</fpage><lpage>14</lpage><pub-id pub-id-type=\"pmid\">28127051</pub-id><pub-id pub-id-type=\"pmcid\">5428335</pub-id></mixed-citation></ref><ref id=\"CR197\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Hao</surname><given-names>C</given-names></name><name><surname>Yunjia</surname><given-names>Y</given-names></name><name><surname>Qin</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>M</given-names></name><name><surname>Mao</surname><given-names>W</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity against enterovirus 71 of sulfated rhamnan isolated from the green alga Monostroma latissimum</article-title><source>Carbohyd Polym</source><year>2018</year><volume>200</volume><fpage>43</fpage><lpage>53</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsVSqur%2FE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.carbpol.2018.07.067</pub-id></mixed-citation></ref><ref id=\"CR198\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Horby</surname><given-names>PW</given-names></name><name><surname>Hayden</surname><given-names>FG</given-names></name><name><surname>Gao</surname><given-names>GF</given-names></name></person-group><article-title xml:lang=\"en\">A novel coronavirus outbreak of global health concern</article-title><source>The Lancet</source><year>2020</year><volume>395</volume><issue>10223</issue><fpage>470</fpage><lpage>473</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXnsVaisLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S0140-6736(20)30185-9</pub-id></mixed-citation></ref><ref id=\"CR199\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wankar</surname><given-names>JN</given-names></name><name><surname>Chaturvedi</surname><given-names>VK</given-names></name><name><surname>Bohara</surname><given-names>C</given-names></name><name><surname>Singh</surname><given-names>MP</given-names></name><name><surname>Bohara</surname><given-names>RA</given-names></name></person-group><article-title xml:lang=\"en\">Role of nanomedicine in management and prevention of COVID-19</article-title><source>Front Nanotechnol</source><year>2020</year><volume>2</volume><pub-id pub-id-type=\"doi\">10.3389/fnano.2020.589541</pub-id></mixed-citation></ref><ref id=\"CR200\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Weiss</surname><given-names>C</given-names></name><name><surname>Carriere</surname><given-names>M</given-names></name><name><surname>Fusco</surname><given-names>L</given-names></name><name><surname>Capua</surname><given-names>I</given-names></name><name><surname>Regla-Nava</surname><given-names>JA</given-names></name><name><surname>Pasquali</surname><given-names>M</given-names></name><name><surname>Scott</surname><given-names>JA</given-names></name><name><surname>Vitale</surname><given-names>F</given-names></name><name><surname>Unal</surname><given-names>MA</given-names></name><name><surname>Mattevi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Toward nanotechnology-enabled approaches against the COVID-19 pandemic</article-title><source>ACS Nano</source><year>2020</year><volume>14</volume><issue>6</issue><fpage>6383</fpage><lpage>6406</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFWjsLnI</pub-id><pub-id pub-id-type=\"pmid\">32519842</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsnano.0c03697</pub-id></mixed-citation></ref><ref id=\"CR201\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wilson</surname><given-names>DR</given-names></name><name><surname>Sen</surname><given-names>R</given-names></name><name><surname>Sunshine</surname><given-names>JC</given-names></name><name><surname>Pardoll</surname><given-names>DM</given-names></name><name><surname>Green</surname><given-names>JJ</given-names></name><name><surname>Kim</surname><given-names>YJ</given-names></name></person-group><article-title xml:lang=\"en\">Biodegradable STING agonist nanoparticles for enhanced cancer immunotherapy</article-title><source>Nanomed Nanotechnol Biol Med</source><year>2018</year><volume>14</volume><issue>2</issue><fpage>237</fpage><lpage>246</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhvVOrsLjK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nano.2017.10.013</pub-id></mixed-citation></ref><ref id=\"CR202\"><mixed-citation publication-type=\"book\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Ye</surname><given-names>F</given-names></name><name><surname>Cui</surname><given-names>M</given-names></name><name><surname>Shibata</surname><given-names>R</given-names></name><name><surname>Xu</surname><given-names>R</given-names></name><name><surname>Cheng</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>DY</given-names></name></person-group><source>Diagnostic methodology and technology in molecular genetic pathology. Molecular genetic pathology</source><year>2013</year><publisher-loc>New York</publisher-loc><publisher-name>Springer</publisher-name><fpage>211</fpage><lpage>300</lpage><pub-id pub-id-type=\"doi\">10.1007/978-1-4614-4800-6_9</pub-id></mixed-citation></ref><ref id=\"CR203\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>S</given-names></name><name><surname>Peng</surname><given-names>Z</given-names></name><name><surname>Xiao</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Yan</surname><given-names>R</given-names></name><name><surname>Luo</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Detection and analysis of nucleic acid in various biological samples of COVID-19 patients</article-title><source>Travel Med Infect Dis</source><year>2020</year><volume>37</volume><pub-id pub-id-type=\"pmid\">32311437</pub-id><pub-id pub-id-type=\"pmcid\">7165102</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tmaid.2020.101673</pub-id></mixed-citation></ref><ref id=\"CR204\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wu</surname><given-names>K</given-names></name><name><surname>Saha</surname><given-names>R</given-names></name><name><surname>Su</surname><given-names>D</given-names></name><name><surname>Krishna</surname><given-names>VD</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Cheeran</surname><given-names>MCJ</given-names></name><name><surname>Wang</surname><given-names>JP</given-names></name></person-group><article-title xml:lang=\"en\">Magnetic-nanosensor-based virus and pathogen detection strategies before and during COVID-19</article-title><source>ACS Appl Nano Mater</source><year>2020</year><volume>3</volume><issue>10</issue><fpage>9560</fpage><lpage>9580</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFSjs7vN</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsanm.0c02048</pub-id></mixed-citation></ref><ref id=\"CR206\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xiang</surname><given-names>SD</given-names></name><name><surname>Scholzen</surname><given-names>A</given-names></name><name><surname>Minigo</surname><given-names>G</given-names></name><name><surname>David</surname><given-names>C</given-names></name><name><surname>Apostolopoulos</surname><given-names>V</given-names></name><name><surname>Mottram</surname><given-names>PL</given-names></name><name><surname>Plebanski</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Pathogen recognition and development of particulate vaccines: does size matter?</article-title><source>Methods</source><year>2006</year><volume>40</volume><issue>1</issue><fpage>1</fpage><lpage>9</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD28XpvFGiu7k%3D</pub-id><pub-id pub-id-type=\"pmid\">16997708</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ymeth.2006.05.016</pub-id></mixed-citation></ref><ref id=\"CR207\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>G</given-names></name><name><surname>Zeng</surname><given-names>S</given-names></name><name><surname>Zhang</surname><given-names>B</given-names></name><name><surname>Swihart</surname><given-names>MT</given-names></name><name><surname>Yong</surname><given-names>KT</given-names></name><name><surname>Prasad</surname><given-names>PN</given-names></name></person-group><article-title xml:lang=\"en\">New generation cadmium-free quantum dots for biophotonics and nanomedicine</article-title><source>Chem Rev</source><year>2016</year><volume>116</volume><issue>19</issue><fpage>12234</fpage><lpage>12327</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhsFemurjF</pub-id><pub-id pub-id-type=\"pmid\">27657177</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.chemrev.6b00290</pub-id></mixed-citation></ref><ref id=\"CR208\"><mixed-citation publication-type=\"other\">Xu W, Ling P, Zhang T (2013) Polymeric micelles, a promising drug delivery system to enhance bioavailability of poorly water-soluble drugs. J Drug Deliv</mixed-citation></ref><ref id=\"CR209\"><mixed-citation publication-type=\"other\">Yadav HK, Almokdad AA, Sumia I, Debe MS (2019) Polymer-based nanomaterials for drug-delivery carriers. In Nanocarriers for drug delivery. Elsevier pp 531–556</mixed-citation></ref><ref id=\"CR210\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yan</surname><given-names>X</given-names></name><name><surname>Zhou</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name><name><surname>Jin</surname><given-names>Z</given-names></name><name><surname>Zhao</surname><given-names>K</given-names></name></person-group><article-title xml:lang=\"en\">An overview of biodegradable nanomaterials and applications in vaccines</article-title><source>Vaccine</source><year>2020</year><volume>38</volume><issue>5</issue><fpage>1096</fpage><lpage>1104</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitlehurfF</pub-id><pub-id pub-id-type=\"pmid\">31813649</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.vaccine.2019.11.031</pub-id></mixed-citation></ref><ref id=\"CR211\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Shu</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Yu</surname><given-names>Z</given-names></name><name><surname>Fang</surname><given-names>M</given-names></name><name><surname>Yu</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Clinical course and outcomes of critically ill patients with SARS-CoV-2 pneumonia in Wuhan, China: a single-centered, retrospective, observational study</article-title><source>Lancet Respir Med</source><year>2020</year><volume>8</volume><issue>5</issue><fpage>475</fpage><lpage>481</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjvVWqt70%3D</pub-id><pub-id pub-id-type=\"pmid\">32105632</pub-id><pub-id pub-id-type=\"pmcid\">7102538</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-2600(20)30079-5</pub-id></mixed-citation></ref><ref id=\"CR212\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ye</surname><given-names>S</given-names></name><name><surname>Shao</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Guo</surname><given-names>N</given-names></name><name><surname>Zuo</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name><name><surname>Lu</surname><given-names>Z</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>He</surname><given-names>Q</given-names></name><name><surname>Han</surname><given-names>H</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral activity of graphene oxide: how sharp-edged structure and charge matter</article-title><source>ACS Appl Mater Interfaces</source><year>2015</year><volume>7</volume><issue>38</issue><fpage>21571</fpage><lpage>21579</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsFSitrbM</pub-id><pub-id pub-id-type=\"pmid\">26370151</pub-id><pub-id pub-id-type=\"doi\">10.1021/acsami.5b06876</pub-id></mixed-citation></ref><ref id=\"CR213\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yousaf</surname><given-names>SS</given-names></name><name><surname>Isreb</surname><given-names>A</given-names></name><name><surname>Khan</surname><given-names>I</given-names></name><name><surname>Mewsiga</surname><given-names>E</given-names></name><name><surname>Elhissi</surname><given-names>A</given-names></name><name><surname>Ahmed</surname><given-names>W</given-names></name><name><surname>Alhnan</surname><given-names>MA</given-names></name></person-group><article-title xml:lang=\"en\">Impact of nanosizing on the formation and characteristics of polymethacrylate films: micro-versus nano-suspensions</article-title><source>Pharm Dev Technol</source><year>2021</year><volume>26</volume><issue>7</issue><fpage>729</fpage><lpage>739</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Ggtr7L</pub-id><pub-id pub-id-type=\"pmid\">34034606</pub-id><pub-id pub-id-type=\"doi\">10.1080/10837450.2021.1931886</pub-id></mixed-citation></ref><ref id=\"CR214\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yu</surname><given-names>CY</given-names></name><name><surname>Chan</surname><given-names>KG</given-names></name><name><surname>Yean</surname><given-names>CY</given-names></name><name><surname>Ang</surname><given-names>GY</given-names></name></person-group><article-title xml:lang=\"en\">Nucleic acid-based diagnostic tests for the detection SARS-CoV-2: an update</article-title><source>Diagnostics</source><year>2021</year><volume>11</volume><issue>1</issue><fpage>53</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhs1CjtLzK</pub-id><pub-id pub-id-type=\"pmid\">33401392</pub-id><pub-id pub-id-type=\"pmcid\">7823986</pub-id><pub-id pub-id-type=\"doi\">10.3390/diagnostics11010053</pub-id></mixed-citation></ref><ref id=\"CR215\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zazo</surname><given-names>H</given-names></name><name><surname>Colino</surname><given-names>CI</given-names></name><name><surname>Lanao</surname><given-names>JM</given-names></name></person-group><article-title xml:lang=\"en\">Current applications of nanoparticles in infectious diseases</article-title><source>J Control Release</source><year>2016</year><volume>224</volume><fpage>86</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XovFCgtA%3D%3D</pub-id><pub-id pub-id-type=\"pmid\">26772877</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jconrel.2016.01.008</pub-id></mixed-citation></ref><ref id=\"CR216\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>Q</given-names></name><name><surname>Honko</surname><given-names>A</given-names></name><name><surname>Zhou</surname><given-names>J</given-names></name><name><surname>Gong</surname><given-names>H</given-names></name><name><surname>Downs</surname><given-names>SN</given-names></name><name><surname>Vasquez</surname><given-names>JH</given-names></name><name><surname>Fang</surname><given-names>RH</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Griffiths</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Cellular nanosponges inhibit SARS-CoV-2 infectivity</article-title><source>Nano Lett</source><year>2020</year><volume>20</volume><issue>7</issue><fpage>5570</fpage><lpage>5574</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFKiu7rM</pub-id><pub-id pub-id-type=\"pmid\">32551679</pub-id><pub-id pub-id-type=\"doi\">10.1021/acs.nanolett.0c02278</pub-id></mixed-citation></ref><ref id=\"CR217\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>F</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Vijver</surname><given-names>MG</given-names></name><name><surname>Peijnenburg</surname><given-names>WJ</given-names></name></person-group><article-title xml:lang=\"en\">Probing nano-QSAR to assess the interactions between carbon nanoparticles and a SARS-CoV-2 RNA fragment</article-title><source>Ecotoxicol Environ Saf</source><year>2021</year><volume>219</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhtFynurvJ</pub-id><pub-id pub-id-type=\"pmid\">34044308</pub-id><pub-id pub-id-type=\"pmcid\">8133531</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ecoenv.2021.112357</pub-id></mixed-citation></ref><ref id=\"CR218\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>K</given-names></name><name><surname>Di</surname><given-names>L</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Yue</surname><given-names>P</given-names></name><name><surname>Song</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Traditional herbal medicine and nanomedicine: Converging disciplines to improve therapeutic efficacy and human health</article-title><source>Adv Drug Deliv Rev</source><year>2021</year><volume>178</volume><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVelsrzI</pub-id><pub-id pub-id-type=\"pmid\">34499982</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.addr.2021.113964</pub-id></mixed-citation></ref><ref id=\"CR219\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>SH</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>SJ</given-names></name><name><surname>Yao</surname><given-names>L</given-names></name><name><surname>Pan</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>YD</given-names></name></person-group><article-title xml:lang=\"en\">Diagnosis of COVID-19 pneumonia via a novel deep learning architecture</article-title><source>J Comput Sci Technol</source><year>2022</year><volume>37</volume><issue>2</issue><fpage>330</fpage><lpage>343</lpage><pub-id pub-id-type=\"pmid\">35496726</pub-id><pub-id pub-id-type=\"pmcid\">9035772</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11390-020-0679-8</pub-id></mixed-citation></ref><ref id=\"CR220\"><mixed-citation publication-type=\"other\">Zhang F, Abudayyeh OO, Gootenberg JS (2020a) A protocol for detection of COVID-19 using CRISPR diagnostics. A protocol for detection of COVID-19 using CRISPR diagnostics 8.</mixed-citation></ref><ref id=\"CR221\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhao</surname><given-names>W</given-names></name><name><surname>Zhong</surname><given-names>Z</given-names></name><name><surname>Xie</surname><given-names>X</given-names></name><name><surname>Yu</surname><given-names>Q</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Relation between chest CT findings and clinical conditions of coronavirus disease (COVID-19) pneumonia: a multicenter study</article-title><source>Ajr Am J Roentgenol</source><year>2020</year><volume>214</volume><issue>5</issue><fpage>1072</fpage><lpage>1077</lpage><pub-id pub-id-type=\"pmid\">32125873</pub-id><pub-id pub-id-type=\"doi\">10.2214/AJR.20.22976</pub-id></mixed-citation></ref><ref id=\"CR222\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhu</surname><given-names>J</given-names></name><name><surname>Shi</surname><given-names>X</given-names></name></person-group><article-title xml:lang=\"en\">Dendrimer-based nanodevices for targeted drug delivery applications</article-title><source>J Mater Chem B</source><year>2013</year><volume>1</volume><issue>34</issue><fpage>4199</fpage><lpage>4211</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXht1entrfP</pub-id><pub-id pub-id-type=\"pmid\">32261015</pub-id><pub-id pub-id-type=\"doi\">10.1039/c3tb20724b</pub-id></mixed-citation></ref><ref id=\"CR223\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ziegler</surname><given-names>CG</given-names></name><name><surname>Allon</surname><given-names>SJ</given-names></name><name><surname>Nyquist</surname><given-names>SK</given-names></name><name><surname>Mbano</surname><given-names>IM</given-names></name><name><surname>Miao</surname><given-names>VN</given-names></name><name><surname>Tzouanas</surname><given-names>CN</given-names></name><name><surname>Cao</surname><given-names>Y</given-names></name><name><surname>Yousif</surname><given-names>AS</given-names></name><name><surname>Bals</surname><given-names>J</given-names></name><name><surname>Hauser</surname><given-names>BM</given-names></name></person-group><article-title xml:lang=\"en\">SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues</article-title><source>Cell</source><year>2020</year><volume>181</volume><issue>5</issue><fpage>1016</fpage><lpage>1035</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXpsVKkuro%3D</pub-id><pub-id pub-id-type=\"pmid\">32413319</pub-id><pub-id pub-id-type=\"pmcid\">7252096</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.cell.2020.04.035</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"review-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">13197</journal-id><journal-id journal-id-type=\"doi\">10.1007/13197.0975-8402</journal-id><journal-title-group><journal-title>Journal of Food Science and Technology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Food Sci Technol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0022-1155</issn><issn pub-type=\"epub\">0975-8402</issn><publisher><publisher-name>Springer India</publisher-name><publisher-loc>New Delhi</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s13197-022-05528-8</article-id><article-id pub-id-type=\"manuscript\">5528</article-id><article-id pub-id-type=\"doi\">10.1007/s13197-022-05528-8</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Review Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Bioactive metabolites in functional and fermented foods and their role as immunity booster and anti-viral innate mechanisms</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Varsha</surname><given-names>Kontham Kulangara</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Narisetty</surname><given-names>Vivek</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Brar</surname><given-names>Kamalpreet Kaur</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref><xref ref-type=\"aff\" rid=\"Aff4\">4</xref></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Madhavan</surname><given-names>Aravind</given-names></name><xref ref-type=\"aff\" rid=\"Aff5\">5</xref></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Alphy</surname><given-names>Maria Paul</given-names></name><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref></contrib><contrib contrib-type=\"author\" id=\"Au6\"><name><surname>Sindhu</surname><given-names>Raveendran</given-names></name><xref ref-type=\"aff\" rid=\"Aff8\">8</xref></contrib><contrib contrib-type=\"author\" id=\"Au7\"><name><surname>Awasthi</surname><given-names>Mukesh Kumar</given-names></name><xref ref-type=\"aff\" rid=\"Aff9\">9</xref></contrib><contrib contrib-type=\"author\" id=\"Au8\"><name><surname>Varjani</surname><given-names>Sunita</given-names></name><xref ref-type=\"aff\" rid=\"Aff10\">10</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au9\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-4191-8048</contrib-id><name><surname>Binod</surname><given-names>Parameswaran</given-names></name><address><email>binodkannur@niist.res.in</email></address><xref ref-type=\"aff\" rid=\"Aff6\">6</xref><xref ref-type=\"aff\" rid=\"Aff7\">7</xref><xref ref-type=\"corresp\" rid=\"IDs13197022055288_cor9\">j</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.254567.7</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 9075 106X</institution-id><institution content-type=\"org-division\">School of Medicine</institution><institution content-type=\"org-name\">University of South Carolina</institution></institution-wrap><addr-line content-type=\"postcode\">29209</addr-line><addr-line content-type=\"city\">Columbia</addr-line><addr-line content-type=\"state\">SC</addr-line><country country=\"US\">USA</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Moolec Science, Innovation Centre</institution></institution-wrap><addr-line content-type=\"street\">Gallows Hill, CV34 6UW</addr-line><addr-line content-type=\"city\">Warwick</addr-line><country country=\"GB\">UK</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.21100.32</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1936 9430</institution-id><institution content-type=\"org-division\">Department of Civil Engineering, Lassonde School of Engineering</institution><institution content-type=\"org-name\">York University</institution></institution-wrap><addr-line content-type=\"postcode\">M3J 1P3</addr-line><addr-line content-type=\"city\">North York, Toronto</addr-line><addr-line content-type=\"state\">Ontario</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff4\"><label>4</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.468122.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 5906 3081</institution-id><institution content-type=\"org-name\">Centre Technologique des Résidus Industriels en Abitibi Témiscamingue</institution></institution-wrap><addr-line content-type=\"street\">433 Boulevard du collège, J9X0E1</addr-line><addr-line content-type=\"city\">Rouyn-Noranda</addr-line><country country=\"CA\">Canada</country></aff><aff id=\"Aff5\"><label>5</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.418917.2</institution-id><institution-id institution-id-type=\"ISNI\">0000 0001 0177 8509</institution-id><institution content-type=\"org-name\">Rajiv Gandhi Centre for Biotechnology</institution></institution-wrap><addr-line content-type=\"postcode\">695 014</addr-line><addr-line content-type=\"city\">Thiruvananthapuram</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff6\"><label>6</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.419023.d</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1808 3107</institution-id><institution content-type=\"org-division\">Microbial Processes and Technology Division</institution><institution content-type=\"org-name\">CSIR-National Institute for Interdisciplinary Science and Technology</institution></institution-wrap><addr-line content-type=\"postcode\">695 019</addr-line><addr-line content-type=\"city\">Trivandrum</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff7\"><label>7</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.469887.c</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 7744 2771</institution-id><institution content-type=\"org-name\">Academy of Scientific and Innovative Research (AcSIR)</institution></institution-wrap><addr-line content-type=\"postcode\">201002</addr-line><addr-line content-type=\"city\">Ghaziabad</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff8\"><label>8</label><institution-wrap><institution content-type=\"org-division\">Department of Food Technology</institution><institution content-type=\"org-name\">T K M Institute of Technology</institution></institution-wrap><addr-line content-type=\"postcode\">691505</addr-line><addr-line content-type=\"city\">Kollam</addr-line><addr-line content-type=\"state\">Kerala</addr-line><country country=\"IN\">India</country></aff><aff id=\"Aff9\"><label>9</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.144022.1</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1760 4150</institution-id><institution content-type=\"org-division\">College of Natural Resources and Environment</institution><institution content-type=\"org-name\">Northwest A &amp; F University</institution></institution-wrap><addr-line content-type=\"postcode\">712 100</addr-line><addr-line content-type=\"city\">Yangling</addr-line><addr-line content-type=\"state\">Shaanxi</addr-line><country country=\"CN\">China</country></aff><aff id=\"Aff10\"><label>10</label><institution-wrap><institution content-type=\"org-name\">Gujarat Pollution Control Board</institution></institution-wrap><addr-line content-type=\"street\">Paryavaran Bhavan, CHH Road, Sector 10 A</addr-line><addr-line content-type=\"postcode\">382010</addr-line><addr-line content-type=\"city\">Gandhinagar</addr-line><addr-line content-type=\"state\">Gujarat</addr-line><country country=\"IN\">India</country></aff></contrib-group><author-notes><corresp id=\"IDs13197022055288_cor9\"><label>j</label><email>binodkannur@niist.res.in</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>24</day><month>6</month><year>2022</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>9</month><year>2023</year></pub-date><volume>60</volume><issue seq=\"2\">9</issue><fpage>2309</fpage><lpage>2318</lpage><history><date date-type=\"registration\"><day>1</day><month>6</month><year>2022</year></date><date date-type=\"rev-recd\"><day>19</day><month>4</month><year>2022</year></date><date date-type=\"accepted\"><day>18</day><month>5</month><year>2022</year></date><date date-type=\"online\"><day>24</day><month>6</month><year>2022</year></date></history><permissions><copyright-statement content-type=\"compact\">© Association of Food Scientists &amp; Technologists (India) 2022</copyright-statement><copyright-year>2022</copyright-year><copyright-holder>Association of Food Scientists &amp; Technologists (India)</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par15\">Live microorganisms in the fermented foods termed probiotics and their secondary metabolites with bioactive potential were considered as potential anti-viral capabilities through various mechanisms. Given the importance of functional and fermented foods in disease prevention, there is a need to discuss the contextualization and deep understanding of the mechanism of action of these foods, particularly considering the appearance of coronavirus (COVID-19) pandemic, which is causing health concerns and increased social services globally. The mechanism of probiotic strains or their bioactive metabolites is due to stimulation of immune response through boosting T-lymphocytes, cytokines, and cell toxicity of natural killer cells. Proper consumption of these functional and fermented foods may provide additional antiviral approaches for public benefit by modulating the immune functions in the hosts.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd>Anti-viral metabolites</kwd><kwd>Immunity</kwd><kwd>Fermented foods</kwd><kwd>COVID-19</kwd><kwd>Functional foods</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>19</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Association of Food Scientists &amp; Technologists (India)</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2022</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/13197_2022_Article_5528.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>7</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>ReviewPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Kontham Kulangara Varsha and Vivek Narisetty these authors contributed equally and Co-first authors.</p></notes><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s13197-022-05528-8\">https://doi.org/10.1007/s13197-022-05528-8</ext-link>.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par42\">The process of fermentation has long been used to increase the shelf life, flavour and functional properties of food. In addition to help food last longer, fermentation increases the nutritional value, and the probiotic bacteria present in fermented food confer health benefits that comprise reducing the risk of type 2 diabetes and cardiovascular diseases by decreasing total and LDL cholesterol (Marco et al. <xref ref-type=\"bibr\" rid=\"CR48\">2017</xref>). Multiple clinical trials investigated the advantages of fermented food and exposed the ability of kimchi and yogurt to downsize the risk of type 2 diabetes (Chen et al. <xref ref-type=\"bibr\" rid=\"CR7\">2014</xref>), Chungkookjang to decrease obesity (Byun et al. <xref ref-type=\"bibr\" rid=\"CR4\">2016</xref>), and the capacity of fermented milk and rye bread to control infection and irritable bowel syndrome (Laatikainen et al. <xref ref-type=\"bibr\" rid=\"CR39\">2016</xref>). Among this health promoting and infectivity demoting effects include the antiviral activity of certain fermented foods owing to the presence of live bacteria in it, and the examples are Chr. Hansen (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.chr-hansen.com/\">http://www.chr-hansen.com/</ext-link>); Kingdom Supercultures (<ext-link ext-link-type=\"uri\" xlink:href=\"https://kingdomsupercultures.com/\">https://kingdomsupercultures.com/</ext-link>); Probitat (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.probitat.eu/\">http://www.probitat.eu/</ext-link>); 3FBIO Ltd (ENOUGH) (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.enough-food.com/\">https://www.enough-food.com/</ext-link>); Fermbiotics (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fermbiotics.com/\">https://www.fermbiotics.com/</ext-link>).</p><p id=\"Par43\">Functional foods (FFs) are distinguished by their nutraceuticals. Nutraceuticals are either whole foods or food ingredients that provide health benefits, such as disease prevention and/or treatment. This is generally associated with their microvascular, anti-inflammatory, and anti-oxidation properties in highly affected individuals. Active FFs are rich in polyphenols, terpenoids, flavonoids, and unsaturated fatty acids ingredients are among the widely active functional foods to be consumed (Alkhatib et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018</xref>). Recently, Acquah et al. (<xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>) reported that some bioactive peptides presented many similar hormonal and neurological activities of human system (Acquah et al. <xref ref-type=\"bibr\" rid=\"CR1\">2020</xref>). On the other side, fermented food products contain probiotics (García-Burgos et al. <xref ref-type=\"bibr\" rid=\"CR13\">2020</xref>). Probiotics are “live microorganisms which when administered in adequate amounts confer a health benefit on the host”. Covid-19 was observed to be a severe acute respiratory syndrome (SARS) which is named as SARS corona virus 2 or SARS-CoV-2 (Lai et al. <xref ref-type=\"bibr\" rid=\"CR40\">2020</xref>). Broadly, coronaviruses are large and enveloped mainly found in humans and mammals and known to cause respiratory, gastrointestinal, and neurological disease. Through genetic recombination and mutation, corona viruses can be more infectious. Stringent measures were taken by various countries based on their resource limitations, geography, population, and political factors. Although these severe interventions, since Feb. 2020 and as of March 17, 2022, the outbreak has infected almost 464 million people and killed over 6.06 million. Previous studies have shown that 65% of airborne MERS-CoV virus remains viable in the air and infectious after 60 min (Pyankov et al. <xref ref-type=\"bibr\" rid=\"CR61\">2018</xref>). The presence of SARS-CoV-2 in hospitals and entrance to department stores in Wuhan, China (Liu et al. <xref ref-type=\"bibr\" rid=\"CR45\">2020</xref>), air outlet fans in a COVID-19 outbreak center in Singapore (Ong et al. <xref ref-type=\"bibr\" rid=\"CR57\">2020</xref>) and hospital isolation rooms in Nebraska (Santarpia et al. <xref ref-type=\"bibr\" rid=\"CR63\">2020</xref>) were observed. Recently, some preliminarily results have shown that the virus can survive up to 3 h as aerosol and infect cells throughout this period (van Doremalen et al. <xref ref-type=\"bibr\" rid=\"CR67\">2020</xref>). A ferret model of SARS-CoV-2 infection that reiterates aspects of human disease has also confirmed the potential of virus airborne transmission (Kim et al. <xref ref-type=\"bibr\" rid=\"CR34\">2020</xref>). A recent study reported that SARS-CoV-2 can be viable for 4 h on copper, 24 h on cardboard, 2–3 days on plastic and stainless steel (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Generally, SARS-CoV-2 is transmitted via respiratory pathways, but they may spread via multiple dominant routes. An acceptor individual must receive an infectious dose of the virus from a donor, either directly through the air or indirectly through deposits of the virus on various surfaces. Molecular based detection techniques, like plaque assay, Enzyme-Linked Immunosorbent Assay (ELISA), Lateral-Flow (immuno) Assay (LFA), Polymerase Chain Reaction (PCR) and Surface Plasmon Resonance (SPR) assay are identified quality and quantitative techniques for virus detection. Approximately 120 candidates are hardly working to formulate vaccines based on either nucleic acids, inactivated or live attenuated virus, and recombinant proteins (Le et al. 2020). Other approaches based on monoclonal antibodies, hyperimmune globulin will be another option. The antiviral role of FFs and fermented foods for the defence of COVID-19 is lacking and has not yet been established. However, recent reports showed that diabetes is considered a risk factor for the development and diagnosis of COVID-19 (Guo et al. <xref ref-type=\"bibr\" rid=\"CR22\">2020</xref>). Fermented foods generally contain single or multiple genera of live probiotic microorganisms that has positive impact on the host beneficially preserving the intestinal microbiota that in return has modulatory role in immune responses and human health (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). Herein, we present antiviral role of fermented foods containing of probiotic and bioactive compounds; mechanism to stimulate immunity; case studies of viral infections and their prevention or treatment and different types of fermented foods available worldwide. This review leads to inculcating the knowledge of benefits of probiotics and fermented foods in reducing the risk of COVID-19 or other viral infections.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Schematic representation of applications of probiotics and bioactive metabolites</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec2\"><title>Antiviral activity of fermented foods containing probiotics and bioactive compounds</title><p id=\"Par45\">The plethora of fermented foods is derived from plants and animal resources. Plant-based fermented foods are tempeh, <italic>kimchi</italic>, <italic>tempoyak</italic>, and <italic>tapai</italic>. While animal-based fermented foods include kefir, cheese, yogurt, sauces, other dairy and meat products (Tamang et al., 2016; Raji et al. <xref ref-type=\"bibr\" rid=\"CR62\">2017</xref>; Khalil et al. <xref ref-type=\"bibr\" rid=\"CR32\">2018</xref>; Lee et al. <xref ref-type=\"bibr\" rid=\"CR44\">2020</xref>). Alcoholic beverages like fermented porridge made of sorghum and maize in South American countries (Tamang et al. <xref ref-type=\"bibr\" rid=\"CR66\">2020</xref>). Fermented foods are enriched with anti-microbial end products such as various organic acids, ethanol and peptides or bacteriocins and several studies report the antiviral potential of fermented foods <italic>in-vitro</italic> and <italic>in-vivo</italic>. The probiotic bacteria and bioactive compounds in fermented foods possess antiviral activities against gut and respiratory and viruses. These active foods stimulate immune system function by increasing the synthesis of pro-inflammatory cytokines and T lymphocytes (CD3<sup>+</sup>, CD16<sup>+</sup>, CD56<sup>+</sup>) (Muhialdin et al. <xref ref-type=\"bibr\" rid=\"CR54\">2021</xref>). <italic>Lactobacillus plantarum</italic> LBP-K10 isolated from kimchi synthesised cyclic di-peptides that inhibited the growth of the influenza A (H3N2) virus (Kwak et al. <xref ref-type=\"bibr\" rid=\"CR38\">2013</xref>), while another study reported declined survival of feline calicivirus and murine norovirus proliferation during <italic>Dongchimi</italic> fermentation along with an increase in lactic acid bacteria (LAB) (Lee et al. <xref ref-type=\"bibr\" rid=\"CR42\">2012</xref>). Likewise, soy extracts fermented with <italic>Aspergillus fumigatus</italic> F-993 or <italic>A. awamori</italic> FB-133 showed therapeutic potential by decreasing hepatitis A virus titers <italic>in-vitro</italic> (Ghanem et al. <xref ref-type=\"bibr\" rid=\"CR14\">2020</xref>).</p><p id=\"Par46\">The cell free supernatant of yogurt has antiviral activity for RNA viruses such as enterovirus 71 and influenza, porcine epidemic diarrhoea virus and Coxsackie A and B viruses (Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>). Polyphenols, bioactive peptides, exopolysaccharides, linoleic acid, and vitamins are among the bioactive compounds found in fermented foods (Hayes and García-Vaquero <xref ref-type=\"bibr\" rid=\"CR24\">2016</xref>). Spanish sausage release angiotensin-converting enzyme inhibitor (ACE-I) when <italic>L. pentosus</italic> and <italic>S. carnosus</italic> used as a inoculum for the fermentation (Mora et al. <xref ref-type=\"bibr\" rid=\"CR51\">2015</xref>). The fermentation of <italic>Ruditapes philippinarum</italic> clams with <italic>Bacillus natto</italic> stimulate hyper-production of ACE-I peptide synthesis exerts anti-cancer property (Chen et al. <xref ref-type=\"bibr\" rid=\"CR8\">2018</xref>). However, further studies assessing the full potential of the probiotics to combat COVID-19 should be carried out (Olaimat et al. <xref ref-type=\"bibr\" rid=\"CR55\">2020</xref>) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>).</p></sec><sec id=\"Sec3\"><title>Clinical impact of probiotics and bioactive compounds against viral infections in humans</title><p id=\"Par47\">High mutation rates of RNA viruses lead to their rapid evolution and better environmental adaptability (Carrasco-Hernandez et al. <xref ref-type=\"bibr\" rid=\"CR6\">2017</xref>). Various case studies report the ability of fermented and probiotic food to reduce respiratory tract infections. In one case study, some COVID-19 patients exhibited <italic>Bifidobacterium</italic> and <italic>Lactobacillus</italic> dysbiosis. The aged patients who suffered severely from COVID-19 had poor gut microbiota diversity (Dhar et al., 2020). Probiotics <italic>L. acidophilus, Bifidobacterium</italic>, and <italic>Saccharomyces boulardii</italic>, as well as minerals and vitamins, were found to reduce the complications of massive antibiotic-associated diarrhoea and <italic>Clostridium difficile</italic> infections. (Horowitz et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). The simultaneous intake of probiotics with azithromycin decreases the severity of <italic>Candida albicans</italic> infection. In another study, COVID-19-like symptoms in a young boy disappeared after 2 days of probiotic administration. (Ji et al. <xref ref-type=\"bibr\" rid=\"CR28\">2020</xref>). The administration of probiotics as an adjunct protected 97% patients against SARS-CoV2 infection. According to the recent trail, patients with severe disease recovered more easily by probiotics therapy (87.5% vs. 40.4%, p = 0.037) compared to non-severe ones. A human clinical trial describes oral intake of the probiotic <italic>L. fermentum</italic> CECT5716 increased NK cells proliferation after vaccination when compared to the group without probiotic consumption (Olivares et al. <xref ref-type=\"bibr\" rid=\"CR56\">2007</xref>). A study conducted on elderly people found to be having a significant increase in NK cell activity (Makino et al. <xref ref-type=\"bibr\" rid=\"CR46\">2010</xref>). At the same time, a randomized control clinical trial among women consumed 1073R-1-yogurt yogurt (n = 479) demonstrated IFN-γ production with no increase in NK cell activity (Kinoshita et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>).</p><p id=\"Par48\">A control trial revealed that the probiotic strain <italic>Lactobacillus</italic> GG holds an adjuvant potential (Davidson et al. <xref ref-type=\"bibr\" rid=\"CR11\">2011</xref>). In children, for acute rotavirus diarrhoea, probiotic administration can relieve symptoms (Grandy et al. <xref ref-type=\"bibr\" rid=\"CR17\">2010</xref>). Multiple clinical trials are underway where the adeptness of probiotic and other dietary supplements is being investigated to alleviate the symptoms of COVID-19 infection (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>).</p><p id=\"Par49\">In the last two years of CoviD-19 pandemic, priority was to prevent the spreading, reduce the infections, save the lives, inculcate the knowledge of vaccination and healthy diet. Frontline health teams would benefit from the development of cutting-edge technology and the collection of available evidence. Sufficient nutrition improves health and boosts immunity, which aids in the prevention and treatment of infections. In this review we have discussed the role of probiotics in combating COVID-19 based on recent evidences, as well as their role as immune-modulators and antiviral agents. Further investigations on impact of probiotic strains and their bioactive compounds on COVID-19 affected individuals are awaiting (Kurian et al. <xref ref-type=\"bibr\" rid=\"CR37\">2021</xref>). However, more research needs to be done to study the production titres of interleukins, interferons, antibodies, and viral count due to probiotic administration during viral infection especially SARS-CoV-2.</p></sec><sec id=\"Sec4\"><title>Immune modulatory mechanisms by probiotics and fermented food</title><p id=\"Par50\">The health benefits conferred by fermented food are mainly attributed to the presence of live probiotic bacteria present which colonize the intestine and impart favourable effects. Lymphocytes, NK cells, macrophages and neutrophils are capable of mediating antibody dependent cell mediated cytotoxicity (ADCC) against virus infected cells and various probiotic and fermented food are reported to induce these immune cells. Multiple clinical trials have shown the ability of fermented food products with probiotics to improve NK cell activity and <italic>L. casei</italic> strain Shirota is a principal probiotic strain with this ability (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Additionally, consumption of dairy yogurt containing probiotics increased NK cell, IL-12, IFNγ and IgG1 levels in a randomized, open-label, placebo-controlled study conducted in 200 healthy volunteers (Lee et al. <xref ref-type=\"bibr\" rid=\"CR43\">2017</xref>). Probiotics play critical role in resistance against viral infections particularly in elderly people with age-related decline in lymphoid cell activity, by modulating the immune responses of hosts. Randomized control trials show that probiotic consumption significantly increased NK cell activity and CD56-positive lymphocytes in peripheral circulation in healthy elderly individuals (Gui et al. <xref ref-type=\"bibr\" rid=\"CR20\">2020</xref>). Probiotic intakes enhance cellular immunity in elderly who had poor pretreatment immune responses. Dietary supplementation of probiotic <italic>B. lactis</italic> HN019 increased helper (CD4 (+)) and activated (CD25 (+)) T lymphocytes, polymorphonuclear cells and natural killer cells in healthy elderly volunteers (Gill et al. <xref ref-type=\"bibr\" rid=\"CR16\">2001</xref>). Probiotic consumption found to increase natural and acquired immunity in mice and significantly improved serum antibody responses to antigens administered orally and systemically (Gill et al. <xref ref-type=\"bibr\" rid=\"CR15\">2000</xref>). Oral administration of heat killed probiotics from Mongolian dairy products augmented IFN -α, IL-12, and IFN -γ productions and increased NK cell activity leading to alleviated influenza symptoms in mice (Takeda and Okumura <xref ref-type=\"bibr\" rid=\"CR65\">2007</xref>). Similarly, oral administration of <italic>L. rhamnoses</italic> CRL1505 improved resistance to RSV infection in infant mice via IFN-γ and IL-10 secretion, which resulted in the activation of CD103<sup>+</sup> and CD11b<sup>high</sup> dendritic cells and the generation of CD3<sup>+</sup>CD4<sup>+</sup>IFN-γ<sup>+</sup> Th1 cells, with subsequent attenuation of strong Th2 reactions associated with RSV challenge (Chiba et al. <xref ref-type=\"bibr\" rid=\"CR9\">2013</xref>). An in vitro model study demonstrated the ability of probiotic bacteria to decrease vesicular stomatitis viral infection by production of nitric oxide and inflammatory cytokines such as IL-6 and IFN-γ (Ivec et al. <xref ref-type=\"bibr\" rid=\"CR27\">2007</xref>). When <italic>L. casei</italic> DK128 was administered into a mice intranasally, reduction in the weight and viral loads was observed, which might elicit the protection against different subtypes of influenza viruses, and mice are observed to be immune to primary infection and subsequently developed heterosubtypic secondary virus infection. The protective effect was linked to an increase in alveolar macrophage cells in the lungs and airways, the early induction of virus-specific antibodies, and lower levels of pro-inflammatory cytokines and innate immune cells (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>) (Jung et al. <xref ref-type=\"bibr\" rid=\"CR29\">2017</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Possible mechanism of anti-viral activity by probiotics</p></caption><graphic mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/13197_2022_5528_Fig2_HTML.png\"/></fig></p><p id=\"Par52\">Exopolysaccharides (EPS) secreted by probiotic microrganisms in fermented food contribute towards their immunomodulatory ability and antiviral potential <italic>in-vivo</italic>. Consumption of yogurt consisting of the starter culture <italic>L. delbrueckii</italic> ssp. <italic>bulgaricus</italic> OLL1073R-1 and secreted EPS reduced influenza virus titer in mice. The prognosis observed significant increase of anti-influenza virus antibodies such as IgA and IgG<sub>1</sub> along with augmented NK cell activity. In knockout mice, the presence of myeloid differentiation factor 88, EPS produced by this probiotic strain activated NK cells through IL-12- and IL-18-mediated IFN-γ production. The same probiotic strain demonstrated resistance flu virus by inducing NK cell activity in human subjects (Makino et al. <xref ref-type=\"bibr\" rid=\"CR47\">2016</xref>). EPS produced by <italic>L. delbrueckii</italic> TUA4408L improved the resistance to rotaviral infection by preventing the viral replication, activation of Toll-like receptor 3 rendering antiviral innate immune response in porcine intestinal epithelial cells. The study also reports <italic>L. delbrueckii</italic> TUA4408L and its EPS activated interferon regulatory factor (IRF)-3 and nuclear factor κB (NF-κB) signalling pathways leading to improved expression of the antiviral factors IFN-β, Myxovirus resistance gene A (MxA) and RNaseL (Kanmani et al. <xref ref-type=\"bibr\" rid=\"CR30\">2018</xref>). In addition to EPS, short chain fatty acids (SCFAs) produced by these microorganisms are observed to regulate immune responses (Parada Venegas et al. <xref ref-type=\"bibr\" rid=\"CR58\">2019</xref>). Indirectly D-phenyl lactate produced by various lactic acid bacteria regulates the immune reactions stimulated by G-protein coupled receptors by activating regulatory hydroxycarboxylic acids (Peters et al. <xref ref-type=\"bibr\" rid=\"CR59\">2019</xref>). Lactate and pyruvate also contribute to enhanced immune responses in mice models by inducing GPR31-mediated dendrite protrusion of intestinal CX3CR1<sup>+</sup> cells (Morita et al. <xref ref-type=\"bibr\" rid=\"CR52\">2019</xref>). The modulation of gut microbiota seems to be a one of the approaches to combat viral infections including COVID-19, but it needs to be further confirmed through animal models (Akour <xref ref-type=\"bibr\" rid=\"CR2\">2020</xref>). Overall, research indicates that fermented foods and probiotics contain high or low molecular weight bioactive metabolites that elicit modulations in the immune system rendering health benefits.</p></sec><sec id=\"Sec5\"><title>Future outlook</title><p id=\"Par53\">Fermentation by microbes applies various unconventional and uncharacterised enzymes to produce hydrolysates of protein with varying compositions of peptides. By utilizing highly efficient bacteria and by optimizing conditions of fermentation, a wide array of proteases could be produced to hydrolyse peptides and proteins with varying amino acid composition and different chain length could be synthesised in accordance to different enzyme specificity, possibly producing unique sequences of peptide with novel anti-viral bioactive properties.</p><p id=\"Par54\">It is predictable that the greatest perplexing part for researchers dealing with functional food bioactive peptides is to warrant the full bioavailability of the bioactive component after their consumption. The bioavailability of bioactive peptide is based on the capability of the peptides to resist proteolytic activity in the intestine and serum, and their potential to enter into the blood stream and consequently, exercising their biological activity. Hence, it is critical to improve the processing settings in order to retain their biological activities. Encapsulation is one of the well-characterised technologies to improve the bioavailability of functionally active peptides. This technique has been extensively applied in the nutraceutical, and food industries to encapsulate biologically active components. Encapsulation of bioactive is economically viable as this technique could avoid the interaction of the peptides with environment and/or other components in the food matrix, decrease the consequence from processing, intensify peptide stability, protect peptides against digestive enzymes and improve bioavailability.</p><p id=\"Par55\">Since functionally active foods are graded as a normal or enriched with bioactive, it has certain side effects, although the research work in this perspective is not adequate to support this statement. Even though much cell line based, and animal model studies have been widely explored to examine the bioactivity of food derived bioactive, inadequate data on human experiments is available. More research data on human clinical trials are required to validate the efficiency of food derived bioactivities. Along with this other safety parameters, like cytotoxicity, allergic response of the functional food should be evaluated before commercialization.</p><p id=\"Par56\">Viral diseases are considered as an immune compromised state resulting from poor consumption of micronutrients, vitamins and other trace elements. Several research previously indicated enhanced function of the immune system by taking those many fermented and functional foods including essential fatty and amino acids, and the above-mentioned minerals and vitamins (Calder and Kew <xref ref-type=\"bibr\" rid=\"CR5\">2002</xref>). Satisfactory diet-based consumption, and supplementation of such functional foods, contribute to sustaining optimal levels in the humans, which improves several characteristics of the immune function, and provides an important antiviral prevention of COVID-19. On the other hand, less strong immune activation has been proved to be the primary threat factor for COVID-19, which makes it appropriate to define the defensive role of functional food particles benefits in the perspective of preventing COVID-19 and other viral diseases (Grant et al. <xref ref-type=\"bibr\" rid=\"CR18\">2020</xref>).</p><p id=\"Par57\">Examination of outcome of viral disease management in high-risk populations and aged people is very central. Large number of viral and COVID 19 infection rate is reported in older adults and persons with other co-morbidities. The prevalence of COVID-19 in people with diabetes is high and now considered a hazard factor for the progression COVID-19 (McGurnaghan et al. <xref ref-type=\"bibr\" rid=\"CR49\">2021</xref>). Therefore, best “immune-modulating” functional foods could provide the finest prevention and progression of the viral diseases. The use of functional foods to combat viral diseases would especially benefit the elderly, which has become a growing sector of the world population.</p></sec><sec id=\"Sec6\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par58\">Fermentation by microbe’s functions as a potent technique to supplement foods with biologically active peptides from various animal or plant sources. These bioactive compounds produced in the fermented foods tends to boost the immune response either directly or indirectly against the viral infections by modulating the lymphocytes, NK cells, macrophages and neutrophils that are capable of mediating antibody dependent cell mediated cytotoxicity. Several works have been documented to unveil novel bioactive components from fermented foods and other edible plant by products. It is possible that the varieties of biopeptides from various fermented food will continue to develop in the coming years. Further awareness of these health beneficial fermented foods and their compositions could help in improving the health and wellness of the society.</p><p id=\"Par59\"><table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Antiviral bioactives from functional and fermented food</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"2\"><p>Probiotic microbes or Fermented foods</p></th><th align=\"left\"><p>Salient Feature</p></th><th align=\"left\"><p>Mode of action</p></th><th align=\"left\"><p>References</p></th></tr></thead><tbody><tr><td align=\"left\" colspan=\"2\"><p><italic>Bifidobacterium animalis</italic></p></td><td align=\"left\"><p>Prevent</p><p>upper respiratory tract infection</p></td><td align=\"left\"><p>Inhibit viral replication</p></td><td align=\"left\"><p>(Smith et al. <xref ref-type=\"bibr\" rid=\"CR64\">2013</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Prevent gastroenteritis COVID virus</p></td><td align=\"left\"><p>Reduces granulocyte, diminishes virus recovery</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. lactis</italic></p></td><td align=\"left\"><p>Prevent respiratory tract infection</p></td><td align=\"left\"><p>Activates plasmacytoid dendrtitic cell</p></td><td align=\"left\"><p>(Kokubo et al. <xref ref-type=\"bibr\" rid=\"CR36\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>L. plantarum</italic></p></td><td align=\"left\"><p>Gastroenteritis coronaviruses (TGEV)</p></td><td align=\"left\"><p>Reduces inflammation and tissue injury</p></td><td align=\"left\"><p>(Yang et al. <xref ref-type=\"bibr\" rid=\"CR72\">2017</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p><italic>Lactobacillus casei</italic> DN-114 001; Dan Active/Actimel</p></td><td align=\"left\"><p>Reduced incidence and duration of RTIs</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Guillemard et al. <xref ref-type=\"bibr\" rid=\"CR21\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Kefir</p></td><td align=\"left\"><p>Zika, hepatitis C, influenza, rotaviruses</p></td><td align=\"left\"><p>Enhanced macrophage synthesis, increases phagocytosis, enhanced synthesis of (CD4+), CD8 + cells, immunoglobulins, neutrophilss, and various cytokines (IL-2, IL-12, INF-γ).</p></td><td align=\"left\"><p>(Hamida et al. <xref ref-type=\"bibr\" rid=\"CR23\">2021</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Yoghurt</p></td><td align=\"left\"><p>Inhibit Enterovirus</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Choi et al. <xref ref-type=\"bibr\" rid=\"CR10\">2010</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Fermented ginseng extracts</p></td><td align=\"left\"><p>Inhibit influenza virus H1N1, H3N2, H5N1, and H7N9 strains</p></td><td align=\"left\"><p>Viral inoculation with extract of ginseng formed better immune responses against the 2<sup>0</sup> infection with homologous and heterosubtypic virus.</p></td><td align=\"left\"><p>(Wang et al. <xref ref-type=\"bibr\" rid=\"CR68\">2018</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Black ginseng</p></td><td align=\"left\"><p>Inhibit influenza virus</p></td><td align=\"left\"><p>Black ginseng improved the levels of GM-CSF and IL-10 at the time of infection</p></td><td align=\"left\"><p>(Kim et al. <xref ref-type=\"bibr\" rid=\"CR33\">2019</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Dietary xylitol</p></td><td align=\"left\"><p>Inhibit influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Yin et al. <xref ref-type=\"bibr\" rid=\"CR73\">2014</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Chongkukjang (Traditional Korean feremented food)</p></td><td align=\"left\"><p>Influenza virus A</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Wei et al. <xref ref-type=\"bibr\" rid=\"CR69\">2015b</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Resveratrol (From red grapes)</p></td><td align=\"left\"><p>Inhibit Epstein-Barr virus</p></td><td align=\"left\"><p>Downregulation of antiapoptotic proteins</p></td><td align=\"left\"><p>(De Leo et al. <xref ref-type=\"bibr\" rid=\"CR12\">2012</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Zingiberofficinale (Ginger)</p></td><td align=\"left\"><p>Anti-chikungunya activity</p></td><td align=\"left\"><p>−</p></td><td align=\"left\"><p>(Kaushik et al. <xref ref-type=\"bibr\" rid=\"CR31\">2020</xref>)</p></td></tr><tr><td align=\"left\" colspan=\"2\"><p>Curcumin</p></td><td align=\"left\"><p>Inhibit Zika and chikungunya viruses</p></td><td align=\"left\"><p>Curcumin interferes with virus-cell binding.</p></td><td align=\"left\"><p>(Mounce et al. <xref ref-type=\"bibr\" rid=\"CR53\">2017</xref>)</p></td></tr></tbody></table></table-wrap></p><p id=\"Par60\"><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Ongoing clinical trials where probiotics and dietary supplements are used against COVID-19</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>No.</p></th><th align=\"left\"><p>Clinical trial identifier</p></th><th align=\"left\"><p>Intervention</p></th><th align=\"left\"><p>Aim</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>NCT04621071</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Evaluation of the efficacy of probiotics to dcrease the duration and symptoms of COVID-19</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>NCT04458519</p></td><td align=\"left\"><p>Probiorinse</p></td><td align=\"left\"><p>Reduction of severity of COVID-19 symptoms</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>NCT04390477</p></td><td align=\"left\"><p>Probiotic</p></td><td align=\"left\"><p>Effect of probiotic on COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>NCT04366180</p></td><td align=\"left\"><p><italic>L. coryniformis</italic> K8</p></td><td align=\"left\"><p>Effect of probiotic in the incidence and prevention of COVID-19 infection in health workers</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>NCT04734886</p></td><td align=\"left\"><p><italic>L. reuteri</italic> DSM 17,938</p></td><td align=\"left\"><p>Impact of probiotic supplementation on SARS-CoV-2 specific antibody response following COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>NCT04666116</p></td><td align=\"left\"><p>Dietary supplements including probiotics</p></td><td align=\"left\"><p>To check the changes in viral load in COVID-19 infection</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>NCT04847349</p></td><td align=\"left\"><p>Probiotics</p></td><td align=\"left\"><p>Efficacy of probiotic intervention to boost the immunity in unvaccinated people infected previously with SARS-CoV-2</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>NCT04420676</p></td><td align=\"left\"><p>Synbiotic (Omnibiotic AAD)</p></td><td align=\"left\"><p>To check the ability to reduce gastrointestinal problems in COVID-19 patients</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>NCT04813718</p></td><td align=\"left\"><p>Omni-Biotic Pro Vi 5</p></td><td align=\"left\"><p>Analysis of post-Covid syndrome</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>NCT04399252</p></td><td align=\"left\"><p><italic>L. rhamnosus</italic> GG</p></td><td align=\"left\"><p>In order to study the consequence of microbiome in COVID-19 exposed household contacts</p></td></tr></tbody></table></table-wrap></p></sec></body><back><sec sec-type=\"author-contribution\"><title>Author's contribution</title><p>KKV, VN, AM, KKB, and MPA: Collecting articles and Writing original draft. RS: Conceptualization, Methodology, Writing - original draft. MKA, SV: Writing, Reviewing and Editing, Formal analysis. PB: Project Administration, Conceptualization and Visualization.</p></sec><sec><title>Funding</title><p>(Information that explains whether and by whom the research was supported): No funding to mention and no funding received for the particular work mentioned in the manuscript.</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec><p>The authors declare that the work described has not been published before (except in the form of an abstract, a published lecture or academic thesis),  it is not under consideration for publication elsewhere, its submission to JFST publication has been approved by all authors as well as the responsible authorities – tacitly or explicitly – at the institute where the work has been carried out, if accepted, it will not be published elsewhere in the same form, in English or in any other language, including electronically without the written consent of the copyright holder, and  JFST will not be held legally responsible should there be any claims for compensation or dispute on authorship.</p></sec><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Acquah</surname><given-names>C</given-names></name><name><surname>Agyei</surname><given-names>D</given-names></name><name><surname>Obeng</surname><given-names>EM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aptamers: an emerging class of bioaffinity ligands in bioactive peptide applications</article-title><source>Crit Rev Food Sci Nutr</source><year>2020</year><volume>60</volume><fpage>1195</fpage><lpage>1206</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVCms7c%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10408398.2018.1564234</pub-id><pub-id pub-id-type=\"pmid\">30714390</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Akour</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Probiotics and COVID-19: is there any link?</article-title><source>Lett Appl Microbiol</source><year>2020</year><volume>71</volume><fpage>229</fpage><lpage>234</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhs1ShtLrJ</pub-id><pub-id pub-id-type=\"doi\">10.1111/lam.13334</pub-id><pub-id pub-id-type=\"pmid\">32495940</pub-id><pub-id pub-id-type=\"pmcid\">7300613</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alkhatib</surname><given-names>A</given-names></name><name><surname>Tsang</surname><given-names>C</given-names></name><name><surname>Tuomilehto</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Olive oil nutraceuticals in the prevention and management of diabetes: from molecules to lifestyle</article-title><source>Int J Mol Sci</source><year>2018</year><volume>19</volume><fpage>2021</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisFKltb7J</pub-id><pub-id pub-id-type=\"doi\">10.3390/ijms19072024</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Byun</surname><given-names>M-S</given-names></name><name><surname>Yu</surname><given-names>O-K</given-names></name><name><surname>Cha</surname><given-names>Y-S</given-names></name><name><surname>Park</surname><given-names>T-S</given-names></name></person-group><article-title xml:lang=\"en\">Korean traditional Chungkookjang improves body composition, lipid profiles and atherogenic indices in overweight/obese subjects: a double-blind, randomized, crossover, placebo-controlled clinical trial</article-title><source>Eur J Clin Nutr</source><year>2016</year><volume>70</volume><fpage>1116</fpage><lpage>1122</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhtVWgu77K</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2016.77</pub-id><pub-id pub-id-type=\"pmid\">27302672</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Calder</surname><given-names>PC</given-names></name><name><surname>Kew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The immune system: a target for functional foods?</article-title><source>Br J Nutr</source><year>2002</year><volume>88</volume><fpage>S165</fpage><lpage>S176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XovFOlsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1079/BJN2002682</pub-id><pub-id pub-id-type=\"pmid\">12495459</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Carrasco-Hernandez</surname><given-names>R</given-names></name><name><surname>Jácome</surname><given-names>R</given-names></name><name><surname>López Vidal</surname><given-names>Y</given-names></name><name><surname>Ponce de León</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Are RNA viruses candidate agents for the next global pandemic? a review</article-title><source>ILAR J</source><year>2017</year><volume>58</volume><fpage>343</fpage><lpage>358</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXisVCmtbjK</pub-id><pub-id pub-id-type=\"doi\">10.1093/ilar/ilx026</pub-id><pub-id pub-id-type=\"pmid\">28985316</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>M</given-names></name><name><surname>Sun</surname><given-names>Q</given-names></name><name><surname>Giovannucci</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Dairy consumption and risk of type 2 diabetes: 3 cohorts of US adults and an updated meta-analysis</article-title><source>BMC Med</source><year>2014</year><volume>12</volume><fpage>215</fpage><pub-id pub-id-type=\"doi\">10.1186/s12916-014-0215-1</pub-id><pub-id pub-id-type=\"pmid\">25420418</pub-id><pub-id pub-id-type=\"pmcid\">4243376</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Gao</surname><given-names>X</given-names></name><name><surname>Wei</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Isolation, purification and the anti-hypertensive effect of a novel angiotensin I-converting enzyme (ACE) inhibitory peptide from <italic>Ruditapes philippinarum</italic> fermented with <italic>Bacillus natto</italic></article-title><source>Food Funct</source><year>2018</year><volume>9</volume><fpage>5230</fpage><lpage>5237</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFOgsLnL</pub-id><pub-id pub-id-type=\"doi\">10.1039/C8FO01146J</pub-id><pub-id pub-id-type=\"pmid\">30206615</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Chiba</surname><given-names>E</given-names></name><name><surname>Tomosada</surname><given-names>Y</given-names></name><name><surname>Vizoso-Pinto</surname><given-names>MG</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Immunobiotic Lactobacillus rhamnosus improves resistance of infant mice against respiratory syncytial virus infection</article-title><source>Int Immunopharmacol</source><year>2013</year><volume>17</volume><fpage>373</fpage><lpage>382</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXhtlGqtrbK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.intimp.2013.06.024</pub-id><pub-id pub-id-type=\"pmid\">23838113</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Choi</surname><given-names>H-J</given-names></name><name><surname>Song</surname><given-names>J-H</given-names></name><name><surname>Park</surname><given-names>K-S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of yogurt against enterovirus 71 in vero cells</article-title><source>Food Sci Biotechnol</source><year>2010</year><volume>19</volume><fpage>289</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXhvVGisLzE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-010-0042-x</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Davidson</surname><given-names>LE</given-names></name><name><surname>Fiorino</surname><given-names>A-M</given-names></name><name><surname>Snydman</surname><given-names>DR</given-names></name><name><surname>Hibberd</surname><given-names>PL</given-names></name></person-group><article-title xml:lang=\"en\">Lactobacillus GG as an immune adjuvant for live-attenuated influenza vaccine in healthy adults: a randomized double-blind placebo-controlled trial</article-title><source>Eur J Clin Nutr</source><year>2011</year><volume>65</volume><fpage>501</fpage><lpage>507</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3MXkt1Wlt7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/ejcn.2010.289</pub-id><pub-id pub-id-type=\"pmid\">21285968</pub-id><pub-id pub-id-type=\"pmcid\">3071884</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>De Leo</surname><given-names>A</given-names></name><name><surname>Arena</surname><given-names>G</given-names></name><name><surname>Lacanna</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Resveratrol inhibits Epstein Barr virus lytic cycle in Burkitt’s lymphoma cells by affecting multiple molecular targets</article-title><source>Antiviral Res</source><year>2012</year><volume>96</volume><fpage>196</fpage><lpage>202</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38Xhs12gtLfE</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2012.09.003</pub-id><pub-id pub-id-type=\"pmid\">22985630</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>García-Burgos</surname><given-names>M</given-names></name><name><surname>Moreno-Fernández</surname><given-names>J</given-names></name><name><surname>Alférez</surname><given-names>MJM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">New perspectives in fermented dairy products and their health relevance</article-title><source>J Funct Foods</source><year>2020</year><volume>72</volume><fpage>104059</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFylu7vM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jff.2020.104059</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ghanem</surname><given-names>KZ</given-names></name><name><surname>Mahran</surname><given-names>MZ</given-names></name><name><surname>Ramadan</surname><given-names>MM</given-names></name><etal/></person-group><article-title xml:lang=\"en\">A comparative study on flavour components and therapeutic properties of unfermented and fermented defatted soybean meal extract</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>5998</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmvFSlur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-62907-x</pub-id><pub-id pub-id-type=\"pmid\">32265465</pub-id><pub-id pub-id-type=\"pmcid\">7138831</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Prasad</surname><given-names>J</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of natural and acquired immunity by Lactobacillus rhamnosus (HN001), Lactobacillus acidophilus (HN017) and Bifidobacterium lactis (HN019)</article-title><source>Br J Nutr</source><year>2000</year><volume>83</volume><fpage>167</fpage><lpage>176</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3cXhsVymur8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114500000210</pub-id><pub-id pub-id-type=\"pmid\">10743496</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gill</surname><given-names>HS</given-names></name><name><surname>Rutherfurd</surname><given-names>KJ</given-names></name><name><surname>Cross</surname><given-names>ML</given-names></name><name><surname>Gopal</surname><given-names>PK</given-names></name></person-group><article-title xml:lang=\"en\">Enhancement of immunity in the elderly by dietary supplementation with the probiotic Bifidobacterium lactis HN019</article-title><source>Am J Clin Nutr</source><year>2001</year><volume>74</volume><fpage>833</fpage><lpage>839</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD3MXovVSgtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1093/ajcn/74.6.833</pub-id><pub-id pub-id-type=\"pmid\">11722966</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grandy</surname><given-names>G</given-names></name><name><surname>Medina</surname><given-names>M</given-names></name><name><surname>Soria</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in the treatment of acute rotavirus diarrhoea. a randomized, double-blind, controlled trial using two different probiotic preparations in Bolivian children</article-title><source>BMC Infect Dis</source><year>2010</year><volume>10</volume><fpage>253</fpage><pub-id pub-id-type=\"doi\">10.1186/1471-2334-10-253</pub-id><pub-id pub-id-type=\"pmid\">20735858</pub-id><pub-id pub-id-type=\"pmcid\">2940902</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Grant</surname><given-names>W</given-names></name><name><surname>Lahore</surname><given-names>H</given-names></name><name><surname>McDonnell</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Evidence that vitamin D supplementation could reduce risk of influenza and COVID-19 infections and deaths</article-title><source>Nutrients</source><year>2020</year><volume>12</volume><fpage>988</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVemsbnK</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu12040988</pub-id><pub-id pub-id-type=\"pmid\">32252338</pub-id><pub-id pub-id-type=\"pmcid\">7231123</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guan</surname><given-names>W</given-names></name><name><surname>Ni</surname><given-names>Z</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical characteristics of coronavirus disease 2019 in China</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1708</fpage><lpage>1720</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSjsb8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1056/NEJMoa2002032</pub-id><pub-id pub-id-type=\"pmid\">32109013</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gui</surname><given-names>Q</given-names></name><name><surname>Wang</surname><given-names>A</given-names></name><name><surname>Zhao</surname><given-names>X</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effects of probiotic supplementation on natural killer cell function in healthy elderly individuals: a meta-analysis of randomized controlled trials</article-title><source>Eur J Clin Nutr</source><year>2020</year><volume>74</volume><fpage>1630</fpage><lpage>1637</lpage><pub-id pub-id-type=\"doi\">10.1038/s41430-020-0670-z</pub-id><pub-id pub-id-type=\"pmid\">32514029</pub-id><pub-id pub-id-type=\"pmcid\">7279433</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guillemard</surname><given-names>E</given-names></name><name><surname>Tondu</surname><given-names>F</given-names></name><name><surname>Lacoin</surname><given-names>F</given-names></name><name><surname>Schrezenmeir</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Consumption of a fermented dairy product containing the probiotic Lactobacillus casei DN-114 001 reduces the duration of respiratory infections in the elderly in a randomised controlled trial</article-title><source>Br J Nutr</source><year>2010</year><volume>103</volume><fpage>58</fpage><lpage>68</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhs1Smt73N</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114509991395</pub-id><pub-id pub-id-type=\"pmid\">19747410</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guo</surname><given-names>Z-D</given-names></name><name><surname>Wang</surname><given-names>Z-Y</given-names></name><name><surname>Zhang</surname><given-names>S-F</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface distribution of severe acute respiratory syndrome coronavirus 2 in hospital wards, Wuhan, China, 2020</article-title><source>Emerg Infect Dis</source><year>2020</year><volume>26</volume><fpage>1583</fpage><lpage>1591</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlOlsbrF</pub-id><pub-id pub-id-type=\"doi\">10.3201/eid2607.200885</pub-id><pub-id pub-id-type=\"pmid\">32275497</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hamida</surname><given-names>RS</given-names></name><name><surname>Shami</surname><given-names>A</given-names></name><name><surname>Ali</surname><given-names>MA</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Kefir: a protective dietary supplementation against viral infection</article-title><source>Biomed Pharmacother</source><year>2021</year><volume>133</volume><fpage>110974</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitlynsrbL</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biopha.2020.110974</pub-id><pub-id pub-id-type=\"pmid\">33186795</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"other\">Hayes M, García-Vaquero M (2016) Bioactive Compounds from Fermented Food Products. pp&amp;nbsp;293–310</mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Horowitz</surname><given-names>RI</given-names></name><name><surname>Freeman</surname><given-names>PR</given-names></name><name><surname>Bruzzese</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Efficacy of glutathione therapy in relieving dyspnea associated with COVID-19 pneumonia: a report of 2 cases</article-title><source>Respiratory Med Case Rep</source><year>2020</year><volume>30</volume><fpage>101063</fpage><pub-id pub-id-type=\"doi\">10.1016/j.rmcr.2020.101063</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hu</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Lin</surname><given-names>W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Review article: probiotics, prebiotics and dietary approaches during COVID-19 pandemic</article-title><source>Trends Food Sci Technol</source><year>2021</year><volume>108</volume><fpage>187</fpage><lpage>196</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitVyrsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2020.12.009</pub-id><pub-id pub-id-type=\"pmid\">33519087</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ivec</surname><given-names>M</given-names></name><name><surname>Botić</surname><given-names>T</given-names></name><name><surname>Koren</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Interactions of macrophages with probiotic bacteria lead to increased antiviral response against vesicular stomatitis virus</article-title><source>Antiviral Res</source><year>2007</year><volume>75</volume><fpage>266</fpage><lpage>274</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXmsFKht7k%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2007.03.013</pub-id><pub-id pub-id-type=\"pmid\">17512614</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ji</surname><given-names>L-N</given-names></name><name><surname>Chao</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>Y-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Clinical features of pediatric patients with COVID-19: a report of two family cluster cases</article-title><source>World J Pediatr</source><year>2020</year><volume>16</volume><fpage>267</fpage><lpage>270</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXltVSisbc%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12519-020-00356-2</pub-id><pub-id pub-id-type=\"pmid\">32180140</pub-id><pub-id pub-id-type=\"pmcid\">7091281</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Lee</surname><given-names>Y-T</given-names></name><name><surname>Ngo</surname><given-names>V</given-names></name><name><surname>Le</surname></name><etal/></person-group><article-title xml:lang=\"en\">Heat-killed Lactobacillus casei confers broad protection against influenza A virus primary infection and develops heterosubtypic immunity against future secondary infection</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><fpage>17360</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhsFGitLbF</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-017-17487-8</pub-id><pub-id pub-id-type=\"pmid\">29234060</pub-id><pub-id pub-id-type=\"pmcid\">5727132</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kanmani</surname><given-names>P</given-names></name><name><surname>Albarracin</surname><given-names>L</given-names></name><name><surname>Kobayashi</surname><given-names>H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">AGenomic characterization of Lactobacillus delbrueckii TUA4408L and evaluation of the antiviral activities of its extracellular polysaccharides in porcine intestinal epithelial cells</article-title><source>Front Immunol</source><year>2018</year><volume>9</volume><fpage>27</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXnslyrsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2018.02178</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kaushik</surname><given-names>S</given-names></name><name><surname>Jangra</surname><given-names>G</given-names></name><name><surname>Kundu</surname><given-names>V</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Anti-viral activity of Zingiber officinale (Ginger) ingredients against the Chikungunya virus</article-title><source>VirusDisease</source><year>2020</year><volume>31</volume><fpage>270</fpage><lpage>276</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38Xit1yqurjO</pub-id><pub-id pub-id-type=\"doi\">10.1007/s13337-020-00584-0</pub-id><pub-id pub-id-type=\"pmid\">32420412</pub-id><pub-id pub-id-type=\"pmcid\">7223110</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khalil</surname><given-names>E</given-names></name><name><surname>Abd Manap</surname><given-names>M</given-names></name><name><surname>Mustafa</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotic properties of exopolysaccharide-producing lactobacillus strains isolated from tempoyak</article-title><source>Molecules</source><year>2018</year><volume>23</volume><fpage>398</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhs1Kjt7%2FM</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules23020398</pub-id><pub-id pub-id-type=\"pmid\">29438288</pub-id><pub-id pub-id-type=\"pmcid\">6017292</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>E-H</given-names></name><name><surname>Kim</surname><given-names>S-W</given-names></name><name><surname>Park</surname><given-names>S-J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Greater efficacy of black ginseng (CJ EnerG) over red ginseng against lethal Influenza A virus infection</article-title><source>Nutrients</source><year>2019</year><volume>11</volume><fpage>1879</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXisVGmtbjP</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu11081879</pub-id><pub-id pub-id-type=\"pmid\">31412594</pub-id><pub-id pub-id-type=\"pmcid\">6723933</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>Y-I</given-names></name><name><surname>Kim</surname><given-names>S-G</given-names></name><name><surname>Kim</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Infection and rapid transmission of SARS-CoV-2 in Ferrets</article-title><source>Cell Host Microbe</source><year>2020</year><volume>27</volume><fpage>704</fpage><lpage>709e2</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmsl2qsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.chom.2020.03.023</pub-id><pub-id pub-id-type=\"pmid\">32259477</pub-id><pub-id pub-id-type=\"pmcid\">7144857</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kinoshita</surname><given-names>T</given-names></name><name><surname>Maruyama</surname><given-names>K</given-names></name><name><surname>Suyama</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of OLL1073R-1 yogurt intake on influenza incidence and immunological markers among women healthcare workers: a randomized controlled trial</article-title><source>Food Funct</source><year>2019</year><volume>10</volume><fpage>8129</fpage><lpage>8136</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVaksLrI</pub-id><pub-id pub-id-type=\"doi\">10.1039/C9FO02128K</pub-id><pub-id pub-id-type=\"pmid\">31738351</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kokubo</surname><given-names>T</given-names></name><name><surname>Komano</surname><given-names>Y</given-names></name><name><surname>Tsuji</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The effects of plasmacytoid dendritic cell-Stimulative lactic acid bacteria, lactococcus lactis strain plasma, on exercise-Induced fatigue and recovery via immunomodulatory action</article-title><source>Int J Sport Nutr Exerc Metab</source><year>2019</year><volume>25</volume><fpage>1</fpage><lpage>5</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXmt1art74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1123/ijsnem.2018-0377</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kurian</surname><given-names>SJ</given-names></name><name><surname>Unnikrishnan</surname><given-names>MK</given-names></name><name><surname>Miraj</surname><given-names>SS</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Probiotics in prevention and treatment of COVID-19: current perspective and future prospects</article-title><source>Arch Med Res</source><year>2021</year><volume>52</volume><fpage>582</fpage><lpage>594</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXnvFehu74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.arcmed.2021.03.002</pub-id><pub-id pub-id-type=\"pmid\">33785208</pub-id><pub-id pub-id-type=\"pmcid\">7972717</pub-id></mixed-citation></ref><ref id=\"CR38\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kwak</surname><given-names>M-K</given-names></name><name><surname>Liu</surname><given-names>R</given-names></name><name><surname>Kwon</surname><given-names>J-O</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Cyclic dipeptides from lactic acid bacteria inhibit proliferation of the influenza a virus</article-title><source>J Microbiol</source><year>2013</year><volume>51</volume><fpage>836</fpage><lpage>843</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXksVajsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s12275-013-3521-y</pub-id><pub-id pub-id-type=\"pmid\">24385362</pub-id></mixed-citation></ref><ref id=\"CR39\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Laatikainen</surname><given-names>R</given-names></name><name><surname>Koskenpato</surname><given-names>J</given-names></name><name><surname>Hongisto</surname><given-names>S-M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Randomised clinical trial: low-FODMAP rye bread vs. regular rye bread to relieve the symptoms of irritable bowel syndrome</article-title><source>Aliment Pharmacol Ther</source><year>2016</year><volume>44</volume><fpage>460</fpage><lpage>470</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28Xht1KktbjL</pub-id><pub-id pub-id-type=\"doi\">10.1111/apt.13726</pub-id><pub-id pub-id-type=\"pmid\">27417338</pub-id><pub-id pub-id-type=\"pmcid\">5113694</pub-id></mixed-citation></ref><ref id=\"CR40\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lai</surname><given-names>C-C</given-names></name><name><surname>Shih</surname><given-names>T-P</given-names></name><name><surname>Ko</surname><given-names>W-C</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and coronavirus disease-2019 (COVID-19): The epidemic and the challenges</article-title><source>Int J Antimicrob Agents</source><year>2020</year><volume>55</volume><fpage>105924</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXjs1Ogtr8%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijantimicag.2020.105924</pub-id><pub-id pub-id-type=\"pmid\">32081636</pub-id><pub-id pub-id-type=\"pmcid\">7127800</pub-id></mixed-citation></ref><ref id=\"CR41\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lam</surname><given-names>TT-Y</given-names></name><name><surname>Jia</surname><given-names>N</given-names></name><name><surname>Zhang</surname><given-names>Y-W</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Identifying SARS-CoV-2-related coronaviruses in Malayan pangolins</article-title><source>Nature</source><year>2020</year><volume>583</volume><fpage>282</fpage><lpage>285</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtFCnu77L</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-020-2169-0</pub-id><pub-id pub-id-type=\"pmid\">32218527</pub-id></mixed-citation></ref><ref id=\"CR42\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>MH</given-names></name><name><surname>Yoo</surname><given-names>S-H</given-names></name><name><surname>Ha</surname><given-names>S-D</given-names></name><name><surname>Choi</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Inactivation of feline calicivirus and murine norovirus during Dongchimi fermentation</article-title><source>Food Microbiol</source><year>2012</year><volume>31</volume><fpage>210</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC38XlvFKgsLw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fm.2012.04.002</pub-id><pub-id pub-id-type=\"pmid\">22608225</pub-id></mixed-citation></ref><ref id=\"CR43\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>A</given-names></name><name><surname>Lee</surname><given-names>YJ</given-names></name><name><surname>Yoo</surname><given-names>HJ</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Consumption of dairy yogurt containing Lactobacillus paracasei ssp. paracasei, Bifidobacterium animalis ssp. lactis and heat-treated Lactobacillus plantarum improves immune function including natural killer cell activity</article-title><source>Nutrients</source><year>2017</year><volume>9</volume><fpage>558</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXitVersbjI</pub-id><pub-id pub-id-type=\"doi\">10.3390/nu9060558</pub-id><pub-id pub-id-type=\"pmid\">28561762</pub-id><pub-id pub-id-type=\"pmcid\">5490537</pub-id></mixed-citation></ref><ref id=\"CR44\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Whon</surname><given-names>TW</given-names></name><name><surname>Roh</surname><given-names>SW</given-names></name><name><surname>Jeon</surname><given-names>CO</given-names></name></person-group><article-title xml:lang=\"en\">Unraveling microbial fermentation features in kimchi: from classical to meta-omics approaches</article-title><source>Appl Microbiol Biotechnol</source><year>2020</year><volume>104</volume><fpage>7731</fpage><lpage>7744</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsFejurbJ</pub-id><pub-id pub-id-type=\"doi\">10.1007/s00253-020-10804-8</pub-id><pub-id pub-id-type=\"pmid\">32749526</pub-id></mixed-citation></ref><ref id=\"CR45\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Gayle</surname><given-names>AA</given-names></name><name><surname>Wilder-Smith</surname><given-names>A</given-names></name><name><surname>Rocklöv</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The reproductive number of COVID-19 is higher compared to SARS coronavirus</article-title><source>J Travel Med</source><year>2020</year><volume>27</volume><fpage>98</fpage><lpage>1006</lpage><pub-id pub-id-type=\"doi\">10.1093/jtm/taaa021</pub-id></mixed-citation></ref><ref id=\"CR46\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Ikegami</surname><given-names>S</given-names></name><name><surname>Kume</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Reducing the risk of infection in the elderly by dietary intake of yoghurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>Br J Nutr</source><year>2010</year><volume>104</volume><fpage>998</fpage><lpage>1006</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3cXht1GjsLfO</pub-id><pub-id pub-id-type=\"doi\">10.1017/S000711451000173X</pub-id><pub-id pub-id-type=\"pmid\">20487575</pub-id></mixed-citation></ref><ref id=\"CR47\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Sato</surname><given-names>A</given-names></name><name><surname>Goto</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Enhanced natural killer cell activation by exopolysaccharides derived from yogurt fermented with Lactobacillus delbrueckii ssp. bulgaricus OLL1073R-1</article-title><source>J Dairy Sci</source><year>2016</year><volume>99</volume><fpage>915</fpage><lpage>923</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXitVSksLbE</pub-id><pub-id pub-id-type=\"doi\">10.3168/jds.2015-10376</pub-id><pub-id pub-id-type=\"pmid\">26686726</pub-id></mixed-citation></ref><ref id=\"CR48\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Marco</surname><given-names>ML</given-names></name><name><surname>Heeney</surname><given-names>D</given-names></name><name><surname>Binda</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Health benefits of fermented foods: microbiota and beyond</article-title><source>Curr Opin Biotechnol</source><year>2017</year><volume>44</volume><fpage>94</fpage><lpage>102</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XhvFKjs7zF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.copbio.2016.11.010</pub-id><pub-id pub-id-type=\"pmid\">27998788</pub-id></mixed-citation></ref><ref id=\"CR49\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>McGurnaghan</surname><given-names>SJ</given-names></name><name><surname>Weir</surname><given-names>A</given-names></name><name><surname>Bishop</surname><given-names>J</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Risks of and risk factors for COVID-19 disease in people with diabetes: a cohort study of the total population of Scotland</article-title><source>Lancet Diabetes Endocrinol</source><year>2021</year><volume>9</volume><fpage>82</fpage><lpage>93</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXks1aqsA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/S2213-8587(20)30405-8</pub-id><pub-id pub-id-type=\"pmid\">33357491</pub-id></mixed-citation></ref><ref id=\"CR50\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mirashrafi</surname><given-names>S</given-names></name><name><surname>Moravejolahkami</surname><given-names>AR</given-names></name><name><surname>Balouch Zehi</surname><given-names>Z</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The efficacy of probiotics on virus titres and antibody production in virus diseases: a systematic review on recent evidence for COVID-19 treatment</article-title><source>Clin Nutr ESPEN</source><year>2021</year><volume>46</volume><fpage>1</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.1016/j.clnesp.2021.10.016</pub-id><pub-id pub-id-type=\"pmid\">34857182</pub-id><pub-id pub-id-type=\"pmcid\">8539817</pub-id></mixed-citation></ref><ref id=\"CR51\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mora</surname><given-names>L</given-names></name><name><surname>Escudero</surname><given-names>E</given-names></name><name><surname>Aristoy</surname><given-names>M-C</given-names></name><name><surname>Toldrá</surname><given-names>F</given-names></name></person-group><article-title xml:lang=\"en\">A peptidomic approach to study the contribution of added casein proteins to the peptide profile in Spanish dry-fermented sausages</article-title><source>Int J Food Microbiol</source><year>2015</year><volume>212</volume><fpage>41</fpage><lpage>48</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhtVKhtbjM</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.ijfoodmicro.2015.05.022</pub-id><pub-id pub-id-type=\"pmid\">26116420</pub-id></mixed-citation></ref><ref id=\"CR52\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Morita</surname><given-names>N</given-names></name><name><surname>Umemoto</surname><given-names>E</given-names></name><name><surname>Fujita</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">GPR31-dependent dendrite protrusion of intestinal CX3CR1 + cells by bacterial metabolites</article-title><source>Nature</source><year>2019</year><volume>566</volume><fpage>110</fpage><lpage>114</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmtVyjtL0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41586-019-0884-1</pub-id><pub-id pub-id-type=\"pmid\">30675063</pub-id></mixed-citation></ref><ref id=\"CR53\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mounce</surname><given-names>BC</given-names></name><name><surname>Cesaro</surname><given-names>T</given-names></name><name><surname>Carrau</surname><given-names>L</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Curcumin inhibits Zika and chikungunya virus infection by inhibiting cell binding</article-title><source>Antiviral Res</source><year>2017</year><volume>142</volume><fpage>148</fpage><lpage>157</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlsV2jtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.antiviral.2017.03.014</pub-id><pub-id pub-id-type=\"pmid\">28343845</pub-id></mixed-citation></ref><ref id=\"CR54\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Muhialdin</surname><given-names>BJ</given-names></name><name><surname>Zawawi</surname><given-names>N</given-names></name><name><surname>Abdull Razis</surname><given-names>AF</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented foods and their probiotics bacteria towards respiratory and alimentary tracts viruses</article-title><source>Food Control</source><year>2021</year><volume>127</volume><fpage>108140</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXhvVSmtLfF</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2021.108140</pub-id><pub-id pub-id-type=\"pmid\">33867696</pub-id><pub-id pub-id-type=\"pmcid\">8036130</pub-id></mixed-citation></ref><ref id=\"CR55\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olaimat</surname><given-names>AN</given-names></name><name><surname>Aolymat</surname><given-names>I</given-names></name><name><surname>Al-Holy</surname><given-names>M</given-names></name><etal/></person-group><article-title xml:lang=\"en\">The potential application of probiotics and prebiotics for the prevention and treatment of COVID-19</article-title><source>npj Sci Food</source><year>2020</year><volume>4</volume><fpage>17</fpage><pub-id pub-id-type=\"doi\">10.1038/s41538-020-00078-9</pub-id><pub-id pub-id-type=\"pmid\">33083549</pub-id><pub-id pub-id-type=\"pmcid\">7536434</pub-id></mixed-citation></ref><ref id=\"CR56\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Olivares</surname><given-names>M</given-names></name><name><surname>Díaz-Ropero</surname><given-names>MP</given-names></name><name><surname>Sierra</surname><given-names>S</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Oral intake of Lactobacillus fermentum CECT5716 enhances the effects of influenza vaccination</article-title><source>Nutrition</source><year>2007</year><volume>23</volume><fpage>254</fpage><lpage>260</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXislWgurg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.nut.2007.01.004</pub-id><pub-id pub-id-type=\"pmid\">17352961</pub-id></mixed-citation></ref><ref id=\"CR57\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ong</surname><given-names>SWX</given-names></name><name><surname>Tan</surname><given-names>YK</given-names></name><name><surname>Chia</surname><given-names>PY</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Air, surface environmental, and personal protective equipment contamination by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) from a symptomatic patient</article-title><source>JAMA</source><year>2020</year><volume>323</volume><fpage>1610</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXovVSrtbk%3D</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.3227</pub-id><pub-id pub-id-type=\"pmid\">32129805</pub-id><pub-id pub-id-type=\"pmcid\">7057172</pub-id></mixed-citation></ref><ref id=\"CR58\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Parada Venegas</surname><given-names>D</given-names></name><name><surname>De la Fuente</surname><given-names>MK</given-names></name><name><surname>Landskron</surname><given-names>G</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Short chain fatty acids (SCFAs)-mediated gut epithelial and immune regulation and its relevance for inflammatory bowel diseases</article-title><source>Front Immunol</source><year>2019</year><volume>10</volume><fpage>107</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsVWrtL%2FJ</pub-id><pub-id pub-id-type=\"doi\">10.3389/fimmu.2019.00277</pub-id></mixed-citation></ref><ref id=\"CR59\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Peters</surname><given-names>A</given-names></name><name><surname>Krumbholz</surname><given-names>P</given-names></name><name><surname>Jäger</surname><given-names>E</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Metabolites of lactic acid bacteria present in fermented foods are highly potent agonists of human hydroxycarboxylic acid receptor 3</article-title><source>PLOS Genet</source><year>2019</year><volume>15</volume><fpage>e1008145</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhvFaqsb%2FN</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pgen.1008145</pub-id><pub-id pub-id-type=\"pmid\">31120900</pub-id><pub-id pub-id-type=\"pmcid\">6532841</pub-id></mixed-citation></ref><ref id=\"CR60\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>przeLe</surname><given-names>TT</given-names></name><name><surname>Cramer</surname><given-names>JP</given-names></name><name><surname>Chen</surname><given-names>R</given-names></name><name><surname>Mayhew</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Evolution of the COVID-19 vaccine development landscape</article-title><source>Nat Rev Drug Discov</source><year>2020</year><volume>19</volume><fpage>667</fpage><lpage>668</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvVylsb3M</pub-id><pub-id pub-id-type=\"doi\">10.1038/d41573-020-00151-8</pub-id></mixed-citation></ref><ref id=\"CR61\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Pyankov</surname><given-names>OV</given-names></name><name><surname>Bodnev</surname><given-names>SA</given-names></name><name><surname>Pyankova</surname><given-names>OG</given-names></name><name><surname>Agranovski</surname><given-names>IE</given-names></name></person-group><article-title xml:lang=\"en\">Survival of aerosolized coronavirus in the ambient air</article-title><source>J Aerosol Sci</source><year>2018</year><volume>115</volume><fpage>158</fpage><lpage>163</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhsVyit7jN</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.jaerosci.2017.09.009</pub-id><pub-id pub-id-type=\"pmid\">32226116</pub-id></mixed-citation></ref><ref id=\"CR62\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Raji</surname><given-names>MNA</given-names></name><name><surname>Ab Karim</surname><given-names>S</given-names></name><name><surname>Ishak</surname><given-names>FAC</given-names></name><name><surname>Arshad</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Past and present practices of the Malay food heritage and culture in Malaysia</article-title><source>J Ethnic Foods</source><year>2017</year><volume>4</volume><fpage>221</fpage><lpage>231</lpage><pub-id pub-id-type=\"doi\">10.1016/j.jef.2017.11.001</pub-id></mixed-citation></ref><ref id=\"CR63\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Santarpia</surname><given-names>JL</given-names></name><name><surname>Rivera</surname><given-names>DN</given-names></name><name><surname>Herrera</surname><given-names>VL</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface contamination of SARS-CoV-2 observed in quarantine and isolation care</article-title><source>Sci Rep</source><year>2020</year><volume>10</volume><fpage>12732</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhsVyku7rM</pub-id><pub-id pub-id-type=\"doi\">10.1038/s41598-020-69286-3</pub-id><pub-id pub-id-type=\"pmid\">32728118</pub-id><pub-id pub-id-type=\"pmcid\">7391640</pub-id></mixed-citation></ref><ref id=\"CR64\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Smith</surname><given-names>TJ</given-names></name><name><surname>Rigassio-Radler</surname><given-names>D</given-names></name><name><surname>Denmark</surname><given-names>R</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Effect of Lactobacillus rhamnosus LGG ® and Bifidobacterium animalis ssp. lactis BB-12 ® on health-related quality of life in college students affected by upper respiratory infections</article-title><source>Br J Nutr</source><year>2013</year><volume>109</volume><fpage>1999</fpage><lpage>2007</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXnvVGhsbY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1017/S0007114512004138</pub-id><pub-id pub-id-type=\"pmid\">23020819</pub-id></mixed-citation></ref><ref id=\"CR65\"><mixed-citation publication-type=\"other\">Takeda K, Okumura K (2007) Effects of a Fermented Milk Drink Containing Lactobacillus casei Strain Shirota on the Human NK-Cell Activity. J Nutr 137. <ext-link ext-link-type=\"doi\" xlink:href=\"10.1093/jn/137.3.791S\">https://doi.org/10.1093/jn/137.3.791S</ext-link>. :791S-793S</mixed-citation></ref><ref id=\"CR66\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tamang</surname><given-names>JP</given-names></name><name><surname>Cotter</surname><given-names>PD</given-names></name><name><surname>Endo</surname><given-names>A</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Fermented foods in a global age: east meets West</article-title><source>Compr Rev Food Sci Food Saf</source><year>2020</year><volume>19</volume><fpage>184</fpage><lpage>217</lpage><pub-id pub-id-type=\"doi\">10.1111/1541-4337.12520</pub-id><pub-id pub-id-type=\"pmid\">33319517</pub-id></mixed-citation></ref><ref id=\"CR67\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>van Doremalen</surname><given-names>N</given-names></name><name><surname>Bushmaker</surname><given-names>T</given-names></name><name><surname>Morris</surname><given-names>DH</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Aerosol and surface stability of SARS-CoV-2 as compared with SARS-CoV-1</article-title><source>N Engl J Med</source><year>2020</year><volume>382</volume><fpage>1564</fpage><lpage>1567</lpage><pub-id pub-id-type=\"doi\">10.1056/NEJMc2004973</pub-id><pub-id pub-id-type=\"pmid\">32182409</pub-id></mixed-citation></ref><ref id=\"CR68\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Jung</surname><given-names>Y-J</given-names></name><name><surname>Kim</surname><given-names>K-H</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Antiviral activity of fermented ginseng extracts against a broad range of Influenza viruses</article-title><source>Viruses</source><year>2018</year><volume>10</volume><fpage>471</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXntF2gsLc%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/v10090471</pub-id><pub-id pub-id-type=\"pmid\">30200514</pub-id><pub-id pub-id-type=\"pmcid\">6164659</pub-id></mixed-citation></ref><ref id=\"CR69\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wei</surname><given-names>L</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Qi</surname><given-names>H</given-names></name><name><surname>Wen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Engineering Scheffersomyces stipitis for fumaric acid production from xylose</article-title><source>Bioresour Technol</source><year>2015</year><volume>187</volume><fpage>246</fpage><lpage>254</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXlvVems74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.biortech.2015.03.122</pub-id><pub-id pub-id-type=\"pmid\">25863201</pub-id></mixed-citation></ref><ref id=\"CR70\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wiersinga</surname><given-names>WJ</given-names></name><name><surname>Rhodes</surname><given-names>A</given-names></name><name><surname>Cheng</surname><given-names>AC</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Pathophysiology, transmission, diagnosis, and treatment of coronavirus disease 2019 (COVID-19)</article-title><source>JAMA</source><year>2020</year><volume>324</volume><fpage>782</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhslamu73N</pub-id><pub-id pub-id-type=\"doi\">10.1001/jama.2020.12839</pub-id><pub-id pub-id-type=\"pmid\">32648899</pub-id></mixed-citation></ref><ref id=\"CR71\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>N</given-names></name><name><surname>Qiao</surname><given-names>K</given-names></name><etal/></person-group><article-title xml:lang=\"en\">Application of metabolic controls for the maximization of lipid production in semicontinuous fermentation</article-title><source>Proc Natl Acad Sci USA</source><year>2017</year><volume>114</volume><fpage>E5308</fpage><lpage>E5316</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXhtVWmtLvI</pub-id><pub-id pub-id-type=\"doi\">10.1073/pnas.1703321114</pub-id><pub-id pub-id-type=\"pmid\">28630295</pub-id><pub-id pub-id-type=\"pmcid\">5502627</pub-id></mixed-citation></ref><ref id=\"CR72\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Song</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name></person-group><article-title xml:lang=\"en\">Antiviral effects of a probiotic metabolic products against transmissible gastroenteritis coronavirus</article-title><source>J Probiotics Heal</source><year>2017</year><volume>5</volume><fpage>543</fpage><pub-id pub-id-type=\"doi\">10.4172/2329-8901.1000184</pub-id></mixed-citation></ref><ref id=\"CR73\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yin</surname><given-names>SY</given-names></name><name><surname>Kim</surname><given-names>HJ</given-names></name><name><surname>Kim</surname><given-names>H-J</given-names></name></person-group><article-title xml:lang=\"en\">Protective effect of dietary xylitol on Influenza A virus infection</article-title><source>PLoS One</source><year>2014</year><volume>9</volume><fpage>e84633</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXltlOit78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1371/journal.pone.0084633</pub-id><pub-id pub-id-type=\"pmid\">24392148</pub-id><pub-id pub-id-type=\"pmcid\">3879333</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec14\"><title>Supplementary Information</title><p id=\"Par35\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/13197_2022_5528_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 556 kb)</p></caption></media></supplementary-material></p></sec></app></app-group><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10068</journal-id><journal-id journal-id-type=\"doi\">10.1007/10068.2092-6456</journal-id><journal-title-group><journal-title>Food Science and Biotechnology</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Food Sci Biotechnol</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1226-7708</issn><issn pub-type=\"epub\">2092-6456</issn><publisher><publisher-name>Springer Nature Singapore</publisher-name><publisher-loc>Singapore</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10068-023-01258-5</article-id><article-id pub-id-type=\"manuscript\">1258</article-id><article-id pub-id-type=\"doi\">10.1007/s10068-023-01258-5</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Detection method for genetically modified potato using an ultra-fast PCR system</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0003-0608-6354</contrib-id><name><surname>Shin</surname><given-names>Min Ki</given-names></name><address><email>mkhj1212@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs10068023012585_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Jeon</surname><given-names>Seon Min</given-names></name><address><email>jsm0065@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Koo</surname><given-names>Yong Eui</given-names></name><address><email>kye2020@korea.kr</email></address><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.31501.36</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 0470 5905</institution-id><institution content-type=\"org-division\">Interdisciplinary Program in Bioengineering</institution><institution content-type=\"org-name\">Seoul National University</institution></institution-wrap><addr-line content-type=\"postcode\">08826</addr-line><addr-line content-type=\"city\">Seoul</addr-line><country country=\"KR\">Republic of Korea</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.467691.b</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1773 0675</institution-id><institution content-type=\"org-division\">Food Safety Risk Assessment Department</institution><institution content-type=\"org-name\">National Institute of Food and Drug Safety Evaluation</institution></institution-wrap><addr-line content-type=\"postcode\">28159</addr-line><addr-line content-type=\"city\">Cheongju</addr-line><country country=\"KR\">Republic of Korea</country></aff></contrib-group><author-notes><corresp id=\"IDs10068023012585_cor1\"><label>a</label><email>mkhj1212@korea.kr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>1</day><month>2</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>32</volume><issue seq=\"4\">9</issue><fpage>1185</fpage><lpage>1191</lpage><history><date date-type=\"registration\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"received\"><day>24</day><month>10</month><year>2022</year></date><date date-type=\"rev-recd\"><day>27</day><month>12</month><year>2022</year></date><date date-type=\"accepted\"><day>10</day><month>1</month><year>2023</year></date><date date-type=\"online\"><day>1</day><month>2</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Korean Society of Food Science and Technology 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Korean Society of Food Science and Technology</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Genetically modified (GM) potatoes having resistance to insects and viral diseases, low reducing sugar contents, and black spots for high quality continue to be developed. However, no GM potato has been approved as food or feed in the Republic of Korea as the country adheres to a zero-tolerance policy to unauthorized genetically modified organisms (GMOs). When the self-sufficiency rate is low, a detection method to assess GMOs in crops or other products is necessary. Therefore, a rapid method for two GM potato events (SPS-Y9 and EH92-527-1) using an ultra-fast PCR (UF-PCR) system has been developed, and its specificity, sensitivity, and applicability were demonstrated. UF-PCR can decrease the runtime of PCR by more than half of that needed in conventional methods. However, UF-PCR is not a common method for GMO analysis. This rapid detection method may be useful for GMO analyses in field conditions.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Solanum tuberosum</italic></kwd><kwd>Event specific qualification</kwd><kwd>GM potato</kwd><kwd>UF-PCR</kwd><kwd>Monitoring</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution>National Institute of Food and Drug Safety Evaluation</institution><institution-id institution-id-type=\"doi\" vocab=\"open-funder-registry\">http://dx.doi.org/10.13039/501100003655</institution-id></institution-wrap></funding-source><award-id award-type=\"FundRef grant\">19161MFDS051</award-id><principal-award-recipient><name><surname>Koo</surname><given-names>Yong Eui</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>The Korean Society of Food Science and Technology, co-published with Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>14</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>15</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>The Korean Society of Food Science and Technology</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>1</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>10</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10068_2023_Article_1258.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Food Science</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Nutrition</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par2\">Potato (<italic>Solanum tuberosum</italic>) is widely used as a food and snack ingredient and is one of the four major food crops, along with rice, wheat, and corn. As of 2017, approximately 390 million ton of potatoes had been produced worldwide in an area of ​​19.3 million hectares. The area for cultivation decreased by 3% compared to that in 2000; however, production increased by 20.3% during the same period owing to the increase in yield (FAOSTAT, <xref ref-type=\"bibr\" rid=\"CR8\">2019</xref>). The productivity is steadily improving with the development of potato varieties and production technology.</p><p id=\"Par3\">Genetically modified (GM) crops also contribute to the increase in productivity. Although commercialization has ceased, the first GM potato, the NewLeaf potato having a pest-resistance trait, was approved as food in 1995 (Jaccaud et al., <xref ref-type=\"bibr\" rid=\"CR13\">2003</xref>). Subsequently, various traits, such as resistance against insects, diseases, and modified product qualities, have been introduced into potato (Martinez-Prada et al., <xref ref-type=\"bibr\" rid=\"CR17\">2021</xref>). In 2022, according to the GM Approval Database (<ext-link ext-link-type=\"uri\" xlink:href=\"http://www.isaaa.org/gmapprovaldatabase/default.asp\">www.isaaa.org/gmapprovaldatabase/default.asp</ext-link>) of the International Service for the Acquisition of Agri-biotech Applications, fifty-one GM potato events have been developed and commercialized.</p><p id=\"Par4\">Potato production in Republic of Korea was approximately 0.63 million ton, importing 0.24 million ton in 2019, and it has continued to increase since 2010 (<ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world/</ext-link>). Under these circumstances, the urge for use of GM potato is increasing; however, no GM potato varieties are currently approved as food and feed in Republic of Korea (MFDS, <xref ref-type=\"bibr\" rid=\"CR20\">2022</xref>).</p><p id=\"Par5\">In several countries, unauthorized genetically modified organisms (GMOs) are managed under zero-tolerance or low level presence polices (Fraiture et al., <xref ref-type=\"bibr\" rid=\"CR9\">2016</xref>), which necessitates the development of rapid and accurate detection methods to manage unauthorized GMOs.</p><p id=\"Par6\">Several studies have been conducted to develop methods to identify GM potatoes. Detection methods for NewLeaf, NewLeaf Y, NewLeaf Plus, PH05-026-0048, EH92-527-1, AM04-1020, SPS-E12, AV43-6-G7, and insertion <italic>Cry3A</italic> have been developed using conventional PCR (PCR), real-time PCR (qPCR) (Cote et al., <xref ref-type=\"bibr\" rid=\"CR5\">2005</xref>; Roh et al., <xref ref-type=\"bibr\" rid=\"CR22\">2004</xref>; Song et al., <xref ref-type=\"bibr\" rid=\"CR25\">2017</xref>; Yoo et al., <xref ref-type=\"bibr\" rid=\"CR29\">2013</xref>), loop-mediated isothermal amplification (Tu et al., <xref ref-type=\"bibr\" rid=\"CR27\">2020</xref>), and droplet digital PCR (Gao et al., <xref ref-type=\"bibr\" rid=\"CR10\">2016</xref>). Recent gene detection assays are directed towards simpler and more cost-effective methods.</p><p id=\"Par7\">Most GMO detection methods using PCR or qPCR take approximately 40 min to 2 h or more including data analysis. However, the ultra-fast PCR (UF-PCR) system can decrease the run time of PCR. In addition, the use of UF-PCR system can reduce reagent usage by half. Owing to these advantages, the UF-PCR system is being considered for application in various analyses, such as forensic genotyping (Aboud et al., <xref ref-type=\"bibr\" rid=\"CR1\">2013</xref>), pathogen screening (Das et al., <xref ref-type=\"bibr\" rid=\"CR6\">2022</xref>; Kim et al., <xref ref-type=\"bibr\" rid=\"CR15\">2018</xref>; Lee et al., <xref ref-type=\"bibr\" rid=\"CR16\">2014</xref>), allergen analysis (Kim et al., <xref ref-type=\"bibr\" rid=\"CR14\">2019</xref>), detection of epidemic viruses like SARS CoV-2 (Chen et al., <xref ref-type=\"bibr\" rid=\"CR4\">2022</xref>; Milosevic et al., <xref ref-type=\"bibr\" rid=\"CR19\">2021</xref>), and authenticity determination (Hong et al., <xref ref-type=\"bibr\" rid=\"CR11\">2021</xref>).</p><p id=\"Par8\">However, little research has been conducted in the field of GMO detection using the UF-PCR system (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). Here, we propose a reliable and rapid detection method for GM potatoes.</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Samples</title><p id=\"Par9\">Reference materials of GM soybean (A2704-12, A5547-127, CV127, DAS-44406-6, DAS-68416-4, DAS-81419-2, DP-305423-1, DP-356043-5, FG72, GMB151, RRS, MON87701, MON87705, MON87708, MON87751, MON87769, MON89788, and SYHT0H2), GM maize (3272, 5307, Bt10, Bt11, Bt176, DAS-40278-9, DAS-59122-7, DLL25, DP-004114-3, DP-023211-2, DP-098140-6, DP-202216-6, DP-915635-4, GA21, MIR162, MIR604, MZHG0JG, MZIR098, NK603, T25, TC1507, MON810, MON863, MON87403, MON87411, MON87419, MON87427, MON87429, MON87460, MON88017, MON89034, MON95379, and VCO-01981-5), GM canola (DP-073496-4, GT73, LBFLFK, MON88302, MON94100, MS8, MS11, NS-B50027-4, RF3, and T45), GM cotton (271/3006, COT102, DAS-81910-7, GHB119, GHB614, GHB811, LLcotton25, MON531, MON1445, MON15985, MON88701, MON88702, MON88913, and T304xGHB119), and GM potato (AM04-1020, EH92-527-1, PH05-026-0048, SPS-E12, and SPS-Y9) were kindly provided by the developers or purchased from the Institute for Reference Materials and Measurements (IRMM, Geel, Belgium) (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). Non-GM (NGM) seeds of soybean, maize, canola, cotton, rice, wheat, barley, and potato tissues were also used.<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>GM reference materials used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event name</p></th><th align=\"left\"><p>Developer</p></th><th align=\"left\"><p>Crop</p></th><th align=\"left\"><p>Event Name</p></th><th align=\"left\"><p>Developer</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>RRS</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87411</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON89788</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>VCO-01981-5</p></td><td align=\"left\"><p>Genective</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A2704-12</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87403</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-356043-5</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87419</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DP-305423-1</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZHG0JG</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>A5547-127</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MZIR098</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87701</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-202216-6</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>CV127</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87429</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87769</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-023211-2</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87705</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON95379</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87708</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-915635-4</p></td><td align=\"left\"><p>Corteva</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>FG72</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON531</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>SYHT0H2</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON1445</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-44406-6</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON15985</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-68416-4</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>281/3006</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>DAS-81419-2</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>LLcotton25</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>MON87751</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88913</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Soybean</p></td><td align=\"left\"><p>GMB151</p></td><td align=\"left\"><p>BASF plant Science</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB614</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON810</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>T304-40 × GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>GA21</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>COT102</p></td><td align=\"left\"><p>Syngenta</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>TC1507</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88701</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>NK603</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB119</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON863</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>DAS-81910-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt11</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>GHB811</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>T25</p></td><td align=\"left\"><p>Bayer CropScience</p></td><td align=\"left\"><p>Cotton</p></td><td align=\"left\"><p>MON88702</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt176</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>GT73</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DLL25</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>T45</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-59122-7</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON88302</p></td><td align=\"left\"><p>Monsanto</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON88017</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS8</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR604</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>RF3</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>Bt10</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>DP-073496-4</p></td><td align=\"left\"><p>Pioneer</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON89034</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MS11</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MIR162</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>LBFLFK</p></td><td align=\"left\"><p>BASF plant Science</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-098140-6</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>NS-B50027-4</p></td><td align=\"left\"><p>NUSEED</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>3272</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Canola</p></td><td align=\"left\"><p>MON94100</p></td><td align=\"left\"><p>Bayer CropScience</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87460</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>5307</p></td><td align=\"left\"><p>Syngenta</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>EH92-527-1</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>MON87427</p></td><td align=\"left\"><p>Monsanto</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>SPS-E12</p></td><td align=\"left\"><p>Simplot</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DAS-40278-9</p></td><td align=\"left\"><p>Dow AgroScience</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>AM04-1020</p></td><td align=\"left\"><p>BASF</p></td></tr><tr><td align=\"left\"><p>Maize</p></td><td align=\"left\"><p>DP-004114-3</p></td><td align=\"left\"><p>Pioneer</p></td><td align=\"left\"><p>Potato</p></td><td align=\"left\"><p>PH05-026-0048</p></td><td align=\"left\"><p>BASF</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec4\"><title>DNA extraction</title><p id=\"Par10\">NGM crops, GM soybean, GM maize, GM canola, GM cotton seeds, freeze-dried GM potato tissue, and processed products were ground separately. Samples of GM soybean mix (18 events), maize mix (33 events), canola mix (10 events), and cotton mix (14 events) were prepared by mixing 0.1 g of ground powder from each event. Genomic DNA (gDNA) was extracted from the powder samples using a Maxwell RCS PureFood GMO and Authentication Kit (Promega, Madison, WI, USA) according to the manufacturer’s instructions. DNA concentrations were determined using a QUBIT 3.0 Quantitation Kit (Thermo Fisher Scientific, Wilmington, DE, USA). DNA samples were diluted in nuclease-free water to a final concentration of 10 ng/µL and stored at − 20 °C until use.</p></sec><sec id=\"Sec5\"><title>Primers</title><p id=\"Par11\">Event-specific primers for the two GM potato events were used in PCR assays. <italic>APRT</italic> (GenBank accession number CK270447), which encodes adenine phosphoribosyltransferase, was selected as an endogenous reference gene. A previously reported APRT F/R primer set for <italic>APRT</italic> detection was used (MHLW, <xref ref-type=\"bibr\" rid=\"CR21\">2020</xref>). The SPS-Y9 mF1/mR1, and Event527-bf1/St527-R1 primer pairs were based on the 5' and 3' integration flanking sequences of T-DNA and the host genome of GM potato, respectively (Shin et al., <xref ref-type=\"bibr\" rid=\"CR23\">2020</xref>; JRC-GMO-Matrix, Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>). All primers were synthesized by Bioneer (Daejeon, Republic of Korea), and the oligonucleotide sequences are listed in Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>The alignment sites of event-specific primers for GM potato SPS-Y9 and EH92-527-1. The arrows indicate the location of the primers. RB, right border; LB, left border; P-Vnt1, promoter for the native Rpi-vnt1 gene of <italic>Solanum venturii</italic>; Vnt1, Rpi-vnt1 gene encoding sequence of <italic>Solanum venturii</italic>; T-Vnt1, termination sequence of Rpi-vnt1 gene of <italic>Solanum venturii</italic>; P-Agp, promoter for the ADP glucose pyrophosphorylase gene of <italic>Solanum tuberosum</italic> var. Ranger Russet; VInv, vacuolar invertase gene fragment of <italic>Solanum tuberosum</italic> var. Ranger Russet; P-Gbss, promoter for the granule-bound starch synthase gene of <italic>Solanum tuberosum</italic>; P-nos, constitutive promoter of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; nptII, sequence encoding the NPTII enzyme (neomycin phosphotranspherase) from Tn5 transposon of E. coli; T-nos, termination sequence of the nopaline synthase gene of <italic>Agrobacterium tumefaciens</italic> T-DNA; Anti-gbss, antisense sequence encoding the gbss enzyme, involved in the starch synthase of <italic>Solanum tuberosum</italic></p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig1_HTML.png\"/></fig><table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Oligonucleotide primers used in this study</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Target</p></th><th align=\"left\"><p>Name</p></th><th align=\"left\"><p>Sequence (5′ → 3′)</p></th><th align=\"left\"><p>Length (bp)</p></th><th align=\"left\"><p>Source</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>APRT F</p><p>APRT R</p></td><td align=\"left\"><p>TGAAAACGATCCCGATCG</p><p>CAATCCCAGCGATACGTTC</p></td><td align=\"char\" char=\".\"><p>90</p></td><td align=\"left\"><p>MHLW (Japan)</p></td></tr><tr><td align=\"left\"><p>SPS-Y9</p></td><td align=\"left\"><p>Y9-mF1</p><p>Y9-mR1</p></td><td align=\"left\"><p>GAAGACAGCTATCAATATACTTTAGTAA</p><p>AATCTTTTGCCATACTCATACAGAG</p></td><td align=\"char\" char=\".\"><p>129</p></td><td align=\"left\"><p>MFDS (Korea)</p></td></tr><tr><td align=\"left\"><p>EH92</p><p>-527-1</p></td><td align=\"left\"><p>Event527-bf1</p><p>St527-R1</p></td><td align=\"left\"><p>GTGTCAAAACACAATTTACAGCA</p><p>TCCCTTAATTCTCCGCTCATGA</p></td><td align=\"char\" char=\".\"><p>93</p></td><td align=\"left\"><p>JRC (EU)</p></td></tr></tbody></table></table-wrap></p></sec><sec id=\"Sec6\"><title>UF-PCR system</title><p id=\"Par12\">UF-PCR, which is based on the same principle as that of SYBR Green qPCR, was used by replacing SYBR Green with EvaGreen as an intercalating dye (Shin et al., <xref ref-type=\"bibr\" rid=\"CR24\">2022</xref>). PCR analysis was performed using a GENECHECKER UF-150 Ultra-fast Real-Time PCR system (Genesystem, Daejeon, Republic of Korea) with Rapi:chip. The final reaction volume was 10 µL consisting of 5 µL SSO Fast EvaGreen Supermixes (Bio-Rad Laboratories, Berkeley, CA, USA), 1 µL each of forward and reverse primers (0.8 µM), and 2 µL DNA template (50 ng). All PCR reactions were performed under the following conditions: one cycle of initial denaturation at 95 °C for 1 min, 40 cycles of denaturation at 95 °C for 5 s, annealing at 54 °C for 5 s, extension at 72 °C for 5 s, and one cycle of elongation at 72 °C for 5 s. No template control (NTC) was used as a negative control. The amplicons were analyzed using a microchip electrophoresis system, MCE-202 MultiNA (Shimazu, Nishinokyo, Kyoto, Japan) with MultiNA software v1.12 (Shimazu).</p></sec><sec id=\"Sec7\"><title>Specificity, sensitivity, and application test</title><p id=\"Par13\">The specificity of UF-PCR detection method was evaluated using various crops and GM events. To determine the detection level, gDNA samples of NGM potato and each GM potato event were mixed in concentrations ranging from 10 to 0.005% (w/w), and a sensitivity test was conducted. In addition, to check whether the method can be used for processed foods, a test was conducted on foods containing potato available in the market.</p></sec></sec><sec id=\"Sec8\"><title>Results and discussion</title><sec id=\"Sec9\"><title>Specificity</title><p id=\"Par14\">The UF-PCR assay was performed to assess the specificity of this detection method by analyzing NGM crops (soybean, maize, canola, cotton, wheat, rice, barley, and potato), GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048).</p><p id=\"Par15\">Results showed that only a single peak was detected for each target of GM potatoes, without specific amplification of non-target crops, GM events, and NTC (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>). In addition, the melting temperature (Tm) values of amplicons were 77.5 °C and 76.84 °C for SPS-Y9 and EH92-527-1, respectively (Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>). Because amplicons have specific Tm values and do not cross-react, the two GM potato events could be selectively distinguished from each other by their Tm values.<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Primer specificity for two GM potato events using Ultra-fast PCR. (<bold>A</bold>, <bold>C</bold>) Specificity results for Non-GM crops (eight crops; soybean, maize, canola, cotton, rice, wheat, potato, and barley) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (seven crops; soybean, maize, canola, rice, wheat, potato, and barley), lane 18: no template; (<bold>B</bold>, <bold>D</bold>) Specificity results for GM soybean (mix of 18 events), GM maize (mix of 33 events), GM canola (mix of 10 events), GM cotton (mix of 14 events), and GM potato (AM04-1020, SPS-Y9, EH92-527-1, SPS-E12, and PH05-026-0048) DNA, lane M: Size marker, lane 1: positive, lanes 2–17: (eighty GM events; soybean, maize, canola, cotton, and potato), lane 18: no template</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/10068_2023_1258_Fig2_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>Sensitivity results of the ultra-fast PCR method</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Events</p></th><th align=\"left\" rowspan=\"2\"><p>Conc</p></th><th align=\"left\" colspan=\"2\"><p>Ct value</p></th><th align=\"left\" colspan=\"2\"><p>Tm (°C) value</p></th></tr><tr><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th><th align=\"left\"><p>Mean</p></th><th align=\"left\"><p>SD</p></th></tr></thead><tbody><tr><td align=\"left\"><p>APRT</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>24.86</p></td><td align=\"char\" char=\".\"><p>0.25</p></td><td align=\"char\" char=\".\"><p>79.52</p></td><td align=\"char\" char=\".\"><p>0.21</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>SPS-Y9</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>26.34</p></td><td align=\"char\" char=\".\"><p>0.42</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>77.45</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.21</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>30.62</p></td><td align=\"char\" char=\".\"><p>0.40</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.12</p></td><td align=\"char\" char=\".\"><p>0.85</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\" rowspan=\"5\"><p>EH92-527-1</p></td><td align=\"left\"><p>100%</p></td><td align=\"left\"><p>23.80</p></td><td align=\"char\" char=\".\"><p>0.40</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>76.77</p></td><td align=\"char\" char=\".\" rowspan=\"5\"><p>0.27</p></td></tr><tr><td align=\"left\"><p>10%</p></td><td align=\"left\"><p>29.75</p></td><td align=\"char\" char=\".\"><p>0.30</p></td></tr><tr><td align=\"left\"><p>0.5%</p></td><td align=\"left\"><p>35.18</p></td><td align=\"char\" char=\".\"><p>0.69</p></td></tr><tr><td align=\"left\"><p>0.1%</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr><tr><td align=\"left\"><p>NTC</p></td><td align=\"left\"><p>ND</p></td><td align=\"char\" char=\".\"><p>-</p></td></tr></tbody></table><table-wrap-foot><p><italic>SD</italic> Standard deviation</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec10\"><title>Sensitivity</title><p id=\"Par16\">To evaluate the sensitivity of UF-PCR method for GM potato events, PCR assays were performed with gDNA samples that were serially diluted to concentrations ranging from 10 to 0.005% (wt/wt) per reaction. The Ct and Tm values of each event were analyzed. For all GM potato events, the assay was sufficiently identifiable even at 0.5% (0.25 ng) level of the target DNA. Based on the copy number of potato (one copy number for haploid genome of potato is considered as 1.8 pg (Arumuganathan and Earle, <xref ref-type=\"bibr\" rid=\"CR2\">1991</xref>)), 0.25 ng is equivalent to approximately 139 copies.</p><p id=\"Par17\">These results demonstrate that the rapid detection system used in this study can accurately detect GM incorporation as low as 0.5%. This level of detection is consistent with the low-level presence (LLP) of GMOs, that are not approved in their countries, but are already authorized in another countries (Stein and Rodríguez-Cerezo, <xref ref-type=\"bibr\" rid=\"CR26\">2010</xref>).</p></sec><sec id=\"Sec11\"><title>Application of UF-PCR in commercial processed products</title><p id=\"Par18\">Nine potato-containing processed products were tested to confirm the applicability of the UF-PCR method to GM potato, SPS-Y9 and EH92-527-1. The selected products included chips, snacks, ready-made meals, French fries, salads, and other items. Extracted gDNA samples were used for UF-PCR analysis, and the results are shown in Table <xref ref-type=\"table\" rid=\"Tab4\">4</xref>. In all the products, the APRT F/R primer set resulted in amplification, and the Tm values of the amplicons were approximately 79.3 °C. The primer sets for SPS-Y9 and EH92-527-1 did not produce amplicons for any of these samples. This suggests that the domestic distribution of processed potato products selected for this study did not contain two unapproved GM potato events.<table-wrap id=\"Tab4\"><label>Table 4</label><caption xml:lang=\"en\"><p>The ultra-fast PCR results for potato DNA in processed products</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" rowspan=\"2\"><p>Products</p></th><th align=\"left\"><p>APRT</p></th><th align=\"left\"><p>SPS-Y9</p></th><th align=\"left\"><p>EH92-527-1</p></th></tr><tr><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th><th align=\"left\"><p>Ct / Tm (°C)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>Creamer potatoes</p></td><td align=\"left\"><p>28.15 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Baked potatoes</p></td><td align=\"left\"><p>30.09 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato salad</p></td><td align=\"left\"><p>31.10 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Frozen raw potatoes</p></td><td align=\"left\"><p>29.15 / 78.82</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Stick potatoes</p></td><td align=\"left\"><p>28.59 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Half cooked potatoes</p></td><td align=\"left\"><p>31.04 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Crinklecut potatoes</p></td><td align=\"left\"><p>30.11 / 79.15</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Potato chips</p></td><td align=\"left\"><p>24.65 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>Deep fried potatoes</p></td><td align=\"left\"><p>30.85 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (SPS-Y9)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>35.04 / 77.50</p></td><td align=\"left\"><p>ND</p></td></tr><tr><td align=\"left\"><p>0.5% Spiked sample (EH92-527-1)</p></td><td align=\"left\"><p>34.69 / 79.48</p></td><td align=\"left\"><p>ND</p></td><td align=\"left\"><p>34.22 / 76.51</p></td></tr></tbody></table><table-wrap-foot><p><italic>ND</italic> not determined</p></table-wrap-foot></table-wrap></p><p id=\"Par19\">A major challenge of this study was that obtaining products containing unauthorized GM potato was difficult. To address this issue, GM potato-spiked samples were tested together and confirmed to be correctly detected. Therefore, this assay method can be applied to analyze processed products based on the positive results for endogenous <italic>APRT</italic> of potato and GM spiked samples. The rapid detection method of GM potato using the UF-PCR system could be analyzed by selecting specific primers and identifying a specific amplification product with a unique melting temperature. In addition, this method was highly sensitive in detecting low copy numbers of DNA in GM potato samples. Recently, with the development of genetic analysis technology and equipment, high-speed analysis methods have been proposed to reflect the high-throughput demand for processing various samples. This reduces the analysis time, which normally takes 2 h, to as little as 40 min, thereby allowing more samples to be processed, faster results, and repetition of the test within the allotted time if case of any issues.</p><p id=\"Par20\">This study proposes a GM potato detection method using an UF-PCR system, which is a shortened test method. The method uses a very thin Rapi:chip and has the advantage of being able to finish the PCR reaction within a short time because of the high efficiency of heat transfer of the reaction mixtures and short duration between cycles.</p><p id=\"Par21\">The UF-PCR analysis method reviewed in this study is specific, sensitive, and reliable for identifying GM potato events. Moreover, the assay is eco-friendly because it uses a small amount of reagents, and if necessary, it can perform analysis using a small portable analyzer in field conditions rather than in the laboratory.</p><p id=\"Par22\">In conclusion, the reaction time and usage of reagents of the UF-PCR system are less than half of those for the conventional method, and the equipment is portable; therefore it can be utilized for on-site sample analysis. Therefore, we propose the use of this method that can perform a quick test within approximately 20 min to inspect and monitor GM potato events.</p></sec></sec></body><back><ack><title>Acknowledgements</title><p>This work was supported by the Ministry of Food and Drug Safety of Korea.</p></ack><sec><title>Funding</title><p>This study was funded by the Ministry of Food and Drug Safety of Korea (Grant Number 19161MFDS051).</p></sec><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par23\">The authors declare no conflict of interest.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Aboud</surname><given-names>M</given-names></name><name><surname>Oh</surname><given-names>HH</given-names></name><name><surname>McCord</surname><given-names>B</given-names></name></person-group><article-title xml:lang=\"en\">Rapid direct PCR for forensic genotyping in under 25 min</article-title><source>Electrophoresis.</source><year>2013</year><volume>34</volume><fpage>1539</fpage><lpage>1547</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXos12qsr0%3D</pub-id><pub-id pub-id-type=\"doi\">10.1002/elps.201200570</pub-id><pub-id pub-id-type=\"pmid\">23529977</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Arumuganathan</surname><given-names>K</given-names></name><name><surname>Earle</surname><given-names>ED</given-names></name></person-group><article-title xml:lang=\"en\">Nuclear DNA content of some important plant species</article-title><source>Plant Molecular Biology Reporter.</source><year>1991</year><volume>9</volume><fpage>208</fpage><lpage>218</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DyaK38XhslSjurY%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/BF02672069</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"other\">Chen XJ, Liu YT, Zhan XA, Gao YB, Sun ZY, Wen WJ, Zheng WD. Ultrafast PCR detection of COVID-19 by using a microfluidic chip-based system. Bioengineering (Basel). 9: 548 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.3390/bioengineering9100548\">https://doi.org/10.3390/bioengineering9100548</ext-link></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cote</surname><given-names>MJ</given-names></name><name><surname>Meldrum</surname><given-names>AJ</given-names></name><name><surname>Raymond</surname><given-names>P</given-names></name><name><surname>Dollard</surname><given-names>C</given-names></name></person-group><article-title xml:lang=\"en\">Identification of genetically modified potato cultivars using event specific polymerase chain reaction</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2005</year><volume>53</volume><fpage>6691</fpage><lpage>6696</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2MXmsFGjs74%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf050591i</pub-id><pub-id pub-id-type=\"pmid\">16104786</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"other\">Das D, Hsieh HC, Chen CS, Chen WL, Chuang HS. Ultrafast and sensitive screening of pathogens by functionalized janus microbeads-enabled rotational diffusometry in combination with isothermal amplification. Small Science. 2: 202200010 (2022) <ext-link ext-link-type=\"doi\" xlink:href=\"10.1002/smsc.202200010\">https://doi.org/10.1002/smsc.202200010</ext-link></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"other\">Food and Agriculture Organization of the United Nations. FAOSTAT Statistical Database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.fao.org/faostat\">https://www.fao.org/faostat</ext-link>. Accessed Aug. 27, 2022.</mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Fraiture</surname><given-names>MA</given-names></name><name><surname>Roosens</surname><given-names>NHC</given-names></name><name><surname>Taverniers</surname><given-names>I</given-names></name><name><surname>Loose</surname><given-names>MD</given-names></name><name><surname>Deforce</surname><given-names>D</given-names></name><name><surname>Herman</surname><given-names>P</given-names></name></person-group><article-title xml:lang=\"en\">Biotech Rice: Current developments and future detection challenges in food and feed chain</article-title><source>Trends in Food Science &amp; Technology.</source><year>2016</year><volume>52</volume><fpage>66</fpage><lpage>79</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC28XmtFCntr4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.tifs.2016.03.011</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Gao</surname><given-names>H</given-names></name><name><surname>Yu</surname><given-names>X</given-names></name><name><surname>Deng</surname><given-names>T</given-names></name><name><surname>Sun</surname><given-names>M</given-names></name><name><surname>Xiao</surname><given-names>X</given-names></name><name><surname>Huang</surname><given-names>X</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Event-specific detection of transgenic potato AV43-6-G7 using real-time and digital PCR methods</article-title><source>BMC Biotechnology.</source><year>2016</year><volume>16</volume><fpage>74</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXlvV2gt7s%3D</pub-id><pub-id pub-id-type=\"doi\">10.1186/s12896-016-0303-8</pub-id><pub-id pub-id-type=\"pmid\">27784303</pub-id><pub-id pub-id-type=\"pmcid\">5081928</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Hong</surname><given-names>YW</given-names></name><name><surname>Kim</surname><given-names>JJ</given-names></name><name><surname>Yu</surname><given-names>YC</given-names></name><name><surname>Kim</surname><given-names>HS</given-names></name><name><surname>Moon</surname><given-names>GI</given-names></name><name><surname>Park</surname><given-names>EM</given-names></name></person-group><article-title xml:lang=\"en\">Ultra-fast PCR method for the distinguishing between Miichthys miiuy and Sciaenops ocellatus</article-title><source>Food Science and Biotechnology.</source><year>2021</year><volume>30</volume><issue>9</issue><fpage>1225</fpage><lpage>1231</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXit1Ggs7rE</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-00954-4</pub-id><pub-id pub-id-type=\"pmid\">34603821</pub-id><pub-id pub-id-type=\"pmcid\">8423894</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"other\">International Service for the Acquisition of Agri-Biotech Applications (ISAAA). GM approval database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.isaaa.org/gmapprovaldatabase\">https://www.isaaa.org/gmapprovaldatabase</ext-link>. Accessed Sep. 18, 2022.</mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Jaccaud</surname><given-names>E</given-names></name><name><surname>Hohne</surname><given-names>M</given-names></name><name><surname>Meyer</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of screening methods for the identification of genetically modified potatoes in raw materials and finished products</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2003</year><volume>51</volume><fpage>550</fpage><lpage>557</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD38XpsFCnsLs%3D</pub-id><pub-id pub-id-type=\"doi\">10.1021/jf0208031</pub-id><pub-id pub-id-type=\"pmid\">12537422</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kim</surname><given-names>MJ</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Suh</surname><given-names>SM</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Rapid on-site detection of shrimp allergen tropomyosin using a novel ultrafast PCR system</article-title><source>Food Science and Biotechnology</source><year>2019</year><volume>28</volume><issue>2</issue><fpage>591</fpage><lpage>597</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXhvVerurfP</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-018-0479-x</pub-id><pub-id pub-id-type=\"pmid\">30956872</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"other\">Kim SH, Lee YS, Joo IS, Kwak HS, Chung GT, Kim SH. Rapid detection for salmonella spp. by ultrafast real-time PCR assay. J. Food Hyg. Saf. 33: 50–57 (2018)</mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>SW</given-names></name><name><surname>Lee</surname><given-names>SH</given-names></name><name><surname>Kim</surname><given-names>ES</given-names></name><name><surname>Kim</surname><given-names>DJ</given-names></name><name><surname>Park</surname><given-names>SH</given-names></name><name><surname>Lee</surname><given-names>EJ</given-names></name><name><surname>Lee</surname><given-names>SY</given-names></name><name><surname>Lee</surname><given-names>JS</given-names></name><name><surname>Lim</surname><given-names>CS</given-names></name><name><surname>Kim</surname><given-names>WK</given-names></name><name><surname>In</surname><given-names>KH</given-names></name></person-group><article-title xml:lang=\"en\">Rapid detection of mycobacterium tuberculosis using a novel ultrafast chip-type real-time polymerase chain reaction system</article-title><source>Chest.</source><year>2014</year><volume>146</volume><issue>5</issue><fpage>1319</fpage><lpage>1326</lpage><pub-id pub-id-type=\"doi\">10.1378/chest.14-0626</pub-id><pub-id pub-id-type=\"pmid\">24967657</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Martinez-Prada</surname><given-names>MM</given-names></name><name><surname>Curtin</surname><given-names>SJ</given-names></name><name><surname>Gutierrez-Gonzalez</surname><given-names>JJ</given-names></name></person-group><article-title xml:lang=\"en\">Potato improvement through genetic engineering</article-title><source>GM Crops &amp; Food.</source><year>2021</year><volume>12</volume><fpage>479</fpage><lpage>496</lpage><pub-id pub-id-type=\"doi\">10.1080/21645698.2021.1993688</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Milosevic</surname><given-names>J</given-names></name><name><surname>Lu</surname><given-names>M</given-names></name><name><surname>Greene</surname><given-names>W</given-names></name><name><surname>He</surname><given-names>HZ</given-names></name><name><surname>Zheng</surname><given-names>SY</given-names></name></person-group><article-title xml:lang=\"en\">An ultrafast one-step quantitative reverse transcription-polymerase chain reaction assay for detection of SARS-CoV-2</article-title><source>Frontiers in Microbiology.</source><year>2021</year><volume>12</volume><pub-id pub-id-type=\"doi\">10.3389/fmicb.2021.749783</pub-id><pub-id pub-id-type=\"pmid\">34803970</pub-id><pub-id pub-id-type=\"pmcid\">8600229</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"other\">Ministry of Food and Drug Safety (MFDS). GMO status database. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.foodsafetykorea.go.kr/portal/board/board.do\">https://www.foodsafetykorea.go.kr/portal/board/board.do</ext-link>. Accessed July. 15, 2022.</mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"other\">Ministry of Health, Labour and Welfare (MHLW). Detection method for genetically modified DNA (2020). Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf\">https://www.mhlw.go.jp/file/06-Seisakujouhou11130500-Shokuhinanzenbu/0000090554.pdf</ext-link>. Accessed Feb. 16, 2022.</mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Roh</surname><given-names>JK</given-names></name><name><surname>Lee</surname><given-names>T</given-names></name><name><surname>Jung</surname><given-names>SI</given-names></name><name><surname>Kim</surname><given-names>TA</given-names></name><name><surname>Park</surname><given-names>YH</given-names></name><name><surname>Kim</surname><given-names>YM</given-names></name></person-group><article-title xml:lang=\"en\">Qualitative and Quantitative PCR methods for detection of three lines of genetically modified potatoes</article-title><source>Journal of Agricultural and Food Chemistry.</source><year>2004</year><volume>52</volume><fpage>3269</fpage><lpage>3274</lpage><pub-id pub-id-type=\"doi\">10.1021/jf0499020</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"other\">Shin MK, Jeon SM, Hong SJ, Choi CH, Kim HA, Park JS. Research on the establishment of test method for labeling and management of GMOs. Ministry of Food and Drug Safety (2020)</mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Shin</surname><given-names>MK</given-names></name><name><surname>Jeon</surname><given-names>SM</given-names></name><name><surname>Koo</surname><given-names>YE</given-names></name></person-group><article-title xml:lang=\"en\">Development of a rapid detection method for genetically modified rice using the ultra-fast PCR system</article-title><source>Food Science and Biotechnology</source><year>2022</year><volume>31</volume><issue>2</issue><fpage>175</fpage><lpage>182</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XmsF2gs78%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s10068-021-01025-4</pub-id><pub-id pub-id-type=\"pmid\">35095287</pub-id><pub-id pub-id-type=\"pmcid\">8783574</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Song</surname><given-names>JY</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Detection of unapproved genetically modified potatoes in Korea using multiplex polymerase chain reaction</article-title><source>Food Control.</source><year>2017</year><volume>80</volume><fpage>19</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXmsFejtbw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodcont.2017.04.026</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Stein</surname><given-names>AJ</given-names></name><name><surname>Rodríguez-Cerezo</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Low-level presence of new GM crops: An issue on the rise for countries where they lack approval</article-title><source>AgBioForum.</source><year>2010</year><volume>13</volume><issue>2</issue><fpage>173</fpage><lpage>182</lpage></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tu</surname><given-names>YK</given-names></name><name><surname>Lin</surname><given-names>YC</given-names></name><name><surname>Feng</surname><given-names>YW</given-names></name><name><surname>Tseng</surname><given-names>YY</given-names></name><name><surname>Chen</surname><given-names>HW</given-names></name></person-group><article-title xml:lang=\"en\">Visual, sensitive and rapid event-specific detection of genetically modified potato EH92-527-1 by loop-mediated isothermal amplification method</article-title><source>Bioscience, Biotechnology, and Biochemistry.</source><year>2020</year><volume>84</volume><fpage>43</fpage><lpage>52</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXhsleitrjO</pub-id><pub-id pub-id-type=\"doi\">10.1080/09168451.2019.1661766</pub-id><pub-id pub-id-type=\"pmid\">31495297</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"other\">Visual Capitalist. Mapped: Food production around the world. Available from: <ext-link ext-link-type=\"uri\" xlink:href=\"https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world\">https://www.visualcapitalist.com/cp/mapped-food-production-around-the-world</ext-link>. Accessed Aug. 18, 2022.</mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Yoo</surname><given-names>MR</given-names></name><name><surname>Kim</surname><given-names>JH</given-names></name><name><surname>Yea</surname><given-names>MC</given-names></name><name><surname>Kim</surname><given-names>HY</given-names></name></person-group><article-title xml:lang=\"en\">Development of detection method of unapproved genetically modified potato (EH92-527-1) in Korea using duplex polymerase chain reaction</article-title><source>Korean Journal of Food Science and Technology.</source><year>2013</year><volume>45</volume><fpage>156</fpage><lpage>160</lpage><pub-id pub-id-type=\"doi\">10.9721/KJFST.2013.45.2.156</pub-id></mixed-citation></ref></ref-list></ref-list><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"other\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10953</journal-id><journal-id journal-id-type=\"doi\">10.1007/10953.1572-8927</journal-id><journal-title-group><journal-title>Journal of Solution Chemistry</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Solution Chem</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0095-9782</issn><issn pub-type=\"epub\">1572-8927</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10953-023-01308-w</article-id><article-id pub-id-type=\"manuscript\">1308</article-id><article-id pub-id-type=\"doi\">10.1007/s10953-023-01308-w</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Article</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">In Memoriam: Jacques Desnoyers (1935–2023)</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><name><surname>Quirion</surname><given-names>Francois</given-names></name><address><email>francois.quirion@physicochimie.fr</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs1095302301308w_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Perron</surname><given-names>Gérald</given-names></name><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Desnoyers</surname><given-names>Christine</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au4\"><name><surname>Desnoyers</surname><given-names>Marie</given-names></name></contrib><contrib contrib-type=\"author\" id=\"Au5\"><name><surname>Desnoyers</surname><given-names>Nichola</given-names></name></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution content-type=\"org-name\">Consultant R&amp;D Physicochimie</institution></institution-wrap><addr-line content-type=\"street\">261 Boulevard Pereire</addr-line><addr-line content-type=\"postcode\">75017</addr-line><addr-line content-type=\"city\">Paris</addr-line><country country=\"FR\">France</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution content-type=\"org-name\">Retired from INRS-Énergie Matériaux et Télécommunications</institution></institution-wrap><addr-line content-type=\"city\">Varennes</addr-line><addr-line content-type=\"state\">QC</addr-line><country country=\"CA\">Canada</country></aff></contrib-group><author-notes><corresp id=\"IDs1095302301308w_cor1\"><label>a</label><email>francois.quirion@physicochimie.fr</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>20</day><month>6</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>52</volume><issue seq=\"1\">8</issue><fpage>859</fpage><lpage>860</lpage><history><date date-type=\"registration\"><day>31</day><month>5</month><year>2023</year></date><date date-type=\"online\"><day>20</day><month>6</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>9</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>5</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10953_2023_Article_1308.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>29</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>EditorialNotes</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Geochemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Oceanography</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Inorganic Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Condensed Matter Physics</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>Christine Desnoyers, Marie Desnoyers and Nichola Desnoyers are son and daughters of Professor Desnoyers.</p></notes></front><body><p id=\"Par1\">It is with great sadness that we report the death of Jacques Desnoyers who passed away peacefully in Quebec City on January 12th at the age of 87, surrounded by his loved ones.<fig id=\"Figa\" position=\"anchor\"><graphic id=\"MO1\" mime-subtype=\"JPEG\" position=\"anchor\" specific-use=\"web\" xlink:href=\"MediaObjects/10953_2023_1308_Figa_HTML.jpg\"/></fig></p><p id=\"Par2\">Jacques Desnoyers was born on 28th January 1935 in Ottawa where he obtained his Ph.D. in 1961 at the University of Ottawa. After a year of post-doctorate studies in England (University of London and of Manchester), he joined, in 1962, the chemistry department at the <italic>Université de Sherbrooke</italic> as full professor and later as director. In 1983, he became the scientific director of <italic>Institut national de la recherche scientifique</italic> (INRS) where he also acted as interim general director (3 March 1987 to 31 July 1988). In 1991, he returned to research as full professor at <italic>INRS</italic>. During the same period, he was appointed as Ombudsman at the <italic>Université du Québec</italic>. In 2002, he retired from INRS. Meanwhile, he accepted the position of scientific advisor for the <italic>Gouvernement du Québec</italic> until 2006. He finally retired fully in 2007.</p><p id=\"Par3\">He developed and promoted flow microcalorimetry with Patrick Picker. He investigated the thermodynamic properties of electrolytes and non-electrolytes in aqueous, non-aqueous and mixed solvents, surfactant solutions, swollen micelles and microemulsions, and polymer-surfactant systems. His research found applications in the fields of low-temperature batteries, Li-ion batteries, enhanced oil recovery and soil remediation.</p><p id=\"Par4\">His work was internationally recognized and he frequently collaborated with other Canadian groups but also with groups around the world, including the USA, France and Italy. After his retirement, he received the 2007 Setaram Award from the <italic>Associazione italiana di calorimetria e analisi termica</italic> (AICAT) for his outstanding scientific achievements in the field of calorimetry.</p><p id=\"Par5\">Throughout his career, Jacques Desnoyers strongly believed in the importance of transmitting knowledge which he did through teaching, directing thesis, writing scientific articles and book chapters and reviewing manuscripts for many scientific journals.</p><p id=\"Par6\">He will be remembered for his <italic>joie de vivre</italic>, his human qualities, his professional ethical principles and for his continuous effort to promote the use of thermodynamic properties as powerful tools for the investigation of simple and complex systems.</p></body><back><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"research-article\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">11696</journal-id><journal-title-group><journal-title>Chemical Papers</journal-title><abbrev-journal-title abbrev-type=\"publisher\">Chem. Pap.</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">0366-6352</issn><issn pub-type=\"epub\">2585-7290</issn><publisher><publisher-name>Springer International Publishing</publisher-name><publisher-loc>Cham</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s11696-023-02771-x</article-id><article-id pub-id-type=\"manuscript\">2771</article-id><article-id pub-id-type=\"doi\">10.1007/s11696-023-02771-x</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Original Paper</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Optimization of microwave-enhanced extraction parameters to recover phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves</article-title></title-group><contrib-group><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au1\"><contrib-id contrib-id-type=\"orcid\">http://orcid.org/0000-0002-5114-2100</contrib-id><name><surname>Alara</surname><given-names>Oluwaseun Ruth</given-names></name><address><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref><xref ref-type=\"corresp\" rid=\"IDs1169602302771x_cor1\">a</xref></contrib><contrib contrib-type=\"author\" id=\"Au2\"><name><surname>Abdurahman</surname><given-names>Nour Hamid</given-names></name><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"aff\" rid=\"Aff2\">2</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Ali</surname><given-names>Hassan Alsaggaf</given-names></name><xref ref-type=\"aff\" rid=\"Aff3\">3</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Centre for Research in Advanced Fluid and Processes (Fluid Centre)</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff2\"><label>2</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.440438.f</institution-id><institution-id institution-id-type=\"ISNI\">0000 0004 1798 1407</institution-id><institution content-type=\"org-division\">Department of Chemical Engineering, College of Engineering</institution><institution content-type=\"org-name\">Universiti Malaysia Pahang</institution></institution-wrap><addr-line content-type=\"street\">Lebuhraya Tun Razak</addr-line><addr-line content-type=\"postcode\">26300</addr-line><addr-line content-type=\"city\">Gambang</addr-line><addr-line content-type=\"state\">Pahang</addr-line><country country=\"MY\">Malaysia</country></aff><aff id=\"Aff3\"><label>3</label><institution-wrap><institution content-type=\"org-name\">Eastern Unity Technology</institution></institution-wrap><addr-line content-type=\"street\">Suite 01, 12th Floor Plaza, 138 Annex Hotel Maya, Jalan Ampang</addr-line><addr-line content-type=\"postcode\">50450</addr-line><addr-line content-type=\"city\">Kuala Lumpur</addr-line><country country=\"MY\">Malaysia</country></aff></contrib-group><author-notes><corresp id=\"IDs1169602302771x_cor1\"><label>a</label><email>alara@ump.edu.my</email><email>ruthoalao@gmail.com</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>10</day><month>4</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>77</volume><issue seq=\"9\">8</issue><fpage>4217</fpage><lpage>4233</lpage><history><date date-type=\"registration\"><day>6</day><month>3</month><year>2023</year></date><date date-type=\"received\"><day>23</day><month>11</month><year>2022</year></date><date date-type=\"accepted\"><day>4</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>10</day><month>4</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© Institute of Chemistry, Slovak Academy of Sciences 2023</copyright-statement><copyright-statement content-type=\"comment\" specific-use=\"RightsRetention\">Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>Institute of Chemistry, Slovak Academy of Sciences</copyright-holder></permissions><abstract id=\"Abs1\" xml:lang=\"en\"><title>Abstract</title><p id=\"Par1\">Vegetables are industrial crops endowed with both nutritional and medicinal values. The overwhelming contributions of vegetables to human living in the form of nutrients and medicine cannot be under emphasised. Thus, this study examined the recoveries of phenolic compounds and antioxidants from <italic>Corchorus olitorius</italic> leaves using a microwave-enhanced extraction technique. Furthermore, the phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were comprehensively identified using liquid chromatography-mass spectrometry quadrupole of flight (LC-QToF-MS). At the optimized conditions of microwave-enhanced extraction (extraction time of 131 s, microwave power 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%), total phenolic content (TPC) of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for 2,2-diphenyl-1-picrylhydrazyl (DPPH) and 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS) assays, respectively, were achieved. Furthermore, an aggregate of fourteen phenolic compounds that include 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose was tentatively identified in the leaf extract of <italic>C. olitorius</italic>. In general, this study has established <italic>C. olitorius</italic> leaves as sources of phenolic compounds and natural antioxidants. Thus, the intake can continue to be promoted as a way forward in solving the problem of food insecurity.</p></abstract><kwd-group xml:lang=\"en\"><title>Keywords</title><kwd><italic>Corchorus olitorius</italic></kwd><kwd>Response surface methodology</kwd><kwd>Total phenols</kwd><kwd>Antioxidant</kwd><kwd>Liquid chromatography quadrupole time-of-flight mass spectrometry</kwd><kwd>Microwave-enhanced extraction</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>12</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>53</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Institute of Chemistry, Slovak Academy of Sciences</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>Yes</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>Unnumbered</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/11696_2023_Article_2771.pdf</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>21</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>OriginalPaper</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Chemistry/Food Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Industrial Chemistry/Chemical Engineering</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biochemistry, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Medicinal Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Materials Science, general</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Biotechnology</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"ESMHint\"><title>Supplementary Information</title><p>The online version contains supplementary material available at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s11696-023-02771-x\">https://doi.org/10.1007/s11696-023-02771-x</ext-link>.</p></notes><notes notes-type=\"CopyrightComment\" specific-use=\"RightsRetention\"><title>Copyright comment</title><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p></notes></front><body><sec id=\"Sec1\"><title>Introduction</title><p id=\"Par12\">Vegetables have been categorised as industrial crops with valuable means of nutrients and a source of income. Other than the valuable nutrients in vegetables, some of them possess diverse medicinal characteristics that are good for human and animal health. Although indigenous vegetables are long been employed in day-to-day diets in different communities of the world, several of these vegetables are underutilised as most have embodiments of unknown and underestimated pharmacological features. Most attention is based on the consumption of maize, rice, and cassava. It is not disputed these crops are endowed with great nutritional values; nevertheless, vegetables can be prioritised due to their functional properties (food and medicine). Additionally, United Nation Sustainable Development Goals (SDG 2) established the importance of figuring out sustainable means of producing foods for humans, especially natural sources. This is highly needed in this time of post-Covid-19 recovery as the world is being faced with hunger, food insecurity and consumption of low-quality foods that cause micronutrient deficiencies (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>)</p><p id=\"Par13\"><italic>Corchorus olitorius</italic> (Jute mallow) is a leafy vegetable mostly found in Tropical regions including the Middle East, Africa and some parts of Asia (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). It belongs to the family of Malvaceae (Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). This is one of the vegetables consumed mainly in different African countries including Nigeria, Tanzania, South Africa, and Ethiopia (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). Consuming this vegetable has been encouraged to curb malnutrition and food insecurity, especially in developing nations such as African countries. The consumption of <italic>C. olitorius</italic> leaves is much encouraged as it is believed to overcome nutritional and food insecurity among children under the ages of five, lactating mothers and pregnant women in sub-Saharan Africa (Andersen et al. <xref ref-type=\"bibr\" rid=\"CR9\">2003</xref>; Kamga et al. <xref ref-type=\"bibr\" rid=\"CR17\">2013</xref>). This vegetable is mostly consumed in the form of soup or sauce alongside solid foods from rice, yam or cassava in African cooking cuisine. It is endowed with different vitamins, enrich in fibre, and antioxidants (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). A brief description of <italic>C. olitorius</italic>: it is an annual vegetable that can reach a height of about 2.4 m; the leaves are finely lobed or serrated margin, simple, lanceolate, and alternate; it has only a few branches and can be unbranched; seeds are used when planting; the flowers are yellowish small with five petals (2–3 cm in diameters) and can survive in any kinds of soil (loamy, sandy or clay) (Loumerem and Alercia <xref ref-type=\"bibr\" rid=\"CR21\">2016</xref>). The leaves of <italic>C. olitorius</italic> are enriched in beta carotene, vitamins A and C, folic acid, calcium, iron, and diverse phenolic compounds (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). The leaves of <italic>C. olitorius</italic> were reported to contain nutritional values more than stems and fruits. This is the reason why it is generally used as food. The leaves had been revealed to be made up of about seventeen active nutrients compounds such as protein, carbohydrate, fat, ash, fibre, potassium, calcium, sodium, phosphorous, iron, beta-carotene, riboflavin, thiamine, ascorbic acid, niacin, and so on. Moreover, the leaves had been reported to possess hepatoprotective, antidiabetic, antioxidant, diuretic, antipyretic, analgesic, antitumor, antimicrobial, anti-obesity, hypoglycemic, and gastroprotective properties (Abir et al. <xref ref-type=\"bibr\" rid=\"CR1\">2019</xref>; Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>; Saliu et al. <xref ref-type=\"bibr\" rid=\"CR28\">2019</xref>).</p><p id=\"Par14\">The presence of phenolic compounds in plant materials is continually studied due to their important link with the occurrence of natural antioxidants that possess outstanding applications in human health (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Cheynier <xref ref-type=\"bibr\" rid=\"CR13\">2012</xref>). The antioxidants from leafy vegetables such as <italic>C. olitorius</italic> can act in place of synthetic antioxidants readily available in the market. These synthetic antioxidants are being reported to pose side effects on consumers (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). However, natural antioxidants can act as protection against degenerative diseases and alter the DNA, proteins and lipids (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). Besides, plants are surviving the exposure to ultraviolet radiation in the environment; this has positioned them as a perfect antioxidant source. Likewise, the consumptions of plant-sourced antioxidants are relatively safe and widely accepted; this has propelled the introduction of plant polyphenols into cosmeceutical and pharmaceutical products in place of synthetic antioxidants (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>; Lobo et al. <xref ref-type=\"bibr\" rid=\"CR20\">2010</xref>). The process of extraction is important when targeting to achieve significant phenolic compounds and antioxidants from plant materials. This process can either be conventional or non-conventional; the latter is currently being used due to its ability to recover a tangible quantity of phenolic compounds and antioxidants from plant materials (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Herrero et al. <xref ref-type=\"bibr\" rid=\"CR16\">2012</xref>). Out of several unconventional techniques that are being used, the microwave-enhanced method has been generally accepted due to its ability to leach a significant quantity of phenolic compounds from the considered sample within a reduced duration of time and not alter the bioactive elements in the extracts (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>, <xref ref-type=\"bibr\" rid=\"CR7\">b</xref>; Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>).</p><p id=\"Par15\">Several parameters can alter the efficiency of recovered phenolic compounds and antioxidants from plant matrices when using microwave-enhanced technique; these are temperature, extraction time, solvent/sample ratio, solvent concentration, microwave power, and solvent selection. Out of different available solvents used in the extraction processes, ethanol and water are the most environment-friendly. It had been explained that pure ethanol cannot extract the totality of phenolic compounds especially those that are hydrophilic (water-soluble) (Thoo et al. <xref ref-type=\"bibr\" rid=\"CR32\">2010</xref>). Previous studies had suggested that the use of aqueous ethanol or binary mixture contributes to the recovery of antioxidant from plant sample (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>). Thus, mixing these two solvents (water and ethanol) in rightful proportion tends to improve the recovery of bioactive materials from plant matrices (Khoddami et al. <xref ref-type=\"bibr\" rid=\"CR18\">2013</xref>). As the selection of extraction is important in achieving significant recovery, process optimization is as well essential to achieve this goal.</p><p id=\"Par16\">There are two common techniques used for process optimization (one leading to the other); these are one-factor-at-a-time (OFAT) experimental trials and response surface methodology (RSM). Although OFAT suffers drawbacks such as no room for interaction between the factors; nonetheless, OFAT is widely employed to determine the range of values used for the response surface methodology. RSM is utilised to settle the limitations of OFAT experimental trials which include inadequate data evaluation, being expensive and time-consuming (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par17\">Although diverse studies had been carried out on <italic>C. olitorius</italic> leaves concerning the proximate constituents, total phenols and antioxidants (Ndlovu et al. <xref ref-type=\"bibr\" rid=\"CR24\">2020</xref>; Youssef et al. <xref ref-type=\"bibr\" rid=\"CR35\">2019</xref>). Nevertheless, there is yet a report on the optimization studies involving microwave-enhanced extraction of the total phenols and antioxidants from <italic>C. olitorius</italic> leaves and comprehensive identifications of the phenolic compounds. Hence, this study focuses on unveiling the optimised microwave-enhanced extraction conditions to achieve maxima phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. At the optimized conditions, the comprehensive identifications of different phenolic compounds in the extracts were carried out using a fast liquid chromatography quadrupole time-of-flight mass spectrometry (LC-QToF-MS).</p></sec><sec id=\"Sec2\" sec-type=\"materials|methods\"><title>Materials and methods</title><sec id=\"Sec3\"><title>Cultivation of <italic>C. olitorius</italic> seeds and sample</title><p id=\"Par18\">The seeds of <italic>C. olitorius</italic> were cultivated in a small garden in the Universiti Malaysia Pahang (UMP) for two months (from February and April 2019); every agricultural practice such as watering, weed control and thinning was adequately observed to achieve productive results. Sandy-loam soil was used for the cultivation. In the maturity stage, the leaves of <italic>C. olitorius</italic> were harvested and washed to evacuate dirt under running water. Then, the leaves were allowed to drain off the water and dried at room temperature until a constant weight was reached. The dried leaves were milled using a blender and sieved using a laboratory sieve (400/425 µm pore size). Afterwards, the ground sample was kept in a zip-lock bag and kept in a freezer at − 20 °C before the extraction process.</p></sec><sec id=\"Sec4\"><title>Chemicals and reagents used</title><p id=\"Par19\">Ethanol, gallic acid, methanol, 2,2-diphenyl-1-picrylhydrazyl (DPPH), sodium carbonate, 2,2′-azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid) (ABTS), potassium persulfate, and Folin–Ciocalteu reagent were procured from Sigma-Aldrich (Selangor, Malaysia).</p></sec><sec id=\"Sec5\"><title>Microwave-enhanced extraction through experimental design</title><p id=\"Par20\">The solvents used in this study were water and ethanol. An Ethos microwave oven (Milestone, Italy) was employed, and this extractor is configured using a control compartment with power, time and temperature control. Since the microwave-enhanced extraction technique is influenced by microwave power, temperature, time, solvent concentration, and solvent/sample ratio; thus, these factors were first studied by carrying out OFAT experimental trials to evaluate the effect of an individual factor on the recovered phenolic compounds and antioxidants as well as the range for individual factor. These ranges of values were employed for the optimization studies. For the extraction, 5 g of <italic>C. olitorius</italic> leaf sample was adequate mixed together with an estimated proportion of ethanol–water solvent (experimental design). The mixture was filtered, and a rotary evaporator set at 50 °C was utilised to concentrate the extract.</p><p id=\"Par21\">After the determination of preliminary ranges for the five considered factors through OFAT experimental trials, a face-centred central composite design (FCCCD) was used to optimise the recovery of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Four factors were considered during the optimization study since temperature (kept at 40 °C) showed an insignificant effect on the dependent factors (total phenolic compounds (TPC) and antioxidant activity (DPPH and ABTS)). Thus, the independent factors were extraction time (<italic>X</italic><sub>1</sub>: 60–180 s), microwave power (<italic>X</italic><sub>2</sub>: 200–400 W), solvent/sample (<italic>X</italic><sub>3</sub>: 8–16 mL/1 g), and ethanol concentration (<italic>X</italic><sub>4</sub>: 30–50%), while the dependent factors were TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>). Thirty randomised runs of experiments with six centre points were obtained using FCCCD (Table S1 in supplementary materials). The regression analysis for this study was initiated using Eq. (<xref ref-type=\"disp-formula\" rid=\"Equ1\">1</xref>).<disp-formula id=\"Equ1\"><label>1</label><alternatives><mml:math display=\"block\" id=\"Equ1_Math\"><mml:mrow><mml:mi>Y</mml:mi><mml:mo>=</mml:mo><mml:msub><mml:mi>β</mml:mi><mml:mn>0</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mrow><mml:mo>+</mml:mo><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn><mml:mo>,</mml:mo><mml:mi>i</mml:mi><mml:mo>&lt;</mml:mo><mml:mi>j</mml:mi></mml:mrow><mml:mrow><mml:mi>k</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:mrow></mml:munderover><mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>j</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ij</mml:mi></mml:mrow></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>i</mml:mi></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mi>j</mml:mi></mml:msub><mml:mo>+</mml:mo></mml:mrow></mml:mrow><mml:munderover><mml:mo movablelimits=\"false\">∑</mml:mo><mml:mrow><mml:mi>i</mml:mi><mml:mo>=</mml:mo><mml:mn>2</mml:mn></mml:mrow><mml:mi>k</mml:mi></mml:munderover><mml:mrow><mml:msub><mml:mi>β</mml:mi><mml:mrow><mml:mi mathvariant=\"italic\">ii</mml:mi></mml:mrow></mml:msub><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mi>i</mml:mi></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mrow></mml:math><tex-math id=\"Equ1_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$Y = \\beta_{0} + \\sum\\limits_{i = 1}^{k} {\\beta_{1} X_{1} } + \\sum\\limits_{i = 1,i &lt; j}^{k = 1} {\\sum\\limits_{j = 2}^{k} {\\beta_{ij} X_{i} X_{j} + } } \\sum\\limits_{i = 2}^{k} {\\beta_{ii} X_{i}^{2} }$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ1.gif\"/></alternatives></disp-formula>where <italic>Y</italic> indicates the dependent factors which are TPC (<italic>Y</italic><sub>1</sub>), DPPH (<italic>Y</italic><sub>2</sub>) and ABTS (<italic>Y</italic><sub>3</sub>), <italic>β</italic><sub>0</sub> is a constant, <italic>β</italic><sub><italic>i</italic></sub> indicates the linear coefficient (actual effect), <italic>β</italic><sub><italic>ij</italic></sub> represents the interaction coefficients between two considered extraction factors, <italic>β</italic><sub><italic>ii</italic></sub> is the second-order polynomial coefficient, and <italic>k</italic> is the number of dependent factors (<italic>k</italic> = 4). The significant values were removed, while only the insignificant values were used to construct a reduced model (Che Sulaiman et al. <xref ref-type=\"bibr\" rid=\"CR12\">2017</xref>).</p><p id=\"Par22\">Having achieved the predicted yields of the considered responses, the adequacy of the constructed models was tested. These were carried out by comparing the actual and predicted values. A pair <italic>t</italic>-test analysis was employed to verify the postulated models.</p></sec><sec id=\"Sec6\"><title>Determination of TPC in the <italic>C. olitorius</italic> leaf extract</title><p id=\"Par23\">The TPC in extract from <italic>C. olitorius</italic> leaves was estimated following the previously described procedures (Alara and Abdurahman <xref ref-type=\"bibr\" rid=\"CR2\">2019</xref>). Briefly, 200 µL of Folin–Ciocalteu reagent was added to 100 µL of leaf extract of <italic>C. olitorius</italic>; then, 0.2 mM Na<sub>2</sub>CO<sub>3</sub> using the volume of 600 µL was added and the mixture was allowed to stay 2 h in the dark at room temperature for adequate reactions. After this, the absorbance of the reacted mixture was read at 765 nm.</p></sec><sec id=\"Sec7\"><title>Determination of antioxidant activity</title><sec id=\"Sec8\"><title>DPPH analysis</title><p id=\"Par24\">The free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, the DPPH solution was prepared to achieve 0.1 mM. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves was presented as IC<sub>50</sub> (this is the concentration of the <italic>C. olitorius</italic> leaf extract needed to inhibit 50% formation of DPPH radicals). These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 200 µL of the extract and 0.1 mM DPPH solution (2 mL volume) were thoroughly mixed. The absorbance was read at 517 nm after the mixture was left to adequate react for 30 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1478<italic>x</italic> + 19.781, <italic>R</italic><sup>2</sup> = 0.9986).</p></sec><sec id=\"Sec9\"><title>ABTS analysis</title><p id=\"Par25\">For the ABTS analysis, the free radical scavenging potential of extracts from <italic>C. olitorius</italic> leaves using DPPH assay was done following the procedure previously published (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>). In brief, an equal ratio of 2.45 mM potassium persulfate and 7 mM ABTS<sup>+</sup> solution was dissolved in 120 mL of methanol to achieve the stock solution of ABTS. This stock solution was allowed to react for 12 h before its usage. Then, 2 mL of stock solution was further diluted in 120 mL of methanol to obtain a fresh solution with an absorbance of 1.10 ± 0.02 at 734 nm. The antioxidant activity of the extract from <italic>C. olitorius</italic> leaves using ABTS was presented as IC<sub>50</sub>. These were done by preparing four different concentrations for each run of extract (0.1, 0.2, 0.3, and 0.5 mg/mL). From each concentration, 2.85 mL of a fresh solution of ABTS and 0.15 mL of extract were thoroughly mixed. At 734 nm, the absorbance was recorded after the mixture had been left to react for 120 min. IC<sub>50</sub> was determined from the linear graph between the percentage of inhibition and extract concentration (<italic>y</italic> = 0.1008<italic>x</italic> + 46.858, <italic>R</italic><sup>2</sup> = 0.9977).</p></sec></sec><sec id=\"Sec10\"><title>LC-QToF-MS analysis of phenolic compounds</title><p id=\"Par26\">Every phenolic compound in the leaf extract of <italic>C. olitorius</italic> at the optimized microwave-enhanced extraction conditions was identified using LC-QToF-MS (Waters Vion IMS, USA). A C<sub>18</sub> column with the dimension of 100 mm × 2.10 mm × 1.80 µm was used. There are two mobile phases used in this analysis; they are water + 0.1% formic acid (A) and 100% acetonitrile (B). The operating conditions were a seal wash period of 5 min, a pre-injector volume of 0 µL, low pressure of 0 psi, high-pressure limit of 18,000 psi, gradient elution was 90% A and 10% B (1.25 min), 45% A and 55% B (4.17 min), 10% A and 90% B (6.25 min), 90% A and 10% B (8.34 min), with the injection volume of 20 µL, the flow rate of 0.50 mL/min, spray voltage of 4 keV, mass range of 100–1000 m/z, sample temperature of 15 °C, column temperature of 40 °C, gas flow of 0.50 mL/min, desolvation gas flow rate of 800 L/h, desolvation temperature of 550 °C, low collision energy of 4 eV, high collision energy ramp between 10 and 45 eV, source temperature of 120 °C. The identifications of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> were performed by utilising SYNAPT mass spectrometer (Waters) coupled with an electrospray ionization operated in negative ion mode. Waters® UNIFY Software 1.0.0 installed in equipment was used to identify the phenolic compounds.</p></sec><sec id=\"Sec11\"><title>Data analysis</title><p id=\"Par27\">Each experimental trial was repeated thrice, and the data were recorded as mean values ± standard deviation. The Design Expert® software (Version 7, Stat. Ease Inc., Minneapolis, USA) was employed to carry out the optimization study. The model significance was based on a <italic>p</italic> value less than 0.05 and vice versa for the insignificant model. Three-dimensional plots were used to understand the interactions between two factors. A Chi-squared test was used to verify the validity of the predicted models.</p></sec></sec><sec id=\"Sec12\"><title>Results and discussions</title><sec id=\"Sec13\"><title>One-factor-at-a-time experimental trials for the extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</title><p id=\"Par28\">From the beginning of this study, the preliminary experimental trials were to study the effects of the factors influencing the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves using a microwave-enhanced extraction technique. These factors were extraction time, microwave power, solvent/sample ration, ethanol concentration, and temperature. Likewise, the experimental ranges for the subsequent optimization study were determined using OFAT. OFAT experimental trials involve varying a factor, while others are kept constant.</p><sec id=\"Sec14\"><title>Contributing impact of extraction time</title><p id=\"Par29\">Extraction time is an essential factor when considering solid–liquid extraction; this is due to its impact on mass transfer and solubility. Figure <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ai and Aii showed the impact of extraction time on the recoveries of phenolics and antioxidants from <italic>C. olitorius</italic>. This was done by keeping solvent/sample ratio, microwave power, ethanol concentration, and temperature at 8 mL/g, 200 W, 20%, and 20 °C, respectively. It can be observed that TPC recovery increased with extraction time to achieve a higher yield at 120 s before the decline in the yield set in. This might be because the extended time of extraction can be dangerous to phenolic compounds in the plant matrix; leading to degradation when expose to microwave radiation (Alara et al. <xref ref-type=\"bibr\" rid=\"CR7\">2021b</xref>; Azwanida <xref ref-type=\"bibr\" rid=\"CR10\">2015</xref>). On contrary, the recoveries of antioxidant through DPPH and ABTS declined with increasing time; the minima yield of antioxidants was achieved at 120 s prior to the further increase in the yield with increasing extraction time. This decrease occurs because the antioxidant activity is inversely proportional to IC<sub>50</sub> values (linear regression between the percentage of inhibition and extract concentration) (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>). Thus, a lesser IC<sub>50</sub> value indicates higher antioxidant potential. The maxima scavenging potentials of <italic>C. olitorius</italic> leaves for both DPPH and ABTS were reached at 120 s. For the optimization study, 60–180 s was the range of values. However, 120 s was considered as the fixed extraction time when studying the impacts of other factors.<fig id=\"Fig1\"><label>Fig. 1</label><caption xml:lang=\"en\"><p>Multiple parameters that influence the recoveries from phenolics and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO1\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig1_HTML.png\"/></fig></p></sec><sec id=\"Sec15\"><title>Contributing impact of solvent/sample ratio</title><p id=\"Par30\">During the solid–liquid extraction, the solvent/sample ratio can impact the recoveries of phenolic compounds and antioxidants. This is because this factor has an impact on the concentration gradient between the solutes in the plant sample and the solvent (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). In this study, the weight of the <italic>C. olitorius</italic> leaves was kept constant, while the volume of the solvent was varied. The effect of solvent/sample ratio was studied by keeping extraction time, microwave power, ethanol concentration, and temperature at 120 s, 200 W, 20%, and 20 °C, respectively. As seen in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bi, the recovery of TPC tends to increase as the solvent/sample ratio improves until the maximum recovery of TPC was obtained when the solvent/sample ratio reached 12 mL/g. Beyond the solvent/sample ratio of 12 mL/g, the occurrence of phenolic compounds in the leaf extract of <italic>C. olitorius</italic> declined. For the maxima, antioxidant potentials through DPPH and ABTS (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Bii); the lowest IC<sub>50</sub> values (strong antioxidant) were reached at a solvent/sample ratio of 12 mL/g. Beyond 12 mL/g, the antioxidant potentials declined with increasing solvent volume. This is industrial valuable because reducing solvent usage that achieves higher recovery favour the production cost (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Bouras et al. <xref ref-type=\"bibr\" rid=\"CR11\">2015</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Moreover, the higher solvent volume tends to increase the concentration gradient; this can then lead to an increased diffusion rate of solute into the solvent. Therefore, 8–16 mL/g was chosen as a range of values for the optimization. However, 12 mL/g was considered as the fixed solvent/sample ratio when studying the impacts of other factors.</p></sec><sec id=\"Sec16\"><title>Contributing impact of microwave power</title><p id=\"Par31\">In microwave-enhanced extraction, the plant sample absorbs microwave energy, this energy changes to heat and evaporation kick-starts from the left-over moisture content in the plant matrix. The evaporated water then causes the pressure body-up within the cell wall, leading to cell rupture in the plant sample. Thus, the bioactive compounds within the plant matrix will leach out into the surrounding solvent after cell rupture (Mandal et al. <xref ref-type=\"bibr\" rid=\"CR23\">2007</xref>).</p><p id=\"Par32\">In this, the impact of microwave power on the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves was investigated by fixing the following factors: extraction time (120 s), solvent/sample ratio (12 mL/g), ethanol concentration (20%), and temperature (20 °C) (Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ci and Cii). There was a slight increase in the TPC recovery as the power increased. At microwave power of 300 W, maximum TPC was achieved, after which the recovery of phenolic compounds tends to decrease against increasing microwave power. In contrast, the antioxidants (IC<sub>50</sub> values) for both DPPH and ABTS declined with increasing microwave power. This is justifiable because a lower IC<sub>50</sub> value indicates higher antioxidant potential. The minima IC<sub>50</sub> values for DPPH and ABTS were obtained at 300 W of microwave power. Beyond 300 W, the phenolic compounds and antioxidants declined with increasing power. Increasing the microwave power can propel increasing temperature. At higher temperatures, the viscosity and surface tension of extracting solvent will be reduced as the diffusion coefficient and solubility increase. Nonetheless, overexposure of the bioactive compounds already leached out into the solvent to higher microwave power propels the degradation of these phenolic compounds and antioxidants (thermal-sensitive phytochemicals) (Quoc <xref ref-type=\"bibr\" rid=\"CR26\">2021</xref>). Thus, 200–400 W was selected as a range of values for the optimization. Nevertheless, 300 W was considered as the fixed microwave power when studying the remaining factors.</p></sec><sec id=\"Sec17\"><title>Contributing impact of ethanol concentration</title><p id=\"Par33\">Organic solvents are mostly used in the extraction of phenolic compounds and antioxidants from plant materials. These solvents are mostly mixed with water because it enhances the water-soluble phytochemicals in the plant sample. Moreover, the mixture of organic solvents with water tends to create a moderate polar medium. Out of several organic solvents used in extraction, ethanol has been considered safe because it is environment-friendly (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>).</p><p id=\"Par34\">The influence of ethanol concentration was investigated by fixing extraction time, microwave power, solvent/sample ratio, and temperature at 120 s, 300 W, 12 mL/g, and 20 °C, respectively. As provided in Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Di and Dii, the recovery of TPC tends to increase and IC<sub>50</sub> values (showing higher antioxidants) declined as ethanol concentration in water increased until the maximum recoveries of TPC and antioxidants (both DPPH and ABTS) were achieved ethanol concentration of 40%. Beyond this concentration, the phenolic compounds and antioxidants in the leaf extract of <italic>C. olitorius</italic> declined. Hence, the ethanol concentration ranged between 30 and 50% for the optimization. However, 40% was considered the fixed concentration for the next factor.</p></sec><sec id=\"Sec18\"><title>Contributing impact of temperature</title><p id=\"Par35\">In solid–liquid extraction, the higher temperature tends to upsurge the release of phenolic compounds and antioxidants from the plant matrix. This is due to its influence on mass transfer (diffusion coefficient), equilibrium (solubility) and stability of phytochemicals (Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>). Although, heating can soften the plant tissue and weaken the interactions of phenolic compounds; hence, it promotes the migration of bioactive compounds into the extracting solvent. Nonetheless, a higher temperature can cause degradation and oxidation of phenolic compounds and antioxidants in the plant sample.</p><p id=\"Par36\">At fixed extraction time (120 s), microwave power (300 W), solvent/sample ratio (12 mL/g), and an ethanol concentration of 40%, the impact of the temperature was as well studied. In Fig. <xref ref-type=\"fig\" rid=\"Fig1\">1</xref>Ei and Eii, it can be observed that temperature did not influence the recoveries of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. This might be because the increasing temperature beyond a certain limit can profuse solvent loss through augment and vaporization; leading to the destruction of the phenolic compounds and antioxidants (Alara et al. <xref ref-type=\"bibr\" rid=\"CR3\">2018a</xref>; Tchabo et al. <xref ref-type=\"bibr\" rid=\"CR31\">2018</xref>).</p></sec></sec><sec id=\"Sec19\"><title>Optimization study and modelling of microwave-enhanced extraction of <italic>C. olitorius</italic> leaves</title><sec id=\"Sec20\"><title>Model fittings</title><p id=\"Par37\">The results from OFAT experimental trials have established the impacts of extraction time, solvent/sample ratio, microwave power, and ethanol concentration. These four factors were optimized through microwave-enhanced extraction to recover higher yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. In this study, FCCCD was employed to optimize this process. This was used because it is the simplest form of RSM. A total of 30 experimental trials were generated (Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>). The obtained outcomes from the randomized experimental trials are presented in Table <xref ref-type=\"table\" rid=\"Tab1\">1</xref>. These results were further analysed using analysis of variance to investigate the contributing impacts of individual factors considered and their interactions. The effectiveness of the RSM model is best explained through the satisfaction of model performance by measuring the determination coefficients (<italic>R</italic><sup>2</sup>), <italic>F</italic>-values, lack of fit, coefficient of variation, adjusted <italic>R</italic><sup>2</sup>, adequate precision, and <italic>p</italic> values (the value less than 0.05 indicates significant and vice versa).<table-wrap id=\"Tab1\"><label>Table 1</label><caption xml:lang=\"en\"><p>Design of experiments for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>Std</p></th><th align=\"left\"><p>Extraction time, <italic>X</italic><sub>1</sub> (s)</p></th><th align=\"left\"><p>Microwave power, <italic>X</italic><sub>2</sub> (W)</p></th><th align=\"left\"><p>Solvent/sample, <italic>X</italic><sub>3</sub> (mL/g)</p></th><th align=\"left\"><p>Ethanol concentration, <italic>X</italic><sub>4</sub> (%)</p></th><th align=\"left\"><p>TPC, <italic>Y</italic><sub>1</sub> (mg GAE/10 g d.b.)</p></th><th align=\"left\"><p>DPPH, <italic>Y</italic><sub>2</sub> (µg/mL)</p></th><th align=\"left\"><p>ABTS, <italic>Y</italic><sub>3</sub> (µg/mL)</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.69 ± 4.07</p></td><td align=\"char\" char=\"±\"><p>113.79 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>74.13 ± 1.08</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.00 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>109.47 ± 2.12</p></td><td align=\"char\" char=\"±\"><p>70.24 ± 1.05</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.42 ± 4.29</p></td><td align=\"char\" char=\"±\"><p>125.98 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>86.68 ± 2.43</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.72 ± 3.00</p></td><td align=\"char\" char=\"±\"><p>108.50 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>69.22 ± 1.54</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>327.50 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>120.00 ± 1.06</p></td><td align=\"char\" char=\"±\"><p>80.37 ± 1.58</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>326.33 ± 1.09</p></td><td align=\"char\" char=\"±\"><p>119.60 ± 2.14</p></td><td align=\"char\" char=\"±\"><p>80.47 ± 1.99</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>329.92 ± 4.76</p></td><td align=\"char\" char=\"±\"><p>117.00 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>77.79 ± 2.19</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>325.14 ± 4.11</p></td><td align=\"char\" char=\"±\"><p>103.72 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>64.50 ± 1.56</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>342.17 ± 2.33</p></td><td align=\"char\" char=\"±\"><p>120.02 ± 2.28</p></td><td align=\"char\" char=\"±\"><p>80.83 ± 1.05</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>341.89 ± 2.36</p></td><td align=\"char\" char=\"±\"><p>99.92 ± 3.05</p></td><td align=\"char\" char=\"±\"><p>60.60 ± 2.11</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>336.78 ± 2.56</p></td><td align=\"char\" char=\"±\"><p>94.48 ± 1.16</p></td><td align=\"char\" char=\"±\"><p>55.25 ± 1.55</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>330.63 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>70.67 ± 1.65</p></td><td align=\"char\" char=\"±\"><p>31.36 ± 1.00</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 2.11</p></td><td align=\"char\" char=\"±\"><p>120.15 ± 2.04</p></td><td align=\"char\" char=\"±\"><p>81.00 ± 1.05</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>340.18 ± 2.23</p></td><td align=\"char\" char=\"±\"><p>107.99 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.66 ± 1.00</p></td></tr><tr><td align=\"left\"><p>15</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>332.82 ± 2.05</p></td><td align=\"char\" char=\"±\"><p>90.45 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>51.22 ± 1.02</p></td></tr><tr><td align=\"left\"><p>16</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>325.52 ± 2.99</p></td><td align=\"char\" char=\"±\"><p>70.29 ± 1.77</p></td><td align=\"char\" char=\"±\"><p>31.08 ± 1.03</p></td></tr><tr><td align=\"left\"><p>17</p></td><td align=\"left\"><p>60</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>343.47 ± 4.22</p></td><td align=\"char\" char=\"±\"><p>101.04 ± 1.22</p></td><td align=\"char\" char=\"±\"><p>61.81 ± 1.06</p></td></tr><tr><td align=\"left\"><p>18</p></td><td align=\"left\"><p>180</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.78 ± 3.08</p></td><td align=\"char\" char=\"±\"><p>85.28 ± 2.06</p></td><td align=\"char\" char=\"±\"><p>46.00 ± 1.08</p></td></tr><tr><td align=\"left\"><p>19</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>200</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>346.89 ± 2.17</p></td><td align=\"char\" char=\"±\"><p>102.02 ± 2.09</p></td><td align=\"char\" char=\"±\"><p>62.82 ± 1.00</p></td></tr><tr><td align=\"left\"><p>20</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>400</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.51 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>85.20 ± 1.18</p></td><td align=\"char\" char=\"±\"><p>45.96 ± 0.99</p></td></tr><tr><td align=\"left\"><p>21</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>8</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>342.70 ± 2.65</p></td><td align=\"char\" char=\"±\"><p>102.39 ± 2.35</p></td><td align=\"char\" char=\"±\"><p>63.10 ± 1.69</p></td></tr><tr><td align=\"left\"><p>22</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>16</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>339.54 ± 1.98</p></td><td align=\"char\" char=\"±\"><p>104.14 ± 1.23</p></td><td align=\"char\" char=\"±\"><p>64.93 ± 2.01</p></td></tr><tr><td align=\"left\"><p>23</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>30</p></td><td align=\"char\" char=\"±\"><p>333.32 ± 4.03</p></td><td align=\"char\" char=\"±\"><p>90.82 ± 2.55</p></td><td align=\"char\" char=\"±\"><p>51.59 ± 1.03</p></td></tr><tr><td align=\"left\"><p>24</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>50</p></td><td align=\"char\" char=\"±\"><p>344.55 ± 1.00</p></td><td align=\"char\" char=\"±\"><p>68.84 ± 1.34</p></td><td align=\"char\" char=\"±\"><p>29.58 ± 1.03</p></td></tr><tr><td align=\"left\"><p>25</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>344.00 ± 3.11</p></td><td align=\"char\" char=\"±\"><p>86.40 ± 2.07</p></td><td align=\"char\" char=\"±\"><p>47.30 ± 1.26</p></td></tr><tr><td align=\"left\"><p>26</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.74 ± 2.16</p></td><td align=\"char\" char=\"±\"><p>90.08 ± 2.02</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 1.00</p></td></tr><tr><td align=\"left\"><p>27</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.91 ± 2.78</p></td><td align=\"char\" char=\"±\"><p>90.26 ± 1.02</p></td><td align=\"char\" char=\"±\"><p>51.01 ± 1.00</p></td></tr><tr><td align=\"left\"><p>28</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.68 ± 3.27</p></td><td align=\"char\" char=\"±\"><p>90.01 ± 1.15</p></td><td align=\"char\" char=\"±\"><p>50.78 ± 1.22</p></td></tr><tr><td align=\"left\"><p>29</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.77 ± 1.97</p></td><td align=\"char\" char=\"±\"><p>90.09 ± 1.04</p></td><td align=\"char\" char=\"±\"><p>50.84 ± 2.14</p></td></tr><tr><td align=\"left\"><p>30</p></td><td align=\"left\"><p>120</p></td><td align=\"char\" char=\".\"><p>300</p></td><td align=\"char\" char=\".\"><p>12</p></td><td align=\"char\" char=\".\"><p>40</p></td><td align=\"char\" char=\"±\"><p>345.92 ± 2.00</p></td><td align=\"char\" char=\"±\"><p>90.23 ± 2.22</p></td><td align=\"char\" char=\"±\"><p>51.00 ± 1.14</p></td></tr></tbody></table></table-wrap></p><p id=\"Par38\">Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref> presents the results obtained for the analysis of variance in recovering phenolic compounds and antioxidants (DPPH and ABTS assays). It can be observed that the three models of the responses (TPC, DPPH and ABTS assays) are significant. Out of the four considered factors for the optimization study, microwave power and ethanol concentration are the most significant with a <italic>p</italic> value &lt; 0.0001, showing model significance at a 99.9% confidence level. Extraction time is also significant, whereas solvent/sample ratio was insignificant for the three responses. Moreover, the lack of fit for the models was far greater than 0.05; this shows that the models are significant. The <italic>R</italic><sup>2</sup> values of the TPC, DPPH and ABTS assays ranged between 0.98 and 0.99; these are indications that the predictive models can explain at least 98% of the result variations. The adjusted and predicted <italic>R</italic><sup>2</sup> values for the three predictive models were closer to 1, indicating significant correlations between the actual and predicted results (Alara et al. <xref ref-type=\"bibr\" rid=\"CR6\">2021a</xref>). The lower value of the coefficient of variation below 10% shows that the experimental results were reproducible and reliable. The obtained results indicate for CV values for TPC, and antioxidant assays (DPPH and ABTS) were 0.33, 2.04 and 3.44, respectively. It should be noted that the contributions of the quadratic models are notable. Thus, the predictive models for the phenolic compounds and antioxidant assays are well suitable for the prediction.<table-wrap id=\"Tab2\"><label>Table 2</label><caption xml:lang=\"en\"><p>Analysis of variance for the optimization of microwave-enhanced extraction of TPC and antioxidants from <italic>C. olitorius</italic> leaves</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\" colspan=\"6\"><p>TPC</p></th><th align=\"left\" colspan=\"6\"><p>DPPH</p></th><th align=\"left\" colspan=\"6\"><p>ABTS</p></th></tr><tr><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th><th align=\"left\"><p>Source</p></th><th align=\"left\"><p>Sum of squares</p></th><th align=\"left\"><p>DF</p></th><th align=\"left\"><p>Mean square</p></th><th align=\"left\"><p><italic>F</italic>-value</p></th><th align=\"left\"><p><italic>p</italic> value Prob &gt; <italic>F</italic></p></th></tr></thead><tbody><tr><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>1856.45</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>132.60</p></td><td align=\"char\" char=\".\"><p>105.49</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6946.43</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>496.17</p></td><td align=\"char\" char=\".\"><p>121.69</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p>Model</p></td><td align=\"char\" char=\".\"><p>6922.90</p></td><td align=\"left\"><p>14</p></td><td align=\"char\" char=\".\"><p>494.49</p></td><td align=\"char\" char=\".\"><p>117.32</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>27.21</p></td><td align=\"char\" char=\".\"><p>21.64</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>902.70</p></td><td align=\"char\" char=\".\"><p>221.39</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub>: Extraction time</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>895.35</p></td><td align=\"char\" char=\".\"><p>212.43</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>111.20</p></td><td align=\"char\" char=\".\"><p>88.47</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1195.12</p></td><td align=\"char\" char=\".\"><p>293.10</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub>: Microwave power</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1185.20</p></td><td align=\"char\" char=\".\"><p>281.19</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>5.01</p></td><td align=\"char\" char=\".\"><p>3.99</p></td><td align=\"left\"><p>0.0643</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>3.66</p></td><td align=\"char\" char=\".\"><p>0.90</p></td><td align=\"left\"><p>0.3582</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub>: Solvent/sample ratio</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>4.12</p></td><td align=\"char\" char=\".\"><p>0.98</p></td><td align=\"left\"><p>0.3386</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>450.50</p></td><td align=\"char\" char=\".\"><p>358.38</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1532.18</p></td><td align=\"char\" char=\".\"><p>375.77</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub>: Ethanol concentration</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1520.03</p></td><td align=\"char\" char=\".\"><p>360.63</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>9.64</p></td><td align=\"char\" char=\".\"><p>7.67</p></td><td align=\"left\"><p>0.0143</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>89.07</p></td><td align=\"char\" char=\".\"><p>21.84</p></td><td align=\"left\"><p>0.0003</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>2</sub></p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>92.26</p></td><td align=\"char\" char=\".\"><p>21.89</p></td><td align=\"left\"><p>0.0003</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>11.90</p></td><td align=\"char\" char=\".\"><p>9.47</p></td><td align=\"left\"><p>0.0077</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.28</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"left\"><p>0.0276</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>24.50</p></td><td align=\"char\" char=\".\"><p>5.81</p></td><td align=\"left\"><p>0.0292</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>13.62</p></td><td align=\"char\" char=\".\"><p>10.83</p></td><td align=\"left\"><p>0.0049</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>103.79</p></td><td align=\"char\" char=\".\"><p>25.45</p></td><td align=\"left\"><p>0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"left\"/><td align=\"char\" char=\".\"><p>110.57</p></td><td align=\"char\" char=\".\"><p>26.23</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.20</p></td><td align=\"char\" char=\".\"><p>4.93</p></td><td align=\"left\"><p>0.0422</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>114.01</p></td><td align=\"char\" char=\".\"><p>27.96</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>3</sub></p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>113.53</p></td><td align=\"char\" char=\".\"><p>26.94</p></td><td align=\"left\"><p>0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>130.64</p></td><td align=\"char\" char=\".\"><p>103.93</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>819.82</p></td><td align=\"char\" char=\".\"><p>201.06</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>828.86</p></td><td align=\"char\" char=\".\"><p>196.65</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>6.84</p></td><td align=\"char\" char=\".\"><p>5.44</p></td><td align=\"left\"><p>0.0340</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.092</p></td><td align=\"char\" char=\".\"><p>0.022</p></td><td align=\"left\"><p>0.8829</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub></p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>0.070</p></td><td align=\"char\" char=\".\"><p>0.017</p></td><td align=\"left\"><p>0.8990</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>12.91</p></td><td align=\"char\" char=\".\"><p>10.27</p></td><td align=\"left\"><p>0.0059</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.94</p></td><td align=\"char\" char=\".\"><p>15.19</p></td><td align=\"left\"><p>0.0014</p></td><td align=\"left\"><p><italic>X</italic><sub>1</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>61.19</p></td><td align=\"char\" char=\".\"><p>14.52</p></td><td align=\"left\"><p>0.0017</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>1.12</p></td><td align=\"char\" char=\".\"><p>0.89</p></td><td align=\"left\"><p>0.3605</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>73.87</p></td><td align=\"char\" char=\".\"><p>18.12</p></td><td align=\"left\"><p>0.0007</p></td><td align=\"left\"><p><italic>X</italic><sub>2</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>74.01</p></td><td align=\"char\" char=\".\"><p>17.56</p></td><td align=\"left\"><p>0.0008</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>46.51</p></td><td align=\"char\" char=\".\"><p>37.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>582.53</p></td><td align=\"char\" char=\".\"><p>142.87</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>3</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>580.61</p></td><td align=\"char\" char=\".\"><p>137.75</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>106.86</p></td><td align=\"char\" char=\".\"><p>85.01</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>184.58</p></td><td align=\"char\" char=\".\"><p>45.27</p></td><td align=\"left\"><p> &lt; 0.0001</p></td><td align=\"left\"><p><italic>X</italic><sub>4</sub><sup>2</sup></p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"left\"><p>1</p></td><td align=\"char\" char=\".\"><p>185.44</p></td><td align=\"char\" char=\".\"><p>44.00</p></td><td align=\"left\"><p> &lt; 0.0001</p></td></tr><tr><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>18.86</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>1.26</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>61.16</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.08</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Residual</p></td><td align=\"char\" char=\".\"><p>63.22</p></td><td align=\"left\"><p>15</p></td><td align=\"char\" char=\".\"><p>4.21</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>16.10</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>1.61</p></td><td align=\"char\" char=\".\"><p>2.92</p></td><td align=\"left\"><p>0.1243</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>49.50</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>4.95</p></td><td align=\"char\" char=\".\"><p>2.12</p></td><td align=\"left\"><p>0.2103</p></td><td align=\"left\"><p>Lack of fit</p></td><td align=\"char\" char=\".\"><p>52.42</p></td><td align=\"left\"><p>10</p></td><td align=\"char\" char=\".\"><p>5.24</p></td><td align=\"char\" char=\".\"><p>2.42</p></td><td align=\"left\"><p>0.1701</p></td></tr><tr><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>2.76</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>0.55</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>11.66</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.33</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Pure error</p></td><td align=\"char\" char=\".\"><p>10.81</p></td><td align=\"left\"><p>5</p></td><td align=\"char\" char=\".\"><p>2.16</p></td><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>1875.30</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>7007.59</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Correlation total</p></td><td align=\"char\" char=\".\"><p>6986.12</p></td><td align=\"left\"><p>29</p></td><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>0.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>2.04</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Coefficient of variation (%)</p></td><td align=\"char\" char=\".\"><p>3.44</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>118.47</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>368.81</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>PRESS</p></td><td align=\"char\" char=\".\"><p>389.33</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9899</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9913</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>R-square</p></td><td align=\"char\" char=\".\"><p>0.9910</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9806</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9831</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adjusted R-square</p></td><td align=\"char\" char=\".\"><p>0.9825</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9368</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9474</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Predicted R-square</p></td><td align=\"char\" char=\".\"><p>0.9443</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr><tr><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>28.799</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>39.281</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/><td align=\"left\"><p>Adequate precision</p></td><td align=\"char\" char=\".\"><p>38.612</p></td><td align=\"left\"/><td align=\"char\" char=\".\"/><td align=\"char\" char=\".\"/><td align=\"left\"/></tr></tbody></table><table-wrap-foot><p><italic>DF</italic> degree of freedom</p></table-wrap-foot></table-wrap></p></sec><sec id=\"Sec21\"><title>Effects of microwave-enhanced extraction factors on TPC and antioxidant assays (DPPH and ABTS) in <italic>C. olitorius</italic> leaves</title><p id=\"Par39\">The influences of microwave-enhanced extraction factors on the phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. The phenolic compounds in the extract of <italic>C. olitorius</italic> leaves varied from 325.52 ± 2.99 to 346.89 ± 2.17 mg GAE/10 g d.b. (Table <xref ref-type=\"table\" rid=\"Tab2\">2</xref>). The least TPC was obtained at extraction time of 180 s, microwave power of 400 W, solvent/sample ratio of 16 mL/g, and ethanol concentration of 50%, whereas the highest TPC was obtained at extraction time of 120 s, microwave power of 200 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 40%. The antioxidant assays (DPPH and ABTS) from the leaf extract of <italic>C. olitorius</italic> ranged from 68.84 ± 1.34 to 125.98 ± 1.16 µg/mL and 29.58 ± 1.03 to 86.68 ± 2.43 µg/mL, respectively. The least (extraction time of 60 s, microwave power of 400 W, solvent/sample ratio of 8 mL/g, and ethanol concentration of 30%) and highest (extraction time of 120 s, microwave power of 300 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%) yields of antioxidants from <italic>C. olitorius</italic> leaves were obtained. Analysis of variance analysis (ANOVA) results indicate that extraction time, microwave power and ethanol concentration significantly influenced the recoveries of phenolic compounds from <italic>C. olitorius</italic> leaves; however, the solvent/sample ratio was insignificant. The quadratic effects of all the factors except microwave power are significant (<italic>p</italic> value &lt; 0.05). Besides, the interactions between all the considered factors significantly impact the microwave-enhanced extraction of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves except for the interactions between solvent/sample ratio and ethanol concentration (<italic>X</italic><sub>3</sub><italic>X</italic><sub>4</sub>) on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves which were insignificant (<italic>p</italic> value &gt; 0.05). The quadratic model for the coded values of TPC, DPPH and ABTS assays is estimated from Eqs. (<xref ref-type=\"disp-formula\" rid=\"Equ2\">2</xref>–<xref ref-type=\"disp-formula\" rid=\"Equ4\">4</xref>).<disp-formula id=\"Equ2\"><label>2</label><alternatives><mml:math display=\"block\" id=\"Equ2_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>1</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>345.43</mml:mn><mml:mo>-</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.49</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.53</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>5</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.78</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.92</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>0.62</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.86</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>0.65</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.23</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>0.66</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>4.24</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>6.42</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ2_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{1} &amp; = &amp; 345.43 - 1.23X_{1} - 2.49X_{2} - 0.53X_{3} + 5X_{4} - 0.78X_{1} X_{2} - 0.86X_{1} X_{3} - 0.92X_{1} X_{4} \\\\ \\quad - 0.62X_{2} X_{3} - 2.86X_{2} X_{4} - 0.65X_{3} X_{4} - 2.23X_{1}^{2} - 0.66X_{2}^{2} - 4.24X_{3}^{2} - 6.42X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ2.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ3\"><label>3</label><alternatives><mml:math display=\"block\" id=\"Equ3_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>2</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>88.89</mml:mn><mml:mo>-</mml:mo><mml:mn>7.08</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.15</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.45</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.36</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.23</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.55</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.67</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.16</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.076</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.89</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.99</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.44</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ3_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{2} &amp; = &amp; 88.89 - 7.08X_{1} - 8.15X_{2} + 0.45X_{3} - 9.23X_{4} - 2.36X_{1} X_{2} + 1.23X_{1} X_{3} - 2.55X_{1} X_{4} \\\\ \\quad - 2.67X_{2} X_{3} - 7.16X_{2} X_{4} + 0.076X_{3} X_{4} + 4.89X_{1}^{2} + 5.34X_{2}^{2} + 14.99X_{3}^{2} - 8.44X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ3.gif\"/></alternatives></disp-formula><disp-formula id=\"Equ4\"><label>4</label><alternatives><mml:math display=\"block\" id=\"Equ4_Math\"><mml:mrow><mml:mtable><mml:mtr><mml:mtd columnalign=\"right\"><mml:msub><mml:mi>Y</mml:mi><mml:mn>3</mml:mn></mml:msub></mml:mtd><mml:mtd columnalign=\"left\"><mml:mo>=</mml:mo></mml:mtd><mml:mtd columnalign=\"right\"><mml:mrow><mml:mn>49.67</mml:mn><mml:mo>-</mml:mo><mml:mn>7.05</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>8.11</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.48</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>9.19</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.40</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>1.24</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>2.63</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>1</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow><mml:mrow/><mml:mspace width=\"1em\"/><mml:mo>-</mml:mo><mml:mn>2.66</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:mo>-</mml:mo><mml:mn>7.20</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>2</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>0.066</mml:mn><mml:msub><mml:mi>X</mml:mi><mml:mn>3</mml:mn></mml:msub><mml:msub><mml:mi>X</mml:mi><mml:mn>4</mml:mn></mml:msub><mml:mo>+</mml:mo><mml:mn>4.86</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>1</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>5.34</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>2</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>+</mml:mo><mml:mn>14.97</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>3</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup><mml:mo>-</mml:mo><mml:mn>8.46</mml:mn><mml:msubsup><mml:mi>X</mml:mi><mml:mrow><mml:mn>4</mml:mn></mml:mrow><mml:mn>2</mml:mn></mml:msubsup></mml:mrow></mml:mtd></mml:mtr><mml:mtr><mml:mtd columnalign=\"right\"><mml:mrow/></mml:mtd></mml:mtr></mml:mtable></mml:mrow></mml:math><tex-math id=\"Equ4_TeX\">\\documentclass[12pt]{minimal}\n",
       "\t\t\t\t\\usepackage{amsmath}\n",
       "\t\t\t\t\\usepackage{wasysym}\n",
       "\t\t\t\t\\usepackage{amsfonts}\n",
       "\t\t\t\t\\usepackage{amssymb}\n",
       "\t\t\t\t\\usepackage{amsbsy}\n",
       "\t\t\t\t\\usepackage{mathrsfs}\n",
       "\t\t\t\t\\usepackage{upgreek}\n",
       "\t\t\t\t\\setlength{\\oddsidemargin}{-69pt}\n",
       "\t\t\t\t\\begin{document}$$\\begin{aligned} Y_{3} &amp; = &amp; 49.67 - 7.05X_{1} - 8.11X_{2} + 0.48X_{3} - 9.19X_{4} - 2.40X_{1} X_{2} + 1.24X_{1} X_{3} - 2.63X_{1} X_{4} \\\\ \\quad - 2.66X_{2} X_{3} - 7.20X_{2} X_{4} + 0.066X_{3} X_{4} + 4.86X_{1}^{2} + 5.34X_{2}^{2} + 14.97X_{3}^{2} - 8.46X_{4}^{2} \\\\ \\end{aligned}$$\\end{document}</tex-math><graphic mime-subtype=\"GIF\" specific-use=\"web\" xlink:href=\"11696_2023_2771_Article_Equ4.gif\"/></alternatives></disp-formula>where <italic>X</italic><sub>1</sub> represents extraction time (s), <italic>X</italic><sub>2</sub> indicates microwave power (W), <italic>X</italic><sub>3</sub> represents solvent/sample ratio (mL/g), <italic>X</italic><sub>4</sub> is the ethanol concentration, <italic>Y</italic><sub>1</sub>, <italic>Y</italic><sub>2</sub> and <italic>Y</italic><sub>3</sub> are the responses TPC, DPPH and ABTS assays, accordingly.</p><p id=\"Par40\">The three-dimensional plot is the representation of regression equations to show the interaction between independent and dependent factors. This plot is presented in different shapes; this illustrates whether the factor or its mutual interaction is significant or not (Halee et al. <xref ref-type=\"bibr\" rid=\"CR15\">2020</xref>). The contributing impacts of individual factors and their mutual interactions on the phenolic compounds and antioxidants are presented using three-dimensional graphical representations in Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a–f, <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a–f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a–f. It should be noted that antioxidant increases with reduced IC<sub>50</sub> values (Mahmud et al. <xref ref-type=\"bibr\" rid=\"CR22\">2019</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the phenolic compounds from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO2\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig2_HTML.png\"/></fig><fig id=\"Fig3\"><label>Fig. 3</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the DPPH assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO3\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig3_HTML.png\"/></fig><fig id=\"Fig4\"><label>Fig. 4</label><caption xml:lang=\"en\"><p>Contributing impacts of individual factor and their mutual interactions on the ABTS assay in recovering antioxidants from <italic>C. olitorius</italic> leaves</p></caption><graphic id=\"MO4\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig4_HTML.png\"/></fig></p><p id=\"Par41\">The contributing effect of interactions between microwave power and extraction time on TPC at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is shown in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>a. A gradual increase in extraction time and a slight decline in microwave power slowly increase the yield of TPC. Nonetheless, beyond 131.37 s and 305.08 W, TPC started to decline. For Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>b, increasing both solvent/sample and extraction time at a fixed microwave power of 300 W and ethanol concentration of 40% gradually increased the recovery of TPC from <italic>C. olitorius</italic> leaves. After the certain limits of microwave power of 305.08 W and extraction time of 131.37 s, the phenolic compounds declined. In Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c, TPC from <italic>C. olitorius</italic> leaves increases with increasing ethanol concentration and reduces the extraction time. After these factors reached 45.07% and 69.85 s, respectively, TPC gradually decreases. The interaction between solvent/sample ratio and microwave power is presented in Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d; increasing solvent/sample ratio, and reducing microwave power increased the recovery of phenolic compounds. At solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recovery of TPC decreased. For the interaction between ethanol concentration and extraction time (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>c), increasing ethanol concentration and reducing microwave power favour increasing phenolic compounds (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>e). At the ethanol concentration and microwave power of 45.07% and 257.62 W, optimal yields of TPC and antioxidants were achieved; beyond these values, the recoveries declined. Finally, the effects of interactions between ethanol concentration and solvent/sample ratio on the recoveries of TPC from <italic>C. olitorius</italic> leaves (Fig. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>f). Reducing the solvent/sample ratio and increasing the ethanol concentration favour the recovery of optimal TPC from the considered sample. Hence, the optimal predicted TPC yield of 343.327 mg GAE/10 g d.b. at the microwave-enhanced extraction conditions including extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p><p id=\"Par42\">Moreover, the influence of interactions between microwave power and extraction time on antioxidants (DPPH and ABTS assays) in the leaf extract of <italic>C. olitorius</italic> at a fixed solvent/sample ratio of 12 mL/g and ethanol concentration of 40% is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>a and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>a. Increasing the extraction time and a slight decrease in microwave power gradually increases the antioxidant yields. The antioxidants in leaf extract declined as the microwave power and extraction time reached 131.37 s and 305.08 W, respectively. Besides, the interaction between solvent/sample ratio and extraction time is presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>b and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>b showed that increasing solvent/sample ratio and declining extraction time favoured antioxidant recoveries from <italic>C. olitorius</italic> leaf at a fixed microwave power of 300 W and ethanol concentration of 40%. Beyond microwave power of 305.08 W and extraction time of 131.37 s, the antioxidants declined. For Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, the antioxidants increase with increasing ethanol concentration and extraction time; beyond 45.07% and 69.85 s, respectively, gradual declination of antioxidants was observed. Increasing solvent/sample ratio and reducing microwave power increased the recoveries of phenolic compounds (Figs. <xref ref-type=\"fig\" rid=\"Fig2\">2</xref>d and <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>d). On reaching solvent/sample ratio and microwave power of 11.48 mL/g and 257.62 W, respectively, the recoveries of antioxidants declined. Similarly, in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>c and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>c, increasing ethanol concentration and reducing microwave power increase the recoveries of antioxidants from leaf extract of <italic>C. olitorius</italic> (Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>e and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>e). Beyond the ethanol concentration and microwave power of 45.07% and 257.62 W, respectively, the recoveries of antioxidants declined. In conclusion, the interactions between ethanol concentration and solvent/sample ratio on the recoveries of antioxidants from <italic>C. olitorius</italic> leaves as presented in Figs. <xref ref-type=\"fig\" rid=\"Fig3\">3</xref>f and <xref ref-type=\"fig\" rid=\"Fig4\">4</xref>f reflected that increasing solvent/sample ratio and reducing ethanol concentration yielded increasing antioxidants. Thus, the optimal predicted antioxidant yields (both DPPH and ABTS assays) were 68.8396 and 29.6220 µg/mL, respectively, at the microwave-enhanced extraction conditions of extraction time, microwave power, solvent/sample ratio, and ethanol concentration at 131.37 s, 305.08 W, 11.67 mL/g, and 50%, respectively.</p></sec><sec id=\"Sec22\"><title>Validations of the generated models</title><p id=\"Par43\">Based on the predicted values from the Design Expert® software, the optimal conditions to achieve phenolic compounds (TPC of 343.327 mg GAE/10 g d.b.) and antioxidants (IC<sub>50</sub> values of 68.8396 and 29.6220 µg/mL for DPPH and ABTS assays, respectively) from extracts from <italic>C. olitorius</italic> leaves were found at extraction time of 131.37 s, microwave power of 305.08 W, solvent/sample ratio of 11.67 mL/g, and ethanol concentration of 50% with the desirability of 0.942. The validation experiments were carried out using approximated values of the predicted values such as extraction time of 131 s, microwave power of 305 W, solvent/sample ratio of 12 mL/g, and ethanol concentration of 50%. Three replicate experiments were carried out using these conditions, and the mean was determined to affirm the generated outcomes from the software. The experimental trials amounted to TPC of 343.098 ± 3.05 mg GAE/10 g d.b., IC<sub>50</sub> values of 68.89 ± 1.08 and 29.76 ± 1.00 µg/mL for DPPH and ABTS assays, respectively, at 131 s, 305 W, 12 mL/g, and 50% of extraction time, microwave power, solvent/sample ratio, and ethanol concentration, respectively. Using Chi-squared analysis, the results obtained indicated no significant difference between the experimental and predicted values from the software.</p><p id=\"Par44\">As earlier reported, some previous studies had evaluated the contents of total phenols and antioxidants in the leaf extract of <italic>C. olitorius</italic>; however, no study was reported on recovering phenolic compounds and antioxidants from this plant through microwave-enhanced technique. A recent finding reported the recoveries of total phenols (0.10–9.81 µ GA/g), DPPH inhibition (1.4–70.3%) and ABTS inhibition (83–93%) (Obeng et al. <xref ref-type=\"bibr\" rid=\"CR25\">2020</xref>). In another recent study, a total antioxidant capacity of 165.66 ± 1.30 mg ascorbic acid/g dry extract was reported in the leaf extract of <italic>C. olitorius</italic> (Ali et al. <xref ref-type=\"bibr\" rid=\"CR8\">2021</xref>). The TPC of 2.5–32.0 Mg TE/g was reported in a recent study; it was established that phenolic compounds and antioxidants from this plant are unaffected by cultivation treatments. Moreover, it had been reported that the mode of extraction can alter the phenolic compounds in the <italic>C. olitorius</italic> leaves (Guzzetti et al. <xref ref-type=\"bibr\" rid=\"CR14\">2021</xref>). The antioxidant from <italic>C. olitorius</italic> leaves was reported to be 91.65 ± 0.15% using DPPH inhibition with the maceration extraction technique (Racha et al. <xref ref-type=\"bibr\" rid=\"CR27\">2018</xref>). The results from the current study established the importance of using the microwave-enhanced extraction method to recover optimal yields of phenolic compounds and antioxidants from <italic>C. olitorius</italic> leaves. Other than this, this plant can further be used in combating food insecurity because it is rich in phenolic compounds.</p></sec></sec><sec id=\"Sec23\"><title>LC-QToF-MS analysis of the optimized extract from <italic>C. olitorius</italic> leaves</title><p id=\"Par45\">The use of LC-QToF-MS analysis to characterize phenolic compounds has been a current trend in natural product chemistry; this analysis has proven to be efficient in tentatively identifying phenolic compounds in diverse plant samples (Alara et al. <xref ref-type=\"bibr\" rid=\"CR5\">2018c</xref>, <xref ref-type=\"bibr\" rid=\"CR4\">2018b</xref>; Tan et al. <xref ref-type=\"bibr\" rid=\"CR29\">2020</xref>; Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). Unifi software processed the data. Thus, the phenolic profile of the leaf extract of <italic>C. olitorius</italic> at optimized microwave-enhanced extraction conditions was characterized using the LC-QToF-MS system (Fig. <xref ref-type=\"fig\" rid=\"Fig5\">5</xref>). The characterization and identification of phytochemicals in the leaf extract of <italic>C. olitorius</italic> were done by comparing retention time (RT), mass spectrometric (MS), and mass error between mass observed (which must be less than 10 ppm for viable identified compound) (Tang et al. <xref ref-type=\"bibr\" rid=\"CR30\">2020</xref>). A total of fourteen phenolic compounds were identified in the extract of <italic>C. olitorius</italic> leaves as presented in Table <xref ref-type=\"table\" rid=\"Tab3\">3</xref>. The identified phytochemicals were 1-galloyl-glucose, 1,3,5-O-tricaffeoylquinic acid, procyanidin C-1, 4,4′,5,6-tetrahydroxystilbene, 3,4,5-O-tricaffeoylquinic acid, 5-desgalloylstachyurin, sanguiin H-4, corilagin_1, 1-O-galloylpedunculagin, laevigatin A, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose as presented in the Supplementary materials (Figs. S1–S14).<fig id=\"Fig5\"><label>Fig. 5</label><caption xml:lang=\"en\"><p>Negative ionization plots for the identified chemical compounds in the extract from <italic>C. olitorius</italic> leaf</p></caption><graphic id=\"MO5\" mime-subtype=\"PNG\" specific-use=\"web\" xlink:href=\"MediaObjects/11696_2023_2771_Fig5_HTML.png\"/></fig><table-wrap id=\"Tab3\"><label>Table 3</label><caption xml:lang=\"en\"><p>The tentative assignments of chemical compounds in the leaf extract of <italic>C. olitorius</italic></p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><th align=\"left\"><p>S/N</p></th><th align=\"left\"><p>Chemical compound</p></th><th align=\"left\"><p>RT (min)</p></th><th align=\"left\"><p>[M-H]- (m/z)</p></th><th align=\"left\"><p>Mass error (ppm)</p></th><th align=\"left\"><p>Molecular formula</p></th><th align=\"left\"><p>Response</p></th><th align=\"left\"><p>Adducts</p></th></tr></thead><tbody><tr><td align=\"left\"><p>1</p></td><td align=\"left\"><p>1-Galloyl-glucose</p></td><td align=\"char\" char=\".\"><p>0.58</p></td><td align=\"char\" char=\".\"><p>331.0673</p></td><td align=\"char\" char=\".\"><p>0.8</p></td><td align=\"left\"><p>C<sub>13</sub>H<sub>16</sub>O<sub>10</sub></p></td><td align=\"left\"><p>6533</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>2</p></td><td align=\"left\"><p>1,3,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>0.59</p></td><td align=\"char\" char=\".\"><p>677.1499</p></td><td align=\"char\" char=\".\"><p>−1.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>3235</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>3</p></td><td align=\"left\"><p>Procyanidin C-1</p></td><td align=\"char\" char=\".\"><p>0.61</p></td><td align=\"char\" char=\".\"><p>865.1981</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>45</sub>H<sub>38</sub>O<sub>18</sub></p></td><td align=\"left\"><p>105,272</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>4</p></td><td align=\"left\"><p>4,4′,5,6-Tetrahydroxystilbene</p></td><td align=\"char\" char=\".\"><p>0.75</p></td><td align=\"char\" char=\".\"><p>289.0725</p></td><td align=\"char\" char=\".\"><p>2.5</p></td><td align=\"left\"><p>C<sub>14</sub>H<sub>12</sub>O<sub>4</sub></p></td><td align=\"left\"><p>6478</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>5</p></td><td align=\"left\"><p>3,4,5-O-Tricaffeoylquinic acid</p></td><td align=\"char\" char=\".\"><p>3.11</p></td><td align=\"char\" char=\".\"><p>677.1517</p></td><td align=\"char\" char=\".\"><p>0.7</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>30</sub>O<sub>15</sub></p></td><td align=\"left\"><p>7900</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>6</p></td><td align=\"left\"><p>5-Desgalloylstachyurin</p></td><td align=\"char\" char=\".\"><p>4.00</p></td><td align=\"char\" char=\".\"><p>783.0674</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>3123</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>7</p></td><td align=\"left\"><p>Sanguiin H-4</p></td><td align=\"char\" char=\".\"><p>5.31</p></td><td align=\"char\" char=\".\"><p>633.0731</p></td><td align=\"char\" char=\".\"><p>−0.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2263</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>8</p></td><td align=\"left\"><p>Corilagin_1</p></td><td align=\"char\" char=\".\"><p>5.85</p></td><td align=\"char\" char=\".\"><p>679.0783</p></td><td align=\"char\" char=\".\"><p>−0.8</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>22</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2547</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>9</p></td><td align=\"left\"><p>1-O-Galloylpedunculagin</p></td><td align=\"char\" char=\".\"><p>5.95</p></td><td align=\"char\" char=\".\"><p>981.0849</p></td><td align=\"char\" char=\".\"><p>−0.1</p></td><td align=\"left\"><p>C<sub>41</sub>H<sub>28</sub>O<sub>26</sub></p></td><td align=\"left\"><p>3356</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>10</p></td><td align=\"left\"><p>Laevigatin A</p></td><td align=\"char\" char=\".\"><p>6.47</p></td><td align=\"char\" char=\".\"><p>847.0839</p></td><td align=\"char\" char=\".\"><p>−0.9</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>26</sub>O<sub>23</sub></p></td><td align=\"left\"><p>2292</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>11</p></td><td align=\"left\"><p>Pedunculagin</p></td><td align=\"char\" char=\".\"><p>6.94</p></td><td align=\"char\" char=\".\"><p>829.0728</p></td><td align=\"char\" char=\".\"><p>−1.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>24</sub>O<sub>22</sub></p></td><td align=\"left\"><p>6165</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>12</p></td><td align=\"left\"><p>2,4,6-Tri-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.40</p></td><td align=\"char\" char=\".\"><p>681.0937</p></td><td align=\"char\" char=\".\"><p>−1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>3128</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>13</p></td><td align=\"left\"><p>1,3,6-Trigalloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>7.91</p></td><td align=\"char\" char=\".\"><p>681.0954</p></td><td align=\"char\" char=\".\"><p>1.4</p></td><td align=\"left\"><p>C<sub>27</sub>H<sub>24</sub>O<sub>18</sub></p></td><td align=\"left\"><p>2265</p></td><td align=\"left\"><p>–H</p></td></tr><tr><td align=\"left\"><p>14</p></td><td align=\"left\"><p>1,2,3,6-Tetra-O-galloyl-<italic>β</italic>-D-glucose</p></td><td align=\"char\" char=\".\"><p>8.14</p></td><td align=\"char\" char=\".\"><p>833.1060</p></td><td align=\"char\" char=\".\"><p>0.6</p></td><td align=\"left\"><p>C<sub>34</sub>H<sub>28</sub>O<sub>22</sub></p></td><td align=\"left\"><p>5153</p></td><td align=\"left\"><p>–H</p></td></tr></tbody></table></table-wrap></p><p id=\"Par46\">In simplicity, identified 1-galloyl-glucose, 1-O-galloylpedunculagin, pedunculagin, 2,4,6-tri-O-galloyl-<italic>β</italic>-D-glucose, 1,3,6-trigalloyl-<italic>β</italic>-D-glucose, and 1,2,3,6-tetra-O-galloyl-<italic>β</italic>-D-glucose belong to tannins (naturally occurring polyphenols). Another identified tannin was 5-desgalloylstachyurin. These compounds are rich in antioxidants and can alleviate different forms of cancer and free radicals (Zhang et al. <xref ref-type=\"bibr\" rid=\"CR37\">2009</xref>). Other identified compounds were 1,3,5-O-tricaffeoylquinic acid and 3,4,5-O-tricaffeoylquinic acid; these compounds were reported as polyphenols that are rich in antioxidants (Wang et al. <xref ref-type=\"bibr\" rid=\"CR33\">2017</xref>). Procyanidin C-1 belongs to polyflavonoids and bioflavonoids; this compound has been reported natural occurring antioxidant from a plant source (Xu et al. <xref ref-type=\"bibr\" rid=\"CR34\">2021</xref>). The identified stilbene monomer in the leaf extract of <italic>C. olitorius</italic> was 4,4′,5,6-tetrahydroxystilbene. This phytochemical has been reported to possess diverse pharmacological features which antioxidants (Zhang and Chen <xref ref-type=\"bibr\" rid=\"CR36\">2018</xref>). Sanguiin H-4 had been reported to possess antioxidant properties (Lachowicz et al. <xref ref-type=\"bibr\" rid=\"CR19\">2020</xref>). Corilagin_1 and laevigatin A are compounds that are rich in antioxidants as well. Therefore, extract from <italic>C. olitorius</italic> leaf has been established as a source of phenolic compounds and natural antioxidants.</p></sec></sec><sec id=\"Sec24\" sec-type=\"conclusions\"><title>Conclusion</title><p id=\"Par47\">Seeking alternative sources of antioxidants, especially plant-based, will continue to be studied. Given this, this study investigated <italic>C. olitorius</italic> leaves as a source of phenolic compounds and antioxidants by considering a technique that can recover yields (microwave-enhanced extraction) using ethanol/water as extracting solvent. The optimized process established optimal recoveries of phenolic compounds and antioxidants which were higher compared to previously reported results on <italic>C. olitorius</italic> leaves. In addition to this, the phenolic compounds that were comprehensively identified in the extract (fourteen in total) ascertain <italic>C. olitorius</italic> leaves as great sources of antioxidants. This outcome is an indication that <italic>C. olitorius</italic> leaves can continue to be encouraged in daily consumption, a way of combating food insecurity in the world. Thus, future study can focus on isolation and investigation of the tentatively identified phenolic compounds. These can further explain the pharmacological importance of this plant sample.</p></sec></body><back><ack><title>Acknowledgements</title><p>The authors thankfully acknowledge Eastern Unity Technology, Malaysia for the financial support through the grant (UIC190806).</p></ack><sec sec-type=\"ethics-statement\"><title>Declarations</title><sec id=\"FPar1\" sec-type=\"COI-statement\"><title>Conflict of interest</title><p id=\"Par48\">We declare none.</p></sec></sec><ref-list id=\"Bib1\"><title>References</title><ref-list><ref id=\"CR1\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Abir</surname><given-names>RR</given-names></name><name><surname>Marjia</surname><given-names>M</given-names></name><name><surname>Rakhi</surname><given-names>NN</given-names></name><name><surname>Saha</surname><given-names>O</given-names></name><name><surname>Hossain</surname><given-names>MA</given-names></name><name><surname>Rahaman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">In vitro comparative analysis of antibacterial activity of different fractions of <italic>Corchorus capsularis</italic> and <italic>Corchorus olitorius</italic> leaves extracts</article-title><source>Bangladesh J Microbiol</source><year>2019</year><volume>36</volume><fpage>69</fpage><lpage>73</lpage><pub-id pub-id-type=\"doi\">10.3329/bjm.v36i2.45530</pub-id></mixed-citation></ref><ref id=\"CR2\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolics from Hibiscus sabdariffa calyces: Kinetic modelling and process intensification</article-title><source>Ind Crops Prod</source><year>2019</year><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2019.05.053</pub-id></mixed-citation></ref><ref id=\"CR3\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name></person-group><article-title xml:lang=\"en\"><italic>Vernonia cinerea</italic> leaves as the source of phenolic compounds, antioxidants, and anti-diabetic activity using microwave-assisted extraction technique</article-title><source>Ind Crop Prod</source><year>2018</year><volume>122</volume><fpage>533</fpage><lpage>544</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXht1Wru7nJ</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.06.034</pub-id></mixed-citation></ref><ref id=\"CR4\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Azhari</surname><given-names>NH</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Metabolic profiling of flavonoids, saponins, alkaloids, and terpenoids in the extract from <italic>Vernonia cinerea</italic> leaf using LC-Q-TOF-MS</article-title><source>J Liq Chromatogr Relat Technol</source><year>2018</year><pub-id pub-id-type=\"doi\">10.1080/10826076.2018.1511995</pub-id></mixed-citation></ref><ref id=\"CR5\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name><name><surname>Hassan</surname><given-names>Z</given-names></name><name><surname>Kabbashi</surname><given-names>NA</given-names></name></person-group><article-title xml:lang=\"en\">Dataset on LC-Q-TOF/MS tentative identification of phytochemicals in the extract of <italic>Vernonia amygdalina</italic> leaf through positive ionization</article-title><source>Data Br</source><year>2018</year><volume>21</volume><fpage>1686</fpage><lpage>1689</lpage><pub-id pub-id-type=\"doi\">10.1016/j.dib.2018.10.159</pub-id></mixed-citation></ref><ref id=\"CR6\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ali</surname><given-names>HA</given-names></name><name><surname>Zain</surname><given-names>NM</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from Carica papaya leaves: an optimization study and LC-QTOF-MS analysis</article-title><source>Futur Foods</source><year>2021</year><volume>3</volume><fpage>100035</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XhtlSmurnO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.fufo.2021.100035</pub-id></mixed-citation></ref><ref id=\"CR7\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Alara</surname><given-names>OR</given-names></name><name><surname>Abdurahman</surname><given-names>NH</given-names></name><name><surname>Ukaegbu</surname><given-names>CI</given-names></name></person-group><article-title xml:lang=\"en\">Extraction of phenolic compounds: a review</article-title><source>Curr Res Food Sci</source><year>2021</year><volume>4</volume><fpage>200</fpage><lpage>214</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXht1Cjsb%2FK</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.crfs.2021.03.011</pub-id><pub-id pub-id-type=\"pmid\">33899007</pub-id><pub-id pub-id-type=\"pmcid\">8058613</pub-id></mixed-citation></ref><ref id=\"CR8\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ali</surname><given-names>MM</given-names></name><name><surname>Ahmed</surname><given-names>KS</given-names></name><name><surname>Hossain</surname><given-names>H</given-names></name><name><surname>Roy</surname><given-names>B</given-names></name><name><surname>Rokeya</surname><given-names>B</given-names></name><name><surname>Rahman</surname><given-names>MT</given-names></name><name><surname>Jahan</surname><given-names>IA</given-names></name><name><surname>Rahman</surname><given-names>MM</given-names></name></person-group><article-title xml:lang=\"en\">Total antioxidant capacity and profiling of polyphenolic compounds in jute leaves by hplc-dad</article-title><source>Food Res</source><year>2021</year><volume>5</volume><fpage>343</fpage><lpage>348</lpage><pub-id pub-id-type=\"doi\">10.26656/fr.2017.5(1).358</pub-id></mixed-citation></ref><ref id=\"CR9\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Andersen</surname><given-names>LT</given-names></name><name><surname>Thilsted</surname><given-names>SH</given-names></name><name><surname>Nielsen</surname><given-names>BB</given-names></name><name><surname>Rangasamy</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Food and nutrient intakes among pregnant women in rural Tamil Nadu</article-title><source>South India Public Heal Nutr</source><year>2003</year><volume>6</volume><fpage>131</fpage><lpage>137</lpage><pub-id pub-id-type=\"doi\">10.1079/PHN2002367</pub-id></mixed-citation></ref><ref id=\"CR10\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Azwanida</surname><given-names>NN</given-names></name></person-group><article-title xml:lang=\"en\">A review on the extraction methods use in medicinal plants, principle</article-title><source>Strength Limit Med Aromat Plants</source><year>2015</year><volume>04</volume><fpage>3</fpage><lpage>8</lpage><pub-id pub-id-type=\"doi\">10.4172/2167-0412.1000196</pub-id></mixed-citation></ref><ref id=\"CR11\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Bouras</surname><given-names>M</given-names></name><name><surname>Chadni</surname><given-names>M</given-names></name><name><surname>Barba</surname><given-names>FJ</given-names></name><name><surname>Grimi</surname><given-names>N</given-names></name><name><surname>Bals</surname><given-names>O</given-names></name><name><surname>Vorobiev</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Optimization of microwave-assisted extraction of polyphenols from Quercus bark</article-title><source>Ind Crops Prod</source><year>2015</year><volume>77</volume><fpage>590</fpage><lpage>601</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2MXhsF2gt73F</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2015.09.018</pub-id></mixed-citation></ref><ref id=\"CR12\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Che Sulaiman</surname><given-names>IS</given-names></name><name><surname>Basri</surname><given-names>M</given-names></name><name><surname>Fard Masoumi</surname><given-names>HR</given-names></name><name><surname>Chee</surname><given-names>WJ</given-names></name><name><surname>Ashari</surname><given-names>SE</given-names></name><name><surname>Ismail</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Effects of temperature, time, and solvent ratio on the extraction of phenolic compounds and the anti-radical activity of <italic>Clinacanthus nutans</italic> Lindau leaves by response surface methodology</article-title><source>Chem Cent J</source><year>2017</year><volume>11</volume><fpage>1</fpage><lpage>11</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXitVShtLzE</pub-id><pub-id pub-id-type=\"doi\">10.1186/s13065-017-0285-1</pub-id></mixed-citation></ref><ref id=\"CR13\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Cheynier</surname><given-names>V</given-names></name></person-group><article-title xml:lang=\"en\">Phenolic compounds: from plants to foods</article-title><source>Phytochem Rev</source><year>2012</year><volume>11</volume><fpage>153</fpage><lpage>177</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXktFeqsro%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11101-012-9242-8</pub-id></mixed-citation></ref><ref id=\"CR14\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Guzzetti</surname><given-names>L</given-names></name><name><surname>Panzeri</surname><given-names>D</given-names></name><name><surname>Ulaszewska</surname><given-names>M</given-names></name><name><surname>Sacco</surname><given-names>G</given-names></name><name><surname>Forcella</surname><given-names>M</given-names></name><name><surname>Fusi</surname><given-names>P</given-names></name><name><surname>Tommasi</surname><given-names>N</given-names></name><name><surname>Fiorini</surname><given-names>A</given-names></name><name><surname>Campone</surname><given-names>L</given-names></name><name><surname>Labra</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Assessment of dietary bioactive phenolic compounds and agricultural sustainability of an African leafy vegetable <italic>Corchorus olitorius</italic> L</article-title><source>Front Nutr</source><year>2021</year><volume>8</volume><fpage>1</fpage><lpage>10</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitFWqsrzI</pub-id><pub-id pub-id-type=\"doi\">10.3389/fnut.2021.667812</pub-id></mixed-citation></ref><ref id=\"CR15\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Halee</surname><given-names>A</given-names></name><name><surname>Supavititpatana</surname><given-names>P</given-names></name><name><surname>Ruttarattanamongkol</surname><given-names>K</given-names></name><name><surname>Jittrepotch</surname><given-names>N</given-names></name><name><surname>Rojsuntornkitti</surname><given-names>K</given-names></name><name><surname>Kongbangkerd</surname><given-names>T</given-names></name></person-group><article-title xml:lang=\"en\">Optimisation of the microwave-assisted extraction of natural antioxidants from defatted black rice bran of <italic>oryza sativa</italic> L. CV. homnin</article-title><source>J Microbiol Biotechnol Food Sci</source><year>2020</year><volume>9</volume><fpage>1134</fpage><lpage>1140</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhvFOksrrP</pub-id><pub-id pub-id-type=\"doi\">10.15414/JMBFS.2020.9.6.1134-1140</pub-id></mixed-citation></ref><ref id=\"CR16\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Herrero</surname><given-names>M</given-names></name><name><surname>Plaza</surname><given-names>M</given-names></name><name><surname>Cifuentes</surname><given-names>A</given-names></name><name><surname>Ibáñez</surname><given-names>E</given-names></name></person-group><article-title xml:lang=\"en\">Extraction techniques for the determination of phenolic compounds in food</article-title><source>Compr Sampl Sample Prep</source><year>2012</year><volume>4</volume><fpage>159</fpage><lpage>180</lpage><pub-id pub-id-type=\"doi\">10.1016/B978-0-12-381373-2.00132-0</pub-id></mixed-citation></ref><ref id=\"CR17\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Kamga</surname><given-names>RT</given-names></name><name><surname>Kouamé</surname><given-names>C</given-names></name><name><surname>Atangana</surname><given-names>AR</given-names></name><name><surname>Chagomoka</surname><given-names>T</given-names></name><name><surname>Ndango</surname><given-names>R</given-names></name></person-group><article-title xml:lang=\"en\">Nutritional evaluation of five African indigenous vegetables</article-title><source>J Hortic Res</source><year>2013</year><volume>21</volume><fpage>99</fpage><lpage>106</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2cXht1SqsL%2FL</pub-id><pub-id pub-id-type=\"doi\">10.2478/johr-2013-0014</pub-id></mixed-citation></ref><ref id=\"CR18\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Khoddami</surname><given-names>A</given-names></name><name><surname>Wilkes</surname><given-names>MA</given-names></name><name><surname>Roberts</surname><given-names>TH</given-names></name></person-group><article-title xml:lang=\"en\">Techniques for analysis of plant phenolic compounds</article-title><source>Molecules</source><year>2013</year><volume>18</volume><fpage>2328</fpage><lpage>2375</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC3sXjs12htrw%3D</pub-id><pub-id pub-id-type=\"doi\">10.3390/molecules18022328</pub-id><pub-id pub-id-type=\"pmid\">23429347</pub-id><pub-id pub-id-type=\"pmcid\">6270361</pub-id></mixed-citation></ref><ref id=\"CR19\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lachowicz</surname><given-names>S</given-names></name><name><surname>Oszmiański</surname><given-names>J</given-names></name><name><surname>Rapak</surname><given-names>A</given-names></name><name><surname>Ochmian</surname><given-names>I</given-names></name></person-group><article-title xml:lang=\"en\">Profile and content of phenolic compounds in leaves, flowers, roots, and stalks of <italic>Sanguisorba officinalis</italic> L. Determined with the LC-DAD-ESI- QTOF-MS/MS analysis and their in vitro antioxidant, antidiabetic, antiproliferative potency</article-title><source>Pharmaceuticals</source><year>2020</year><volume>13</volume><fpage>1</fpage><lpage>22</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitVCgurrP</pub-id><pub-id pub-id-type=\"doi\">10.3390/ph13080191</pub-id></mixed-citation></ref><ref id=\"CR20\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Lobo</surname><given-names>V</given-names></name><name><surname>Patil</surname><given-names>A</given-names></name><name><surname>Phatak</surname><given-names>A</given-names></name><name><surname>Chandra</surname><given-names>N</given-names></name></person-group><article-title xml:lang=\"en\">Free radicals, antioxidants and functional foods: impact on human health</article-title><source>Pharmacogn Rev</source><year>2010</year><volume>4</volume><fpage>118</fpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:STN:280:DC%2BC387ivVOgtA%3D%3D</pub-id><pub-id pub-id-type=\"doi\">10.4103/0973-7847.70902</pub-id><pub-id pub-id-type=\"pmid\">22228951</pub-id><pub-id pub-id-type=\"pmcid\">3249911</pub-id></mixed-citation></ref><ref id=\"CR21\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Loumerem</surname><given-names>M</given-names></name><name><surname>Alercia</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Descriptors for jute (<italic>Corchorus olitorius</italic> L.)</article-title><source>Genet Resour Crop Evol</source><year>2016</year><volume>63</volume><fpage>1103</fpage><lpage>1111</lpage><pub-id pub-id-type=\"doi\">10.1007/s10722-016-0415-y</pub-id></mixed-citation></ref><ref id=\"CR22\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mahmud</surname><given-names>IA</given-names></name><name><surname>Mirghani</surname><given-names>SME</given-names></name><name><surname>Yusof</surname><given-names>F</given-names></name><name><surname>Al-khatib</surname><given-names>A</given-names></name></person-group><article-title xml:lang=\"en\">Effects of time, temperature, and solvent ratio on the extraction of non-extractable polyphenols with anticancer activity of barhi date palm kernels extracts using response surface methodology</article-title><source>J. Biotechnol. Eng.</source><year>2019</year><volume>1</volume><fpage>1</fpage><lpage>36</lpage><pub-id pub-id-type=\"doi\">10.20944/preprints201907.0055.v1</pub-id></mixed-citation></ref><ref id=\"CR23\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Mandal</surname><given-names>V</given-names></name><name><surname>Mohan</surname><given-names>Y</given-names></name><name><surname>Hemalath</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">Microwave assisted extraction-an innovative and promising extraction tool for medicinal plant research</article-title><source>Pharmacogn Rev</source><year>2007</year><volume>1</volume><fpage>7</fpage><lpage>18</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD2sXhtVSit7zI</pub-id></mixed-citation></ref><ref id=\"CR24\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Ndlovu</surname><given-names>S</given-names></name><name><surname>Pullabhotla</surname><given-names>RVSR</given-names></name><name><surname>Ntuli</surname><given-names>NR</given-names></name></person-group><article-title xml:lang=\"en\">Response of <italic>Corchorus olitorius</italic> leafy vegetable to cadmium in the soil</article-title><source>Plants</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>13</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXitFGnsLbE</pub-id><pub-id pub-id-type=\"doi\">10.3390/plants9091200</pub-id></mixed-citation></ref><ref id=\"CR25\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Obeng</surname><given-names>E</given-names></name><name><surname>Kpodo</surname><given-names>FM</given-names></name><name><surname>Tettey</surname><given-names>CO</given-names></name><name><surname>Essuman</surname><given-names>EK</given-names></name><name><surname>Adzinyo</surname><given-names>OA</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant total phenols and proximate constituents of four tropical leafy vegetables</article-title><source>Sci African</source><year>2020</year><volume>7</volume><fpage>e00227</fpage><pub-id pub-id-type=\"doi\">10.1016/j.sciaf.2019.e00227</pub-id></mixed-citation></ref><ref id=\"CR26\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Quoc</surname><given-names>LPT</given-names></name></person-group><article-title xml:lang=\"en\">Microwave-assisted extraction of phenolic compounds from coffee (<italic>Coffea robusta</italic> L. Linden) bee pollen</article-title><source>Herb Pol</source><year>2021</year><volume>67</volume><fpage>37</fpage><lpage>44</lpage><pub-id pub-id-type=\"doi\">10.2478/hepo-2021-0015</pub-id></mixed-citation></ref><ref id=\"CR27\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Racha</surname><given-names>A</given-names></name><name><surname>Yakoub</surname><given-names>O</given-names></name><name><surname>Abdehedi</surname><given-names>O</given-names></name></person-group><article-title xml:lang=\"en\">Flavonoids, phenols, antioxidant, and antimicrobial activities in various extracts from Tossa jute leave (<italic>Corchorus olitorus</italic> L.)</article-title><source>Ind Crops Prod</source><year>2018</year><volume>118</volume><fpage>206</fpage><lpage>213</lpage><pub-id pub-id-type=\"doi\">10.1016/j.indcrop.2018.03.047</pub-id></mixed-citation></ref><ref id=\"CR28\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Saliu</surname><given-names>JA</given-names></name><name><surname>Ademiluyi</surname><given-names>AO</given-names></name><name><surname>Boligon</surname><given-names>AA</given-names></name><name><surname>Oboh</surname><given-names>G</given-names></name><name><surname>Schetinger</surname><given-names>MRC</given-names></name><name><surname>Rocha</surname><given-names>JBT</given-names></name></person-group><article-title xml:lang=\"en\">Dietary supplementation of jute leaf (<italic>Corchorus olitorius</italic>) modulates hepatic delta-aminolevulinic acid dehydratase (δ-ALAD) activity and oxidative status in high-fat fed/low streptozotocin-induced diabetic rats</article-title><source>J Food Biochem</source><year>2019</year><volume>43</volume><fpage>e12949</fpage><pub-id pub-id-type=\"doi\">10.1111/jfbc.12949</pub-id><pub-id pub-id-type=\"pmid\">31368580</pub-id></mixed-citation></ref><ref id=\"CR29\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tan</surname><given-names>HM</given-names></name><name><surname>Leong</surname><given-names>KH</given-names></name><name><surname>Song</surname><given-names>J</given-names></name><name><surname>Mohd Sufian</surname><given-names>NSF</given-names></name><name><surname>Mohd Hazli</surname><given-names>UHA</given-names></name><name><surname>Chew</surname><given-names>LY</given-names></name><name><surname>Kong</surname><given-names>KW</given-names></name></person-group><article-title xml:lang=\"en\">Antioxidant and LC-QToF-MS/MS analysis of polyphenols in polar and non-polar extracts from Strobilanthes crispus and Clinacanthus nutans</article-title><source>Int Food Res J</source><year>2020</year><volume>27</volume><fpage>903</fpage><lpage>914</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3MXitlynu7s%3D</pub-id></mixed-citation></ref><ref id=\"CR30\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tang</surname><given-names>J</given-names></name><name><surname>Dunshea</surname><given-names>FR</given-names></name><name><surname>Suleria</surname><given-names>HAR</given-names></name></person-group><article-title xml:lang=\"en\">LC-ESI-QTOF/MS characterization of phenolic compounds from medicinal plants (Hops and Juniper Berries) and their antioxidant activity</article-title><source>Foods</source><year>2020</year><volume>9</volume><fpage>1</fpage><lpage>25</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB3cXhtVektLnE</pub-id><pub-id pub-id-type=\"doi\">10.3390/foods9010007</pub-id></mixed-citation></ref><ref id=\"CR31\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Tchabo</surname><given-names>W</given-names></name><name><surname>Ma</surname><given-names>Y</given-names></name><name><surname>Kwaw</surname><given-names>E</given-names></name><name><surname>Xiao</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>M</given-names></name><name><surname>Apaliya</surname><given-names>MT</given-names></name></person-group><article-title xml:lang=\"en\">Impact of extraction parameters and their optimization on the nutraceuticals and antioxidant properties of aqueous extract mulberry leaf</article-title><source>Int J Food Prop</source><year>2018</year><volume>21</volume><fpage>717</fpage><lpage>732</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1cXovFyrsb4%3D</pub-id><pub-id pub-id-type=\"doi\">10.1080/10942912.2018.1446025</pub-id></mixed-citation></ref><ref id=\"CR32\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Thoo</surname><given-names>YY</given-names></name><name><surname>Ho</surname><given-names>SK</given-names></name><name><surname>Liang</surname><given-names>JY</given-names></name><name><surname>Ho</surname><given-names>CW</given-names></name><name><surname>Tan</surname><given-names>CP</given-names></name></person-group><article-title xml:lang=\"en\">Effects of binary solvent extraction system, extraction time and extraction temperature on phenolic antioxidants and antioxidant capacity from mengkudu (<italic>Morinda citrifolia</italic>)</article-title><source>Food Chem</source><year>2010</year><volume>120</volume><fpage>290</fpage><lpage>295</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXhsFClu7zO</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2009.09.064</pub-id></mixed-citation></ref><ref id=\"CR33\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Hwang</surname><given-names>SH</given-names></name><name><surname>Quispe</surname><given-names>YNG</given-names></name><name><surname>Arce</surname><given-names>PHG</given-names></name><name><surname>Lim</surname><given-names>SS</given-names></name></person-group><article-title xml:lang=\"en\">Investigation of the antioxidant and aldose reductase inhibitory activities of extracts from Peruvian tea plant infusions</article-title><source>Food Chem</source><year>2017</year><volume>231</volume><fpage>222</fpage><lpage>230</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC2sXltlOnurw%3D</pub-id><pub-id pub-id-type=\"doi\">10.1016/j.foodchem.2017.03.107</pub-id><pub-id pub-id-type=\"pmid\">28450000</pub-id></mixed-citation></ref><ref id=\"CR34\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Xu</surname><given-names>Q</given-names></name><name><surname>Fu</surname><given-names>Q</given-names></name><name><surname>Li</surname><given-names>Z</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>R</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Ju</surname><given-names>Z</given-names></name><name><surname>Campisi</surname><given-names>J</given-names></name><name><surname>Kirkland</surname><given-names>JL</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name></person-group><article-title xml:lang=\"en\">The flavonoid procyanidin C1 has senotherapeutic activity and increases lifespan in mice</article-title><source>Nat Metab</source><year>2021</year><volume>3</volume><fpage>1706</fpage><lpage>1726</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BB38XotFKmsLg%3D</pub-id><pub-id pub-id-type=\"doi\">10.1038/s42255-021-00491-8</pub-id><pub-id pub-id-type=\"pmid\">34873338</pub-id><pub-id pub-id-type=\"pmcid\">8688144</pub-id></mixed-citation></ref><ref id=\"CR35\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Youssef</surname><given-names>AF</given-names></name><name><surname>Younes</surname><given-names>NA</given-names></name><name><surname>Youssef</surname><given-names>M</given-names></name></person-group><article-title xml:lang=\"en\">Genetic diversity in <italic>Corchorus olitorius</italic> L. revealed by morphophysiological and molecular analyses</article-title><source>Mol Biol Rep</source><year>2019</year><volume>46</volume><fpage>2933</fpage><lpage>2940</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BC1MXmsFCnu7g%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11033-019-04754-2</pub-id><pub-id pub-id-type=\"pmid\">30887258</pub-id></mixed-citation></ref><ref id=\"CR36\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Biological effects of tetrahydroxystilbene glucoside: an active component of a rhizome extracted from <italic>Polygonum multiflorum</italic></article-title><source>Oxid Med Cell Longev</source><year>2018</year><volume>2018</volume><fpage>3641960</fpage><pub-id pub-id-type=\"doi\">10.1155/2018/3641960</pub-id><pub-id pub-id-type=\"pmid\">30524653</pub-id><pub-id pub-id-type=\"pmcid\">6247474</pub-id></mixed-citation></ref><ref id=\"CR37\"><mixed-citation publication-type=\"journal\"><person-group person-group-type=\"author\"><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Kim</surname><given-names>SH</given-names></name><name><surname>Hagerman</surname><given-names>AE</given-names></name><name><surname>Lü</surname><given-names>J</given-names></name></person-group><article-title xml:lang=\"en\">Anti-cancer, anti-diabetic and other pharmacologic and biological activities of penta-galloyl-glucose</article-title><source>Pharm Res</source><year>2009</year><volume>26</volume><fpage>2066</fpage><lpage>2080</lpage><pub-id assigning-authority=\"ChemPort ( Chemical Abstract Service )\" pub-id-type=\"other\">1:CAS:528:DC%2BD1MXptlaku7w%3D</pub-id><pub-id pub-id-type=\"doi\">10.1007/s11095-009-9932-0</pub-id><pub-id pub-id-type=\"pmid\">19575286</pub-id><pub-id pub-id-type=\"pmcid\">2822717</pub-id></mixed-citation></ref></ref-list></ref-list><app-group><app id=\"App1\" specific-use=\"web-only\"><sec id=\"Sec25\"><title>Supplementary Information</title><p id=\"Par49\">Below is the link to the electronic supplementary material.<supplementary-material content-type=\"local-data\" id=\"MOESM1\" xlink:title=\"Supplementary Information\"><media mime-subtype=\"msword\" mimetype=\"application\" position=\"anchor\" xlink:href=\"MediaObjects/11696_2023_2771_MOESM1_ESM.docx\"><caption xml:lang=\"en\"><p>Supplementary file1 (DOCX 934 KB)</p></caption></media></supplementary-material></p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>ABTS</term><def><p id=\"Par2\">2,2′-Azino-bis(3-ethylbenzothiazoline-6′-sulfonic acid)</p></def></def-item><def-item><term>ANOVA</term><def><p id=\"Par3\">Analysis of variance analysis</p></def></def-item><def-item><term>DNA</term><def><p id=\"Par4\">Deoxyribonucleic acid</p></def></def-item><def-item><term>DPPH</term><def><p id=\"Par5\">2,2-Diphenyl-1-picrylhydrazyl</p></def></def-item><def-item><term>FCCCD</term><def><p id=\"Par6\">Face-centred central composite design</p></def></def-item><def-item><term>LC-QToF-MS</term><def><p id=\"Par7\">Liquid chromatography-mass spectrometry quadrupole of flight</p></def></def-item><def-item><term>OFAT</term><def><p id=\"Par8\">One-factor-at-a-time</p></def></def-item><def-item><term>TPC</term><def><p id=\"Par9\">Total phenolic content</p></def></def-item><def-item><term>RSM</term><def><p id=\"Par10\">Response surface methodology</p></def></def-item><def-item><term>SDG 2</term><def><p id=\"Par11\">Sustainable development goals</p></def></def-item></def-list></glossary><notes notes-type=\"Misc\"><title>Publisher's Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article><article article-type=\"correction\" dtd-version=\"1.2\" xml:lang=\"en\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><front><journal-meta><journal-id journal-id-type=\"publisher-id\">10934</journal-id><journal-id journal-id-type=\"doi\">10.1007/10934.1573-4854</journal-id><journal-title-group><journal-title>Journal of Porous Materials</journal-title><abbrev-journal-title abbrev-type=\"publisher\">J Porous Mater</abbrev-journal-title></journal-title-group><issn pub-type=\"ppub\">1380-2224</issn><issn pub-type=\"epub\">1573-4854</issn><publisher><publisher-name>Springer US</publisher-name><publisher-loc>New York</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type=\"publisher-id\">s10934-023-01452-0</article-id><article-id pub-id-type=\"manuscript\">1452</article-id><article-id pub-id-type=\"doi\">10.1007/s10934-023-01452-0</article-id><article-categories><subj-group subj-group-type=\"heading\"><subject>Correction</subject></subj-group></article-categories><title-group><article-title xml:lang=\"en\">Correction to: Synthesis and characterization of polyurethane rigid foam by using feedstocks received from renewable and recyclable resources</article-title></title-group><contrib-group><contrib contrib-type=\"author\" id=\"Au1\"><name><surname>Sarim</surname><given-names>Muntajab</given-names></name><address><email>muntajabsarim@gmail.com</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><contrib contrib-type=\"author\" corresp=\"yes\" id=\"Au2\"><name><surname>Nikje</surname><given-names>Mir Mohammad Alavi</given-names></name><address><email>alavi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref><xref ref-type=\"corresp\" rid=\"IDs10934023014520_cor2\">b</xref></contrib><contrib contrib-type=\"author\" id=\"Au3\"><name><surname>Dargahi</surname><given-names>Maryam</given-names></name><address><email>dargahi@sci.ikiu.ac.ir</email></address><xref ref-type=\"aff\" rid=\"Aff1\">1</xref></contrib><aff id=\"Aff1\"><label>1</label><institution-wrap><institution-id institution-id-type=\"GRID\">grid.411537.5</institution-id><institution-id institution-id-type=\"ISNI\">0000 0000 8608 1112</institution-id><institution content-type=\"org-division\">Department of Chemistry, Faculty of Science</institution><institution content-type=\"org-name\">Imam Khomeini International University (IKIU)</institution></institution-wrap><addr-line content-type=\"city\">Qazvin</addr-line><country country=\"IR\">Iran</country></aff></contrib-group><author-notes><corresp id=\"IDs10934023014520_cor2\"><label>b</label><email>alavi@sci.ikiu.ac.ir</email></corresp></author-notes><pub-date date-type=\"pub\" publication-format=\"electronic\"><day>17</day><month>3</month><year>2023</year></pub-date><pub-date date-type=\"pub\" publication-format=\"print\"><month>8</month><year>2023</year></pub-date><volume>30</volume><issue seq=\"25\">4</issue><fpage>1357</fpage><lpage>1357</lpage><history><date date-type=\"registration\"><day>13</day><month>3</month><year>2023</year></date><date date-type=\"online\"><day>17</day><month>3</month><year>2023</year></date></history><permissions><copyright-statement content-type=\"compact\">© The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature 2023</copyright-statement><copyright-year>2023</copyright-year><copyright-holder>The Author(s), under exclusive licence to Springer Science+Business Media, LLC, part of Springer Nature</copyright-holder></permissions><related-article ext-link-type=\"doi\" related-article-type=\"corrected-article\" xlink:href=\"10.1007/s10934-023-01425-3\"/><custom-meta-group><custom-meta><meta-name>publisher-imprint-name</meta-name><meta-value>Springer</meta-value></custom-meta><custom-meta><meta-name>volume-issue-count</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-article-count</meta-name><meta-value>31</meta-value></custom-meta><custom-meta><meta-name>issue-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>issue-pricelist-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-holder</meta-name><meta-value>Springer Science+Business Media, LLC, part of Springer Nature</meta-value></custom-meta><custom-meta><meta-name>issue-copyright-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-contains-esm</meta-name><meta-value>No</meta-value></custom-meta><custom-meta><meta-name>article-numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-month</meta-name><meta-value>3</meta-value></custom-meta><custom-meta><meta-name>article-registration-date-day</meta-name><meta-value>13</meta-value></custom-meta><custom-meta><meta-name>article-toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>toc-levels</meta-name><meta-value>0</meta-value></custom-meta><custom-meta><meta-name>volume-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>journal-product</meta-name><meta-value>ArchiveJournal</meta-value></custom-meta><custom-meta><meta-name>numbering-style</meta-name><meta-value>ContentOnly</meta-value></custom-meta><custom-meta><meta-name>article-grants-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>metadata-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>abstract-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodypdf-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bodyhtml-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>bibliography-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>esm-grant</meta-name><meta-value>OpenAccess</meta-value></custom-meta><custom-meta><meta-name>online-first</meta-name><meta-value>false</meta-value></custom-meta><custom-meta><meta-name>pdf-file-reference</meta-name><meta-value>BodyRef/PDF/10934_2023_Article_1452.pdf</meta-value></custom-meta><custom-meta><meta-name>output-medium</meta-name><meta-value>All</meta-value></custom-meta><custom-meta><meta-name>pdf-type</meta-name><meta-value>Typeset</meta-value></custom-meta><custom-meta><meta-name>target-type</meta-name><meta-value>OnlinePDF</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-online-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-year</meta-name><meta-value>2023</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-month</meta-name><meta-value>6</meta-value></custom-meta><custom-meta><meta-name>issue-print-date-day</meta-name><meta-value>27</meta-value></custom-meta><custom-meta><meta-name>issue-type</meta-name><meta-value>Regular</meta-value></custom-meta><custom-meta><meta-name>article-type</meta-name><meta-value>Erratum</meta-value></custom-meta><custom-meta><meta-name>journal-subject-primary</meta-name><meta-value>Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Catalysis</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Characterization and Evaluation of Materials</meta-value></custom-meta><custom-meta><meta-name>journal-subject-secondary</meta-name><meta-value>Physical Chemistry</meta-value></custom-meta><custom-meta><meta-name>journal-subject-collection</meta-name><meta-value>Chemistry and Materials Science</meta-value></custom-meta><custom-meta><meta-name>open-access</meta-name><meta-value>false</meta-value></custom-meta></custom-meta-group></article-meta><notes notes-type=\"Misc\"><p>The online version of the original article can be found at <ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p></notes></front><body><p id=\"Par1\">Correction to: <bold>Journal of Porous Materials</bold></p><p id=\"Par2\"><ext-link ext-link-type=\"doi\" xlink:href=\"10.1007/s10934-023-01425-3\">https://doi.org/10.1007/s10934-023-01425-3</ext-link></p><p id=\"Par6\">In this article, the author name Maryam Dargah has been corrected to Maryam Dargahi.</p><p id=\"Par7\">The original article has been corrected.</p></body><back><notes notes-type=\"Misc\"><title>Publisher’s Note</title><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></notes></back></article></records>"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"records\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n",
      "<meta-value>Regular</meta-value>\n"
     ]
    }
   ],
   "source": [
    "a = soup.find_all('meta-name', string='issue-type')\n",
    "for i in a:\n",
    "    print(i.find_next_sibling())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
